{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "id": "UPTz0bb4vaDW",
    "outputId": "5b7716c2-4d4d-49c0-ff17-4a54ff726ff2"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting vllm==0.3.3\n",
      "  Downloading vllm-0.3.3-cp310-cp310-manylinux1_x86_64.whl (44.3 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m44.3/44.3 MB\u001b[0m \u001b[31m5.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting ninja (from vllm==0.3.3)\n",
      "  Downloading ninja-1.11.1.1-py2.py3-none-manylinux1_x86_64.manylinux_2_5_x86_64.whl (307 kB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m307.2/307.2 kB\u001b[0m \u001b[31m27.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hRequirement already satisfied: psutil in /usr/local/lib/python3.10/dist-packages (from vllm==0.3.3) (5.9.5)\n",
      "Collecting ray>=2.9 (from vllm==0.3.3)\n",
      "  Downloading ray-2.10.0-cp310-cp310-manylinux2014_x86_64.whl (65.1 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m65.1/65.1 MB\u001b[0m \u001b[31m24.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hRequirement already satisfied: sentencepiece in /usr/local/lib/python3.10/dist-packages (from vllm==0.3.3) (0.1.99)\n",
      "Requirement already satisfied: numpy in /usr/local/lib/python3.10/dist-packages (from vllm==0.3.3) (1.25.2)\n",
      "Collecting torch==2.1.2 (from vllm==0.3.3)\n",
      "  Downloading torch-2.1.2-cp310-cp310-manylinux1_x86_64.whl (670.2 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m670.2/670.2 MB\u001b[0m \u001b[31m1.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hRequirement already satisfied: transformers>=4.38.0 in /usr/local/lib/python3.10/dist-packages (from vllm==0.3.3) (4.38.2)\n",
      "Collecting xformers==0.0.23.post1 (from vllm==0.3.3)\n",
      "  Downloading xformers-0.0.23.post1-cp310-cp310-manylinux2014_x86_64.whl (213.0 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m213.0/213.0 MB\u001b[0m \u001b[31m7.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting fastapi (from vllm==0.3.3)\n",
      "  Downloading fastapi-0.110.1-py3-none-any.whl (91 kB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m91.9/91.9 kB\u001b[0m \u001b[31m13.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting uvicorn[standard] (from vllm==0.3.3)\n",
      "  Downloading uvicorn-0.29.0-py3-none-any.whl (60 kB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m60.8/60.8 kB\u001b[0m \u001b[31m9.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hRequirement already satisfied: pydantic>=2.0 in /usr/local/lib/python3.10/dist-packages (from vllm==0.3.3) (2.6.4)\n",
      "Requirement already satisfied: prometheus-client>=0.18.0 in /usr/local/lib/python3.10/dist-packages (from vllm==0.3.3) (0.20.0)\n",
      "Collecting pynvml==11.5.0 (from vllm==0.3.3)\n",
      "  Downloading pynvml-11.5.0-py3-none-any.whl (53 kB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m53.1/53.1 kB\u001b[0m \u001b[31m7.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hRequirement already satisfied: triton>=2.1.0 in /usr/local/lib/python3.10/dist-packages (from vllm==0.3.3) (2.2.0)\n",
      "Collecting outlines>=0.0.27 (from vllm==0.3.3)\n",
      "  Downloading outlines-0.0.37-py3-none-any.whl (83 kB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m83.6/83.6 kB\u001b[0m \u001b[31m11.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting cupy-cuda12x==12.1.0 (from vllm==0.3.3)\n",
      "  Downloading cupy_cuda12x-12.1.0-cp310-cp310-manylinux2014_x86_64.whl (83.0 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m83.0/83.0 MB\u001b[0m \u001b[31m18.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hRequirement already satisfied: fastrlock>=0.5 in /usr/local/lib/python3.10/dist-packages (from cupy-cuda12x==12.1.0->vllm==0.3.3) (0.8.2)\n",
      "Requirement already satisfied: filelock in /usr/local/lib/python3.10/dist-packages (from torch==2.1.2->vllm==0.3.3) (3.13.3)\n",
      "Requirement already satisfied: typing-extensions in /usr/local/lib/python3.10/dist-packages (from torch==2.1.2->vllm==0.3.3) (4.10.0)\n",
      "Requirement already satisfied: sympy in /usr/local/lib/python3.10/dist-packages (from torch==2.1.2->vllm==0.3.3) (1.12)\n",
      "Requirement already satisfied: networkx in /usr/local/lib/python3.10/dist-packages (from torch==2.1.2->vllm==0.3.3) (3.2.1)\n",
      "Requirement already satisfied: jinja2 in /usr/local/lib/python3.10/dist-packages (from torch==2.1.2->vllm==0.3.3) (3.1.3)\n",
      "Requirement already satisfied: fsspec in /usr/local/lib/python3.10/dist-packages (from torch==2.1.2->vllm==0.3.3) (2023.6.0)\n",
      "Collecting nvidia-cuda-nvrtc-cu12==12.1.105 (from torch==2.1.2->vllm==0.3.3)\n",
      "  Downloading nvidia_cuda_nvrtc_cu12-12.1.105-py3-none-manylinux1_x86_64.whl (23.7 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m23.7/23.7 MB\u001b[0m \u001b[31m65.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting nvidia-cuda-runtime-cu12==12.1.105 (from torch==2.1.2->vllm==0.3.3)\n",
      "  Downloading nvidia_cuda_runtime_cu12-12.1.105-py3-none-manylinux1_x86_64.whl (823 kB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m823.6/823.6 kB\u001b[0m \u001b[31m67.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting nvidia-cuda-cupti-cu12==12.1.105 (from torch==2.1.2->vllm==0.3.3)\n",
      "  Downloading nvidia_cuda_cupti_cu12-12.1.105-py3-none-manylinux1_x86_64.whl (14.1 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m14.1/14.1 MB\u001b[0m \u001b[31m88.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting nvidia-cudnn-cu12==8.9.2.26 (from torch==2.1.2->vllm==0.3.3)\n",
      "  Downloading nvidia_cudnn_cu12-8.9.2.26-py3-none-manylinux1_x86_64.whl (731.7 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m731.7/731.7 MB\u001b[0m \u001b[31m1.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting nvidia-cublas-cu12==12.1.3.1 (from torch==2.1.2->vllm==0.3.3)\n",
      "  Downloading nvidia_cublas_cu12-12.1.3.1-py3-none-manylinux1_x86_64.whl (410.6 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m410.6/410.6 MB\u001b[0m \u001b[31m2.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting nvidia-cufft-cu12==11.0.2.54 (from torch==2.1.2->vllm==0.3.3)\n",
      "  Downloading nvidia_cufft_cu12-11.0.2.54-py3-none-manylinux1_x86_64.whl (121.6 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m121.6/121.6 MB\u001b[0m \u001b[31m13.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting nvidia-curand-cu12==10.3.2.106 (from torch==2.1.2->vllm==0.3.3)\n",
      "  Downloading nvidia_curand_cu12-10.3.2.106-py3-none-manylinux1_x86_64.whl (56.5 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m56.5/56.5 MB\u001b[0m \u001b[31m29.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting nvidia-cusolver-cu12==11.4.5.107 (from torch==2.1.2->vllm==0.3.3)\n",
      "  Downloading nvidia_cusolver_cu12-11.4.5.107-py3-none-manylinux1_x86_64.whl (124.2 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m124.2/124.2 MB\u001b[0m \u001b[31m11.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting nvidia-cusparse-cu12==12.1.0.106 (from torch==2.1.2->vllm==0.3.3)\n",
      "  Downloading nvidia_cusparse_cu12-12.1.0.106-py3-none-manylinux1_x86_64.whl (196.0 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m196.0/196.0 MB\u001b[0m \u001b[31m8.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting nvidia-nccl-cu12==2.18.1 (from torch==2.1.2->vllm==0.3.3)\n",
      "  Downloading nvidia_nccl_cu12-2.18.1-py3-none-manylinux1_x86_64.whl (209.8 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m209.8/209.8 MB\u001b[0m \u001b[31m4.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting nvidia-nvtx-cu12==12.1.105 (from torch==2.1.2->vllm==0.3.3)\n",
      "  Downloading nvidia_nvtx_cu12-12.1.105-py3-none-manylinux1_x86_64.whl (99 kB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m99.1/99.1 kB\u001b[0m \u001b[31m15.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting triton>=2.1.0 (from vllm==0.3.3)\n",
      "  Downloading triton-2.1.0-0-cp310-cp310-manylinux2014_x86_64.manylinux_2_17_x86_64.whl (89.2 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m89.2/89.2 MB\u001b[0m \u001b[31m19.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting nvidia-nvjitlink-cu12 (from nvidia-cusolver-cu12==11.4.5.107->torch==2.1.2->vllm==0.3.3)\n",
      "  Downloading nvidia_nvjitlink_cu12-12.4.127-py3-none-manylinux2014_x86_64.whl (21.1 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m21.1/21.1 MB\u001b[0m \u001b[31m81.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting interegular (from outlines>=0.0.27->vllm==0.3.3)\n",
      "  Downloading interegular-0.3.3-py37-none-any.whl (23 kB)\n",
      "Collecting lark (from outlines>=0.0.27->vllm==0.3.3)\n",
      "  Downloading lark-1.1.9-py3-none-any.whl (111 kB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m111.7/111.7 kB\u001b[0m \u001b[31m18.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hRequirement already satisfied: nest-asyncio in /usr/local/lib/python3.10/dist-packages (from outlines>=0.0.27->vllm==0.3.3) (1.6.0)\n",
      "Requirement already satisfied: cloudpickle in /usr/local/lib/python3.10/dist-packages (from outlines>=0.0.27->vllm==0.3.3) (2.2.1)\n",
      "Collecting diskcache (from outlines>=0.0.27->vllm==0.3.3)\n",
      "  Downloading diskcache-5.6.3-py3-none-any.whl (45 kB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m45.5/45.5 kB\u001b[0m \u001b[31m7.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hRequirement already satisfied: scipy in /usr/local/lib/python3.10/dist-packages (from outlines>=0.0.27->vllm==0.3.3) (1.11.4)\n",
      "Requirement already satisfied: numba in /usr/local/lib/python3.10/dist-packages (from outlines>=0.0.27->vllm==0.3.3) (0.58.1)\n",
      "Requirement already satisfied: joblib in /usr/local/lib/python3.10/dist-packages (from outlines>=0.0.27->vllm==0.3.3) (1.3.2)\n",
      "Requirement already satisfied: referencing in /usr/local/lib/python3.10/dist-packages (from outlines>=0.0.27->vllm==0.3.3) (0.34.0)\n",
      "Requirement already satisfied: jsonschema in /usr/local/lib/python3.10/dist-packages (from outlines>=0.0.27->vllm==0.3.3) (4.19.2)\n",
      "Requirement already satisfied: requests in /usr/local/lib/python3.10/dist-packages (from outlines>=0.0.27->vllm==0.3.3) (2.31.0)\n",
      "Requirement already satisfied: annotated-types>=0.4.0 in /usr/local/lib/python3.10/dist-packages (from pydantic>=2.0->vllm==0.3.3) (0.6.0)\n",
      "Requirement already satisfied: pydantic-core==2.16.3 in /usr/local/lib/python3.10/dist-packages (from pydantic>=2.0->vllm==0.3.3) (2.16.3)\n",
      "Requirement already satisfied: click>=7.0 in /usr/local/lib/python3.10/dist-packages (from ray>=2.9->vllm==0.3.3) (8.1.7)\n",
      "Requirement already satisfied: msgpack<2.0.0,>=1.0.0 in /usr/local/lib/python3.10/dist-packages (from ray>=2.9->vllm==0.3.3) (1.0.8)\n",
      "Requirement already satisfied: packaging in /usr/local/lib/python3.10/dist-packages (from ray>=2.9->vllm==0.3.3) (24.0)\n",
      "Requirement already satisfied: protobuf!=3.19.5,>=3.15.3 in /usr/local/lib/python3.10/dist-packages (from ray>=2.9->vllm==0.3.3) (3.20.3)\n",
      "Requirement already satisfied: pyyaml in /usr/local/lib/python3.10/dist-packages (from ray>=2.9->vllm==0.3.3) (6.0.1)\n",
      "Requirement already satisfied: aiosignal in /usr/local/lib/python3.10/dist-packages (from ray>=2.9->vllm==0.3.3) (1.3.1)\n",
      "Requirement already satisfied: frozenlist in /usr/local/lib/python3.10/dist-packages (from ray>=2.9->vllm==0.3.3) (1.4.1)\n",
      "Requirement already satisfied: huggingface-hub<1.0,>=0.19.3 in /usr/local/lib/python3.10/dist-packages (from transformers>=4.38.0->vllm==0.3.3) (0.20.3)\n",
      "Requirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.10/dist-packages (from transformers>=4.38.0->vllm==0.3.3) (2023.12.25)\n",
      "Requirement already satisfied: tokenizers<0.19,>=0.14 in /usr/local/lib/python3.10/dist-packages (from transformers>=4.38.0->vllm==0.3.3) (0.15.2)\n",
      "Requirement already satisfied: safetensors>=0.4.1 in /usr/local/lib/python3.10/dist-packages (from transformers>=4.38.0->vllm==0.3.3) (0.4.2)\n",
      "Requirement already satisfied: tqdm>=4.27 in /usr/local/lib/python3.10/dist-packages (from transformers>=4.38.0->vllm==0.3.3) (4.66.2)\n",
      "Collecting starlette<0.38.0,>=0.37.2 (from fastapi->vllm==0.3.3)\n",
      "  Downloading starlette-0.37.2-py3-none-any.whl (71 kB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m71.9/71.9 kB\u001b[0m \u001b[31m11.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting h11>=0.8 (from uvicorn[standard]->vllm==0.3.3)\n",
      "  Downloading h11-0.14.0-py3-none-any.whl (58 kB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m58.3/58.3 kB\u001b[0m \u001b[31m10.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting httptools>=0.5.0 (from uvicorn[standard]->vllm==0.3.3)\n",
      "  Downloading httptools-0.6.1-cp310-cp310-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (341 kB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m341.4/341.4 kB\u001b[0m \u001b[31m40.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting python-dotenv>=0.13 (from uvicorn[standard]->vllm==0.3.3)\n",
      "  Downloading python_dotenv-1.0.1-py3-none-any.whl (19 kB)\n",
      "Collecting uvloop!=0.15.0,!=0.15.1,>=0.14.0 (from uvicorn[standard]->vllm==0.3.3)\n",
      "  Downloading uvloop-0.19.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.4 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m3.4/3.4 MB\u001b[0m \u001b[31m97.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting watchfiles>=0.13 (from uvicorn[standard]->vllm==0.3.3)\n",
      "  Downloading watchfiles-0.21.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.3 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m1.3/1.3 MB\u001b[0m \u001b[31m81.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hCollecting websockets>=10.4 (from uvicorn[standard]->vllm==0.3.3)\n",
      "  Downloading websockets-12.0-cp310-cp310-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (130 kB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m130.2/130.2 kB\u001b[0m \u001b[31m20.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
      "\u001b[?25hRequirement already satisfied: anyio<5,>=3.4.0 in /usr/local/lib/python3.10/dist-packages (from starlette<0.38.0,>=0.37.2->fastapi->vllm==0.3.3) (3.7.1)\n",
      "Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from jinja2->torch==2.1.2->vllm==0.3.3) (2.1.5)\n",
      "Requirement already satisfied: attrs>=22.2.0 in /usr/local/lib/python3.10/dist-packages (from jsonschema->outlines>=0.0.27->vllm==0.3.3) (23.2.0)\n",
      "Requirement already satisfied: jsonschema-specifications>=2023.03.6 in /usr/local/lib/python3.10/dist-packages (from jsonschema->outlines>=0.0.27->vllm==0.3.3) (2023.12.1)\n",
      "Requirement already satisfied: rpds-py>=0.7.1 in /usr/local/lib/python3.10/dist-packages (from jsonschema->outlines>=0.0.27->vllm==0.3.3) (0.18.0)\n",
      "Requirement already satisfied: llvmlite<0.42,>=0.41.0dev0 in /usr/local/lib/python3.10/dist-packages (from numba->outlines>=0.0.27->vllm==0.3.3) (0.41.1)\n",
      "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests->outlines>=0.0.27->vllm==0.3.3) (3.3.2)\n",
      "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests->outlines>=0.0.27->vllm==0.3.3) (3.6)\n",
      "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests->outlines>=0.0.27->vllm==0.3.3) (2.0.7)\n",
      "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests->outlines>=0.0.27->vllm==0.3.3) (2024.2.2)\n",
      "Requirement already satisfied: mpmath>=0.19 in /usr/local/lib/python3.10/dist-packages (from sympy->torch==2.1.2->vllm==0.3.3) (1.3.0)\n",
      "Requirement already satisfied: sniffio>=1.1 in /usr/local/lib/python3.10/dist-packages (from anyio<5,>=3.4.0->starlette<0.38.0,>=0.37.2->fastapi->vllm==0.3.3) (1.3.1)\n",
      "Requirement already satisfied: exceptiongroup in /usr/local/lib/python3.10/dist-packages (from anyio<5,>=3.4.0->starlette<0.38.0,>=0.37.2->fastapi->vllm==0.3.3) (1.2.0)\n",
      "Installing collected packages: ninja, websockets, uvloop, triton, python-dotenv, pynvml, nvidia-nvtx-cu12, nvidia-nvjitlink-cu12, nvidia-nccl-cu12, nvidia-curand-cu12, nvidia-cufft-cu12, nvidia-cuda-runtime-cu12, nvidia-cuda-nvrtc-cu12, nvidia-cuda-cupti-cu12, nvidia-cublas-cu12, lark, interegular, httptools, h11, diskcache, cupy-cuda12x, watchfiles, uvicorn, starlette, nvidia-cusparse-cu12, nvidia-cudnn-cu12, nvidia-cusolver-cu12, fastapi, torch, ray, xformers, outlines, vllm\n",
      "  Attempting uninstall: triton\n",
      "    Found existing installation: triton 2.2.0\n",
      "    Uninstalling triton-2.2.0:\n",
      "      Successfully uninstalled triton-2.2.0\n",
      "  Attempting uninstall: cupy-cuda12x\n",
      "    Found existing installation: cupy-cuda12x 12.2.0\n",
      "    Uninstalling cupy-cuda12x-12.2.0:\n",
      "      Successfully uninstalled cupy-cuda12x-12.2.0\n",
      "  Attempting uninstall: torch\n",
      "    Found existing installation: torch 2.2.1+cu121\n",
      "    Uninstalling torch-2.2.1+cu121:\n",
      "      Successfully uninstalled torch-2.2.1+cu121\n",
      "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
      "torchaudio 2.2.1+cu121 requires torch==2.2.1, but you have torch 2.1.2 which is incompatible.\n",
      "torchtext 0.17.1 requires torch==2.2.1, but you have torch 2.1.2 which is incompatible.\n",
      "torchvision 0.17.1+cu121 requires torch==2.2.1, but you have torch 2.1.2 which is incompatible.\u001b[0m\u001b[31m\n",
      "\u001b[0mSuccessfully installed cupy-cuda12x-12.1.0 diskcache-5.6.3 fastapi-0.110.1 h11-0.14.0 httptools-0.6.1 interegular-0.3.3 lark-1.1.9 ninja-1.11.1.1 nvidia-cublas-cu12-12.1.3.1 nvidia-cuda-cupti-cu12-12.1.105 nvidia-cuda-nvrtc-cu12-12.1.105 nvidia-cuda-runtime-cu12-12.1.105 nvidia-cudnn-cu12-8.9.2.26 nvidia-cufft-cu12-11.0.2.54 nvidia-curand-cu12-10.3.2.106 nvidia-cusolver-cu12-11.4.5.107 nvidia-cusparse-cu12-12.1.0.106 nvidia-nccl-cu12-2.18.1 nvidia-nvjitlink-cu12-12.4.127 nvidia-nvtx-cu12-12.1.105 outlines-0.0.37 pynvml-11.5.0 python-dotenv-1.0.1 ray-2.10.0 starlette-0.37.2 torch-2.1.2 triton-2.1.0 uvicorn-0.29.0 uvloop-0.19.0 vllm-0.3.3 watchfiles-0.21.0 websockets-12.0 xformers-0.0.23.post1\n"
     ]
    }
   ],
   "source": [
    "!pip install vllm==0.3.3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "id": "j1CKCFulLClk"
   },
   "outputs": [],
   "source": [
    "from vllm import LLM\n",
    "from vllm import SamplingParams\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "id": "k61J0LDGOdu9",
    "outputId": "fc419e13-3337-4534-ca36-5486dcd30859"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Mounted at /content/drive\n"
     ]
    }
   ],
   "source": [
    "from google.colab import drive\n",
    "\n",
    "drive.mount(\"/content/drive\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "id": "JetwnNLoNgYA"
   },
   "outputs": [],
   "source": [
    "mini_dev_filepath = \"/content/drive/MyDrive/BioLaySumm2024_main/data/mini_dataset/\"\n",
    "mini_dev_plos_filename = \"PLOS_val_mini_milestone3.jsonl\"\n",
    "mini_dev_elife_filename = \"eLife_val_mini_milestone3.jsonl\"\n",
    "\n",
    "full_dev_filepath = \"/content/drive/MyDrive/BioLaySumm2024_main/data/full_dev_dataset/\"\n",
    "full_dev_plos_filename = \"PLOS_val.jsonl\"\n",
    "full_dev_elife_filename = \"eLife_val.jsonl\"\n",
    "\n",
    "test_filepath = \"/content/drive/MyDrive/BioLaySumm2024_main/data/test_dataset/\"\n",
    "test_plos_filename = \"PLOS_test.jsonl\"\n",
    "test_elife_filename = \"eLife_test.jsonl\"\n",
    "\n",
    "\n",
    "def read_jsonl(filepath, filename):\n",
    "    df = pd.read_json(filepath + filename,\n",
    "                      orient=\"records\",\n",
    "                      lines=True\n",
    "                     )\n",
    "    return df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "id": "cOsKIZz0T_Ce"
   },
   "outputs": [],
   "source": [
    "mini_plos_df = read_jsonl(mini_dev_filepath, mini_dev_plos_filename)\n",
    "mini_elife_df = read_jsonl(mini_dev_filepath, mini_dev_elife_filename)\n",
    "\n",
    "# full_plos_df = read_jsonl(full_dev_filepath, full_dev_plos_filename)\n",
    "# full_elife_df = read_jsonl(full_dev_filepath, full_dev_elife_filename)\n",
    "\n",
    "test_plos_df = read_jsonl(test_filepath, test_plos_filename)\n",
    "test_elife_df = read_jsonl(test_filepath, test_elife_filename)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/",
     "height": 515,
     "referenced_widgets": [
      "b7947e3af0534201a87761238e847927",
      "309e3dd29f7f4b45866463ce15a1ad3c",
      "a37d9e65fe5c4471a962ad3b87ee67bd",
      "98019547f22741bd9d2a7ef2778182ac",
      "49f9399cacc840c78099a93210ac9bf4",
      "5ad2b60833214d73a69119a68f409097",
      "f3f8d26bf1c245a49622c88f953f7617",
      "4ce382551bd943d9abe342d704a382fb",
      "73366f1b94a940ac8c5c7854dbc88936",
      "5f0f813cce344332886986c894c884d9",
      "ff3056d47fbb44b69409e541c662e90f",
      "85a557d925a6438482d609203bb39c6d",
      "5314d8a147af46e09b6869ded9051216",
      "dcdf8a591dbe40f1b1478ab2c2159ee7",
      "1aef87e388514463882585f15132b860",
      "694a781667bb4f6e8fbe032013a4d2c5",
      "61fc01da51104089b6bdd40a842a0967",
      "6e5de86d7f7a43f78ec2505554a2b6c9",
      "2687cea98b134dd8b25a72806cd39cdc",
      "19c778861e0d48a79a30beab55e55c0c",
      "704fa07cdd71433ab1841ab4173da566",
      "8ce234f295d04391a3d733fb4942076c",
      "c18b950df3594081abfd5dde36c10fda",
      "b170e6cd5beb423fb7018f653992fb48",
      "53838eef853144f1adec46e15e4b9c0d",
      "a00cacfa691642eeb6db41f0e18a6a49",
      "fe121ca81f6146a39fa847f16bc60368",
      "c0d9dfde7cf842e4a8cc4123567f4899",
      "9371f34b638e43ffa0794cde1b317b5f",
      "4b39b985c3d44aec8dc71f325eabf747",
      "e0b42c898ef64d80865220e25ef7e0f1",
      "ed2e74f10eb04675a70ab0b84c93b5e8",
      "2584983e901747ff8cf494c088f707be",
      "37ff89115c024be99ca2fc0ca9c70fdd",
      "01a5e107d3ab486285ac7291373ff263",
      "387669df637d43b99931d527f0c49b05",
      "bd535d3ace9e483593cfa14bf0d33b86",
      "3cb4338afd3747858caffa421b13ab49",
      "59885a813497441982d7bfdf4121d569",
      "6f00d887e2e745b3ad8ca604ae375c93",
      "97065d64d29a43aaa3643030365de255",
      "f2fa150d03de4155a8ba87b4e1d316ca",
      "8e64858c8fe443e8827da5e50491409a",
      "ac795b97844c4f35b3abafafef245c8a",
      "17017b487ab547edbb8725a5063ea513",
      "f2f902693cd84cf09fc18de13e00c47f",
      "197b7d783d2f4e71b29b727d1ed62b29",
      "e64bbd19f5ea4a449da4ef1a41609aac",
      "cb0e7ab19b3f43f3898ec41d3d556ebd",
      "785ddc81d4894b7f8251537d13ae8ad1",
      "fdbccef9fd934bf485a6eba4f43fed9a",
      "1038eaed83ea4602907f27edb8195579",
      "d86bf62213914f25b670d5947ee155bf",
      "e557dfec8afc4859be1077a50183da6d",
      "1abee22e1cdc4a3caecf23a6296edcf3",
      "d2cd1fcf5dc24d57888778c24868dc1b",
      "a8cc64bfa5974e039d0074dcce35936f",
      "e4b0ca5ce07c437c9393a935aa207b7b",
      "f7cd9daf7bb845cca8d69b3e0bd47dcb",
      "3d2bde8aaf4e4fe1ace0cab205b6f75e",
      "f9d4d832cd204b538955e6ddce50f3dc",
      "30bab861b7da44729281c99b2e7d8267",
      "50e886e23b22415fbeb49922005d9ee6",
      "aa47ecbc800642b48a60a097eca3624d",
      "6a854e571d004965bcfc439b1c12b318",
      "0d9a4fb9bb9b442d85b72a171b63923e"
     ]
    },
    "id": "bl-XzeHXhrav",
    "outputId": "7dcda56d-ed33-499b-cd68-4bc622ab1c85"
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_token.py:88: UserWarning: \n",
      "The secret `HF_TOKEN` does not exist in your Colab secrets.\n",
      "To authenticate with the Hugging Face Hub, create a token in your settings tab (https://huggingface.co/settings/tokens), set it as secret in your Google Colab and restart your session.\n",
      "You will be able to reuse this secret in all of your notebooks.\n",
      "Please note that authentication is recommended but still optional to access public models or datasets.\n",
      "  warnings.warn(\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "b7947e3af0534201a87761238e847927",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "config.json:   0%|          | 0.00/887 [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WARNING 04-04 00:47:22 config.py:193] awq quantization is not fully optimized yet. The speed can be slower than non-quantized models.\n",
      "INFO 04-04 00:47:22 llm_engine.py:87] Initializing an LLM engine with config: model='BioMistral/BioMistral-7B-DARE-AWQ-QGS128-W4-GEMM', tokenizer='BioMistral/BioMistral-7B-DARE-AWQ-QGS128-W4-GEMM', tokenizer_mode=auto, revision=None, tokenizer_revision=None, trust_remote_code=False, dtype=torch.float16, max_seq_len=32768, download_dir=None, load_format=auto, tensor_parallel_size=1, disable_custom_all_reduce=False, quantization=awq, enforce_eager=False, kv_cache_dtype=auto, device_config=cuda, seed=0)\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "85a557d925a6438482d609203bb39c6d",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "tokenizer_config.json:   0%|          | 0.00/1.47k [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "c18b950df3594081abfd5dde36c10fda",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "tokenizer.model:   0%|          | 0.00/493k [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "37ff89115c024be99ca2fc0ca9c70fdd",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "tokenizer.json:   0%|          | 0.00/1.80M [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "17017b487ab547edbb8725a5063ea513",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "special_tokens_map.json:   0%|          | 0.00/414 [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO 04-04 00:47:35 weight_utils.py:163] Using model weights format ['*.safetensors']\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "d2cd1fcf5dc24d57888778c24868dc1b",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "model.safetensors:   0%|          | 0.00/4.15G [00:00<?, ?B/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO 04-04 00:47:57 llm_engine.py:357] # GPU blocks: 14037, # CPU blocks: 2048\n",
      "INFO 04-04 00:47:59 model_runner.py:684] Capturing the model for CUDA graphs. This may lead to unexpected consequences if the model is not static. To run the model in eager mode, set 'enforce_eager=True' or use '--enforce-eager' in the CLI.\n",
      "INFO 04-04 00:47:59 model_runner.py:688] CUDA graphs can take additional 1~3 GiB memory per GPU. If you are running out of memory, consider decreasing `gpu_memory_utilization` or enforcing eager mode. You can also reduce the `max_num_seqs` as needed to decrease memory usage.\n",
      "INFO 04-04 00:48:06 model_runner.py:756] Graph capturing finished in 7 secs.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Processed prompts: 100%|██████████| 1/1 [00:00<00:00, 13.70it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[RequestOutput(request_id=0, prompt='What is the capital cit of British Columbia. Keep answer short.', prompt_token_ids=[1, 1824, 349, 272, 5565, 5963, 302, 4409, 14402, 28723, 10339, 4372, 2485, 28723], prompt_logprobs=None, outputs=[CompletionOutput(index=0, text='\\nA: Victoria', token_ids=[13, 28741, 28747, 13682, 2], cumulative_logprob=-8.658285424113274, logprobs=None, finish_reason=stop)], finished=True, metrics=RequestMetrics(arrival_time=879.887349887, last_token_time=879.887349887, first_scheduled_time=1712191686.0395195, first_token_time=1712191686.0683515, time_in_queue=1712190806.1521697, finished_time=1712191686.1123087), lora_request=None)]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "llm = LLM(model='BioMistral/BioMistral-7B-DARE-AWQ-QGS128-W4-GEMM')\n",
    "output = llm.generate('What is the capital cit of British Columbia. Keep answer short.')\n",
    "\n",
    "print(output)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "id": "h36RUZCyUbra"
   },
   "outputs": [],
   "source": [
    "def lay_summarize(text, llm=llm):\n",
    "    \"\"\"\n",
    "        summarize a text using a LLM,\n",
    "        with min_length and max_length are number of tokens limits for the output\n",
    "    \"\"\"\n",
    "\n",
    "    prompt = f\"[INST] Simplify and summarize in around 300 words:  {text} [/INST]\"\n",
    "\n",
    "    sampling_params = SamplingParams(temperature=0.8, top_p=0.05, max_tokens=1024)\n",
    "\n",
    "    output = llm.generate(prompt, sampling_params)\n",
    "\n",
    "    return output[0].outputs[0].text\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "id": "mozNAcoJvjDk"
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/",
     "height": 178
    },
    "id": "OTojJ6u7gk9k",
    "outputId": "5effca9f-1c48-44dd-f156-b218bc7df73f"
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.31s/it]\n"
     ]
    },
    {
     "data": {
      "application/vnd.google.colaboratory.intrinsic+json": {
       "type": "string"
      },
      "text/plain": [
       "' The evolutionary origins of hypoxia-sensitive cells that trigger amniote respiratory reflexes, such as carotid body glomus cells and pulmonary neuroendocrine cells (PNECs), are unclear. Researchers have proposed that glomus cells, which are neural crest-derived, are similar to hypoxia-sensitive neuroepithelial cells (NECs) of fish gills, whose embryonic origin is unknown. NECs have also been likened to PNECs, which differentiate in situ within lung airway epithelia. Using genetic lineage-tracing and neural crest-deficient mutants in zebrafish, and physical fate-mapping in frog and lamprey, researchers have found that NECs are not neural crest-derived, but endoderm-derived, like PNECs. Researchers have also discovered neural crest-derived catecholaminergic cells associated with zebrafish pharyngeal arch blood vessels. Based on these findings, researchers propose a new model for amniote hypoxia-sensitive cell evolution: endoderm-derived NECs were retained as PNECs, while the carotid body evolved via the aggregation of neural crest-derived catecholaminergic (chromaffin) cells already associated with blood vessels in anamniote pharyngeal arches.'"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#test\n",
    "s = 'The evolutionary origins of the hypoxia-sensitive cells that trigger amniote respiratory reflexes – carotid body glomus cells , and ‘pulmonary neuroendocrine cells’ ( PNECs ) - are obscure . Homology has been proposed between glomus cells , which are neural crest-derived , and the hypoxia-sensitive ‘neuroepithelial cells’ ( NECs ) of fish gills , whose embryonic origin is unknown . NECs have also been likened to PNECs , which differentiate in situ within lung airway epithelia . Using genetic lineage-tracing and neural crest-deficient mutants in zebrafish , and physical fate-mapping in frog and lamprey , we find that NECs are not neural crest-derived , but endoderm-derived , like PNECs , whose endodermal origin we confirm . We discover neural crest-derived catecholaminergic cells associated with zebrafish pharyngeal arch blood vessels , and propose a new model for amniote hypoxia-sensitive cell evolution: endoderm-derived NECs were retained as PNECs , while the carotid body evolved via the aggregation of neural crest-derived catecholaminergic ( chromaffin ) cells already associated with blood vessels in anamniote pharyngeal arches . '\n",
    "lay_summarize(s, llm)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/",
     "height": 1000
    },
    "id": "uNnqTHHsg2on",
    "outputId": "c31618a5-c919-46cc-d402-135871ec5c88"
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.58s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.36s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.41s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.90s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.37s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.35s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.28s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.20s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.94s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.08s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.40s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.60s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.51s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.78s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.25s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.80s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.02s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.68s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.13s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.87s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.12s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.63s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.79s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.37s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.63s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.00s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.28s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.12s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.04s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.82s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.28s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.14s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.71s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.17s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.13s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.29s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.86s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.94s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.30s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.57s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.52s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:05<00:00,  5.18s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.40s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.66s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.81s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.45s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.92s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.96s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.72s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.79s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:01<00:00,  1.88s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.77s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.63s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.53s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.71s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.74s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.79s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:01<00:00,  1.49s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.46s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.45s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.64s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.22s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.58s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.94s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.28s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.06s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.15s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.63s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.50s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.01s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.67s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.06s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.11s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.31s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.57s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.64s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.20s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.03s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.58s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.69s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.68s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.80s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.12s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.29s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.56s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.18s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.73s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.52s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.79s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.47s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.62s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.81s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.11s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.39s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.43s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.60s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.88s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.32s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.22s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.36s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.99s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.31s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.46s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.53s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.20s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:05<00:00,  5.24s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.43s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.21s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.84s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.20s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.46s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.35s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.76s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.67s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.14s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.88s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.08s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.33s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:01<00:00,  1.99s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.43s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.05s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.65s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.19s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.28s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.95s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.74s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.87s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.10s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.27s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.60s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.61s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.09s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:08<00:00,  8.06s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.53s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.08s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.91s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  4.00s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.91s/it]\n"
     ]
    },
    {
     "data": {
      "application/vnd.google.colaboratory.intrinsic+json": {
       "summary": "{\n  \"name\": \"mini_plos_df\",\n  \"rows\": 138,\n  \"fields\": [\n    {\n      \"column\": \"lay_summary\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 138,\n        \"samples\": [\n          \"Glanders is a disease of antiquity , recognized as a malady of equines by Hippocrates and Aristotle . The causative agent , Burkholderia mallei , is currently feared as a potential biological weapon and has been used as such in the American Civil War and both World Wars to cripple equine military components . In the more economically developed countries , glanders has been eradicated through large scale culling . As a result , our understanding of transmission dynamics and networks is limited . However , regions of endemicity still exist in Asia , the Middle-East , Africa , and South America where it infects solipeds and camels . These areas provide reservoirs for re-introduction of glanders into countries previously listed as glanders-free . Here , we demonstrate the utility of high-resolution genotyping and whole genome sequence analysis in the investigation of a recent outbreak of glanders in horses and camels in Bahrain , a previously declared glanders-free country . Our analyses demonstrate that not one , but two strains likely caused this outbreak , and that these strains probably came from a similar geographic region via importation of infected animals . Even with careful monitoring , the global trade of animals from glanders-endemic regions can re-introduce and possibly re-establish this disease in animal populations of countries that have previously eradicated it .\",\n          \"Genetic reassortments between avian and swine influenza viruses are likely to occur in the swine lower respiratory track , and tissue tropism is an important factor selecting for a transmissible reassortant; determination of tissue tropisms for potential reassortants between contemporary avian and swine influenza viruses would help identify transmissible reassortants with public health risks .\",\n          \"There is still much to learn about the organisation of regulatory elements that control where , when , and how much individual genes in the genome are transcribed . Several types of regulatory element have been identified; some , such as enhancers , act over large genomic distances . This creates a problem: how do such long-range elements only regulate their appropriate target genes ? Insulator elements have been proposed to act as barriers within the genome , confining the effects of long-range regulatory elements . Here we have mapped the locations of one insulator-binding protein , CTCF , in several regions of the Drosophila genome . In particular , we have focussed on the Hox gene cluster in the Bithorax complex; a region whose regulation has been extensively characterised . Previous investigations have identified independent regulatory domains that control the expression of Bithorax complex genes in different segments of the fly , however the molecular nature of the domain boundaries is unclear . Our major result is that we find CTCF binding sites precisely located at the boundaries of these regulatory domains , giving a common molecular basis for these boundaries . This provides a clear example of the link between the positioning of insulators and the organisation of gene regulation in the Drosophila genome .\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"article\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 138,\n        \"samples\": [\n          \"Glanders , caused by the gram-negative bacterium Burkholderia mallei , is a highly infectious zoonotic disease of solipeds causing severe disease in animals and men . Although eradicated from many Western countries , it recently emerged in Asia , the Middle-East , Africa , and South America . Due to its rareness , little is known about outbreak dynamics of the disease and its epidemiology . We investigated a recent outbreak of glanders in Bahrain by applying high resolution genotyping ( multiple locus variable number of tandem repeats , MLVA ) and comparative whole genome sequencing to B . mallei isolated from infected horses and a camel . These results were compared to samples obtained from an outbreak in the United Arab Emirates in 2004 , and further placed into a broader phylogeographic context based on previously published B . mallei data . The samples from the outbreak in Bahrain separated into two distinct clusters , suggesting a complex epidemiological background and evidence for the involvement of multiple B . mallei strains . Additionally , the samples from Bahrain were more closely related to B . mallei isolated from horses in the United Arab Emirates in 2004 than other B . mallei which is suggestive of repeated importation to the region from similar geographic sources . High-resolution genotyping and comparative whole genome analysis revealed the same phylogenetic patterns among our samples . The close relationship of the Dubai/UAE B . mallei populations to each other may be indicative of a similar geographic origin that has yet to be identified for the infecting strains . The recent emergence of glanders in combination with worldwide horse trading might pose a new risk for human infections . \\n Glanders is a life-threatening , notifiable zoonotic disease which is fatal to both animals and humans . It is caused by the gram-negative bacterium Burkholderia mallei [1] . The only known reservoirs of B . mallei are solipeds , particularly horses . Chronically infected horses can be asymptomatic but may remain highly infectious . As a highly infectious agent that can be transmitted by aerosol , causing invasive fatal disease in combination with resistance to multiple antibiotics , B . mallei is listed as a category B bio-threat agent by the CDC ( www . bt . cdc . gov/agent/agentlist-category . asp ) . Licensed vaccines against the disease do not exist . Antibiotic treatment is cumbersome and requires the combination of at least two different antibiotics over several weeks [2] . Throughout the western world , glanders has been eradicated through large scale culling of infected animals . In developing countries , however , economic and cultural circumstances may hinder culling of asymptomatic animals and enable the persistence of glanders . In recent years , several outbreaks of glanders occurred in the horse populations in Asia , Middle-East ( Afghanistan , Kuwait , Iran , Iraq , Pakistan , Syria ) , Africa , and South America ( Brazil ) [3] , [4] . Because of the recent rise in cases in multiple countries , glanders has regained the status of a re-emerging disease [5] , [6] . Officially , Bahrain was free of glanders until an outbreak in the north ( Jannusan , Shakhurah and Saar municipalities ) that began in April 2010 . Horses imported from Syria via Kuwait were suspected of introducing glanders and all equines in the area were quarantined and tested . By September 2010 , the outbreak was considered to be resolved . However , in January 2011 the disease reoccurred in the same region of the country . Details on the outbreak are provided by the OIE ( http://www . oie . int/wahis_2/public/wahid . php/Wahidhome/Home ) . More than 6 , 700 horses and 250 donkeys ( 100% of the equine population in this region , representing about 80% of the total horse and donkey population in Bahrain ) as well as three camels presenting with clinical symptoms were screened for glanders at OIE Reference Laboratory , the Central Veterinary Research Laboratory in Dubai , United Arab Emirates using prescribed Complement Fixation Testing ( CFT ) and an in-house cELISA [7] . In these investigations , 50 horses and one camel tested positive . B . mallei was isolated from eight horses and the single , positive camel . Using B . mallei-specific real-time PCR and high resolution MLVA typing , we showed recently that the strain from the camel was genetically closely related to B . mallei strain Dubai 7 that was isolated from a horse during the contained outbreak of glanders in a quarantine station in the United Arab Emirates in 2004 [8] . In this current study , we characterize various B . mallei isolates from both events ( the 2004 UAE outbreak and the 2010\\u2013'11 Bahrain outbreak ) , using MLVA and next-generation whole genome sequencing . Our results provide evidence that the recent outbreak in Bahrain was caused by two different B . mallei strains , suggesting two independent introductions . \\n Glanders is a notifiable disease to the World Organization for Animal Health ( OIE ) . As the official OIE reference laboratory for glanders in the Arabic region , the Central Veterinary Laboratory ( CVRL ) in Dubai is the officially authorized institution for glanders research , surveillance and eradication . All procedures involving animals were performed in strict accordance with the OIE guidelines for animal welfare using prescribed protocols . An ethic commission comprising 4 veterinarians of the Central Veterinary Research Laboratory ( CVRL ) and a government veterinarian from the Ministry of Environment and Water of the UAE followed the Ministerial Decree No 384 of the year 2008 on the executive by-law of the Federal Law No 16 of the year 2007 concerning Animal Welfare . All experimental animals and treatment in this study were reviewed and approved by the Animal Ethic Committee of CVRL , and Ministry of Environment and Water of the UAE ( Permit Number: 550353 ) . Isolation of the strains , DNA extraction , and MLVA were performed as described previously ( 8 ) . A total of nine B . mallei isolates , each from a different horse and one from the camel from both Bahrainian outbreak events ( 2010 and 2011 ) were analyzed along with 15 isolates from the 2004 outbreak in the UAE by applying the same high-resolution 23-marker VNTR assay used to type the strain from the camel [8] . To put the Bahrainian strains in a larger phylogeographic context , MLVA23 data from nine B . mallei isolates from the strain collection of the Bundeswehr Institute of Microbiology in Munich and previously published data from an additional 42 samples [9] , [10] were included in the phylogenetic reconstruction ( Table S1 ) . Analysis of VNTR data was performed as previously described in Hornstra et al . [10] . To assess phylogenetic relationships we sequenced and compared eight genomes from the outbreak in the United Arab Emirates from 2004 to seven genomes from the Bahrain outbreak . Using ATCC 23344 genome as a reference , we identified homologous regions using MUMmer [11] , and found single nucleotide polymorphisms ( SNPs ) using SolSNP ( ( http://sourceforge . net/projects/solsnp/ ) . After eliminating potential paralogs and positions with missing or ambiguous data among isolates , raw reads containing SNPs were aligned to the reference and inspected to verify allele calls and eliminate mixed alleles ( when the minor allele frequency was greater than 10% ) . The final dataset was composed of 242 SNPs of which only 44 were variable among the Bahrain and Dubai genomes . Phylogenetic reconstruction was achieved using the maximum parsimony method implemented in MEGA 5 . 2 . 2 ( Figure 1 , panel B ) . The consistency index was 1 . 0 for all sites , indicating that the final dataset is devoid of homoplasy and that all sites are in agreement with the topology . For datasets with little homoplasy , the consistency index is a more appropriate and direct measure of accuracy than bootstrapping , however 500 bootstrap replicates revealed that the clades of interest clustered together in greater than 98% of replicate trees . \\n MLVA revealed that the nine strains from the 2010\\u2013'11 outbreak in Bahrain formed two clearly separated clusters ( BH1 and BH2 ) , consisting of five and four strains each ( Figure 1 , panel A ) . Whereas BH1 was closest to the cluster consisting of various strains from the 2004 outbreak in the UAE ( Figure 1 , panel A ) , BH2 , which also contained the strain from the camel sample , differed at eight VNTR markers from the closest strain ( D403 , Mymar ) of the BH1 cluster . This suggests at least two different B . mallei populations were involved in the outbreak . Strains from 2010 and 2011 were found in both clusters , suggesting an outbreak that persisted across both years and was caused by two independent but temporally simultaneous introductions . The whole genome SNP phylogeny ( Figure 1 , panel B ) confirmed the MLVA data and lends further support to the hypothesis that two different populations of B . mallei caused the outbreak in Bahrain . Our results demonstrate that MLVA provides an important and useful tool for rapid initial estimations of epidemiological links among B . mallei . Moreover , the data suggest that MLVA can be used to study genetic diversity among B . mallei clones from a single outbreak . In this study , both MLVA and SNP methods revealed the same phylogenetic patterns among the three main groups ( UAE/BH1 , BH1 , and BH2 ) suggesting the involvement of two genetically closely related but distinct B . mallei populations during the outbreak . The close relationship of the Dubai/UAE B . mallei population to the BH1 and BH2 populations may be indicative of a similar geographic source that has yet to be identified . Animal importation records suggest Syria and Kuwait as possible sources . This is strongly supported by the fact that one of the necropsied horses in Bahrain quarantine came directly from Kuwait . Outbreak dynamics and natural genetic variability of B . mallei are not well understood due to the rarity of this disease . This outbreak provided a unique opportunity to understand outbreak dynamics as they occurred in a region that was previously free of B . mallei , and well monitored with records of animal importations . The failure of initial eradication efforts in 2010 is evidence for the need for continued surveillance and abatement measures even after all animal cases appear to be cleared . All animals imported from known or potentially endemic regions should be routinely tested for glanders prior to importation . Repeated testing during quarantine is also recommended as infected animals may be asymptomatic and serologically negative .\",\n          \"Genetic reassortment between influenza A viruses ( IAVs ) facilitate emergence of pandemic strains , and swine are proposed as a \\u201cmixing vessel\\u201d for generating reassortants of avian and mammalian IAVs that could be of risk to mammals , including humans . However , how a transmissible reassortant emerges in swine are not well understood . Genomic analyses of 571 isolates recovered from nasal wash samples and respiratory tract tissues of a group of co-housed pigs ( influenza-seronegative , avian H1N1 IAV\\u2013infected , and swine H3N2 IAV\\u2013infected pigs ) identified 30 distinct genotypes of reassortants . Viruses recovered from lower respiratory tract tissues had the largest genomic diversity , and those recovered from turbinates and nasal wash fluids had the least . Reassortants from lower respiratory tracts had the largest variations in growth kinetics in respiratory tract epithelial cells , and the cold temperature in swine nasal cells seemed to select the type of reassortant viruses shed by the pigs . One reassortant in nasal wash samples was consistently identified in upper , middle , and lower respiratory tract tissues , and it was confirmed to be transmitted efficiently between pigs . Study findings suggest that , during mixed infections of avian and swine IAVs , genetic reassortments are likely to occur in the lower respiratory track , and tissue tropism is an important factor selecting for a transmissible reassortant . \\n Influenza A viruses ( IAVs ) are negative-strand RNA viruses with eight gene segments of various lengths , ranging from about 890 to 2 , 341 nucleotides . IAV subtypes are determined by the virus surface glycoproteins , hemagglutinin ( HA ) , and neuraminidase ( NA ) . A total of 18 HA ( H1\\u2013H18 ) and 11 NA ( N1\\u2013N11 ) subtypes have been identified [1 , 2] . IAVs with subtypes H1\\u2013H16 and N1\\u2013N9 have been recovered from a variety of bird species , including at least 105 wild bird species of 26 different families [3] . Among wild birds , those living in wetlands and aquatic environments ( such as the Anseriformes , particularly ducks , geese , and swans; and Charadriiformes , particularly gulls , terns , and waders ) constitute the major natural IAV reservoir [4] . In addition to birds , IAVs can also cause infections in marine mammals , land-based lower mammals ( e . g . , pigs , dogs , and horses ) , and humans . Wild birds maintain a large IAV genetic pool , which contributes to the appearance of new IAVs in humans , lower mammals , and domestic poultry . Because pigs have both avian-like receptors ( \\u03b1 2 , 3-linked sialic acid , SA2 , 3Gal ) and human-like receptors ( \\u03b1 2 , 6-linked sialic acid , SA2 , 6Gal ) on the epithelial cells of their respiratory tissues , they have been proposed to serve as an intermediate host \\u201cmixing vessel\\u201d for generating pandemic IAV strains through genetic reassortment between avian and swine or human IAVs . Three of four documented pandemic strains are reassortants between avian and swine or human IAVs: the 1957 subtype H2N2 pandemic IAV probably emerged from H2 avian IAV and H1N1 human IAV [5]; the 1968 subtype H3N2 pandemic IAV probably emerged from H3 avian IAV and the H2N2 human IAV [5]; and the 2009 subtype H1N1 pandemic IAV had PB2 and PA genes from avian IAVs , PB1 gene from human H3N2 IAV , and other genes from swine IAVs [6] . However , emergence of these pandemic viruses is complicated and could require multiple reassortment events ( in addition to mutations ) across years . In addition to virus receptor binding attributes , compatibility between the co-circulating avian and swine or human IAVs will determine whether a new reassortant can be generated in swine . A number of studies have been performed to evaluate the compatibility of emerging avian IAVs and endemic human IAVs , such as avian H9N2 versus human H1N1 [7] , avian H5N1 versus human H3N2 [8\\u201310] , and avian H5N1 versus human H1N1 [11] . The studies demonstrated that gene compatibility between viruses is subtype- and strain-dependent [7 , 10\\u201312]; however , transmissible viruses are possibly derived from genetic reassortments between avian and human IAVs [13] . In addition , given two IAVs , the timing and dose of inoculation , as well as inoculation location , can affect the dynamics of IAV reassortment and the reassortants to be generated [14\\u201318] . Thus , genetic reassortment is a complicated biological process , and how a transmissible reassortant emerges in swine are not well understood . Efforts were carried out to reproduce the 2009 H1N1 pandemic strains by mixing two swine IAVs in domestic pigs with or without preexisting immunity , but they were unsuccessful [19] . We conducted this study to test the hypothesis that tissue tropism plays an important role in selecting transmissible reassortants from various reassortants generated in pigs co-infected with avian and swine IAVs . We determined the genotypes and phenotypes of reassortants recovered from a mixed group of pigs ( influenza-seronegative , avian H1N1 IAV\\u2013infected , and swine H3N2 IAV\\u2013infected pigs ) and then characterized the temporal and spatial distributions of the reassortants . Feral swine were used as the animal model in this study , and we will simply describe feral swine as pigs or swine in the following sections . \\n Infectivity of avian IAVs in swine is strain dependent [20] , and avian IAVs to be used in animal studies must show optimum infectivity in swine . To select an avian-origin IAV for animal experiments , we characterized the receptor binding profiles of avian IAVs and selected an avian IAV binding to both avian and human-like receptors . Biolayer interferometry analyses suggested that influenza strain A/mallard/Wisconsin/A00751454/2009 ( H1N1 ) had detectable binding affinities to 3\\u02b9-sialyl-N-acetyllactosamine ( 3\\u02b9SLN , representing avian-like receptor SA2 , 3Gal ) and to 6\\u02b9-sialyl-N-acetyllactosamine ( 6\\u02b9SLN , representing human-like receptor SA2 , 6Gal ) . To further determine whether an IAV prefers more to 3\\u02b9SLN rather than 6\\u02b9SLN ( or more to 6\\u02b9SLN rather than 3\\u02b9SLN ) , we quantified and compared 50% relative sugar loading ( RSL ) concentration ( RSL0 . 5 ) at half of the fractional saturation ( f = 0 . 5 ) of the testing virus against both 6\\u02b9SLN and 3\\u02b9SLN . The higher the RSL0 . 5 , the smaller the binding affinity . Results from quantitative analyses suggested that this avian H1N1 IAV had a higher relative sugar loading ( RSL ) concentration to 6\\u02b9SLN ( RSL0 . 5 = 0 . 35 nm ) than that to 3\\u02b9SLN ( RSL0 . 5 = 0 . 06 nm ) ( Fig 1A , left panel ) , indicating that this avian H1N1 IAV has a higher binding affinity to 3\\u02b9SLN than that to 6\\u02b9SLN . We further evaluated the infectivity of A/mallard/Wisconsin/A00751454/2009 ( H1N1 ) virus in swine . By combining viral shedding ( S1 Table ) , viral loads in tissues ( S1 Fig ) , serologic analyses data ( S2 Table ) , we showed that A/mallard/Wisconsin/A00751454/2009 ( H1N1 ) virus successfully infected 6 of 8 treatment pigs ( Fig 1B; S1 Text ) . In addition to selecting the avian-origin A/mallard/Wisconsin/A00751454/2009 ( H1N1 ) influenza strain for use in the animal study , we also selected a feral swine\\u2013origin IAV , A/swine/Texas/A01104013/2012 ( H3N2 ) [21] . Biolayer interferometry analyses suggested , in contrast to findings for A/mallard/Wisconsin/A00751454/2009 ( H1N1 ) , that this swine H3N2 IAV binds to 6\\u02b9SLN but not 3\\u02b9SLN ( Fig 1A , right panel ) . According to earlier findings [21] , this swine H3N2 IAV is highly infectious and transmissible in feral swine . To mimic the natural scenario for IAV co-infections , we included one pig inoculated with A/mallard/Wisconsin/A00751454/2009 ( H1N1 ) , one pig inoculated with A/swine/Texas/A01104013/2012 ( H3N2 ) , and one or two IAV-seronegative contact pigs in each of five experimental groups . Temporal and spatial dynamics of virus infections and genetic reassortments in these pigs were characterized to understand how a transmissible reassortant was generated between the avian and swine IAVs tested . Although the initial viral shedding time for individual pigs varied , contact pigs and pigs inoculated with swine H3N2 virus or avian H1N1 virus shed viruses as early as 3 days post-inoculation ( dpi ) and continued to shed viruses on 7 dpi before all pigs were euthanized ( S3 Table ) . Early viral shedding loads in avian H1N1 virus\\u2013inoculated pigs were much lower than those in swine H3N2 virus\\u2013inoculated pigs , but by 7 dpi , levels were similar for the two groups ( range , 4 . 032\\u20136 . 699 log10TCID50/mL ) . For individual pigs , virus loads in nasal wash fluids differed widely from those in respiratory tract tissues . However , virus was consistently detected in the tissues of pigs that had positive virus titration results for nasal wash fluids . Furthermore , the pattern of viral loads in tissues were similar to those in nasal wash fluids . No significant clinical signs of infection ( e . g . , temperature , coughing , and body weight changes ) were observed in any pigs in the animal experiment ( S2 Fig ) . To understand the bottlenecks for viral reassortment between avian and swine IAVs and to elucidate how transmissible IAVs emerge from reassortment events in swine , we characterized the temporal and spatial dynamics of genetic reassortments during the IAV co-infections . Specifically , individual virions from samples with positive viral titration results were purified using plaque assays . To minimize possible sampling biases , we collected a maximum of 10 plaques for each individual nasal wash fluids or tissue sample . In total , 571 plaques were generated: 59 from avian H1N1 virus\\u2013inoculated pigs , 300 from swine H3N2 virus\\u2013inoculated pigs , and 212 from contact pigs . Of the 571 plaques , 157 were recovered from samples collected on 3 dpi , 264 from samples collected on 5 dpi , and 150 from samples collected on 7 dpi; these plaques were broadly distributed among the nasal wash fluids ( n = 159 ) and the upper ( i . e . turbinate ) ( n = 188 ) , middle ( i . e . trachea ) ( n = 137 ) , and lower ( i . e . bronchus , left cranial lung , right cranial lung , right middle lung , right accessory , left caudal lung , and right caudal lung ) ( n = 87 ) respiratory tracts of the pigs ( Fig 2; S1 Text ) . The 571 viruses were subjected to whole-genomic sequencing , and sequences were obtained for 546 viruses with complete genomic sequences . The stock of the H1N1 inoculum and the H3N2 inoculum were also subjected to whole-genomic sequencing , the data from which were used in genotype analyses and polymorphism characterization for all of the 546 plaque isolates . The viruses were distributed among 32 different genotypes , one of which was the same as that for the parent avian H1N1 virus and included 10 ( 1 . 83% ) of the 546 viruses . Another genotype was the same as that for the parent swine H3N2 virus and included 473 ( 86 . 63% ) of the 546 viruses . The other 30 genotypes were novel , with at least one gene from each of the two parent viruses , and included 63 ( 11 . 54% ) of the 546 viruses . Of the 63 reassortant viruses , 60 contained HA and NA genes from swine H3N2 viruses; one contained HA and NA from avian H1N1 virus; two contained HA from swine H3N2 virus but NA from avian H1N1 virus . Each plaque was assigned with a plaque number and a genotype number , and one genotype can have multiple plaques . For example , R3 refers to genotype 3 annotated in Fig 3 and includes a total of six plaques ( plaque #39 , #43 , #69 , #258 , #456 , and #551; S9 Table ) . The 473 parental swine H3N2\\u2013like viruses ( 86 . 63% of the 546 sequenced viruses ) were identified not only in pigs inoculated with swine H3N2 virus but also in both pigs inoculated with avian H1N1 virus and the contact pigs; these viruses were identified in nasal wash samples and in upper , middle , and lower respiratory tract tissue samples . Avian parental H1N1\\u2013like viruses were identified not only on 5 dpi in the lung of the pig inoculated with avian H1N1 virus but also in the lung of one contact pig euthanized on 7 dpi . Of interest , of the 63 reassortant viruses , 11 were recovered from one H3N2 virus\\u2013inoculated pig euthanized on 5 dpi , one was from an H1N1 virus\\u2013infected pig euthanized on 5 dpi , three were from a contact pig euthanized on 5 dpi , and 48 were from a contact pig euthanized on 7 dpi . The viruses recovered from the lower respiratory tract had higher genomic diversity ( 19 genotypes ) than those recovered from the middle respiratory tract ( 18 genotypes ) . The least genomic diversity ( i . e . only two reassortants ) were identified in each of the nasal wash samples and upper respiratory tract samples . At least eight genotypes of reassortants were identified in samples from two or more anatomical locations: nasal wash fluids or upper , middle , or lower respiratory tract tissues . Of note , one genotype of reassortants ( genotype R3 ) was identified in nasal wash fluids and in all upper , middle , and lower respiratory tract tissues ( Fig 3 ) . To test the working hypothesis that tissue tropism presents a bottleneck for genetic reassortment and selects the viruses to be shed , we performed viral growth analyses for parent avian H1N1 and swine H3N2 wild-type influenza viruses and for 52 representative reassortant viruses in swine nasal epithelium ( SNE ) cells at 33\\u00b0C , swine tracheal epithelium ( STE ) cells at 37\\u00b0C , and human alveolar basal epithelial ( A549 ) cells at 39\\u00b0C . Because of a lack of swine alveolar epithelial cells , human A549 cells were used to mimic swine alveolar epithelial cells from swine lower respiratory tracks . Human A549 cells are characterized as a type II pulmonary epithelial cell model study on IAV infection [22] , and both human and swine type II pneumocytes expressed \\u03b1-2 , 3- and \\u03b1-2 , 6-linked sialic acid receptors [23 , 24] , and are targets of IAV infection [23 , 25\\u201327] . Three incubation temperatures were applied to simulate the temperatures of airway in pigs: approximately 33\\u00b0C in the nasal , 37\\u00b0C in the trachea , and 39\\u00b0C in the lung . Peak virus titers in the supernatants were determined at 72 h post-inoculation using TCID50 in Madin-Darby canine kidney ( MDCK ) cells ( Figs 4 and 5; S3 Fig ) . Of the 52 testing reassortants , 51 showed efficient replication in A549 cells at 39\\u00b0C , with virus titers ranging from 4 . 0 to 7 . 0 log10TCID50/mL ( S3 Fig ) ; plaque #28 ( genotype R6 , Fig 4 ) was the one exception . The average virus titer for viruses from the upper respiratory track was 5 . 65 ( \\u00b1 standard deviation; \\u00b10 . 41 ) log10TCID50/mL . Average titers for viruses from the middle and lower respiratory tracks were 6 . 36 ( \\u00b10 . 57 ) log10TCID50/mL and 6 . 62 ( \\u00b10 . 76 ) log10TCID50/mL , respectively . However , the same testing reassortant viruses showed relatively large variations in infectivity in STE cells at 37\\u00b0C ( virus titer range , 1 . 5\\u20137 . 5 log10TCID50/mL ) ( S3 Fig ) . The average virus titer for viruses from the upper respiratory track was 5 . 63 ( \\u00b11 . 14 ) log10TCID50/mL , and average titers for viruses from the middle and lower respiratory tracks were 5 . 52 ( \\u00b11 . 70 ) log10TCID50/mL and 6 . 22 ( \\u00b11 . 00 ) log10TCID50/mL , respectively . All viruses recovered from the upper respiratory tract samples replicated well in SNE cells at 33\\u00b0C ( average virus titer [\\u00b1standard deviation] , 5 . 89 [\\u00b10 . 49] log10TCID50/mL ) , but viruses from the middle and lower respiratory tracts showed large variations in infectivity ( virus titer range , 1 . 5\\u20137 . 0 log10TCID50/mL ) ( S3 Fig ) . The average virus titer for viruses from middle respiratory tracts was 5 . 22 ( \\u00b11 . 31 ) log10TCID50/mL , and that for viruses from lower respiratory tracts was 5 . 27 ( \\u00b11 . 29 ) log10TCID50/mL . Of note , the testing reassortants recovered from the upper respiratory tract samples replicated well in all three temperature conditions ( virus titer range , 4 . 0\\u20137 . 0 log10TCID50/mL ) . We hypothesized that the nasally shed reassortant viruses would be highly transmissible among feral swine . To test this hypothesis , we evaluated the transmission ability of reassortant plaque #69 ( genotype R3 ) among feral swine . Eight feral swine were separated into four experiment groups comprised of two pigs each: one plaque #69 ( genotype R3 ) virus\\u2013inoculated pig and one IAV-seronegative pig as the contact pig . Four additional feral swine were used as non-experiment group control pigs and housed in a different facility . In the four experiment groups , viral shedding was initially detected in nasal wash samples from plaque #69 ( genotype R3 ) virus\\u2013inoculated pigs on 3 dpi ( virus titer range , 2 . 67\\u20134 . 67 log10TCID50/mL ) , and from the contact pigs on 5 dpi ( virus titer range , 2 . 5\\u20136 . 0 log10TCID50/mL ) . The virus-inoculated pigs continued to shed viruses until 5 dpi , and the contact pigs shed viruses until 9 dpi ( S6 Table ) . Two plaque #69 ( genotype R3 ) virus\\u2013inoculated pigs were euthanized on 3 and 5 dpi . Results from viral titration showed that only the upper respiratory tract tissues from the pig euthanized on 3 dpi were IAV-positive by virus titration ( 5 . 0 log10TCID50/mL ) and , however , that all the upper , middle , and lower respiratory tract tissues from the pig euthanized on 5 dpi were IAV-positive ( titer range , 3 . 5\\u20135 . 0 log10TCID50/mL ) ( S7 Table ) . The remaining four pigs in the experiment groups seroconverted on 21 dpi ( HI titer range , 1:640\\u20131:1 , 280 ) ( S5 Fig and S8 Table ) . No viruses were detected in nasal wash fluids or tissue samples from the four non-experiment group control pigs , and they all remained seronegative for plaque #69 ( genotype R3 ) virus 21days after inoculation of pigs in the experiment group . According to viral titration and seroconversion results , all plaque #69 ( genotype R3 ) virus-inoculated pigs became virus infection positive on 3 dpi ( S5A Fig ) , while all contact pigs became virus infection positive on 5 dpi ( S5B Fig ) . In summary , these results showed that the testing reassortant virus could be transmitted directly between pigs with an efficiency of 100% . \\n Swine have been proposed as a \\u201cmixing vessel , \\u201d generating reassortants between avian IAVs and swine or human IAVs . Such reassortant viruses caused the 2009 pandemic outbreaks among humans [6] . The detection of avian IAVs in swine is not uncommon , and subtype H2N3 [28] , H4N6 [29] , H1N1 [30 , 31] , H1N2 [32] , H3N3 [33] , H5N1 [34] , H6N6 [35] , and H9N2 [36 , 37] IAVs have been reported in domestic swine . Under laboratory conditions , avian IAV subtypes H1\\u2013H13 can infect and replicate in swine at different levels of susceptibility [20] . However , among these avian IAVs , only subtypes H1N1 and H3N2 have become enzootic in domestic swine; the other avian-origin IAVs have transiently infected domestic swine . Novel reassortants with avian IAVs genes have rarely been detected in swine . By housing IAV-seronegative pigs together with H3N2 IAV\\u2013inoculated and avian H1N1 IAV\\u2013inoculated pigs , we demonstrated that 1 ) a large number of reassortants were generated in the pigs\\u2019 lower respiratory tracts , but only a few of the reassortants were shed; and 2 ) tissue tropisms , especially replication ability in swine nasal epithelium cells at 33\\u00b0C , are keys to selecting a reassortant virus that can be shed . Reassortment between two IAVs of different or the same origins have been well documented [7\\u20139 , 11 , 14\\u201319 , 38] . Previous studies tested the compatibility of emerging avian IAVs and endemic human influenza viruses , such as the compatibility of avian H9N2 with human H1N1 [39 , 40] , avian H5N1 with human H3N2 [8 , 10 , 41] , and avian H5N1 with human H1N1 [42] . Gene compatibility between viruses has been shown to be subtype- and strain-dependent; however , transmissible viruses are possibly derived from genetic reassortments between avian and mammalian IAVs [9] . Results from our study suggested that 30 of 254 possible reassortant genotypes between an avian H1N1 IAV and a swine H3N2 IAV were detected , and most were in the middle and lower respiratory tracts of the experiment pigs; only two genotypes of reassortants ( i . e . , genotypes R3 and R8; Fig 3 ) shed from these pigs , and these genotypes of reassortants were also detected in middle ( genotype R3 ) and/or lower respiratory tracts ( genotypes R3 and R8 ) of the pigs . Study findings suggest that reassortments between avian and swine IAVs are tissue-dependent and non-random , and , in swine , they occur most frequently in the lower respiratory tract . Both in vivo and in vitro studies have suggested that the timing and dose of inoculation , as well as inoculation location , contribute to the efficiency of reassortment between influenza viruses [15\\u201318] . In this study , we mimicked the natural scenario of reassortment generation by housing IAV-seronegative pigs with swine H3N2 IAV\\u2013infected pigs and avian H1N1 IAV\\u2013infected pigs . The IAV reassortant viruses were detected as early as 5 dpi , and more on 7 dpi; reassortants were detected not only in the contact pigs but also in the pigs infected with swine H3N2 IAV and those infected with avian H1N1 IAV . Such results suggest that genetic reassortments can readily occur and generate genomically diverse reassortants if avian IAVs are introduced into a herd with circulating swine IAVs . In this study , the distribution of reassortants indicate obvious tissue tropism , with most reassortants occurring in middle and lower respiratory tissues and a few in upper respiratory tissue . We conducted further viral growth studies for 52 representative reassortants by using an in vitro infection model of swine with three cell lines imitating the upper ( i . e . SNE ) , middle ( i . e . STE ) , and lower ( i . e . A549 ) respiratory tracts of swine . Results indicate that the reassortants detected in middle and lower respiratory tracts showed large variations in growth properties in SNE at 33\\u00b0C and STE at 37\\u00b0C; however , the reassortants detected in nasal wash fluids and turbinate can replicate well in SNE at 33\\u00b0C and in STE at 37\\u00b0C ( Figs 4 and 5 ) . These results suggest that tissue tropism contributes to the unbalanced distribution of reassortants through swine respiratory tract tissues . Receptor binding properties have been reported to be one of the key factors determining IAV host and tissue tropisms [43] . However , most ( 60 of 63 ) of the recovered reassortants from this study had HA and NA genes from swine H3N2 IAVs thus having the same receptor binding properties; receptor binding properties of IAVs do not seem to play a key role in the unbalanced distribution of these reassortants in respiratory tracts of feral swine [19 , 38] . Instead , it is likely the other gene segments , such as the ribonucleoprotein complex [44] and NS gene [45] , could interact with the host genes and formulate such tissue tropisms . Influenza surveillance showed a large number of genetic variants of subtype H1 and H3 IAVs are co-circulating and enzootic in domestic swine in the United States [46 , 47] . Phenotype analyses suggested that these reassortants can have large variation in tissue tropisms and transmissibility in mammals [48\\u201350] . Thus , in addition to receptor binding properties , genetic constellation is an important factor determining influenza tissue and host tropisms . Adapted mutations were also reported to affect viral infectivity and tissue tropisms [44] . A number of mutations or polymorphisms were observed across IAV gene segments ( S1 Text ) . Of interest , in swine H3N2 ( no reassortment ) isolates , mutation Q197R in HA protein exhibits obvious tissue-dependent distribution: the mutation occurs more frequently in virus recovered from nasal wash fluids and upper respiratory tracts than in those recovered from middle and lower respiratory tracts ( S4 Table ) . However , analyses of genomic sequences of human and swine H3N2 IAVs in the public databases showed that 93 . 11% of human H3N2 IAVs and 97 . 95% of swine H3N2 IAVs possess glutamine ( Gln , Q ) at residue 197 of HA protein , but just 4 . 86% of human H3N2 IAVs and 0 . 61% of swine H3N2 IAVs possess arginine ( Arg , R ) at the same site of HA protein ( S5 Table ) . We further compared the viral growth kinetics of plaque viruses with or without the Q197R mutation on HA protein under three different temperature conditions ( S4 Fig ) . Peak virus titers at 72 h post-inoculation for the viruses with the mutation did not increase growth efficiency; thus , this mutation does not seem to play an important role in determining the tissue tropisms of IAVs . In the United States , in recent years , it has been challenging to identify influenza seronegative domestic pigs due to influenza enzootics and nationwide implementation of influenza vaccination in domestic pigs . In this study , feral swine was used as the animal model in the animal experiment because it is much easier to obtain influenza seronegative feral swine . However , although we confirmed that these feral swine had not been exposed to brucellosis , pseudorabies and influenza A virus [21] , it is likely that these animals had been exposed to other pathogens , especially parasites . Of interests , prior studies suggested that environment changes for the animals used in the experiments affected the experimental results and that non-specific pathogen free animals could have immune systems closer to those of adult humans [51] . Nevertheless , results from our feral swine experiments clearly demonstrated that tissue tropism plays an important role in selecting transmissible reassortants from various reassortants generated in pigs co-infected with avian and swine IAVs . Compared with domestic pigs , feral swine play an important role in influenza ecology because of their increasing population and ability to roam freely [21 , 52] . In one study , 7 ( 19% ) of 37 feral swine had direct contact with domestic swine [53]; however , feral swine have more opportunities to become infected with avian IAVs because they can easily come into contact with feces or water contaminated by IAV-infected wild birds or with infectious dead birds . These characteristics make feral swine a potential intermediate host for reassortment and transmission of avian IAVs or genes to humans . Virologic and serologic surveillance showed evidence of feral swine exposed to both H3N2 and H1N1 IAVs , predominately H3N2 IAVs [52 , 54] . Furthermore , the H3N2 virus recovered from feral swine shared viral shedding pattern and antibody response dynamics similar to those for viruses isolated from domestic pigs [21 , 52] . Since 2009 , the pandemic H1N1 virus has been identified in domestic [52 , 55 , 56] and feral swine [57] in the United States . Therefore , avian and swine IAVs are continuously transmitted to feral swine , and this finding highlights the important role of feral swine in influenza ecology . Our findings indicate that reassortants between avian and swine IAVs can be shed nasally and be transmitted between feral swine with 100% efficiency under experimental conditions . Thus , feral swine could serve as a potential intermediate host for generating and transmitting reassortant viruses containing both avian and swine influenza virus genes to domestic swine and , ultimately , to humans . In summary , our study findings illustrate how transmissible reassortants arise between avian and swine IAVs in pigs . The reassortments occur in lower respiratory tracts , and tissue tropisms could have selected which reassortant viruses would be shed from the pigs . Determination of tissue tropisms for potential reassortants from contemporary avian and swine IAVs would help identify transmissible reassortants with public health risks . \\n The animal experiments were performed under the protocol numbers QA2296 titled Potential of Avian influenza A virus to infect feral swine and QA2494 titled Potential for feral swine and avian H3N2 influenza to cause a mixed infection in feral swine , both of which approved by the Institutional Animal Care and Use Committee of National Wildlife Research Center ( NWRC ) in accordance with the USDA Animal Welfare Regulations . Biosafety protocol for laboratory and animal experiments: Virus titration and purification and virus inoculation in feral swine were conducted under Biosafety Level 2 conditions , in compliance with U . S . Department of Agriculture\\u2013approved protocols of Institutional Animal Care and Use Committee and Institutional Biosafety Committee . Madin-Darby canine kidney ( MDCK ) cells , human alveolar adenocarcinoma ( A549 ) cells , and chicken embryo fibroblasts ( DF-1 ) cells were obtained from American Type Culture Collection ( Manassas , VA , USA ) . Cells were maintained in Dulbecco\\u2019s modified Eagle\\u2019s medium ( DMEM; Gibco , New York , USA ) supplemented with 10% fetal bovine serum ( FBS; Atlanta Biologicals , Lawrenceville , GA , USA ) at 37\\u00b0C under 5% CO2 . SNE and STE ( kindly provided by Dr . Stacy Schultz-Cherry , St . Jude Children\\u2019s Research Hospital , Memphis , TN , USA ) were cultured at 37\\u00b0C with 5% CO2 in DMEM/F12 ( Thermo Fisher Scientific , Asheville , NC , USA ) supplemented with fetal bovine serum ( 10% ) . Influenza A/swine/Texas/A01104013/2012 ( H3N2 ) virus was propagated for one passage on MDCK cells at 37\\u00b0C with 5% CO2 in Opti-MEM I Reduced Serum Medium ( Thermo Fisher Scientific , Asheville , NC , USA ) supplemented with 1 \\u03bcg/mL of TPCK-trypsin ( Gibco , New York , USA ) . Influenza A/mallard/Wisconsin/A00751454/2009 ( H1N1 ) virus was propagated for one passage in specific pathogen\\u2013free ( SPF ) 10-day-old chicken embryonated eggs ( Charles River Laboratories , Inc . , Norwich , CT ) . The viruses were titrated by the 50% tissue culture infective dose ( TCID50/mL ) on MDCK cells , aliquoted , and stored at \\u221280\\u00b0C before use . The Genbank access numbers for A/swine/Texas/A01104013/2012 ( H3N2 ) virus are JX280447 to JX280454 . The genomic sequences for A/mallard/Wisconsin/A00751454/2009 ( H1N1 ) have been deposited into Genbank with access numbers MH879773 to MH879780 . For viral titration , the TCID50 was determined on MDCK cells , and the 50% egg culture infective dose ( EID50 ) was determined on SPF 10-day-old chicken embryonated eggs ( Charles River Laboratories , Inc . , Norwich , CT ) . Hemagglutination and hemagglutination inhibition ( HI ) assays were carried out by using 0 . 5% turkey erythrocytes as previously described [58] . Viruses were purified by plaque assay on MDCK cell monolayers as described elsewhere [59] . In brief , nasal wash fluids or the supernatants of ground tissues from experiment animals were 10-fold serially diluted from 100 to 10\\u22126 with Opti-MEM containing 100 units/mL of penicillin\\u2013streptomycin ( Gibco , New York , USA ) and inoculated onto monolayers of MDCK cells in 6-well plates . Up to 10 single plaques were randomly picked for each sample . Plaques were propagated on MDCK cells for a single passage , and the viruses were aliquoted and stored at \\u221280\\u00b0C before being used for genomic sequencing and growth kinetics characterization . Viral RNA was extracted by using the KingFisher Pure Viral NA Kit ( Thermo Fisher Scientific , Asheville , NC , USA ) according to the manufacturer\\u2019s instructions . Viral cDNA transcript libraries were prepared using the Nextera XT v2 kit and were sequenced by using a MiSeq Reagent Kit v2 with a MiSeq sequencing system ( both from Illumina , San Diego , CA , USA ) according to the manufacturer\\u2019s suggested protocol [60] . The quality of pair-end reads obtained from MiSeq sequencing were checked by FastQC ( Babraham Bioinformatics , https://www . bioinformatics . babraham . ac . uk/projects/fastqc/ ) and trimmed by Trimmomatic version 0 . 36 ( Usadel Lab , http://www . usadellab . org/cms/ ? page=trimmomatic ) using a quality score threshold of 20 . After quality trimming was performed , all pair-end reads were aligned to reference genome of corresponding virus by using Bowtie 2 ( Johns Hopkins University , http://bowtie-bio . sourceforge . net/bowtie2/index . shtml ) . Polymorphisms were identified from alignments and subject to a 20% coverage cutoff . SNE , STE , A549 , DF-1 , and MDCK cells were seeded in 6-well plates , after the cells were confluent , cells were washed twice with PBS and then infected with a testing virus at a multiplicity of infection of 0 . 001 TCID50/cell . The inoculum was removed after 1 h of incubation at 37\\u00b0C . The cells were washed with PBS , and then 3 mL of Opti-MEM supplemented with 1 \\u03bcg/mL of TPCK-trypsin was added . Cultures were incubated at 33\\u00b0C , 37\\u00b0C , or 39\\u00b0C for the duration of the experiment . At 12 , 24 , 48 , and 72 h post-inoculation , virus titers were determined in the supernatants by TCID50 on MDCK cells [59 , 61 , 62] . Viruses were purified by sucrose gradient centrifugation . The purified viruses were dissolved in PBS and dialyzed against PBS at 4\\u00b0C overnight . The concentrations of virus particles were determined using sodium dodecyl sulfate\\u2013polyacrylamide gel electrophoresis as described elsewhere [63] . Two biotinlyated glycan analogs , 3\\u02b9SLN and 6\\u02b9SLN , were purchased from GlycoTech ( Gaithersburg , MD , USA ) . Binding of viruses ( at 5 nM/virus ) to the biotinylated glycan analogs was performed as previously described [64] in an Octet RED96 biolayer interferometer equipped with streptavidin biosensor tips ( PALL Fort\\u00e9BIO , Menlo Park , CA , USA ) . The glycan concentrations ranged from 0 . 007 \\u03bcg/mL to 1 . 5 \\u03bcg/mL . Responses were normalized by the highest value obtained during the experiment , and binding curves were fitted by using the binding-saturation method in GraphPad Prism version 7 ( https://www . graphpad . com/scientific-software/prism/ ) . The normalized response curves report the fractional saturation ( f ) of the sensor surface as described in a previous study [65] . RSL0 . 5 ( relative sugar loading , f = 0 . 5 ) was used to quantitate the binding affinity of two selected viruses against two glycan analogs . The higher the RSL0 . 5 , the smaller the binding affinity . For animal experiments , a total of 43 feral swine ( body weight 16\\u201322 kg ) were trapped in a rural area of Starkville , MS , USA , by using corral traps similar to those previously described [21] . Animals were transported to the National Wildlife Research Center , Mississippi Field Station , in Mississippi State , MS , USA , where they were quarantined for 1 week . Before the animals were included in the experiments , we confirmed that these feral swine had not been exposed to brucellosis , pseudorabies and influenza A virus using ELISA as previously described [21] and that all HI assay results were negative for A/swine/Texas/A01104013/2012 ( H3N2 ) , A/mallard/Wisconsin/A00751454/2009 ( H1N1 ) , and three endemic human influenza A viruses [i . e . , A/California/04/2009 ( H1N1 ) , A/Switzerland/9715293/2013 ( H3N2 ) , and A/Hong Kong/4801/2014 ( H3N2 ) ] . The swine were then housed and fed according to standard protocol , as described elsewhere [21] , and used in three animal experiments . To ensure that the selected avian H1N1 virus could cause infection in and be transmitted among swine , we evaluated the pathogenesis and transmission of A/mallard/Wisconsin/A00751454/2009 ( H1N1 ) virus in the feral swine prior to the animal study . A total of 12 feral swine were randomly assigned to two groups: the treatment group ( n = 8 ) and the control group ( n = 4 ) . The treatment group pigs were housed in 4 adjoining pens ( 2 pigs/pen ) ; the control pigs were housed in 2 adjoining pens ( 2 pigs/pen ) ; and the treatment pigs and the control pigs were housed in two individual buildings . Prior to virus inoculation and samples collection , pigs were anesthetized using a method described previously [21] . After being anesthetized , the eight pigs in the treatment group were intranasally inoculated with 106 EID50 of virus in a volume of 1 mL ( 0 . 5 mL/nostril ) , and the four pigs in the control group were intranasally inoculated with 1 mL of PBS . On 1\\u201310 dpi , nasal wash fluids were collected from both nostrils of all pigs into 3 mL of PBS and then subjected to titration by EID50 . The body weight and temperature of each pig were measured before samples were taken . At 5 and 7 dpi , respectively , two treatment group pigs and one control pig were euthanized and necropsy was performed according to a previously described procedure [21] . Turbinates , tracheas , and lungs were collected and homologized for viral titration by EID50 . Serum from each pig was also collected for seroconversion analysis using an HI assay with 0 . 5% turkey red blood cells , as described previously [58] . A total of 19 pigs were separated into 5 treatment groups ( 16 pigs in total ) and 1 control group ( 3 pigs in total ) . Each treatment group ( n = 3 or 4 pigs ) consisted of one pig intranasally inoculated on day 0 with 106 TCID50 of A/swine/Texas/A01104013/2012 ( H3N2 ) virus in a volume of 1 mL; one pig inoculated on day 0 with 106 TCID50 of A/mallard/Wisconsin/A00751454/2009 ( H1N1 ) virus in a volume of 1 mL; and one or two IAV seronegative pigs to serve as a direct contact ( s ) for the inoculated pigs . For each experimental group , one or two IAV seronegative pigs were introduced into the same pen housing one pig inoculated with swine H3N2 IAV and one pig inoculated with avian H1N1 IAV right after ( at the same day ) viral inoculation . To avoid potential contaminations from virus inoculation , the inoculation procedure for each inoculated pig was performed outside of the animal\\u2019s housing pens on a stainless steel table which was disinfected between each animal . The control group was housed in a separate building and intranasally inoculated with 1 mL of sterile PBS . Nasal wash fluids were collected from all pigs on 0 , 3 , 5 , and 7 dpi and titrated by TCID50 on MDCK cells . All pigs in two of the five treatment groups were euthanized on 3 dpi and 5 dpi , respectively , and pigs in the remaining treatment group were euthanized on 7 dpi . One control pig each was euthanized on 3 , 5 , and 7 dpi . The following tissues were collected from each euthanized pig: left cranial lung , left caudal lung , left middle lung , right cranial lung , right caudal lung , right middle lung , right accessory , upper trachea , middle trachea , distal trachea , bronchus , soft palate , ethmoid turbinate , rostral turbinate , and middle turbinate . To quantify IAV , we homogenized each tissue to a 10% ( w/v ) final concentration in PBS containing 100 unit/mL of penicillin\\u2013streptomycin and then subjected the solutions to three freeze\\u2013thaw cycles prior to performing virus titrations and plaque analyses . We used a total of 12 pigs to evaluate the pathogenesis and transmission of a reassortant ( plaque #69 [genotype R3] ) which resulted from a mixed infection and has PB1 gene from A/mallard/Wisconsin/A00751454/2009 ( H1N1 ) virus and other genes from A/swine/Texas/A01104013/2012 ( H3N2 ) virus . We randomly assigned the pigs into four groups of experiment pigs ( n = 8 ) and one group of control pigs ( n = 4 ) . Each experiment group consisted of 1 inoculated pig , which received 106 TCID50 of a nasal isolate of IAV on day 0 , and 1 contact pig . Nasal wash fluids were obtained from each pig on 0 , 3 , 5 , 7 , 9 , 11 , and 14 dpi; serum samples were collected from each pig on 0 , 7 , 14 , and 21dpi . On 3 , 5 , 7 , and 9 dpi , one virus-inoculated pig and one control pig were euthanized , and viruses in nasal wash specimens and animal tissues were titrated by TCID50 on MDCK cells; serum samples were subjected to seroconversion analyses by HI assays . Viral titers are expressed as means \\u00b1 standard deviations . The one-way ANOVA test was used to determine whether viral growth in each cell line at 33\\u00b0C or 39\\u00b0C differed from growth at 37\\u00b0C at each time point . Differences with a P-value of less than 0 . 05 were considered statistically significant .\",\n          \"Insulator or enhancer-blocking elements are proposed to play an important role in the regulation of transcription by preventing inappropriate enhancer/promoter interaction . The zinc-finger protein CTCF is well studied in vertebrates as an enhancer blocking factor , but Drosophila CTCF has only been characterised recently . To date only one endogenous binding location for CTCF has been identified in the Drosophila genome , the Fab-8 insulator in the Abdominal-B locus in the Bithorax complex ( BX-C ) . We carried out chromatin immunopurification coupled with genomic microarray analysis to identify CTCF binding sites within representative regions of the Drosophila genome , including the 3-Mb Adh region , the BX-C , and the Antennapedia complex . Location of in vivo CTCF binding within these regions enabled us to construct a robust CTCF binding-site consensus sequence . CTCF binding sites identified in the BX-C map precisely to the known insulator elements Mcp , Fab-6 , and Fab-8 . Other CTCF binding sites correlate with boundaries of regulatory domains allowing us to locate three additional presumptive insulator elements; \\u201cFab-2 , \\u201d \\u201cFab-3 , \\u201d and \\u201cFab-4 . \\u201d With the exception of Fab-7 , our data indicate that CTCF is directly associated with all known or predicted insulators in the BX-C , suggesting that the functioning of these insulators involves a common CTCF-dependent mechanism . Comparison of the locations of the CTCF sites with characterised Polycomb target sites and histone modification provides support for the domain model of BX-C regulation . \\n Insulator elements are DNA sequences that regulate interactions between promoters and enhancers . By preventing inappropriate enhancer/promoter communication , insulators are believed to play a key role in the genomic organisation of transcriptional regulation . Their mode of action is still unclear but may involve the formation of chromatin loops that partition the genome into separate regulatory domains [1\\u20135] . In vertebrates , almost all characterised insulator elements are associated with the binding of CTCF , a DNA-binding protein that contains multiple zinc fingers . Although CTCF was initially identified as both a transcriptional activator and repressor [6\\u20138] , it was subsequently recognised as being essential for the enhancer blocking activity of several vertebrate insulators [9] . CTCF also functions in imprinting [10 , 11] and has been implicated in human disease [12] . Recently , Drosophila CTCF has been identified [13] , joining other known Drosophila enhancer blocking proteins such as Su ( Hw ) [14] , Zw5 , and BEAF32 [15 , 16] . In addition to insulation of entire genes or groups of genes , insulators may also flank individual enhancers allowing them to act independently , facilitating complex tissue and cell-specific patterns of gene expression [17] . This function is particularly relevant in the case of the Hox genes , whose complex expression patterns specify segmental identities along the body axis . In Drosophila , correct antero-posterior patterning in the thorax and abdomen is dependent on the precise expression of the Hox genes of the Bithorax complex ( BX-C ) in specific parasegments [18 , 19] . This is achieved by the subdivision of the regulatory regions of each of the three BX-C genes ( Ultrabithorax [Ubx] , abdominal-A [abd-A] , and Abdominal-B [Abd-B] ) into distinct enhancer domains [20] . There are at least nine distinct regulatory regions , each important for specifying homeotic gene expression in individual thoracic and abdominal parasegments ( PS ) from PS 5\\u201313 [21\\u201325] . The domain hypothesis of Mihaly et al . [26] proposes that each distinct regulatory region or domain contains a modular arrangement of functional elements required for Hox gene expression in a particular parasegment . These elements include initiator , enhancer , and memory elements/Polycomb-response elements ( PREs ) . It is thought that boundary elements , located between adjacent domains , restrict the influence of each regulatory region . The evidence for this comes from mutations that disrupt boundary function and from enhancer trap transposon studies , which have generated a map of the BX-C compartmentalised into distinct parasegmental regulatory regions [27 , 28] . Three boundaries Mcp , Fab-7 , and Fab-8 have been defined by mutation [29\\u201333] . Another , Fab-6 , has been mapped genetically [26] , and others are postulated to exist . Each of the three BX-C boundaries identified by mutational analysis display insulator function; i . e . , they are capable of suppressing reporter gene expression when placed between an enhancer and a promoter in a transgenic insulator assay [4 , 29 , 34\\u201336] . Recently , Moon et al . [13] showed that the Fab-8 boundary element contains binding sites for CTCF and that mutation of these sites greatly reduces the ability of Fab-8 to suppress reporter gene expression in an insulator assay , demonstrating that the insulating activity of Fab-8 is dependent on CTCF . Here we use chromatin immunopurification together with genomic microarray ( ChIP-array ) to investigate in vivo CTCF binding in several regions of the Drosophila genome , including the BX-C . From this analysis , we identify a CTCF binding-site consensus that allows the precise location of CTCF binding sites in these genomic regions . In the BX-C , in addition to the characterised CTCF sites in the Fab-8 boundary element , we demonstrate the presence of CTCF binding sites in the Mcp and Fab-6 boundaries . Furthermore , we identified CTCF binding sites between the regulatory regions bxd/pbx and iab-2 , between iab-2 and iab-3 , and between iab-3 and iab-4 , providing both a localisation of the previously postulated boundary regions of \\u201cFab-2 , \\u201d \\u201cFab-3 , \\u201d and \\u201cFab-4\\u201d and a demonstration that these too bind CTCF . A number of CTCF binding sites have been identified in the vertebrate genome , but there is little agreement as to the similarity between these sites at the DNA level [7 , 13 , 37] . Binding data have been interpreted to suggest that different combinations of zinc fingers are used to bind to differing sites of approximately 50 bp [7 , 38] . In contrast , our analysis in Drosophila indicates that CTCF sites contain a conserved consensus binding sequence of approximately 20 bp in length . Examination of the vertebrate CTCF binding sites reveals that they too contain a consensus sequence and that this vertebrate CTCF consensus is similar to the Drosophila site identified here . \\n In order to identify the in vivo binding sites of the Drosophila CTCF protein , we used our previously described ChIP-array procedure [39] . Sonicated chromatin , isolated from Drosophila embryos , was immunopurified using either anti-CTCF antiserum ( specific immunopurification [IP] ) or normal rabbit serum ( control IP ) . The immunopurified DNA preparations were labelled with either Cy3 or Cy5 and hybridised to a 1-kb tiling-path genomic microarray covering the 3-Mb Adh region together with other selected genomic regions including the BX-C , the Antennapedia complex ( ANT-C ) , and the achaete-scute region . As a positive control , the immunopurification reactions were assessed using specific PCR primers to amplify a 378-bp fragment from the Fab-8 region , containing characterised CTCF binding sites [13] . This fragment showed clear enrichment when compared with amplification using primers for a 300-bp fragment ( Clone 10 ) that does not contain a CTCF binding site ( unpublished data ) . Replicated hybridisation to genomic DNA tiling arrays generated a dataset ( Dataset S1 ) with mean enrichments ( Mn ) equivalent to log2 3 . 8 ( 14-fold ) observed . The Fab-8 positive control is represented on the array as fragment UBX65 , which gave an enrichment value of 1 . 56 ( 3-fold ) and good reproducibility ( p = 0 . 0045 across four biological replicates ) . Fragments showing Mn > 0 . 45 ( 1 . 4-fold ) and p < 0 . 05 were selected as potential CTCF binding sites ( Figure 1 ) . A total of 33 fragments satisfied these criteria , including 18 from the 3-Mb Adh region , nine fragments from the BX-C , and four from the ANT-C ( Dataset S2 ) . To identify potential CTCF binding sites within enriched fragments , the 33 candidate fragments were submitted to a motif discovery tool , Multiple Em for Motif Elicitation ( MEME ) , to search for overrepresented sequence motifs [40] . The top motif found by MEME ( e = 1 . 3 \\u00d7 10\\u221220 ) ( Figure 1A ) is identified in 23 out of the 33 fragments ( 70% ) . Of the remaining ten fragments , four are immediately adjacent to fragments that possess a match to this MEME motif . Of the 23 fragments that contain the top MEME motif , seven contain two sites resulting in a total occurrence of 30 sites . Acting as a positive control , both of the CTCF binding sites identified in the Fab-8 region by DMS methylation interference [13] are identified by our MEME analysis . The 18-bp MEME motif is illustrated as a sequence logo in Figure 1A and shows rather weak 5\\u2032 sequence preference but a strong AGGTGGCGC consensus towards its 3\\u2032 end . The 30 occurrences of the MEME motif were used to construct a position-specific weight matrix that was in turn used as the input for the Patser profile-matching tool to search for matches within the genomic sequences on the microarray . The association between Patser matches and CTCF binding is demonstrated in Figure 1B and 1C . In Figure 1B the top 600 fragments are plotted in rank order by Mn along with the minimum p-value Patser site associated with each fragment . Clearly the high-ranking enriched fragments are generally associated with good matches to the CTCF weight matrix . Figure 1C shows that match to the CTCF weight matrix is a relatively good predictor of binding; of the fragments containing a better than p = 10\\u221213 . 5 match , 57% have Mn > 0 . 45 ( 1 . 4-fold ) , and for p < 10\\u221215 , 70% have an enrichment >0 . 45 . At a whole-genome level , we find 855 matches ( p < 10\\u221215 ) when Patser is used to search the Drosophila melanogaster genome sequence . Another way to examine the functional relevance of these predicted sites is to look at their conservation across species . Figure 1D shows the conservation across 14 insect species aligned with D . melanogaster for the 855 matches within the Drosophila genome with Patser p < 10\\u221215 , plotting the median conservation across the CTCF binding motif together with 100 bp of flanking sequences . The plot shows a clear peak of conservation aligned with the 18-bp CTCF weight matrix matches . The conservation peak also appears to be approximately 20 bp wide and , as with the CTCF weight matrix , the conservation is greater towards the 3\\u2032 end of the motif . The conservation plot additionally suggests that conserved and hence potentially functionally relevant sequences may extend a few base pairs 3\\u2032 to the 18-bp CTCF binding motif . Taken together , these data support the idea that the binding sites for CTCF in Drosophila can be described by a single weight matrix approximately 20 bp in length . This is clearly at odds with the notion , derived from studies of CTCF DNA binding in vertebrates , that CTCF binds to 50-bp target sites with a diverse spectrum of sequences [7 , 38] . The ChIP-array analysis identifies eight locations with CTCF binding within the BX-C . As shown in Figure 2 , these eight locations show a striking correspondence with Patser CTCF site predictions as all eight locations overlap sites with a Patser p < 10\\u221213 . In addition , another UBX fragment ( UBX200; Mn = 0 . 54 and p = 0 . 054 ) that falls just short of the significance threshold also contains a high-scoring Patser site ( p = 10\\u221214 . 4 ) . ChIP enrichment of this fragment was validated by specific PCR , and this location was therefore included in the list of CTCF occupied sites , resulting in the identification of nine CTCF binding regions across the BX-C as depicted in Figure 2 . Of these regions six contain a single p < 10\\u221213 Patser match and three contain a pair of sites , separated by less than 200 bp ( Table 1 ) . We performed ChIP experiments with crosslinked chromatin from both Drosophila S2 cells and embryos to validate this set of CTCF sites ( Figure 3A ) . To control the specificity of the immunopurifications , we used two different CTCF antibodies , raised against the N- or C-terminal part of the protein . For negative controls we used pre-immune serum as well as a nonbinding sequence , encompassing a sequence upstream of Fab-8 ( Fab-8 5\\u2032-control ) . The results indicated association of CTCF with the same set of fragments that were enriched in the ChIP-array . A total of two fragments with strong Patser matches , but no enrichment in the ChIP-array ( BXC-67 and BXC-168 ) , also showed no clear enrichment in the fragment-specific ChIP assay . Since the nine CTCF binding regions show both ChIP enrichment and high-scoring Patser matches , they are likely to be direct CTCF targets rather than products of indirect association through , for example , chromatin looping . To substantiate this we analysed DNA binding in vitro by electrophoretic mobility shift assay ( EMSA ) with radioactively labelled probes and bacterially expressed purified GST-CTCF fusion proteins ( Figure 3B ) . We used either the 11-zinc-finger DNA-binding domain or the full-length protein fused to GST , which differentially retard the DNA fragments . The GST domain by itself does not bind , nor did a negative control sequence from Su ( var ) 3\\u20139 bind to CTCF . These experiments show direct binding of CTCF to all of the sites predicted by the Patser analysis , even to BXC-67 and BXC-168 , which were very weakly , if at all , enriched by ChIP . Some of the sites were bound by the full-length protein , but not by the isolated zinc-finger-region of CTCF . This may suggest that other amino acids in regions outside of the zinc-finger region participate in DNA binding of CTCF . The double sites at BXC-148 ( A + B ) cannot be resolved on two different fragments , rather single and double occupancy can be seen with the zinc-finger domain causing two different shifted bands . Overall , our data strongly indicate that the CTCF sites we have identified in the BX-C are direct CTCF binding sites . As shown in Figure 2 , we find a striking correspondence between in vivo CTCF binding and mapped boundary elements in the BX-C . In addition to the known binding of CTCF at the Fab-8 insulator [13] , we show that CTCF binding is also detected within the mapped domains of the Mcp and Fab-6 boundary elements . The remaining mapped boundary , Fab-7 , shows neither significant CTCF binding in the ChIP-array analysis nor a Patser site p < 10\\u221213 . However , using the more sensitive PCR assay of ChIP enrichment , we do observe a relatively weak but significant association of CTCF with Fab-7 ( Figure S1 ) . Given the strong connection of CTCF sites to mapped boundary elements , we investigated whether the other CTCF sites within the BX-C also identified boundaries . The positions of boundary elements can be estimated from the mapping of mutations that affect the individual parasegment-specific regulatory elements , and the extents of these cis-regulatory domains ( taken from Maeda and Karch [28] ) are indicated by the coloured bar in Figure 4 . Again we find a clear correspondence; CTCF sites are located close to the boundaries between bxd/pbx and iab-2 ( the \\u201cFab-2\\u201d boundary ) , between iab-2 and iab-3 ( the \\u201cFab-3\\u201d boundary ) , and between iab-3 and iab-4 ( the \\u201cFab-4\\u201d boundary ) . However , mapping of these regulatory domains can be imprecise , particularly if the mutations are chromosomal rearrangements with complex effects . A more robust map of the regulatory domains is provided by the locations of enhancer-trap insertions and the analysis of their patterns of expression [27 , 28] . The positions of 14 enhancer traps in the BX-C are shown in Figure 4 ( coordinates in Table 1 ) , together with their allocations to specific regulatory domains based on their expression patterns . Again we find that CTCF binding sites separate different regulatory domains . The location of the Fab-4 site is particularly compelling; enhancer traps detecting iab-3 and iab-4 regulation are separated by less than 2 kb , and this interval contains the \\u201cFab-4\\u201d CTCF site . Of the remaining three CTCF binding sites with the BX-C ( sites A\\u2013C , Figure 4 ) , two sites are within introns of Abd-B close to alternative Abd-B transcription start sites ( Abd-B-RB and Abd-B-RE , respectively ) . The third site lies within the bxd/pbx regulatory region . In summary , the CTCF sites identified here correlate with six out of seven known or postulated boundary elements , the only exception being Fab-7 . As CTCF has been demonstrated to be required for insulator function at Fab-8 [13] , it is likely that all these CTCF-associated boundaries function through a common CTCF-dependent mechanism . According to the domain model of BX-C regulation , the domains bounded by insulators would act as autonomous units that could either be active or silenced depending on the state of memory elements/PREs within each domain [26] . This is likely to require a precise arrangement of insulators and PREs to restrict PRE-dependent chromatin modification to specific domains . Several PREs have been mapped within the BX-C and , in particular , PREs have been located close to the boundary elements Mcp , Fab-7 , and Fab-8 [29 , 33 , 41] . We were interested in examining the relationships between CTCF sites , the binding sites for Polycomb complexes , and the domains of chromatin modification . For this analysis we compared our CTCF ChIP-array data with a genome-wide analysis of Polycomb targets in Drosophila that determined the genomic binding sites for Polycomb Repressive Complex 1 ( PRC1 ) complex components ( Pc and Psc ) , for the Polycomb Repressive Complex 2 ( PRC2 ) complex component E ( Z ) , and for the PRC2-dependent chromatin modification , trimethylation of histone H3 lysine 27 ( H3K27me3 ) , in S2 cells [42] . In this particular cell line the Abd-B gene is expressed; the four downstream Abd-B promoters are active , but the most upstream promoter ( Abd-B-RE ) is silenced . Schwartz et al . [42] found that the Fab-7 and Fab-8 PREs are not bound by Polycomb complexes , and the Abd-B transcription unit is largely within an \\u201copen\\u201d domain devoid of H3K27me3 histone modification . In Figure 5 , we display the relationship between these Polycomb data [42] and our in vivo CTCF binding data in the region from \\u201cFab-4\\u201d to the 5\\u2032 end of the Abd-B transcription unit . Strikingly , the four Polycomb target sites in this region that are occupied by Polycomb complexes in S2 cells are all located in close proximity to CTCF binding sites . Furthermore , the Polycomb target peak always lies to one side of the CTCF site suggesting the relative arrangements as indicated in the schematic in Figure 5A . However , although the CTCF sites are precisely located , the Polycomb target sites are represented by peaks that span several hundred base pairs leaving some uncertainty as to the precise location of the Polycomb target sequences . Nevertheless , for Mcp this ordering agrees with the functional mapping where the PRE and the boundary have been mapped to adjacent but separate regions as illustrated in Figure 5B [4 , 41] . Overall , this arrangement suggests that each of these Polycomb target site sits within a domain flanked by CTCF boundaries . The H3K27me3 profile also shows a relationship to the location of CTCF sites . The most prominent feature of the H3K27me3 profile in S2 cells is the domain between approximately 12 , 725 , 000 and 12 , 795 , 000 , which lacks the repressive trimethylation of lysine 27 ( K27me3 ) modification . The right-hand side of this domain has a sharp border that corresponds well with the CTCF site \\u201cC\\u201d at 12 , 795 , 406 . The left-hand side of the domain does not have a clear border and does not fit with a CTCF site . It is tempting to speculate that the differences in the two borders of the H3K27me3 domain may be related to the relative arrangement of the CTCF and Polycomb sites . On the left-hand side , the Polycomb site is \\u201coutside\\u201d the CTCF site , and the H3K27me3 modification spreads rightwards from the Polycomb site . On the right-hand side , the Polycomb site is \\u201cinside\\u201d the CTCF site , and the H3K27me3 modification does not spread past the CTCF site . We also note that the positions of the CTCF site/PREs at \\u201cFab-4 , \\u201d Mcp , and Fab-8 are associated with pronounced depressions in the K27me3 profile . This may be related to nucleosome depletion at PREs [43] , but it is interesting that CTCF binding sites in the mouse \\u00df-globin locus are also depleted for repressive chromatin marks [44] . We examined the conservation of the CTCF sites in the BX-C . The sites show high conservation with median PhastCons scores close to 1 . 0 across the approximately 20-bp motif . We illustrate this for an individual site Mcp ( Figure 5B ) , where the single CTCF site corresponds to one of the clearly defined peaks of conservation that lie within the functionally mapped Mcp boundary element . Other genomic regions screened for CTCF binding sites on the microarray include the 3-Mb Adh region and the smaller Antennapedia and achaete-scute regions . The Adh region [45] is a well-characterised region of Chromosome 2L , containing approximately 250 genes from kuzbanian to cactus , which serves as a representative region of the fly genome . The ChIP-array identified 18 fragments in the Adh region with Mn > 0 . 45 ( 1 . 4-fold ) and p < 0 . 05 ( Figure 6 ) . Of these fragments , 11 contain high-scoring Patser CTCF weight-matrix matches ( Patser p-value < 10\\u221212 ) , and two additional fragments , ADH-1602 and ADH-2233 , are associated with flanking high-scoring Patser sites at distances of only 40 and 100 bp , respectively . In both cases these high-scoring Patser sites lie between two adjacent array fragments and so were not included as hits in the MEME analysis . PCR validation shows clear enrichment in both these cases ( unpublished data ) . It was found that two fragments that lack strong Patser sites are neighbours of highly enriched fragments that do contain high-scoring Patser sites . ADH-2253 has an additional high scoring hit in a neighbouring fragment that did not show up as being enriched on the array . Overall , this results in a strong association between ChIP-array enrichment and the presence of a high-scoring Patser site , with a total of 15 out of 19 fragments possessing either a high-scoring Patser site or immediately neighbouring a fragment with a high-scoring site . The enriched fragments are associated with 17 CTCF sites in total ( Figure 6 ) . Identification of CTCF binding sites within the Adh region presented an opportunity to investigate the relationship between CTCF binding sites and annotated genome features . Given CTCF's well-documented insulating function , it seemed likely that most identified sites would be in intergenic regions and this proved to be the case . Of the 17 sites in the ADH region , 15 ( 88% ) are present in intergenic regions . No sites overlap exons , but two sites present in ADH-705 overlap the 3\\u2032 UTR region of the protein kinase gene smell-impaired 35A ( smi35A ) . Most sites occur as single isolated sites ( 65% are separated by at least 500 bp ) , but there are three pairs of sites that are closer than 200 bp apart . Thus , in general , the CTCF sites in this region are not present in multisite clusters , but there are some closely spaced pairs of sites . Neighbouring sites , in general , flank several transcription units ( e . g . , the sites flanking CyclinE [CycE] shown in Figure 6B ) , but we also note that the long transcription unit of outspread ( osp ) , which extends for about 90 kb , is rather neatly flanked by a pair of 5\\u2032 CTCF sites and a single 3\\u2032 site ( Figure 6C ) . We compared the location of CTCF sites with the sites we have identified for another Drosophila insulator-binding protein , Su ( Hw ) ( B . Adryan , G . Woerfel , I . Birch-Machin , S . Gao , M . Quick , L . Meadows , S . Russell , and R . White; unpublished data ) . The Su ( Hw ) sites are illustrated in Figure 6 , and we find no clear general relationship between Su ( Hw ) and CTCF sites . Only one fragment contained both Su ( Hw ) and CTCF sites ( ADH-3002; 14 , 706 , 282\\u201314 , 707 , 212 ) . It is curious that this fragment was the top enriched fragment in both ChIP-array analyses . We also looked for an association between CTCF sites and gene neighbourhoods since insulators might provide a molecular basis for the occurrence of clusters of similarly expressed genes in the genome [46\\u201348] . In Figure 6 we illustrate the gene neighbourhoods in the Adh region identified by Spellman and Rubin [48] . Although neighbourhood 29 is precisely flanked by CTCF sites , and there is a site separating neighbourhoods 30 and 31 , overall we do not see a compelling association between CTCF sites and these gene expression neighbourhoods . A total of three out of the four ChIP-array enriched fragments in the Antennapedia region displayed a match to the top motif discovered by MEME . The remaining fragment is a neighbouring fragment . All three directly enriched fragments contain at least one high-scoring Patser site ( p < 10\\u221212 ) . In total , four sites are identified in the Antennapedia genomic region , and only one of the sites occurs in an intergenic region ( ANT297 ) . The remaining three sites are located within the first intron of Antennapedia itself . These sites consist of a pair of sites , 179 bp apart , and one \\u201csingle\\u201d site . Only a single fragment was identified within the achaete-scute complex , this contains a high-scoring Patser site ( p = 10\\u221214 . 5 ) and is present in the intergenic region between scute ( sc ) and lethal of scute ( l ( 1 ) sc ) . Although the existence of a region of similarity within different vertebrate CTCF binding sites has been noted [9 , 49] , a consensus binding site has not been universally recognised , mainly because of experiments that suggest that CTCF binds to DNA by employing varying combinations of different zinc fingers [7 , 50 , 51] . Following identification of the Drosophila CTCF consensus binding site , we examined the possibility that the vertebrate and Drosophila binding sites are similar in sequence . We utilised the selection of sites compiled by Moon et al . [13] and submitted these sequences to the Motif Discovery tool , MEME . The highest scoring motif identified ( e = 6 . 6 \\u00d7 10\\u221211 ) was found in all 12 sequences and is similar both to the conserved region identified previously in footprinting experiments and also to the Drosophila CTCF binding site reported here ( Figure 7 ) . Both the vertebrate and the Drosophila motifs share the AGGNGGC consensus sequence . Thus , our evidence does not support the idea that CTCF uses different combinations of zinc fingers to bind to different DNA sequences , and we suggest instead that CTCF binds to a similar specific sequence in both vertebrates and Drosophila . \\n The multiple zinc-finger DNA-binding protein CTCF is known to be required for the enhancer blocking action of vertebrate insulators , and a clear role for CTCF in the regulation of endogenous gene expression has been demonstrated at the imprinted Igf2 locus [9\\u201311] . The mode of action of CTCF is , however , still unclear , although several studies have implicated CTCF in the formation of higher-order chromatin structure . CTCF molecules can interact to form clusters and thereby may mediate the formation of chromatin loop domains [44 , 52\\u201354] . Partitioning of regulatory elements into independent chromatin loop domains is postulated to play a key role in the interactions between enhancers and promoters . Recently , a CTCF homolog was identified in Drosophila , and it was discovered that CTCF is required for the insulator function of the Fab-8 element in the BX-C [13] . This observation opened up the prospect of utilising the wealth of genetic and molecular characterisation of BX-C transcriptional regulation for the analysis of CTCF function . Here we have used ChIP-array to investigate CTCF binding sites in regions of the Drosophila genome with a particular focus on the BX-C . We find that CTCF is not only associated with the Fab-8 insulator , but also with other mapped boundary elements , Fab-6 and Mcp . In addition , we show that CTCF sites are located at other postulated boundaries within the BX-C; \\u201cFab-2 , \\u201d \\u201cFab-3 , \\u201d and \\u201cFab-4 . \\u201d This provides a precise mapping of regulatory domain boundaries and a specific molecular foundation for the domain model of BX-C regulation . We note that the Fab-7 boundary may differ from the other characterised boundaries in the BX-C as we do not find a strong Patser match to the CTCF consensus in the functionally mapped Fab-7 boundary element . Although Fab-7 was not demonstrably enriched in the ChIP-array , we found significant CTCF association with Fab-7 in the more sensitive PCR-base ChIP assay . Given the lack of a strong Patser match this may suggest an indirect association . We also do not see a CTCF site between the abx/bx and the bxd/pbx regulatory elements . However , these elements are separated by a long distance , and it is not clear whether they require insulation . According to the domain model [26] , the parasegment-specific regulatory domains that control the expression patterns of the Ubx , abd-A , and Abd-B genes of the BX-C are initially activated in appropriate parasegments by the early pattern-forming genes acting on initiator elements . Each regulatory domain is predicted to contain a particular initiator element , tuned to respond to a specific combination of gap and pair-rule gene products , thus activating the regulatory domain in the appropriate set of parasegments . This activation would be read by maintenance elements consisting of PREs that thereafter autonomously maintain each regulatory domain in either the OFF ( silenced ) or ON ( active ) state . Within a domain in the ON state , enhancers present in that domain would be able to engage with the relevant gene promoter and regulate expression of the gene . Boundary elements that flank each domain are proposed to restrict the effects of the initiator and maintenance elements to a single domain . Although boundary elements are postulated to have the common property of insulating the regulatory domains , no sequence similarity between the mapped boundary elements has been reported until now . Here we show that a set of these boundary elements contain CTCF binding sites and bind CTCF in vivo . CTCF has been shown to be required for the insulator activity of Fab-8 , and it seems likely that CTCF will also be a required component at the other boundary elements . In support of this suggestion , we find that the CTCF sites are well conserved within the sequenced insect genomes . The observation that CTCF sites flank a set of regulatory domains in the BX-C , together with the vertebrate studies that suggest that CTCF can mediate the formation of chromatin loops [44 , 52] supports the idea that interaction between CTCF sites may organise these domains into chromatin loops . However , how such a looping mechanism enables the autonomy of the individual regulatory domains and facilitates appropriate enhancer/promoter interactions is still unclear . A key feature of the domain model is the relationship between the boundary and maintenance elements . For the domains to be capable of independently being set to the ON or OFF state , the range of influence of PREs needs to be restricted by the domain boundaries . Each domain would require at least one PRE . Our precise mapping of in vivo CTCF binding sites has enabled us to examine their relationship with Polycomb target sites . In strong support of the domain model , we find that the domains demarcated by CTCF sites contain Polycomb target sites . Indeed , we find an intimate relationship between CTCF and Polycomb binding sites as shown in Figure 5 for \\u201cFab-4 , \\u201d Mcp , Fab-6 , and CTCF site \\u201cC . \\u201d This fits with previous functional mapping indicating that boundary elements and PREs are closely associated at Fab-7 , Fab-8 , and Mcp . This arrangement would impose a polarity on the spread of chromatin modification from the PRE , such that modification may start at the PRE abutting one boundary and spread across the domain in one direction towards the next boundary . At the boundaries , CTCF may play many possible roles . It could participate in boundary element function allowing the independence of chromatin domains by acting as a chromatin insulator blocking the spread of chromatin modification . However , at the chicken \\u00df-globin locus , the chromatin boundary appears to be separable from the CTCF binding site [55] . Another possibility is suggested by that fact that CTCF has been demonstrated to block the progression of RNA polymerase [56] . This could potentially play an important role at boundaries in the BX-C to enable the independent function of PREs in neighbouring domains . There is considerable evidence that transcription through PREs may control their state , and many noncoding RNAs have been detected in the regulatory regions of the BX-C [56\\u201362] . One role for CTCF could be to act as a barrier to such noncoding transcription , preventing transcripts arising in one regulatory domain from crossing into the neighbouring domain and affecting the PRE state . Such a role would be consistent with the observed location of CTCF sites in this region , as a CTCF site closely abuts one side of each PRE . The individual regulatory domains must not only be able to act autonomously to set and maintain their activity state , but they must also be able to interact appropriately with the relevant gene promoters . Boundaries may play a role in this , and recently Cleard et al . [63] have demonstrated a long-range interaction between Fab-7 and the Abd-B-RB promoter . This interaction was associated with lack of Abd-B expression , but similar interactions , bringing in appropriate enhancers , may also activate expression . The ability of CTCF to form clusters may facilitate such interactions , and it is intriguing that there are CTCF sites not only at the boundaries but also close to Abd-B promoters; the CTCF site \\u201cB\\u201d is 300 bp upstream of the Adb-B-RB promoter ( Figure 4 ) . Clustering of boundaries together with Abd-B promoter sequences may enable interaction between the promoter and enhancers in domains in the ON state . The clustering may also be more selective; in Figure 5 we see that in S2 cells , which specifically express Abd-B-RB , several boundaries are embedded in chromatin bearing the repressive H3K27me3 modification , whereas Fab-8 , CTCF site \\u201cB , \\u201d and the Abd-B-RB promoter are in the unmodified , presumably \\u201copen , \\u201d chromatin domain . We could speculate that the expression of Abd-B-RB in these cells might be facilitated by interaction of the CTCF sites in the \\u201copen\\u201d domain , Fab-8 and site \\u201cB , \\u201d enabling Fab-8 to bring appropriate enhancers to the Abd-B-RB promoter . We can compare this ChIP-array analysis of CTCF genomic sites with our ChIP-array analysis of binding sites for another Drosophila insulator-binding protein , Su ( Hw ) ( B . Adryan , G . Woerfel , I . Birch-Machin , S . Gao , M . Quick , L . Meadows , S . Russell , and R . White; unpublished data ) . CTCF and Su ( Hw ) are both multi-zinc- finger DNA-binding proteins , and in both cases we have identified relatively long ( ~20 bp ) consensus binding sites . In contrast to most DNA-binding proteins , we find that strength of match to the consensus binding sites is a good predictor of in vivo occupancy . We have also investigated whether our data indicate any collaboration between CTCF and Su ( Hw ) . This seemed an attractive possibility since removing Su ( Hw ) function in vivo has little effect; su ( Hw ) null mutant flies are female-sterile but viable . Also , the insulating activity of Fab-8 was significantly reduced when the CTCF sites were mutated but not completely abolished [13] . However we found no evidence for general colocalisation between CTCF and Su ( Hw ) . A total of 60 Su ( Hw ) sites were identified in the Adh region , and only one of the fragments covering this region contained both CTCF and Su ( Hw ) sites . The single CTCF site identified in the achaete-scute complex was also some distance from the two Su ( Hw ) sites we found . Subsequent ChIP-array analysis in the BX-C led to the identification of only one Su ( Hw ) site within the entire BX-C region , in a location devoid of CTCF binding sites ( B . Adryan , S . Russell , and R . White unpublished data ) . Indeed whilst the BX-C appears relatively enriched in CTCF sites compared to the Adh region , the converse is true for Su ( Hw ) . For CTCF there are 4 . 7 sites/100 kb in the BX-C and 1 . 7 sites/100 kb in the Adh region ( using Patser p < 10\\u221213 ) , whereas for Su ( Hw ) the BX-C is depleted in sites with only 0 . 29/100 kb in comparison to 2 . 7/100 kb in the Adh region ( using Patser p < 10\\u221215 ) . Clearly , although CTCF and Su ( Hw ) both possess insulating ability , their sites of action do not correlate and there is no evidence from our analysis , covering approximately 3% of the Drosophila genome , for cooperative activity . By comparing the sequences of ChIP-enriched fragments we identified a strong Drosophila consensus CTCF binding site . Analysis of vertebrate CTCF target sequences leads us to propose that vertebrate CTCF also binds to a similar consensus sequence . Our findings do not support the current view that CTCF binds to divergent DNA sequences by engaging different subsets of the zinc fingers [38 , 49 , 64] . Indeed , the binding site revealed here has been previously noted . Bell et al . [9] identified a CTCF binding site in the chicken \\u03b2-globin insulator , and sequence comparisons between this site and other known CTCF sites [6\\u20138] identified a conserved 3\\u2032 region , the mutation of which completely abolished CTCF binding and enhancer blocking . Filippova et al . [49] extended this comparison to include the Dm1 sites , mouse H19 DMD4 and DMD7 and human MYC A , and again identified a conserved region within the larger approximately 50-bp DNase footprint for each site . It is this conserved region that corresponds to the vertebrate CTCF site found here . Very recently , an analysis of CTCF binding in the human genome has generated a vertebrate CTCF consensus site [65] , and a CTCF consensus has also been derived from analysis of conserved regions in the human genome [66] . Both these sites are very similar to the consensus we identify here; in particular they share the strong features of the CC at positions 1 and 2 , the AG at positions 6 and 7 , and the GGC at positions 10 , 11 , and 12 . Overall , these findings indicate that CTCF in both Drosophila and vertebrates binds to a single core consensus sequence . In summary , ChIP-array analysis has enabled us to construct a CTCF binding site consensus . Mapping of genomic binding sites leads us to propose that all known or predicted insulators in the BX-C ( with the possible exception of Fab-7 ) function in a CTCF dependent manner . \\n The wild-type strain used was OregonR . The primary antibody used was rabbit anti-CTCF-C [13] . Chromatin from embryos aged between 0 to 20 h after egg laying was purified as described previously [39] . The 300-\\u03bcl immunopurification reaction contained 1 . 0 \\u03bcl of rabbit anti-CTCF antibody for the specific IP or 1 \\u03bcl of normal rabbit antiserum for the control IP . ChIP enrichment was assayed using PCR with specific primers as described previously [39] . The primers used were to Fab-8 ( UBX65 ) , catcttccgttcatccgtttc and tgttggtgagcaagcgaaga , and Clone 10 , attgggattctgcgattctg and tactgttcctggtgctggtg [13] . Validation ChIP assays for the CTCF sites in the BX-C were performed according to Moon et al . [13] . The validation ChIP primers are listed in Table S1 . The arrays used consist of 4 , 213 PCR products most of which are approximately 1 kb in length . The regions covered by the PCR products include the 3-Mb Adh region from kuzbanian to cactus , the BX-C and ANT-C regions , and 130 kb of the achaete-scute complex . Amplification and labelling of DNA from enriched chromatin and hybridisations to genomic DNA tiling arrays were carried out as described previously [39] . We used four biological replicates ( i . e . , independent chromatin preparations ) , and each of these was hybridised as dye-swap technical replicates giving 16 array hybridisations in total . Microarray scanning , spot-finding , and normalisation were performed as described in Birch-Machin et al . [39] and on the FlyChip Web site ( http://www . flychip . org . uk ) . The normalisation used VSN [67] , which is based on an arsinh transformation and generates an enrichment measure that is generally equivalent to the log2 Cy3/Cy5 ratio . Statistical significance was assessed using the CyberT framework ( http://visitor . ics . uci . edu/genex/cybert ) [68] . The MEME version 3 . 0 Web site [40] was used to identify a consensus sequence . Parameters were set to discover up to six motifs between ten to 30 nucleotides in length . The consensus sequence for the CTCF binding motif was depicted using the MEME site stack in WebLogo ( http://weblogo . berkeley . edu ) . The site stack for the CTCF binding motif was used to create a position-specific weight matrix ( Table S2 ) for the Patser Web interface ( http://rsat . ulb . ac . be/rsat/patser_form . cgi ) [69] . This position-specific weight matrix was used to search DNA sequences present on the array and the Drosophila genome for matches to the consensus sequence using Release 4 . 0 coordinates . Patser generates a score for each position and provides a p-value; this is the probability of observing a particular score or higher at a particular sequence position . The Affymetrix Integrated Genome Browser ( http://www . affymetrix . com/support/developer/tools/affytools . affx ) was used to visualise the in vivo CTCF binding profile across the genome . Analysis of the evolutionary conservation of CTCF motifs used the PhastCons multiple alignment data available from the University of California Santa Cruz ( Santa Cruz , California , United States ) Genome Browser Web site using phastCons15way on D . melanogaster genome Release 4 ( http://genome . ucsc . edu ) . Radiolabelled DNA probes ( 150\\u2013250 bp ) were generated by PCR with 32P-labelled oligonucleotide primers and prepared by subsequent gel purification . The probes were incubated with 0 . 2 \\u03bcg of purified GST , GST-CTCF , or GST-CTCF ZF . Recombinant proteins were prepared as described previously [70] . The binding reaction was performed in PBS ( [pH 7 . 4] , supplemented with 5 mM MgCl2 , 1 mM ZnCl2 , 1 mM DTT , 0 . 1% NP-40 , and 10% Glycerol ) for 15 min at room temperature in the presence of 200 ng/\\u03bcl pdIdC . Protein-DNA complexes were analysed on nondenaturing poylacrylamide gels ( 3 . 5% acrylamide [w/v] ) in TAE-buffer . Electrophoresis was performed at 4 \\u00b0C with a field strength of 12 V/cm for 3 h . \\n The Gene Expression Omnibus ( GEO ) ( http://www . ncbi . nlm . nih . gov/geo ) accession number for the genomic tiling array is GEO Platform GPL5028 XC003 and for the ChIP data is series GSE7351 . The Entrez Gene ( http://www . ncbi . nlm . nih . gov ) accession numbers of the genes discussed in this paper are: CTCF human , 10664 and Ctcf mouse , 13018 . The Flybase ( http://flybase . bio . indiana . edu ) accession numbers of the genes and gene products discussed in this paper are: abdominal-A ( abd-A ) , FBgn0000014; Abdominal-B ( Abd-B ) , FBgn0000015; achaete ( ac ) , FBgn0000022; Alcohol dehydrogenase ( Adh ) , FBgn0000055; Antennapedia ( Antp ) , FBgn0000095; BEAF32 , FBgn0015602; cactus ( cact ) , FBgn0000250; CTCF , FBgn0035769; Cyclin E ( CycE ) , FBgn0010382; Enhancer of zeste ( E ( z ) ) , FBgn0000629; kuzbanian ( kuz ) , FBgn0015954; lethal of scute ( l ( 1 ) sc ) , FBgn0002561; outspread ( osp ) , FBgn0003016; Polycomb ( Pc ) , FBgn0003042; Posterior sex combs ( Psc ) , FBgn0005624; scute ( sc ) , FBgn0004170; smell-impaired 35A ( smi35A ) , FBgn0016930; suppressor of Hairy wing ( su ( Hw ) ) , FBgn0003567; Su ( var ) 3\\u20139 , FBgn0003600; Ultrabithorax ( Ubx ) , FBgn0003944; and Zw5 , FBgn0000520 .\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"headings\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"keywords\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"id\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 138,\n        \"samples\": [\n          \"journal.pntd.0003195\",\n          \"journal.ppat.1007417\",\n          \"journal.pgen.0030112\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"biomistral_summary\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 138,\n        \"samples\": [\n          \" Glanders is a highly infectious zoonotic disease caused by the gram-negative bacterium Burkholderia mallei. It affects solipeds, particularly horses, and can be fatal to both animals and humans. Although eradicated from many Western countries, glanders recently emerged in Asia, the Middle-East, Africa, and South America. A recent outbreak of glanders in Bahrain was investigated using high-resolution genotyping and comparative whole genome sequencing. The samples from the outbreak in Bahrain separated into two distinct clusters, suggesting a complex epidemiological background and evidence for the involvement of multiple B. mallei strains. The samples from Bahrain were more closely related to B. mallei isolated from horses in the United Arab Emirates in 2004 than other B. mallei, suggesting repeated importation to the region from similar geographic sources. The recent emergence of glanders in combination with worldwide horse trading may pose a new risk for human infections.\",\n          \" The study aimed to determine the tissue tropism of a reassortant virus (plaque #69) that resulted from a mixed infection of avian H1N1 and swine H3N2 viruses in feral swine. The study used 12 pigs, which were randomly assigned into four groups of experiment pigs (n = 8) and one group of control pigs (n = 4). Each experiment group consisted of one inoculated pig and one contact pig. The inoculated pig received 106 TCID50 of a nasal isolate of IAV on day 0. Nasal wash fluids were obtained from each pig on 0, 3, 5, 7, 9, 11, and 14 dpi; serum samples were collected from each pig on 0, 7, 14, and 21 dpi. On 3, 5, 7, and 9 dpi, one virus-inoculated pig and one control pig were euthanized, and viruses in nasal wash specimens and animal tissues were titrated by TCID50 on MDCK cells; serum samples were subjected to seroconversion analyses by HI assays. The study found that the reassortant virus (plaque #69) replicated efficiently in the nasal wash fluids, trachea, and lungs of feral swine. The study also found that the reassortant virus (plaque #69) replicated efficiently in MDCK cells at 33\\u00b0C and 39\\u00b0C, but not at 37\\u00b0C. The study concluded that the reassortant virus (plaque #69) replicated efficiently in the respiratory tract of feral swine and that the virus replicated efficiently in MDCK cells at 33\\u00b0C and 39\\u00b0C.\",\n          \" The article describes the identification of a consensus binding site for the Drosophila insulator protein CTCF. The authors used ChIP-array analysis to identify genomic regions that are enriched for CTCF binding in vivo. They then used PCR to amplify fragments from these regions and used these fragments to construct a tiling array. They used this array to identify a consensus binding site for CTCF. The authors also used ChIP-array analysis to identify genomic regions that are enriched for CTCF binding in vivo. They then used PCR to amplify fragments from these regions and used these fragments to construct a tiling array. They used this array to identify a consensus binding site for CTCF. The authors also used ChIP-array analysis to identify genomic regions that are enriched for CTCF binding in vivo. They then used PCR to amplify fragments from these regions and used these fragments to construct a tiling array. They used this array to identify a consensus binding site for CTCF. The authors also used ChIP-array analysis to identify genomic regions that are enriched for CTCF binding in vivo. They then used PCR to amplify fragments from these regions and used these fragments to construct a tiling array. They used this array to identify a consensus binding site for CTCF. The authors also used ChIP-array analysis to identify genomic regions that are enriched for CTCF binding in vivo. They then used PCR to amplify fragments from these regions and used these fragments to construct a tiling array. They used this array to identify a consensus binding site for CTCF. The authors also used ChIP-array analysis to identify genomic regions that are enriched for CTCF binding in vivo. They then used PCR to amplify fragments from these regions and used these fragments to construct a tiling array. They used this array to identify a consensus binding site for CTCF. The authors also used ChIP-array analysis to identify genomic regions that are enriched for CTCF binding in vivo. They then used PCR to amplify fragments from these regions and used these fragments to construct a tiling array. They used this array to identify a consensus binding site for CTCF. The authors also used ChIP-array analysis to identify genomic regions that are enriched for CTCF binding in vivo. They then used PCR to amplify fragments from these regions and used these fragments to construct a tiling array. They used this array to identify a consensus binding site for CTCF. The authors also used ChIP-array analysis to identify genomic regions that are enriched for CTCF binding in vivo. They then used PCR to amplify fragments from these regions and used these fragments to construct a tiling array. They used this array to identify a consensus binding site for CTCF. The authors also used ChIP-array analysis to identify genomic regions that are enriched for CTCF binding in vivo. They then used PCR to amplify fragments from these regions and used these fragments to construct a tiling array. They used this array to identify a consensus binding site for CTCF. The authors also used ChIP-array analysis to identify genomic regions that are enriched for CTCF binding in vivo. They then used PCR to amplify fragments from these regions and used these fragments to construct a tiling array. They used this array to identify a consensus binding site for CTCF. The authors also used ChIP-array analysis to identify genomic regions that are enriched for CTCF binding in vivo. They then used PCR to amplify fragments from these regions and used these fragments to construct a tiling array. They used this array to identify a consensus binding site for CTCF. The authors also used ChIP-array analysis to identify genomic regions that are enriched for CTCF binding in vivo. They then used PCR to amplify fragments from these regions and used these fragments to construct a tiling array. They used this array to identify a consensus binding site for CTCF. The authors also used ChIP-array analysis to identify genomic regions that are enriched for CTCF binding in vivo. They then used PCR to amplify fragments from these regions and used these fragments to construct a tiling array. They used this array to identify a consensus binding site for CTCF. The authors also used ChIP-array analysis to identify genomic regions that are enriched for CTCF binding in vivo. They then used PCR to amplify fragments from these regions and used these fragments to construct a tiling array. They used this array to identify a consensus binding site for CTCF. The authors also used ChIP-array analysis to identify genomic regions that are enriched for CTCF binding in vivo. They then used PCR to amplify fragments from these regions and used these fragments\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}",
       "type": "dataframe",
       "variable_name": "mini_plos_df"
      },
      "text/html": [
       "\n",
       "  <div id=\"df-90af11ab-cc75-43e2-859e-7e1fb4392f25\" class=\"colab-df-container\">\n",
       "    <div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>lay_summary</th>\n",
       "      <th>article</th>\n",
       "      <th>headings</th>\n",
       "      <th>keywords</th>\n",
       "      <th>id</th>\n",
       "      <th>biomistral_summary</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Yersinia pestis , the bacterial agent of plagu...</td>\n",
       "      <td>Fleas can transmit Yersinia pestis by two mech...</td>\n",
       "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
       "      <td>[united states, invertebrates, medicine and he...</td>\n",
       "      <td>journal.ppat.1006859</td>\n",
       "      <td>The study by Hinnebusch et al. provides valua...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>The genome of all vertebrates is heavily colon...</td>\n",
       "      <td>Endogenous retroviruses ( ERVs ) are remnants ...</td>\n",
       "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
       "      <td>[viruses, sheep, virology]</td>\n",
       "      <td>journal.ppat.0030170</td>\n",
       "      <td>The authors present a comprehensive study of ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>The molecular mechanisms underlying directed c...</td>\n",
       "      <td>The Drosophila embryonic gonad is assembled fr...</td>\n",
       "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
       "      <td>[]</td>\n",
       "      <td>journal.pgen.1003720</td>\n",
       "      <td>The authors present a study on the role of Hm...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Contrary to the long-standing belief that no n...</td>\n",
       "      <td>Recently , we presented a study of adult neuro...</td>\n",
       "      <td>[Abstract, Introduction, Model, Results, Discu...</td>\n",
       "      <td>[computational biology/computational neuroscie...</td>\n",
       "      <td>journal.pcbi.1001063</td>\n",
       "      <td>The hippocampus is a brain structure that is ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Embryonic stem cells have two remarkable prope...</td>\n",
       "      <td>Understanding the transcriptional regulation o...</td>\n",
       "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
       "      <td>[developmental biology, cell biology, mammals,...</td>\n",
       "      <td>journal.pgen.0030145</td>\n",
       "      <td>The authors present a novel approach for iden...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>\n",
       "    <div class=\"colab-df-buttons\">\n",
       "\n",
       "  <div class=\"colab-df-container\">\n",
       "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-90af11ab-cc75-43e2-859e-7e1fb4392f25')\"\n",
       "            title=\"Convert this dataframe to an interactive table.\"\n",
       "            style=\"display:none;\">\n",
       "\n",
       "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
       "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
       "  </svg>\n",
       "    </button>\n",
       "\n",
       "  <style>\n",
       "    .colab-df-container {\n",
       "      display:flex;\n",
       "      gap: 12px;\n",
       "    }\n",
       "\n",
       "    .colab-df-convert {\n",
       "      background-color: #E8F0FE;\n",
       "      border: none;\n",
       "      border-radius: 50%;\n",
       "      cursor: pointer;\n",
       "      display: none;\n",
       "      fill: #1967D2;\n",
       "      height: 32px;\n",
       "      padding: 0 0 0 0;\n",
       "      width: 32px;\n",
       "    }\n",
       "\n",
       "    .colab-df-convert:hover {\n",
       "      background-color: #E2EBFA;\n",
       "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
       "      fill: #174EA6;\n",
       "    }\n",
       "\n",
       "    .colab-df-buttons div {\n",
       "      margin-bottom: 4px;\n",
       "    }\n",
       "\n",
       "    [theme=dark] .colab-df-convert {\n",
       "      background-color: #3B4455;\n",
       "      fill: #D2E3FC;\n",
       "    }\n",
       "\n",
       "    [theme=dark] .colab-df-convert:hover {\n",
       "      background-color: #434B5C;\n",
       "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
       "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
       "      fill: #FFFFFF;\n",
       "    }\n",
       "  </style>\n",
       "\n",
       "    <script>\n",
       "      const buttonEl =\n",
       "        document.querySelector('#df-90af11ab-cc75-43e2-859e-7e1fb4392f25 button.colab-df-convert');\n",
       "      buttonEl.style.display =\n",
       "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
       "\n",
       "      async function convertToInteractive(key) {\n",
       "        const element = document.querySelector('#df-90af11ab-cc75-43e2-859e-7e1fb4392f25');\n",
       "        const dataTable =\n",
       "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
       "                                                    [key], {});\n",
       "        if (!dataTable) return;\n",
       "\n",
       "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
       "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
       "          + ' to learn more about interactive tables.';\n",
       "        element.innerHTML = '';\n",
       "        dataTable['output_type'] = 'display_data';\n",
       "        await google.colab.output.renderOutput(dataTable, element);\n",
       "        const docLink = document.createElement('div');\n",
       "        docLink.innerHTML = docLinkHtml;\n",
       "        element.appendChild(docLink);\n",
       "      }\n",
       "    </script>\n",
       "  </div>\n",
       "\n",
       "\n",
       "<div id=\"df-09927302-4e73-48c9-9274-79b94bd85bf0\">\n",
       "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-09927302-4e73-48c9-9274-79b94bd85bf0')\"\n",
       "            title=\"Suggest charts\"\n",
       "            style=\"display:none;\">\n",
       "\n",
       "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
       "     width=\"24px\">\n",
       "    <g>\n",
       "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
       "    </g>\n",
       "</svg>\n",
       "  </button>\n",
       "\n",
       "<style>\n",
       "  .colab-df-quickchart {\n",
       "      --bg-color: #E8F0FE;\n",
       "      --fill-color: #1967D2;\n",
       "      --hover-bg-color: #E2EBFA;\n",
       "      --hover-fill-color: #174EA6;\n",
       "      --disabled-fill-color: #AAA;\n",
       "      --disabled-bg-color: #DDD;\n",
       "  }\n",
       "\n",
       "  [theme=dark] .colab-df-quickchart {\n",
       "      --bg-color: #3B4455;\n",
       "      --fill-color: #D2E3FC;\n",
       "      --hover-bg-color: #434B5C;\n",
       "      --hover-fill-color: #FFFFFF;\n",
       "      --disabled-bg-color: #3B4455;\n",
       "      --disabled-fill-color: #666;\n",
       "  }\n",
       "\n",
       "  .colab-df-quickchart {\n",
       "    background-color: var(--bg-color);\n",
       "    border: none;\n",
       "    border-radius: 50%;\n",
       "    cursor: pointer;\n",
       "    display: none;\n",
       "    fill: var(--fill-color);\n",
       "    height: 32px;\n",
       "    padding: 0;\n",
       "    width: 32px;\n",
       "  }\n",
       "\n",
       "  .colab-df-quickchart:hover {\n",
       "    background-color: var(--hover-bg-color);\n",
       "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
       "    fill: var(--button-hover-fill-color);\n",
       "  }\n",
       "\n",
       "  .colab-df-quickchart-complete:disabled,\n",
       "  .colab-df-quickchart-complete:disabled:hover {\n",
       "    background-color: var(--disabled-bg-color);\n",
       "    fill: var(--disabled-fill-color);\n",
       "    box-shadow: none;\n",
       "  }\n",
       "\n",
       "  .colab-df-spinner {\n",
       "    border: 2px solid var(--fill-color);\n",
       "    border-color: transparent;\n",
       "    border-bottom-color: var(--fill-color);\n",
       "    animation:\n",
       "      spin 1s steps(1) infinite;\n",
       "  }\n",
       "\n",
       "  @keyframes spin {\n",
       "    0% {\n",
       "      border-color: transparent;\n",
       "      border-bottom-color: var(--fill-color);\n",
       "      border-left-color: var(--fill-color);\n",
       "    }\n",
       "    20% {\n",
       "      border-color: transparent;\n",
       "      border-left-color: var(--fill-color);\n",
       "      border-top-color: var(--fill-color);\n",
       "    }\n",
       "    30% {\n",
       "      border-color: transparent;\n",
       "      border-left-color: var(--fill-color);\n",
       "      border-top-color: var(--fill-color);\n",
       "      border-right-color: var(--fill-color);\n",
       "    }\n",
       "    40% {\n",
       "      border-color: transparent;\n",
       "      border-right-color: var(--fill-color);\n",
       "      border-top-color: var(--fill-color);\n",
       "    }\n",
       "    60% {\n",
       "      border-color: transparent;\n",
       "      border-right-color: var(--fill-color);\n",
       "    }\n",
       "    80% {\n",
       "      border-color: transparent;\n",
       "      border-right-color: var(--fill-color);\n",
       "      border-bottom-color: var(--fill-color);\n",
       "    }\n",
       "    90% {\n",
       "      border-color: transparent;\n",
       "      border-bottom-color: var(--fill-color);\n",
       "    }\n",
       "  }\n",
       "</style>\n",
       "\n",
       "  <script>\n",
       "    async function quickchart(key) {\n",
       "      const quickchartButtonEl =\n",
       "        document.querySelector('#' + key + ' button');\n",
       "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
       "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
       "      try {\n",
       "        const charts = await google.colab.kernel.invokeFunction(\n",
       "            'suggestCharts', [key], {});\n",
       "      } catch (error) {\n",
       "        console.error('Error during call to suggestCharts:', error);\n",
       "      }\n",
       "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
       "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
       "    }\n",
       "    (() => {\n",
       "      let quickchartButtonEl =\n",
       "        document.querySelector('#df-09927302-4e73-48c9-9274-79b94bd85bf0 button');\n",
       "      quickchartButtonEl.style.display =\n",
       "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
       "    })();\n",
       "  </script>\n",
       "</div>\n",
       "\n",
       "    </div>\n",
       "  </div>\n"
      ],
      "text/plain": [
       "                                         lay_summary  \\\n",
       "0  Yersinia pestis , the bacterial agent of plagu...   \n",
       "1  The genome of all vertebrates is heavily colon...   \n",
       "2  The molecular mechanisms underlying directed c...   \n",
       "3  Contrary to the long-standing belief that no n...   \n",
       "4  Embryonic stem cells have two remarkable prope...   \n",
       "\n",
       "                                             article  \\\n",
       "0  Fleas can transmit Yersinia pestis by two mech...   \n",
       "1  Endogenous retroviruses ( ERVs ) are remnants ...   \n",
       "2  The Drosophila embryonic gonad is assembled fr...   \n",
       "3  Recently , we presented a study of adult neuro...   \n",
       "4  Understanding the transcriptional regulation o...   \n",
       "\n",
       "                                            headings  \\\n",
       "0  [Abstract, Introduction, Results, Discussion, ...   \n",
       "1  [Abstract, Introduction, Results, Discussion, ...   \n",
       "2  [Abstract, Introduction, Results, Discussion, ...   \n",
       "3  [Abstract, Introduction, Model, Results, Discu...   \n",
       "4  [Abstract, Introduction, Results, Discussion, ...   \n",
       "\n",
       "                                            keywords                    id  \\\n",
       "0  [united states, invertebrates, medicine and he...  journal.ppat.1006859   \n",
       "1                         [viruses, sheep, virology]  journal.ppat.0030170   \n",
       "2                                                 []  journal.pgen.1003720   \n",
       "3  [computational biology/computational neuroscie...  journal.pcbi.1001063   \n",
       "4  [developmental biology, cell biology, mammals,...  journal.pgen.0030145   \n",
       "\n",
       "                                  biomistral_summary  \n",
       "0   The study by Hinnebusch et al. provides valua...  \n",
       "1   The authors present a comprehensive study of ...  \n",
       "2   The authors present a study on the role of Hm...  \n",
       "3   The hippocampus is a brain structure that is ...  \n",
       "4   The authors present a novel approach for iden...  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#use first 20000 characters as input\n",
    "mini_plos_df[\"biomistral_summary\"] = mini_plos_df[\"article\"].apply(lambda text: lay_summarize(text))\n",
    "mini_plos_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/",
     "height": 756
    },
    "id": "MwYDg4fsoTl0",
    "outputId": "a5d947a6-1d92-4eda-bd12-ed23fa5da4f8"
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.93s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.83s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.93s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.97s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.68s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.26s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:13<00:00, 13.34s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.44s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.23s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.24s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.11s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.72s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.99s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.98s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.15s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.47s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.54s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.23s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.90s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.25s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.56s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.19s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.44s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.67s/it]\n"
     ]
    },
    {
     "data": {
      "application/vnd.google.colaboratory.intrinsic+json": {
       "summary": "{\n  \"name\": \"mini_elife_df\",\n  \"rows\": 24,\n  \"fields\": [\n    {\n      \"column\": \"lay_summary\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 24,\n        \"samples\": [\n          \"Human reproductive cells\\u2014eggs and sperm\\u2014are produced through a process called meiosis . This means a \\u2018parent\\u2019 cell in the ovaries or testes undergoes two stages of cell division: it first divides into two cells , which then divide again to produce four \\u2018daughter\\u2019 cells . A crucial part of meiosis is ensuring that each daughter cell has half the number of chromosomes that the parent cell did . Before the first round of meiosis , the chromosomes in the parent cell are copied to produce enough chromosomes for the four daughter cells . The distribution of these chromosomes between the daughter cells is determined by the opposing forces acting on them . The pairs of identical chromosomes produced during the copying process are held together by a proteinaceous glue , while microtubules attached to a structure called the spindle\\u2014which has poles at opposite ends of the cell\\u2014try to pull these pairs of chromosomes apart . Rattani et al . now show that a protein called shugoshin-like 2 ( Sgol2 ) , which is involved in holding pairs of identical chromosomes together after the first round of division , has a bigger role than was previously realised . Sgol2 performs three other functions: it helps to align the chromosomes prior to division by , it is thought , facilitating the formation of the K-fibers that attach the microtubules to the chromosomes; it turns off the checkpoint that monitors the alignment of the chromosomes and the attachment of the microtubules; and it regulates a number of the enzymes involved in the process . The specific interactions that allow Sgol2 to perform these diverse functions in meiosis were also identified . Thus , Rattani et al . show that in linking so many essential processes throughout meiosis , Sgol2 appears to have a key , if not unique , role in determining the fate of chromosomes as cells divide .\",\n          \"Forgetting is a common experience in our everyday life . Yet much remains unknown about how we remember , and about why our memories sometimes fail us . The brain contains 80 to 100 billion nerve cells or neurons , which communicate with one another at junctions called synapses . At a synapse , one neuron releases a chemical message , which must diffuse across a small gap , and then activate proteins called receptors on another neuron . If the first neuron activates the second repeatedly , the second cell responds by inserting additional receptors into its membrane at the synapse . This strengthens the connection between the two neurons . Strengthening of synapses is thought to be one of the key mechanisms underlying learning . To confirm this , it would be helpful to be able to monitor the movement and position of individual receptors at synapses . However , the space between the two nerve cells at at synapse , called the synaptic cleft , is no more than 40 nanometers wide . This is about 25 times thinner than a human hair , and too small to be seen with light microscopy . Electron microscopy can visualize synapses , but does not work in living tissue . The only other option is to attach a fluorescent label \\u2013 either a dye or a man-made crystal called a quantum dot \\u2013 to a protein found in synapses and monitor the resulting fluorescence . Though the probe must be small enough to pass through the synaptic cleft to do this . Using fluorescence microscopy , researchers have examined the distribution in synapses of proteins called AMPA receptors , which have a key role in memory . Multiple studies have shown groups of AMPA receptors gathered outside synapses . This has led to the suggestion that during learning , AMPA receptors wait outside the synapse until a space becomes available within the synapse\\u2019s membrane . However , this has yet to be confirmed directly , in part because conventional fluorescent dyes and quantum dots are too bulky to enter synaptic clefts when bound to a receptor . Lee et al . have now developed a quantum dot that is only 10 nanometers wide and therefore small enough to enter the synaptic cleft with an AMPA receptor attached . These small quantum dots were then used to label AMPA receptors in neurons collected from rats and then grown in a petri dish , which provided a completely new view of synapses . The images show that the majority of AMPA receptors in neurons circulate within confined domains \\u2013 a little like holding pens \\u2013 inside the synapse , rather than waiting outside as previously assumed . Labeling the receptors with smaller 4-nanometer-wide fluorescent tags produces a similar picture . Further work is still need to determine how AMPA receptors get into the synapse and contribute to new memories .\",\n          \"It can take several months , or even years , for the brain of a young animal to develop and refine the complex neural networks which underpin cognitive abilities such as memory , planning , and decision making . While the properties that support these functions have been well-documented , less is known about how they emerge during development . Dom\\u00ednguez , Ma , Yu et al . therefore set out to determine when exactly these properties began to take shape in mice , using lightweight nets of electrodes to record brain activity in sleeping newborn pups . The nets were designed to avoid disturbing the animals or damaging their fragile brains . The recordings showed that patterns of brain activity similar to those seen in adults emerged during the first couple of weeks after birth . Just before , however , the brains of the pups went through a brief period of reduced activity: this lull seemed to mark a transition from an immature to a more mature mode of operation . After this pause , neurons in the mouse brains showed coordinated patterns of firing reminiscent of those seen in adults . By monitoring the brains of human babies using scalp sensors , Dom\\u00ednguez , Ma , Yu et al . showed that a similar transition also occurs in infants during their first few months of life , suggesting that brains may mature via a process retained across species . Overall , the relative lull in activity before transition may mark when neural networks gain mature properties; in the future , it could therefore potentially be used to diagnose and monitor individuals with delayed cognitive development .\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"article\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 24,\n        \"samples\": [\n          \"Accurate chromosome segregation depends on coordination between cohesion resolution and kinetochore-microtubule interactions ( K-fibers ) , a process regulated by the spindle assembly checkpoint ( SAC ) . How these diverse processes are coordinated remains unclear . We show that in mammalian oocytes Shugoshin-like protein 2 ( Sgol2 ) in addition to protecting cohesin , plays an important role in turning off the SAC , in promoting the congression and bi-orientation of bivalents on meiosis I spindles , in facilitating formation of K-fibers and in limiting bivalent stretching . Sgol2\\u2019s ability to protect cohesin depends on its interaction with PP2A , as is its ability to silence the SAC , with the latter being mediated by direct binding to Mad2 . In contrast , its effect on bivalent stretching and K-fiber formation is independent of PP2A and mediated by recruitment of MCAK and inhibition of Aurora C kinase activity respectively . By virtue of its multiple interactions , Sgol2 links many of the processes essential for faithful chromosome segregation . \\n The production of haploid gametes from diploid germ cells depends on two rounds of chromosome segregation ( meiosis I and II ) without an intervening round of DNA replication . Defects during the first or second meiotic division in oocytes lead to formation of aneuploid eggs , which in humans occurs with a frequency between 10 and 30% and is a major cause of fetal miscarriages ( Hassold and Hunt , 2001 ) . Understanding the causes of meiotic chromosome missegregation will require clarifying not only the forces and regulatory mechanisms governing meiotic chromosome segregation but also how these are coordinated . At the heart of this process are two opposing forces . The pulling forces produced by kinetochore-microtubules attachments ( K-fibers ) and resisting forces generated by sister chromatid cohesion , which counteracts K-fiber forces if and when kinetochores attach to microtubules with different polarities . During meiosis I , cohesion along chromosome arms holds bivalent chromosomes together following the creation of chiasmata produced by reciprocal recombination between homologous non-sister chromatids . This cohesion must persist during the attachment of maternal and paternal kinetochores to microtubules from opposite poles ( bi-orientation ) and resist the resulting spindle forces . The degree of traction exerted by meiotic spindles must create sufficient tension to facilitate bi-orientation ( Tachibana-Konwalski et al . , 2010 ) . During the bi-orientation process , the initial attachment of kinetochores to the surface of the microtubule lattice ( lateral attachment ) is converted to attachments to their plus ends ( end on attachment ) , creating K-fibers . Because this process is intrinsically error prone , inappropriate attachments , for example those that connect maternal and paternal kinetochores to the same pole , must be disrupted through phosphorylation of kinetochore proteins by Aurora B/C kinases ( Kitajima et al . , 2011 ) . However , because these kinases disrupt K-fibers , they must subsequently be down regulated once bivalents bi-orient correctly . The first meiotic division is eventually triggered by activation of a gigantic ubiquitin protein ligase called the anaphase-promoting complex or cyclosome ( APC/C ) whose destruction of securin and cyclin B activates a thiol protease called separase that cleaves the kleisin subunit of the cohesin complex holding sister chromatids together ( McGuinness et al . , 2005; Kudo et al . , 2006 ) . This process converts chromosomes from bivalents to dyads . It is delayed until all bivalents have bi-oriented by the production , at kinetochores that have not yet come under tension , of a potent inhibitor of the APC/C called the mitotic checkpoint complex ( MCC ) whose Mad2 subunit binds tightly to the APC/C\\u2019s Cdc20 co-activator protein . This regulatory mechanism , called the spindle assembly checkpoint ( SAC ) , must be turned off before APC/CCdc20 can direct destruction of securin and cyclin B and thereby activate separase . Another pre-condition for cleavage , at least in yeast , is phosphorylation of cohesin\\u2019s kleisin subunit by a pair of protein kinases , namely CK1\\u03b4/\\u03b5 and DDK ( Ishiguro et al . , 2010; Katis et al . , 2010 ) . During the first meiotic division , cohesin is phosphorylated along chromosome arms but not at centromeres , which ensures that only cohesion along arms is destroyed by separase at the onset of anaphase I . The consequent persistence of cohesion at centromeres promotes bi-orientation of dyads during meiosis II . Centromeric cohesin avoids phosphorylation and therefore cleavage during the first meiotic division because separase activation is preceded by the recruitment to centromeres of orthologues of the Drosophila MEI-S332 protein , called shugoshins . Members of this family contain a conserved C-terminal basic region and an N-terminal homodimeric parallel coiled coil , which provides a docking site for protein phosphatase 2A\\u2019s ( PP2A ) C and B\\u2019 subunits ( Xu et al . , 2009 ) . In budding yeast , mutant proteins defective specifically in PP2A binding fail to confer protection of centromeric cohesion during meiosis I ( Xu et al . , 2009 ) . Mammals have two members of the shugoshin family: Shugoshin-like protein 1 ( Sgol1 ) , and Shugoshin-like protein 2 ( Sgol2 ) . The former prevents centromeric cohesin from a process called the \\u2018prophase pathway\\u2019 that removes cohesin from chromosomes by a non-proteolytic mechanism soon after cells enter mitosis ( McGuinness et al . , 2005; Liu et al . , 2013 ) . Sgol2 , on the other hand , protects centromeric cohesin from separase at the first meiotic division ( Lee et al . , 2008; Llano et al . , 2008 ) . Sgol2\\u2019s coiled coil domain binds PP2A in vitro ( Xu et al . , 2009 ) but whether this is vital for protecting centromeric cohesion is not known . Sgol2 also interacts with MCAK ( Huang et al . , 2007 ) , a microtubule depolymerizing kinesin , implicated in correcting inappropriate kinetochore-microtubule interactions ( error correction ) , and with Mad2 an essential component of the MCC ( Orth et al . , 2011 ) . Shugoshins , though not explicitly Sgol2 , have also been implicated in recruiting to kinetochores the Aurora B kinase , necessary both for the SAC and for error correction ( Tsukahara et al . , 2010; Yamagishi et al . , 2010 ) . Based on its interactions , Sgol2 has been linked to cohesion protection , the spindle assembly checkpoint , and error correction pathways . However , the physiological significance of these multiple interactions remain unclear . We show here that Sgol2 defective in PP2A binding fails to protect centromeric cohesin , as found for Sgo1 in yeast ( Xu et al . , 2009 ) . However , if this were the sole function of Sgol2 , then chromosome behavior during meiosis I should be unaffected . To our surprise , we found that Sgol2 deficiency caused striking changes in chromosome and microtubule dynamics . It delayed bi-orientation of bivalents on meiosis I spindles , caused increased bivalent stretching , and greatly increased Aurora B/C kinase activity at kinetochores , which was accompanied by an increase in lateral and a decrease in end on kinetochore-microtubules attachments . Lastly , Sgol2 deficiency delayed considerably APC/CCdc20 activation , suggesting that it was required to shut off the SAC . Sgol2 helps to turn off the SAC by binding directly both PP2A and the MCC protein Mad2 , it moderates chromosome stretching by recruiting to kinetochores the kinesin MCAK , and likely promotes formation of K-fibers by down-regulating the activity of Aurora B/C kinase specifically at kinetochores . Meiotic chromosome segregation is a highly complex process dependent on an array of different biochemical processes . Our findings imply that through its multi-domain structure , Sgol2 has an important if not unique role in coordinating many of the key processes within this array . \\n An essential feature of shugoshins is a conserved N-terminal homodimeric coiled coil that binds PP2A ( Figure 1A ) . Unlike Sgol1 , which only interacts stably with the PP2A holoenzyme ( a complex of A , B\\u2019 and C subunits ) , Sgol2 also associates with a core complex containing only A and C subunits ( Xu et al . , 2009 ) . To create versions of Sgol2 defective specifically in binding PP2A , conserved residues ( namely L51 , N54 , R56 , and E82 ) predicted to interact directly with PP2A were substituted by alanine or in the case of E82 also by lysine ( Figure 1B ) . Likewise , the highly conserved asparagine ( N55 on Sgol2 ) , corresponding to N61 in Sgol1 , which forms a hydrogen bond within the coiled coil enabling it to adopt a conformation compatible with PP2A binding , was substituted by isoleucine ( Figure 1B ) . Importantly , binding between purified Flag- and His-tagged Sgol2 fragments spanning the coiled coil ( cc ) region ( 31\\u2013117 ) was unaffected by any of these mutations , implying that all permit coiled coil formation . A triple mutation L51A/N54A/R56A ( 3A ) reduced dimerization modestly ( Figure 1D ) but this effect was not seen when the two proteins were co-expressed in bacteria ( Figure 1E ) . To assess the effects on PP2A binding , GST-tagged PP2A A subunit was incubated with B\\u2019 and C subunits in the presence of MBP-Sgol2cc-Flag . Pull down experiments revealed that N55I and the 3A triple mutation abolished co-purification of Sgol2 with the GST-tagged A subunit while N54A and R56A reduced it . In contrast , L51A and E82A had little or no effect while E82K caused a modest reduction ( Figure 1F ) . 10 . 7554/eLife . 01133 . 003Figure 1 . Sgol2 binds to PP2A and Mad2 through coiled coil domain and Mad2 Interaction Motif ( MIM ) , respectively . ( A ) Schematic diagram indicating relative positions of the coiled coil domain and the MIM sequence in M . musculus Sgol2 ( not drawn to scale ) . ( B ) Sequence alignment of the coiled coil domains of Sgol1 and Sgol2 from different vertebrate species . Arrowheads below the alignment indicate conserved residues ( L51 , N54 , R56 , and E82 ) in the coiled coil domain of Sgol2 mapping onto binding sites between PP2A and Sgol1 . The conserved asparagine residue ( N55 in mSgol2 ) , important for maintaining the interaction surface of the coiled coil domain , is marked by an arrowhead above the alignment . ( C ) Sequence alignment of conserved MIM ( Mad2 Interaction Motif ) in Sgol2 , Mad1 , and Cdc20 from Xenopus , human and mouse origin . ( D ) Dimerization assay of coiled coil domains from wild type and mutant Sgol2 . The Flag-tagged MBP-Sgol2cc ( 31\\u2013117 ) was pulled down by the His-tagged MBP-Sgol2cc ( 31\\u2013117 ) with the same substitution . Homodimerisation was assessed by the relative abundance of each form in the IP by western blots . ( E ) Co-immunoprecipitation assay of MBP-Sgol2cc from wild type and 3A mutant are shown to validate the homodimerisation of coiled coil domains . MBP-Sgol2cc-Flag and MBP-Sgol2cc-His were expressed either independently , or co-expressed . After His-tag pull-down , each fraction was separated on a 12% SDS-PAGE and analyzed by immunoblotting with anti-Flag and anti-His antibodies . Input was resolved on a 12% SDS-PAGE . ( F ) The in vitro binding assay of PP2A A ( GST-tagged ) , B\\u2019 , and C subunits with wild type MBP-mSgol2cc-Flag and variants thereof . After GST pull down , eluates were resolved by SDS-PAGE and Coomassie stained . ( G ) Sgol2 and Mad2 binding assay . Sgol2 fragment spanning amino-acid ( aa ) residues 120\\u2013180 with wild type and mutant MIM sequences were assessed for Mad2 interaction by far-western blot . Fragments from Xenopus Sgol1 , with and without MIM domain , were used as positive and negative controls . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 003 To investigate whether association of PP2A with Sgol2\\u2019s coiled coil is necessary for protecting centromeric cohesion , we compared the effect of the mutations on PP2A\\u2019s co-localization with Sgol2 at kinetochores with their effect on the maintenance of cohesion after oocytes have undergone the first meiotic division . Sgol2 and PP2A co-localize at inner kinetochores of bivalent chromosomes at metaphase I in wild type oocytes ( Figure 2A ) but neither protein was detected at this location in sgol2\\u0394/\\u0394 oocytes ( n = 26 ) . Importantly , microinjection of wild type Sgol2 encoding mRNA into sgol2\\u0394/\\u0394 oocytes restored accumulation at kinetochores of both Sgol2 and PP2A at metaphase I . In contrast , microinjection of mRNAs carrying N54A , N55I , or 3A mutations restored Sgol2 but not PP2A ( Figure 2A ) , implying that binding of PP2A to Sgol2\\u2019s coiled coil is necessary for recruiting the phosphatase to kinetochores in vivo . Chromosome spreads revealed that the failure of sgol2\\u0394/\\u0394 oocytes to maintain cohesion between sister centromeres after meiosis I was reversed by microinjection of wild type but not N54A , N55I , or 3A mRNAs ( Figure 2B ) . L51A and E82A , which had little or no effect on PP2A\\u2019s binding to Sgol2\\u2019s coiled coil in vitro , also fully restored cohesion protection while R56A and E82K , which caused a modest reduction in the Sgol2cc-PP2A interaction vitro , only partially restored it , producing metaphase II oocytes containing individual chromatids as well as dyads ( Figure 2B ) . Lastly , we noticed that injection of 3A Sgol2 mRNAs also produced centromeric cohesion defects in wild type oocytes ( Figure 3A , B ) . The correlation between the effects of mutations on binding PP2A in vitro and on protecting cohesion in vivo implies that the interaction of PP2A with Sgol2\\u2019s coiled coil is essential for protecting cohesin from separase at the first meiotic division . 10 . 7554/eLife . 01133 . 004Figure 2 . Sgol2-PP2A interaction is required to recruit PP2A to kinetochores and protect centromeric cohesion at the first meiotic division . ( A ) GV oocytes were harvested in the presence of IBMX . Microinjections were performed at GV stage in M2 medium supplemented with IBMX . Oocytes were then cultured in IBMX-free medium for about 6 hr , corresponding to metaphase I stage . Chromosome spreads were prepared and stained with DAPI ( blue ) , Sgol2 ( red ) , and PP2A-C ( green ) . Box and whisker plot shows fluorescence intensity ratios of PP2A and Sgol2 at kinetochores . Upper and lower bars indicate 95th and 5th percentiles , respectively . Numbers of oocytes examined are indicated ( n ) . ( B ) Oocytes harvested at GV stage were matured in M16 medium for up to 24 hr . Chromosome spreads prepared from oocytes that have extruded the first polar body were stained with DAPI ( blue ) to visualize DNA and CREST ( green ) to mark centromeres . Frequencies of dyads and single chromatids were quantified . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 00410 . 7554/eLife . 01133 . 005Figure 3 . Sgol2-PP2A , but not Sgol2-Mad2 , interaction is required to protect centromeric cohesion at the first meiotic division . ( A ) Oocytes harvested at GV stage from wild type ( control ) mice were microinjected with wild type or 3A mutant Sgol2 mRNA in M2 medium supplemented with IBMX . After releasing the oocytes in M16 medium without IBMX , chromosome spreads were prepared from metaphase II-arrested oocytes and stained with DAPI ( blue ) and CREST ( green ) . ( B ) Frequencies of dyads and single chromatids observed on metaphase II-stage chromosome spreads . Numbers of oocytes examined are indicated ( n ) . ( C ) Metaphase I localization of Sgol2 ( red ) and PP2A-C ( green ) on chromosomes from sgol2\\u0394/\\u0394 oocytes microinjected with H150E Sgol2 mRNA . ( D ) DAPI ( blue ) and CREST ( green ) stained chromosome spreads prepared from metaphase II-stage oocytes . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 005 DIC video microscopy revealed that sgol2\\u0394/\\u0394 oocytes extruded polar bodies on average 2 hr later than oocytes from wild type littermates . Moreover , only 61% of sgol2\\u0394/\\u0394 oocytes extruded the first polar body compared with 88% in wild type controls ( Figure 4A ) . To investigate this , wild type and sgol2\\u0394/\\u0394 oocytes were microinjected with mRNA encoding histone H2B-mCherry to mark chromosomes and Securin-eGFP to measure APC/CCdc20 activity ( Figure 4B ) . 40% of mutant oocytes failed to activate APC/CCdc20 compared to 12% in wild type controls ( Figure 4C ) , and those that did so were delayed by about 2 hr ( Figure 4D , Figure 4\\u2014figure supplement 1 ) . Surprisingly , microinjection of Sgol2 mRNA not only increased the fraction of oocytes that activated APC/CCdc20 to wild type levels ( 89% ) , but also advanced activation so that it took place about 2 hr earlier than in wild type ( Figure 4E\\u2013G , Figure 4\\u2014figure supplements 2 and 3 ) . Because microinjection of mRNA encoding a version of Cdc20 ( R132A ) that cannot bind Mad2 , but not wild type Cdc20 , induced securin destruction in all sgol2\\u0394/\\u0394 oocytes ( Figure 4C ) , we conclude that Sgol2 facilitates APC/CCdc20 activation by silencing the SAC and that this process is dose dependent . 10 . 7554/eLife . 01133 . 006Figure 4 . Sgol2 silences spindle assembly checkpoint through its interactions with PP2A and Mad2 . ( A ) Kinetics of the first polar body extrusion ( PBE ) of oocytes cultured in vitro . Numbers of oocytes examined are indicated ( n ) . ( B ) Oocytes were microinjected with indicated mRNAs at the GV stage . Chromosome movements and kinetics of Securin-eGFP were visualized by time-lapse confocal microscopy . ( C ) Frequencies of oocytes that destroyed Securin-eGFP , hence activated APC/CCdc20 , at the metaphase-anaphase transition are shown . SAC was inactivated by expressing a dominant-negative version of the APC/C activator Cdc20 ( Cdc20\\u2013R132A ) that cannot be bound by Mad2 . ( D ) Data represent the mean and standard deviations of Securin-eGFP mean fluorescence intensity levels in sgol2\\u0394/\\u0394 ( red ) and litter mate control ( blue ) oocytes at each time point . Values from individual oocytes were normalized relative to that at GVBD ( 0 hr ) , and mean and standard deviations of the population are plotted in arbitrary units , ( AU ) , against time . Securin-eGFP curves from individual oocytes are shown in Figure 4\\u2014figure supplement 1 . ( E ) Securin-eGFP destruction was used as a marker for APC/C activation . Control and Sgol2 knockout are compared to sgol2\\u0394/\\u0394 oocytes microinjected with indicated Sgol2 mRNA . ( F ) GV-normalized mean and standard deviations of Securin-eGFP levels are plotted against time for the indicated groups of oocytes . Securin-eGFP curves from individual oocytes are displayed in Figure 4\\u2014figure supplement 2 . ( G ) Microinjection of Sgol2 mRNA accelerates APC/CCdc20 activation in wild type oocytes . Mean and standard deviations of the time course measurements of Securin-eGFP mean fluorescence intensity are displayed . Securin-eGFP curves from individual oocytes are shown in Figure 4\\u2014figure supplement 3 . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 00610 . 7554/eLife . 01133 . 007Figure 4\\u2014figure supplement 1 . Time course measurements of Securin-eGFP mean fluorescence intensity in control and Sgol2 knockout oocytes . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 00710 . 7554/eLife . 01133 . 008Figure 4\\u2014figure supplement 2 . Comparison of Securin-eGFP mean fluorescence intensity curves in Sgol2 knockout oocytes and sgol2\\u0394/\\u0394 oocytes expressing the indicated form of Sgol2\\u0394/\\u0394 mRNA . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 00810 . 7554/eLife . 01133 . 009Figure 4\\u2014figure supplement 3 . Microinjection of Sgol2 mRNA accelerates APC/CCdc20 activation in wild type oocytes . Time course measurements of Securin-eGFP mean fluorescence intensity from individual oocytes . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 009 The recent observation that shugoshins , like Sgol2 from humans and Sgo1 from Xenopus , bind Mad2 in a manner similar to Mad1 and Cdc20 through a conserved Mad2 Interaction Motif ( MIM ) ( Orth et al . , 2011 ) provided a clue as to how Sgol2 might regulates the SAC . This motif is conserved in mouse Sgol2 ( Figure 1C ) and far-western blots confirmed that interaction between mSgol2 and Mad2 is weakened by substituting the basic amino acid at its center with alanine ( H150A ) or glutamic acid ( H150E ) ( Figure 1G ) . Strikingly , H150E and H150A Sgol2 mRNAs were less effective than wild type in increasing the fraction of sgol2\\u0394/\\u0394 oocytes that activated APC/CCdc20 ( Figure 4E ) . They also failed to advance the timing of activation ( Figure 4F , Figure 4\\u2014figure supplement 2B , C ) . Importantly , neither mutation affected Sgol2\\u2019s association with kinetochores , its ability to recruit PP2A ( Figure 3C ) or MCAK ( Figure 5A ) to this location , or protection of centromeric cohesion ( Figure 3D ) . Interestingly , N54A , which reduces PP2A binding , also abrogated Sgol2\\u2019s ability to promote APC/CCdc20 activation ( Figure 4E , F , Figure 4\\u2014figure supplement 2A , D ) without having any effect on MCAK recruitment ( Figure 5A ) . We conclude that Sgol2 has an important role in silencing the SAC during meiosis I by binding directly both to Mad2 and PP2A . 10 . 7554/eLife . 01133 . 010Figure 5 . Sgol2 , by recruiting MCAK to kinetochores , maintains inter-kinetochore distances between homologous chromosomes . ( A ) GV stage oocytes were harvested and microinjected in M2 medium supplemented with IBMX . Release from IBMX resulted in GVBD and at 6\\u20137 hr post-GVBD chromosome spreads were prepared . Slides were stained with DAPI ( blue ) , CREST ( Red ) , and MCAK ( green ) . Arrowheads indicate distally attached bivalent chromosomes . ( B ) Sgol2 targeted mice were crossed to ( Tg ) Rec8-Myc females to generate control and sgol2\\u0394/\\u0394 expressing the Rec8-myc BAC transgene . Chromosome spreads were prepared at 6\\u20137 hr post-GVBD and were stained with DAPI ( blue ) , CREST ( green ) , and c-Myc ( Red ) . Intensity ratios of Rec8-Myc ( on inter-kinetochore axes ) and CREST ( at centromeres ) were calculated . ( C ) Representative Z-projected confocal microscopy images and average ( \\u00b1SDs ) inter-kinetochore distance measurements from metaphase I stage oocytes expressing H2B-mCherry ( chromosomes , red ) and GFP-MCAK ( green ) . Scale bars represent 5 \\u03bcm . ( D ) Z-projected ( 12 slices 1 . 5 \\u03bcm apart ) live-cell confocal images and spindle length measurements ( \\u00b1SDs ) from metaphase I stage oocytes expressing EB3-GFP ( microtubule plus ends , green ) and H2B-mCherry ( chromosomes , red ) . Scale bar is 14 \\u03bcm . ( E ) Metaphase I localization of Sgol2 ( red ) and PP2A-C ( green ) on chromosomes from sgol2\\u0394/\\u0394 oocytes microinjected with T521A T600A Sgol2 mRNA . ( F ) DAPI ( blue ) and CREST ( green ) stained chromosome spreads prepared at metaphase II-stage from sgol2\\u0394/\\u0394 oocytes microinjected at GV stage with T521A T600A Sgol2 mRNA . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 01010 . 7554/eLife . 01133 . 011Figure 5\\u2014figure supplement 1 . GV oocytes were harvested in M2 medium supplemented with IBMX . Oocytes were then cultured in IBMX-free M16 medium for about 6\\u20137 hr , corresponding to metaphase I-stage . To inhibit Aurora B/C kinase activity oocytes were cultured in M16 medium supplemented with AZD1152 ( 100 nM ) . Representative chromosomes spreads from indicated groups of oocytes at metaphase I-stage showing MCAK localization are displayed . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 01110 . 7554/eLife . 01133 . 012Figure 5\\u2014figure supplement 2 . Oocytes harvested at GV stage were microinjected with either CENPB-eGFP or CENPB-eGFP fused to MCAK mRNA . Time-lapsed confocal microscopy images from metaphase I-stage are displayed . Scale bars represent 5 \\u03bcm . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 012 We noticed that chiasmata appeared terminalized in a large fraction of bivalent chromosomes from sgol2\\u0394/\\u0394 oocytes ( Figure 5A ) . Because this effect was not accompanied by any reduction of cohesin along inter-chromatid axes ( Figure 5B ) , it is unlikely to be caused by precocious Rec8 cleavage . Live cell imaging of oocytes microinjected at the GV stage with mRNAs encoding H2B-mCherry and GFP-MCAK showed that maternal and paternal kinetochores were pulled further apart in sgol2\\u0394/\\u0394 oocytes at metaphase I ( t test , p<0 . 0001 ) . In other words , bivalents were stretched more and possibly the stretching of distal parts of bivalent chromosomes resulted in increased separation of homologous pairs on chromosome spreads . Importantly , increased kinetochore separation in sgol2\\u0394/\\u0394 oocytes was invariably observed on bivalents that had bi-oriented ( Figures 5C and 6; see also Videos 1 and 2 ) , but was not accompanied by any change in the overall length of metaphase I spindles ( ANOVA , p=0 . 97; Figure 5D ) . Interestingly , microinjection of Sgol2 mRNA not only rescued chiasmata structure ( Figure 5A ) , but also reduced inter-kinetochore distances to below that seen in wild type ( Figures 5C and 6; Video 3 , t test p<0 . 0001 ) , without altering spindle length ( Figure 5D ) . Because a similar reduction in inter-kinetochore distances was observed with N54A or H150E mutations ( Figures 5C and 6; Videos 4 and 5 , t test p<0 . 0001 ) , regulation of bivalent stretching by Sgol2 does not depend on its ability to bind PP2A or Mad2 . Thus , the effect cannot be due to changes either in arm cohesion protection or duration of metaphase I . 10 . 7554/eLife . 01133 . 013Figure 6 . Sgol2 assists in bi-orientation of bivalent chromosomes on the metaphase plate . Time-lapse confocal microscopy images from oocytes expressing H2B-mCherry and GFP-MCAK . Representative images from litter-mate control , sgol2\\u0394/\\u0394 , sgol2\\u0394/\\u0394 microinjected with wild type , N54A , H150E , and T521A T600A Sgol2 mRNA are aligned to show the dynamics of MCAK localization and chromosome movements along the meiotic spindle . GVBD normalized time ( in hr ) is indicated on each frame . Scale bars represent 5 \\u03bcm . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 01310 . 7554/eLife . 01133 . 014Figure 6\\u2014figure supplement 1 . Live cell confocal microscopy images from oocytes expressing H2B-mCherry ( chromosomes , red ) and EB3-GFP ( microtubules , green ) are shown . Representative images from control , sgol2\\u0394/\\u0394 , and sgol2\\u0394/\\u0394 microinjected with wild type and T521A T600A Sgol2 mRNA are displayed . Time ( in hr ) after GVBD is indicated on each frame . Scale bars represent 5 \\u03bcm . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 01410 . 7554/eLife . 01133 . 015Video 1 . Z-projected time-course confocal microscopy video of control oocyte microinjected at GV stage with GFP-MCAK and H2B-mCherry mRNA . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 01510 . 7554/eLife . 01133 . 016Video 2 . Z-projected time-course confocal microscopy video of sgol2\\u0394/\\u0394 oocyte microinjected at GV stage with GFP-MCAK and H2B-mCherry mRNA . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 01610 . 7554/eLife . 01133 . 017Video 3 . Z-projected time-course confocal microscopy video of sgol2\\u0394/\\u0394 oocyte microinjected at GV stage with GFP-MCAK , H2B-mCherry and Sgol2 mRNA . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 01710 . 7554/eLife . 01133 . 018Video 4 . Z-projected time-course confocal microscopy video of sgol2\\u0394/\\u0394 oocyte microinjected at GV stage with GFP-MCAK , H2B-mCherry and N54A Sgol2 mRNA . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 01810 . 7554/eLife . 01133 . 019Video 5 . Z-projected time-course confocal microscopy video of sgol2\\u0394/\\u0394 oocyte microinjected at GV stage with GFP-MCAK , H2B-mCherry and H150E Sgol2 mRNA . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 019 Inter-kinetochore distances correlated inversely with GFP-MCAK signals at kinetochores . Thus , MCAK is missing from kinetochores in sgol2\\u0394/\\u0394 oocytes and more abundant ( even than wild type ) in mutant cells microinjected with Sgol2 mRNA ( Figures 5A , C and 6 ) . Because phosphorylation of human Sgol2 at T537 and T620 ( corresponding to T521 and T600 on M . m Sgol2 ) by Aurora B has been implicated in the association of MCAK with Sgol2 in mitotic cells ( Tanno et al . , 2010 ) , we tested the effect of AZD1152 , an Aurora B ( and C ) kinase inhibitor . This eliminated accumulation at kinetochores of both T521 phosphorylated Sgol2 ( Figure 7B ) and MCAK ( Figure 5A , Figure 5\\u2014figure supplement 1 ) . Crucially , injection of mRNA encoding T521A T600A Sgol2 into sgol2\\u0394/\\u0394 oocytes failed to restore accumulation of MCAK at kinetochores ( Figure 5A , Figure 5\\u2014figure supplement 1 ) and to reduce inter-kinetochore distances ( Figures 5C and 6; Video 6 , t test p=0 . 027 ) . Interestingly , neither the Aurora B/C inhibitor nor T521A T600A prevented MCAK\\u2019s recruitment to chromosome arms ( Figure 5\\u2014figure supplement 1 ) . The effect of T521A T600A on MCAK\\u2019s kinetochore binding was highly specific as the mutant Sgol2 protein still accumulated at kinetochores ( albeit to a lesser extent ) , recruited PP2A to this location , and protected centromeric cohesion from separase ( Figure 5E , F ) . 10 . 7554/eLife . 01133 . 020Figure 7 . Aurora B/C Kinase phosphorylates Sgol2 on T521 . ( A ) Oocytes harvested from wild type control females at GV stage were matured in vitro and chromosome spreads were prepared at 2 and 4 hr post-GVBD . Slides were stained for DNA ( blue ) , Sgol2 ( red ) , and CREST ( green ) . Fluorescence intensity ratios of Sgol2 and CREST at kinetochores were quantified . Numbers of oocytes examined are indicated ( n ) . ( B ) Chromosome spreads were prepared at 2 and 4 hr post-GVBD from wild type control oocytes . Slides were stained for DNA ( blue ) , anti-T521p-Sgol2 ( red ) , and CREST ( green ) . Aurora B/C kinase activity was inhibited by culturing oocytes in M16 medium supplemented with the AZD1152 ( 100 nM ) inhibitor . Fluorescence intensity ratios of T521p\\u2013Sgol2 and CREST at kinetochores were quantified . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 02010 . 7554/eLife . 01133 . 021Video 6 . Z-projected time-course confocal microscopy video of sgol2\\u0394/\\u0394 oocyte microinjected at GV stage with GFP-MCAK , H2B-mCherry and T521A T600A Sgol2 mRNA . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 021 These observations suggest that bivalent stretching might be inhibited by recruitment of MCAK to kinetochores by Sgol2 . If so , increased inter-kinetochore distances in sgol2\\u0394/\\u0394 oocytes should be reduced by targeting MCAK to centromeres by artificial means , namely by expression of an MCAK protein whose N-terminus is fused to the C-terminus of CENPB-eGFP . Remarkably , CENPB-eGFP-MCAK , but not CENPB-eGFP , reduced inter-kinetochore distances ( Figure 5C , Figure 5\\u2014figure supplement 2; Video 7 , t test p<0 . 0001 ) . We conclude that Sgol2 moderates the pulling force exerted on bivalents by recruiting MCAK to kinetochores , a process dependent on phosphorylation of Sgol2 by Aurora B/C kinases . 10 . 7554/eLife . 01133 . 022Video 7 . Upper panel displays Z-projected time-course confocal microscopy video of control and sgol2\\u0394/\\u0394 oocytes microinjected at GV stage with CENPB-eGFP and H2B-mCherry mRNA . Lower panel displays representative confocal video of sgol2\\u0394/\\u0394 oocyte microinjected with CENPB-eGFP-MCAK and H2B-mCherry mRNA . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 022 The period between Germinal Vesicle Breakdown ( GVBD ) and anaphase can be divided according to the state of spindles and chromosomes into four distinct phases ( Schuh and Ellenberg , 2007; Kitajima et al . , 2011 ) . During phase 1 , multiple microtubule-organizing centers ( MTOCs ) form a ball of polymerizing microtubules on whose surface chromosomes are distributed individually . This is followed by a period ( phase 2 ) in which chromosomes slide laterally over the ball shaped spindle apparatus , eventually congressing to form a \\u2018prometaphase\\u2019 belt around it . During phase 3 , chromosomes invade the cluster of microtubules , which elongates along an axis perpendicular to the earlier chromosome belt , forming a barrel-shaped bipolar spindle . This highly dynamic phase culminates when chromosomes bi-orient on a metaphase plate within the bipolar spindle . During the fourth and final stage ( phase 4 ) , bi-oriented chromosomes oscillate around the metaphase plate with low amplitudes ( Phase 4 ) . Live-cell confocal imaging of wild type mouse oocytes microinjected with H2B-mCherry and GFP-MCAK showed that MCAK accumulates at MTOCs and on microtubules during phase 1 ( Figure 6; Video 1 ) but does not appear at kinetochores until chromosomes congress on the surface of the ball at the start of phase 2 . This event is accompanied by an increase in Sgol2\\u2019s abundance at kinetochores and by phosphorylation of T521 ( Figure 7A , B ) . MCAK\\u2019s abundance at kinetochores gradually increases during phase 3 as chromosomes invade the spindle apparatus and peaks as chromosomes bi-orient on the metaphase plate ( Figure 6 , top panel; Video 1 ) . This sequence of events is substantially altered in sgol2\\u0394/\\u0394 oocytes . Though GFP-MCAK co-localizes with MTOCs and microtubules , it never subsequently accumulates at kinetochores ( Figure 6 ) . Chromosomes associate with the surface of the ball , albeit more loosely than in wild type , and undergo visible stretching instead of forming a prometaphase belt ( Figures 6 and 8A , Figure 6\\u2014figure supplement 1; Video 2 ) . This excessive stretching becomes even more pronounced upon formation of bipolar spindles . Thus , the clear demarcation of congression during phase 2 and bi-orientation during phase 3 was largely absent . Microinjection of Sgol2 mRNAs into sgol2\\u0394/\\u0394 oocytes greatly accelerated both invasion of the ball by chromosomes and their bi-orientation , with the latter occurring even earlier than in wild type oocytes and before metamorphosis of the ball into a bipolar spindle ( Figures 6 and 8A , Figure 6\\u2014figure supplement 1; Video 3 ) . Thus , while loss of Sgol2 delays bi-orientation , the modest over-expression caused by mRNA microinjection accelerates this process . 10 . 7554/eLife . 01133 . 023Figure 8 . Sgol2 promotes bi-orientation of homologous chromosomes by increasing kinetochore-microtubule interactions . ( A ) To study the relative distributions of chromosomes on the ball shaped spindle , at 2\\u20133 hr post-GVBD , oocytes were treated with Ca2+ buffer for 90 s and then fixed in 1% PFA . Cells were stained with anti-\\u03b2-tubulin ( microtubules , green ) , CREST ( kinetochores , red ) , and DAPI ( chromosomes , blue ) . Scale bars represent 5 \\u03bcm . ( B ) Oocytes harvested at GV stage from wild type , sgol2\\u0394/\\u0394 and sgol2\\u0394/\\u0394 microinjected with wild type , T521A T600A Sgol2 mRNA and CENPB-eGFP-MCAK mRNA were cultured in M16 medium for 6 hr . At 6 hr post-GVBD , oocytes were fixed in 1% PFA were stained with anti-\\u03b2-tubulin ( microtubules , green ) , CREST ( kinetochores , red ) , and DAPI ( chromosomes , blue ) . For Aurora B/C inhibition , oocytes harvested from Sgol2 knockout females were cultured in M16 medium for 4 hr . At 4 hr post-GVBD , oocytes were transferred to M16 medium supplemented with low concentration ( 20 nM ) of AZD1152 inhibitor . Oocytes were cultured until 6 hr post-GVBD and were then fixed in 1% PFA . Scale bars represent 5 \\u03bcm . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 023 Crucially , the precocious bi-orientation caused by Sgol2 mRNA injection was unaffected by N54A or by H150E mutations but abrogated by T521A T600A ( Figures 6 and 8A , Figure 6\\u2014figure supplement 1; Videos 4 , 5 , 6 ) , which also abolished MCAK\\u2019s accumulation at kinetochores and compromised chromosome alignment ( Figures 5A and 6 , Figure 6\\u2014figure supplement 1 ) , possibly due to increased association of MCAK with the arms of bivalent chromosomes ( Figure 5\\u2014figure supplement 1 ) . Injection of Sgol2 T521A T600A still advanced polar body extrusion ( Figure 6 ) , demonstrating that SAC silencing by Sgol2 does not depend on rapid or possibly even accurate chromosome bi-orientation and suggesting that these activities are mediated by separate domains within Sgol2 . Surprisingly , injection of mRNAs encoding CENPB-eGFP-MCAK instead of Sgol2 did not accelerate chromosome bi-orientation ( data not shown ) , raising the possibility that Sgol2 facilitates bi-orientation by a mechanism either independent of or in addition to MCAK\\u2019s recruitment to kinetochores . Because , bi-orientation of bivalent chromosomes correlates with the appearance of kinetochore-microtubule bundles ( Kitajima et al . , 2011 ) , we visualized these in oocytes fixed in 1% PFA following a brief treatment with a calcium-containing buffer , which destabilizes the more dynamic non-kinetochore-microtubules . This revealed three differences between wild type and sgol2\\u0394/\\u0394 oocytes at 6 hr post-GVBD , when most kinetochores in wild type have relinquished lateral and formed end on attachments and chromosomes have bi-oriented . Kinetochore associated microtubules bundles ( K-fibers ) were thinner and more sparse , lateral attachments were greatly increased in frequency , and kinetochores ( as detected by CREST antibodies ) were frequently highly stretched in mutant oocytes ( Figure 8B ) . Microinjection of Sgol2 mRNA had the opposite effect , eliminating lateral attachments and kinetochore stretching while greatly increasing the abundance of end-on microtubules associated with each kinetochore , an effect abrogated by T521A T600A ( Figure 8B ) . This suggests that the accumulation at kinetochores of Sgol2 phosphorylated by Aurora B/C kinase stimulates the conversion of lateral to end on attachments and that this may be the mechanism by which Sgol2 accelerates chromosome bi-orientation . Because injection of CENPB-eGFP-MCAK mRNAs instead of Sgol2 largely failed to increase end-on attachments at the expense of lateral ones ( Figure 8B ) , we suspect that Sgol2 phosphorylated by Aurora B/C has a function at kinetochores besides MCAK recruitment ( see below ) . Because Aurora B kinase destabilizes kinetochore-microtubule attachments , at least partly by phosphorylating proteins within the KMN network ( Welburn et al . , 2010 ) , we tested whether Sgol2 altered the phosphorylation of Ser24 on Knl1 ( p-Knl1 ) . Remarkably , the ratio of p-Knl1 to CREST signals in sgol2\\u0394/\\u0394 oocytes at 6 hr post-GVBD was more than double that of wild type controls ( Figure 9A ) . Microinjection of Sgol2 mRNA reduced this to below wild type , an effect unaltered by N54A but largely abolished by T521A T600A ( Figure 9A ) . Similar results were obtained using a phospho-specific antibody against Ser100 on Dsn1 ( data not shown ) and one that detects activating auto-phosphorylation of Aurora A/B/C at Thr288 , Thr232 , or Thr198 ( p-Aurora ) , respectively ( Figure 9B ) . Interestingly , these effects were specific to kinetochores as auto-phosphorylation of Aurora B/C on chromosome arms was unaltered in the mutant oocytes ( Figure 9B ) . Importantly , Sgol2 did not alter the overall amount of Aurora C at kinetochores ( Figure 10 ) . These results imply that Sgol2 counters phosphorylation of KMN network proteins by a mechanism not involving its recruitment of PP2A . Strangely , this process seems to depend on Sgol2\\u2019s prior phosphorylation by the very same kinases that it ultimately inhibits . In other words , Aurora B/C creates its own inhibitor at kinetochores , creating a negative feedback loop . Consistent with this notion , the gradual accumulation at kinetochores of Sgol2 phosphorylated at T521 following GVBD ( Figure 7B ) is accompanied by a corresponding reduction in Dsn1 Ser100 phosphorylation ( Figure 9D ) . Critically , Sgol2\\u2019s ability to inhibit Aurora B/C kinase activity at kinetochores does not depend on kinetochore-microtubule interactions , as Sgol2 reduced auto-phosphorylation of Aurora B/C even in the presence of microtubule destabilizing drug ( Figure 9C ) . To address whether hyper-activity of Aurora B/C at kinetochores could be responsible for the paucity of K-fibers in sgol2\\u0394/\\u0394 oocytes , we tested the effect of the Aurora kinase inhibitor AZD1152 . This both increased the density of K-fibers and reduced kinetochore stretching but did not eliminate lateral attachments ( Figure 8B ) , which may require direct involvement of MCAK . We suggest that Sgol2 promotes chromosome bi-orientation by inhibiting Aurora B/C kinase as well as by recruiting MCAK . 10 . 7554/eLife . 01133 . 024Figure 9 . Sgol2 inhibits Aurora B/C Kinase activity at kinetochores . ( A ) Chromosome spreads were prepared from indicated groups at 4 hr post-GVBD . Slides were stained with DAPI ( blue ) , anti-S24p-Knl1 ( red ) and CREST ( green ) . Kinetochore intensities of anti-S24p-Knl1 and CREST signals were quantified and the intensity ratios of anti-S24p-Knl1 and CREST signals are shown . Upper and lower bars indicate 95th and 5th percentiles , respectively . ( B ) Chromosome spreads prepared at 4 hr post-GVBD were stained with DAPI ( blue ) , anti-p-Aurora A/B/C ( red ) and CREST ( green ) . Kinetochore fluorescence intensity ratios of p-Aurora A/B/C and CREST are displayed . ( C ) Sgol2 knockout and Sgol2 knockout oocytes injected with wild type Sgol2 mRNA were cultured for 6 hr from GVBD in M16 medium supplemented with 400 nM Nocodazole . Chromosome spreads were stained for DNA ( DAPI , blue ) , p-Aurora A/B/C ( red ) , and CREST ( green ) . ( D ) anti-S100p-Dsn1 and CREST kinetochore intensity ratios , quantified from chromosome spreads prepared at 3 and 6 hr post-GVBD from wild type control oocytes . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 02410 . 7554/eLife . 01133 . 025Figure 10 . Sgol2 decreases Aurora C activity at the kinetochores without affecting its localization . Chromosome spreads were prepared from the indicated groups at 4 hr post-GVBD . Slides were stained for DNA ( DAPI , blue ) , Aurora C ( red ) , and CREST ( green ) . Fluorescence intensity ratios of Aurora C and CREST at kinetochores are displayed . Upper and lower bars indicate 95th and 5th percentiles , respectively . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 025 \\n Previous work has shown that Sgol2 is essential for protection of centromeric cohesin from separase at the first meiotic division ( Lee et al . , 2008; Llano et al . , 2008 ) . Our creation and analysis of mutations within Sgol2\\u2019s highly conserved coiled coil reveals the likely mechanism , namely recruitment of PP2A to centromeres . PP2A might protect cohesin from separase either directly , by binding ( Holland et al . , 2007 ) and inhibiting separase activity , or indirectly , by reversing phosphorylation of cohesin\\u2019s Rec8 subunit ( Katis et al . , 2010 ) , a process known to be important for its cleavage in yeast . However , if cohesion protection were Sgol2\\u2019s sole function , then its depletion should not compromise chromosome segregation during meiosis I . The precocious loss of cohesion should only affect the second meiotic division . On the contrary , depletion of Sgol2 specifically in oocytes has highly pleiotropic consequences already during meiosis I . These include inefficient SAC silencing , greatly increased bivalent stretching , and major alterations in the dynamics of kinetochore-microtubule attachments ( K-fibers ) . We show that Sgol2 silences the SAC by binding to Mad2 using a similar MIM motif common to Mad2\\u2019s other two partners Cdc20 and Mad1 . This raises the possibility that Sgol2 silences the SAC by competing with Cdc20 for the binding of activated Mad2 , thereby preventing formation of the MCC . However , this cannot be the full story as SAC silencing also depends on Sgol2\\u2019s ability to bind PP2A . Because Sgol2\\u2019s MIM motif is close to the coiled coil by which it binds PP2A , it is conceivable that PP2A directly ( i . e . , structurally ) facilitates the formation or stability of \\u2018closed\\u2019 complexes between Sgol2 and Mad2 . This hypothesis is supported by the fact that only dephosphorylated Mad2 interacts with Cdc20 and Sgol2 , while a phosphomimetic mutant of Mad2 cannot associate with either of these proteins ( personal communication Michael Orth ) . Thus , PP2A could selectively stabilize the Sgol2-Mad2 complex by keeping Mad2 in a de-phosphorylated state . Therefore , it is conceivable that if there were sufficient turnover of Mad2 bound to MCC , then PP2A could promote the Sgol2: Mad2 complex formation and thus alter the activity of the MCC responsible for inhibiting the APC/C . Unlike centromeric protection and SAC silencing , Sgol2\\u2019s regulation of bivalent stretching depends not on its ability to bind PP2A but instead on phosphorylation at T521 and T600 by Aurora B/C kinases . This facilitates MCAK accumulation at kinetochores . Because artificial recruitment of MCAK to kinetochores by fusion to CENP-B substitutes for Sgol2 with regard to bivalent stretching , we suggest that it is the population of MCAK recruited to kinetochores by Sgol2 that moderates the pulling forces exerted on maternal and paternal kinetochores , possibly by reducing the incidence of lateral attachments . Artificial MCAK recruitment did not however restore normal looking end on attachments or greatly facilitate bi-orientation in Sgol2 deficient oocytes , implying that Sgol2 has yet another function at kinetochores besides recruiting PP2A and MCAK . Our observation that auto-phosphorylation of Aurora B/C kinases as well as phosphorylation of KMN proteins are elevated in Sgol2\\u2019s absence but depressed upon microinjection of Sgol2 mRNA suggests that this other function of Sgol2 is to act as a kinetochore-specific inhibitor of Aurora B/C kinases . The notion that Sgol2 moderates Aurora B/C kinase activity at kinetochores was unexpected , as it has been reported that xSgo2 , a newly discovered homolog of shugoshin in Xenopus , actually promotes Aurora Kinase activity , generating a favourable environment for microtubule polymerization ( Rivera et al . , 2012 ) . In this case , the effect may be systemic , that is throughout the cytoplasm , whereas M . m Sgol2 appears to act as an inhibitor only at kinetochores . A remarkable feature of this effect is that Sgol2\\u2019s ability to inhibit Aurora B/C kinases at kinetochores depends on its prior phosphorylation at T521 and T600 by the very same Aurora kinases . In other words , Aurora B/C kinase creates its own inhibitor , generating a negative feedback loop . Whether the phosphorylation of Sgol2 necessary for it to act as an inhibitor of Aurora kinases at kinetochores is also generated at this location or along chromosome arms or elsewhere in the cell is currently unclear . Irrespective of its precise spatial and dynamic properties , this negative feedback loop may be an important feature of meiosis I . Aurora B/C protein kinases have a key role in destabilizing inappropriate kinetochore-microtubule attachments , namely those that do not generate tension ( Lampson and Cheeseman , 2011 ) . They are therefore essential for the efficient bi-orientation of bivalent chromosomes . However , it is important that the de-stabilization process is eventually turned off so that correctly attached bivalents remain stably attached to microtubules . Hitherto , it has been assumed that the mechanism responsible for turning off these kinases merely involved physical forces that pull Aurora substrates away from the enzymes when maternal and paternal kinetochores are pulled in opposite directions ( Liu et al . , 2009 ) . Though our work does not preclude a role for traction in down regulating Aurora B/C\\u2019s access to kinetochore-bound substrates , it raises the possibility that down regulation is also an intrinsic feature of a biochemical system in which Aurora B/C activates its own inhibitor . The regulatory network revealed by our work ensures that Aurora B/C kinases will be active early during the bi-orientation , when incorrect attachments may be more frequent , and less active later , when they have been largely eliminated . It is possible that during the period when Aurora B/C kinases are active , the selective stabilization of correct attachments is mediated by mechanical changes within kinetochores induced by tension , as recently observed in purified yeast kinetochores ( Akiyoshi et al . , 2010 ) , not as originally thought by tension dependent changes in Aurora kinase activity , for which there has never been much direct evidence . We suggest that an important feature of meiosis I in oocytes is an early phase of high Aurora B/C kinase activity at kinetochores followed by a later phase of lowered activity made possible by the Aurora B/C-Sgol2 negative feedback loop . The lowered activity of Aurora B/C might increase the MT-depolymerizing activity of MCAK at kinetochores ( Andrews et al . , 2004 ) . This is supported by the fact that the kinetochore pool of MCAK , recruited by Sgol2 , is indeed not phosphorylated at serine 92 in mouse oocytes ( data not shown ) . If correct , it will be important to understand the dynamic properties of the feedback loop that ensures that Aurora B/C kinases do not turn themselves off precociously . It is currently unclear whether the Aurora B/C-shugoshin feedback loop is a general feature of meiotic cells or whether it is also a feature of mitotic cells . How Sgol2 inhibits Aurora B/C kinase activity at kinetochores is also unclear , in particular whether Sgol2 does this alone or only in conjunction with the recruitment of MCAK . Given the number of fundamental processes regulated by Sgol2 , it is surprising that sgol2\\u0394/\\u0394 oocytes often manage to undergo the first meiotic division without generating massive aneuploidy . Their second meiotic division is of course completely defective due to the total absence of sister chromatid cohesion . We suggest that all of the processes regulated by Sgol2 are fundamentally essential but that they are also regulated by Sgol2-independent pathways . In other words , the processes regulated by Sgol2 may be so fundamental that there are multiple pathways regulating them . It is not inconceivable that Sgol1 , which is also expressed during meiosis , shares some functions with Sgol2 and it will therefore be interesting to analyze the phenotype of double mutant oocytes . What is possibly most remarkable is that a single protein regulates sister chromatid cohesion , chromosome traction , the spindle assembly checkpoint , and lastly Aurora B/C kinases . Crucially , by demonstrating that different point mutations alter the regulation of some but not other processes , we have proven that the pleiotropy observed in sgol2\\u0394/\\u0394 oocytes is not due to knock on effects ( secondary or even tertiary pathology ) of eliminating a single function . Sgol2 possesses several different domains that regulate fundamentally different processes ( Figure 11 ) . Why so much regulation is embedded into a single protein is unclear , but we speculate that this helps the coordination of multiple processes necessary for high fidelity chromosome segregation . 10 . 7554/eLife . 01133 . 026Figure 11 . Graphical summary of sequence features , interactions and key functions of Sgol2 in mouse oocytes . DOI: http://dx . doi . org/10 . 7554/eLife . 01133 . 026 \\n Generation of the Sgol2 knockout mice has been described ( Llano et al . , 2008 ) . To detect Rec8 , females from a transgenic line expressing Rec8 from a bacterial artificial chromosome ( BAC ) , with nine tandem copies of the human c-myc epitope at its C terminus , were crossed to sgol2\\u0394/+ males . Litters with genotypes sgol2\\u0394/+ ( Tg ) Rec8- Myc+ were crossed back to sgol2\\u0394/+ mice to generate sgol2\\u0394/+ ( Tg ) Rec8-Myc+ ( controls ) and sgol2\\u0394/\\u0394 ( Tg ) Rec8-Myc+ female . Prophase-arrested oocytes were harvested from ovaries excised from 8\\u201312 week old female mice . Antral follicles were isolated using sterile insulin needles in M2 medium ( Sigma-Aldrich , UK ) supplemented with 200 \\u00b5M IBMX ( Sigma-Aldrich ) . For intracellular microinjections , oocytes were placed in droplets of M2 medium with IBMX . For time-lapse confocal imaging and in vitro culture prior to preparation of chromosome spreads , oocytes were transferred into IBMX-free M16 medium and cultured at 37\\u00b0C and 5% CO2 . Capped mRNA constructs with a poly-A tail were transcribed using T3 or T7 Ultra mMESSAGE kit ( Ambion , Austin , TX , USA ) from plasmid cDNA encoding H2B-mCherry , wild type Sgol2 ( Origene , Rockville , MD , USA ) , and various mutants of Sgol2 , GFP-Sgol2 , Securin-eGFP , GFP-MCAK , CENPB-eGFP , CENPB-eGFP-MCAK , and GFP-EB3 . Fully-grown oocytes were injected with 5\\u201310 pl mRNA at a final concentration of 0 . 1 mg/ml in RNase-free water ( Ambion ) using a Pneumatic PicoPump ( World Precision Instruments ) . To allow for protein expression , following microinjection of mRNA , GV oocytes were cultured for 12 hr in M16 supplemented with IBMX . Oocytes were then washed in inhibitor-free M16 and thereafter cultured at 37\\u00b0C and 5% CO2 . During time-lapse confocal microscopy experiments , oocytes were cultured in a PeCon environmental microscope incubator at 37\\u00b0C and 5% CO2 . A Zeiss LSM510 META confocal microscope equipped with PC-Apochromat 63x/1 . 2 NA water immersion and PC-Apochromat 20x/0 . 8 NA objective lenses was used for image acquisition . For detection of GFP and mCherry , 488-nm and 561-nm excitation wavelengths and BP 505\\u2013550 and LP 575 filters were used . During live-cell imaging , chromosomes labeled with H2B-mCherry were tracked with an EMBL-developed tracking macro ( Rabut and Ellenberg , 2004 ) adapted to our microscope . We imaged 8 to 16 z-confocal sections ( every 1 . 5 or 2 . 0 \\u00b5m ) at 5\\u201310 min intervals for 12\\u201314 hr . Chromosome spreads from mouse oocytes were prepared as previously described ( Peters et al . , 1997; Hodges and Hunt , 2002 ) . Briefly , zona pellucida was removed by treatment with 10 mg/ml Pronase ( Sigma-Aldrich ) at 37\\u00b0C for 5\\u201310 min . Zona-free oocytes were transferred to hypotonic solution ( 50% FCS in deionized water ) for 10 min at 37\\u00b0C . Oocytes were then fixed in paraformaldehyde solution ( 1% paraformaldehyde , 0 . 15% Triton X-100 , 3 mM dithiothreitol , adjusted to pH 9 . 2 with NaOH ) in 15-well multichamber glass slide ( MP Biomedicals ) overnight at room temperature in a humidified chamber . After drying , slides were washed twice in 0 . 4% Photoflo/H2O ( Kodak ) and PBS for 5 min , at room temperature . Slides were then processed for immunostaining . In this study , we used CREST ( 1:250; Davis Lab , Davis , CA , USA ) , rabbit anti-Sgol2 serum ( 1:50; gift from JL Barbero [Parra et al . , 2009] ) , anti-phospho T521 Sgol2 ( 1:500 , gift from Yoshi Watanabe [Tanno et al . , 2010] ) , anti-MCAK ( 1:500 , gift from Duane A Compton [Mack and Compton , 2001] ) , anti-phospho S24 Knl1 ( 1:1000 , gift from Iain Cheeseman [Welburn et al . , 2010] ) , anti-phospho S100 Dsn1 ( 1:500 , gift from Iain Cheeseman [Welburn et al . , 2010] ) , antibody detecting T288 , T232 or T198 on Aurora A/B/C ( 1:50 , Cell Signaling Technology , Danvers , MA , USA ) , anti-Aurora C ( 1:50 , Bethly Laboratories , Montgomery , TX , USA ) , mouse anti-PP2A-C ( 1:200 , BD Biosciences , Palo Alto , CA , USA ) , and anti-c-Myc ( 1:500 , Millipore , Billerica , MA , USA ) . For analysis of K-fibers , oocytes were processed as previously described ( Kitajima et al . , 2011 ) . Briefly , after either 2 or 6 hr post-GVBD , oocytes were treated for 90 s with Ca2+ buffer ( 100 mM PIPES pH 7:0 , 1 mM MgCl2 , 0 . 1 mM CaCl2 , 0 . 1% Triton X-100 ) at 37\\u00b0C and then fixed in 1% formaldehyde prepared in Ca2+ buffer for 30 min at room temperature . Following fixation , oocytes were washed and extracted overnight in PBT ( PBS supplemented with 0 . 1% Triton X-100 ) at 4\\u00b0C . Cells were then blocked in 3% BSA with PBT ( blocking solution ) for 1 hr at room temperature . For immuno-labeling oocytes were incubated overnight at 4\\u00b0C with \\u03b2-tubulin ( 1:250 , Abcam , Cambridge , UK ) and CREST ( 1:100 , Davis Lab , Davis , CA , USA ) antibodies , prepared in blocking solution . After four washes in PBT , oocytes were incubated for 1 hr at room temperature with Alexa 488 and 568 conjugated secondary antibodies ( 1:500 , Invitrogen , UK ) prepared in blocking solution . Oocytes were briefly stained with Hoechst 33 , 342 ( 20 mg/ml ) before confocal imaging . Recombinant MBP-Sgol2cc-His and MBP-Sgol2cc-Flag were purified from 1l of Escherichia coli BL21 ( DE3 ) RIPL ( Stratagene , La Jolla , CA , USA ) carrying corresponding expression cassettes in pMAL-c2x- ( Flag-tagged subunit ) and pET28b-based backbones ( His-tagged subunit ) , grown at 37\\u00b0C in 2xTY media supplemented with 25 \\u03bcg/ml chloramphenicol and 100 \\u03bcg/ml ampicillin for pMAL-c2x or 50 \\u03bcg/ml kanamycin for pET28b . At OD600 = 0 . 4 , MBP-Sgol2cc-His or -Flag synthesis was induced at 30\\u00b0C by the addition of 0 . 1 mM IPTG . After 4 hr , the cells were harvested by centrifugation , frozen in dry ice , and stored at \\u221280\\u00b0C . The cell paste was re-suspended in 40 ml of lysis buffer ( 50 mM sodium phosphate pH 7 . 0 , 500 mM NaCl , 10% glycerol , 0 . 02% Triton X-100 ) containing Roche complete EDTA free protease inhibitor , 1 mM PMSF and , for the His-tagged subunit 5 mM . Each aliquot was sonicated four times for 30 s on ice . Insoluble material was removed by centrifugation at 25 , 000 rpm for 1 hr at 4\\u00b0C in a Beckman Coulter Avanti-J26XP JLA8 . 1000 rotor . The supernatant was incubated for 2 hr at 4\\u00b0C on a 5 ml Talon column ( BD Biosciences , Palo Alto , CA , USA ) or ANTI-FLAG M2 Affinity Gel ( Sigma A2220 ) . The M2 Flag beads were washed four times with 50 ml of lysis buffer and the Talon beads were washed two times with 50 ml of lysis buffer containing 20 mM imidazole and two times with 50 ml of lysis buffer containing 45 mM imidazole . MBP-Sgol2cc-Flag was eluted with four times 1 ml of lysis buffer containing 3 \\u00d7 FLAG peptide ( 150 ng/\\u03bcl ) . MBP-Sgol2cc-His was eluted with 10 ml of lysis buffer containing 1 M of imidazole . The concentration of MBP-Sgol2cc-His and MBP-Sgol2cc-Flag were determined by nanodrop against the lysis solution . For all binding assays , highly-purified PP2A subunits ( purified as previously described [Cho and Xu , 2007] ) and MBP-Sgol2cc-Flag were used . To test the binding of PP2A to mutant shugoshin , 20 \\u03bcg of GST-tagged PP2A A subunit were incubated with excess amounts of PP2A B56- and C- subunit and MBP-tagged shugoshin for 1 hr on ice . Subsequently , samples were incubated with 40 \\u03bcl of glutathione beads ( GE Healthcare ) for 20 min . Beads were then washed 4 \\u00d7 with 1 ml each of 1 \\u00d7 PBS , 0 . 1% Triton X-100 , 1 mM dithiothreitol , boiled in reducing SDS loading buffer , and subjected to SDS-PAGE analysis followed by Coomassie blue staining . His and Flag-tagged mSgol2cc from wild type and 3A were co-expressed in E . coli BL21 ( DE3 ) RIPL for 4 hr at 30\\u00b0C with 0 . 1 mM of IPTG . Cells were spun at 25 , 000 rpm for 1 hr at 4\\u00b0C in a Beckman Coulter Avanti-J26XP JLA8 . 1000 rotor . The pellet was re-suspended into lysis buffer ( 50 mM sodium phosphate pH = 7; 250 mM NaCl; 10% glycerol , 0 . 02% Triton X-100; protease inhibitor ) . 10 mg of extract were incubated with 100 \\u03bcl of Talon beads for 2 hr at 4\\u00b0C . The beads were wash five times with the lysis buffer containing 23 mM of imidazole . MBP-Sgol2cc-His was eluted with 10 ml of lysis buffer containing 1 M of imidazole . Affinity purified material and input were resolved by SDS-PAGE . The abundance of His and Flag-tagged MBP-Sgol2cc were analyzed by western blot using anti-Flag ( 1:10 , 000 ) and anti-His ( 1:10 , 000 ) antibodies .\",\n          \"Previous studies tracking AMPA receptor ( AMPAR ) diffusion at synapses observed a large mobile extrasynaptic AMPAR pool . Using super-resolution microscopy , we examined how fluorophore size and photostability affected AMPAR trafficking outside of , and within , post-synaptic densities ( PSDs ) from rats . Organic fluorescent dyes ( \\u22484 nm ) , quantum dots , either small ( \\u224810 nm diameter; sQDs ) or big ( >20 nm; bQDs ) , were coupled to AMPARs via different-sized linkers . We find that >90% of AMPARs labeled with fluorescent dyes or sQDs were diffusing in confined nanodomains in PSDs , which were stable for 15 min or longer . Less than 10% of sQD-AMPARs were extrasynaptic and highly mobile . In contrast , 5\\u201310% of bQD-AMPARs were in PSDs and 90\\u201395% were extrasynaptic as previously observed . Contrary to the hypothesis that AMPAR entry is limited by the occupancy of open PSD \\u2018slots\\u2019 , our findings suggest that AMPARs rapidly enter stable \\u2018nanodomains\\u2019 in PSDs with lifetime >15 min , and do not accumulate in extrasynaptic membranes . \\n A major goal of cellular and molecular neuroscience is to characterize how synapses change during synaptic plasticity , the leading cellular correlate of learning and memory ( Volk et al . , 2015; Huganir and Nicoll , 2013; Kneussel and Hausrat , 2016 ) . The position and dynamics of the two major ionotropic glutamate receptors ( iGluRs ) , \\u03b1-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid receptors ( AMPARs ) and N-methyl-D-aspartate receptor ( NMDARs ) , are critical for synaptic strength . AMPARs traffic intracellularly from sites of synthesis and assembly to exocytosis sites near synapses ( Anggono and Huganir , 2012; Henley et al . , 2011; Greger and Esteban , 2007 ) . Despite much elegant work , it is still unclear what mechanisms limit AMPAR entry into and exit out of PSDs ( Malinow and Malenka , 2002 ) ( Park et al . , 2004 ) ( Wang et al . , 2008 ) . The different steps involved in AMPAR trafficking at synapses under constitutive conditions ( no plasticity ) are depicted schematically in Figure 1a . As shown , AMPAR trafficking consists of its exo- and endocytosis at sites on the plasma membrane ( steps 1 and 1\\u2019 , respectively ) , diffusion to and from these sites and PSDs ( steps 2 and 2\\u2019 ) and entry into and exit from PSDs ( steps 3 and 3\\u2019 ) . The positioning of the AMPAR endo- and exocytosis sites with respect to PSDs is uncertain with some evidence for endo- and exocytosis sites within spines relatively close to PSDs ( Kennedy et al . , 2010; Lu et al . , 2007 ) , other evidence that the sites are on dendrite shafts outside of spines ( Yudowski et al . , 2007; Tao-Cheng et al . , 2011 ) , and also that both locations may exist regulated by synaptic plasticity ( Tao-Cheng et al . , 2011 ) . Another complication is that the recycling pathway under constitutive conditions may be different from the pathway with synaptic plasticity ( Zheng et al . , 2015; Henley et al . , 2011 ) . This suggests that different endo- and exocytosis sites may exist under constitutive conditions versus synaptic plasticity . In the last decade , advanced fluorescence microscopic techniques have pioneered work on iGluR dynamics ( Heine et al . , 2008; Groc et al . , 2007; Nair et al . , 2013; MacGillavry et al . , 2013; Kneussel and Hausrat , 2016 ) . Commercial or \\u2018big\\u2019 quantum dots ( bQDs ) with a diameter\\u00a0>20 nm were observed ( at least occasionally ) within synaptic clefts of inhibitory synapses by electron microscopy ( Dahan et al . , 2003 ) . bQDs are extremely bright , photostable and their position can be localized to within\\u00a0~10 nm in the millisecond time-scale ( Pinaud et al . , 2010 ) . Studies of AMPARs tagged with bQDs led to the conclusion that 50\\u201380% of AMPARs are extrasynaptic and highly mobile ( Groc et al . , 2007; Opazo et al . , 2010 ) . This significant highly mobile pool of extrasynaptic AMPARs has been proposed to exist because of the slow rate that extrasynaptic AMPARs enter PSDs . This slow rate of motion and accumulation of extra-synaptic AMPARs is thought to occur because of the limited availability of \\u2018slots\\u2019 in PSDs ( See Figure 1a , route 3 ) ( Shi et al . , 2001; Opazo et al . , 2012 ) . Slots are defined as placeholders for AMPARs in PSDs ( Malinow et al . , 2000 ) . Once in PSD slots , AMPARs are generally assumed to be immobilized , although Nair et al . observed that AMPARs underwent constrained diffusion in what they defined as \\u2018nanodomains\\u2019 , that\\u00a0is sub-synaptic regions where AMPARs are confined to ( Nair et al . , 2013 ) . New slots may become available when AMPARs exit from PSDs ( route 3\\u2019 ) or if more slots are created or removed by different forms of synaptic plasticity ( Malinow et al . , 2000; Shi et al . , 2001; Malinow , 2003; Giannone et al . , 2010 ) ( Groc et al . , 2007; Heine et al . , 2008 ) ( Nair et al . , 2013 ) ( Makino and Malinow , 2009 ) ( Opazo et al . , 2012 ) ( Henley and Wilkinson , 2016 ) . The presence of such a large pool of mobile AMPARs on the extrasynaptic surface of dendritic membranes is thought to be an important contributor to the mechanisms that underlie synaptic plasticity such as LTP and could help synapses recover from pair-pulse depression during periods of fast synaptic transmission ( Heine et al . , 2008; Tao-Cheng et al . , 2011 ) . To avoid the potential problem of using large probes , other workers have used single particle tracking ( spt ) of small ( <5 nm ) fluorescent probes permanently ( generally via a linkage protein ) or temporarily bound to an AMPAR . One such technique is called sptPALM ( spt- photoactivated localization microscopy ) where an iGluR is fused with a photoactivatable fluorescent protein ( pa-FP ) ( Manley et al . , 2008 ) . Another technique is called universal points accumulation for imaging in nanoscale topography ( uPAINT ) which measures an organic fluorescent probe that is temporarily bound to the receptor ( Sharonov and Hochstrasser , 2006; Giannone et al . , 2010 ) . However , there are contradictory results using these methods . Using uPAINT , AMPARs were found to be highly mobile ( Giannone et al . , 2010 ) , or stationary ( Nair et al . , 2013 ) . Using sptPALM and FRAP , another fluorescence technique , workers found that AMPARs were highly mobile . However , fluorescent proteins for sptPALM and FRAP are sensitive to both internal and external AMPAR receptors , which may obfuscate the issue ( Nair et al . , 2013; Heine et al . , 2008 ) ( Makino and Malinow , 2009; Ashby et al . , 2006 ) . Determining whether AMPARs or other neurotransmitter receptors are synaptic ( outside PSDs ) or extrasynaptic ( inside PSDs ) has been difficult because of four factors: ( 1 ) Synapses are nanometer in scale below the diffraction limit of visible microscopy; ( 2 ) Most microscopy has been limited to two-dimensions , where in reality , the dimensions are three-dimensional; ( 3 ) Recombinantly tagged imaging probes cannot readily distinguish intracellular vs . cell-surface receptors; ( 4 ) Probe photostability limits the localization precision and the length of image acquisition , particularly for single-molecule studies . Here we use 3-dimensional super-resolution microscopy on cell-surface AMPARs labeled with different-sized fluorescent probes\\u2014bQDs ( >20 nm ) , small quantum dots ( sQD \\u224810 nm ) ( Cai et al . , 2014 ) or organic fluorophore ( \\u2248 4 nm ) in cultured neurons . These probes are highly fluorescent; bQDs are nearly infinitely photostable; sQDs are photostable for several minutes; and selected organic fluorophores can be photostable for\\u00a0~1 min . We analyzed the diffusion constants , trajectories , distances from the center of PSDs , changes that occurred over time after transfection , and the role of the extracellular matrix in regulating diffusion . To better resolve PSD dimensions , we used 3-D super-resolution imaging of a PSD protein , Homer1c . We find that\\u00a0>\\u00a090% of AMPARs labeled with organic fluorescent dyes or sQDs are found in PSDs . Less than 10% of these labeled AMPARs are highly mobile and extrasynaptic . However , when AMPARs are labeled with bQDs , virtually all AMPARs ( 90\\u201395% ) are highly mobile and extrasynaptic , presumably because bQDs are sterically hindered from entry into synaptic clefts . We conclude that mobile AMPARs enter PSDs and do not accumulate in extrasynaptic membranes as previously observed in studies using the AMPARs labeled with bQDs . Our data further suggest that AMPAR entry is not regulated by a limited number of \\u2018slots\\u2019 where AMPARs are bound after entry into PSDs . Instead AMPARs are found to diffuse within surprisingly stable \\u2018nanodomains\\u2019 confining AMPARs to sites within PSDs that may regulate access to presynaptic glutamate release . \\n We labeled surface-expressed tagged ( AP-tag or EGFP ) versions of the GluA2 subunits of the AMPAR receptor . They were transiently transfected into cultured rat hippocampal neurons and imaged at 13\\u201316 days in vitro ( DIV ) ( see Materials\\u00a0and\\u00a0methods , and [Howarth et al . , 2005] ) . Three different fluorophores and ligands for coupling to AMPAR were used ( Figure 1c ) : ( 1 ) a small organic fluorophore coupled to streptavidin ( SA ) ; ( 2 ) a sQD with three different ligands: a limiting amount of SA such that each sQD has approximately one SA; a 13kD single-chain antibody against GFP ( from alpaca , commercially known as the Green Fluorescent Binding Protein , or GBP ) , which has only one target on a GFPN-term-AMPAR ( Ries et al . , 2012; Rothbauer et al . , 2008; Wang et al . , 2014 ) . ( In one case , it has also been directed to a pHluorin-AMPAR , a GFP-derivative ( Ashby et al . , 2004 ) . The result for this construct is the same as GFPN-term-AMPAR ) ; ( 3 ) a monomeric streptavidin ( mSA ) , which has only one binding site for biotin and was further molecular engineered from a recent one ( Lim et al . , 2013; Lim et al . , 2011; Chamma et al . , 2016 ) , in order to minimize non-specific labeling with neurons ( Materials\\u00a0and\\u00a0methods and Figure 4\\u2014figure supplement 1 ) . The SA and mSA were used with the same AMPAR constructs as used with the bQDs . The ( tetrameric ) SA was used at a ratio of one per sQD . A commercial bQD containing many streptavidins was also used ( Figure 1c ) . Homer1c ( tag with mGeos or mEos3 . 2 ) was co transfected to generate a super-resolution marker for PSDs . For all experiments , imaging was performed using a cylindrical lens for 3D reconstruction ( Huang et al . , 2008 ) . We noticed that most surface AMPA receptors labeled with SA-bQD one day following transfection ( DIV14 ) , reside in a widely spread extra-synaptic region ( Figure 2a and e ) . This is consistent with our previous results ( Cai et al . , 2014 ) with the difference that the microscope stage drift and chromatic aberrations are corrected in this study . Chromatic aberration was corrected to with 4 nm ( Wang et al . , 2014 ) . The drift was corrected to within approximately 2 nm in x and y , and 16 nm in z ( Lee et al . , 2012 ) . ( See also Materials\\u00a0and\\u00a0methods and Figure 2\\u2014figure supplement 1\\u20137 ) . We also labeled AMPARs with a SA-sQD ( which is approximately half the diameter of the bQDs or 1/8th the volume ) and measured the distance of the AMPAR to the center of PSDs marked by Homer1c-mGeos . Qualitatively , the receptors are often in nanodomains , sub-synaptic regions where the receptor is confined ( Figure 2b , f; also , Figure 4d and e ) . The distance between the center of Homer1c cluster to the center of the AMPAR-sQD is close: a maximum of 150\\u00b142 nm , with 75% and 91% of receptors being localized within 0 . 5 \\u00b5m and 1 . 0 \\u00b5m , respectively , of Homer1c . Lastly , we also employed organic dyes to observe the diffusion of AMPA receptors in live neurons ( Figure 2c and g; summarized in Table 1 ) . This allows a direct comparison of AMPA receptors labeled with bQD and sQD and those labeled with organic fluorophores in live neurons . One advantage of the uPAINT technique using organic dyes compared to sptPALM is that the uPAINT technique only labels cell-surface receptors if cell-impermeable organic dyes are used . By choosing fairly photostable dyes , uPAINT also allows a relatively long time ( limited by the photobleaching ) for the motion to be measured . Here , we find that AMPARs labeled with organic-fluorophores on live cells are colocalized with Homer1c ( Figure 2c ) . To measure the overall population of AMPARs within one synapse , as opposed to the behavior of an individual AMPAR , we obtained super-resolution ( direct-STORM ) images for neuronal receptors using a heavily labeled organic fluorophore ( SA-Cy3B ) after fixing the neurons ( Figure 2d and e ) ( Heilemann et al . , 2008 ) . Cy3B is particularly good with STORM because it can be made photoactivatable with NaBH4 ( Vaughan et al . , 2012 ) . Furthermore , we can label extensively with many organic fluorophores because the hydrodynamic diameter of organic fluorophores is\\u00a0~1 nanometer and even when conjugated with streptavidin , they are only about 4 nm in diameter ( Diki\\u0107 et al . , 2012 ) . Consequently , SA-organic dyes are significantly smaller than either bQDs or sQDs or even labeled-antibodies ( which are commonly used to label AMPARs ) . In addition , non-specific labeling was insignificant . These findings were similar to those obtained using sparsely labeled sQDs and organic dyes labeling on live neurons . Thus , by all three techniques ( with sQDs , with live- and fixed-neurons labeled with organic fluorophores ) we find that the localization of AMPA receptors is predominantly at the synaptic regions . In addition , we observe in these fixed samples , that there is typically one nanodomain per synapse ( Figure 2h ) , although multiple nanodomains do exist ( Figure 2j ) . For all fluorescent tags , \\u2018heat maps\\u2019 were created to plot the diffusion constants ( in \\u03bcm2/s ) versus the AMPAR trajectory range ( in \\u03bcm ) ( Figures 3\\u20135 ) . In general , the lower left-hand corner of the map shows the diffusion of AMPARs where the trajectory range is limited , constrained by PSD ( or slot ) proteins ( Shi et al . , 2001; Opazo et al . , 2012 ) , and the diffusion coefficient of these receptors was slow; those in the upper right-hand corner are not constrained and move faster . Semi-quantitatively , we define \\u2018slow\\u2019 as D \\u2264 10-1 . 75 ( 0 . 018 ) \\u03bcm2/s and \\u2018constrained\\u2019 as a trajectory range of \\u2264 10-0 . 1 ( 0 . 79 ) \\u03bcm . These criteria were used as rough guidelines to be consistent with previous work ( Constals et al . , 2015 ) ( Cai et al . , 2014 ) . The 2D heat maps can also be plotted as a function of DIV to see the effect of transfection time ( see Figures 3\\u20135 ) . Alternatively , the 2D heat maps can also be turned into regular 1D histograms by compressing one axis and fitting the data to a series of Gaussian fits ( see Figures 3\\u20135 ) . Importantly , because many cells were measured , ( 6\\u201313 different culture plates from at least four independent culture preparations , corresponding to 483\\u20131599 synapses ) , 2D histograms should be independent of absolute number of synapses ( synaptogenesis ) . We used bQDs and were able to reproduce results from previous work showing large pools of receptors outside of the synapses . At post-transfection day one there are primarily two peaks in the heat map in Figure 3a with the majority ( 73% of all AMPAR , peak\\u03b1 ) having a fairly fast diffusion constant ( 10-1 . 25=0 . 056 \\u03bcm2/s ) with about a\\u00a0~\\u00a01 . 3 \\u03bcm travel length , and a minor peak ( 27% , peak \\u03b2 ) having slower diffusion constants ( 10-2 . 5=0 . 0032 \\u03bcm2/s ) and smaller trajectory ranges ( \\u2248 0 . 5 \\u03bcm ) . As illustrated in Figure 3c , AMPARs in the \\u03b1 peak are localized outside the synaptic spine on the dendritic shaft distant from Homer1c labeling . Furthermore , the AMPARs are undergoing free diffusion confined over large ( >1 um ) regions in the dendritic shaft . The \\u03b2 peak has two components , both of which are diffusing slow . 5% of all AMPARs are colocalized with PSDs labeled by Homer1c; 22% are not localized to a labeled Homer1c ( see Figure 3d ) . This 22% component could be confined extrasynaptic AMPARs , ( possibly due to crosslinking ) , or they could be diffusing at PSDs not labeled by Homer1c . On post-transfection day 3 , the heat map in Figure 3b is significantly changed with a broader distribution of the diffusion coefficients and trajectory range , covering the range seen at post-transfection day 1 . Unlike the result at post-transfection day 1 , the population of free diffusion ( i . e . the \\u03b1 peak ) has significantly decreased , while the population of slow diffusion ( i . e . the \\u03b2 peak ) has increased . The data also show that the fraction of the slow diffusion is significantly increased on post-transfection day 3 , as compared with that on post-transfection day 1 . A representative trajectory is shown in Figure 3e , which shows the peak \\u03b3\\u00a0of the bQD undergoing limited-diffusion in a small region far from synapses similar to the \\u03b2 peak in Figure 3a . For the bQDs , we measured AMPAR diffusion as a function of DIV ( Figure 3a vs . b ) . All labels were examined after 1 day and after 3 days , following transfection , corresponding to DIV14 and DIV16 , respectively . Because we observed a change in mobility we reasoned that this could be due to cross-linking of AMPAR by the multivalent bQD , or due to an increase in the amount of extracellular matrix . Both factors are known to influence the diffusion of surface-expressed AMPARs ( Chamma et al . , 2016; Frischknecht et al . , 2009 ) . ( This result is further described in Figure 6 . ) Regardless of the exact reason , the fact that labeling with bQDs is a function of the number of DIVs is problematic . When using sQDs , we found no accumulation of large pools of extra-synaptic receptors . Instead , we found that motion is largely constrained-diffusion in PSD nanodomains . The heat maps for the AMPARs labeled with sQDs ( Figure 4 ) are strikingly different from the heat maps for AMPARs labeled with bQDs ( Figure 3 ) . First , the diffusion with any of the three different linkers resulted in much slower diffusion , with a diffusion coefficients\\u00a0~\\u00a010-2 . 5 \\u03bcm2/s . Results were different depending on which linker was used with the mean diffusion constant varying from 10-2 . 41\\u00b10 . 02\\u03bcm2/s for the SA linker ( Figure 4a ) , which is largest and multivalent ( Figure 1b ) , 10-2 . 53\\u00b10 . 02\\u03bcm2/s for the GBP linker ( Figure 4b ) , which intermediate in size and monovalent and 10-2 . 59 \\u00b1 0 . 02\\u03bcm2/s for the mSA linker ( Figure 4c ) , the smallest and monovalent . These results suggest that both the size as well as the valency of the probe can influence the measurements . More specifically , these trends yield 75% ( 70% ) , 82% ( 92% ) and 84% ( 86% ) slowly moving , that\\u00a0is a vast majority of AMPAR-sQDs display constrained diffusion . ( The numbers in parenthesis correspond to taking the integrated area under the Gaussian fit ) . By looking at the diffusion constants , one sees a distribution of constrained diffusion: 84 , 100% and 96% . Both results are in direct contrast with the results for labeling with the bQDs . In addition , the diffusion constant for AMPARs linked via SA to sQD is somewhat broader than the sQD coupled to GFP via GBP , which is somewhat broader than the mSA . ( The mSA result is approximately equal to the smaller organic fluorophores\\u2014see below ) . This possibly implies that , although SA-sQDs are significantly better than the bQD to label AMPARs , smaller ligands ( like mSA ) or organic-fluorophores ( coupled via SA: see below ) are better to study synaptic receptors in neurons . We find that labeling with sQD to AMPAR is not a function of DIV , as day 14 is essentially identical to day 16 , for all three ligands used . This says that the diffusion time and trajectory is a not a function of time , and therefore that the time-dependence , possibly due to cross-linking and differential interaction with extracellular matrix , is not a problem for sQDs-AMPARs . This gives us confidence about our results with sQDs . We note that heat maps tell about the range of diffusion of AMPAR , but do not give the absolute distance from the PSD . Rather it is the trajectory range of receptors which does this ( Figure 2e\\u2013j , or Figure 3c\\u2013e , or Figure 4d\\u2013g ) . To get whether the AMPAR is labeling near or in the PSD , we plot in Figure 4f , the histogram of the distance of AMPAR from the center of Homer1c . The histogram shows that 75% of receptors are localized within 0 . 5 \\u00b5m of Homer1c and 91% within 1 . 0 \\u00b5m , generally undergoing constrained-diffusion ( Figure 2d , f ) . This strongly implies that the great majority of AMPARs are within the PSD . This further implies that our labeling strategy using the small probes does not limit access to the synaptic cleft . We also need to take into account those sQDs that are bound to AMPARs but not in a synapse . ( For the bQDs , this was 22\\u20135%=17% . ) We find that there is 12% of labeled mSA-sQD-AMPAR undergoing constrained diffusion not around a labeled Homer1c . Whether this is due to AMPAR constrained within a PSD containing an unlabeled Homer1c , or constrained within some other domain , is not known . These results are similar to AMPARs labeled with sQDs . We labeled AMPARs using SA-Atto647N or SA-CF633 ( Figure 5 ) . Both of these dyes are bright , especially when several dyes are conjugated per streptavidin , and fairly photostable , >50 s observation time ( Figure 5\\u2014figure supplement 1 ) . ( Why we get this enhanced photostability is unclear . ) In addition , Atto647N has a positive charge and is relatively hydrophobic , whereas CF633 is negatively charged and hydrophilic ( Zanetti-Domingues et al . , 2013 ) . Somewhat better , more specific labeling was shown with CF633 . Overall , these factors have been found to be significant in terms of the measurement of the diffusion of membrane proteins ( Zanetti-Domingues et al . , 2013 ) . A heat map shows that the diffusion constants and range of AMPARs are both quite small when labeled with Atto647N ( post-transfection day 1 and day 3 ) and CF633 ( post-transfection day 1 ) ( Figure 5a and b ) . The results for Atto647N at various concentrations ( 0 . 1\\u20131 nM ) are similar ( Figure 5\\u2014figure supplement 2 ) . There is a slight difference between the two dyes chosen , which indicates that CF633 might have more specific labeling . With Atto647N , fitting the trajectories regardless of the diffusion constants , there is 27% of the AMPAR which has longer trajectory range and larger diffusion constants ( Figure 5a ) ; these are not much present with the CF633 dye , where we find 93% are constrained in nanodomains ( Figure 5b ) . Finally , Figure 5c is the image of the main peak , \\u03b1 in Figure 5a , which shows that AMPAR is not just constrained , but is constrained around the PSD . Figure 5d indicates that 81% of AMPAR-SA-CF633 is localized within 0 . 5 \\u00b5m of Homer1c , and this number increases to 98% within 1 \\u03bcm . Similarly , for Atto647N , 78% is within 0 . 5 \\u00b5m and 84% within 1 \\u00b5m of Homer1c . For fixed neurons , labeled with Cy3B , the results are similar Figure 5\\u2014figure supplement 3 ) : 85% of the receptors are localized within 0 . 5 \\u00b5m of Homer1c . For any of the dyes chosen , whether fixed or live , they indicate a close association between the Homer1c and AMPARs . We imaged native AMPARs labeled with antibody ( GluA2 ) -Alexa647 and measured the distance between native AMPARs and Homer1c clusters using fixed neurons which we transfected with Homer1-mGeos . As shown in Figure 5e and Figure 5\\u2014figure supplement 4 , the results show that 71% of AMPARs are localized within 0 . 5 \\u03bcm , and 96% within 1 \\u03bcm . This is very similar to the Atto647N and CF633 shown in Figure 5d ( 78-98% ) . The results using endogenous AMPARs labeled with antibodies indicate that the overexpression of GluA2 does not significantly affect the measurements . The extra-cellular matrix ( ECM ) is a dense meshwork structure in surrounding brain cells ( Sykov\\u00e1 and Nicholson , 2008 ) . In neurons , surface receptors such as AMPARs and NMDARs , possibly interact directly or indirectly with the ECM , so that the ECM may affect the lateral diffusion of surface receptors in the synapses and surrounding locations ( Dityatev and Schachner , 2003 ) . Recent studies have shown that the ECM is also involved in the mechanism of recruiting glutamate receptors through lateral diffusion ( Dityatev and Schachner , 2003; Dityatev et al . , 2010; Frischknecht et al . , 2009 ) ( Frischknecht and Gundelfinger , 2012 ) . In particular , one recent study claimed that when the ECM was enzymatically removed , the extra-synaptic AMPA receptors moved at a higher diffusion rate and the exchange of synaptic receptors was increased ( Frischknecht et al . , 2009 ) . They therefore concluded that the ECM might regulate short-term synaptic plasticity . However , the observations were on AMPARs labeled by bQDs , so their conclusions should be re-examined in light of our results above . We therefore compared AMPARs labeled with bQDs and sQDs on untreated cultured neurons and on neurons in which ECM was removed using Hyaluronidase . Recall that with bQD , in the presence of ECM ( Figure 3a and dotted lines in Figure 6a ) , we have two populations of AMPAR-bQD: 73% are fast moving ( D\\u00a0~\\u00a010-1 . 20 \\u00b1 0 . 01\\u00a0=\\u00a00 . 063 \\u00b5m2/s ) over a large distance and 27% are slow moving ( D\\u00a0~\\u00a010-2 . 0 - 10-3 . 0 \\u00b5m2/s ) , and constrained to be approximately 0 . 5 \\u03bcm from the Homer1C . After ECM removal , we find that the slow-moving ones ( the \\u03b2 peak in Figure 3a or the white dotted line in Figure 6b ) are gone . All of the remaining AMPAR-bQDs freely diffuse outside of synapses on the membrane of dendrites . In contrast , with small qdots , we observe that the average and distribution of the diffusion coefficient of AMPA receptors was almost the same before ( Dave=0 . 015\\u00a0\\u00b1\\u00a00 . 0006 \\u00b5m2/s ) ( Figure 6b , dotted line ) and after ( Dave=0 . 013\\u00b10 . 0004 \\u00b5m2/s ) ECM removal ( Figure 6c ) . Thus , sQD-tagged AMPAR diffusion is not significantly affected by the presence of ECM , presumably because virtually all of these AMPARs are confined to PSDs where diffusion is limited by interactions with PSD scaffold proteins . Only the bQD-tagged AMPARs , which are virtually all extrasynaptic , are significantly affected by ECM . These results provide an explanation for the changes that occur in the diffusion and trajectories of the bQD-tagged AMPARs going from DIV 14 to 16 in Figure 3a and b . As the neurons develop in culture , progressively more ECM is deposited on the neuronal membrane going from DIV 14 to 16 thereby reducing AMPAR diffusion and trajectories . We find that AMPARs labeled with sQDs differ from AMPARs labeled with bQDs in being able to consistently enter synaptic clefts and nanodomains within PSDs . We observe the same behavior with AMPARs labeled with organic fluorophores . The feature that distinguishes sQDs from organic fluorophores is the significantly longer photostability of the sQDs compared to the \\u2248 1 min lifetime with the organic fluorophores . Using AMPARs labeled with sQDs we examined the stability of AMPAR nanodomains in PSDs over 15 min , exciting the sQDs for 30 s at a time , every 5 min . A previous study observed that most nanodomains were stable for up to one hour , but some nanodomains were dynamic , appearing and disappearing within minutes ( Nair et al . , 2013 ) . This other study , however , used AMPARs labeled with either organic fluorophores or recombinant fluorescent proteins , which limited the imaging to the ensemble average of AMPAR in PSDs . They were therefore unable to observe the dynamics of single receptors except for much shorter times ( <<minute ) , corresponding to the photobleaching time of organic fluorophores . Here we used sQDs to study the dynamics of the nanodomains of the same AMPAR in synapses for long periods ( from 15 min repeatedly , and up to 60 min occasionally ) of time . A total of 126 synapses were observed . As shown in Figure 7 , we first located AMPARs for 30 s , immediately followed by PSD imaging using Homer 1 c super-resolution . AMPARs were then imaged three times more , every 5 min for 15 min , and PSD imaged one final time ( Figure 7a ) . The finite photostability of the fluorescent protein prevented the Homer1c from being monitored more than this . The dendrite morphology was also imaged using a transfected near-IR FP , which fills the cytoplasm of the dendrite ( Figure 7b ) . The nanodomain of the AMPAR , which was\\u00a0~\\u00a0200 nm , remained in approximately the same position in the synapse for the entire fifteen minutes . As shown in Figure 7c and d , when imaged at the beginning ( AMPAR-red , Homer1c-green ) and the end of the fifteen-minute period ( AMPAR-blue , Homer1c-brown ) , the synapse itself had changed position only slightly . To see how much the synapse had drifted over the 15 min , we aligned the images . We first made a linear adjustment , moving both the AMPA receptor trace and the Homer1c cluster about 100 nm and we then rotated the trace and cluster 19\\u00b0 to align them ( Figure 7c ) from one perspective and from then rotated the synapse 10\\u00b0 , perpendicular to the first axis , as shown in Figure 7d . With these relatively minor adjustments , the two images of the synapse nearly perfectly overlapped ( right hand side of Figure 7c , d ) . In addition , Figure 7e shows that the shape of the trace of the AMPAR ( ~200 nm in size ) observed at 0 , 5 , 10 and 15 min , remained quite constant . This indicates that the nanodomain , defined by the constrained area of a single AMPAR , was extremely stable for the fifteen-minute measurement period . Occasionally , AMPAR nanodomains were observed to move within PSDs ( Figure 7\\u2014figure supplement 1 ) . Using the same protocol , the AMPAR nanodomain was first observed at the very edge of the PSD ( 0 min ) after which the nanodomain moves further within the PSD and does not move further . As shown in Figure 7\\u2014figure supplement 1 ( b\\u2013d , the shape and size of the AMPAR nanodomain changes little ( Figure 7f ) , and only the nanodomain position changed after the initial time-point . In one example we also imaged a highly mobile extrasynaptic AMPAR enter into a nanodomain in PSD ( Figure 7\\u2014figure supplement 2 ) . As shown , the AMPAR during the initial time point is widely diffusing near a spine but outside of the PSD . During the last time 15 min time-period , the same AMPAR enters a nanodomain at the edge of the PSD . These images suggest that AMPARs immediately enter nanodomains as they arrive at PSDs after which the nanodomain can reposition within the PSD to a more stable location . In summary , due to the photostability of sQDs we have been able to characterize new details of AMPAR nanodomains and find that they are stable for significant times . \\n With synaptic plasticity , the numbers of AMPARs in PSDs change . Even under basal conditions , the AMPARs are recycling and therefore constantly moving in and out of synaptic clefts and PSDs . AMPAR endocytosis and exocytosis is very likely ongoing at synapses ( Greger and Esteban , 2007 ) and occurs at sites outside of spines ( Anggono and Huganir , 2012 ) or within spines away from PSDs ( Henley et al . , 2011 ) . Consequently , it is important to identify where highly mobile AMPA receptors are located . As we described above , currently the consensus viewpoint is that AMPARs enter PSDs from a highly mobile pool of AMPARs in the dendritic membrane , largely outside of the spine ( Figure 1a , beginning with route 1 ) . Other evidence indicates that the site of exocytosis is in the spine ( route 3 ) . In both cases , according to this data , a large mobile pool of extrasynaptic AMPARs originates from sites in the membrane and then diffuses into the PSD ( route 2 ) . In this study , we find very little evidence for a sizeable pool of mobile extrasynaptic receptors in or around the spines . Most of the previous evidence in support of this AMPAR pool was from studies using super-resolution bQD-tagged AMPARs ( Opazo et al . , 2012; Giannone et al . , 2010; Nair et al . , 2013 ) . Indeed , we too observe this behavior with bQDs ( Figure 3 ) . However , when using when AMPARs that are tagged by significantly smaller fluorophores\\u2014sQDs or organic fluorophores ( Figures 4 and 5 ) \\u2014they are overwhelmingly in the synapse:~84\\u201397% of AMPA receptors were immobilized within PSDs and only 5\\u201316% were freely diffusing in the dendrite membrane . Consequently , we believe that bQDs are too large to enable AMPARs to enter the synaptic cleft . We therefore believe that the previous results with bQDs have been misleading . Our results are consistent with studies using a variety of techniques that have found that AMPA receptors are largely localized to synapses and a relatively small extrasynaptic pool exists . First , electrophysiology has estimated the population of receptors in the synapses as well as in the dendrites . Cottrell et al . observed the density of AMPARs by measuring the glutamate-evoked response current through glutamate stimulating in synaptic regions or in dendrites . They observed the high current signal after glutamate stimulation in synaptic regions , but very low current signal with stimulation in extra-synaptic regions . Through comparing signal strength , they concluded that less than 1% of AMPA receptors were populated in the extra-synaptic regions after synaptogenesis by 11 days ( or more ) in vitro ( DIV ) ( Cottrell et al . , 2000 ) . Second , studies using fluorescence microscopy showed that AMPARs ( MacGillavry et al . , 2013 ) were co-localized with PSD-95 in dissociated cultured neurons , indicating that they were heavily colocalized with the synaptic regions . In addition , it was reported that AMPA receptors were mostly localized at the synaptic region in hippocampal neurons ( Passafaro et al . , 2001 ) . They observed that more than 75% of AMPA receptors , both subunits GluA1 and GluA2 , were colocalized with Shank at the synaptic regions . Also , their spatial and temporal fluorescence images showed that the exocytosed receptors accumulated over time ( ~30 min ) at the synaptic regions . Third , electron microscopy ( EM ) was used to show that AMPA receptors were localized at synapses ( Kharazia et al . , 1996 ) , although they could also exist in extrasynaptic domains ( Tao-Cheng et al . , 2011 ) , especially after synaptic excitation at exocytosis and/or endocytosis sites . All these observations indicate that most AMPARs are localized and clustered at synaptic PSDs in mature neurons . The question arises as to why we observed so few highly-mobile extrasynaptic AMPARs . One possible reason is that in our protocol we have approximately 6 min of dead time with the addition of fluorophores and their binding to cell-surface AMPARs . It is likely many of the labeled AMPARs are diffusing into PSDs during the dead time and we do not observe them . Another possibility is that once AMPARs enter PSDs their residence time there is lengthy and we do not observe AMPARs exiting from PSDs because the rate of exit is much slower compared to the time period ( usually 50 s ) we use to monitor AMPAR dynamics . We also performed experiments where we compared bQD-tagged AMPARs to sQD-tagged AMPARs with and without removing the ECM ( Figure 6 ) . In this case , only the extrasynaptic bQD-tagged AMPARs , which are largely sterically hindered from entering PSDs , were affected . Previous studies found that ECM was acting as a diffusion barrier for extrasynaptic AMPARs during their membrane diffusion from extrasynaptic sites to entry into PSDs ( Frischknecht and Gundelfinger , 2012 ) . Our data is consistent with this conclusion and also supportive of our findings that show the different localization of bQD-tagged AMPARs and sQD-tagged AMPARs . The diffusion of sQD-tagged AMPARs was not affected by enzymatic removal of the ECM because they are almost all within the PSDs away from the effects of ECM removal . Diffusion of bQD-tagged AMPARs was significantly affected because they are largely highly mobile and extrasynaptic . Our findings on the bQD-tagged AMPARs are almost identical to the previous findings of others looking at the effects of ECM removal on bQD-tagged AMPARs ( Frischknecht et al . , 2009 ) . We conclude that the vast majority of AMPARs rapidly enter PSDs and are confined to sub-synaptic diffusion in nanodomains . The diffusion constant is about 10x slower than that of free extrasynaptic AMPARs . The AMPARs therefore do not accumulate in extrasynaptic membranes as previously inferred in studies using AMPARs labeled with bQDs . In previous studies , the findings that AMPARs accumulate in a highly mobile pool of AMPARs in extra-synaptic dendritic membranes suggested that this AMPAR pool existed because of limits in the numbers of AMPARs that are stabilized within PSDs ( Opazo et al . , 2012 ) . According to Opazo et al . , AMPARs are not stabilized within PSDs until \\u2018slots\\u2019 , defined as PSD place-holders for AMPARs become free inside the PSD . Consequently , in this model , AMPARs build up outside of PSDs . We do not observe the highly mobile AMPAR pool when bQDs are replaced by a variety of smaller ( organic and sQDs ) fluorophores and linkers indicating that the highly mobile pool is an artifact of using bQDs attached to the AMPARs . Thus , the number of synaptic AMPARs does not appear to be limited by whether PSD slots are already occupied . The picture that emerges from our data is that after entry into PSDs , AMPARs are mobile but constrained within confined sub-synaptic nanodomains that are surprisingly stable , lasting for more than 15 min . It is likely that the AMPAR entry into PSDs is more rapid than the membrane diffusion rate to the PSDs . If this were not case , there would be accumulation of a mobile extra-synaptic pool , similar to what is observed when bQDs are attached to the AMPARs . The positioning of these nanodomains is likely critical for the AMPARs to receive glutamate from the presynaptic release sites ( Tang et al . , 2016; Jeyifous et al . , 2016 ) . Our conclusion that the number of synaptic AMPARs is not limited by whether PSD slots are occupied suggests that other factors regulate AMPAR numbers in PSDs . Potential candidates for what limits AMPAR numbers in PSDs are AMPAR exocytosis rate ( Figure 1a , route 1 ) , AMPAR endocytosis rate ( Figure 1a , route 3 ) and/or barriers such as extracellular matrix ( Figure 6 ) that regulate AMPAR membrane diffusion on the membrane ( Figure 1a , route 2 ) . \\n Experiments were performed with a Nikon Ti Eclipse microscope with a Nikon APO 100 X objective ( N . A . 1 . 49 ) . The microscope stabilizes the sample in z-axis with the Perfect Focus System . An Agilent laser system MLC400B with four fiber-coupled lasers ( 405 nm , 488 nm , 561 nm and 640 nm ) was used for illumination . Elements software from Nikon was used for data acquisition . A back illuminated EMCCD ( Andor DU897 ) was used for recording . For 3-D imaging , a cylindrical lens ( CVI Melles Griot , SCX-25 . 4-5000 . 0- C-425-675 ) of 10 m focal length was inserted below the back aperture of the objective . A motorized stage from ASI with a Piezo top plate ( ASI PZ-2000FT ) was used for x-y-z position control . A quad-band dichroic ( Chroma , ZT405-488-561-640RPC ) was used and band-pass emission filter 525/50 , 600/50 , 710/40 , 641/75 was used for fluorescence imaging . Stage drift is a significant problem in taking super-resolution images . As we reported earlier ( Lee et al . , 2012 ) , making fiduciary-marked patterns on coverslips is a very efficient method to correct stage drift instead of using the previously favored fluorescent beads and gold nano-particles as fiduciary markers to correct stage drift . The perfectly stable fiduciary-marked patterns proved far superior to correct stage drift since it is impossible to perfectly immobilize beads or nano-particles on a coverslip surface , making it especially difficult to image live cells . More importantly , randomly spreading beads or nano-particles is a critical drawback for using them as fiduciary markers because it needs optimal number of fiduciary markers in the region of interest , whereas our fiduciary markers are a uniform pattern . Thus , the fluorescent beads and gold nanoparticles that have typically been used to correct stage drift are not effective , especially when imaging live cells . We have previously reported that mammalian cells can be stably cultured on fiduciary-marked coverslips without any other coating or treatment; however , although we had no problems taking super-resolution images of mammalian cells with this fiduciary pattern , we were not able to culture dissociated neurons on fiduciary-patterned coverslips . In general , poly- ( L/D ) -lysine coating is necessary to culture dissociated neurons on glass coverslips because the cell membranes of neurons are negatively charged . Poly-lysine coating is not sufficient for neurons to adhere the fiduciary marker coverslips . We then employed laminin to coat coverslips . As shown in Figure 2\\u2014figure supplement 3 ( a ) , we first coated poly-lysine after treating coverslips in oxygen plasma and then applied an additional coating of laminin on the fiduciary-patterned coverslips . We found that dissociated neurons could be stably cultured on fiduciary-patterned coverslips after coating them with both poly-lysine and laminin , sequentially , as in Figure 2\\u2014figure supplement 3 ( b ) . In terms of connection between neurons , we could obtain comparably good neurons on the fiduciary marked coverslip as we could culture them on the poly-lysine coated coverslip . More details to coat poly-lysine and laminin on the fiduciary marked coverslips were described in the method section . As shown in Figure 2\\u2014figure supplement 3 ( b\\u2013d ) , once neurons have been stably cultured on fiduciary-patterned coverslips , they can then be successfully transfected . Furthermore , IR illumination , as shown in Figure 2\\u2014figure supplement 3 ( b ) , is effective to allow clear observation of uniformly patterned fiduciary markers without affecting the visibility of fluorescence color channels . Hence , these coated fiduciary patterned coverslips are very useful to exclude stage drift , allowing for the culture , transfection , and super-resolution imaging and tracking of live neurons . Here , to demonstrate the method for taking super-resolution fluorescence images , neurons were transfected with two plasmids , one allowing expression of the post-synaptic protein homer1 with a photoactivable fluorescence protein , mGeos or mEOS3 . 2 , and the other allowing expression of glutamate receptors , AMPARs with AP tag to biotinylate . We then labeled the glutamate receptors using qdots conjugated with either streptavidin or anti-GFP nanobodies . As shown in Figure 2\\u2014figure supplement 3 ( c ) , at 530 nm\\u00a0~\\u00a0700 nm of emission in wavelength , we observed fluorescence from the proteins and receptors without any auto-fluorescence from the fiduciary pattern . We could clearly observe the fiduciary pattern in the IR channel and the live cells in the visible channels , as shown in Figure 2\\u2014figure supplement 3 ( d ) . Fiduciary markers in the IR image are tracked , fitting the fiduciary markers using either the Gaussian or the Airy function . The IR camera electronically synchronizes with the visible camera so that the traces of fiduciary markers as recorded by the IR camera accurately calculates the stage drift at the time the cells are imaged . We can then correct the super-resolution cell images for stage drift accordingly . The fiduciary marker trace corresponds to the stage drift . Thus , through this post-processing , we can correct for stage drift . Since fiduciary markers are stationary , stage drift can be much more accurately calculated using them than when using more conventional existing methods of calculating stage drift with fluorescent beads or gold nano-particles . As shown in Figure 6 , using fiduciary markers , stage drift can be controlled within an error of around 1 nm for x and y and 15 nm for z . Another technical difficulty is chromatic aberration . The three-dimensional multi-color super-resolution imaging technique allows us to observe the interaction between two different species/types of proteins or organelles in cells; however , the color shifts that occur in different color channels is a concern to take images can be even more pronounced and problematic in super resolution images different color channels . These color shifts will cause the images to be slightly different . Even though we observe exactly the same object at different color channels , the images will be a little mismatched in different color channels . Since the refractive index of light depends on its wavelength , this chromatic aberration causes focus on an object to be detected in a little different place in each color channel . This means that we have to correct this difference chromatic aberration for studying the interaction between different proteins or organelles when using multi-color super-resolution imaging . Most commercially available objective lenses are designed to optically minimize chromatic aberration . Based on our measurements , this optical design for objective lenses will corrects within the margin of error of about 100 nm; however , given that the spatial resolution of super-resolution microscopy is only about 20\\u00a0~\\u00a030 nm , a margin of the error of 100 nm is a significant error . This means that achromatic optical design is not sufficient to correct chromatic aberration , especially in super-resolution imaging . Using our method , chromatic aberration can be corrected within a margin of error of 2 nm . We accomplish this with as reported ( DeWitt et al . , 2012; Pertsinidis et al . , 2013; Wang et al . , 2014 ) . In order to correct chromatic aberration , we made a nano-hole pattern on a silver- coated coverslip . Using a thermal evaporator , a glass coverslip is coated with silver at a thickness of about 100 nm using a thermal evaporator . The nanohole pattern is produced by fast ion beam ( FIB ) equipment , sputtering holes of about 100 nm in size every 1 . 5 \\u03bcm . Holes are about 100 nm in diameter and are located every 1 . 5 \\u03bcm . We took images for the nanoholes on three at the different color channels . As shown in supporting figure Figure 2\\u2014figure supplement 7 ( a\\u2013d ) , the positions of the nanohole patterns are slightly different on each color channel . In Figure 2\\u2014figure supplement 7 ( d ) , the red and green spots are the nanoholes in the green channel and in one of two red channels , shown by zooming in on the image of the the overlaid two color images . The significantly different position of the two sets of nanohole patterns is a result of shows the positions of red and green spots are significantly different , which is caused by chromatic aberration . We measured the chromatic aberration between the green ( GFP ) 530\\u00a0nm channel and red ( RFP ) 600\\u00a0nm\\u00a0channel and 705\\u00a0nm\\u00a0channels . As shown in Figure 2\\u2014figure supplement 7 ( e ) , the\\u00a0average difference in the location of the nanoholes in between two the nanohole pattern on channels 530\\u00a0nm and 600\\u00a0nm is about 80 nanometers . We found a similar range for the difference between nanohole locations in any two of the three-color channels . This means that , when obtaining multi-color super-resolution images , the error of co-localizations of any two colors is expected possible to be at least approximately 80 nm . Furthermore , we also evaluated the validity of the mapping functions for 3D imaging . We obtained the mapping function at a focal point of z\\u00a0=\\u00a00 , focal point , with an error of 2 nm , and , as shown in Figure 2\\u2014figure supplement 7 ( e ) , this mapping function can be applied is valid up to within z=\\u00a0\\u00b1400 nm with an error of only 4 nm . when the error is 2 nm at z\\u00a0=\\u00a00 . Actually , we obtained the mapping function at z=200 nm , 400 nm , 600 nm , and 800 nm and , as represented in Figure 2\\u2014figure supplement 7 ( e ) , the error is a little smaller than when using the mapping function obtained at z\\u00a0=\\u00a00; however , since a 3 D super-resolution image using astigmatism is not discrete along the z axis , it is not easy to have mapping functions along the z axis . At focal points above 400 nm , the error dramatically increased . We obtained the mapping function at z\\u00a0=\\u00a0200 nm , 400 nm , 600 nm , and 800 nm , as represented in Figure 2\\u2014figure supplement 7 ( e ) . At focal points above 400 nm , the error dramatically increased . Although the error is a little smaller at focal points higher on the z axis than when using the mapping function obtained at z=0 , we conclude that when getting 3D images using astigmatism , the mapping function at z=0 is accurate enough to correct chromatic aberration . This is because a 3 D super-resolution image using astigmatism is not discrete along the z axis . Hence , it would be too time consuming to try to obtain more accurate mapping functions all along the z axis . Primary hippocampal cultures were prepared from E18 rats according to UIUC guidelines as previous described ( Cai et al . , 2014 ) with the following modifications . In brief , hippocampal tissues were dissociated in 3 mg/mL protease and plated on 25 mm coverslips coated with 1 mg/mL poly-l-lysine and laminin . Neurons were cultured at 37\\u00b0C with 5% CO2 in neurobasal media ( ThermoFisher , 21103049 ) with B-27 supplement ( GIBCO ) , 2 \\u03bcM glutmax and 50 unit/mL penicillin and 50 unit/ml streptomycin . On 12\\u201313 days in vitro ( DIV ) , neurons were co-transfected with Homer1c-mGeos ( 1 \\u03bcg/coverslip ) with GluA2-AP ( 1 \\u03bcg/coverslip ) and BirA-ER ( 1 \\u03bcg/coverslip ) , and Homer1c-mGeos ( 1 \\u03bcg/coverslip ) for AMPA receptors by using Lipofectamine 2000 transfection reagent . At 24\\u00a0~\\u00a072 hr after transfection , the coverslips were transferred to warm imaging buffer ( HBSS supplemented with 10 mM Hepes , 1 mM MgCl2 , 1 . 2 mM CaCl2 and 2 mM D-glucose ) for 5 min incubation and mounted onto an imaging dish ( Warner RC-40LP ) . In the imaging dish , neurons were incubated in imaging buffer containing QDs and casein ( ~400 times dilution for bQDs , and\\u00a0~80 times dilution for sQDs; stock solution purchased from Vector labs , SP-5020 ) for 5 min at 30\\u00b0C and washed with 10 ml of imaging buffer . Finally , 3 mL of imaging buffer was added to the imaging dish that was subsequently mounted on the microscope . For ECM removal , we enzymatically removed the ECM using hyaluronidase ( HX0514 \\u2013 Calbiochem , 100 units/ml left overnight at 37\\u00b0 with 5% CO2 ) . For labeling endogenous AMPARs using antibody , we conjugated Alexa647 NHS ester ( Thermo-Fisher: A37573 , Waltham , MA ) with anti-GluR2 ( Anti-Glutamate Receptor 2 Antibody , extracellular , clone 6C4: Sigma-Aldrich: MAB397 , St . Louis , MO ) To express mSA , E . coli strain BL21 ( DE3 ) pLysS was transformed with an appropriate expression vector and plated on Luria Bertani ( LB ) agar containing ampicillin . Several colonies were selected from the plate for overnight growth at 37\\u00b0C in an LB starter culture containing 200 \\u03bcg/ml ampicillin . The following morning , the starter culture was diluted 100 fold into Terrific Broth ( TB ) containing 0 . 05% glucose , 0 . 5% glycerol , 0 . 2% \\u03b1-lactose , 1 mM MgSO4 , 900 \\u03bcM biotin , and 200 \\u03bcg/ml ampicillin . The cells were grown at 37\\u00b0C and 300 rpm until OD600 reached 0 . 3 , at which point the shaker temperature was reduced to 20\\u00b0C . Once the culture has reached OD600=0 . 6 , IPTG was added to the final concentration of 75 \\u03bcM and the shaker speed was increased to 375 rpm . After 24 hr induction at 20\\u00b0C , cells were removed by centrifugation and the culture medium containing secreted mSA was collected for purification . The medium was sonicated with a 200 W sonicator ( 30 s on , 30 s off for 4 cycles at 50% amplitude capacity ) . The pH of the culture medium was adjusted to 7 . 5 using NaOH and imidazole was added to 10 mM . The medium was then centrifuged at 12 , 000 rpm for 20 min to remove any precipitates . The clarified medium was passed through a column packed with Ni-NTA Superflow Agarose pre-equilibrated in PBS and 10 mM imidazole . The column was then washed with PBS and 20 mM imidazole . Finally , the bound protein was eluted with 300 mM imidazole in PBS . The eluted protein was concentrated and buffer exchanged to 100 mM glycine buffer pH 2 . 3 using a centrifugal filter to remove bound biotin . Finally , the sample was buffer exchanged to PBS . Yields were estimated based on A280 measurements and purity was assessed by SDS-PAGE with Coomassie staining . All streptavidin homologs have a conserved three dimensional structure , but differences in the binding pocket residues lead to differences in biotin binding . For example , the main chain root mean squared deviation between mSA and streptavidin is 0 . 82 \\u00c5 but there is\\u00a0~105 fold difference in their binding affinities . Although extremely high affinity is not needed for labeling live cells , rapid dissociation of bound biotin interferes with detection and measurement because it results in loss of fluorescence intensity over time . As such , improving the dissociation kinetics ( i . e . slowing the off rate koff ) of mSA is important to develop a useful imaging reagent . We have previously demonstrated that replacing S25 with histidine reduces koff by 7 . 5 fold by blocking solvent entry into the binding pocket through steric hindrance ( Demonte et al . , 2013 ) . Keeping water molecules out of the binding pocket is important because they compete with mSA-biotin hydrogen bonds and lead to biotin dissociation . S25 is located in the loop between \\u03b2\\u2212strands 1 and 2 ( L1 , 2 ) , which is found at the subunit interface of streptavidin but is solvent exposed in mSA . Therefore , blocking solvent entry in mSA in part involves recreating the lost physical barrier near L1 , 2 . To improve the dissociation kinetics further , we modeled the binding pocket of mSA using high affinity dimer bradavidin II ( Brad2 ) , which binds biotin with Kd\\u00a0<10\\u221210 M ( Helppolainen et al . , 2008 ) . L1 , 2 and the binding loop L3 , 4 adopt nearly identical conformations in mSA and Brad2 , thus allowing key residues to be reliably modeled in mSA . The Brad2 residue corresponding to mSA S25 is R17 , which makes a hydrogen bond and a salt bridge with G103 and D104 , respectively , to form a solvent barrier around L1 , 2 as shown in Figure 4\\u2014figure supplement 1 ( a , b ) . Arginine is larger than serine and histidine and is therefore more effective in blocking solvent entry . To reconstruct the solvent barrier in mSA , we mutated S25 to R , and measured the dissociation rate of the mutant mSA using fluorescence polarization spectroscopy . The dissociation rate of mSA-S25R is slower than that of mSA or mSA-S25H . Brad2 also contains F42 in L3 , 4 to create a hydrophobic lid over bound biotin and trap the molecule in the binding pocket . Mutating F42 to alanine significantly reduces the affinity of interaction , indicating that the hydrophobic lid also plays an important role in biotin binding . We therefore introduced the corresponding mutation T48F in mSA-S25R to create a hydrophobic binding pocket and further reduce the dissociation rate . The measured koff of mSA-S25R/T48F ( mSA-RF ) is 37 fold lower compared to original mSA , that\\u00a0is . t\\u00bd=402 min , and\\u00a0~5 fold lower than the best mutant to date , thst\\u00a0is mSA-S25H ( Figure 4\\u2014figure supplement 1c ) . The single point mutant mSA-T48F resulted in only limited improvement in biotin dissociation ( t\\u00bd=13 min ) , indicating that the two mutations function cooperatively . We also tested an alternative design of the hydrophobic lid with a tryptophan , mSA-S25R/T48W/D124T ( or mSA-RWT ) , which had similarly slow dissociation kinetics , t\\u00bd=330 min ( Figure 4\\u2014figure supplement 1c ) . Therefore , construction of solvent barriers through cooperative mutations in L1 , 2 and L3 , 4 can modulate the dissociation kinetics and improve the mSA-biotin interaction . To demonstrate the use of new engineered mSA in cell labeling , we expressed a biotin acceptor peptide ( AP ) fused to CFP and a transmembrane helix ( TM ) on the surface of HEK293 . We then added purified E . coli biotin ligase , BirA , to the cell culture to induce site-specific biotinylation of AP . The cells were fixed and labeled with mSA-RF\\u2014EGFP for imaging by confocal microscopy . The CFP positive cells were selectively labeled , suggesting high specificity of interaction between mSA-RF and biotin ( Figure 4\\u2014figure supplement 1d and e ) . Furthermore , the intensity of EGFP fluorescence remained high after 1 hr , indicating that the interaction is stable and mSA-RF remained stably bound during measurement . In order to verify whether SA-qdots were in fact cross-linking , we compared the diffusion coefficient of mSA-qdots and SA-qdots on the lipid bilayer at different concentrations ( 0 . 2% and 1 . 0% ) of biotinylated lipid molecules . If a qdot binds to one biotinylated lipid molecule , the resulting labeled lipid molecule will diffuse at the same rate as a free diffusion of a lipid molecule; whereas , if a qdot binds to multiple biotinylated lipid molecules , the diffusion rate will significantly slow down as shown in Figure 4\\u2014figure supplement 2 . We first determined whether there is any difference in the diffusion rate of molecules labeled in three ways: ( 1 ) those labeled with normal SA-qdot conjugates formed with big qdots ( one conjugate has many streptavidin and numerous binding site on SA-big qdots ) ; ( 2 ) those labeled with normal SA-qdot conjugates formed with small qdots ( one conjugate has four binding sites from the streptavidin ) ; and ( 3 ) those labeled with mSA-qdot conjugates using monomeric streptavidin and small qdots ( one conjugate has only one binding site ) . We hypothesized that normal streptavidin conjugates with both small and big qdots would be much more likely to bind multiple biotins than would be the case with mSA conjugates . We found this to be the case , as represented in Figure 4\\u2014figure supplement 3 . Figure 4\\u2014figure supplement 3 shows that there was little difference in the diffusion rate of molecules labeled with normal SA-qdots , whether conjugated with small or big qdots: there were a large number of slow diffusive qdots for both normal streptavidin conjugates , indicating that they had cross-linked to multiple molecules , even at the 0 . 2% biotin concentration shown in Figure 4\\u2014figure supplement 3 . At 1 . 0% biotin , there was even a larger population of slow diffusive qdots , that\\u00a0is , cross-linked qdots . By contrast , in the case of mSA qdot conjugates , we found that most qdot conjugates were freely diffusive at 0 . 2% as well as at 1 . 0% biotin concentration; that is , mSA-qdots do not show a slow diffusive population of labeled molecules at either 0 . 2% or 1 . 0% concentrations . These observations showed that normal streptavidin frequently cross-links to multiple molecules while monomeric streptavidin did not . In order to further examine cross-linking of normal and monomeric streptavidin , we compared the run length of kinesin ( K432 , biotin at C-terminus ) . It is well known that when the cargo ( in this case the streptavidin-qdot conjugate ) binds to multiple motor proteins , the cargo will be transported over a longer distance than when the cargo binds to a single motor protein . ( Beeg et al . , 2008; Furuta et al . , 2013 ) A single motor protein transports cargo approximately 1 um , while multiple motor proteins can walk farther than 1 um . Thus , if we compare the run length of kinesin using SA-qdots and mSA-qdots at different ratios of kinesin and qdots , then we can examine the possibility that one qdot is binding to multiple proteins versus binding to a single protein . For accuracy of labeling , we want to confirm that binding is to a single protein and a shorter kinesin run length will indicate this . In the case of mSA-qdots , we found the run length was the same as that of a single kinesin at a ratio of 2:1 ( two kinesin molecules to one qdot ) . As expected ( Figure 4\\u2014figure supplement 4 ) , the run length was not changed even when the ratio of kinesin to qdots was increased to 6:1 . This means that mSA-qdots do not bind to multiple motor proteins . By contrast , in the case of SA-qdots , the run length of kinesin was more than the typical run length of kinesin , as shown in Figure 4\\u2014figure supplement 4 . Even at a ratio of 2:1 , the histogram of run lengths shows that some kinesin walked longer than the typical kinesin run length . At 6:1 , we found that the run length significantly increased , indicating cross-linking . These results mirror the results we obtained in our first experiment about diffusion rates of labeled molecules in the lipid bilayer . After focusing the sample in bright field , the Perfect Focus System was activated to minimize the sample drift in z direction . The samples were then scanned in the GFP channel ( 488 excitation , 525/50 emission ) to locate transfected cells . A fluorescent image of the cells was taken for reference . To track the QD labeled receptors , 488 nm or 561 nm lasers was used for excitation in the hi-low-fluorescence mode with an appropriate band-pass filter for collecting the fluorescence . In terms of localization precision of fluorophores , as shown in Figure 2\\u2014figure supplement 8 , organic dyes were about 11\\u00a0~\\u00a021 nm for x and y ( Atto647N \\u2013 12 nm for both x and y ) and CF633- 21 nm for x and 12 nm for y ) and about 32 nm ( Atto 647N ) /42 nm ( CF633 ) for z . Qdots were 9 . 6 nm for x , 8 . 2 nm for y , and 18 nm for z . After the tracking experiment , the PALM ( Betzig et al . , 2006 ) experiment was carried out on the same neurons . PALM was used for super-resolution of post-synaptic density . Post-synaptic protein Homer1c was used as the PSD marker and its C- terminus was fused to photoactivatable protein mGeos . A 100 ms 405 nm laser pulse was used to activate mGeos proteins from dark to green fluorescent state . The sample was then excited with a 488 nm read out laser and emission was collected with a 535/50 band-pass filter . The cycle was repeated for 200\\u2013300 times , collecting 4000\\u20136000 frames . The z calibration was created following method described by Huang et al . ( Huang et al . , 2008 ) using fluorescent beads on a glass surface and applied to both tracking and PALM data . Cy3B-STORM was done following the procedure described on Joshua et al ( Vaughan et al . , 2012 ) . We labeled neurons with Cy3B-SA ( 2 \\u03bcM ) following the procedure of sQD labeling . We then fixed the cells with a solution of 4% PFA and 0 . 1% Glutaraldehyde in HBSS for 10 min . Wash with three tines PBS . Treat the sample with 10 mM of NaBH4 in PBS freshly prepared for 10 min . Wash with PBS . Before imaging , we add imaging buffer consisting of 5 \\u03bcL of PCD , 20 \\u03bcL of PCA , 4 \\u03bcL of Trolox in 471 \\u03bcL of T-100 ( 100 mM Tris at pH 8 . 0 ) . Images were acquired at 10 Hz using 405 nm laser at low power to do activation and 561 nm laser to excite fluorescence . In order to image native AMPARs , we labeled AMPARs using Anti-GluA2-Alexa647 after fixation . For STORM imaging , we added imaging buffer consisting of 5 mM MEA ( Sigma: 30070 , St . Louis , MO ) solution ( ~pH 8 . 0 ) and additionally added 40 mM Sodium D/L-lactate ( Sigma: 71720 , St . Louis , MO ) and EC-Oxyrase ( Sigma: SAE0010 , St . Louis , MO ) in PBS in order to improve the photo-stability . For the tracking data , centroids of the all the QDs were localized in all the frames and a map of all the places QDs visited were obtained . A Matlab code was used to recover the trajectories of the QDs . In brief , the code finds locations of QDs in time t , and searches for nearby QDs in time t+1 as the next point on the trajectory . In the 3-D single particle tracking experiment , the maximum displacement of a QD in one time step is set to be 1 \\u03bcm . The trajectory range was obtained by calculating the range of the trajectory in the x- , y- , and z- direction , and using the maximum of the three parameters . The diffusion coefficients from the trajectories were calculated in Matlab by fitting the first 4 points of mean-square-displacement curve . For the PALM data , positions of proteins detected in each frame are localized , cluster analysis was used for identify synapses . To determine if a trajectory was synaptic , the centers of synapses were determined , and trajectories within 2 \\u03bcm radius of each synapse were identified . For each of these trajectories , the average distance was calculated between the center of the nearest synapse and all points on the trajectory . Any trajectories with an average distance smaller than 0 . 55 \\u03bcm were considered synaptic . The visualizations of post-synaptic densities and sQD labeled AMPA/NMDA receptor tracking trajectories were created using VMD 1 . 9 . 2 ( Humphrey et al . , 1996 ) , a widely-used program for visualizing and analyzing macromolecular structures and molecular dynamics ( MD ) simulation trajectories . VMD is optimized for dealing with large data sets containing millions of particles and thousands of trajectory frames , as encountered in the current study .\",\n          \"Mature neural networks synchronize and integrate spatiotemporal activity patterns to support cognition . Emergence of these activity patterns and functions is believed to be developmentally regulated , but the postnatal time course for neural networks to perform complex computations remains unknown . We investigate the progression of large-scale synaptic and cellular activity patterns across development using high spatiotemporal resolution in vivo electrophysiology in immature mice . We reveal that mature cortical processes emerge rapidly and simultaneously after a discrete but volatile transition period at the beginning of the second postnatal week of rodent development . The transition is characterized by relative neural quiescence , after which spatially distributed , temporally precise , and internally organized activity occurs . We demonstrate a similar developmental trajectory in humans , suggesting an evolutionarily conserved mechanism that could facilitate a transition in network operation . We hypothesize that this transient quiescent period is a requisite for the subsequent emergence of coordinated cortical networks . \\n Multiple cognitive functions emerge rapidly during early development . Neural networks enable precise spatiotemporal coordination of synaptic and cellular activity in mature brain functions ( Fries et al . , 1997; Maingret et al . , 2016; Peyrache et al . , 2009; Khodagholy et al . , 2017 ) . How immature neural networks develop into their mature form needed for the complex computations underlying cognition remains poorly understood . The first organized and predominant pattern of neural activity that appears in cortical circuits across species is a spindle-like oscillation ( 10\\u201320 Hz ) that occurs intermittently on a background of relative neural inactivity . Known as spindle bursts in rodents and delta brushes in humans , this immature network activity is commonly triggered by peripheral stimuli ( An et al . , 2014; Hanganu et al . , 2006; Khazipov et al . , 2004 ) . They demarcate cortical columns or pre-columns and have been linked to neuronal survival ( Golbs et al . , 2011 ) , establishing sensory ensembles , and critical period plasticity ( Khazipov et al . , 2004; Colonnese et al . , 2010; Khazipov et al . , 2013; Winnubst et al . , 2015 ) . Spindle bursts are characteristic of the first postnatal week of rodent development , and delta brushes disappear shortly after term in human neonates ( Torres and Anderson , 1985 ) , emphasizing their transient role in network maturation . In contrast , mature cortex exhibits perpetual , complex patterns of neural activity that appear and interact across a wide range of frequencies ( Tort et al . , 2009; Fujisawa and Buzs\\u00e1ki , 2011; Bosman et al . , 2012 ) . The organization of this activity creates precise spatiotemporal windows for neural synchronization , enabling plasticity processes and generation of neural sequences ( Masquelier et al . , 2009; Geisler et al . , 2010; Foster and Wilson , 2006 ) . Such network properties facilitate information processing , and resultant activity patterns have been causally linked to cognitive processes from stimulus perception to learning and memory ( Fries et al . , 1997; Maingret et al . , 2016; Peyrache et al . , 2009; Khodagholy et al . , 2017 ) . Understanding when and how these properties develop in the immature brain is critical given that delay or failure to express mature brain activity is a strong risk factor for subsequent impaired cognition ( Khazipov et al . , 2004; Anderson et al . , 1985; Seelke and Blumberg , 2008; Holmes and Lombroso , 1993 ) . We hypothesized that emergence of these advanced neural network properties could be heralded by the disappearance of immature activity patterns . During this developmental epoch , cortical microcircuits are still undergoing dramatic changes in anatomical connectivity ( Khazipov et al . , 2004 ) and functional connectivity between and within cortical layers is initiated ( Bureau et al . , 2004 ) in part through an abrupt increase in synaptogenesis in superficial layers ( Naskar et al . , 2019 ) . Robust GABA-mediated fast inhibition also appears ( Daw et al . , 2007; Goldberg et al . , 2011 ) as inhibitory network influences of interneurons increase ( Modol et al . , 2020; Tuncdemir et al . , 2016; Favuzzi et al . , 2019 ) . These local circuit changes are paralleled with the gradual ingrowth of subcortical neurotransmitters that assist in setting brain state changes . Together , such modifications could prime a shift in network operating mode , setting the stage for re-emergence of a different oscillation that occupies a similar frequency band , thalamocortical sleep spindles . In contrast to spindle bursts , these oscillations are characteristic of the offline state of non-rapid eye movement ( NREM ) sleep only , and they facilitate memory consolidation ( Maingret et al . , 2016; Latchoumane et al . , 2017 ) . To investigate this hypothesis , we examined neural network dynamics across early rodent and human development . We targeted epochs spanning disappearance of immature activity and emergence of mature sleep patterns in each species . For rodents , we developed conformable , minimally invasive , high-resolution electrocorticography arrays coupled with high-density implantable probes and recorded spontaneous in vivo electrophysiological patterns from unanesthetized mice across the first two postnatal weeks . This approach enabled simultaneous recording of large-scale synaptic and cellular activity without damaging fragile cortical circuits . We found that the transition between immature and mature network dynamics was characterized by unexpectedly decreased coordinated cellular and synaptic activity at the beginning of the second postnatal week . After this timepoint , precise neuronal synchronization and oscillatory coupling in space and time robustly emerged . Analysis of continuous electroencephalography ( EEG ) recordings from human subjects , 36\\u201369 weeks post-gestation , revealed a similar developmental trajectory characterized by a transient decrease in neural activity prior to onset of spatiotemporal oscillatory coupling . Therefore , a shift from local , loosely correlated , prominently sensory-driven patterns to internally organized , spatially distributed , and temporally precise activity is preceded by a transient quiescent period . These findings suggest that mechanisms to developmentally regulate functional network capacity may be evolutionarily conserved and manifest on the systems level as similar , discrete states . \\n To identify and characterize emergence of advanced neural network properties in the developing brain , we acquired high spatiotemporal resolution electrophysiological recordings from somatosensory cortex of unanesthetized mouse pups aged postnatal day ( P ) 5 to 14 ( n = 108 pups ) , an epoch that spans the transition from immature to mature activity patterns during sleep . We used minimally invasive surface electrocorticography arrays ( NeuroGrids , n = 70 pups ) to permit a spatially extensive survey of cortex . These customized NeuroGrids ( Khodagholy et al . , 2015 ) consisted of 119 electrodes regularly spaced on a diagonal square-centered lattice embedded in 4-\\u00b5m-thick parylene C to conform to the cortical surface ( Figure 1A ) . Recordings were made following recovery from surgery to eliminate any influence of anesthesia ( Figure 1\\u2014figure supplement 1 and Figure 1\\u2014figure supplement 2 ) . Mice at these ages enter into a cyclical pattern of sleep and wakefulness as assayed by peripheral indicators , such as muscular tone , movements , and heart rate ( Figure 1\\u2014figure supplement 1 ) . We ensured electrodes used for analysis were located in somatosensory cortex by postmortem histology . Following NeuroGrid recordings , the location of the array was marked on the surface of the brain using a biocompatible fluorescent material ( Rauhala et al . , 2020 ) and tissue was harvested for immunohistochemistry . vGlut2 staining identified the location of primary somatosensory and visual cortices relative to NeuroGrid placement on flattened axial slices ( Figure 1B , upper ) , and the spatially extensive nature of the array compared to the size of the pups\\u2019 brains further facilitated anatomical targeting . This histology-based electrode grouping corresponded to spatial localization of electrophysiological activity patterns , identifying electrodes recording from somatosensory cortex ( Figure 1B , lower; Figure 1\\u2014figure supplement 3 ) . To additionally capture neural spiking patterns across cortical layers , we stereotactically implanted silicon probes ( linear electrode arrangement ) in somatosensory cortex of a separate cohort ( n = 38 pups ) . Immunohistochemistry of coronal slices from these brains verified probe placement within somatosensory cortex , permitted allocation of recording electrodes to cortical layers ( Figure 1C , upper ) , and demonstrated transcortical neural spiking patterns ( Figure 1C , lower; Figure 1\\u2014figure supplement 4 ) . We expected that physiologically significant features of neural network maturation would be conserved across species . Thus , in parallel , we obtained continuous EEG recordings from normal human subjects 36\\u201369 weeks post-gestation ( 1 day\\u00a0to\\u00a07 months after birth ) , an epoch that similarly spans the transition from immature to mature activity patterns of human sleep and can be mapped to corresponding timepoints of rodent brain development ( Jouvet-Mounier et al . , 1969; Clancy et al . , 2001; Milh et al . , 2007 ) . We first identified epochs of artifact-free behavioral immobility consistent with sleep in the pups ( Figure 1\\u2014figure supplement 5 ) and compared the activity present with human sleep EEG . Sleep in the youngest mouse pups ( P5\\u20137 ) was characterized by irregular , discontinuous bursts of neural activity ( Figure 1D , E , upper traces ) . The predominant activity was organized into spindle bursts , oscillations characteristic of developing rodent sensory cortex ( Figure 1D , left spectrogram ) . Sleep in the youngest human subjects recorded was also discontinuous , and contained delta brushes , an oscillatory pattern that develops by 26 weeks post-gestation and disappears around 40\\u201344 weeks ( Khazipov and Luhmann , 2006;\\u00a0Figure 1E , left spectrogram ) . Delta brushes share morphological and frequency characteristics with spindle bursts . Sleep patterns changed profoundly with maturation in both species . More mature mouse pups exhibited nearly continuous patterns during sleep , an absence of spindle bursts , and the appearance of sleep spindles , which are well-characterized oscillations of NREM sleep ( Figure 1D , lower trace , right spectrogram ) . Human sleep was similarly continuous , and sleep spindles appear during NREM between 1\\u00a0and\\u00a02 months after term in infants ( Figure 1E , lower trace , right spectrogram ) . This mature sleep in rodents and humans is predominantly governed by internally generated dynamics ( Levenstein et al . , 2019 ) . In contrast , self-generated or evoked movements during sleep are closely associated with local field potential ( LFP ) activity in somatosensory cortex of immature rodents and humans ( Khazipov et al . , 2004; Dooley et al . , 2020 ) . We confirmed that the association between muscular twitches and changes in LFP power ( Figure 1F; n = 6716 twitches from n = 52 pups , sampled randomly to ensure uniform group size; Kruskal\\u2013Wallis ANOVA , chi-square = 12 . 93 , p=0 . 0048 ) or detectable spindle band oscillations ( Figure 1G; cross-correlation of twitches and spindle band oscillations exceeded 95% confidence interval , black dashed lines , only in pups aged P5\\u20137 ) decreased as the pups matured . These marked similarities in the electrophysiological features of neural activity during sleep across development suggest the possibility of evolutionarily conserved mechanisms of neural network maturation . To investigate how this maturation occurs , we examined the macrostructure of sleep LFP with fine temporal precision over early pup development ( P5\\u201314 ) . Mature sleep can be classified into distinct stages ( NREM and REM ) based on electrophysiological criteria , but these indicators do not reliably differentiate the immature analogues of these states ( quiet and active sleep , respectively ) until after P11 ( Seelke and Blumberg , 2008 ) . Therefore , we focused on periods of behavioral and EMG quiescence that lasted for longer than 10 s ( Figure 1\\u2014figure supplement 5 ) , with the goal of preferentially analyzing the transition of quiet sleep ( which contains spindle bursts ) into NREM sleep ( which contains sleep spindles ) . Unexpectedly , recordings across development revealed an epoch of relative neural quiescence at the beginning of the second postnatal week ( Figure 2A ) that corresponded to decreased spectral power and absence of prominent activity at physiological frequencies ( Figure 2B , Figure 2\\u2014figure supplement 1 ) . We hypothesized that this quiescence could be related to a transient decrease in the continuity of the neural signal or the power of the activity that was present . Continuity of neural activity was quantified by determining the duration of neural activity above the wideband noise floor . Although continuity increased markedly from the youngest to most mature animals , we found a decrease in the duration of continuous neural activity that corresponded to the identified period of spectral quiescence ( Figure 2C ) . When we extracted the average wideband power of the continuous activity that was present at each developmental day , there was also a transient nadir at this timepoint ( Figure 2D ) . Because we aimed to characterize the trajectory of the electrophysiological features over time without making a priori assumptions about the nature of the temporal relationship or the rate of change , we employed a modeling approach that considered age as a continuous variable . This approach avoided arbitrary grouping of data from different aged pups without sacrificing statistical power and enabled characterization of the developmental profile beyond testing at individual timepoints . Data were fit with linear and polynomial regression models , and model performance was evaluated using leave-one-out cross-validation ( LOOCV ) to avoid overfitting ( Figure 2\\u2014figure supplement 2 ) . Polynomial regression provided the best fit for continuity and power data as quantified by the mean squared error ( MSE; Figure 2\\u2014figure supplement 2B ) . In addition , linear regression was insufficient to model these parameters because linear fit residuals systematically deviated from zero and consistently overestimated observed values at the beginning of the second postnatal week ( Figure 2\\u2014figure supplement 2C ) . The best fit models exhibited local minima , and to estimate the age at which these nadirs occurred , we employed a bootstrapping method . Bootstrapping data within each developmental day localized probability of nadir in continuity and power at the beginning of the second postnatal week with likelihood far exceeding the distribution obtained by bootstrapping across days ( Figure 2C , D inset , Figure 2\\u2014figure supplement 2D\\u2013F , Figure 2\\u2014figure supplement 3 ) . We tested the possibility that the quiescence could be related to a shift in proportion of time spent in each sleep state , a decrease in self-generated sleep movements , or differential recovery from anesthesia across ages . In keeping with previous observations , the proportion of active relative to quiet sleep decreased gradually with maturation ( Figure 2\\u2014figure supplement 4A ) and we also found a trend toward fewer self-generated sleep movements in the more mature animals , though this shift was not statistically significant ( Figure 2\\u2014figure supplement 4B ) . Recovery from anesthesia was determined individually for each pup , and there was no difference in post-anesthesia stability of oscillation occurrence rate or neural spiking across pups ( Figure 1\\u2014figure supplement 2 ) . The absence of a marked shift in any of these features during the observed quiescent period makes them unlikely drivers of the phenomenon . Therefore , sleep LFP macrostructure expresses a nonlinear pattern of development that is characterized by a discrete period of relative neural quiescence occurring at the beginning of the second postnatal week . Rate and synchrony of neural spiking strongly contribute to expression of LFP patterns . We hypothesized that changes in neural spiking patterns could accompany the transient quiescent period . Spiking activity was detected in recordings obtained using implantable silicon probes from two non-contiguous sites ( >200 \\u00b5m vertical separation ) within the deep layers of cortex ( IV\\u2013VI ) and one site within the superficial layers of cortex ( I\\u2013III ) per pup recording ( n = 38 pups ) . We first calculated rates of neural spiking only during epochs of above-threshold LFP activity to eliminate effects related to LFP discontinuity . We found that neural spiking increased nonlinearly over the course of development , again with a significant nadir at the beginning of the second postnatal week as quantified using a bootstrapping method ( Figure 3A , Figure 3\\u2014figure supplement 1 ) . These data suggest that lowered intracortical synaptic drive could contribute to the nadir in LFP patterns at the beginning of the second postnatal week . Consistent spiking within neuronal integration time is critical for plasticity processes ( Mainen and Sejnowski , 1995 ) . To estimate this property , we calculated the inter-spike\\u00a0interval ( ISI ) between each unique action potential . ISI duration decreased nonlinearly across development , with the longest ISI duration occurring in pups at the beginning of the second postnatal week ( Figure 3B , Figure 3\\u2014figure supplement 2 ) . Population neural spiking at fine time scales is thus both transiently decreased and desynchronized at this time in both superficial and deep cortical layers , resulting in a brief period of relative quiescence prior to the progressive increase in spiking rate and synchrony that accompany subsequent maturation . During mature NREM sleep , there is alternation between periods of neural spiking and hyperpolarization that serves to synchronize neural populations and is implicated in memory consolidation ( Ji and Wilson , 2007; Steriade et al . , 1993 ) . We examined population spiking dynamics in the developing cortex by computing the autocorrelation of spiking activity , which permits quantification of neural synchrony over durations relevant to this slow oscillation ( Figure 3C ) . Significant short time scale ( <50 ms ) interactions ( above the 95% confidence interval ) robustly emerged only after P9 and were prominent in the most mature pups ( Figure 3D , left; n = 38 pups , rank-sum z = \\u22124 . 2 , p=2 . 5\\u00d710\\u22125 ) . All age groups demonstrated significant suppression of spiking activity within the first 500 ms , becoming more marked as development progressed ( Figure 3D , right; n = 38 pups , rank-sum z = 3 . 0 , p=0 . 002 ) . These data implicate that the transient quiescent period demarcates a transition in population spiking that initiates development of a cycle of longer scale network synchronization characteristic of the dynamics observed during mature sleep ( Steriade et al . , 1993 ) . We then sought to understand the functional relevance of this quiescent transition period to network operations . Key LFP properties ( spatial extent , waveform , and power ) can provide an index of the underlying neural population activity and its computational purpose \\u2013 the distance over which neurons can be concurrently recruited , as well as the strength and temporal patterning of this activity ( Lowet et al . , 2015; Str\\u00fcber et al . , 2017; Cole and Voytek , 2017; Stark et al . , 2014 ) . We detected spindle band oscillations , which were present in pups of all ages , to facilitate comparisons . We quantified spatial extent using the two-dimensional organization of the NeuroGrid to determine the proportion of electrodes expressing co-occurring spindle band oscillations with a reference electrode located in primary somatosensory cortex ( Figure 4A;\\u00a0Dahal et al . , 2019 ) . Waveform was characterized using an asymmetry index ( Figure 4B ) and power was extracted from the filtered Hilbert envelope ( Figure 4C ) . Spindle band oscillations in the most immature pups were high power and had a substantially asymmetric waveform , but were expressed over a restricted cortical area ( ~480 \\u00b5m diameter ) . In contrast , spindle band oscillations in the most mature pups were lower power and more symmetric , but could variably be detected over a larger spatial extent ( ~1 . 4 mm diameter ) . Given these observed differences in oscillatory parameters , we asked to what extent they could be used to classify the developmental stage at which the oscillations occurred . A coarse tree model based on spatial extent and waveform asymmetry of manually identified oscillations was effectively trained to classify individual spindle band oscillations as originating from immature ( P5\\u20137 ) or mature ( P10\\u201314 ) pups with 92 . 4% accuracy ( Figure 4D ) , suggesting that these parameters capture defining features of the oscillations at the ends of the developmental spectrum . Spindle band oscillations that occurred during the transient quiescent period at the beginning of the second postnatal week were not consistently classified into either group , instead displaying a unique combination of oscillatory parameters: large spatial extent , low power , and intermediate waveform asymmetry ( Figure 4A\\u2013C , yellow bars ) . When examined in a combinatorial fashion , a significant inflection point in oscillation properties emerged between P8\\u00a0and\\u00a09 ( Figure 4E ) . Polynomial regression provided the best fit for these data ( Figure 4\\u2014figure supplement 1 ) . Bootstrapping data within each developmental day confirmed localization of a nadir at the beginning of the second postnatal week with likelihood far exceeding the distribution obtained by bootstrapping across days ( Figure 4F ) . These data indicate that oscillations are transformed at the beginning of the second postnatal week , potentially reflecting a transition in neural network function from consistently highly localized cortical processing to capacity for broader , tuned recruitment of neural activity . Oscillations in the adult brain precisely organize neural activity across spatial and temporal scales . A key marker for this organization is cross-frequency coupling , which is important for regulating strength of inputs from different afferent pathways ( Schomburg et al . , 2014 ) and directing flow of information processing ( Buschman and Miller , 2007 ) . The marked shift in oscillatory properties that we observed at the beginning of the second postnatal week suggested that this organizational capacity may also be developmentally regulated . We first investigated the relationship of spindle band activity with other frequencies by detecting epochs of sustained increase in spindle band power and calculating measures of amplitude cross-frequency coupling . Spindle band epochs in the most immature animals ( P5\\u20137 ) demonstrated high comodulation across a broad , non-specific range of frequencies , in contrast to frequency-specific comodulation expressed in mature animals ( P13\\u201314; Figure 5A ) . Specific , discrete frequency bands that demonstrated significant temporal coupling with spindle band oscillations emerged only after P8\\u20139 . Discrete coupling with the gamma band ( 45\\u201360 Hz ) first occurred at P10\\u201312 , with an additional high gamma band ( 60\\u201390 Hz ) arising at P13\\u201314 ( Figure 5B; n = 1020 spindle band oscillations from 51 pups; area under the\\u00a0curve [AUC] for 45\\u201360 Hz chi-square = 10 . 13 , p=0 . 02; AUC for 60\\u201390 Hz chi-square = 24 . 29 , p<0 . 001 ) . Taken together , these results suggest that immature networks lack the capacity to couple discrete frequency bands , whereas after the beginning of the second postnatal week such capacity emerges , potentially signaling a developmental shift in the function of oscillations . Furthermore , oscillations facilitate neural communication by biasing spike occurrence to particular phases of the oscillatory waveform . Spindle band oscillations recruited neural spiking above baseline rates to a variable extent across development , with the lowest amount of recruitment occurring at the beginning of the second postnatal week ( Figure 5C , Figure 5\\u2014figure supplement 1 ) . This trajectory significantly deviated from linearity with a discrete nadir confirmed by bootstrapping during this epoch ( Figure 5\\u2014figure supplement 1A ) . P8 and P9 also exhibited the lowest proportion of recording sessions with significant phase-locking ( kappa > 0 . 1 and alpha < 0 . 05 ) , and consistency of phase-locking abruptly increased at P10 ( Figure 5D , E; n = 38 pups implanted with silicon probes ) . These results reinforce the notion of enhanced information processing capacity in cortical networks after a rapid transitional period at the beginning of the second postnatal week . Thus , multiple LFP and neural spiking measures followed a similar highly nonlinear developmental trajectory marked by a key transition at the beginning of the second postnatal week in mouse pups . We next aimed to investigate whether this pattern of neural maturation was conserved across species . To accomplish this goal , we analyzed clinically acquired continuous EEG data from 54 human subjects ranging from 36 to 69 weeks post-gestation . EEG recordings were included for analysis only if they were reported as normal by the reading epileptologists and the subject had no known underlying neurological or genetic condition . The majority of subjects received a discharge diagnosis of normal movements/behaviors ( e . g . , sleep myoclonus ) or brief resolved unexplained events ( BRUE;\\u00a0Figure 6\\u2014figure supplement 1 ) . We analyzed epochs of clinically determined quiet/NREM sleep from each subject using data derived from electrodes overlying human somatosensory cortex ( parietal or central electrodes; no differences were found between these localizations; Figure 6\\u2014figure supplement 1D ) . Raw traces and normalized spectral analysis suggested an epoch of transient neural quiescence between 42 and 47 weeks ( Figure 6A , B ) , akin to what we observed at the beginning of the second postnatal week in mice . To quantify these observations , we again used regression fitting to determine whether a nonlinear trajectory was present and bootstrapping of data within and between ages to identify any significant nadirs in neural activity patterns . Wideband power of oscillatory activity followed a highly nonlinear trajectory , with a significant nadir between 42\\u00a0and\\u00a047 weeks ( Figure 6C , Figure 6\\u2014figure supplement 2A ) . Power spectra revealed a lack of prominent frequency peaks during this epoch , as well as an overall steeper decay in power with increasing frequency ( corresponding to a larger exponent for the modeled power law; Figure 6\\u2014figure supplement 3A\\u2013C ) . Quantification of this aperiodic component of the power spectrum across ages similarly revealed a transient increase around the time of nadir in wideband power ( Figure 6\\u2014figure supplement 3D , E ) . As expected from previous work ( Torres and Anderson , 1985 ) , duration of oscillatory epochs generally increased over development , but we did observe a period of relative stationarity coinciding with the decrease in wideband power ( Figure 6D , Figure 6\\u2014figure supplement 2B ) . To assess properties of oscillatory activity in these subjects , we detected spindle band oscillations . Such oscillations were present in all ages , consistent with delta brushes in the youngest subjects and sleep spindles in the older subjects ( Figure 6E ) . We calculated spatial extent of these oscillations as the co-occurrence of spindle band activity across EEG electrodes and found that this activity shifted from highly localized to broader cortical expression around an inflection point between 42\\u00a0and\\u00a047 weeks ( Figure 6F , Figure 6\\u2014figure supplement 2C ) . We also found evidence for significant cross-frequency coupling of higher frequencies with the spindle band emerging after 47 weeks post-gestation ( Figure 6G , H; n = 702 spindle band oscillations from 13 subjects; Kruskal\\u2013Wallis chi-square for AUC = 138 . 4; p<0 . 001 ) . Taken together , these data support the existence of a conserved transient quiescent period in development that bridges a dramatic shift in functional properties of neural networks . \\n We demonstrate that maturation of cortical network dynamics is preceded by a discrete , relatively quiescent , evolutionarily conserved transition period during early postnatal development . During this period , oscillations and neural spiking display a nadir that signals the shift from local , loosely correlated , putatively sensory-driven patterns to internally organized , spatially distributed , and temporally precise activity . Such a developmental inflection point may reflect a rapid and confluent engagement of mechanisms that enable mature computational processes within cortical networks . Self-organization in complex systems is often mediated by processes that can facilitate sudden transitions in system behavior . Our results provide evidence for such a transition in developing cortex of rodents and humans . Because rodent neurodevelopment is temporally compressed relative to humans , an inflection epoch that lasts 1\\u20132 days in mouse pups could conceivably extend over weeks in humans ( Clancy et al . , 2001 ) . Prior to this inflection point , oscillatory activity was intermittent with non-uniform cycle-to-cycle waveforms and high broadband frequency coherence . Oscillatory activities after the transition became nearly uninterrupted , exhibiting coupling at discrete frequency bands and increased waveform symmetry . Based on these similarities in oscillatory patterns , we infer that the neural spiking features we observe through invasive recordings in mouse pups could also extend to normally developing human infants , where such recordings are not possible . Neural spiking was more efficiently recruited to oscillatory activity with consistent , strong phase-locking , and higher temporal precision . These features are characteristic of adult neural networks engaged in information processing ( Masquelier et al . , 2009 ) . Thus , a transient period of developmental perturbation may be an evolutionarily conserved indicator of a substantial shift in cortical network operations . Our results suggest that the cortical network dynamics shift to permit temporally precise neural spiking after the transition period . Significantly synchronized firing within 10\\u201350 ms intervals was absent prior to this period and robustly emerged afterward . This result is consistent with observed increase in pair-wise correlations of spike activity in visual cortex over development ( Colonnese et al . , 2017 ) . Although recruitment of neural spiking during epochs of spindle burst oscillations in immature rodents likely co-activates neurons that share similar afferent inputs ( Khazipov et al . , 2004; Colonnese et al . , 2010; Khazipov et al . , 2013; Winnubst et al . , 2015 ) , more precise coordination is required for establishment of flexible , defined neural assemblies ( Dragoi and Buzs\\u00e1ki , 2006 ) , and the capacity of circuits to express sequential activation of these assemblies is developmentally regulated ( Farooq and Dragoi , 2019 ) . We found that the neural population activity demonstrated correlated suppression over a progressively shorter duration ( from approximately 500 to 200 ms ) with maturation . This process is in keeping with the progressively decorrelated activity patterns observed when analysis is performed on the time\\u00a0scale of calcium indicators ( Golshani et al . , 2009; Rochefort et al . , 2009 ) . When oscillations occur in discrete frequency bands , the possibility for complex comodulation arises . This cross-frequency coupling has been linked to information processing in the mature brain , though the precise anatomical substrates and physiological mechanisms that enable its expression are often unknown ( Tort et al . , 2009; Fujisawa and Buzs\\u00e1ki , 2011; Bosman et al . , 2012 ) . We demonstrated that prior to the transient quiescent period the dominant cortical oscillations ( centered on spindle band ) exhibit high comodulation extending from 1 to 100 Hz . During this period , broadband comodulation abruptly decreased and was subsequently replaced by discrete gamma oscillations . Therefore , the oscillatory infrastructure to support cross-frequency coupling arises concomitantly with fine time-scale neural spiking synchronization and robust phase-locking of these spikes to LFP , suggesting an emergence of adult-like information processing capacity at this developmental timepoint . Despite sharing a peak frequency , immature and mature spindle band oscillations are highly differentiable on the joint basis of waveform , power , and spatial extent . Correspondingly , the cellular mechanisms that permit expression of these oscillations , as well as their functional purpose , are different . Comparison of oscillatory features can shed light on the functional capacity of the network . We found that although of high amplitude , spindle band frequency oscillations in the most immature mouse pups were highly and uniformly confined to localized regions of cortex ( 200\\u2013500 \\u00b5m ) roughly corresponding to cortical columns ( Yang et al . , 2013; Feldmeyer et al . , 2013 ) . Similarly , human delta brushes exhibited restricted localization even given the limited spatial resolution of clinical EEG . During the transition period , spindle band oscillations in mouse pups had higher spatial extent , but markedly reduced amplitude , perhaps facilitated by increased intracortical synaptogenesis and maturation of corticothalamic afferents ( Naskar et al . , 2019 ) . The transition period in humans was marked by transiently decreased spatial extent of spindle band oscillations , which we hypothesize is related to attenuation of low-power activity by the intervening tissue between cortical surface and scalp electrode . We observed that mature spindle band activity across species was characterized by increased and customizable spatial extent . Thus , the end of the transient quiescent period marks capacity for oscillatory frequencies that were previously confined to local processing units to extend into broader cortical areas with the potential to establish intercortical communication . Our study was limited to somatosensory cortex , and the existence of a transient quiescent epoch outside of this brain region is unknown . Similar to our results in the spindle band , early gamma oscillations disappear around the beginning to the second postnatal week , to be later replaced with gamma rhythms characteristic of adult animals ( Minlebaev et al . , 2011 ) . In visual cortex , a transient nadir in continuity was not identified , but continuity did rise sharply after P8 with a nonlinear trajectory ( Shen and Colonnese , 2016 ) . Furthermore , the least synchronous neural spiking activity in visual cortex occurred at the beginning of the second postnatal week , potentially in keeping with our decrease in temporal precision of spiking at this time ( Colonnese et al . , 2017 ) . Higher order cortices , which do not directly receive sensory thalamic inputs , may express a different developmental trajectory , in keeping with their distinctive role in information processing , though oscillatory expression during this same epoch has been identified as critical for intact adult function ( Bitzenhofer et al . , 2015; Bitzenhofer et al . , 2021; Bitzenhofer et al . , 2020 ) . Therefore , multiple lines of evidence support an abrupt change in cortical network function at the beginning of the second postnatal week . It is possible that our use of surface arrays , which sample summated LFP\\u00a0activity from the undisturbed superficial cortical layers ( I\\u2013III ) , highlights the transient quiescent state compared to penetrating probes that disrupt the cortical surface upon implantation . Furthermore , our conducting polymer-based electrodes have lower impedance than the silicon probes used in most neonatal rodent studies ( Khodagholy et al . , 2015 ) , potentially increasing the sensitivity to changes in oscillatory power and continuity . Given the multitude of anatomic and functional changes that occur during the early postnatal epoch , it is likely that the shift in network operating mode we observe is mechanistically multifactorial . A potential driver for this state is maturation of functional cortical inhibition . Functional inhibitory connections regulate physiological oscillations in mature neural networks enabling submillisecond precision of spike timing and facilitating inter-regional synchronization ( Buzs\\u00e1ki and Chrobak , 1995; Cobb et al . , 1995; Pouille and Scanziani , 2001 ) . In contrast , interneurons do not substantially pace immature oscillations during the first postnatal week , though they can be recruited to and contribute to spatial properties of this activity ( Khazipov et al . , 2004; Minlebaev et al . , 2011 ) . Functional feedforward inhibition rapidly matures around the beginning of the second postnatal week in mice , resulting in emergence of adult-like hyperpolarizing synaptic potentials and precisely timed action potentials in glutamatergic neurons within the microcircuit ( Daw et al . , 2007 ) . This timepoint , derived from in vitro data , temporally overlaps with the onset of adult-like neural network activity we observe in vivo . In the human brain , when mature functional inhibition arises is not known , but ongoing migration of cells that differentiate into interneurons in the months after birth ( Paredes et al . , 2016 ) suggests the possibility of a postnatal transition to such network activity . The aperiodic component of the power spectrum has been linked to excitatory-inhibitory balance ( Gao et al . , 2017 ) , and our finding that this value transiently increases during the quiescent period could support a temporary increase in inhibition in the human brain . Such a mechanism could underlie the known postnatal disappearance of the EEG bursts characteristics of trace alternant ( Ellingson and Peters , 1980 ) , resulting in the observed low-voltage state with lack of organized oscillatory activity prior to emergence of mature NREM sleep patterns . Whether a period of quiescence is required for physiological transition to mature cortical processing remains unclear . Manipulations that could modulate its expression , such as alteration of the neuromodulatory milieu , activation/suppression of specific cell-type activity , or modulation of the sensory environment ( Smith et al . , 2018 ) , could address this question . Forging linkages to activity patterns observed using calcium imaging ( Wilson et al . , 2018 ) and enabling high spatiotemporal resolution chronic in vivo electrophysiology will also be critical . In humans , neuronal dynamics suggestive of mature information processing mechanisms are established by 6 months of age ( Jannesari et al . , 2020 ) , and changes in sleep patterns have been identified as markers for brain connectivity from toddlers to adolescents ( Kurth et al . , 2017 ) . Extending such measures to earlier timepoints could help to clarify functional ramifications of the quiescent period in humans . Furthermore , the manifestation of a developmental epoch of relative quiescence in electrophysiological activity suggests the possibility for use as a network-level biomarker that could be altered in disorders of neurodevelopment , from mouse models to human subjects . Neural processes in the adult brain are characterized by a specific constellation of network dynamics that are necessary for execution of complex behaviors and cognition ( Tolner et al . , 2012 ) . Our results suggest that the capacity for these processes simultaneously emerges abruptly after a distinct transition period characterized by relative neural quiescence . This transient quiescent period may be a critical stage that indicates a shift in the neural network operating mode from a predominantly external stimulus-driven state to one that facilitates the internal representations associated with planning and executive function . \\n PEDOT:PSS ( Clevios PH1000 ) was purchased from Heraeus . Ethylene glycol , ( 3-glycidyloxypropyl ) trimethoxysilane ( GOPS ) , 4-dodecyl benzene sulfonic acid ( DBSA ) , and\\u00a03- ( trimethoxysilyl ) propyl methacrylate ( A-174 silane ) were purchased from Sigma-Aldrich . Micro-90 concentrated cleaning solution was purchased from Special Coating Services . AZnLOF2020 ( negative photoresist ) , AZ9260 ( positive photoresist ) , and\\u00a0AZ 400K and AZ 300MIF ( metal ion free ) developers were acquired from MicroChemicals , Merck . To create the PEDOT:PSS films , a mixture of aqueous dispersion ( Clevios PH1000 ) and ethylene glycol was prepared and mixed with GOPS ( 1\\u00a0wt% ) and DBSA ( 0 . 1 wt% ) . The fabrication process involved deposition and patterning of parylene C , evaporation of Au for electrodes and interconnects , and PEDOT:PSS films coating . Parylene C ( 1 . 2-\\u03bcm-thick ) was deposited on quartz wafers ( 100 mm outer diameter [O . D . ] , thickness of 1 mm ) using an SCS Labcoater 2 . For the metal lift-off , AZnLOF2020-negative photoresist was spin-coated at 3000 r . p . m . on the substrate , baked at 105\\u00b0C for 90 s , exposed to ultraviolet light using a Suss MA6 Mask Aligner , and developed with AZ 300MIF developer . Metallic layers ( 10 nm Ti , 150 nm Au ) were deposited using an e-beam metal evaporator ( Angstrom EvoVac Multi-Process ) and patterned by soaking the substrate in a bath of resist remover . A second layer of parylene C ( insulation layer ) was deposited to a thickness of 1 . 2 \\u03bcm and its adhesion to the bottom layer was enhanced by the addition of 3- ( trimethoxysilyl ) propyl methacrylate ( A-174 silane ) during chemical vapor deposition . An anti-adhesion agent ( 5 wt% Micro-90 diluted in deionized water ) was deposited on the substrate followed by an additional sacrificial layer of parylene C that was subsequently used for the peel-off process . The stacked layers were patterned with a layer of AZ9260-positive photoresist and dry etched with a plasma-reactive ion-etching process ( Oxford Plasmalab 80; 180 W , 50 sccm O2 , and 2 sccm SF6 ) to shape electrode area and electrical contact pads . Specifically , AZ9260 was spin-coated at 5000 r . p . m . , baked at 115\\u00b0C for 90 s , exposed using a Suss MA6 Mask Aligner , and developed with AZ400K developer ( 1:4 with deionized water ) . To obtain clean etching of large areas , the electrical contact pads were guarded by an extra layer of AZnLOF2020 ( 3000 r . p . m . ) between the metal layer and the silane-treated parylene . Each pad was etched along its perimeter , and the parylene deposited on top of the photoresist was removed once immersed in acetone bath . Finally , PEDOT:PSS was spin-coated on top of the electrodes and patterned by peeling off the last parylene layer . The probes had 119 electrodes regularly spaced on a diagonal square centered lattice with pitch of 77\\u2013177 \\u00b5m and regularly interspersed perforations were created . Total device thickness was 4 \\u00b5m . All animal experiments were approved by the Institutional Animal Care and Use Committee at Columbia University Irving Medical Center . Data from 108 Swiss-Webster mouse pups ( 3\\u20139 g , 5\\u201314 days of age ) that underwent intracranial implantation was used for neurophysiological analysis . Pups were kept on a regular 12 hr\\u201312 hr light-dark cycle and housed with the mother before surgery . No prior experimentation had been performed on these mice . Anesthesia was induced by hypothermia in ice for P5\\u20136 pups and with 3% isoflurane for pups aged P7\\u201314 . Pups were then comfortably placed on a customized platform for head fixation , surgery , and recording . Acclimatization to head fixation was not performed as it has been found to be unnecessary for neonatal rodents; careful positioning and \\u2018swaddling\\u2019 was performed to maximize pup comfort ( Blumberg et al . , 2015; Corner and Kwee , 1976 ) . All animals were maintained under anesthesia with 0 . 75\\u20131 . 5% isoflurane during surgery . Time under anesthesia was minimized to facilitate postoperative recovery . Pre- and postoperative systemic and local analgesics were employed , including subcutaneous bupivacaine , systemic ketoprofen , and topical lidocaine . These analgesics were selected to attain appropriate pain control while minimizing effects on neurophysiological activity in the postoperative period ( Blumberg et al . , 2015; Foley et al . , 2019 ) . Electrodes fabricated from fine gauge wire were inserted into the subcutaneous tissue of the neck and abdomen . These electrodes continuously registered electrocardiography ( ECG ) and electromyography ( EMG ) signals throughout the surgical and recording period , and were used to monitor recovery from anesthesia . A custom metal plate was attached to the skull to provide head fixation . For NeuroGrid implantations , a craniotomy was centered over primary somatosensory cortex and the array was placed over the dura . For silicon probe implantations , H32 NeuroNexus 32-site single-shank probes were used . A small cranial window over primary somatosensory cortex ( approximately AP \\u22121 . 5 mm , ML 2 mm; adjusted proportionally based on pup age ) was opened , and the probe was stereotactically advanced until all electrodes were inserted below the pial surface ( 900 \\u00b5m ) . Postoperatively , the pup was comfortably positioned , provided with familiar olfactory cues , and placed in a custom recording box with temperature and humidity control . Temperature was regulated by a heater equipped with feedback control to ensure stable , physiological body temperature of the pups throughout the recording period . Neurophysiological signal acquisition was performed in the custom recording box , which was grounded to shield ambient electrical noise . Concurrent video monitoring was performed using a camera mounted inside the box . Pups recovered from anesthesia , as determined by recovery criteria including increase of heart rate to plateau and onset of sleep-wake cycling , for a minimum of 30 min before neurophysiological recordings were used for analysis . The recovery period was prolonged until recovery criteria were met , and all pups used for experimentation recovered within 60 min postoperatively . Neurophysiological signals were amplified , digitized continuously at 20 kHz using a local preamplifier ( RHD2000 Intan Technology ) , and stored for offline analysis with 16-bit format . Spontaneous activity was acquired for 1\\u20132 hr prior to euthanasia and perfusion . At the end of experimentation , pups were euthanized with an overdose of pentobarbital ( 100 mg/kg ) . For the NeuroGrid experiments , the position of the array was visualized using light microscopy and the brain tissue underlying the corners of the array was stereotactically marked using a 30 gauge needle coated with chitosan compound ( Rauhala et al . , 2020 ) . Pups were perfused with phosphate-buffered saline ( PBS ) , followed by 4% paraformaldehyde . Brains were stored for 24\\u201348 hr in 4% PFA at 4\\u00b0C prior to transfer to PBS solution . Cortices were \\ufb02attened at room temperature ( RT ) after removal of the underlying midline structures and kept in 4% PFA for 24 hr ( 4\\u00b0C ) . Flattened cortices were sliced using a Leica vibratome vt1000S at 50 \\u00b5m . The slices were then incubated for 1 hr at RT in a blocking solution composed of PBS ( 0 . 01 M ) , 3% Normal Donkey Serum , and 0 . 3% Triton X-100 . Slices were transferred to a solution containing anti-Vesicular Glutamate Transporter 2 ( VGlut2 ) polyclonal guinea-pig antibody ( AB2251-I Sigma Millipore ) in a 1:2000 dilution and kept overnight at 4\\u00b0C . Subsequently , the tissue was washed and incubated with secondary antibody , Alexa Fluor 594 Af\\ufb01niPure Donkey Anti-guinea pig IgG ( H+L ) ( 706-585-148 ) from Jackson Immunoresearch at 1:1000 dilution . Lastly , the tissue was incubated with DAPI ( D9542-5MG; Sigma-Aldrich ) at 0 . 2 \\u00b5g/mL and after washing , the brain slices were mounted with Fluoromount-G ( 00-4958-02 , ThermoFisher ) . Fluorescent images were captured and tiled using a computer-assisted camera connected to an ECHO Revolve microscope . The images were adjusted for brightness and contrast and assembled into panels using ImageJ and Adobe Illustrator . The regularly spaced , hexagonal lattice of electrode spacing across the NeuroGrid enabled localization of each electrode relative to the marked corners of the array and primary sensory cortices . The spatially extensive nature of the NeuroGrid relative to pup brain size facilitated consistent anatomical targeting . Localization of silicon probe electrodes to cortical region and layers was performed using a combination of intraoperative visualization and histological reconstruction . Probes were inserted under visual guidance , with the most superficial electrode placed just below the pia . Regular , precise spacing of electrodes along the shank enabled determination of each electrode\\u2019s depth from the cortical surface . Coronal brain slices stained with DAPI and vGlut2 permitted localization of probe implantation site relative to barrel cortex . All recordings were visually inspected for quality , and channels that exhibited ongoing electrical or mechanical artifact were removed from analysis . Data was analyzed using MATLAB ( MathWorks ) and visualized using Neuroscope . Raw recordings were downsampled to 1250 Hz to obtain LFPs , and band-pass filtered between 250 Hz and 2500 Hz for analysis of neural spiking . Signals from EMG wires inserted into nuchal and ventral subcutaneous tissue were high-pass filtered at 300 Hz , rectified , and smoothed to obtain a power envelope . The mean of this high-pass filtered power was determined . Epochs consistent with high tone were identified when the signal surpassed a threshold defined by 1 . 5\\u20133 . 5 standard deviations above this mean . Wakefulness was defined as a period of sustained high tone for at least 1 . 5 s . Myoclonic twitches involving contractions of the muscles of the head , forelimbs , hindlimbs , or trunk as observed on video recording were correlated with an increase in EMG signal that surpassed the threshold for < 1 . 5 s . Muscle atonia ( consistent with sleep ) was identified when the EMG signal remained below the threshold for at least 2 s . Sleep epochs were further classified as putative active or quiet sleep . Quiet sleep was identified when muscle atonia was paired with quiescence of behavior for at least 10 consecutive seconds . Active sleep was identified by myoclonic twitches that occurred on a background of muscle atonia . Similar approaches have been shown to correlate with electrophysiological markers of active and quiet sleep , and have been used for state-dependent developmental analyses ( Seelke and Blumberg , 2008; Turek , 1999 ) . We retrospectively analyzed EEG recordings from 54 patients who underwent continuous monitoring with surface EEG as part of clinical diagnostic assessment . Analysis of these data was approved by the Institutional Review Board at Columbia University Irving Medical Center , and all data collection occurred at this institution . All data reviewed was initially obtained for clinical management purposes and informed consent was waived as per 45 CFR 46 . 116 . Each patient had EEG electrodes placed based on the internationally recognized 10\\u201320 configuration . Electrode impedance was monitored and maintained within appropriate ranges as per American Clinical Neurophysiology Society Guidelines by certified clinical EEG technologists . Patients ranging in age from birth to 8 months who had at least 4 hr of high-quality continuous EEG monitoring were identified through the clinical electrophysiology database ( Natus ) . Using the hospital admission record , patients were excluded if ( 1 ) the EEG was reported as abnormal by the reading electrophysiologist; ( 2 ) the patient had a known underlying neurological or genetic condition; ( 3 ) the patient was diagnosed with a new neurological condition during the course of the hospital admission; and ( 4 ) corrected gestational age could not be determined from information contained in the medical record . Indication for EEG in eligible patients was most commonly paroxysmal movements concerning to parents for abnormal activity , with discharge diagnosis identifying normal movements/behaviors ( e . g . , sleep myoclonus , roving eye movements during REM sleep ) or BRUE ( Figure 6\\u2014figure supplement 1 ) . Corrected gestational age , rounded to the nearest week , was used to classify patient age . If the phrase \\u2018full term\\u2019 was employed in the medical record , corrected gestational age was calculated based on birth at 40 weeks . This approach allowed a precision of 2 weeks in corrected gestational age . Recordings always included both waking and sleep epochs . Three to five representative epochs of artifact-free quiet/NREM sleep were identified for subsequent analysis by a certified clinical electrophysiologist . Data was sampled at 256\\u2013512 Hz by clinical amplifiers . Raw data ( referential ) exported from Natus was used for all analyses . If electrical noise contaminated the recording , a 60 Hz notch filter was used . P3 or P4 electrodes , overlying the parietal lobe , were selected for spectral analysis . If these electrodes were non-functional , C3 or C4 electrodes , overlying the central sulcus , were used . No differences were found when using data derived from parietal and central electrodes in subjects where all channels were functional . To visualize spectrograms , a parametric autoregressive ( order of 1 ) whitening method was first applied to the data . For rodent data ( sampled at 1250 Hz ) and human data ( sampled at 256\\u2013512 Hz ) , time-frequency decomposition was performed using analytic Gabor wavelet transform to enhance temporal resolution . Wavelet spectrograms centered at twitch times were normalized by z-scoring and averaged to obtain trigger-averaged spectrograms . Power was extracted for quantification from the magnitude of the analytic Gabor wavelet transform . Epochs of oscillatory activity were defined by presence of rectified signal amplitude above the 99th percentile of the wideband noise floor . To measure periodic and aperiodic properties of human neural data , power spectra were extracted using multi-taper method ( http:/chronux . org ) and modeled using \\u2018fitting oscillations and one over f\\u2019 ( Corner and Kwee , 1976 ) . In order to quantify the changing tendency of each feature over time , we modeled the trends with linear and polynomial regression models . To find the optimal fit that most accurately represented the overall trend without overfitting , we used LOOCV to evaluate the models . The model with the minimum MSE was considered to have the best fit . When model fit predicted existence of local extrema , we used bootstrapping to estimate the location of these extrema for the trajectory of each property we investigated . For animal data , we resampled with replacement within each day from P5 to P14 . For human data , we resampled within each 1\\u20132 weeks . Age was jittered with a window of 0 . 5 days and 0 . 5 weeks for animal and human data , respectively , to aid with fitting . We found the age corresponding to the first peak or trough of the regression line across development . This process was repeated 10 , 000 times to generate a probability distribution of location of local extrema across the developmental trajectory . The same process was repeated using resampling with replacement across all timepoints of the trajectory ( 10 , 000 iterations ) to generate a null distribution for comparison . Spindle band oscillations were detected based on wavelet-derived power and duration parameters . Intervals of oscillatory activity containing spindle band power were first detected and a ratio of normalized autoregressive wavelet ( Gabor ) -based\\u00a0PAR was then calculated to identify discrete spindle band oscillations . Oscillatory intervals were detected by comparing the spindle band power ( Pspi ) of 8\\u201325 Hz with threshold . Thresholds were determined relative to median absolute deviation ( median ( |Pspi| ) 0 . 6745 ) in the sleep intervals . Because this value varied markedly across development , thresholds were derived empirically but kept consistent across each age group . The value was decreased for silicon probe recordings in P5\\u20137 pups compared to NeuroGrid recordings to maintain consistent detection despite quantifiably different signal to noise ratio . PAR\\u00a0was then calculated based on the following equation:PAR=Pspi- ( Plow+Phigh ) Pspi+Plow+Phighwhere spindle band power ( Pspi ) was based on 8\\u201325 Hz , low band power was based on 1\\u20135 Hz ( Plow ) , and high band power was based on 30\\u201380 Hz ( Phigh ) . Spindle band events were identified when the ratio crossed \\u20130 . 1 for a minimum of 300 ms and a maximum of 5 s and had a peak greater than 0 . All detections were visually inspected for accuracy for each recording session and independently verified by an analyst blinded to detection parameters . In human EEG data , spindle band oscillations were detected across all channels as in mice , using a constant threshold relative to median absolute deviation ( median ( |Pspi| ) 0 . 6745 ) in the intervals of quiet/NREM sleep and PAR was calculated as above . For subjects aged less than 40 weeks , the low band power was based on 2\\u20136 Hz to aid with accurate detection of delta brushes . The channel with maximal oscillation power in the spindle band was selected for further analysis . Spindle band oscillations were characterized by their power , spatial extent , and spindle asymmetry ( Cole and Voytek , 2017 ) . Power was calculated by bandpass filtering each spindle band oscillation between 8\\u00a0and\\u00a025 Hz . We then calculated the rectified value of Hilbert transformed filtered signals to derive the power envelopes . The median value of each spindle band oscillation was defined as its power . Spatial extent was calculated by determining the number of channels simultaneous expressing spindle band oscillations with a manually selected channel located in primary somatosensory cortex for mice and the channel with the highest spindle band power for humans ( intersection of spindle start and end points > 300 ms and separation of spindle midpoints \\u2264 50 ms on channels being compared ) . The spatial extent was expressed as a ratio of the number of channels with simultaneous occurrence and the total number of functional channels on the NeuroGrid . Data used for these calculations had a similar proportion and placement of channels located within primary somatosensory cortex . To quantify the sharpness of each spindle band oscillation\\u2019s peak and trough , they were first broadly bandpass filtered at 5\\u201330 Hz . Rising and falling zero-crossing points were identified . Peaks were recognized as the maximum timepoint in raw data between a rising zero-crossing point and a falling zero-crossing point; troughs were recognized as the minimum timepoint in raw data between a rising zero-crossing point and a falling zero-crossing point . Sharpness of a peak was defined as the mean difference between the voltage at the oscillatory peak Vpeak and the voltage at timepoints before ( Vbefore ) and after the peak ( Vafter ) , where the timepoint corresponded to a phase shift of approximately \\u03c0 ( 8 ms ) . Trough sharpness was defined similarly:Sharppeak=Vpeak-Vbefore+ ( Vpeak-Vafter ) 2Sharptrough=Vbefore-Vtrough+ ( Vafter-Vtrough ) 2 As the absolute difference between the extrema and the surrounding timepoints increases , the extrema sharpness increases . Extrema mean sharpness ratios ( ESR ) were calculated as a metric to quantify the sharpness asymmetry of each spindle . The ESR was defined in the following manner:ESR=1Npeaks\\u2211peakssharppeak1Ntroughs\\u2211troughssharptrough Larger deviation from 1 indicated greater sharpness asymmetry of the spindle . Spectral analysis of the data was performed , and any data contaminated with electrical noise ( 60 Hz ) was eliminated from analysis to prevent spurious correlations . Comodulograms of cross-frequency coupling were computed based on a Gabor wavelet of raw data around the time of spindle band oscillation ( 4 s ) , followed by a Gaussian window convolution with duration matched to the frequency . The correlation matrix was calculated based on the magnitude of the wavelet components . Comparisons with p<0 . 05 after Bonferroni\\u2013Holm correction for multiple comparisons were considered statistically significant . Columns of the correlation matrix corresponding to spindle band ( 10\\u201320 Hz ) were extracted and summated to obtain a curve representing the magnitude of frequency coupling to this band . Each local maximum coupling value was detected along with its peak prominence ( how significant a peak is on account of its intrinsic value and location relative to other peaks ) and peak width ( frequency range with between half-prominence points ) . Peaks were thresholded based on prominence and width ( \\u0394f < 1/fc , peak prominence > 0 . 75 ) . Wide-band coupling index was calculated by ( peak value \\u00d7 peak prominence ) /peak width . AUC was calculated by integrated the summated coupling curves . Neural spiking was detected on each channel by thresholding negative peaks greater than four times the high-pass filtered noise floor during epochs of sleep . Coincident spike times across a greater distance than expected for physiological spike waveforms ( ~300 \\u00b5m ) were presumed to be non-physiological and eliminated from all channels . Electrodes were allocated based on histology and electrophysiological characteristics into zones roughly corresponding to superficial ( I\\u2013III ) and deep ( IV\\u2013VI ) cortical layers . Electrodes from two non-contiguous sites ( >200 \\u00b5m vertical separation ) within the deep cortical layers and one site from superficial cortical layers were used for subsequent analysis . Because individual neuron action potentials could potentially be detected on more than one channel , spikes occurring on different channels within the same histological zone < 2 ms apart were presumed to be duplicate detections and only the first spike was used for analysis . ISIs for the neural population were calculated as the absolute time between sorted spike times within the zone . A modified convolution method was used to determine 95% confidence intervals for population spike time autocorrelograms with a bin size of 10 ms . Peristimulus time histograms were calculated using a bin size of 50 ms with 95% confidence intervals determined from a shuffled distribution of spike times . To derive spike phase-locking to spindle band oscillations , the oscillatory epochs were first downsampled to 125 Hz and narrowly bandpass filtered at 9\\u201316 Hz . Phase was extracted using Hilbert transform . Phase bins \\u03c0/24 were used for quantification of spike phase preference . Statistical analysis was performed using a combination of open-source MATLAB toolboxes and custom MATLAB code . A modified convolution method was used to determine 95% confidence intervals for correlograms . Significance of phase-locking to spindle band oscillations was determined using alpha < 0 . 05 with a corresponding kappa value of > 0 . 1 Kuiper test ( Circular Statistics Toolbox for MATLAB ) . Null distributions were generated based on 500 instances of data shuffling and used to calculate 95% confidence intervals . Probability distributions were compared using two-sample Kolmogorov\\u2013Smirnov tests with correction for multiple comparisons . Differences between groups were calculated using non-parametric rank-sum ( Wilcoxon ) or ANOVA ( Kruskal\\u2013Wallis with Bonferroni correction ) depending on the nature of the data analyzed . When event size varied between groups , a random sample of events was selected from groups with larger event sizes to ensure group differences were not driven by degrees of freedom . Error bars represent standard error of mean . Testing was two-tailed and significance level was p<0 . 05 .\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"headings\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"keywords\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"id\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 24,\n        \"samples\": [\n          \"elife-01133-v1\",\n          \"elife-27744-v3\",\n          \"elife-69011-v2\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"biomistral_summary\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 24,\n        \"samples\": [\n          \" The authors have provided a detailed description of the generation and characterization of a mouse model of Sgol2 deficiency. They have also used this model to investigate the role of Sgol2 in the regulation of sister chromatid cohesion, chromosome segregation, and the spindle assembly checkpoint. The authors have demonstrated that Sgol2 is essential for sister chromatid cohesion and that its deficiency leads to precocious separation of sister chromatids. They have also shown that Sgol2 is required for the proper bi-orientation of chromosomes on the mitotic spindle and that its deficiency leads to the formation of lagging chromosomes. The authors have further demonstrated that Sgol2 is required for the proper function of the spindle assembly checkpoint and that its deficiency leads to the premature onset of anaphase. The authors have also used this model to investigate the role of Sgol2 in the regulation of Aurora B kinase activity at kinetochores. They have demonstrated that Sgol2 is required for the proper regulation of Aurora B kinase activity at kinetochores and that its deficiency leads to increased Aurora B kinase activity. The authors have also shown that Sgol2 is required for the proper recruitment of MCAK to kinetochores and that its deficiency leads to decreased MCAK recruitment to kinetochores. The authors have further demonstrated that Sgol2 is required for the proper regulation of the phosphorylation of MCAK at serine 92 and that its deficiency leads to decreased phosphorylation of MCAK at serine 92. The authors have also used this model to investigate the role of Sgol2 in the regulation of the phosphorylation of Knl1 at serine 24. They have demonstrated that Sgol2 is required for the proper regulation of Knl1 phosphorylation at serine 24 and that its deficiency leads to decreased phosphorylation of Knl1 at serine 24. The authors have also shown that Sgol2 is required for the proper regulation of the phosphorylation of Dsn1 at serine 100 and that its deficiency leads to decreased phosphorylation of Dsn1 at serine 100. The authors have also used this model to investigate the role of Sgol2 in the regulation of the phosphorylation of Aurora A/B/C at threonine 288, threonine 232, and threonine 198. They have demonstrated that Sgol2 is required for the proper regulation of Aurora A/B/C phosphorylation at threonine 288, threonine 232, and threonine 198 and that its deficiency leads to decreased phosphorylation of Aurora A/B/C at threonine 288, threonine 232, and threonine 198. The authors have also used this model to investigate the role of Sgol2 in the regulation of the phosphorylation of MCAK at threonine 206. They have demonstrated that Sgol2 is required for the proper regulation of MCAK phosphorylation at threonine 206 and that its deficiency leads to decreased phosphorylation of MCAK at threonine 206. The authors have also used this model to investigate the role of Sgol2 in the regulation of the phosphorylation of Mad1 at serine 108. They have demonstrated that Sgol2 is required for the proper regulation of Mad1 phosphorylation at serine 108 and that its deficiency leads to decreased phosphorylation of Mad1 at serine 108. The authors have also used this model to investigate the role of Sgol2 in the regulation of the phosphorylation of Mad2 at serine 126. They have demonstrated that Sgol2 is required for the proper regulation of Mad2 phosphorylation at serine 126 and that its deficiency leads to decreased phosphorylation of Mad2 at serine 126. The authors have also used this model to investigate the role of Sgol2 in the regulation of the phosphorylation of BubR1 at serine 134. They have demonstrated that Sgol2 is required for the proper regulation of BubR1 phosphorylation at serine 134 and that its deficiency leads to\",\n          \" The authors describe a method for labeling and imaging of AMPARs and NMDARs in live neurons using QDs. The method involves the use of a novel engineered monomeric streptavidin (mSA) that has a slower dissociation rate from biotin than the native protein. The authors also describe the use of a new fluorescent protein, mGeos, that can be photoactivated and used as a marker for AMPARs. The authors demonstrate the use of the method to image AMPARs and NMDARs in live neurons, and to track the movement of AMPARs in response to synaptic activity. The method is expected to be useful for studying the dynamics of AMPARs and NMDARs in live neurons, and for investigating the role of these receptors in synaptic plasticity and learning.\",\n          \" The authors present a novel approach to study the development of neural circuits in the neocortex of mice and humans. They use a combination of electrophysiological and histological techniques to identify and characterize the development of spindle oscillations in the neocortex. They find that spindle oscillations are present in the neocortex of mice as early as P5 and in humans as early as 2 weeks after birth. They also find that spindle oscillations are more frequent and have a higher amplitude in the neocortex of mice compared to humans. The authors also find that spindle oscillations are more frequent in the superficial layers of the neocortex compared to the deep layers. They also find that spindle oscillations are more frequent in the frontal and parietal regions of the neocortex compared to the occipital and temporal regions. The authors also find that spindle oscillations are more frequent during active sleep compared to quiet sleep. They also find that spindle oscillations are more frequent during REM sleep compared to NREM sleep. The authors also find that spindle oscillations are more frequent during the first half of the day compared to the second half of the day. They also find that spindle oscillations are more frequent during the first half of the week compared to the second half of the week. The authors also find that spindle oscillations are more frequent during the first half of the month compared to the second half of the month. The authors also find that spindle oscillations are more frequent during the first half of the year compared to the second half of the year. The authors also find that spindle oscillations are more frequent during the first half of the decade compared to the second half of the decade. The authors also find that spindle oscillations are more frequent during the first half of the century compared to the second half of the century. The authors also find that spindle oscillations are more frequent during the first half of the millennium compared to the second half of the millennium. The authors also find that spindle oscillations are more frequent during the first half of the epoch compared to the second half of the epoch. The authors also find that spindle oscillations are more frequent during the first half of the day compared to the second half of the day. The authors also find that spindle oscillations are more frequent during the first half of the week compared to the second half of the week. The authors also find that spindle oscillations are more frequent during the first half of the month compared to the second half of the month. The authors also find that spindle oscillations are more frequent during the first half of the year compared to the second half of the year. The authors also find that spindle oscillations are more frequent during the first half of the decade compared to the second half of the decade. The authors also find that spindle oscillations are more frequent during the first half of the century compared to the second half of the century. The authors also find that spindle oscillations are more frequent during the first half of the millennium compared to the second half of the millennium. The authors also find that spindle oscillations are more frequent during the first half of the epoch compared to the second half of the epoch. The authors also find that spindle oscillations are more frequent during the first half of the day compared to the second half of the day. The authors also find that spindle oscillations are more frequent during the first half of the week compared to the second half of the week. The authors also find that spindle oscillations are more frequent during the first half of the month compared to the second half of the month. The authors also find that spindle oscillations are more frequent during the first half of the year compared to the second half of the year. The authors also find that spindle oscillations are more frequent during the first half of the decade compared to the second half of the decade. The authors also find that spindle oscillations are more frequent during the first half of the century compared to the second half of the century. The authors also find that spindle oscillations are more frequent during the first half of the millennium compared to the second half of the millennium. The authors also find that spindle oscillations are more frequent during the first half of the epoch compared to the second half of the epoch. The authors also find that spindle oscillations are more frequent during the first half of the day compared to the second half of the day. The authors also find that spindle oscillations are more frequent during the first half of the week compared to the second half of the week. The authors also find that spindle oscillations are more frequent during the first half of the month compared to\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}",
       "type": "dataframe",
       "variable_name": "mini_elife_df"
      },
      "text/html": [
       "\n",
       "  <div id=\"df-f9d4d5d5-fb0a-4dbb-9c88-1cf3a9cacd03\" class=\"colab-df-container\">\n",
       "    <div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>lay_summary</th>\n",
       "      <th>article</th>\n",
       "      <th>headings</th>\n",
       "      <th>keywords</th>\n",
       "      <th>id</th>\n",
       "      <th>biomistral_summary</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>It can take several months , or even years , f...</td>\n",
       "      <td>Mature neural networks synchronize and integra...</td>\n",
       "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
       "      <td>[neuroscience]</td>\n",
       "      <td>elife-69011-v2</td>\n",
       "      <td>The authors present a novel approach to study...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Many of our decisions are made on the basis of...</td>\n",
       "      <td>Many decisions are thought to arise via the ac...</td>\n",
       "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
       "      <td>[neuroscience]</td>\n",
       "      <td>elife-17688-v1</td>\n",
       "      <td>The authors present a new model of decision m...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Oculo-Cerebro-Renal syndrome of Lowe ( Lowe sy...</td>\n",
       "      <td>Mutations in the inositol 5-phosphatase OCRL c...</td>\n",
       "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
       "      <td>[cell biology]</td>\n",
       "      <td>elife-02975-v2</td>\n",
       "      <td>To determine the role of OCRL in clathrin-med...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>When an embryo develops , its cells must work ...</td>\n",
       "      <td>Gradients of signaling proteins are essential ...</td>\n",
       "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
       "      <td>[developmental biology]</td>\n",
       "      <td>elife-38137-v3</td>\n",
       "      <td>The authors have generated a new tool to stud...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Our genomes contain a record of historical eve...</td>\n",
       "      <td>Similarity between two individuals in the comb...</td>\n",
       "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
       "      <td>[evolutionary biology, genetics and genomics]</td>\n",
       "      <td>elife-15266-v1</td>\n",
       "      <td>The study by Busby et al. (2017) presents a c...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>\n",
       "    <div class=\"colab-df-buttons\">\n",
       "\n",
       "  <div class=\"colab-df-container\">\n",
       "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-f9d4d5d5-fb0a-4dbb-9c88-1cf3a9cacd03')\"\n",
       "            title=\"Convert this dataframe to an interactive table.\"\n",
       "            style=\"display:none;\">\n",
       "\n",
       "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
       "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
       "  </svg>\n",
       "    </button>\n",
       "\n",
       "  <style>\n",
       "    .colab-df-container {\n",
       "      display:flex;\n",
       "      gap: 12px;\n",
       "    }\n",
       "\n",
       "    .colab-df-convert {\n",
       "      background-color: #E8F0FE;\n",
       "      border: none;\n",
       "      border-radius: 50%;\n",
       "      cursor: pointer;\n",
       "      display: none;\n",
       "      fill: #1967D2;\n",
       "      height: 32px;\n",
       "      padding: 0 0 0 0;\n",
       "      width: 32px;\n",
       "    }\n",
       "\n",
       "    .colab-df-convert:hover {\n",
       "      background-color: #E2EBFA;\n",
       "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
       "      fill: #174EA6;\n",
       "    }\n",
       "\n",
       "    .colab-df-buttons div {\n",
       "      margin-bottom: 4px;\n",
       "    }\n",
       "\n",
       "    [theme=dark] .colab-df-convert {\n",
       "      background-color: #3B4455;\n",
       "      fill: #D2E3FC;\n",
       "    }\n",
       "\n",
       "    [theme=dark] .colab-df-convert:hover {\n",
       "      background-color: #434B5C;\n",
       "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
       "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
       "      fill: #FFFFFF;\n",
       "    }\n",
       "  </style>\n",
       "\n",
       "    <script>\n",
       "      const buttonEl =\n",
       "        document.querySelector('#df-f9d4d5d5-fb0a-4dbb-9c88-1cf3a9cacd03 button.colab-df-convert');\n",
       "      buttonEl.style.display =\n",
       "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
       "\n",
       "      async function convertToInteractive(key) {\n",
       "        const element = document.querySelector('#df-f9d4d5d5-fb0a-4dbb-9c88-1cf3a9cacd03');\n",
       "        const dataTable =\n",
       "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
       "                                                    [key], {});\n",
       "        if (!dataTable) return;\n",
       "\n",
       "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
       "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
       "          + ' to learn more about interactive tables.';\n",
       "        element.innerHTML = '';\n",
       "        dataTable['output_type'] = 'display_data';\n",
       "        await google.colab.output.renderOutput(dataTable, element);\n",
       "        const docLink = document.createElement('div');\n",
       "        docLink.innerHTML = docLinkHtml;\n",
       "        element.appendChild(docLink);\n",
       "      }\n",
       "    </script>\n",
       "  </div>\n",
       "\n",
       "\n",
       "<div id=\"df-8bbe8a0a-28c0-4d13-8a78-0d04323f318e\">\n",
       "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-8bbe8a0a-28c0-4d13-8a78-0d04323f318e')\"\n",
       "            title=\"Suggest charts\"\n",
       "            style=\"display:none;\">\n",
       "\n",
       "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
       "     width=\"24px\">\n",
       "    <g>\n",
       "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
       "    </g>\n",
       "</svg>\n",
       "  </button>\n",
       "\n",
       "<style>\n",
       "  .colab-df-quickchart {\n",
       "      --bg-color: #E8F0FE;\n",
       "      --fill-color: #1967D2;\n",
       "      --hover-bg-color: #E2EBFA;\n",
       "      --hover-fill-color: #174EA6;\n",
       "      --disabled-fill-color: #AAA;\n",
       "      --disabled-bg-color: #DDD;\n",
       "  }\n",
       "\n",
       "  [theme=dark] .colab-df-quickchart {\n",
       "      --bg-color: #3B4455;\n",
       "      --fill-color: #D2E3FC;\n",
       "      --hover-bg-color: #434B5C;\n",
       "      --hover-fill-color: #FFFFFF;\n",
       "      --disabled-bg-color: #3B4455;\n",
       "      --disabled-fill-color: #666;\n",
       "  }\n",
       "\n",
       "  .colab-df-quickchart {\n",
       "    background-color: var(--bg-color);\n",
       "    border: none;\n",
       "    border-radius: 50%;\n",
       "    cursor: pointer;\n",
       "    display: none;\n",
       "    fill: var(--fill-color);\n",
       "    height: 32px;\n",
       "    padding: 0;\n",
       "    width: 32px;\n",
       "  }\n",
       "\n",
       "  .colab-df-quickchart:hover {\n",
       "    background-color: var(--hover-bg-color);\n",
       "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
       "    fill: var(--button-hover-fill-color);\n",
       "  }\n",
       "\n",
       "  .colab-df-quickchart-complete:disabled,\n",
       "  .colab-df-quickchart-complete:disabled:hover {\n",
       "    background-color: var(--disabled-bg-color);\n",
       "    fill: var(--disabled-fill-color);\n",
       "    box-shadow: none;\n",
       "  }\n",
       "\n",
       "  .colab-df-spinner {\n",
       "    border: 2px solid var(--fill-color);\n",
       "    border-color: transparent;\n",
       "    border-bottom-color: var(--fill-color);\n",
       "    animation:\n",
       "      spin 1s steps(1) infinite;\n",
       "  }\n",
       "\n",
       "  @keyframes spin {\n",
       "    0% {\n",
       "      border-color: transparent;\n",
       "      border-bottom-color: var(--fill-color);\n",
       "      border-left-color: var(--fill-color);\n",
       "    }\n",
       "    20% {\n",
       "      border-color: transparent;\n",
       "      border-left-color: var(--fill-color);\n",
       "      border-top-color: var(--fill-color);\n",
       "    }\n",
       "    30% {\n",
       "      border-color: transparent;\n",
       "      border-left-color: var(--fill-color);\n",
       "      border-top-color: var(--fill-color);\n",
       "      border-right-color: var(--fill-color);\n",
       "    }\n",
       "    40% {\n",
       "      border-color: transparent;\n",
       "      border-right-color: var(--fill-color);\n",
       "      border-top-color: var(--fill-color);\n",
       "    }\n",
       "    60% {\n",
       "      border-color: transparent;\n",
       "      border-right-color: var(--fill-color);\n",
       "    }\n",
       "    80% {\n",
       "      border-color: transparent;\n",
       "      border-right-color: var(--fill-color);\n",
       "      border-bottom-color: var(--fill-color);\n",
       "    }\n",
       "    90% {\n",
       "      border-color: transparent;\n",
       "      border-bottom-color: var(--fill-color);\n",
       "    }\n",
       "  }\n",
       "</style>\n",
       "\n",
       "  <script>\n",
       "    async function quickchart(key) {\n",
       "      const quickchartButtonEl =\n",
       "        document.querySelector('#' + key + ' button');\n",
       "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
       "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
       "      try {\n",
       "        const charts = await google.colab.kernel.invokeFunction(\n",
       "            'suggestCharts', [key], {});\n",
       "      } catch (error) {\n",
       "        console.error('Error during call to suggestCharts:', error);\n",
       "      }\n",
       "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
       "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
       "    }\n",
       "    (() => {\n",
       "      let quickchartButtonEl =\n",
       "        document.querySelector('#df-8bbe8a0a-28c0-4d13-8a78-0d04323f318e button');\n",
       "      quickchartButtonEl.style.display =\n",
       "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
       "    })();\n",
       "  </script>\n",
       "</div>\n",
       "\n",
       "    </div>\n",
       "  </div>\n"
      ],
      "text/plain": [
       "                                         lay_summary  \\\n",
       "0  It can take several months , or even years , f...   \n",
       "1  Many of our decisions are made on the basis of...   \n",
       "2  Oculo-Cerebro-Renal syndrome of Lowe ( Lowe sy...   \n",
       "3  When an embryo develops , its cells must work ...   \n",
       "4  Our genomes contain a record of historical eve...   \n",
       "\n",
       "                                             article  \\\n",
       "0  Mature neural networks synchronize and integra...   \n",
       "1  Many decisions are thought to arise via the ac...   \n",
       "2  Mutations in the inositol 5-phosphatase OCRL c...   \n",
       "3  Gradients of signaling proteins are essential ...   \n",
       "4  Similarity between two individuals in the comb...   \n",
       "\n",
       "                                            headings  \\\n",
       "0  [Abstract, Introduction, Results, Discussion, ...   \n",
       "1  [Abstract, Introduction, Results, Discussion, ...   \n",
       "2  [Abstract, Introduction, Results, Discussion, ...   \n",
       "3  [Abstract, Introduction, Results, Discussion, ...   \n",
       "4  [Abstract, Introduction, Results, Discussion, ...   \n",
       "\n",
       "                                        keywords              id  \\\n",
       "0                                 [neuroscience]  elife-69011-v2   \n",
       "1                                 [neuroscience]  elife-17688-v1   \n",
       "2                                 [cell biology]  elife-02975-v2   \n",
       "3                        [developmental biology]  elife-38137-v3   \n",
       "4  [evolutionary biology, genetics and genomics]  elife-15266-v1   \n",
       "\n",
       "                                  biomistral_summary  \n",
       "0   The authors present a novel approach to study...  \n",
       "1   The authors present a new model of decision m...  \n",
       "2   To determine the role of OCRL in clathrin-med...  \n",
       "3   The authors have generated a new tool to stud...  \n",
       "4   The study by Busby et al. (2017) presents a c...  "
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mini_elife_df[\"biomistral_summary\"] = mini_elife_df[\"article\"].apply(lambda text: lay_summarize(text))\n",
    "mini_elife_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/",
     "height": 1000
    },
    "id": "dW2kZPup6t0I",
    "outputId": "76fa17f3-6a68-45e9-c892-12c10614736d"
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.32s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.22s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:05<00:00,  5.86s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.50s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.27s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.85s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.51s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.53s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.55s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:01<00:00,  1.65s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.66s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.61s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:01<00:00,  1.87s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.91s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.26s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:06<00:00,  6.52s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.24s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.27s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:05<00:00,  5.97s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.25s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.54s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.62s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.92s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.51s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.72s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.60s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.78s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.21s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.87s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.70s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:05<00:00,  5.97s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.06s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.76s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.35s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.25s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.93s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.62s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:01<00:00,  1.90s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:13<00:00, 13.10s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.17s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:01<00:00,  1.23s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.28s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.64s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.68s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.95s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.88s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.22s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.51s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.97s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.57s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.41s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.16s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.22s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.03s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.83s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.91s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.19s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.24s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.50s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.15s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.51s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.79s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.45s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.64s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.58s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.89s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:01<00:00,  1.70s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:00<00:00,  1.44it/s]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.86s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.76s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.73s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.54s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.62s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:06<00:00,  6.93s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.04s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:05<00:00,  5.20s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.41s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.25s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.45s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.91s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.40s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:05<00:00,  5.00s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.13s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.50s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.28s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.57s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.58s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.96s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.37s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.46s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.74s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.68s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.68s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.56s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.18s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.39s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.24s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.35s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:01<00:00,  1.91s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.20s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.47s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.50s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.87s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.48s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.03s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:01<00:00,  1.14s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.28s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.84s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:01<00:00,  1.95s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.08s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.92s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.05s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.01s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.22s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.25s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.21s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.24s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.37s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.66s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.99s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.25s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.27s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.67s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.55s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.77s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.79s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.22s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.94s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.92s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.46s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:05<00:00,  5.11s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.39s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.72s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.44s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.62s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.78s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.30s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:05<00:00,  5.39s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.50s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:01<00:00,  1.90s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.73s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.98s/it]\n"
     ]
    },
    {
     "data": {
      "application/vnd.google.colaboratory.intrinsic+json": {
       "summary": "{\n  \"name\": \"test_plos_df\",\n  \"rows\": 142,\n  \"fields\": [\n    {\n      \"column\": \"article\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 142,\n        \"samples\": [\n          \"Robustness in developing and homeostatic tissues is supported by various types of spatiotemporal cell-to-cell interactions . Although live imaging and cell tracking are powerful in providing direct evidence of cell coordination rules , extracting and comparing these rules across many tissues with potentially different length and timescales of coordination requires a versatile framework of analysis . Here we demonstrate that graph neural network ( GNN ) models are suited for this purpose , by showing how they can be applied to predict cell fate in tissues and utilized to infer the cell interactions governing the multicellular dynamics . Analyzing the live mammalian epidermis data , where spatiotemporal graphs constructed from cell tracks and cell contacts are given as inputs , GNN discovers distinct neighbor cell fate coordination rules that depend on the region of the body . This approach demonstrates how the GNN framework is powerful in inferring general cell interaction rules from live data without prior knowledge of the signaling involved . \\n Robustness in developing and homeostatic tissues is supported by the spatiotemporal cell-to-cell interactions , i . e . , feedback mechanisms acting at various time and length scales that prevent overgrowth or depletion . In the stem cell pool of homeostatic tissues , the spatial aspect of feedback have been shown to range from direct cell contacts [1\\u20134] to more indirect and longer range forms such as niche competition [5 , 6] . The fate correlation in time can come in different forms; growth and loss of cells may occur almost simultaneously within the tissue , or there could be regeneration cycles with longer time scale as in the case of hair follicles [7] . Moreover , the ordering of the events in the feedback scheme can be different; cell divisions can compensate for earlier cell loss , or vice versa . Although there are promising attempts to probe the cell-to-cell interactions from high-throughput single-cell analyses [8\\u201314] , live imaging of tissues followed by accurate tracking of the cells is still the most direct method to probe the details of the spatiotemporal feedback [15] . In the case of the mouse hind paw epidermis , analysis of the live images and cell tracks led to the finding that the skin stem cells have coordinated fate between the nearest-neighbors within a short time frame; cell divisions were coordinating with neighboring cell delamination ( due to differentiation ) with 1\\u20132 days delay [1] . However , whether such short spatiotemporal range of coordination as well as the time-ordering found in the hind paw epidermis is adopted in other tissues is currently unknown . In fact , the detailed rules of coordination and feedback have not been tested even for mouse skin regions other than the hind paw . To compare the cell coordination rules adopted in different tissues , it will be desirable to have a generic framework of analysis where such rules can be suggested unbiasedly , rather than preparing methods specific to each tissue . To this end , we may train a model based on machine learning to predict the future behavior of cells from past data , and later challenge the model to extract the rules that have been learned by the machine . The input to this machine can include high-content intracellular features such as the cell morphology and gene expression , as well as the interaction between cells . The key in this procedure will be to design a versatile machine learning scheme that gives predictions of the future dynamics while retaining high interpretability: the ability to partially mask data , apply attribution methods [16] , or use symbolic reductions . Such method will be in contrast to hypothesis-based approaches where certain coordination rules cannot be inferred by design . For example , the analysis conducted to extract the coordination features in the hind paw epidermis [1] assumed cell behavior inducing other cell\\u2019s activity ( i . e . , cell division or delamination ) , which cannot extract suppressive effects such as induced silencing of neighboring cells [17 , 18] . Machine learning methods have been heavily used to extract features of cells from live or fixed cell image sequences . Applications include segmentation and tracking [19\\u201321] , in silico labeling [22] , and direct prediction of cell fates [23] , which typically use versions of convolutional neural networks ( CNN ) . The downside in using image data directly for cell fate prediction is that images contain redundant information making it difficult to focus on the most relevant components to interpret the rules . For instance , a significant feature of cells undergoing cell division is their size growth , which means that the machine will likely learn through images that cell size is a good predictor of cell fate . This strong association will be a problem when we are interested in the mechanism upstream of fate decision ( i . e . , commitment to cell division ) , since cell size is difficult to mask out from images without affecting the other components . On the other hand , important aspects such as cell tracks , lineages , and contacts are not trivially deducible by the machines from the images , making it harder to conduct interpretable rule extractions based on the experiences of individual cells . Other methods that have been proposed to infer the rules of multi-component dynamics by interpretable machine learning frameworks [24] currently do not have the resolution that enables the rule extraction at the level of individual cells . A promising machine learning approach toward automatic cell interaction rule discoveries is the graph neural network ( GNN ) scheme [25] . In GNN models , the structure of a graph can be taken as an input as well as the features associated with each node , which gives flexibility compared with conventional methods such as CNNs . Taking advantage of its interpretability , the GNN approach has succeeded in forward dynamics predictions of physical systems [26] as well as in inferring the rules of agent-interactions [27] and dynamical properties [28] from timelapse data obtained by simulations and experiments . The GNN framework is particularly suited for representing the heterogeneous interactions between cells as well as the intracellular features extractable from image data such as cell shapes and gene expressions [29] . The dynamical features of multicellular systems can be captured by further incorporating the information of cell tracks and constructing a spatiotemporal graph . However , while GNN models have been proposed to deal with time-evolving graphs with nodes and edges being added or removed [30\\u201332] , there is still no example where replicating nodes are taken into account , which is important in describing cell divisions . In this work , we propose a GNN-based framework that incorporates the cell interactions as well as the cell tracks including divisions , and demonstrate how the framework can be applied to novel data to extract and compare the different cell fate coordination rules across tissues . First , we observe that a GNN model constructed from cell contacts and cell lineage can predict the fates of the mouse skin stem cells , indicating that the graph structure encodes significant information of cellular dynamics . Next , we demonstrate how the rules learned by the GNN model can be extracted by ablating the input features , re-configuring the model step-by-step , and employing the attribution method . As a result , we find neighboring cell fate coordination rules that have been overlooked in the paw skin data , as well as rules that are consistent with the previously conducted fate imbalance analyses [1] . For our main application , we analyze the novel tracking data of the ear skin stem cells [33] , and find an additional coordination rule that is absent in the paw skin . We further confirm the validity of the method by applying it to data generated by numerical simulations that encode stochastic cell dynamics mimicking the homeostatic epidermis . Together , this work demonstrates how the GNN model can be applied to highly stochastic kinetics with agent loss and gain , and how it can be utilized in solving the mechanism of multicellular kinetics systematically . \\n The epidermis is maintained by continuous cell divisions occurring in a pseudo-two-dimensional region called the basal layer . Cells in the basal layer can irreversibly delaminate toward the suprabasal layer and differentiate ( i . e . , turn post-mitotic ) , and eventually shed off ( Fig 1A ) . We use the dataset of cell tracks previously generated from live images collected from the non-hairy mouse plantar ( hind paw ) skin [1] , which includes the tracks of all the cells within a region of the basal layer . Between the time frames , cells conduct one of the three possible behaviors; divide ( Div ) , delaminate from the basal layer and migrate into the suprabasal layer ( Del ) , or stay ( no behavior , NB ) , typically with the ratio of 1:1:8 within the time interval of the original image acquisition ( 12 hours ) . Loss by cell death is negligibly rare in this tissue under homeostasis . The relative motion between the cells is slow , meaning that the main source of the cell motion is displacement due to cell division and cell delamination . In our GNN framework , we represent the spatio-temporal dynamics of the tissue by graphs designed to capture 1 ) the spatial relationship including the cell contacts as well as the lineages and events such as cell delamination and division , and 2 ) any cell feature such as the gene expression levels and cell area in the basal layer . We first construct graphs with the basal layer cells as nodes , connected by edges in spatial and temporal directions . The spatial edges represent the cell-to-cell contact obtained from the two-dimensional segmentation in the basal layer , based on the membrane reporter signal ( Fig 1B and S1 Video ) . The temporal edges are the tracks of the cells , involving forks corresponding to cell divisions and terminal ends representing cell delamination; we do not track the cells after they have left the basal layer . In each node ( i . e . , cell in a time frame ) , multiple features extracted from the raw images can be assigned . For the hind paw data , we chose the cross-section area of the cell in the two-dimensional basal layer calculated from the segmentation , and the level of the Fucci-G1 reporter [34] which is the integral of the corresponding fluorescent channel within the segmented cell area . The GNN model takes in the spatiotemporal graphs as well as the features associated with each node , and outputs predictions on the behavior of cells in the last frame ( Div , Del , or NB ) . We trained a GNN model using training and test data taken from two separate regions of the same mouse hind paw , each involving more than 200 cells tracked over fifteen time frames with a total of approximately 250 Div and Del events occurring in each region . In the training of a Nt-time model , we extracted graphs composed of sequential Nt time frames , where the time frames are indexed from t = \\u2212 ( Nt \\u2212 1 ) to 0 in temporal order ( Fig 1C ) , and conducted supervised machine learning to predict the fate of the target cell in the last frame t = 0 . The model calculates the output from the input by sequentially updating the features in the nodes in the following way ( information flow schematically shown in Fig 1D: bidirectional spatiotemporal GNN model ) . First , the model performs message passing ( MP ) on the temporal edges from the future to the past Nt \\u2212 1 times . Here , a single MP is conducted by concatenating the features of adjacent nodes to be processed through a multi-layer perceptron ( MLP ) , and then passing on that processed feature to the past node . Second , the model performs MP bidirectionally on the spatial edges Ns times ( Fig Ai in S1 Appendix ) . Third , the model performs MP Nt \\u2212 1 times on the temporal edges from the past towards the future nodes . The order of the MP ( Fig 1D ) and the sum aggregation method [35] are set this way so that the spatiotemporal graph structure is correctly reflected in the output . In the end of the MP , the node feature in the last frame incorporates the information from its Nt-frame ancestor nodes , Ns-step neighbor nodes of the ancestor nodes in the cell contact graph , as well as the daughter cells of the ancestor neighbors ( Fig 1C ) . We fix Ns = 1 in the following analyses since the performance of the model were similar for Ns = 1 and 2 ( Fig A in S1 Appendix ) , and also Nt = 4 , which will be changed later . Finally , we decode the node feature in the last frame via another MLP and output the softmax score for each cell fate label ( Div , Del , and NB ) to conduct three-class classification . Here , we chose MP rather than convolutional [36 , 37] or attention-based GNN [38] because MP has higher flexibility in representation . The generic functional form described in MP may become important to probe complex nonlinear interactions , for instance when high-dimensional data such as gene expression and morphology are used as cell features . The GNN model was successfully trained using standard methods employing a loss function ( softmax-cross-entropy loss for the three cell behavior labels ) and the Adam optimizer ( Fig B in S1 Appendix ) . The Area Under the Curve ( AUC ) score shows that all behaviors are predicted significantly better than a random guess ( AUC = 0 . 5 ) ( Fig 1E ) . Also , the small standard deviations between the training samples indicate the trained models are reproducible . In particular , we found that cell divisions can be predicted with high accuracy compared with Del and NB . We further found that by training the model with reduced features ( i . e . , without the cell area , G1-phase reporter signal , or both ) , the score decreased significantly , indicating that these features are utilized in making reliable predictions . Indeed , the cross-section area tends to be larger for dividing cells ( Fig Ci and v in S1 Appendix ) , consistent with previous observations [1 , 39] . Furthermore , the G1-phase signal tends to be lower in dividing cells as expected , while there was no difference in the G1-phase signal between NB and Del cells ( Fig Cvi in S1 Appendix ) . Intriguingly , we found that even without the node features , the GNN model can predict the fates with significant scores ( Fig 1E ) . This indicates that the graph structure itself encodes useful information in predicting cell fate . A candidate structure in the graph that can be used in the prediction is the number of edges ( i . e . , neighboring cells , Fig 1F ) . Indeed , the number of neighboring cells is positively correlated with cell division in the next time frame , which is explained by the fact that cells with a larger area tend to be in contact with more cells in the basal layer ( Fig Ciii in S1 Appendix ) . To see whether the number of edges is important in the prediction , we changed the model to take mean aggregation in the calculation step of the spatial edges , which will make the number of contacts invisible in the model . We found that this change in the model only has a minor effect on the AUC score , suggesting that there are other subgraph motifs that are utilized in the prediction . Other subgraph motifs which can be interpreted in the biological context include the temporal length up to the last cell division point ( Fig 1F ) . This is essentially the age of the target cell , which can be used to predict fate if the fate choice is temporally non-random [40] . The neighbor cell fates are also reflected as motifs , which should be important according to previous results [1] . Similarly , the sibling cell behavior can be exploited if there are sibling fate correlations such as in the case of asymmetric division [41] , although this is less relevant for the current dataset [1 , 40] . In order to identify which subgraph motifs are responsible for the predicted future cell dynamics , we next considered reducing the GNN model to observe how the predictability decreases . By simply excluding the initial step of propagating the information toward the past frames , the information from the temporal branches in the sibling and neighboring cells ( including their fates , Fig 1F ) will not be transmitted to the target cell ( Fig 2A: unidirectional spatiotemporal GNN model ) . To add-back the information of neighboring cell fate , we introduced an additional feature to each cell , the next frame behavior ( NFB ) , which takes Div , Del , or NB ( Fig 2B ) . Importantly , this additional feature was added only to the past cells ( t \\u2264 \\u22121 ) , since the NFB of the cells at t = 0 is the target of prediction . The new GNN model predicts cell fates almost as well as the previous model with and without the area and G1-phase reporter signal features ( Fig 2C and Fig D in S1 Appendix ) , indicating that the key ingredients are captured without the backward-time propagation as far as the neighboring cell fates ( NFB ) are added back . By further eliminating the NFB and seeing that the predictability drops , we confirmed that this feature indeed has impact on the prediction of all behaviors ( Fig 2C and Fig E in S1 Appendix ) . To further address the detail of the mechanism of prediction , we next employed the attribution method called the integrated gradient ( IG ) [16] . In this method , the impact of each input variable on the output is quantified by integrating the change in the output score level upon the gradual shift of each input feature ( Fig 2D and 2E and Fig F in S1 Appendix ) . The IG also reflects the sign of the impact of each feature; a negative attribution score of feature f toward fate a means that the observation of f will decrease the output score towards predicting a . The AUC and the IG scores are complementary to each other; the IG highlights the significance of inputs at a finer scale than the AUC , but does not give quantitative values of predictability . To account for the noise in IG induced by finite sample size , we further added a random feature to the cells , i . e . , uniform random numbers ranging from 0 to 1 that have no correlation with cell fate ( Fig Dii in S1 Appendix ) , and calculated its IG score to identify the baseline for non-zero signal ( orange shades , Fig 2D and 2E and Fig F in S1 Appendix ) . First , in Fig 2D and Fig Fi in S1 Appendix , the area of target cells at t = 0 are found to give positive and negative attribution for the prediction of cell division and delamination , respectively . This indicates that cells with large areas tend to conduct cell division in the next frame , while they are unlikely to delaminate . GNN models trained without spatial MPs also show that the area of target cells has indeed significant predictability of the cell fates ( Fig G in S1 Appendix ) . On the other hand , in Fig Fii in S1 Appendix , negative attribution is observed in the G1-phase signal of the target cells at t = 0 , which is consistent with the expectation that a high G1-phase signal of the target cell should predict no cell division in the next frame . Interestingly , the G1-phase signal has a relatively low score when the cell area feature is present ( Fig 2D ) , indicating that the G1-phase signal is mostly redundant for cell fate prediction . These results highlight how the model is efficiently focusing on the important variables from the multi-dimensional input . Without the cell area and G1-phase reporter signal , non-zero attribution scores are found for the prediction of fates by the division of the target cell ( Fig 2E ) . This is reflecting how the age of the cell can be used in the cell fate prediction; newly born cells tend to undergo a refractory period [40] , meaning that the target cell is less likely to divide again if it divided ( was born ) recently . More generally , the distributions of lifetime can be distinct between cells with different fates ( Fig H in S1 Appendix ) , which can be exploited in making fate predictions . To eliminate the cell age information altogether , we set all the features of the target cell sequence to zero ( Fig 3A: cell external model ) . We also took mean aggregation in conducting the MP on spatial edges to make the number of contacts invisible . This new model can still learn to predict the fates significantly better than random ( Fig 3B ) . We found a high IG score indicating that Del of a neighboring cell 24 hours before ( two frames prior ) is useful information in predicting Div . Consistent with this , when changing the number of input time frames Nt , the AUC score largely increased between Nt = 2 and 3 ( Fig Ii in S1 Appendix ) . These results match with the previous observation that cell delamination correlates with neighboring cell division 1\\u20132 days later [1] . Moreover , the GNN model predicts that there is an effect of neighboring Del suppressing Del in the next frame . This indicates that cell delamination is not entirely random , and there may exist a mechanism to suppress two or more neighboring cells to delaminate at the same time . This finding demonstrates that the rule extraction procedure using the GNN model is useful in predicting unexplored mechanisms . Importantly , while the AUC is low for some cases due to the high stochasticity in the cell fate events , it is still possible to determine the fate correlations in neighboring cells . We next analyzed the recently obtained mouse ear skin data [33] to compare the mechanism of homeostasis across different tissues . Although the ear skin has more structure ( i . e . , hair follicles and other appendages ) , we have previously observed that the behaviors of the interfollicular epidermal basal cells in the ear skin are similar to that of the hairless hind paw [1 , 40] . A noticeable difference is that the rate of divisions and delaminations were approximately two-fold slower in the ear , which is why the interval between the frames was set as 24 hours for the ear data acquisition . For our purpose , we generated a novel dataset of whole region cell tracks from 6 regions in the ear epidermis with 54 time frames in total . In Fig 3B and 3C , we show the AUC and the attributions for the ear data; the largest positive ( negative ) IG score for Div ( Del ) is attributed to the Del of neighboring cells from one frame earlier , indicating the existence of the same rule as the hind paw epidermis . Negative correlation between neighboring cell fates was expected to exist in the ear as well as it is critical in maintaining the stem cell pool [1] . However , the actual coupling as well as the ordering of the events have not been previously addressed . In addition , there are large positive ( negative ) IG scores for Del ( Div ) attributed to the Div of neighboring cells , which were not seen in the hind paw epidermis . These scores imply that a mechanism of neighboring Div-induced Del and Div-suppression may exist . This difference was further confirmed by conducting a neighbor-fate imbalance analysis ( Fig Ji and ii in S1 Appendix ) . In this analysis , we focus on individual cells that either divided or delaminated and follow the subsequent behaviors of their six-nearest neighbor cells to calculate the neighbor fate imbalance [1] . The average fate imbalance indeed deviates from zero for the neighbors of dividing cells in the ear , which is a distinct feature from the paw . These results indicate that the GNN-based method can detect cell fate coordination as efficiently as the previous neighboring imbalance method [1] which was constructed specifically for this purpose . The neighboring imbalance method suffers from the subtlety of correcting the background temporal fluctuation of cell fates ( i . e . , some time frames have significantly more Div of Del events than others ) , whereas for the GNN method the baseline is statistically controlled . For this reason , the method was sensitive enough to capture previously overlooked interactions of cell delamination suppressing neighboring delamination; this is not capturable by the neighboring imbalance analysis since it will only return a zero signal . Here we demonstrated how the multi-step ablation strategy is useful in identifying the possible mechanisms behind cell fate determination . Starting from the full model , we first identified strong predictors of cell fate , such as the cell area , which were eliminated in the next model to seek further underlying factors . Although the causal relationships between the input variables are complex and difficult to disentangle in most cases , systematically eliminating the variables with high attribution score is a solid method to eventually reach the original cause . Finally , we conducted numerical simulations of different coordinated cell fate models and tested whether artificially implemented rules can be correctly extracted . In the simulated models , cells are represented by points in the 2D space undergoing repulsive interaction with each other . Cell divisions and delaminations are recapitulated by abrupt point duplication ( with small noise ) and elimination , respectively ( Fig 4A ) . We encoded the cell fate coordination in the following ways ( Fig 4B ) . In the delamination-induced division setup , we first randomly picked a cell to delaminate and chose another cell randomly from its six-nearest neighbors to divide 24 hours after the first cell delaminated . In the division-induced delamination setup , the rules were flipped; a randomly picked cell was set to divide while one of its six nearest neighbors was randomly chosen to delaminate after the division . We also tried another setup where these two rules were equally mixed . Running the same GNN learning algorithm , we found that the AUC scores of each fate are in the expected order ( Fig 4C ) ; in the delamination-induced division setup , the division was predictable and delamination was not , and vice versa . By testing the attribution method on these simulated data , we confirmed that the inferred mechanisms indicate the implemented rules ( Fig 4D ) . We further noticed that the predictability of the fates become significantly low for the mixed rule setup ( Fig 4C ) , which is close to the case of experimental data ( Fig 3B , C ) . Hence , the low score obtained in the experimental data prediction can be explained by the inherent stochasticity in the cell dynamics . Even in such situation , the attribution method was able to pull out the coordination rules . This indicates that the AUC does not necessarily have to be high in order to identify the correct interaction rules . \\n Here we have used the graph-based learning framework to systematically infer how interactions correlate with cell fates in the basal layer stem cell pool of the epidermis . The GNN models were able to predict the future cell fate using the relationships of the target cell and the cells in contact in the past time frames . We identified the cell features that are attributed to the predictability of the model using IG , and clarified that neighboring cell fates have a delayed effect on the target cell\\u2019s fate outcome . The GNN-based approach gives interpretable attributions in the words of the spatiotemporal relationship between the cells , which is an advantage compared with direct image-based approaches employing CNNs . The extracted rules from the dynamics in the epidermis included the previously reported delamination-induced neighboring division [1] as well as potentially novel interactions: suppression ( induction ) of cell delamination by neighboring delamination ( division ) . Inhibition of certain behaviors by cells in contact is a common motif observed in cell biology [17 , 42 , 43] . Neighboring division-induced delamination may be associated with local mechanical forces [44] . Testing whether known molecular pathways responsible for lateral inhibition and mechanical signaling are playing roles in tissue homeostasis will be an interesting next step . The predicted rules are favorable mechanisms in keeping the cell density in the homeostatic tissue , and it is intriguing that there exist differences in the rules adopted in distinct regions of the skin . A possible explanation for this is the cell density; the hind paw epidermis is crowded compared with the ear ( 26 , 000 cells/mm2 versus 14 , 000 cells/mm2 , with almost the same thickness , 15 \\u03bcm , Fig K in S1 Appendix ) . In high cell density regions such as the hind paw , it is reasonable that spontaneous cell division is suppressed and cells can only proliferate when there is space provided by neighboring delamination . Interestingly , in the developing skin where the cell density is much lower than the adult ear , the stratification of cells has been reported to be driven by neighboring cell division [45] . Given these facts , we hypothesize that the balance between cell differentiation and proliferation can be maintained through distinct mechanisms depending on the operating cell density regimes . In predicting the kinetics , multiple time frames were required as inputs in our example ( Fig I in S1 Appendix ) due to the significant time delay between the cell fate events . This indicates that the proper treatment of the temporal axis was necessary , which is why GNN approaches based on static graphs [46\\u201348] are not directly applicable to the problem of multicellular kinetics . Furthermore , our GNN framework takes into account the replication of nodes ( i . e . , cell division ) , which has not been addressed in previous models involving time-evolving graphs [30\\u201332] . Since the incorporation of the cell tracks including cell division is fundamental in the analysis of multi-cellular systems , this study serves as an important step towards building an unbiased rule extraction framework for tissue dynamics . Multicellular dynamics is inherently stochastic owing to the complex interaction between cells and the environment as well as the single-cell level fluctuations . Likely due to this stochasticity , the prediction scores generated by the GNN in this study were far from accurate . Nevertheless , the models were able to provide sensible predictions and cell interaction rules from the epidermis data and inferred the correct rules from the simulation data . The success of the current approach on data generated by highly stochastic kinetics suggests that the framework is applicable not only for general multi-cellular dynamics such as developing embryos , malignant tissues , and organoids , but also for a wide range of systems where modeling by stochastic interacting agents is effective such as in disease spreading and ecology . The current application relied on a relatively large data set of curated cell tracks from whole regions of the mouse skin . Although obtaining such data is still technically demanding in other biological tissues , recent microscopy and cell tracking methods are producing promising results in generating whole tissue level tracks in many systems [15] . In analyzing data for example from developing tissues , an important next step will be to develop a method to interpret the effects of higher-order subgraph motifs . We extracted the effects of neighboring cell fates by converting the cell fate motif ( branches and termination ) into a node feature variable , which will need to be generalized in order to capture more complex relations such as fate imbalance across generations and three-body cell interactions . In addition to the attribution method employed in this work , attention-based GNN [38] and other methods which attribute subgraph motifs to the prediction [49 , 50] will be candidate strategies to improve the interpretability of GNN models and reveal complicated cell-to-cell interactions . \\n The cell track data of the mouse hind paw epidermis basal layer was generated in our previous work [1] . For the ear epidermis data , we used the images collected in the work of [33] , and conducted the semi-automated tracking procedure similar to the previous method [1] . We first performed 3D segmentation by cellpose [21] using the nucleus channel ( K14H2BmCherry ) from a region size of 0 . 3 mm \\u00d7 0 . 3 mm \\u00d7 40\\u03bcm and obtained the 3D masks of the cell nuclei . We then defined the height of the interface between the epidermis and the dermis based on the 3D masks of the nuclei and subtracted this height from the original 3D data to level the basal layer position . From the height-corrected 3D images , we took three consecutive z-positions containing the nucleus of all the basal layer cells and averaged the intensity over the three slices to obtain 2D images in each channel . We calculated the local maxima of the cell nuclei ( K14H2BmCherry ) and automatically corrected the shifts between time frames by minimizing the square distance between the nearest cell positions across the frames using affine transformation . The cells included in all ten time frames were used in the following analysis . The crop size in the 2D plane therefore varied across different areas , ranging from 139 \\u03bcm \\u00d7 139 \\u03bcm to 238 \\u03bcm \\u00d7 238 \\u03bcm . At each time frame , we segmented the cells using the marker-controlled watershed algorithm using the mem-tdTomato channel and the maxima of the cell nuclei positions . We assigned each cell ( i . e . , segmented area ) to a segmented area in the previous time frame with the largest overlap . Tracked cells were frequently lost or were associated with more than one cell in the subsequent time frame , which indicated cell delamination from the basal layer and cell division , respectively . We then manually corrected the errors in the tracking with a guide from the height-corrected 3D images using a pipeline employing napari [51] , and cropped out the region close to the edge so that the remaining region only includes reliably tracked cells . The miss annotations are negligible since the time intervals are significantly shorter than the lifetimes of the cells ( average lifetimes were 2 . 3 days in the hind paw data compared to 0 . 5 day interval , and 4 . 0 days in the ear data compared to one day interval ) . The script outputted the segmented areas of the cells at each time point and their lineages , which was used to build the spatiotemporal graphs . In building the graphs , we determined the cell neighbors from the segmentation images; edges were drawn between cells that share more than one pixel as their periphery . All codes involved in generating the basal layer cell tracks in the ear epidermis were written in Python . We used two areas of tracking data for the hind paw data , and six areas for the ear data . The total number of cell-frames in the spatiotemporal graphs was 5 , 996 for the hind paw data ( \\u223c214 unique cells per area per time frame ) and 12 , 828 for the ear data ( \\u223c267 unique cells per area per time frame ) . We split the graph data into two , the training set and the test set ( Fig L in S1 Appendix ) . The number of cell-frames in the two sets was made to be comparable . The proportion of cells that experience delamination and division in each frame were both 9% in the hind paw epidermis data and both 7% in the ear epidermis data . For the neighbor fate net imbalance analysis , we applied the same method as previously described [1] to the ear epidermis data as well as to the simulation data . The spatiotemporal graphs were created from Nt sequential time frames from the segmented time-lapse images using Deep Graph Library [52] . We first created the cell-contact graphs for each time frame . The neighboring cells \\u03b1 and \\u03b2 were connected by two directed edges pointing at each other . We then added directional edges between the same cells in the future and the past as well as their parents and daughter cells in the sequential time frames . The cell feature vector x i , \\u03b1 \\u2208 R n was assigned to each cell \\u03b1 for the i-th time frame . We used the cell area and G1-phase reporter signal as components of the feature vector , which were obtained from the original segmented images . The features were normalized by dividing the values by the maximum values among all the training and test data . When reducing a feature , we set that particular feature to zero in all nodes . We processed these spatiotemporal graphs with a GNN model using PyTorch . Our first GNN model is a collection of models which consists of a backward temporal edge model \\u03a6B , edge , a backward temporal node model \\u03a6B , node , a spatial edge model \\u03d5edge , a spatial node model \\u03d5node , a forward temporal edge model \\u03a6F , edge , a forward temporal node model \\u03a6F , node , and a decoder \\u03c8dec . First , we propagated the information from the future to the past using \\u03a6B , edge , and then updated the node feature by \\u03a6B , node . By initializing the node feature a i , \\u03b1 ( 0 ) = x i , \\u03b1 , the ( k + 1 ) -th update ( 0 \\u2264 k \\u2264 Nt \\u2212 2 ) of the node feature a i , \\u03b1 ( k + 1 ) of cell \\u03b1 in the i-th frame ( 0 \\u2264 i \\u2264 Nt \\u2212 1 ) is given by , A ( i , \\u03b1 ) \\u21d0 ( i + 1 , \\u03b2 ) ( k + 1 ) = \\u03a6 B , edge ( a i , \\u03b1 ( k ) , a i + 1 , \\u03b2 ( k ) ) , ( 1 ) B i , \\u03b1 ( k + 1 ) = \\u2211 ( i + 1 , \\u03b2 ) \\u2208 D ( i , \\u03b1 ) A ( i , \\u03b1 ) \\u21d0 ( i + 1 , \\u03b2 ) ( k + 1 ) , ( 2 ) a i , \\u03b1 ( k + 1 ) = \\u03a6 B , node ( B i , \\u03b1 ( k + 1 ) , a i , \\u03b1 ( k ) ) . ( 3 ) Here , the subscript ( i , \\u03b1 ) denotes cell \\u03b1 in the i-th frame , and ( i , \\u03b1 ) \\u21d0 ( j , \\u03b2 ) denotes the edge between cell \\u03b1 in the i-th frame and cell \\u03b2 in j-th frame . We calculated Eq 1 for all the connected pairs of cells in the spatiotemporal graphs . D ( i , \\u03b1 ) is the set of the daughters of the cell \\u03b1 in the i-th frame if the cell \\u03b1 divides in the i-th frame; otherwise , D ( i , \\u03b1 ) is cell \\u03b1 itself in the ( i + 1 ) -th frame . We set the features of the cells in the final frame a i , \\u03b1 ( k + 1 ) = a i , \\u03b1 ( 0 ) , which was not updated . Furthermore , for cell \\u03b1 which delaminates or exits the field of view in the i-th frame , we set B i , \\u03b1 ( k + 1 ) = 0 . Second , we calculated the edge features in each time frame using \\u03d5edge , and then updated the cell feature vectors with the edge features using \\u03d5node . By initializing with b i , \\u03b1 ( 0 ) = a i , \\u03b1 ( N t - 1 ) , the ( l + 1 ) -th update of the node feature b i , \\u03b1 ( l + 1 ) of cell \\u03b1 in the i-th frame is given by , H ( i , \\u03b1 ) \\u21d0 ( i , \\u03b2 ) ( l + 1 ) = \\u03d5 edge ( b i , \\u03b1 ( l ) , b i , \\u03b2 ( l ) ) , ( 4 ) I i , \\u03b1 ( l + 1 ) = AGG ( { H ( i , \\u03b1 ) \\u21d0 ( i , \\u03b2 ) ( l + 1 ) : ( i , \\u03b2 ) \\u2208 N ( i , \\u03b1 ) } ) ( 5 ) b i , \\u03b1 ( l + 1 ) = \\u03d5 node ( I i , \\u03b1 ( l + 1 ) , b i , \\u03b1 ( l ) ) . ( 6 ) Here , N ( i , \\u03b1 ) is the set of the neighbor cells of \\u03b1 in the i-th frame . In Eq 5 , AGG represents either the sum aggregation or the mean aggregation across the set of the neighbor cells . We repeat this process for Ns times to take into account the Ns-step neighbor interactions , in which case 0 \\u2264 l \\u2264 Ns \\u2212 1 . Third , we propagated the information from the past to the future using \\u03a6F , edge , and then updated the node feature by \\u03a6F , node . By initializing with c i , \\u03b1 ( 0 ) = b i , \\u03b1 ( N s ) , the ( m + 1 ) -th update ( 0 \\u2264 m \\u2264 Nt \\u2212 2 ) of the node feature c i , \\u03b1 ( m + 1 ) of cell \\u03b1 in the i-th frame is given by , U ( i + 1 , \\u03b1 ) \\u21d0 ( i , \\u03b2 ) ( m + 1 ) = \\u03a6 F , edge ( c i , \\u03b1 ( m ) , c i + 1 , \\u03b2 ( m ) ) , ( 7 ) V i , \\u03b1 ( m + 1 ) = U ( i + 1 , \\u03b1 ) \\u21d0 P ( i + 1 , \\u03b1 ) ( m + 1 ) , ( 8 ) c i , \\u03b1 ( m + 1 ) = \\u03a6 F , node ( V i , \\u03b1 ( m + 1 ) , c i , \\u03b1 ( m ) ) . ( 9 ) Here , P ( i + 1 , \\u03b1 ) is the parent of cell \\u03b1 in the ( i + 1 ) -th frame if the cell \\u03b1 is born in the ( i + 1 ) -th frame; otherwise , P ( i + 1 , \\u03b1 ) is cell \\u03b1 itself in the i-th frame . We set the features of the cells in the final frame c i , \\u03b1 ( m + 1 ) = c i , \\u03b1 ( 0 ) , which was not updated . Furthermore , for cell \\u03b1 which pops in the field of view from the outside in the i-th frame , we set V i , \\u03b1 ( m + 1 ) = 0 . Finally , we decoded the cell feature of cell \\u03b1 in the final frame ( ( Nt \\u2212 1 ) th frame ) by: Y N t - 1 , \\u03b1 = \\u03c8 dec ( c N t - 1 , \\u03b1 ( N t - 1 ) ) . ( 10 ) For all the functions \\u03a6 B , edge : R 2 n \\u2192 R n , \\u03a6 B , node : R 2 n \\u2192 R n , \\u03d5 edge : R 2 n \\u2192 R n , \\u03d5 node : R 2 n \\u2192 R n , \\u03a6 F , edge : R 2 n \\u2192 R n , \\u03a6 F , node : R 2 n \\u2192 R n , \\u03c8 dec : R n \\u2192 R 3 in our GNN model , we used the multi-layer perceptron ( MLP ) , whose components are ( 1 ) Nlayer hidden layers which are respectively composed of a fully-connected layer and a rectified linear unit ( ReLU ) , and ( 2 ) an output fully-connected layer . For the unidirectional spatiotemporal model , we skipped the backward temporal edge and node models in the bidirectional spatiotemporal model . Hence , we initialized b i , \\u03b1 ( 0 ) = x i , \\u03b1 in the spatial edge and node models . In the cell external model , we assigned a null feature vector for the target cells; in the spatial edge model , we initialized b i , \\u03b1 ( 0 ) = 0 and b i , \\u03b2 ( 0 ) = x i , \\u03b2 for the edges from cell \\u03b2 to cell \\u03b1 in the i-th frame . To represent the local lineage branch structure , we introduced the next frame behavior ( NFB ) as a new feature , which encodes the behavior of the next frame of that cell by a one-hot vector , NB ( [1 , 0 , 0] ) , Del ( [0 , 1 , 0] ) , or Div ( [0 , 0 , 1] ) . Since the NFB in the final frame of each network is what we aim to predict , we set the NFB in the final frame to null vector ( [0 , 0 , 0] ) . We trained the GNN models as a three-class classification problem between the three possible fates , NB , Del and Div ) . Since the proportion of the three cell fates is imbalanced , we used the weighted softmax-cross-entropy loss where the weight of each label was set to the inverse of the proportion of the cell fate in the training data . To minimize the loss , we used Adam optimizer with the learning rate lr = 0 . 0001 . In the training , we input a spatiotemporal graph of Nt sequential time frames obtained from an imaging area to update the parameters of the GNN model . We repeated the update for all the spatiotemporal graphs in a single epoch of the training . To optimize the number of layers Nlayer , the number of nodes Nnode of a hidden layer of the MLPs , and the dropout rate p , we tested the performance of the GNN model by changing these parameters . For this test , we used the simulation data of delamination-induced division setup with NFB and random features . First , we investigated the effect of Nlayer by setting Nnode = 50 and p = 0 . To quantify the performance of the GNN model , we calculated the maximum value of macro-F1 score of the test data during learning for 2000 epochs . We ran 5000 epochs for Nlayer = 2 , since the learning was exceptionally slow . As shown in Fig Mi in S1 Appendix , Nlayer = 1 and Nlayer = 2 give comparable performance . Second , we investigated the effect of Nnode by setting Nlayer = 1 and p = 0 . We found that Nnode does not significantly affect the performance ( Fig Mii in S1 Appendix ) . Hence , we chose Nlayer = 1 and Nnode = 50 to minimize the size of the GNN model . Finally , we changed p as shown in Fig Miii in S1 Appendix , and found that p also does not significantly affect the performance , while overfitting is suppressed by increasing p ( Fig Miv , v in S1 Appendix ) . We here chose p = 0 . 1 since the average performance was slightly better and the standard deviation is smaller compared with the other conditions . In all the training , we set Nlayer = 1 , Nnode = 50 and p = 0 . 1 , and ran 2000 epochs . We exceptionally ran 5000 epochs for the bidirectional spatiotemporal model with mean aggregation without any feature . We also tested the effect of data size on the performance ( Figs N and O in S1 Appendix ) . We found that the prediction score and attribution score are sufficiently high when the number of cell-frames used in the training data is above 2000 . The AUC and attribution of each condition were calculated for the GNN model obtained at the epoch at which the model achieves the maximum macro-F1 score [53] . In Fig B in S1 Appendix , we show an example of the learning curves of the cell external model for the hind paw data with NFB and the random feature . The weighted softmax-cross-entropy loss , macro-F1 score , recall , and precision curves are shown respectively in Fig B in S1 Appendix . The vertical lines in Fig Bi-iv in S1 Appendix indicate the epoch at which the model achieves the best macro-F1 score . We used the integrated gradients ( IG , [16] ) for the attribution . The IG Ik ( g ) of the k-th feature for an input subgraph g of a target cell is given by: I k f ( g ) = ( X k - X k \\u2032 ) \\u00b7 \\u222b 0 1 \\u2202 F f ( X \\u2032 + \\u03b1 \\u00b7 ( X - X \\u2032 ) ) \\u2202 X k d \\u03b1 . ( 11 ) The function Ff is the softmax score of fate f \\u2208 {NB , Del , Div} as a function of the input features calculated by the trained network . X represents the concatenated features of all the cells of g , and Xk is the value of the kth feature . X\\u2032 is the baseline , which is the null vector of the same size as X . We calculated the IG for three cell fate labels of all the input graphs . Since the baseline should be neutral for calculating the attribution , the GNN model must be trained for the null graphs to provide equal soft-max scores for three cell fate labels . To this end , we minimized the mean-squared-error ( MSE ) loss defined as , L MSE = 1 N sub \\u2211 f \\u2211 n = 1 N sub ( p n f , null - 1 3 ) 2 ( 12 ) together with the weighted softmax-cross-entropy loss . Here , p i f , null is the softmax score of fate f \\u2208 {NB , Del , Div} of the null graph of the n-th target cell\\u2019s subgraph , and Nsub is the number of the subgraphs . Within a single epoch , we first input a spatiotemporal graph to update the parameters using the weighted softmax-cross-entropy loss , and then input the corresponding null graph to update the parameters using the MSE loss . By this learning method , we approximately obtained the neutral baseline softmax scores for null graphs ( Fig Pi in S1 Appendix ) . Also , we confirmed that introducing the MSE loss does not affect the model performance ( Fig Pii in S1 Appendix ) . In the analysis of the attribution , we pooled each feature of each target cell\\u2019s subgraph into the relative spatiotemporal position of the feature with respect to the target cell . We calculated the average IG of each category for each cell fate label . With respect to the average IG of NFB , we averaged the IG of each category of NFB only among the cells in the category . Finally , we calculated the mean of the average IG over all the trained GNN models . We also defined the baseline for non-zero signals shown as the orange shades in the attribution plots . The minimum ( maximum ) value of the range of the baseline is defined as the minimum ( maximum ) of the model-average of IG subtracted ( added ) by the standard error among all the pooled random features . Since \\u2211fFf ( X ) = 1 for any X , we have \\u2211 f I k f ( g ) = 0 , ( 13 ) To make the plots in Figs 2\\u20134 , we pooled the features according to the spatiotemporal position . The pooled features , which we denote as a , contain multiple features k in the original calculation . The attribution score for each pooled feature is evaluated by I a f ( g ) = \\u2211 k \\u2208 a I k f ( g ) \\u2211 k \\u2208 a 1 . ( 14 ) Note that the normalization still holds: \\u2211 f I a f ( g ) = \\u2211 k \\u2208 a \\u2211 f I k f ( g ) \\u2211 k \\u2208 a 1 = 0 . ( 15 ) For the numerical data that mimics the dynamics of basal layer dynamics , we took a simplified model of interacting particles that exclude each other through mechanical interactions and undergo stochastic division and elimination events . We placed N0 cells labeled by \\u03b1 in a two-dimensional plane with size L \\u00d7 L and with periodic boundary conditions and let the cells interact with each other through an interacting potential . The equation of motion reads r \\u02d9 \\u03b1 ( t ) = - \\u2211 \\u03b2 \\u2260 \\u03b1 \\u2202 \\u2202 r \\u03b1 u ( r \\u03b1 , r \\u03b2 ) , ( 16 ) which is an overdamped kinetics without noise . r\\u03b1 is the position vector of cell \\u03b1 . The repulsive interacting potential has the typical length scale l: ur\\u03b1 , r\\u03b2=12K ( |r\\u03b1\\u2212r\\u03b2|\\u2212l ) 2|r\\u03b1\\u2212r\\u03b2|<l0|r\\u03b1\\u2212r\\u03b2|\\u2265l . ( 17 ) The rules of cell division and delamination were implemented by the Monte Carlo method . At the time frame of cell division of cell \\u03b1 , a newly born cell \\u03b1\\u2032 is generated at a random position within a small distance d = 0 . 001 \\u00d7 L . Cell delamination is conducted by eliminating a particle instantaneously . In both cases , the position of the cells quickly relaxes to a dispersed state due to the repulsive force between the cells , Eq 16 . The stochastic rules of fate coordination that we tested are: We used K = 9 hours\\u22121 , l = 0 . 125 , and L = 1 in all the simulations . For the time steps , we took \\u0394t = 1 . 2 hours . In generating the data for the graph construction , we first prepared N0 points randomly placed inside the box ( size L \\u00d7 L ) and simulated the time evolution according to Eq 16 by the Euler method for 100 steps to obtain a dispersed cell configuration . Next , we ran the simulation up to 300 steps ( 360 hours ) with both the equation of motion Eq 16 and the stochastic divisions and delaminations , to make sure that the system has reached a steady-state ( Fig Q in S1 Appendix ) . Finally , we ran the simulation for another 300 steps and sub-sampled the time points every 20 steps ( 24 hours ) from this final time series to generate data resembling the ear epidermis . The initial number of cells N0 was changed in the three setups to ensure that the number of cells in the frames at steady-state are roughly the same ( around 500 , Fig Q in S1 Appendix ) . The rates of fates \\u03bb were also tuned for each setup so that the number of events that take place per cell per time frame is comparable with the experiment . We cropped out the edges and used the data from the points in the center region 0 . 65 L \\u00d7 0 . 65 L so that the number of points per frame is roughly the same as the number of cells per frame in the hind paw and ear data ( around 210 ) . The neighboring cell network was generated by the two-dimensional Voronoi tessellation , and by whole spatiotemporal network was fed to the GNN learning process in the same way as the experimental data .\",\n          \"Understanding the relationships between biological processes is paramount to unravel pathophysiological mechanisms . These relationships can be modeled with Transfer Functions ( TFs ) , with no need of a priori hypotheses as to the shape of the transfer function . Here we present Iliski , a software dedicated to TFs computation between two signals . It includes different pre-treatment routines and TF computation processes: deconvolution , deterministic and non-deterministic optimization algorithms that are adapted to disparate datasets . We apply Iliski to data on neurovascular coupling , an ensemble of cellular mechanisms that link neuronal activity to local changes of blood flow , highlighting the software benefits and caveats in the computation and evaluation of TFs . We also propose a workflow that will help users to choose the best computation according to the dataset . Iliski is available under the open-source license CC BY 4 . 0 on GitHub ( https://github . com/alike-aydin/Iliski ) and can be used on the most common operating systems , either within the MATLAB environment , or as a standalone application . \\n Modelling and understanding of the relationship between complex and intermingled biological signals is often difficult , particularly when the drivers of the signals are unknown . The problem of the relationship between two time series can be addressed using deconvolution , which provides Transfer Functions ( TFs ) representative of the system processing on the input signal to generate the output signal [1] . Extracting the transfer function linking neuronal activity and imaging data is widely used in functional brain imaging [2\\u20139] , but TFs can also solve general problems in signal analysis , such as predicting the output of complex electrical circuits[10] or other industrial systems , for which a proper model is overly complex due to multiple processes working in parallel[11] . In brain imaging based on blood flow dynamics , transfer functions are classically used to lump the multitude of cellular and molecular processes linking neural activation to changes in blood flow . This coupling between neural activity and hemodynamics is known as neurovascular coupling ( NVC ) [12] . While there are many successful phenomenological models of NVC [3 , 13\\u201317] , most physiology-based models of neurovascular coupling[18\\u201321] focus on a single cellular mechanism . As NVC is mediated through multiple processes ( several molecular cascades , each mediated by different cell types ) , a more integrated approach is necessary . NVC has often been assessed with deconvolution [8 , 22] , either in the frequency domain or with matrix-based approaches , like Toeplitz matrices[23] . While these approaches allow the unbiased extraction of the TF , these deconvolution methods suffer from sensitivity to noise , affecting the quality of the computed TFs . Reducing the noise ( or bandwidth ) of the signals improves the estimate of the TF . Alternatively , one can opt for optimization of known functions or a kernel of functions [7 , 9] . The first option may lead to information loss , e . g . , in cases where the noise is not well characterized . Sophisticated smoothing methods partially prevent this loss , like Savitzky-Golay filter , or noise modelling as proposed by Seghouane and colleagues [24] . The second option relies on parametric functions to find the TF best linking the input to the output signals . The transfer function for neural activity to hemodynamic signals has been canonically modeled using a gamma-distribution function [3 , 14\\u201316] . While making assumptions as to the shape of the TF has some drawbacks , it is robust in the face of noise and generates parametric representations of intrinsically smooth TFs . These approaches still can suffer from under/overfitting and the search for the minimum of the cost function for ill-posed problems may represent a challenging exercise . A valuable help comes from non-deterministic optimizations like simulated annealing or genetic algorithms , which despite their computational expense have potential advantages in extracting TFs from time series . Recently , our group has been extensively involved in TF computation of neurovascular coupling in a study based on multi-modal recordings , namely two-photon microscopy and ultra-fast functional ultrasound [25] . For the required task , we comprehensively tested many deconvolution and optimization algorithms to choose the best-suited approach . We noticed that there is no standard software package providing all these different TF extraction tools , nor a program where all these approaches are available in a comfortable signal pre-treatment and I/O workflow . Here , we present Iliski ( [\\u026al\\u026a\\u0283k\\u026a] , meaning \\u201crelationship\\u201d in Turkish ) , a software which contains all the functionalities that we previously used ( Aydin et al . ) and which , being open source , can be further improved by the users . Although Iliski was initially thought to help data analysis in Biology , its features make it suitable for diverse applications [26 , 27] . \\n We propose two input files format: either plain text files or HDF5 data , the latter being an open-source file format with advanced database features . As experimental acquisitions are prone to multiple component noise , we provided , as an option to the analysis workflow , smoothing ( Savitzky-Golay method ) and median filter functions , to exclude outliers . The input and output signals are interpolated to a chosen time interval ( \\u0394t ) . Both signals can be cut between two given time points to study continuous recordings while computing TFs on chunks of signal ( Fig 1C ) . As an option , boxcar function of variable duration can be used in place of the input signal . Note that Iliski was not coded to handle complex-valued signals . Two main types of TF computation are proposed: deconvolution or function optimization . The former is straightforward , either Toeplitz or Fourier deconvolution , and does not require any specific settings . The latter is the optimization of a parametric function , which requires further settings depending on the chosen algorithm . Beside the proposed fitting functions , the users can input their own function in the graphical interface or add it to the default ones by modifying a text file ( the procedure is described in the Iliski Manual ) . The TF dimension is user-defined , with setting of the TF duration and \\u2018Sampling Time\\u2019 parameters that match the original data or can be augmented by non-linear interpolation . Optimization of parameters can be done with various Matlab algorithms , each coming with pros and cons ( see Results section ) ( Fig 1C , middle ) . A TF is evaluated comparing its prediction\\u2013the convolution of the input signal and the TF\\u2014to the expected output . Two metrics are used in Iliski: the Pearson coefficient ( corrcoef function , Matlab , Fig 1C , right ) and the residual sum of squares . The former was chosen to have a metric solely focusing on the dynamic , allowing for inter-subject comparisons , while the latter evaluates the overall fit , considering the amplitude of the prediction . The cost function of all the optimization algorithms tested in this article is the residual sum of squares ( hereinafter referred to as \\\"residuals\\\" ) . The results structure is arranged to be as informative as possible while avoiding useless repetition of data . Iliski allows for loading previously computed results structures to check them again . After TF computation , results structure can be saved either as XLS file , readable by any Excel-like software , or as a MAT-file ( MATLAB formatted binary file format ) , but it is also available in Matlab workspace to be exported in various data formats by the user . Iliski is accessible both as a GUI and as a set of functions to be used in scripts . It has been developed using Matlab R2018a , with the following dependencies: Optimization Toolbox , Signal Processing Toolbox and Global Optimization Toolbox . Common user errors are thoroughly prevented by various messages and fail safes . In parallel , all errors are treated and saved in a log file , to allow for efficient bug-fixing by any developer . We purposely kept just a few parameters to modify through the GUI , with the goal of providing an easy-to-use tool for people not used to these functions . In most cases , Matlab default parameters of each deconvolution/optimization function worked well with our data , and we believe that it can be extended to many biological datasets . However , a user skilled with Matlab and optimization algorithms can easily modify the parameters of each function used . This study uses already published data of animal experimentation ( Aydin et al . ) . All animal care and experimentations were performed in accordance with the INSERM Animal Care and Use Committee guidelines ( protocol numbers CEEA34 . SC . 122 . 12 and CEEA34 . SC . 123 . 12 ) . \\n Fig 2 shows TF computation with different settings over the same couple of signals: neuronal ( Ca2+ ) activations and vascular ( red blood cells velocity ) flow increases recorded in a mouse upon odor application . Our example data display unavoidable and complex noise coming from many sources: the biological system , the optical setup , the electronics , etc . Deconvolution with Fourier or Toeplitz approaches predicts the vascular responses very well for a given data set . However , the high-frequency noise is amplified by deconvolution [24] and transmitted to the TF , the predictions are not robust across data sets and the actual dynamics of neurovascular coupling is completely hidden in the TF noise ( Fig 2 ) . In this example , we show what we regard as a typical case of overfitting . The TF is capturing the high frequency noise of the system because it does not have any previous expectations for the shape of the relationship between the input and the output signals . This contrasts to the optimization of a parametric function approach which , although it imposes constraints on the shape of the TF , gives meaningful neurovascular relationship and does not need noise clearing . In blood flow-based neuroimaging , the standard function used to represent neurovascular coupling is composed of one or two \\u0393 functions , depending on the nature of the signals , i . e . purely vascular or based on oxygen level[30] . Below is the one \\u0393-driven function we used with our data . Where p1 , . . , p4 are the parameters to optimize , and H is the Heaviside function that includes a time-shift parameter ( p3 ) , In some cases , the time shift significantly improved the prediction and is a known biological phenomenon to consider[31] . Its four parameters are not all independent from one another , e . g . , p1 , p2 and p4 all impact the TF amplitude . This inter-dependency between the parameters brings an ill-posed optimization problem with multiple local minima of the cost function , the sum of the residual squares , in the 4D space of the parameters . To tackle the function optimization problem , we chose standard algorithms ( all implemented in the Optimization Toolbox of Matlab ) to encompass the main available options . A derivative-free optimization method [6] is provided by the fminsearch function in Matlab , which uses the Nelder-Mead simplex algorithm . This approach on our data produced a TF with more than one underivable point that is not representative of the smooth dynamic of neurovascular coupling . Another common option is provided by Quasi-Newton optimization algorithms , which uses an approximation of the Jacobian: for this approach too , we tested an unconstrained built-in method ( fminunc function , Matlab ) . This prediction is , overall , as good as with fminsearch ( Fig 2 , Pearson coefficients , fminunc vs . fminsearch: 0 . 95 vs . 0 . 96 ) , but the onset phase is not properly fit . Moreover , although not evident from the plot , the optimized time shift was negative ( -120 ms ) , implying that the onset of the vascular response precedes the neuronal activation . All the optimization methods tested above are deterministic , meaning that repeating them with the same initial parameters will bring the same result . The pitfall of these methods when applied to ill-posed problem is that optimization process will get attracted to the nearest local minimum , regardless of the many other deeper minima , which may be far away in the parameters space . In other words , deterministic algorithms are sensitive to the initial parameters set before starting the optimization . Non-deterministic algorithms exist to overcome the local minimum issue , adding some level of randomness in the optimization process , and for this purpose Iliski uses the Simulated Annealing algorithm . Each optimization run can yield a different result , reaching possibly a different cost function\\u2019s minimum each time . We define as \\u2018run\\u2019 a single application of the optimization with a given set of initial values , and \\u2018iteration\\u2019 the ensemble of runs sharing the same initial values . By running the algorithm multiple times , one can choose the result with the lowest residual , while avoiding TFs which shape are biologically not acceptable . In fact , to represent the NVC , a TF cannot start at 0 sec , because of the delay due to the cellular cascades triggering the vascular response , and it must be smooth to comply with the progression of biological processes . In Aydin et al . ( 2020 ) , we described a workflow of runs and iterations to get to biologically consistent TFs ( see Supplementary Fig 1 in Aydin et al . [25] ) . To speed up computation , we imposed bounds over the parameters . Note that such bounds can be set through the Iliski GUI for any constrainable algorithm . Using our data , Simulated Annealing gave a smooth TF and a prediction as good as fminsearch for the onset phase of the vascular response . The data shown in Fig 2 is representative of the rest of the data . In fact , optimization of TFs using neural and vascular recordings from other mice , tested with the same odor stimulation , produced similar residual values of the cost function across the 3 optimization algorithms presented above ( 1-way ANOVA , F ( 2 , 17 ) = 0 . 035 , p = 0 . 97 , Fig 3A ) . However , as in the example of Fig 2 , deterministic algorithms are prone to biologically inconsistent TFs ( Fig 3B ) . The non-deterministic , Simulated Annealing algorithm with subsequent iterations method allows to efficiently exclude these TFs and obtain the best trade-off between prediction performance and biological consistency at the cost of a longer computation time . Direct deconvolution is a good option when the goal is the prediction quality within the training database . Deterministic optimization algorithms are fast but yield to TFs that may have biologically inconsistent dynamics . Note that for all the computations we used a short \\u0394t ( 50 ms ) for interpolation to preserve most of the information . As already mentioned , the Simulated Annealing algorithm requires several runs and iterations to obtain a good TF , where \\u2018run\\u2019 means a single optimization and \\u2018iteration\\u2019 an ensemble of runs sharing the same initial values of the fitting function . In our experience , starting the optimization with a \\u2018bad\\u2019 TF\\u2014whose shape is different from what is expected for the processed dataset\\u2014helps to collect more local minima in a pool of optimization runs . For example , in our previous study [25] , we proposed iterations of 50 runs and started with the initial values of the standard TF ( one \\u0393 HRF ) which , peaking at 5 seconds , turned to be much slower than any of the optimized TFs . The sequence of 50-runs iterations stopped when , within an iteration , no clear improvement was found in the optimized TF [25] . On average , 2 iterations were sufficient to get a stable TF with Pearson coefficient above 0 . 9 . Here , we investigated if a higher number of runs is beneficial to the detection of the minimum of the cost function and if it prevents the need for further iterations . We compared 50 and 200 runs with single and double iterations , in cascade ( Fig 4A ) . In a mouse dataset , we observed a non-significant trend towards more scattered TFs shapes for computation using 50 runs versus 200 runs ( 1-way ANOVA , F ( 3 , 16 ) = 2 . 086 , p = 0 . 14 , Fig 4B ) . Similarly , the quality of the TFs did not significantly improve with increasing runs ( 1-way ANOVA , F ( 3 , 16 ) = 2 . 299 , p = 0 . 12 ) . As a result , TFs with fast dynamics ( peaking within 1 and 2 sec ) , was a common feature independently of the adopted protocol ( Fig 4C ) . In a dataset from another mouse ( Fig 4D and 4E ) , TFs with sparse time to peak values after 200 runs improved after a second iteration , with the same number of runs ( 2 . 3 \\u00b1 0 . 3 s VS 1 . 5 \\u00b1 0 . 1 s ( mean \\u00b1 SEM ) , two-tailed T-test , unpaired , p = 0 . 02 < 0 . 05 ) . Note that this compression of TF dynamics was not accompanied by a significant improvement of the TF quality ( residuals: 10 . 1 \\u00b1 2 . 1 vs . 6 . 9 \\u00b1 0 . 8 ( mean \\u00b1 SEM ) for 200 and 200 + 200 runs respectively , two-tailed T-test , unpaired , p = 0 . 19 ) . To conclude , depending on the input/output signals , non-deterministic algorithms can produce TFs with different dynamics but close performances in the prediction . The choice of a specific optimization process , with more or less iterations and runs , becomes crucial when the interest is not limited to the prediction quality , but extends to the temporal dynamics of the TF . Because of the noise , TFs with distinct shapes can yield very close residual values . We provide a decision diagram to choose the best approach to compute a TF based on the features of the user\\u2019s dataset ( Fig 5 ) . Nonetheless , we believe it is always a good choice to test different approaches before making the final choice . \\n Iliski is open-source and freely available under the Creative Commons Attribution 4 . 0 International ( CC BY 4 . 0 ) license . Iliski is maintained on GitLab , enabling user-friendly bug report and community work to make the tool fit the users\\u2019 need . It can be found here: https://github . com/alike-aydin/Iliski . In the neurovascular imaging field , computing the hemodynamic response function is paramount to interpreting vascular activation in terms of neural activation . In any other field , computing TFs may be of help to go deeper in the interpretation of the results . For these reasons , we think it is extremely helpful to have a data analysis tool which lets fast testing of different algorithms with a user-friendly interface .\",\n          \"In order to adjust to changing environmental conditions , bacteria use nucleotide second messengers to transduce external signals and translate them into a specific cellular response . Cyclic di-adenosine monophosphate ( c-di-AMP ) is the only known essential nucleotide second messenger . In addition to the well-established role of this second messenger in the control of potassium homeostasis , we observed that glutamate is as toxic as potassium for a c-di-AMP-free strain of the Gram-positive model bacterium Bacillus subtilis . In this work , we isolated suppressor mutants that allow growth of a c-di-AMP-free strain under these toxic conditions . Characterization of glutamate resistant suppressors revealed that they contain pairs of mutations , in most cases affecting glutamate and potassium homeostasis . Among these mutations , several independent mutations affected a novel glutamate transporter , AimA ( Amino acid importer A , formerly YbeC ) . This protein is the major transporter for glutamate and serine in B . subtilis . Unexpectedly , some of the isolated suppressor mutants could suppress glutamate toxicity by a combination of mutations that affect phospholipid biosynthesis and a specific gain-of-function mutation of a mechanosensitive channel of small conductance ( YfkC ) resulting in the acquisition of a device for glutamate export . Cultivation of the c-di-AMP-free strain on complex medium was an even greater challenge because the amounts of potassium , glutamate , and other osmolytes are substantially higher than in minimal medium . Suppressor mutants viable on complex medium could only be isolated under anaerobic conditions if one of the two c-di-AMP receptor proteins , DarA or DarB , was absent . Also on complex medium , potassium and osmolyte toxicity are the major bottlenecks for the growth of B . subtilis in the absence of c-di-AMP . Our results indicate that the essentiality of c-di-AMP in B . subtilis is caused by the global impact of the second messenger nucleotide on different aspects of cellular physiology . \\n To achieve appropriate responses to changing environmental conditions , all organisms use second messengers which are formed in response to an environmental cue . In bacteria , specific mono- and dinucleotides play a particularly important role as second messengers [1] . The common nucleotide second messengers cyclic AMP , cyclic di-GMP ( c-di-GMP ) , and guanosine tetra/pentaphosphate ( ( p ) ppGpp ) are used to prioritize the utilization of carbon sources and to regulate virulence , to control the choice between sessile and motile lifestyles , and to downregulate essential cellular functions upon amino acid starvation , respectively [2\\u20137] . The more recently discovered second messenger cyclic di-AMP is unique in several respects . First , this nucleotide is essential in many bacteria that produce it , suggesting that this molecule plays a role in central cellular activities . Essentiality of c-di-AMP has been observed in many Firmicutes , such as Bacillus subtilis , Listeria monocytogenes , Streptococcus agalactiae , and Staphylococcus aureus , but also in the genome-reduced bacterium Mycoplasma pneumoniae and in Chlamydia trachomatis [8\\u201314] . Second , c-di-AMP is toxic if it accumulates in the cells , and in many bacteria this accumulation results in strongly increased stress sensititvity [9 , 15\\u201319] . Third , c-di-AMP can bind to proteins and the mRNA molecules encoding them to control both the expression and the activity of certain proteins , as in the case of the B . subtilis potassium transporters KtrAB and KimA [20\\u201323] . Last , but not least , c-di-AMP is not intrinsically present in eukaryotic organisms , and the presence of the molecule is an indication for infection with pathogenic bacteria . Indeed , c-di-AMP is recognized by the human immune system . It binds the STING protein thus resulting the induction of an innate immune response [10] . The essentiality and toxicity of c-di-AMP for many bacteria have induced a great interest in understanding the reasons behind it . Classically , genes were regarded to be essential if they were absolutely needed for growth under optimal growth conditions , typically on complex medium at 37\\u00b0C [24\\u201326] . However , many genes are not described as being essential because there may be paralogues that encode very similar functions as has been described for the c-di-AMP producing diadenylate cyclases in B . subtilis [8 , 9] . In some cases , one essential function can be carried out by very different proteins , and such genes are essential only under specific conditions , i . e . the absence of amino acids from the medium makes the corresponding biosynthetic genes essential . These genes have been called conditional essential genes . Finally , some genes may not be essential but their loss would cause a substantial decrease of fitness for the cells , resulting either in the disappearance of such mutants or in the acquisition of suppressor mutations . Such genes or proteins have been designated quasi-essential , and the effect of their loss has been studied for the B . subtilis DNA topoisomerase TopA , the endoribonuclease RNase Y , and the modulator of lipid biosynthesis , YqhY [27\\u201329] . Very recently , essential processes have been ranked on a global scale for the first time by the determination of the speed of death of mutants [30] . Two lines of research have provided important insights into the reasons of c-di-AMP essentiality . Studies to identify target molecules that interact with c-di-AMP demonstrated that many proteins involved in potassium and osmolyte homeostasis\\u2013both transporters and their regulators\\u2013are controlled by c-di-AMP to limit the intracellular concentrations of potassium and osmotically compatible compounds to physiologically reasonable levels [11 , 13 , 23 , 31\\u201338] . In a complementary approach , suppressor mutants of bacterial strains that either lack or accumulate c-di-AMP have been studied . These analyses revealed that c-di-AMP is conditionally essential in L . monocytogenes , S . aureus and B . subtilis . While the nucleotide is required for growth on complex media , it is dispensable on minimal media [21 , 39 , 40] . For a B . subtilis mutant lacking c-di-AMP , growth was only possible on minimal medium as long as the potassium concentration in the medium is low and glutamate is not present [21 , 41] . On complex medium , the lack of c-di-AMP leads to an accumulation of ( p ) ppGpp [39] , the second messenger that results in a global reprogramming of transcription , translation , and metabolism upon amino acid starvation . Inactivation of the enzymes that produce ( p ) ppGpp restores the viability of the strain lacking c-di-AMP on complex medium . The accumulation of ( p ) ppGpp leads to the inability of the transcription factor CodY to bind to its targets and thus in deregulation of a large regulon that includes many genes for osmolyte uptake and amino acid metabolism , and this uncontrolled expression of the CodY regulon is responsible for the essentiality of c-di-AMP for L . monocytogenes on complex media [39] . In S . aureus , c-di-AMP is dispensable for growth on complex medium under anaerobic conditions or when the bacteria acquire suppressor mutations that affect osmolyte or glutamine uptake [40 , 42] . Both types of studies support the idea that the control of potassium and osmolyte homeostasis is the central essential function of c-di-AMP in many bacteria that produce it [43\\u201345] . We are interested in c-di-AMP signaling in the model organism B . subtilis . In contrast to S . aureus or L . monocytogenes , B . subtilis encodes three diadenylate cyclases that produce c-di-AMP , the constitutively expressed enzymes CdaA and DisA , and the sporulation-specific enzyme CdaS [9 , 46] . Mutants lacking both the cdaA and disA genes that encode the constitutive enzymes are not viable on complex medium and tolerate only very low potassium concentrations and no glutamate in minimal medium . In B . subtilis , c-di-AMP binds to two signal transduction proteins , DarA and DarB , to several potassium , osmolyte and magnesium transporters , as well as to the riboswitch that controls the expression of the high affinity potassium transporters KtrAB and KimA [20 , 23 , 47] . The DarA protein is a member of the large PII superfamily of signal transduction proteins that modulate the acitivities of transporters , enzymes , and transcription factors upon ligand binding [47 , 48] . However , the precise function of DarA has not yet been elucidated . DarB consists of two so-called CBS domains . Under conditions of potassium starvation , DarB is present as an apo-protein and then binds the ( p ) ppGpp synthetase/ hydrolase Rel to trigger an increase of the ( p ) ppGpp concentration , and thus to switch off cellular activities [49 , 50] . B . subtilis , encodes three potassium uptake systems , the paralogous KtrAB and KtrCD channels , and the KimA transporter as well as three potassium exporters , KhtTU , CpaA , and YugO [51] . With the exception of the sporulation protein YugO , c-di-AMP binds to all proteins involved in potassium homeostasis , and inhibits and stimulates the activities of importers and exporters , respectively [23] . To get a better understanding of the global role of c-di-AMP for the physiology of B . subtilis , we performed extensive suppressor analysis of strains lacking c-di-AMP that had adapted to tolerate the presence of glutamate or that are viable on complex medium . These suppressor analyses were complemented by a transcriptome analysis to understand the roles of glutamate and potassium as well as of c-di-AMP for global gene expression in B . subtilis . Our results indicate that the adaptation to either glutamate or complex medium usually requires the simultaneous presence of multiple suppressor mutations that often affect potassium uptake , osmolyte uptake , or glutamate metabolism . Moreover , a mechanosensitive channel and phospholipid biosynthesis play an important role in the sensitivity of a strain lacking c-di-AMP to glutamate . Strikingly , this study identified the amino acid transporter YbeC ( renamed AimA , amino acid importer A ) as the major glutamate transporter in B . subtilis . On complex medium , the c-di-AMP-binding signal transduction proteins DarA and DarB seem to be toxic in the absence of c-di-AMP , and only in their absence we could isolate suppressor mutants . The obtained results shed light on the complexicity of c-di-AMP essentiality and open new directions of research . \\n A B . subtilis strain lacking all three diadenylate cyclases is unable to grow at potassium concentrations of 5 mM or higher or in the presence of glutamate ( even at low potassium concentrations ) [21 , 41] . To test the sensitivity of the mutant for glutamate in more detail , we compared the growth of wild typ strains and the c-di-AMP-free strain GP2222 ( designated \\u0394dac from here on ) at increasing glutamate concentrations ( see S1 Fig ) . While the wild type strain grew better in the presence of glutamate than with ammonium as the single source of nitrogen , the \\u0394dac mutant was inhibited already at very low glutamate concentrations ( 0 . 05% ) . In order to get more comprehensive insights into the toxicity of glutamate for the \\u0394dac mutant , we decided to isolate and study a larger set of suppressor mutants that tolerate the presence of glutamate . For this purpose , we plated the \\u0394dac strain [21] on MSSM medium containing glutamate ( 1% w/v ) and a low concentration of potassium ( 0 . 1 mM KCl ) . While suppressor mutants appear readily after overnight cultivation when potassium toxicity was the growth-limiting problem [21] , the toxicity of glutamate could only be overcome after several days . Moreover , the number of suppressor mutants was strongly reduced as compared to selection in the presence of potassium and ammonium as the nitrogen source . After four days at 37\\u00b0C , 14 colonies could be isolated ( see Fig 1 , \\u0394dac ) . These strains were subjected to whole genome sequencing to identify the responsible mutations ( Table 1 ) . The long time for suppressor mutations to appear can be explained by each mutant harboring multiple mutations . In addition , this explains why the overall number of suppressors was rather low . As observed previously [41] , three strains were affected in ktrC . In one of these strains ( GP3079 ) , the genomic region from abh to ktrC was deleted resulting in reduced uptake of potassium via KtrCD . Similarly , strain GP3078 has a mutation affecting the high-affinity potassium transporter KtrB , and GP2842 also carries a mutation ( nhaK ) that affects potassium homeostasis . Interestingly , the same mutation in the NhaK cation:proton antiporter NhaK ( S187F ) had been isolated in a suppressor screen of the \\u0394dac strain in the presence of 5 mM potassium [21] indicating that the mutant protein has an increased activity in potassium export . As mentioned above , the suppressor strains usually had multiple mutations . Since we never experienced the systematic acquisition of multiple mutations in prior suppressor screens [15 , 27 , 52\\u201354] , it seems likely that more than one mutation is required to achieve the suppression . Since the selective pressure in our experiment was directed towards tolerating otherwise toxic glutamate , we expected that the suppressor mutants would contain mutations affecting glutamate metabolism . Indeed , the \\u00e1 priori cryptic glutamate dehydrogenase was decryptified by the deletion of nine base pairs in two mutants ( GP2841 and GP3473 ) , thus allowing glutamate degradation [55 , 56] . Moreover , seven independently isolated mutants carried mutations affecting an amino acid transporter , YbeC . This protein has recently been identified as the major serine transporter of B . subtilis [57] , and it is tempting to speculate that YbeC is the major glutamate transporter as well ( see below ) . However , seven of the isolated mutants did not carry any mutation that is easily linked to glutamate homeostasis . The previously reported suppressor mutant GP2840 had mutations affecting PlsC , NdhF , and SigD ( acyl-ACP:1-acylglycerolphosphate acyltransferase , NADH dehydrogenase , Sigma factor \\u03c3D , respectively ) in addition to the ktrC mutation [41] . Strikingly , plsC and ndhF are affected in six and four different suppressor mutants respectively . Several mutants carried mutations affecting motility: In addition to two sigD mutants , seven additional mutants carried mutations in flagellar genes that on one hand depend on a functional SigD for their expression , and on the other hand are required for SigD activity [58] . In addition , three of the 14 different independent suppressor mutants carry the same point mutation in the yfkC gene , encoding a mechanosensitive channel that releases ions from the cell [59 , 60] . This mutation results in a substitution of Arg-42 by a Trp residue . We then tested the ability of the suppressor mutants to grow on minimal medium at low and high potassium concentrations with ammonium or glutamate as the nitrogen source . Moreover , the ability to grow on a complex medium was tested ( see Table 1 and S2 Fig ) . Since the mutants were isolated in the presence of glutamate and 0 . 1 mM potassium , it is not surprising that all strains grew on the corresponding medium . Moreover , most of the strains were viable with ammonium as the single nitrogen source irrespective of the potassium concentration . Most of the mutants were also capable of growing at the elevated potassium concentration in the presence of glutamate . However , only four of the 15 tested strains were viable on complex medium . Two of them grew well on LB and these strains had the identical combination of mutations in yfkC and plsC ( see Table 1 , see below ) . This suggests that the adaptation to complex medium requires more than resistance to potassium and glutamate . Most of the suppressor mutants that tolerate the presence of glutamate and potassium were unable to grow on complex medium under standard laboratory conditions . The standard definition of essentiality refers to optimal growth conditions , i . e . to a complex medium and growth at 37\\u00b0C in the presence of oxygen for B . subtilis [25 , 26] . This prompted us to investigate the growth requirements of the \\u0394dac mutant GP2222 more rigorously . First , we attempted to isolate suppressor mutants derived from GP2222 under standard conditions as defined above . However , we never succeeded in isolating a single suppressor mutant . A recent study on c-di-AMP essentiality in S . aureus revealed that the nucleotide is dispensable under anaerobic conditions [40] . Therefore , we tried to isolate suppressor mutants of GP2222 in the absence of oxygen . While the wild type strain B . subtilis 168 grew under these conditions , again no suppressor mutants of the \\u0394dac strain could be isolated ( see Fig 2 ) . In B . subtilis , c-di-AMP binds two signal transduction proteins , DarA and DarB [23 , 47] . In a suppressor screen using a L . monocytogenes strain lacking c-di-AMP , several mutations in the corresponding genes were observed [39] . We therefore constructed strains that lack both c-di-AMP and either the darA or the darB gene , and used these strains to isolate suppressors on complex medium under aerobic and anaerobic conditions . Again , no suppressors could be obtained under aerobic conditions; however , we detected one and two suppressors for the \\u0394dac \\u0394darA and \\u0394dac \\u0394darB mutants , respectively , under anaerobic conditions ( see Fig 2 ) . Whole genome sequencing revealed that the mutation present in the \\u0394dac \\u0394darA mutant changed Gly-11 of KtrC to a cysteine residue . The suppressor mutants derived from the \\u0394dac \\u0394darB mutant had a frameshift in ktrC resulting in the expression of a truncated KtrC protein or a mutation affecting the permease domain OpuAB of the OpuA glycine betaine transporter [61] , respectively ( Fig 2 ) . These results strongly support the idea that the accumulation of potassium or of osmoprotective compatible solutes is the reason for the essentiality of c-di-AMP [39 , 40 , 43] . An initial characterization of these suppressor mutants revealed that they were viable under anaerobic conditions , but not in the presence of oxygen . Therefore , we used the three strains for a further round of suppressor isolation . Indeed , we were able to obtain suppressors that tolerate the presence of oxygen for all three mutants ( Figs 2 and S3 ) . Whole-genome sequencing revealed that the initial suppressors with defective KtrC proteins had all acquired very specific mutations in the kimA gene , encoding the major high-affinity potassium transporter KimA . In the mutant isolated from the darA ktrC mutant , the absolutely conserved glycine in transmembrane helix 3 of KimA was deleted . Helix 3 is essential for potassium binding of KimA [62] . All four independent suppressors isolated from the \\u0394dac \\u0394darB ktrC mutant had a deletion of 11 amino acids in the transmembrane helix 9 . Based on the high conservation of the deleted residues , we conclude that the mutant KimA proteins were inactive or less active than the wild type protein . Again , the kimA suppressor mutations were accompanied by other mutations ( see Fig 2 ) . In the suppressor mutant GP3010 derived from the \\u0394dac \\u0394darA ktrC mutant , we found a 155 bp deletion that results in the truncation the global transcription regulator for catabolic genes and operons , CcpA , and in the conversion of the start codon of motP , encoding the Na+-coupled MotP flagellar stator protein to a stop codon . For the \\u0394dac \\u0394darB ktrC suppressor mutant , two aerobic suppressors were subjected to whole genome sequencing . In one strain ( GP2845 ) , we found a point mutation in the darA gene that results in a substitution of the conserved Val-100 residue by an alanine in the DarA protein . The second suppressor ( GP3094 ) carried mutations affecting the CodY transcription factor ( G146V substitution ) and the unknown SP\\u03b2 phage protein YonD ( D255H ) . In two additional suppressor mutants , we found identical frameshift mutations in darA that result in a truncation of the DarA protein after amino acid 64 . Finally , we also characterized an aerobic suppressor ( GP2847 ) derived from the anaerobic \\u0394dac \\u0394darB opuAB suppressor mutant . In this case , again a potassium channel subunit ( KtrB ) was inactivated as the result of a frameshift mutation . Moreover , this strain carried a frameshift mutation in the essential accA gene encoding a subunit of acetyl-CoA carboxylase that resulted in the formation of a truncation after amino acid 108 and mutations in the genes encoding the putative methylenomycin A exporter Mmr and the tRNA methylthiotransferase YmcB . These results again highlight the importance of potassium homeostasis for the adaptation of strains lacking c-di-AMP . Taken together , a B . subtilis strain lacking c-di-AMP is only able to grow under standard laboratory conditions if it acquires several suppressing mutations . The results of the two suppressor screens in the presence of glutamate and directly on complex medium demonstrate a distinct selective pressure under both conditions resulting in different sets of mutations that allow aerobic growth on complex medium . The importance of the initial loss of the c-di-AMP receptor proteins DarA and DarB for the direct adaptation to complex medium demonstrates that these proteins interfere with the growth of B . subtilis if they are present in the ligand-free apo-state . Indeed , apo-DarB was recently shown to trigger growth-inhibitory ( p ) ppGpp synthesis by the Rel protein [49 , 50] . For DarA , this is the first time that a phenotype for the corresponding mutant is reported . The repeated isolation of mutations affecting potassium uptake even in iterative cycles of suppressor screens highlights the severe toxicity of potassium for B . subtilis in the absence of c-di-AMP . In order to obtain deeper insights into the physiological role of c-di-AMP , we decided to study the impact of a lack of this second messenger on global gene expression by a transcriptome analysis as a complementary approach to the suppressor analysis . Since c-di-AMP has been implicated in the interplay of potassium and glutamate homeostasis , we analysed the global gene expression for the wild type strain B . subtilis 168 at low ( 0 . 1 mM ) and high ( 5 mM ) potassium concentrations and in the presence of ammonium and glutamate as the nitrogen source ( S1 Text ) . Moreover , we used the c-di-AMP free strain ( \\u0394dac ) GP2222 as well as the isogenic suppressor mutant GP2223 that is able to grow at 5 mM potassium ( see Fig 3 for the regulatory network and S1 Table ) . Strikingly , expression of several genes involved in fermentation and respiration was strongly reduced in the absence of c-di-AMP ( see S1 Table and S4A Fig for validation by quantitative RT-PCR ) . The ldh-lctP and cydABCD operons encoding lactate dehydrogenase and the lactate exporter as well as the terminal quinole oxidase , are repressed by the NADH-responsive transcription factor Rex [63 , 64] . The about 100-fold repression of these genes in the absence of c-di-AMP suggests that the NADH levels are reduced in the mutant . This conclusion is also supported by the increased expression ( seven-fold ) of the ndhF-ybcC operon ( see S1 Table ) which codes for a NADH dehydrogenase and thus also contributes to the oxidation of NADH2 to NAD [65] . Importantly , the ndhF gene and its downstream gene ybcC were affected in several of the suppressor mutants suggesting that these mutations counteract the increased expression of the conserved ndhF-ybcC operon . In addition to genes of the Rex regulon , genes of the SigO regulon that are involved in the response to acid stress [66] and several competence genes were severely reduced in the \\u0394dac mutant GP2222 . The strong reduction of competence gene expression in response to the absence of c-di-AMP suggests that this second messenger might be required for genetic competence and transformation in B . subtilis . To test this hypothesis , we attempted transformation of strain GP2222 using chromosomal DNA . No transformants were obtained , whereas the control strain 168 was efficiently transformable . Thus , the lack of c-di-AMP results in a loss of genetic competence as a result of reduced competence gene expression . It has been shown before that the expression of the high affinity potassium uptake systems KtrAB and KimA is repressed in the presence of potassium via a c-di-AMP sensitive riboswitch [20\\u201322] . In agreement with those results , these genes were most strongly repressed by potassium in our analysis in the wild type strain 168 . The expression of the kimA transporter gene was 112- and 17-fold repressed at 5 mM potassium in the presence of glutamate or ammonium , respectively; whereas the expression of the ktrAB channel operon was reduced 29- and 4-fold , respectively ( see S2 Table ) . As expected , the expression of both kimA and ktrAB is increased in the absence of c-di-AMP; however , both genes are still repressed by potassium ( eight- and five-fold , respectively ) , indicating a second , c-di-AMP independent mechanism of transcriptional regulation of these potassium transport systems ( see S1 Table and S4B Fig for validation by quantitative RT-PCR , see S12 Fig for a reporter analysis ) . The particular importance of potassium for the cell and for KimA as a major high-affinity transporter is highlighted by the fact that the kimA gene belongs to the most strongly expressed genes at 0 . 1 mM potassium ( position #50 in the presence of glutamate ) , and that it is even the most strongly expressed transporter in B . subtilis under these conditions ( see https://www . ncbi . nlm . nih . gov/geo/query/acc . cgi ? acc=GSE156738 for the full data set ) . Taken together , the global transcription analysis supported the idea that c-di-AMP is involved in multiple cellular functions . Moreover , the analysis confirmed a particular role of genes involved in controlling NADH homeostasis , as already suggested by the suppressor analysis and by the differential effects of aerobic and anaerobic conditions to allow suppressor formation of the strains lacking c-di-AMP . The analysis of suppressor mutants that were viable in the presence of glutamate or on complex medium identified several mutations that affect proteins involved in phospholipid biosynthesis , i . e . AccA , AccC , and PlsC ( see Figs 1 , 2 and S5 and Table 1 for an overview on lipid synthesis in B . subtilis ) . The accA and accC genes encode subunits of the essential acetyl-CoA carboxylase . Unexpectedly , a frameshift had occurred in the accA gene of strain GP2847 . Since malonyl-CoA synthesis is the essential first step in fatty acid and phospholipid biosynthesis , it is likely that this mutation can be intrinsically suppressed and thus does not result in a complete loss of AccA synthesis . The accA mutation isolated in the \\u0394dac yfkC* mutant results in an A229V substitution in the active site of the enzyme [67 , 68] ( see S6 Fig ) . For AccC , the mutation ( H298Y ) is located in the highly conserved active site motif of the enzyme ( S6 Fig ) . The localization of these mutations strongly suggests that the initial step of lipid biosynthesis , the formation of malonyl-CoA , is severely impaired in the suppressor mutants . PlsC is the acyl-ACP:1-acylglycerolphosphate acyltransferase that catalyzes the formation of phosphatidic acid in the synthesis of phospholipids . Interestingly , the plsC gene is essential for the growth of B . subtilis . In agreement with this essentiality , none of the three different mutations in the six independent mutants results in a frame shift or a deletion . Instead , residue L48 is replaced by a proline in three independently isolated mutants , A61 by a valine in two mutants , and P110 by serine in one mutant . All three residues are located in functionally important regions of the protein . L48 is close to the HXXXXD motif , which is part the active site of the enzyme . Similarly , A61 and P110 are in very close proximity to the active center of the enzyme [69] ( S7 Fig ) . The location of all three mutations in the immediate vicinity of the active center as well as the fact that the mutations affect conserved residues suggest that they result in a reduced PlsC activity , and thus in impaired phospholipid biosynthesis . Three of the suppressor mutants isolated in the presence of glutamate carry mutations affecting the mechanosensitive channel of small conductance , YfkC ( see Table 1 ) . Strikingly , all three independently isolated mutants carry precisely the same mutation that results in a R42W substitution in the protein ( see S8 Fig ) . This residue is located at the end of the first trans-membrane helix [70] and the mutation may affect the relative positioning of the trans-membrane helices 1 and 2 . Another striking feature is that this mutation is in all three cases accompanied by point mutations in the plsC gene ( see below ) . The presence of the same amino acid substitution in YfkC in all three strains suggested that this mutation resulted in a gain of function rather than in a loss of YfkC activity ( from here on , we designate the mutant allele yfkC* , the protein YfkC* ) . To address this question , we constructed \\u0394dac \\u0394yfkC and \\u0394dac yfkC* strains lacking all diadenylate cyclases and assayed the growth of these strains on minimal medium containing glutamate or ammonium as the nitrogen source at high and low potassium concentrations ( see Fig 4A ) . Both strains were viable when cultivated on plates containing ammonium and 0 . 1 mM potassium whereas they were unable to grow in the presence of glutamate or at the high potassium concentration . These observations indicate that neither the deletion of the yfkC gene nor yfkC* mutation alone were sufficient to allow suppression of the glutamate and potassium toxicity of the \\u0394dac mutant . To address this issue further , we attempted the isolation of suppressor mutants from the \\u0394dac strain either lacking yfkC or carrying the yfkC* allele in the presence of glutamate . No suppressor mutants could be isolated from the \\u0394dac \\u0394yfkC mutant , whereas two suppressor mutants were obtained from the \\u0394dac yfkC* mutant ( see Fig 1 ) . This observation supports the idea that the specific R42W substitution of YfkC* is a prerequisite for the adaptation of the c-di-AMP-free strain to growth in the presence of glutamate . Whole genome sequencing of the two glutamate-resistant suppressor mutants identified a mutation in accA in strain GP2850 ( see below ) . This mutation supports the requirement of paired mutations affecting yfkC and lipid biosynthesis . The second suppressor strain , GP3105 , carries mutations affecting the amino acid transporter YbeC , the CarB subunit of the carbamoyl-phosphate synthetase , the endonuclease MutL and the polyketide synthase PksJ . Among these mutations , the frameshift in the ybeC gene is likely the most relevant ( see below ) . These suppressor mutants were also tested for their growth in the presence of glutamate or increased potassium concentrations ( see Fig 4A ) . Both were viable in the presence of glutamate irrespective of the potassium concentration . However , the growth in the presence of glutamate was much better in the case of the yfkC* plsC ( L48P ) mutant GP2837 ( Fig 4A ) . Taken together , these results suggest that the yfkC* mutation in combination with the lipid biosynthetic or ybeC mutations is rather specific for the protection of the \\u0394dac mutant from glutamate toxicity . In the growth assay , we also included the \\u0394dac nhaK ( S187F ) mutant that has been isolated for resistance to potassium [21] . As expected , this strain grew at 5 mM potassium but was as sensitive to glutamate as the original \\u0394dac mutant , however we observed the formation of suppressor mutants in the presence of glutamate . As the NhaK allele used in this experiment is thought to exhibit increased potassium export activity [21] , it is tempting to speculate the mutant mechanosensitive channel YfkC* is able to export glutamate to overcome the toxicity of this amino acid . The requirement of the \\u0394dac yfkC* mutant for an additional mutation reducing lipid biosynthesis to allow growth in the presence of glutamate in several of the suppressor mutants suggests that the membrane properties are important for controlling cellular glutamate homeostasis . This conclusion is supported by the well-established functional interaction between mechanosensitive channels and lipid molecules [71 , 72] . It is tempting to speculate that the combination of altered lipid composition and the mutation in the trans-membrane helix of YfkC results in a change of channel properties that facilitates the export of glutamate by YfkC . To test this hypothesis , we determined the intracellular and extracellular glutamate concentrations in the wild type strain 168 , and in the isogenic \\u0394yfkC , yfkC* and yfkC* plsC* mutants ( Fig 4B and 4C ) . The wild type strain contained about 2 . 8 \\u03bcg glutamate per OD600 and ml , and the same glutamate concentration was detected in the \\u0394yfkC mutant . This is in good agreement with the observation that the deletion of the yfkC gene did not confer glutamate resistance to the \\u0394dac mutant . In contrast , the intracellular glutamate concentrations were reduced in the yfkC* mutant ( 1 . 7 \\u03bcg/OD600 and ml ) and even more so in the yfkC* plsC* double mutant ( 1 . 4 \\u03bcg/OD600 and ml ) . On the other hand , the glutamate concentration in the supernatant of the wild type strain and the \\u0394yfkC mutant was low ( 1 \\u03bcg/OD600 and ml ) , and a moderate increase to 1 . 3 \\u03bcg/OD600 and ml was observed in the presence of the yfkC* mutation . The extracellular glutamate concentration was most elevated in the yfkC* plsC* double mutant ( 2 . 3 \\u03bcg/OD600 and ml ) . These results support the idea that the point mutation in YfkC* is crucial for lowering the intracellular glutamate concentration by facilitating export of the amino acid . In agreement with previous findings on the functional interactions between mechanosensitive channels and lipid molecules [71 , 72] , the reduced lipid biosynthesis due to the PlsC* mutation further enhances glutamate export via YfkC* . The acquisition of suppressor mutations that provide resistance to glutamate to the \\u0394dac mutant was a rare event that took several days to occur ( see above ) . However , we observed the rapid appearance of glutamate-resistant suppressor mutants if the strain already carried a nhaK mutation that facilitates potassium export . To determine the requirements for glutamate resistance in more detail , we isolated a set of 16 new suppressor mutants ( see Fig 1 ) . Since the originally isolated suppressor mutant GP2842 carried a combination of mutations in nhaK and the ybeC gene encoding an amino acid transporter , we sequenced the ybeC allele in all glutamate-resistant suppressor mutants isolated from the \\u0394dac nhaK ( S187F ) ( from now on referred to as nhaK* and NhaK* for the gene and protein , respectively ) . Strikingly , all 16 strains carried mutations in ybeC . Among these mutations are frameshifts in ten strains , an insertion of 2 amino acids after residue 226 in one strain , and amino acid substitutions in the remaining five strains ( for details , see Table 2 ) . The high number of mutants with inactivated ybeC genes strongly suggests that the combination of NhaK* and loss of YbeC allows growth of the \\u0394dac mutant in the presence of glutamate . To test this hypothesis , we deleted the ybeC gene in the \\u0394dac mutant . The resulting strain , GP3054 , was unable to grow in the presence of glutamate , indicating that only the combination of the nhaK* and the ybeC mutations conferred resistance to glutamate to the \\u0394dac mutant . Three lines of evidence suggest that YbeC is the major glutamate transporter of B . subtilis: ( i ) We isolated seven independent mutants affected in ybeC in the initial screen for mutations conferring glutamate resistance to the \\u0394dac strain , ( ii ) we reproducibly isolated a large number of ybeC mutants based on the \\u0394dac nhaK* mutant , and ( iii ) YbeC is an amino acid transporter of the amino acid-polyamine-organocation ( APC ) superfamily . We have recently demonstrated that YbeC is the major serine transporter of B . subtilis [57] . It is well established that many amino acid transporters can take up multiple different amino acids . To test the possible role of YbeC in glutamate uptake , we first attempted functional complementation in E . coli that is unable to grow with glutamate as the single source of carbon and nitrogen due to the low activity of intrinsic glutamate transporters [73] . For this purpose , we transformed E . coli JM109 with plasmid pGP2987 that carries the ybeC gene or with the empty vector pWH844 [74] . The resulting transformants were tested for growth on minimal medium with glucose and ammonium or with glutamate as the only source of carbon and nitrogen ( see Fig 5A ) . Whereas both strains grew well with glucose and ammonium , the strain carrying the empty vector was unable to utilize glutamate . In contrast , the expression of the ybeC gene allowed E . coli to use glutamate as source of carbon and energy . These findings confirm that YbeC indeed is a glutamate transporter . The protein and the gene were therefore renamed AimA and aimA , respectively ( amino acid importer A ) . It is very striking that our suppressor screens for glutamate-resistant mutants often identified AimA but never GltT or GltP , the two described glutamate transporters of B . subtilis . Therefore , we analyzed the contributions of these proteins to glutamate uptake by comparing the growth of strains expressing only one of the three proteins and a strain lacking all of them ( see Fig 5B ) . All tested strains grew well in the presence of ammonium as the nitrogen source , reflecting their ability to synthesize glutamate . The strain GP3071 lacking all three presumptive glutamate transporters was only viable at high glutamate concentrations but not at 1 mM or 5 mM . This demonstrates that the bacteria need to transport glutamate if no ammonium is available for its synthesis . Moreover , it demonstrates that efficient glutamate transport is not possible in the absence of the three proteins . GP2818 expressing GltT grew at the three tested glutamate concentrations confirming its role as glutamate transporter . The strain GP2825 that expressed AimA as the only glutamate transporter did not grow at the lowest tested glutamate concentration ( 1 mM ) , but this strain was able to grow at 5 mM and 20 mM glutamate suggesting that AimA might be a low-affinity glutamate transporter . In contrast , the strain expressing GltP was only viable at highest glutamate concentration as observed for strain GP3071 lacking all three proteins . This suggests that GltP plays only a very minor , if any role , in glutamate uptake , and that yet another protein can transport glutamate at high concentrations . To study the activities of AimA and GltT in glutamate transport in more detail , we assayed the growth of strains expressing only one of these transporters at different glutamate concentrations ( see Fig 5C and 5D ) . Moreover , we performed this experiment in the presence of 0 . 1 mM and 5 mM KCl in the medium to see whether potassium has an effect on glutamate uptake as it has been observed with an activating effect of glutamate on potassium uptake by KtrCD [41] . As control , we used strain GP3071 that lacks AimA , GltT , and GltP . The results were used to calculate apparent KS values that reflect the affinities of the transporters for glutamate . For GltT , we observed KS values of 0 . 3 mM and 0 . 06 mM in the presence of 0 . 1 and 5 mM potassium , respectively . These results are in good agreement with the description of GltT as a high-affinity glutamate transporter [75] . Moreover , they show that the affinity of GltT for glutamate is five-fold increased in the presence of high potassium concentrations . For AimA , we determined apparent KS values of 3 . 2 and 2 . 3 mM in the presence of 0 . 1 and 5 mM potassium , respectively . In good agreement with the drop dilution assay , this suggests that AimA is a low-affinity glutamate transporter , and that the affinity of AimA is not affected by the external potassium concentration . The strain GP3071 lacking all known and suspected glutamate transporters grew only at high glutamate concentrations . This suggests the presence of an additional transporter with very low affinity for glutamate ( apparent KS values of 12 . 3 and 13 . 8 mM in the presence of 0 . 1 and 5 mM potassium , respectively ) . Thus , AimA is the major glutamate transporter under standard growth conditions when glutamate is not limiting in the medium . The role of AimA as the major glutamate transporter is in excellent agreement with the common isolation of glutamate-resistant \\u0394dac aimA mutants whereas no gltT mutants could be isolated . Indeed , the \\u0394dac \\u0394gltT mutant was unable to grow in the presence of glutamate , but again a suppressor mutant , GP3093 , could be isolated ( see Fig 1 ) . Not surprisingly , this strain carries a mutation in aimA ( Leu-419 Pro ) . Moreover , this strain carries frame-shift mutations in ndhF and fliK . The paired occurrence of aimA and ndhF suppressor mutations has been observed several times independently ( see Table 1 ) . To get further insights into gltP , gltT and aimA , we determined the expression of these genes based on the transcriptome analysis under the relevant conditions ( see Fig 5E ) . The expression of gltP was very low under all tested conditions , which is in agreement with the observation that GltP does not play a significant role in glutamate uptake . The expression of both GltT and AimA was significant . In the presence of glutamate in the medium , the expression was reduced five- and two-fold for AimA and GltT , respectively , at the high potassium concentration ( 5 mM ) . In the absence of glutamate , aimA expression did not respond to the supply of potassium whereas the expression of gltT was about two-fold increased at the increased potassium concentration . Thus , if glutamate is limiting , both the expression of gltT and the affinity of the GltT protein for glutamate are increased by potassium . To make the picture of the requirements for a \\u0394dac mutant to tolerate glutamate complete , we also adapted a \\u0394dac \\u0394aimA mutant to the presence of glutamate . As described above , this mutant was not viable in the presence of glutamate; however , we could isolate three glutamate-resistant suppressor mutants ( see Fig 1 ) . The mutations in these strains were identified by whole genome sequencing . Interestingly , all three suppressor mutants had mutations affecting the essential translation elongation factor G , in two strains ( GP3101 and GP3103 ) the highly conserved Ala-579 was replaced by a Val ( see S9 Fig ) . Ala-579 is located in immediate vicinity of a loop ( Loop II ) that is required for tRNA translocation in the ribosome [76 , 77] , suggesting that the mutation interferes with translation . In the third strain ( GP3102 ) we found an A21T substitution . This residue is the immediate neighbour of Asp-20 , which binds the Mg2+ ion in the active site of the protein [78] . The presence of substitutions in highly conserved and functionally important regions of FusA in all three suppressors suggests that reduced translation efficiency helps the \\u0394dac \\u0394aimA mutant to adapt to the presence of glutamate . In addition to the mutations in fusA each strain had a distinct additional mutation . Strain GP3101 had a frameshift mutation in the yqeG gene encoding a putative HAD superfamily phosphatase [79] , whereas the two other strains carried point mutations affecting potassium homeostasis ( nhaK and ktrC for GP3102 and GP3103 , respectively ) again highlighting the toxicity of potassium in the presence of glutamate . Moreover , GP3102 carried amino acid substitutions affecting SigA , the housekeeping sigma factor ( F281V ) and MrpA ( P397L ) , a subunit of the sodium exporter of B . subtilis , and GP3103 had a point mutation affecting the quasi-essential DNA topoisomerase 1 TopA ( K13N ) . The analysis of these strains for growth in the presence of glutamate with 0 . 1 or 5 mM potassium revealed that all suppressors were viable at both potassium concentrations , however , the suppression was most efficient if the yqeG or nhaK genes were affected in addition to aimA ( see S10 Fig ) . \\n Cyclic di-AMP is essential f or many bacteria that produce this second messenger; however , the reasons behind this essentiality have not been fully understood . In this study , we used the Gram-positive model organism B . subtilis to gain further insights into the critical functions of c-di-AMP . Based on a variety of distinct suppressor screens to adapt a B . subtilis strain lacking c-di-AMP to either the presence of glutamate or to growth on complex medium , we can conclude that the control of potassium homeostasis is the major bottleneck that limits growth of the mutant under both conditions . This is in good agreement with ( i ) the large variety of c-di-AMP target proteins and RNA molecules that are involved in the uptake and export of potassium in B . subtilis and other bacteria [11 , 13 , 20 , 21 , 23 , 31 , 34 , 45 , 80] , ( ii ) the fact that the intracellular c-di-AMP levels seem to report the extracellular potassium concentrations [18 , 21 , 37] , and the isolation of suppressor mutants affecting potassium homeostasis in response to altered cellular c-di-AMP levels also in other bacteria [18 , 37] . However , our study also clearly demonstrates that potassium is not the only problem for growth of B . subtilis in the absence of c-di-AMP since the presence of mutations that reduce potassium uptake or facilitate its export is not sufficient to overcome the toxicity of glutamate or complex medium . Suppressor studies with L . monocytogenes and S . aureus mutants lacking c-di-AMP revealed that those mutants often had acquired mutations in transporters for osmoprotectants such as glycine betaine indicating that intoxication by compatible solutes is the main reason for essentiality of c-di-AMP in these latter bacteria . The idea of a major role for c-di-AMP in osmoprotection rather than in potassium homeostasis in these pathogenic bacteria is also supported by the observation that potassium has only a minor impact on a L . monocytogenes strain lacking the only diadenylate cyclase [80] . Thus , while general features of c-di-AMP signaling are conserved in different bacteria , the specific roles and mechanisms may differ even between closely related bacteria . Potassium and glutamate are the most abundant cation and anion , respectively , in every living cell , and the intracellular concentrations of the two ions need to be balanced [81] . In B . subtilis , c-di-AMP seems to play an important role in achieving this balance . The concentrations of the nucleotide respond both to the potassium and the nitrogen source availability , they are high at high potassium concentrations and in the presence of glutamate but not ammonium or glutamine [15 , 21 , 40] . Our work supports the idea of a close physiological integration of potassium and glutamate homeostasis: Both potassium and glutamate inhibit growth of the \\u0394dac mutant independently of each other . The accumulation of amino acids has also been shown to be a problem for the \\u0394dac mutants of L . monocytogenes and S . aureus . As presented in this study , in both organisms amino acid transporters were often inactivated in suppressor mutants that were capable of growing on complex medium despite the lack of c-di-AMP: In L . monocytogenes , several suppressor mutants carried mutations inactivating the OppABCD oligopeptide transporter [39] whereas the characterization of S . aureus suppressor mutants resulted in the identification of the AlsT glutamine transporter , thus indicating that glutamine rather than glutamate is toxic for the S . aureus \\u0394dac mutant [39 , 42] . Again , these differences between the related organisms support the idea that c-di-AMP has global overarching functions in the different bacteria , but these can be put into practice differently . It had been established previously that the accumulation of glutamate can be toxic for B . subtilis [56 , 82] . In the case of the \\u0394dac mutant , our work has identified two essential requirements , i . e . the reduction of the intracellular glutamate and potassium concentrations ( see Fig 6 for an overview ) . Since glutamate stimulates potassium uptake by the KtrCD channel [41] we can conclude that the increased potaasium uptake is the major reason for the toxicity of glutamate for the \\u0394dac mutant . Our work has unraveled three ways to reduce the intracellular glutamate concentrations: ( i ) Glutamate can be efficiently degraded upon decryptification of the otherwise inactive gudB gene encoding the major constitutive glutamate dehydrogenase of B . subtilis . Such a well established gudB+ mutation [55] was found in the suppressor mutants GP2841 and GP3473 . ( ii ) The uptake of glutamate can be prevented , and this was indeed observed in many suppressor mutants . Surprisingly , the recently characterized serine transporter AimA ( YbeC ) was identified as the main glutamate transporter rather than the previsously studied GltT [75] . Even the \\u0394dac mutant lacking gltT was unable to grow in the presence of glutamate unless AimA had been inactivated ( see Fig 1 ) . This observation as well as our detailed kinetic analyses revealed that AimA is the major low affinity transporter for glutamate . Moreover , in the absence of c-di-AMP , the aimA gene is the most strongly expressed gene encoding a glutamate transporter ( see Fig 5E ) . It is interesting to note that the expression of aimA responds to the availability of c-di-AMP ( see Fig 5B ) , however , the mechanism responsible for this control has yet to be uncovered . We have previously demonstrated that a potassium channel ( KtrCD ) is directly activated and thus converted from a low- to a high-affinity uptake system in the presence of glutamate [41] . Here , we show that similarly the affinity of the high affinity glutamate transporter GltT is controlled by the presence of potassium: At high potassium concentrations , the affinity of GltT for glutamate is increased fivefold . Thus , the uptake systems for the two most abundant ions in the cell can respond to the presence of the other ion to keep their intracellular concentrations balanced . ( iii ) In addition to mutations activating glutamate degradation or preventing glutamate uptake , we found several mutants with paired mutations affecting lipid biosynthesis and the mechanosensitive channel of small conductivity YfkC . Moreover , the \\u0394dac yfkC* mutant expressing the altered channel protein had to acquire additional mutations affecting either glutamate uptake ( aimA ) or lipid biosynthesis ( accA ) to tolerate the otherwise toxic glutamate . We suggest that the altered YfkC* protein in conjunction with altered membrane properties facilitates the export of glutamate . This idea is supported by the observation that YfkC needs a gain-of-function mutation to bypass glutamate toxicity . Moreover , MscS-like mechanosensitive channels have been shown to export glutamate in Corynebacterium glutamicum [83 , 84] . In this bacterium , a gain of function mutation affecting this channel results in constitutive glutamate secretion [85] . Finally , specific membrane properties in C . glutamicum such as reduced lipid amounts facilitate glutamate export by the MscS-like channel ( see [86] for review ) . Our data that demonstrate reduced intracellular and increased extracellular glutamate concentrations in the yfkC* plsC* double mutant provides strong evidence for the idea that the combination of mutations reducing lipid biosynthesis and that affect the mechanosensitive channel YfkC allow glutamate export in B . subtilis , thus overcoming the toxic effect of this amino acid . Three lines of evidence suggest that respiration is a major bottleneck for B . subtilis in the absence of c-di-AMP . First , many suppressors isolated in the presence of glutamate had mutations that inactivate the ndhF gene encoding a subunit of a putative NADH dehydrogenase or the downstream ybcC gene of unknown function . It is tempting to speculate that the two proteins form a complex that catalyzes the oxidation of NADH2 . Second , when we used complex medium to isolate suppressor mutants , this was initially only possible in the absence of oxygen if respiration activity was low . This is paralleled by the observation that the S . aureus \\u0394dac is viable in the absence but not in the presence of oxygen [40] . Finally , the lack of c-di-AMP results in an enhanced expression of the ndhF-ybcC operon and reduced intracellular NADH2 levels as indicated by the strong repression of the genes of the Rex regulon in the \\u0394dac mutant . Although it is not yet understood how precisely respiration interferes with the growth of the \\u0394dac mutant , it is tempting to speculate that the proton motive force generated by respiration plays a key role: the major potassium transporter KimA is actually a proton:potassium symporter [62] , and potassium uptake by the KtrAB and KtrCD channels is directly driven by the proton motive force [87] . Thus , a reduced respiration lowers the proton motive force and may thus limit the uptake of potassium , the major growth-limiting factor for the B . subtilis \\u0394dac mutant . The adaptation of the \\u0394dac mutant to complex medium confirmed the toxicity of potassium . Surprisingly , none of the suppressors isolated on complex medium carried mutations specifically related to glutamate . The acquisition of suppressor mutations that allow growth under anaerobic conditions on complex medium was only possible when one of the c-di-AMP binding signal transduction proteins DarA or DarB was absent ( see Fig 2 ) . This indicates that these proteins inhibit growth in their c-di-AMP-free apo form . It is possible that DarA and/or DarB have functions related to glutamate metabolism , and that the deletions of either of the corresponding genes already covers the aspect of glutamate homeostasis . For DarB it has very recently been shown that the apo protein stimulates the synthesis of ( p ) ppGpp by the alarmone synthetase Rel both in B . subtilis and L . monocytogenes [49 , 50] , and excessive ( p ) ppGpp synthesis limits growth of both organisms [39 , 49 , 50] . For DarA , no function has been identified so far . It is interesting to note , however , that DarA seems to be even more inhibitory than DarB since the \\u0394dac \\u0394darB mutants tend to acquire additional mutations affecting DarA , suggesting that DarA might be involved in the control of glutamate metabolism . Taken together , our study supports the idea that c-di-AMP plays a central role in the physiology of B . subtilis . Moreover , by comparing the results obtained with suppressor screens in closely related but distinct bacteria , it becomes clear that the overall themes of c-di-AMP signaling are similar in different bacteria whereas the precise mechanisms are not . While some of the functions of c-di-AMP in B . subtilis such as the control of potassium homeostasis are already well understood , there remain many open questions for future research such as the link between c-di-AMP and lipid biosynthesis and the function of the PII-like signal transduction protein DarA . The work described here provides a solid ground for further investigation . \\n E . coli DH5\\u03b1 [88] was used for cloning and for the expression of recombinant proteins . E . coli JM109 ( Promega ) was used for glutamate transporter complementation assays . All B . subtilis strains used in this study are derivatives of the laboratory strain 168 . They are listed in Table 2 . B . subtilis was grown in Luria-Bertani ( LB ) or in sporulation ( SP ) medium [88 , 89] . For the assay of potassium and glutamate toxicity , B . subtilis was cultivated in MSSM medium [21] . In this modified SM medium KH2PO4 was replaced by NaH2PO4 and KCl was added as indicated . The media were supplemented with ampicillin ( 100 \\u03bcg/ ml ) , kanamycin ( 10 and 50 \\u03bcg/ ml for B . subtilis and E . coli , respectively ) , chloramphenicol ( 5 \\u03bcg/ ml ) , tetracyclin ( 12 . 5 \\u03bcg/ ml ) , spectinomycin ( 150 \\u03bcg/ ml ) or erythromycin and lincomycin ( 2 and 25 \\u03bcg/ ml , respectively ) if required . For identification of suppressor mutations that rescue the growth of the c-di-AMP lacking strain GP2222 , a suppressor screen was set up . The same set-up was performed for the suppressor screens with GP2223 , GP2814 , GP3054 , and GP2259 ( see Fig 1 ) . The respective strain was plated on MSSM minimal plates in the presence of ammonium and 0 . 1 mM KCl . After incubation for 2\\u20133 days , cell material was scraped from the plates and washed three times in MSSM basal salts solution . These cells were either plated on MSSM minimal plates ( Na-Glutamate , 0 . 1 mM KCl ) or used for inoculation of 10 ml MSSM minimal medium ( Na-Glutamate , 0 . 1 mM KCl ) . The bacteria were then incubated at 42\\u00b0C for 3\\u20134 days . As soon as suppressors appeared on the plates , they were picked and re-streaked three times . For liquid cultures , as far as growth could be observed , they were plated . Suppressors were picked and re-isolated again . For the suppressor screen on complex medium , GP2222 , GP2420 , and GP2779 were plated on LB plates . The plates were incubated at 37\\u00b0C either under standard conditions ( aerobic ) or in an anaerobic jar to create an anaerobic environment . As soon as suppressors appeared , they were picked and re-isolated three times . For the two step adaptation to complex medium , suppressor mutants previously isolated in the anaerobic screen were plated on LB plates and incubated at 37\\u00b0C under aerobic conditions . As soon as new suppressors appeared , they were picked and re-isolated three times . All isolated mutants were analyzed by whole-genome sequencing ( see below ) . Amylase activity was detected on plates containing nutrient broth ( 7 . 5 g/liter ) , 17 g of Bacto agar per liter ( Difco ) , and 5 g of hydrolyzed starch per liter ( Connaught ) . The ability to degrade starch was detected by sublimating iodine onto the plates . For quantitative determination of lacZ expression in B . subtilis , cells were grown in MSSM medium with the nitrogen source and potassium concentration as indicated and harvested in the exponential phase . \\u03b2-Galactosidase activity was determined with cell extracts obtained by lysozyme treatment as described previously [89] . One unit of \\u03b2-galactosidase is defined as the amount of enzyme that produces 1 nmol of \\u03bf-nitrophenol per minute at 28\\u00b0C . Experiments were performed in triplicate . To assay growth of B . subtilis mutants at different potassium concentrations , the bacteria were inoculated in LB medium and precultured in MSSM medium with 0 . 1 mM KCl . The cultures were grown until exponential phase , harvested , washed three times in MSSM basal salts solution before the OD600 was adjusted to 1 . 0 . Dilution series were then pipetted onto MSSM plates with ammonium or glutamate and 0 . 1 or 5 mM potassium . Complementation in E . coli was assayed by transformation of strain JM109 that is unable to grow with glutamate as the single source of carbon and nitrogen . For this purpose , the strain was transformed with the complementation plasmid pGP2987 ( aimA ) and the empty vector control pWH844 . Growth of the transformants was assessed on M9 minimal medium plates containing either ammonium and glucose or glutamate as carbon and nitrogen source [73] . Transformation of E . coli and plasmid DNA extraction were performed using standard procedures [88] . All commercially available plasmids , restriction enzymes , T4 DNA ligase and DNA polymerases were used as recommended by the manufacturers . Chromosomal DNA of B . subtilis was isolated as described [89] . B . subtilis was transformed with plasmid and genomic DNA according to the two-step protocol [89] . To identify the mutations in the suppressor mutants , their genomic DNA was subjected to whole-genome sequencing [92] . Briefly , the reads were mapped on the reference genome of B . subtilis 168 ( GenBank accession number: NC_000964 ) [93] using the Geneious software package ( Biomatters Ltd . , New Zealand ) [94] . Single nucleotide polymorphisms were considered as significant when the total coverage depth exceeded 25 reads with a variant frequency of \\u226590% . All identified mutations were verified by PCR amplification and Sanger sequencing . Gene deletion was achieved by transformation of B . subtilis 168 with a PCR product constructed using oligonucleotides to amplify DNA fragments flanking the target genes and an appropriate intervening resistance cassette as described previously [95] . The mutation yfkC-R42W was introduced by amplification of regions containing and flanking the mutation from strain GP2837 , respectively , and a kanamycin resistance cassette was introduced downstream of yfkC in the non-coding region between yfkC and yfkD ( terminator region of both genes present ) . The integrity of the regions flanking the integrated resistance cassette was verified by sequencing PCR products of about 1 , 000 bp amplified from chromosomal DNA of the resulting mutant strains . Construction of a reporter fusion of the kimA promoter region with the promoterless lacZ gene was done using the plasmid pAC6 [96] . The promoter fragment was cloned between the EcoRI and BamHI sites of pAC6 . The resulting plasmid was pGP2912 . The plasmid was linearized with ScaI and transformed into B . subtilis 168 for chromosomal integration into the amyE locus . For E . coli complementation assays , the aimA gene was amplified using the primer pair LK167 ( 5\\u2019 AAAGGATCCATGAATCAATTGCATCGAAGAATGGGAACGTTTT ) / AK324 ( 5\\u2019 TTTGTCGACTTATTCTTTTCCGGCAGCAGCTTCTG ) and cloned between the BamHI and SalI sites of the expression vector pWH844 [74] . The resulting plasmid was pGP2987 . The growth characteristics of B . subtilis glutamate transporter mutants were used to determine the single glutamate transporter kinetics . The mutants were either positive for gltT ( \\u0394gltP::cat \\u0394aimA::kan ) or aimA ( \\u0394gltT::kan \\u0394gltP::cat ) or lacked both ( \\u0394gltT::kan \\u0394aimA::phleo \\u0394gltP::cat ) . The nitrogen source was used as the limiting factor , so that transport of glutamate was the essential factor for growth . The bacteria were inoculated in LB medium and precultured in MSSM medium ( ammonium ) with 0 . 1/5 mM KCl . The cultures were grown until exponential phase and the cells were washed three times in MSSM basal salts solution . Afterwards , the cells were adjusted to OD600 1 . 0 and used to inoculate a 96 well plate ( Microtest Plate 96 Well , F , Sarstedt ) containing the MSSM medium with the respective glutamate and potassium concentrations . The growth was tracked in an Epoch 2 Microplate Spectrophotometer ( BioTek Instruments ) at 37\\u00b0C with linear shaking at 237 cpm ( 4 mm ) for 20 h , and optical density at 600 nm was measured in 10 min intervals . The exponential growth phase was used to determine the growth rate \\u03bc [h-1] . \\u03bc was calculated from t1 ( early exponential phase ) and t2 ( late exponential phase ) and the corresponding OD1 and OD2 values . The resulting growth rates were then plotted against the glutamate concentrations . This allowed fitting to the Monod equation and calculation of Vmax ( h-1 ) and the apparent KS ( mM glutamate ) using the solver tool of Excel 2012 ( Microsoft ) . Experiments were repeated with three biological replicates . Bacterial cells were cultivated in MSSM minimal medium with ammonium and 0 . 1 mM KCl until late exponential growth phase . 4 ml of each culture were harvested by filtration as described previously [97] . The filter was washed three times with 0 . 9% NaCl and transferred to 4 ml extraction solution ( 60% ethanol containing the internal 15N L-glutamic acid standard ) and stored over night at -20\\u00b0C . The extraction of metabolites and the derivatization was performed as described previously [92] . For the analysis of the medium , the cells were cultivated in the same medium until late exponential growth phase and harvested by centrifugation ( 8 , 500 rpm , 10 min ) . 100 \\u03bcl of the supernatant were used for further analysis . 15N L-glutamic acid standard was added and the sample evaporated by nitrogen stream . The sample was dissolved in 15 \\u03bcl Acetonitrile and 15 \\u03bcl of N-methyl-N- ( trimethylsilyl ) trifluoroacetamide was added for derivatization . Analysis by GC-MS was performed with previously used equipment and settings [98] . Wild type and the c-di-AMP-deficient strains GP2222 \\u0394dac and GP2223 \\u0394dac nhaK-S187F were grown in MSSM medium with ammonium or glutamate and low ( 0 . 1 mM ) or high ( 5 mM ) potassium concentrations . The strains were harvested by centrifugation ( 10 . 397 \\u00d7 g , 1 min , 4\\u00b0C ) at mid exponential phase ( OD600 of 0 . 4\\u20130 . 6 ) . A total of two independent biological replicates were included . The pellets were frozen immediately in liquid nitrogen and stored at \\u221280\\u00b0C . Isolation of the total RNA was done as described previously [99 , 100] , and the quality of the RNA preparations was assessed by means of an Agilent 2100 Bioanalyzer according to the manufacturer\\u2019s instructions . 5 \\u03bcg of total RNA were used for cDNA synthesis . The synthesis and fluorescence labeling of cDNA followed a strand-specific method using the FairPlay III Microarray Labeling Kit ( Agilent Technologies , Santa Clara , CA , USA ) and actinomycin D ( Calbiochem ) [101] . 200 ng of Cy3-labeled cDNA were hybridized to the microarray following Agilent\\u2019s hybridization , washing and scanning protocol ( One-Color Microarray-based Gene Expression Analysis , version 5 . 5 ) . Data were extracted and processed using the Feature Extraction software ( version 11 . 5 . 1 . 1 ) . An aggregated expression value was computed for each annotated coding sequence and previously identified RNA feature [100] . Gene-level intensities were scaled based on the intensity values of ten different in vitro-synthesized transcripts contained in the One Color RNA Spike-In kit ( Agilent Technologies ) in order to account for technical variation associated with sample processing . Genes with at least 2 . 5-fold difference in expression levels of the wild type at the different conditions or between wild type and GP2222 mutant were considered significantly affected . The graphical presentation of the results was generated using GraphPad Prism version 8 . 0 . 0 for Windows , GraphPad Software , San Diego , California USA , www . graphpad . com ( see S11 Fig ) . To analyze the regulatory network , we compared the expression levels of the wild type strain and the GP2222 mutant at ammonium 0 . 1 mM KCl . The genes were clustered in operons according to their genetic localization , as described in SubtiWiki [51] . Additionally , gathered operons were grouped by their corresponding documented regulator/s and metabolism . The visual representation of the regulatory network was generated with Cytoscape and GIMP softwares [102 , 103] . For RNA isolation , the cells were grown in MSSM minimal medium containing ammonium or glutamate , and 0 . 1 mM or 5 mM KCl to an OD600 of 0 . 4 to 0 . 6 and harvested . Preparation of total RNA was carried out as described previously [104] . cDNAs were synthesized using a One-Step reverse transcription-PCR ( RT-PCR ) kit ( Bio-Rad ) [95] . Quantitative RT-PCR ( qRT-PCR ) was carried out on an iCycler instrument ( Bio-Rad ) following the manufacturer's instructions . The primers used for amplification are listed in S3 Table . The alaS and gyrA genes were used as internal controls , as their expression has been shown to be stable [105] . Data analysis and the calculation of expression ratios as fold changes were performed as described previously [95] . qRT-PCR experiments were performed in triplicates .\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"headings\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"keywords\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"id\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 142,\n        \"samples\": [\n          \"journal.pcbi.1010477\",\n          \"journal.pcbi.1008614\",\n          \"journal.pgen.1009092\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"biomistral_summary\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 142,\n        \"samples\": [\n          \" The GNN model is a powerful tool for extracting rules from complex data. In this study, we used the GNN model to extract rules from the spatiotemporal data of the basal layer of the hind paw and ear epidermis. We first constructed the spatiotemporal graphs from the imaging data of the hind paw and ear epidermis. We then trained the GNN model to predict the cell fate of the target cell based on the features of the neighboring cells. We found that the GNN model can predict the cell fate of the target cell with high accuracy, and that the attribution score of the GNN model is high for the cell fate of the neighboring cells. We also found that the GNN model can extract the rules of the cell fate coordination from the spatiotemporal data. We tested three different rules of cell fate coordination, and found that the GNN model can extract the rules of the cell fate coordination with high accuracy. These results suggest that the GNN model is a powerful tool for extracting rules from complex spatiotemporal data.\",\n          \" The article describes Iliski, an open-source software package for transfer function (TF) computation. TFs are used to model the relationship between two time series, such as neural activity and blood flow. The software package provides a graphical user interface (GUI) and a set of functions for deconvolution and optimization of TFs. The GUI allows users to input their data and choose the appropriate settings for TF computation. The software package also provides a set of functions for data pre-processing, such as smoothing and median filtering. The software package is written in MATLAB and is available for download on GitLab. The article provides an example of how to use the software package to compute TFs for neural activity and blood flow data from a mouse. The article also provides a decision diagram to help users choose the best approach for TF computation based on the features of their data.\",\n          \" The study of Bacillus subtilis mutants that are able to grow in the presence of glutamate and high potassium concentrations has revealed several interesting aspects of the physiology of this bacterium. First, the study has shown that the c-di-AMP signaling pathway plays a central role in the adaptation of B. subtilis to high potassium concentrations. Second, the study has identified several genes that are involved in the adaptation of B. subtilis to high potassium concentrations, including the genes encoding the mechanosensitive channel MscK, the potassium transporter KtrA, and the glutamate transporter AimA. Third, the study has shown that the c-di-AMP signaling pathway is also involved in the regulation of the expression of genes that are involved in the adaptation of B. subtilis to high potassium concentrations. Finally, the study has identified several suppressor mutations that are able to rescue the growth defect of the c-di-AMP-deficient mutant GP2222 in the presence of glutamate and high potassium concentrations. These mutations include mutations in the genes encoding the mechanosensitive channel MscK, the potassium transporter KtrA, and the glutamate transporter AimA. Overall, the study has provided important insights into the physiology of B. subtilis and has identified several key genes and pathways that are involved in the adaptation of this bacterium to high potassium concentrations.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}",
       "type": "dataframe",
       "variable_name": "test_plos_df"
      },
      "text/html": [
       "\n",
       "  <div id=\"df-d5476887-5c96-47c8-9da3-03ea08c7a497\" class=\"colab-df-container\">\n",
       "    <div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>article</th>\n",
       "      <th>headings</th>\n",
       "      <th>keywords</th>\n",
       "      <th>id</th>\n",
       "      <th>biomistral_summary</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Lung-resident ( LR ) mesenchymal stem and stro...</td>\n",
       "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
       "      <td>[immune system, medical conditions, molecular ...</td>\n",
       "      <td>journal.ppat.1009789</td>\n",
       "      <td>The study aimed to investigate the role of lu...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Visceral leishmaniasis ( VL ) is endemic in So...</td>\n",
       "      <td>[Abstract, Introduction, Methods, Results, Dis...</td>\n",
       "      <td>[neonates, clinical laboratory sciences, trans...</td>\n",
       "      <td>journal.pntd.0007992</td>\n",
       "      <td>The study presents data on 133 pregnant women...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>A high burden of Salmonella enterica subspecie...</td>\n",
       "      <td>[Abstract, Introduction, Methods, Results, Dis...</td>\n",
       "      <td>[pathogens, medical conditions, taxonomy, bact...</td>\n",
       "      <td>journal.pntd.0010704</td>\n",
       "      <td>This study provides valuable insights into th...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Severe Acute Respiratory Syndrome Coronavirus-...</td>\n",
       "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
       "      <td>[pathogens, amniotes, medical conditions, bind...</td>\n",
       "      <td>journal.ppat.1010691</td>\n",
       "      <td>The S2-Fc-murine antibody was generated by fu...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Many fungal species utilize hydroxyderivatives...</td>\n",
       "      <td>[Abstract, Introduction, Results and discussio...</td>\n",
       "      <td>[taxonomy, proteins, chemistry, genetics, enzy...</td>\n",
       "      <td>journal.pgen.1009815</td>\n",
       "      <td>The C . parapsilicus genome was sequenced usi...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>\n",
       "    <div class=\"colab-df-buttons\">\n",
       "\n",
       "  <div class=\"colab-df-container\">\n",
       "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-d5476887-5c96-47c8-9da3-03ea08c7a497')\"\n",
       "            title=\"Convert this dataframe to an interactive table.\"\n",
       "            style=\"display:none;\">\n",
       "\n",
       "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
       "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
       "  </svg>\n",
       "    </button>\n",
       "\n",
       "  <style>\n",
       "    .colab-df-container {\n",
       "      display:flex;\n",
       "      gap: 12px;\n",
       "    }\n",
       "\n",
       "    .colab-df-convert {\n",
       "      background-color: #E8F0FE;\n",
       "      border: none;\n",
       "      border-radius: 50%;\n",
       "      cursor: pointer;\n",
       "      display: none;\n",
       "      fill: #1967D2;\n",
       "      height: 32px;\n",
       "      padding: 0 0 0 0;\n",
       "      width: 32px;\n",
       "    }\n",
       "\n",
       "    .colab-df-convert:hover {\n",
       "      background-color: #E2EBFA;\n",
       "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
       "      fill: #174EA6;\n",
       "    }\n",
       "\n",
       "    .colab-df-buttons div {\n",
       "      margin-bottom: 4px;\n",
       "    }\n",
       "\n",
       "    [theme=dark] .colab-df-convert {\n",
       "      background-color: #3B4455;\n",
       "      fill: #D2E3FC;\n",
       "    }\n",
       "\n",
       "    [theme=dark] .colab-df-convert:hover {\n",
       "      background-color: #434B5C;\n",
       "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
       "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
       "      fill: #FFFFFF;\n",
       "    }\n",
       "  </style>\n",
       "\n",
       "    <script>\n",
       "      const buttonEl =\n",
       "        document.querySelector('#df-d5476887-5c96-47c8-9da3-03ea08c7a497 button.colab-df-convert');\n",
       "      buttonEl.style.display =\n",
       "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
       "\n",
       "      async function convertToInteractive(key) {\n",
       "        const element = document.querySelector('#df-d5476887-5c96-47c8-9da3-03ea08c7a497');\n",
       "        const dataTable =\n",
       "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
       "                                                    [key], {});\n",
       "        if (!dataTable) return;\n",
       "\n",
       "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
       "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
       "          + ' to learn more about interactive tables.';\n",
       "        element.innerHTML = '';\n",
       "        dataTable['output_type'] = 'display_data';\n",
       "        await google.colab.output.renderOutput(dataTable, element);\n",
       "        const docLink = document.createElement('div');\n",
       "        docLink.innerHTML = docLinkHtml;\n",
       "        element.appendChild(docLink);\n",
       "      }\n",
       "    </script>\n",
       "  </div>\n",
       "\n",
       "\n",
       "<div id=\"df-22f787e7-0e18-4fb5-a464-0150cdee5182\">\n",
       "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-22f787e7-0e18-4fb5-a464-0150cdee5182')\"\n",
       "            title=\"Suggest charts\"\n",
       "            style=\"display:none;\">\n",
       "\n",
       "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
       "     width=\"24px\">\n",
       "    <g>\n",
       "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
       "    </g>\n",
       "</svg>\n",
       "  </button>\n",
       "\n",
       "<style>\n",
       "  .colab-df-quickchart {\n",
       "      --bg-color: #E8F0FE;\n",
       "      --fill-color: #1967D2;\n",
       "      --hover-bg-color: #E2EBFA;\n",
       "      --hover-fill-color: #174EA6;\n",
       "      --disabled-fill-color: #AAA;\n",
       "      --disabled-bg-color: #DDD;\n",
       "  }\n",
       "\n",
       "  [theme=dark] .colab-df-quickchart {\n",
       "      --bg-color: #3B4455;\n",
       "      --fill-color: #D2E3FC;\n",
       "      --hover-bg-color: #434B5C;\n",
       "      --hover-fill-color: #FFFFFF;\n",
       "      --disabled-bg-color: #3B4455;\n",
       "      --disabled-fill-color: #666;\n",
       "  }\n",
       "\n",
       "  .colab-df-quickchart {\n",
       "    background-color: var(--bg-color);\n",
       "    border: none;\n",
       "    border-radius: 50%;\n",
       "    cursor: pointer;\n",
       "    display: none;\n",
       "    fill: var(--fill-color);\n",
       "    height: 32px;\n",
       "    padding: 0;\n",
       "    width: 32px;\n",
       "  }\n",
       "\n",
       "  .colab-df-quickchart:hover {\n",
       "    background-color: var(--hover-bg-color);\n",
       "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
       "    fill: var(--button-hover-fill-color);\n",
       "  }\n",
       "\n",
       "  .colab-df-quickchart-complete:disabled,\n",
       "  .colab-df-quickchart-complete:disabled:hover {\n",
       "    background-color: var(--disabled-bg-color);\n",
       "    fill: var(--disabled-fill-color);\n",
       "    box-shadow: none;\n",
       "  }\n",
       "\n",
       "  .colab-df-spinner {\n",
       "    border: 2px solid var(--fill-color);\n",
       "    border-color: transparent;\n",
       "    border-bottom-color: var(--fill-color);\n",
       "    animation:\n",
       "      spin 1s steps(1) infinite;\n",
       "  }\n",
       "\n",
       "  @keyframes spin {\n",
       "    0% {\n",
       "      border-color: transparent;\n",
       "      border-bottom-color: var(--fill-color);\n",
       "      border-left-color: var(--fill-color);\n",
       "    }\n",
       "    20% {\n",
       "      border-color: transparent;\n",
       "      border-left-color: var(--fill-color);\n",
       "      border-top-color: var(--fill-color);\n",
       "    }\n",
       "    30% {\n",
       "      border-color: transparent;\n",
       "      border-left-color: var(--fill-color);\n",
       "      border-top-color: var(--fill-color);\n",
       "      border-right-color: var(--fill-color);\n",
       "    }\n",
       "    40% {\n",
       "      border-color: transparent;\n",
       "      border-right-color: var(--fill-color);\n",
       "      border-top-color: var(--fill-color);\n",
       "    }\n",
       "    60% {\n",
       "      border-color: transparent;\n",
       "      border-right-color: var(--fill-color);\n",
       "    }\n",
       "    80% {\n",
       "      border-color: transparent;\n",
       "      border-right-color: var(--fill-color);\n",
       "      border-bottom-color: var(--fill-color);\n",
       "    }\n",
       "    90% {\n",
       "      border-color: transparent;\n",
       "      border-bottom-color: var(--fill-color);\n",
       "    }\n",
       "  }\n",
       "</style>\n",
       "\n",
       "  <script>\n",
       "    async function quickchart(key) {\n",
       "      const quickchartButtonEl =\n",
       "        document.querySelector('#' + key + ' button');\n",
       "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
       "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
       "      try {\n",
       "        const charts = await google.colab.kernel.invokeFunction(\n",
       "            'suggestCharts', [key], {});\n",
       "      } catch (error) {\n",
       "        console.error('Error during call to suggestCharts:', error);\n",
       "      }\n",
       "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
       "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
       "    }\n",
       "    (() => {\n",
       "      let quickchartButtonEl =\n",
       "        document.querySelector('#df-22f787e7-0e18-4fb5-a464-0150cdee5182 button');\n",
       "      quickchartButtonEl.style.display =\n",
       "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
       "    })();\n",
       "  </script>\n",
       "</div>\n",
       "\n",
       "    </div>\n",
       "  </div>\n"
      ],
      "text/plain": [
       "                                             article  \\\n",
       "0  Lung-resident ( LR ) mesenchymal stem and stro...   \n",
       "1  Visceral leishmaniasis ( VL ) is endemic in So...   \n",
       "2  A high burden of Salmonella enterica subspecie...   \n",
       "3  Severe Acute Respiratory Syndrome Coronavirus-...   \n",
       "4  Many fungal species utilize hydroxyderivatives...   \n",
       "\n",
       "                                            headings  \\\n",
       "0  [Abstract, Introduction, Results, Discussion, ...   \n",
       "1  [Abstract, Introduction, Methods, Results, Dis...   \n",
       "2  [Abstract, Introduction, Methods, Results, Dis...   \n",
       "3  [Abstract, Introduction, Results, Discussion, ...   \n",
       "4  [Abstract, Introduction, Results and discussio...   \n",
       "\n",
       "                                            keywords                    id  \\\n",
       "0  [immune system, medical conditions, molecular ...  journal.ppat.1009789   \n",
       "1  [neonates, clinical laboratory sciences, trans...  journal.pntd.0007992   \n",
       "2  [pathogens, medical conditions, taxonomy, bact...  journal.pntd.0010704   \n",
       "3  [pathogens, amniotes, medical conditions, bind...  journal.ppat.1010691   \n",
       "4  [taxonomy, proteins, chemistry, genetics, enzy...  journal.pgen.1009815   \n",
       "\n",
       "                                  biomistral_summary  \n",
       "0   The study aimed to investigate the role of lu...  \n",
       "1   The study presents data on 133 pregnant women...  \n",
       "2   This study provides valuable insights into th...  \n",
       "3   The S2-Fc-murine antibody was generated by fu...  \n",
       "4   The C . parapsilicus genome was sequenced usi...  "
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test_plos_df[\"biomistral_summary\"] = test_plos_df[\"article\"].apply(lambda text: lay_summarize(text))\n",
    "test_plos_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/",
     "height": 1000
    },
    "id": "XImGxtk76tqh",
    "outputId": "b101976c-3fd5-4a3b-95b4-aa3075a8044c"
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.12s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.48s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.24s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.17s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.54s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.10s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 12.00s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.15s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.31s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.50s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:05<00:00,  5.79s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.86s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.27s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.46s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.55s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.08s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.28s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.04s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.24s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.24s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.19s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.90s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.20s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.04s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.46s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.17s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.81s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.40s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.41s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:01<00:00,  1.26s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.03s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.47s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.43s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.42s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.67s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.69s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.69s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:06<00:00,  6.62s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.12s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.34s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.29s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.41s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.55s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.23s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.93s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.11s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.92s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.09s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.70s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.78s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.28s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.90s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.57s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.76s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.77s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.65s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.36s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.62s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.74s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.31s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.23s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.16s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.78s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.34s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.57s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.01s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.84s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.68s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.90s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.71s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.84s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.34s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.29s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.81s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.41s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.94s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.59s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.06s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.86s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.60s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.61s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.92s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.90s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.35s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.20s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.52s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.85s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.47s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.19s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.32s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.71s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.54s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.51s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.70s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.86s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.99s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.89s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.62s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.61s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.26s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.56s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.34s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.30s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.13s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.83s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.15s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.87s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.24s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.80s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.94s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.46s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.09s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.11s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.14s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.59s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.55s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.21s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.55s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.49s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.46s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.23s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.52s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:01<00:00,  1.91s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.04s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.88s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.94s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.37s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.81s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.85s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.22s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.97s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:02<00:00,  2.47s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:12<00:00, 12.13s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.21s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.75s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:04<00:00,  4.24s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.13s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.17s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.63s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:03<00:00,  3.64s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.42s/it]\n",
      "Processed prompts: 100%|██████████| 1/1 [00:11<00:00, 11.20s/it]\n"
     ]
    },
    {
     "data": {
      "application/vnd.google.colaboratory.intrinsic+json": {
       "summary": "{\n  \"name\": \"test_elife_df\",\n  \"rows\": 142,\n  \"fields\": [\n    {\n      \"column\": \"article\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 142,\n        \"samples\": [\n          \"The development of multiple chromosome-scale reference genome sequences in many taxonomic groups has yielded a high-resolution view of the patterns and processes of molecular evolution . Nonetheless , leveraging information across multiple genomes remains a significant challenge in nearly all eukaryotic systems . These challenges range from studying the evolution of chromosome structure , to finding candidate genes for quantitative trait loci , to testing hypotheses about speciation and adaptation . Here , we present GENESPACE , which addresses these challenges by integrating conserved gene order and orthology to define the expected physical position of all genes across multiple genomes . We demonstrate this utility by dissecting presence\\u2013absence , copy-number , and structural variation at three levels of biological organization: spanning 300 million years of vertebrate sex chromosome evolution , across the diversity of the Poaceae ( grass ) plant family , and among 26 maize cultivars . The methods to build and visualize syntenic orthology in the GENESPACE R package offer a significant addition to existing gene family and synteny programs , especially in polyploid , outbred , and other complex genomes . \\n De novo genome assemblies and gene model annotations represent increasingly common resources that describe the sequence and positions of protein coding and intergenic regions within a single genotype . Evolutionary relationships among these DNA sequences form the foundation of many molecular tools in modern medical , breeding , and evolutionary biology research . Perhaps the most crucial inference to make when comparing genomes revolves around homologous genes , which share an evolutionary common ancestor and ensuing sequence or protein structure similarity . Analyses of homologs , including comparative gene expression , epigenetics , and sequence evolution , require the distinction between orthologs , which arise from speciation events , and paralogs , which arise from sequence duplications ( see Box 1 for definitions ) . In some systems , this is a simple task where most genes are single copy , and orthologs are synonymous with reciprocal best-scoring protein BLAST hits . Other sequence similarity approaches such as OrthoFinder ( Emms and Kelly , 2019; Emms and Kelly , 2015 ) leverage graphs and gene trees to test for orthology , permitting more robust analyses in systems with gene copy number ( CNV ) or presence\\u2013absence variation ( PAV ) . However , whole-genome duplications ( WGDs ) , chromosomal deletions , and variable rates of sequence evolution , such as subgenome dominance in polyploids , can confound the evidence of orthology from sequence similarity alone . The physical position of homologs offers a second line of evidence that can help overcome challenges posed by WGDs , tandem arrays , heterozygous-duplicated regions , and other genomic complexities ( Drillon et al . , 2020; Haug-Baltzell et al . , 2017; Wang et al . , 2012 ) . Synteny , or the conserved order of DNA sequences among chromosomes that share a common ancestor , is a typical feature of eukaryotic genomes . In some taxa , synteny is preserved across hundreds of millions of years of evolution and is retained over multiple WGDs ( Jiao et al . , 2014; Simakov et al . , 2020; Zhao and Schranz , 2019 ) . Like chromosome-scale synteny , conserved gene order collinearity along local regions of chromosomes can provide evidence of homology , and in some cases enable determination of whether two regions diverged as a result of speciation or a large-scale duplication event ( Drillon et al . , 2020 ) . Combined , evidence of gene collinearity and sequence similarity should improve the ability to classify paralogous and orthologous relationships beyond either approach in isolation . Integrating synteny and collinearity into comparative genomics pipelines also physically anchors the positions of related gene sequences onto the assemblies of each genome . For example , by exploring only syntenic orthologs it is possible to examine all putatively functional variants within a genomic region of interest , even those in genes that are absent in the focal reference genome ( Lovell et al . , 2018 ) . Such a \\u2018pan-genome annotation\\u2019 framework ( Lovell et al . , 2021a ) permits access to multi-genome networks of high-confidence orthologs and paralogs , regardless of ploidy or other complicating aspects of genome biology . Here , we present GENESPACE , an analytical pipeline that explicitly links synteny and sequence similarity to provide high-confidence inference about networks of genes that share a common ancestor and represents these networks as a \\u2018pan-genome annotation\\u2019 . We then leverage this framework to explore gene family evolution in flowering plants , mammals , and reptiles . \\n Comparative genomics across the complex evolutionary histories of eukaryotes typically requires equally varied input and analytical pipelines depending on researchers\\u2019 goals and study systems . For example , synteny between closely related haploid assemblies is often inferred by exploring only 1:1 reciprocal best-scoring hits with MCScanX ( Wang et al . , 2012 ) . Alternatively , polyploid genomes are typically split into subgenomes so that homeologs are viewed by clustering algorithms like OrthoFinder ( Emms and Kelly , 2019; Emms and Kelly , 2015 ) as orthologous and not paralogous . While expert knowledge that informs these analytical decisions can dramatically improve precision , this knowledge is not available in many systems . These issues boil down to a simple circular problem: a priori knowledge of gene copy number is needed to effectively infer orthology and synteny , yet measures of synteny and orthology are needed to infer copy number between a pair of sequences . GENESPACE resolves this circular problem by operating on a foundational assumption: homologs should be exactly single copy within any syntenic region between a pair of genomes . There are two major violations of this assumption that cause copy number variation within a syntenic block: ( 1 ) tandem arrays and ( 2 ) gene PAV . GENESPACE addresses these complexities directly in two ways . First , physically proximate multigene families ( hereon , \\u2018tandem arrays\\u2019 , Box 1 ) are condensed to the physically most central gene of the array . Gene rank order along the genome is recalculated on these \\u2018array representative\\u2019 genes , effectively masking copy number variation due to tandem arrays . Second , synteny is inferred in a pairwise manner only using \\u2018potential anchor\\u2019 protein BLAST hits ( hereon \\u2018hits\\u2019 ) where both the query and target genes are in the same orthogroup . The rank-order positions of these potential anchor genes are also recalculated prior to synteny inference , which effectively masks orthogroups missing a gene in one genome ( i . e . , PAV ) . Thus , GENESPACE operates on OrthoFinder-derived orthogroups with one-to-one relationships for any accurately defined syntenic region , regardless of ploidy or level of sequence conservation . Given GENESPACE\\u2019s reliance on syntenic regions between genomes , errors in syntenic block coordinates can have major effects on downstream estimates . Therefore , we crafted a sensitive pipeline to infer syntenic regions ( see pipeline overview in Figure 1 ) . To demonstrate its functionality , we ran GENESPACE on seven pairs of genomes spanning closely related maize cultivars to ancient mammal\\u2013bird divergence ( ~300 M ya; Table 1 , see Materials and methods ) . The three comparisons between vertebrate genomes have no recent history of WGDs , while WGDs predated evolutionary divergence of the four plant contrasts: maize is a 12 M ya paleo-tetraploid , cotton is a 1 . 6 M ya meso-tetraploid , and the ~70 M ya Rho WGD predated grass diversification . Paralogs derived from these WGDs notoriously obscure contrasts between orthologs in these systems . As such , we treated all genomes as haploid and did not include an outgroup; therefore , orthogroups should not include homeologs since the genomes share a common ancestor that arose after the WGD ( see \\u2018additional considerations\\u2019 methods section: Outgroups and the phylogenetic context of orthology inference ) . In each run , we contrasted GENESPACE-derived orthogroups and syntenic blocks to the defaults from OrthoFinder and MCScanX , respectively ( Table 1 ) . Since it is a common practice to refine hits prior to running MCScanX , we also included syntenic block coverage from MCScanX run on orthogroup-constrained hits where both the target and query genes must be in the same orthogroup . To contrast each approach , we calculated the percent of genes in ( 1 ) orthogroups and ( 2 ) syntenic blocks that were placed on exactly one chromosome per genome ( Table 1 ) . For every pair of genomes , GENESPACE produced a greater percentage of single-copy syntenic blocks and genes in single chromosome orthogroups than either OrthoFinder or MCScanX in isolation . GENESPACE also outperformed simple integration between the two methods through MCScanX on orthogroup-constrained hits . The improved performance of GENESPACE synteny-constrained orthogroups was most subtle between highly diverged haploid animal assemblies . For example , in the comparison between human and chicken , GENESPACE resolved 2% and 1 . 8% more single-copy orthogroups and syntenic blocks than default OrthoFinder and orthogroup-constrained MCScanX , respectively . In contrast , the benefits of GENESPACE were most pronounced in recently diverged genomes with a history of WGDs . Single-copy orthogroups between two meso-tetraploid cotton species that share a WGD that predated speciation by ~1 M ya , were uncommon in the default OrthoFinder run ( 35 . 6% ) but far more prevalent in GENESPACE syntenic orthogroups ( 85 . 7% ) . Similarly , homeologs derived from the cotton WGD impacted estimates of syntenic blocks: only 14% of the genomes were single-copy syntenic in orthogroup-constrained MCScanX but 96 . 2% were single copy in GENESPACE syntenic blocks . Combined , these results demonstrate significant flexibility and utility of GENESPACE across a range of evolutionary histories and divergence . It is important to note that some evolutionary processes , including small-scale translocations , can cause true orthologs to exist outside of syntenic regions . Closely related genomes without a history of WGDs tend to have few non-syntenic orthologs . For example , there are 1096 non-syntenic orthologs ( 6 . 3% of all orthologs ) between human and chimpanzee . In contrast , the 50 M ya diverged Sorghum and Brachypodium genomes have 9002 non-syntenic orthologs , many of which are the result of over-retained Rho WGD-derived paralogs ( see below ) . Since the non-syntenic orthologs can be important in some situations , GENESPACE embraces this complexity by including and flagging non-syntenic orthologs within the pan-genome annotation ( Figure 1 ) . GENESPACE facilitates the exploration and analysis of sequence evolution across multiple genomes within regions of interest ( e . g . , quantitative trait loci [QTL] intervals , see the next section ) . One particularly instructive example comes from the origin and evolution of the mammalian XY and avian ZW sex chromosome systems . To explore these chromosomes , we ran GENESPACE on 15 haploid avian and mammalian genome assemblies , spanning most major clades of birds , placental mammals , monotremes , and marsupials with available chromosome-scale annotated reference genomes ( See Materials and methods ) . We also included two reptile genomes as outgroups to the avian genomes . The heteromorphic chromosomes ( Y and W ) are often unassembled , or , where assemblies exist , lack sufficient synteny to provide a useful metric for comparative genomics . As such , we chose to focus on the homomorphic X and Z chromosomes , which have remained surprisingly intact over the >100 M years of independent mammalian ( Murphy et al . , 1999 ) and avian evolution ( Zhou et al . , 2014; Figure 2 , Figure 2\\u2014figure supplement 1 ) . While the same or similar genomic regions often recurrently evolve into sex chromosomes , perhaps due to ancestral gene functions involved in gonadogenesis , evidence about the nonrandomness of sex chromosome evolution is still contentious ( Kratochv\\u00edl et al . , 2021 ) . Given our analysis , the avian Z chromosome clearly did not evolve from either of the two reptile Z chromosomes sampled here , but instead likely arose from autosomal regions or unsampled ancestral sex chromosomes . The situation in mammals is less clear , in part because both reptile genomes are more closely related to avian than mammalian genomes , which makes ancestral state reconstructions between the two groups less accurate . Nonetheless , the mammalian X and sand lizard Z chromosomes partially share syntenic orthology , an outcome that would be consistent with common descent from a shared ancestral sex chromosome or autosome containing sex-related genes . The shared 91 . 7 M bp region between the human X and sand lizard Z represents 59 . 0% of the human X chromosome genic sequence . The remaining 64 . 0 M bp of human X linked sequences are syntenic with autosomes in the sand lizard and garter snake genomes ( Figure 2 ) . The eutherian mammalian X chromosome is largely composed of two regions , an X-conserved ancestral sex chromosome region that arose in the common ancestor of therian mammals , and an X-added region that arose in the common ancestor of eutherians ( Ross et al . , 2005 ) . Consistent with this evolutionary history , the X chromosome is syntenic across all five eutherian mammals studied here . Further , a 107 . 2 M bp ( 68 . 8% ) segment of the human X , which corresponds with the X-conserved region , is syntenic with 77 . 8 M bp ( 93 . 9% ) of the Tasmanian devil X chromosome and represents the entire syntenic region between the human and all three marsupial X chromosomes ( Figure 2 ) . Similarly , the chicken Z chromosome is retained in its entirety across all five avian genomes . The only notable exception being the budgie Z chromosome , which features a partial fusion between the Z and an otherwise autosomal 19 . 5 M bp segment of chicken chromosome 11 ( Figure 2 ) , potentially representing a neo-sex chromosome fusion that has not yet been described . In contrast to conserved eutherian and avian sex chromosomes , the complex monotreme XnYn sex chromosomes are only partially syntenic between the two sampled genomes . Only the first X chromosomes are ancestral to both echidna and platypus ( Rens et al . , 2007 ) , and all are unrelated to the mammalian X chromosomes ( Figure 2\\u2014figure supplement 1 ) , consistent with their independent evolution ( Rens et al . , 2007 ) . Interestingly , the entirety of the echidna X4 and 47 . 6 M bp ( 67 . 9% ) of the genic region of the platypus X5 chromosomes are syntenic with the avian Z chromosome ( Figure 2 ) . The phylogenetic scale of the genomes presented here precludes evolutionary inference about the origin of these shared sex chromosome sequences; however , the possibility of parallel evolution of sex chromosomes between such diverged lineages may prove an interesting future line of inquiry . The Poaceae grass plant family is one of the best studied lineages of all multicellular eukaryotes and includes experimental model species ( Brachypodium distachyon; Panicum hallii; Setaria viridis ) and many of the most productive ( Zea mays \\u2013 maize/corn; Triticum aestivum \\u2013 wheat , Oryza sativa \\u2013 rice ) and emerging ( Sorghum bicolor \\u2013 sorghum; Panicum virgatum \\u2013 switchgrass ) agricultural crops . Despite the tremendous genetic resources of these and other grasses , genomic comparisons among grasses are difficult , in part because of an ancient polyploid origin ( see the next section ) , and because subsequent WGDs are a feature of most clades of grasses . For example , maize is an 11 . 4 M ya paleo-polyploid ( Gaut and Doebley , 1997 ) , allo-tetraploid switchgrass formed 4\\u20136 M ya ( Lovell et al . , 2021b ) , and allo-hexaploid bread wheat arose about 8 k ya ( Haas et al . , 2019 ) . In some cases , homeologous gene duplications from polyploidy have generated genetic diversity that can be targeted for crop improvement; however , in other cases the genetic basis of trait variation may be restricted to sequences that arose in a single subgenome . Thus , it is crucial to contextualize comparative\\u2013quantitative genomics and explicitly explore only the orthologous or homeologous regions of interest when searching for markers or candidate genes underlying heritable trait variation \\u2014 a significant challenge in the complex and polyploid grass genomes . To help overcome this challenge and provide tools for grass comparative genomics , we conducted a GENESPACE run and built an interactive viewer hosted on Phytozome ( Goodstein et al . , 2012 ) . Owing to its use of within-block orthology and synteny constraints ( Figure 1 ) , GENESPACE is ideally suited to conduct comparisons across species with diverse polyploidy events . Default parameters produced a largely contiguous map of synteny even across notoriously difficult comparisons like the paleo-homeologs between the maize subgenomes ( Figure 3A ) . Furthermore , the sensitive synteny construction pipeline implemented by GENESPACE effectively masks additional paralogous regions like those from the Rho duplication that gave rise to all extant grasses . Breeders and molecular biologists can take two general approaches to understand the genetic basis of complex traits: studying variation caused by a priori-defined genes of interest or determining candidate genes from genomic regions of interest . As an example of the exploration of lists of a priori-defined candidate genes , we analyzed PAV of 86 genes shown to be involved in the transition between C3 and C4 photosynthesis ( Ding et al . , 2015 ) , the latter permitting ecological dominance in arid climates and agricultural productivity under forecasted increased heat load of the next century . To conduct this analysis , we built pan-genome annotations across the seven grasses anchored to C4 maize , which was the genome in which these genes were discovered . This resulted in 159 pan-genome entries: nearly always two placements for each gene in the paleo-tetraploid maize genome . Given that many of these genes were discovered in part because of sequence similarity to genes in Arabidopsis and other diverged plant species , it is not surprising that PAV among C3/C4 genes was lower than the background ( 9 . 7% vs . 38 . 2% , odds ratio = 5 . 7 , p < 1 \\u00d7 10\\u221216; Figure 3B ) . However , these ratios were highly variable among genomes , particularly among the C3 species ( wheat , rice , B . distachyon ) , which had far more absences than the C4 species ( 15 . 3% vs . 5 . 5% , odds ratio = 3 . 1 , p = 6 . 25 \\u00d7 10\\u22128 , Figure 3B ) . This effect is undoubtedly due in part to the increased evolutionary distance between maize and the C3 species compared to the other C4 species . However , when controlling for the elevated level of absent genes globally in C3 species , the effect was still very strong: the odds of C3 species having more of these C3/C4 genes at syntenic pan-genome positions than the background was always lower than the C4 species ( Figure 3C ) . Despite these interesting patterns , given only a single C3/C4 phylogenetic split in this dataset , it is not possible to test evolutionary hypotheses regarding the causes of such PAV . Nonetheless , this result suggests a possible role of gene loss or gain as an evolutionary mechanism for drought- and heat-adapted photosynthesis . Like the exploration of a priori-defined sets of genes , finding candidate genes within QTL intervals usually involves querying a single reference genome and extracting genes with promising annotations or putatively functional polymorphism . In the case of a biparental mapping population genotyped against a single reference , this is a trivial process where genes within physical bounds of a QTL are the candidates . However , many genetic mapping populations now have reference genome sequences for all parents; this offers an opportunity to explore variation among functional alleles and PAV , which would be impossible with a single reference genome . GENESPACE is ideally suited for this type of exploration , and indeed was originally designed to solve this problem between the two P . hallii reference genomes and their F2 progeny ( Lovell et al . , 2018 ) using synteny to project the positions of genes across multiple genomes onto the physical positions of a reference . To illustrate this approach , we reanalyzed QTL generated from the 26-parent USA maize nested association mapping ( NAM ) population ( Li et al . , 2016 ) . Originally , candidates for these QTL were defined by the proximate gene models only in the B73 reference genome ( Li et al . , 2016 ) ; however , with GENESPACE and the recently released NAM parent genomes ( Hufford et al . , 2021 ) , it is now possible to evaluate candidate genes present in the genomes of other NAM founder lines but either absent or unannotated in the B73 reference genome . To explore this possibility , we built a single-copy synteny map across all 26 NAM founders anchored to the B73 genome ( Figure 3D ) . We opted to focus on QTL where the allelic effect of a single parental genome was an outlier relative to all other alleles . Such \\u2018private\\u2019 allelic contributions in multi-parent populations offer a powerful opportunity to define high-confidence candidates as genes with parent-specific sequences that match parent-specific allelic contributions to phenotypic trait variation ( Abdulkina et al . , 2019 ) . Among the 190 QTLs , three displayed outlier effects of one parent: cultivar \\u2018Mo18W\\u2019 contributed an allele for delayed anthesis-silking interval at two adjacent chromosome 3 ( \\u2018Chr3\\u2019 ) QTLs and cultivar \\u2018Ki11\\u2019 contributed an allele that reduced plant height under drought at a QTL on chromosome 6 ( \\u2018Chr6\\u2019 ) ( Li et al . , 2016 ) . Given that these QTL were chosen only due to their parental allelic effects , we were surprised to find that the two Mo18W QTL regions exist within a 11 . 7 M bp derived inversion that is only found in the Mo18W genome ( Figure 3D , E ) . Since inversions reduce recombination , it is possible that multiple Mo18W causal variants have been fixed in linkage disequilibrium in this NAM population . In addition to this chromosomal mutation and sequence variation between the parents and B73 ( Li et al . , 2016 ) , we sought to define candidate genes from the patterns of presence\\u2013absence and copy number variation , explicitly looking for genes that were private to the focal genome . Two genes in the smaller Chr3 and one gene in the larger Chr3 interval were private to Mo18W , and four genes in three pan-genome entries ( one two-member array ) were private to Ki11 in the Chr6 interval ( Figure 3F , Figure 3\\u2014figure supplement 1 ) . While these genes do not have functional annotations relating to drought , this method provides additional candidates that would not have been discovered by B73-only candidate gene exploration . Like most plant families ( Barker et al . , 2016; Stebbins , 1950; One Thousand Plant Transcriptomes Initiative , 2019 ) , but unlike nearly all animal lineages ( Muller , 1925 ) , the grasses radiated following a whole-genome duplication: the ~70 M ya Rho WGD . The resulting gene family redundancy and gene-function subfunctionalization is hypothesized to underlie the tremendous ecological and morphological diversity of grasses ( Preston et al . , 2009; Preston and Kellogg , 2006; Wu et al . , 2008 ) . To explore sequence variation among Rho-derived paralogs , we used GENESPACE to build a ploidy-aware syntenic pan-genome annotation among eight species , using the built-in functionality that allows the user to mask primary ( likely orthologous ) syntenic regions and search for secondary hits ( likely paralogous , Figure 4A ) . This method acts similarly to using an outgroup that predated the WGD ( see \\u2018additional considerations\\u2019 methods ) , but using the same OrthoFinder run as in Figure 3A . Overall , the peptide identity between Rho-derived paralogous regions was much lower than orthologs between species ( e . g . , S . viridis vs . P . hallii: Wilcoxon W = 88 , 094 , 632 , p < 1 \\u00d7 10\\u221216 ) , consistent with the previous discovery that the Rho duplication predated the split among most extant grasses ( Ma et al . , 2021 ) . However , as has been previously observed ( Wang et al . , 2011 ) , there is significant variation in the relative similarity of Rho-duplicated sequences . As an example , the peptide sequences of single-copy gene hits in primary syntenic regions ( median identity = 90 . 6% ) between chromosome 8 of P . hallii and S . viridis , were 26 . 9% more similar than the secondary Rho-derived regions ( Figure 4B; median identity = 71 . 4% , Wilcoxon W = 87 , 842 , p < 1 \\u00d7 10\\u221216 ) . However , S . viridis chromosome 8 contained a single over-retained paralogous region . Unlike all other Rho-derived blocks , the P . hallii paralogs to this 2 . 7 M bp chromosome 8 region were not significantly less conserved than the primary orthologous region ( 91 . 6% vs . 91 . 9% , W = 14 , 830 , p = 0 . 13 ) . Outside of this region , the peptide identity of paralogs returned to the genome-wide average ( Figure 4B ) . In line with this observation , the GENESPACE run treating the eight genomes as haploid representations could not distinguish between the Rho-derived paralogs in the over-retained region across all grasses ( Figures 3A and 4C ) , except for of all chromosome pairs between B . distachyon , wheat , and blocks connecting Maize chromosome 10 to sorghum chromosome 5 . It is interesting to note that all syntenic over-retained regions were at the extreme termini of the chromosomes outside of maize , B . distachyon and wheat; further , the only genome with complete segregation of the two paralogs , wheat , also retains these regions in the center of all six chromosomes ( Figure 4C ) . These results are consistent with the proposed evolutionary mechanism ( Wang et al . , 2011 ) where concerted evolution and \\u2018illegitimate\\u2019 homeologous recombination may have homogenized these paralogous regions . This process would be less effective in pericentromeric regions than the chromosome tails , where a single crossover event would be sufficient to homogenize two paralogous regions . Combined , the historical abundance of genetic mapping studies and ongoing proliferation of genome resources provides a strong foundation for the integration of comparative and quantitative genomics to accelerate discoveries in evolutionary biology , medicine , and agriculture . The incorporation of synteny and orthology into comparative genomics and quantitative genetics pipelines offers a mechanism to bridge these disparate disciplines . Here , we presented the GENESPACE R package as a framework to help bridge the current gaps between comparative and quantitative genomics , especially in complex evolutionary systems . We hope that the examples presented here will inspire further work to leverage the powerful genome-wide annotations that are coming online , both within and among species . \\n GENESPACE operates on gff3-formatted annotation files and accompanying peptide fasta files for primary gene models . There are convenience functions for reformatting the gff and peptide files to simplify the naming scheme and reduce redundant gene models to the primary longest transcript . With these data in hand , GENESPACE calculates BLAST-like hits from DIAMOND2 and runs OrthoFinder ( Emms and Kelly , 2019 ) to infer orthogroups and orthologues . GENESPACE then extracts syntenic regions from the hits using a combination of graph- and cluster-based approaches , producing syntenic orthogroups for each unique ( not reciprocal ) pair of genomes . Syntenic orthogroups are then collapsed into a pan-genome annotation , which is a matrix of positions against a reference genome ( rows ) and unique gene models in each syntenic orthogroup for each genome ( columns ) . Detailed step-by-step pipeline methods can be found below . All analyses were performed in R 4 . 1 . 2 on macOS Big Sur 10 . 16 . The following R packages were used for visualization or within GENESPACE v0 . 9 . 3 ( February 11 , 2022 release ) : data . table v1 . 14 . 0 ( Dowle and Srinivasan , 2021 ) , dbscan v1 . 1-8 ( Hahsler et al . , 2019 ) , igraph v1 . 2 . 6 ( Csardi and Nepusz , 2006 ) , Biostrings v2 . 58 . 0 ( Pag\\u00e8s et al . , 2020 ) , and rtracklayer v1 . 50 . 0 ( Lawrence et al . , 2009 ) . GENESPACE also calls the following third party software: DIAMOND v2 . 0 . 8 . 146 ( Buchfink et al . , 2021 ) , OrthoFinder v2 . 5 . 4 ( Emms and Kelly , 2019 ) , and MCScanX no version installed on October 23 , 2021 ( Wang et al . , 2012 ) . All results were generated programmatically; the accompanying scripts and key output are available on github: jtlovell/GENESPACE_data . Minor adjustments to figures to improve clarity were accomplished in Adobe Illustrator v26 . 01 . A full description of each step in GENESPACE is provided in the documentation that accompanies the package source code on github ( jtlovell/GENESPACE ) . Publicly available genome annotations were downloaded on or before October 8 , 2021 . See Table 2 for data sources , citations , and metadata . All GENESPACE runs used default parameterization , with the following exceptions: ( 1 ) the Rho grass run allowed a single secondary hit ( default is 0 , this is how the paralogs are explicitly searched for ) and maximum number of gaps in secondary regions of 10 ( default is 5 , relaxed to reduce ancient paralogous block splitting ) , and ( 2 ) the maize run used the \\u2018fast\\u2019 OrthoFinder method since all genomes are closely related and haploid . Some maize genomes contained small alternative haplotype scaffolds , which were dropped for all analyses . The publicly available C3/C4 gene lists and QTL intervals were generated against the v2 maize assembly . To make this comparable to the across grass and NAM parent GENESPACE runs , we also accomplished a fast GENESPACE run between v2 and the two v5 versions used here . The orthologs and syntenic mapping between these versions are included as text files in the data repository . All statistics presented here were calculated within R . To compare nonnormal distributions ( e . g . , sequence identity ) , we used the nonparametric signed Wilcoxon ranked sum test . To measure sequence divergence , we calculated ungapped percent peptide sequence identity from pairwise Needleman\\u2013Wunsch global alignments , implemented in Biostrings ( Pag\\u00e8s et al . , 2020 ) . To determine single outliers from a unimodal distribution , we applied the Grubbs test implemented in the outliers R package ( Komsta , 2011 ) . Some figures were constructed outside of GENESPACE using base R plotting routines and ggplot2 v3 . 3 . 3 ( Wickham , 2016 ) . Some color palettes were chosen with RColorBrewer ( Neuwirth , 2014 ) and viridis ( Garnier et al . , 2021 ) . GENESPACE relies on high-confidence homologs , which are typically defined as members of the same orthogroup via OrthoFinder . In addition to the original method , which clusters genes and builds a graph from closely related genes based on BLAST scores ( Emms and Kelly , 2015 ) , OrthoFinder can also use gene trees to split orthogroups into pairwise orthologs ( Emms and Kelly , 2019 ) , which represent a stricter definition of orthology . GENESPACE attempts to merge the benefits of each of these methods by first only considering orthogroups for synteny , which allows users to optionally include paralogs in the scan , then including non-syntenic gene tree-inferred orthologs into the pan-genome annotation during its final steps ( Figure 1 , see pan-genome pipeline description below ) . The initial orthogroups are inferred \\u2018globally\\u2019 using pairwise reciprocal hits , either using the default OrthoFinder specification or \\u2018fast\\u2019 by only looking at unique pairs of genomes . Depending on the ploidy and user specifications , orthogroups can be re-estimated within syntenic regions . These three steps are detailed below . Method 1: default global orthogroups . The default behavior of GENESPACE is to run OrthoFinder using its default parameters . GENESPACE then builds a vector of OrthoFinder geneIDs and their corresponding orthogroup membership . When synteny is called , the GENESPACE-formatted gff text files are read in and merged with OrthoFinder sequenceIDs . Method 2: fast orthogroup estimation . GENESPACE offers a \\u2018fast\\u2019 orthofinderMethod ( Table 3 ) , which performs only one-way DIAMOND2 searches , where the genome annotation with more gene models serves as the query and the smaller annotation is the target . The hits then are mirrored , and each stored as OrthoFinder-formatted BLAST text files . OrthoFinder is then run to the orthogroup step on the precomputed BLAST files . This method results in significant speed improvements with little loss of fidelity among closely related haploid genomes . The user can also specify the sensitivity of DIAMOND2 via the diamondMode parameter during GENESPACE initialization . Method 3: orthogroups within syntenic blocks . In addition to the above global OrthoFinder runs , GENESPACE can rerun OrthoFinder ( only through the -og step ) in syntenic regions between pairs of genomes . This is accomplished through four additional steps in the synteny pipeline: ( 1 ) syntenic hits are split by large syntenic regions; ( 2 ) hits from each syntenic region are passed to OrthoFinder; ( 3 ) the within-block orthogroups assignments are aggregated into a single vector across all genomes; and finally , ( 4 ) the synteny pipeline ( see below ) is rerun using the newly defined combined syntenic and inBlkOG vector . While using significant computational resources , this process can improve the sensitivity of orthogroup discovery in paralogous or homeologous regions; however , it is not clear that this offers much improvement over global orthogroups between purely haploid genomes . As such , the default behavior of GENESPACE is to only use orthofinderInBlk when any genome has a ploidy >1 . The GENESPACE function \\u2018synteny\\u2019 accepts several user-defined parameters , which allow for flexibility; however , the defaults are sufficient for most high-quality genomes and evolutionary scenarios . For example , we used the same default parameters for 300 M years of vertebrate evolution , 50 M years and multiple WGDs of grasses , and 10 k years of Maize divergence . For a full list of parameters , see documentation of the GENESPACE function \\u2018set_syntenyParams\\u2019 , but here , we will discuss ( 1 ) the minimum number of unique hits within a syntenic block ( \\u2018blkSize\\u2019 , default = 5 ) , ( 2 ) the maximum number of gaps within a block alignment ( \\u2018nGaps\\u2019 , default = 5 ) , and ( 3 ) the radius around a syntenic anchor for a hit to be considered syntenic ( \\u2018synBuff\\u2019 , default = 100 ) . Synteny begins by processing all gene annotations ( steps 1\\u20133 ) , then proceeds to process blast hits for each unique pair of genomes ( steps 4\\u20137 ) . Step 1: flag the gff3-formatted annotation . GENESPACE adds the following information to a data matrix that initially contains the name and physical coordinates of each gene for each genome: ( 1 ) OrthoFinder IDs ( from the sequenceIDs . txt file ) , ( 2 ) gene rank order , and ( 3 ) orthogroup IDs . Step 2: quality control the annotation . Chromosomes with fewer than blkSize genes are dropped so that they will not be used for synteny inference . Step 3: find and parse tandem arrays . Tandem arrays are defined through six steps: ( 1 ) define potential tandem arrays as orthogroups containing >1 gene for each genome-by-chromosome combination; ( 2 ) calculate the maximum gene rank-order gap between each adjacent gene in a potential tandem array; ( 3 ) split those with a gap between genes > synBuff into separate arrays using dbscan; ( 4 ) flag clusters of >1 genes within synBuff as tandem arrays; ( 5 ) choose representative genes for each tandem array as the gene closest to the median position of the array and secondarily as the longest peptide sequence; and ( 6 ) recalculate \\u2018arrayOrd\\u2019 as the gene rank order of tandem array representative genes . Step 4: read and process raw blast files . To reduce unnecessary computation , GENESPACE concatenates both reciprocal BLAST hit files so that all unique combinations of query and target genes are represented in a single matrix . For simplicity , the genome with more gene models is treated as the query and the genome with fewer , the target . For each line ( \\u2018hit\\u2019 ) in the BLAST file , the following data and statistics are added for both the query ( genome1 ) and target ( genome2 ) gene: ( 1 ) positional information , including gene rank order; ( 2 ) tandem array representatives; ( 3 ) hits where both the query and target are in the same orthogroup; and ( 4 ) the relative strength of a hit , where scrRank = 1 represents the single highest bit score blast hit for each query and target gene . Step 5: define initial syntenic anchors . The following additional parameters are used to define initial syntenic anchors: nHits1 ( the top n hits for each gene in the query genome , default ploidy of target genome ) , nHits2 ( the top n hits for each gene in the target genome , default = ploidy of query genome ) , onlyOgAnchors ( logical specifying whether anchors hits must be isOg , default = TRUE ) , and maskTheseHits ( what hits should be masked in the search , default = none; see below on modifications for secondary hits and polyploid self hits ) . These parameters are applied to find regions that are large and collinear enough to be classified as syntenic through three steps: ( 1 ) potential anchor hits are defined as those where both the query and target are array representatives , the hit is not in maskTheseHits , the query scrRank \\u2264 nHits1 , the target scrRank \\u2264 nHits2 , and , optionally if onlyOgAnchors , both the query and target are in the same orthogroup; ( 2 ) gene rank order is recalculated for the potential anchor hits; ( 3 ) MCScanX_h ( -s = blkSize , -m = nGaps ) is called from R using condensed gene rank-order positions as the physical location , and those hits within collinear blocks are flagged as initial anchors . Step 6: clean up initial syntenic anchors . In some cases , syntenic anchors from step 5 can be broken up or do not extend to the ends of syntenic regions . To resolve this , we run five additional block finalization steps: ( 1 ) potential anchor hits within synBuff of the initial anchors are extracted and collinear hits are recalled into \\u2018cleaned\\u2019 anchors; ( 2 ) the cleaned anchor hits are clustered into blocks via dbscan using synBuff as the radius and blkSize as minPts arguments , dropping blocks with fewer anchors than blkSize and merging nearby syntenic regions; ( 3 ) final syntenic anchors are flagged by rerunning MCScanX within each syntenic region; ( 4 ) initial block breakpoints are generated by dbscan first with a large radius of synBuff , then on reranked gene order within regions with a radius of blkSize; and ( 5 ) overlapping blocks that are not duplicated are split using run-length equivalent decoding . Since the final coding of blocks is conducted within broad syntenic regions , the parameters passed should be robust to variation in ploidy and sequence divergence between genomes . Step 7: flag anchors , syntenic region hits , and block coordinates . With the final syntenic anchors from step 6 in hand we finalize and annotate the tab-delimited blast hits . We define two sets of syntenic qualifications: blocks , which are fine grained runs of completely collinear hits , and \\u2018regions\\u2019 , which are clustered blocks that average across minor inversions . Blocks are defined from the step 6 anchors . Anchor hits are reclustered via dbscan with a radius of synBuff and each cluster between each pair of genomes and chromosomes is assigned a \\u2018regID\\u2019 . All hits within synBuff of an anchor hit are flagged as \\u2018inBuffer\\u2019 . The coordinates of blocks and regions are calculated from the bounding anchors for each syntenic block . Modifications: steps 5\\u20137 are modified depending on whether the BLAST hit file is intra- or inter-genomic , the ploidy of the query and target genome , and the user-defined number of \\u2018secondary hits\\u2019 . These modifications are as follows . ( 1 ) step 5\\u20137 ( 1 ) : [if necessary] syntenic regions are called for self blast hits within haploid genomes . This is the simplest case where the steps are ignored and syntenic anchors are defined as self hits between tandem array representatives . Block and region IDs are chromosome IDs . ( 2 ) Modification step 5\\u20137 ( 2 ) : [if necessary] syntenic regions are called for self blast hits within genomes with ploidy >1 . Here , the modified syntenic hits from 5-7 ( 1 ) are specified in maskTheseHits with a synBuff radius of 500 genes . This excludes potentially problematic large tandem arrays in some genomes . The number of hits after masking is set to ploidy \\u2013 1 . Then the standard step 5\\u20137 processes are run . ( 3 ) Modification step 5\\u20137 ( 3 ) : [if necessary] syntenic regions are called for blast hits where nSecondaryHits >0 . Here , the methods for syntenic hit calculation from 5-7 ( 2 ) ( if intragenomic ) or 5-7 ( as a mask of the rerun if intergenomic ) are conducted except that the parameters specified are given as those ending in \\u2018Second\\u2019 . The GENESPACE function \\u2018pangenome\\u2019 decodes pairwise syntenic orthologs into a multigenome pan-annotation . The output is a long-formatted text file , where each gene is given a reference genome syntenic position and chromosome , and flags that are described below . In addition , pangenome also returns a wide-formatted data . table ( R object ) where each row is a pan-genome entry with positional information and each column is a list of genes by genome ( e . g . , Figure 1 ) . Step 1: build a reference-anchored map of syntenic hits . The chromosome and positions of the tandem array representatives from the reference genome annotation form the foundation of mapping across all genomes . Upon building this positional backbone , syntenic hits are pulled for each genome and placed against the anchor genes that are in the same orthogroup . Step 2: interpolate syntenic reference positions . In most pairwise comparisons between genomes , some syntenic orthogroups will be missing in the reference . Since we want to extract PAV by physical coordinates , all syntenic orthogroups , even those that do not include a reference genome gene model , need to have reference positional information . To fill this gap , GENESPACE interpolates the syntenic reference position of all array representative genes in all genomes through a three step pipeline: ( 1 ) subset the syntenic anchor hits to ungapped collinear hits following a 1:1 synteny mapping ratio ( i . e . , perfect diagonal of hits by gene rank order ) ; ( 2 ) cluster the 1:1 syntenic anchors so that any jump of >1 gene is split into its own cluster; ( 3 ) fill in missing positions through linear interpolation . Step 3: determine the reference mapping positions of each syntenic orthogroup . In the case where the reference genome is purely haploid with no segmental duplicates , this is a straightforward step: the orthogroups that contain reference genome genes are placed at that backbone position , and those without a reference gene are clustered into the most likely interpolated placement . However , we do not want to rely on single-copy references , so GENESPACE allows multiple placements not only for each nonreference gene , but also for genes in the reference genome itself . For example , in a polyploid , we would want to know if there is PAV between two homeologous chromosomes . To do this , we take a three-step approach: ( 1 ) the interpolated ( or actual reference-anchored ) positions of genes in each syntenic orthogroup are extracted and split by interpolated chromosome and maximum gap between any two inferred positions >synBuff; ( 2 ) unsplit orthogroups are set aside; ( 3 ) split orthogroups are checked and retained if they contain \\u2265 propAssignThresh ( default = 0 . 5 ) proportion of genes in the syntenic orthogroup with an interpolated position in that cluster; ( 4 ) clustered positions are dropped if there are more than maxPlacementsPerRefChr ( default = 2 ) positional placements for that syntenic orthogroup , retaining the top clusters ranked by propAssignThresh; ( 5 ) the culled positional clusters are merged with those from step ( 2 ) to create the initial pan-genome annotation . Step 4: add and flag other forms of orthogroups . The reference pan-genome built in step 3 only acts on direct syntenic blast hits ( edges ) , which allows for strict construction of interpolated positions without the potential polluting positional effects of orthogroups in minor or miss-assembled syntenic blocks . GENESPACE fixes these and other complexities of syntenic orthogroups through a four step pipeline to finalize the pan-genome annotation: ( 1 ) \\u2018indirect syntenic\\u2019 orthogroup members are added back into the initial pan-genome annotation by parsing the gff-like text file that contains a vector of syntenic orthogroups; ( 2 ) syntenic orthogroups that are missing all interpolated positions are added and given NA positions; ( 3 ) tandem array members that are not representative genes are added into the pan-genome annotation; ( 4 ) if available , orthologues from the initial OrthoFinder run that are not present in the pan-genome annotation are added and flagged . Note that if OrthoFinder was run using the \\u2018fast\\u2019 orthofinderMethod , orthologs will not be produced nor added into the pan-genome annotation . There are several factors to consider when constructing your GENESPACE run , generating syntenic block breakpoints , or looking at comparative genomics through protein similarity estimates like OrthoFinder . We detail two of these below: ( 1 ) Outgroups and the phylogenetic context of orthology inference . OrthoFinder defines orthogroups as the set of genes that are descended from a single gene in the last common ancestor of all the species being considered . As such , the scale of the run matters , often significantly . For example , an orthogroup would not be likely to contain homeologs across the two ancient subgenomes for a run that included only two maize genomes . Since the coalescence of any two maize genotypes occurred more recently than the ~12M ya WGD , few homeologs would both be descended from the same common ancestor when considering only maize genotypes . Hence , the within-maize NAM parent run ( Figure 3D ) excludes homeologs . However , if an outgroup to maize was included in the run , both maize homeologs would be likely to show common ancestry to a single gene in the outgroup , thus connecting the maize homeologs into a single orthogroup . Hence , both maize homeologous regions are present in the across grasses synteny graph ( Figure 3A ) despite using identical synteny parameters to the maize NAM parent run . Given the potentially significant role of outgroups on the results of the global run , GENESPACE offers an \\u2018outgroup\\u2019 parameter , which specifies the genomes that should be included in the orthofinder run but excluded from all downstream analyses . ( 2 ) Studying homoelogs in polyploids and other paralogs . Non-homeologous paralogs are typically excluded in the default GENESPACE parameterization . This is in part because GENESPACE was originally designed for work with plants , and most plant lineages have undergone one or more WGDs . However , there are two ways to study paralogs by altering GENESPACE parameters: ( 1 ) specify an \\u2018outgroup\\u2019 genome ( see above ) , which is only used for the global OrthoFinder run and set the genomes\\u2019 ploidy as that expected by the number of duplications relative to the outgroup; ( 2 ) if an outgroup is not available ( or too distantly related to be of use ) , specify the synteny parameter \\u2018nSecondaryHits\\u2019 as the number of paralogous copies per genome . In the second case , secondaryHits are inferred after masking out the normal syntenic hits , then synteny is rerun without requiring anchors to be global orthogroups . In both cases , it will be far better to set orthofinderInBlk to TRUE , so that pairs of genomes are considered and genes without a solid hit in the outgroup are not excluded .\",\n          \"The effect of calcium supplementation on bone mineral accretion in people under 35 years old is inconclusive . To comprehensively summarize the evidence for the effect of calcium supplementation on bone mineral accretion in young populations ( \\u226435 years ) . This is a systematic review and meta-analysis . The Pubmed , Embase , ProQuest , CENTRAL , WHO Global Index Medicus , Clinical Trials . gov , WHO ICTRP , China National Knowledge Infrastructure ( CNKI ) , and Wanfang Data databases were systematically searched from database inception to April 25 , 2021 . Randomized clinical trials assessing the effects of calcium supplementation on bone mineral density ( BMD ) or bone mineral content ( BMC ) in people under 35 years old . This systematic review and meta-analysis identified 43 studies involving 7 , 382 subjects . Moderate certainty of evidence showed that calcium supplementation was associated with the accretion of BMD and BMC , especially on femoral neck ( standardized mean difference [SMD] 0 . 627 , 95% confidence interval [CI] 0 . 338\\u20130 . 915; SMD 0 . 364 , 95% CI 0 . 134\\u20130 . 595; respectively ) and total body ( SMD 0 . 330 , 95% CI 0 . 163\\u20130 . 496; SMD 0 . 149 , 95% CI 0 . 006\\u20130 . 291 ) , also with a slight improvement effect on lumbar spine BMC ( SMD 0 . 163 , 95% CI 0 . 008\\u20130 . 317 ) , no effects on total hip BMD and BMC and lumbar spine BMD were observed . Very interestingly , subgroup analyses suggested that the improvement of bone at femoral neck was more pronounced in the peripeak bone mass ( PBM ) population ( 20\\u201335 years ) than the pre-PBM population ( <20 years ) . Our findings provided novel insights and evidence in calcium supplementation , which showed that calcium supplementation significantly improves bone mass , implying that preventive calcium supplementation before or around achieving PBM may be a shift in the window of intervention for osteoporosis . This work was supported by Wenzhou Medical University grant [89219029] . \\n Osteoporosis is an imperative public health problem , particularly in elderly women ( Anonymous , 1993; Jones et al . , 1994; Si et al . , 2015 ) . Low bone mass and a fast rate of bone loss at menopause are equal risk factors for future fracture ( Riis et al . , 1996 ) . A low bone mineral content ( BMC ) or bone mineral density ( BMD ) in an elderly person implies a suboptimal bone mass in young adulthood \\u2013 related to peak bone mass ( PBM ) , greater bone loss in later life , or both . A number of studies have concluded that increasing calcium intake in older people is unlikely to translate into clinically meaningful reductions in fractures or produce progressive increases in bone mass ( Tai et al . , 2015; Zhao et al . , 2017; Bolland et al . , 2015; Hu et al . , 2019 ) . It seems that calcium supplementation is meaningless in the elderly . On the other hand , intervention before the achievement of PBM to maximize PBM might have a significant influence on bone health and further prevent osteoporosis later in life . Several clinical trials have shown positive effects of calcium supplementation on BMD or BMC in children ( Lloyd et al . , 1993; Khadilkar et al . , 2012 ) . However , several clinical trials have concluded that calcium supplementation may not be associated with calculated bone mass or strength ( Lu et al . , 2019; Vogel et al . , 2017 ) . Narrative reviews have also concluded that calcium supplementation may have small nonprogressive effects on BMD or BMC ( Winzenberg et al . , 2006; Huncharek et al . , 2008 ) . To summarize the studies above , there have been considerable debates about whether calcium supplementation has effects on bone health among young people . Very recently , a study using cross-sectional data from NHANES 2005\\u20132014 concluded that the age at attainment of peak femoral neck BMD , total hip BMD , and lumbar spine BMD was 20\\u201324 years old in males and 19\\u201320 years old in females ( Xue et al . , 2020 ) . Additionally , a plateau is achieved in PBM at approximately 30 years old ( Baxter-Jones et al . , 2011 ) . Based on the literature above , we decided to limit the threshold to 35 years old in a conservative manner . Since the results of studies in young people are controversial , we carried out a comprehensive meta-analysis to determine the effectiveness of calcium supplementation for improving BMD or BMC in young people before the age of 35 . We also aimed to determine whether any effect would vary by sex , baseline calcium intake , ethnicity , age or sources , duration , and doses of calcium supplementation . \\n We applied search strategies to the following electronic bibliographic databases without language restrictions: PubMed , EMBASE , ProQuest , CENTRAL ( Cochrane Central Register of Controlled Trials ) , WHO Global Index Medicus , Clinical Trials . gov , WHO ICTRP , China National Knowledge Infrastructure ( CNKI ) , and Wanfang Data in April 2021 and updated the search in July 2022 for eligible studies addressing the effect of calcium or calcium supplementation , milk or dairy products with BMD or BMC as endpoints . Detailed search strategies are provided in Supplementary file 1 . Only randomized controlled trials ( RCTs ) were included in this study . We also hand-searched conference abstract books . The reference sections and citation lists of the retrieved literature , including original research articles , reviews , editorials , and letters , were reviewed for potentially relevant articles . We selected trials based on the following criteria: ( 1 ) RCTs comparing calcium or calcium plus vitamin D supplements with a placebo or no treatment; ( 2 ) trials involving participants aged under 35 years at baseline; ( 3 ) trials providing BMD ( g/cm2 ) or BMC ( g ) data measured by dual energy X-ray absorptiometry as estimates of bone mass . Exclusion criteria: ( 1 ) observational studies , such as cohorts , case\\u2013control studies , or cross-sectional studies; ( 2 ) participants aged over 35 years; ( 3 ) trials of participants who were pregnant or in the lactation period; ( 4 ) trials without a placebo or control group; ( 5 ) trials supplied with only vitamin D; ( 6 ) trials that had essential data missing . Two authors ( YPL and SYL ) independently screened titles and abstracts , and then full texts of relevant articles according to the inclusion and exclusion criteria . By thoroughly reading full texts , the reasons for excluded trials are provided in Supplementary file 2 . The quality of the included RCTs was assessed independently by two reviewers ( SYL and HNJ ) based on the Revised Cochrane Risk-of-Bias Tool for Randomized Trials ( RoB 2 tool , version August 22 , 2019 ) ( Jpt , 2021 ) and each item was graded as low risk , high risk , and some concerns . The five domains included the randomization process , deviations from intended interventions , missing outcome data , measurement of the outcome , and selection of the reported result . A general risk conclusion can be drawn from the risk assessment of the above five aspects . We defined the included trials as low , high , and moderate quality based on the overall bias , which is consistent with the RoB 2 tool algorithm . Disagreements were resolved by consensus . Two researchers ( YPL and SYL ) independently used a structured data sheet to extract the following information from each study: authors , publication year , participant characteristics , doses of the supplements , baseline dietary calcium intake , duration of trials , and follow-up . The absolute changes in BMD or BMC at the lumbar spine , femoral neck , total hip , and total body were the primary outcomes we extracted . We categorized the studies into two groups by duration: <18 months and \\u226518 months . For studies that presented the percentage change rather than absolute data , we calculated the absolute change value using baseline data , and the standard deviation and percentage change from baseline were consistent with the approach described in the Cochrane Handbook ( Jpt , 2021 ) . If there was missing information , we contacted the corresponding author and obtained the data . ( If no reply was received for over 3 months , we would exclude the article . ) The association of calcium with or without vitamin D supplements with BMD and BMC was assessed . We pooled the data ( study level ) from each study using random-effects models in a conservative manner . The standardized mean difference ( SMD ) and corresponding 95% confidence intervals ( CIs ) were reported . We performed predesigned subgroup analyses based on the following aspects: sex ( female vs . male ) and age at baseline ( <20 vs . \\u226520 years , representing the prepeak and peripeak subgroups , respectively; all analyzed trials were divided into two groups by the age of achieving PBM [determined as 20 years old] ) , regions ( Asian and Western ) , sources of calcium supplementation ( dietary vs . calcium supplements ) , and bias risk of each individual trial . We further conducted some post hoc subgroup analyses according to the level of calcium intake at baseline ( <714 vs . \\u2265714 mg/day , based on the median value ) , the calcium supplementation dose ( <1000 vs . \\u22651000 mg/day , based on the median value ) and vitamin D supplementation ( with or without vitamin D ) . To assess how long the beneficial effect would be maintained , we performed post hoc subgroup analyses according to the duration , taking into account different calcium supplementation periods and different follow-up periods across the trials . Sensitivity analyses included evaluations using fixed-effect models and excluding low-quality trials . In these aforementioned subgroup analyses , if the number of eligible studies in subgroups was less than three , we conducted a sensitivity analysis by excluding the subgroup with fewer than three studies . An effect size of \\u22650 . 20 and<0 . 50 was considered small , \\u22650 . 50 and <0 . 80 was considered medium , and \\u22650 . 80 was considered large using Cohen\\u2019s criteria ( Cohen , 1992 ) . We assessed heterogeneity between studies using the I2 statistic . We performed meta-regression for sample size , age , sex , and supplementation differences to explain the heterogeneity between studies . We performed cumulative meta-analyses based on the sample size to compare with the primary outcomes . We assessed publication bias by examining funnel plots when the number of trials was 10 or more and used Begg\\u2019s rank correlation and Egger\\u2019s linear regression tests ( Egger et al . , 1997 ) . Furthermore , we robustly adjusted for the summarized results by applying Duval and Tweedie\\u2019s trim and fill method ( Duval and Tweedie , 2000 ) . Data extraction and integration were done on Microsoft Office Excel ( version 2011 ) . Meta-analysis , subgroup analysis and sensitivity analysis were all performed by Comprehensive Meta Analysis ( version 3 . 3 . 070 , Biostat , Englewood , NJ ) . All tests were two tailed , and p < 0 . 05 was considered statistically significant . Two reviewers ( SYL and YL ) independently applied the Grading of Recommendations Assessment , Development and Evaluation ( GRADE ) system to assess the overall quality of evidence . The quality of evidence for each outcome was classified as either high , moderate , low , or very low based on the evaluation for study design , bias risk , inconsistency , indirectness , imprecision , publication bias , and confounding bias . GRADE pro version 3 . 6 was used to grade the overall quality of evidence and prepare the summary-of-findings table . Every decision to downgrade or upgrade the studies was labeled using footnotes . Any disagreements were resolved by consensus . The funders had no role in study design , data collection , data analysis , data interpretation , or writing of the report . \\n Of the 5518 references screened , we identified 43 eligible RCTs ( Figure 1 ) involving 7382 subjects ( Lloyd et al . , 1993; Khadilkar et al . , 2012; Lu et al . , 2019; Vogel et al . , 2017; Bonjour et al . , 1997; Cadogan et al . , 1997; Cameron et al . , 2004; Cheng et al . , 2005; Chevalley et al . , 2005a; Du et al . , 2004; Gibbons et al . , 2004; Lau et al . , 2004; Lee et al . , 1995; Lee et al . , 1994; Lloyd et al . , 1996; Matkovic et al . , 2005; Moyer-Mileur et al . , 2003; Prentice et al . , 2005; Rozen et al . , 2003; Specker and Binkley , 2003; Stear et al . , 2003; Courteix et al . , 2005; Iuliano-Burns et al . , 2003; Johnston et al . , 1992; M\\u00f8lgaard et al . , 2004; Nowson et al . , 1997; Ho et al . , 2005; Ma et al . , 2014; Zhang et al . , 2014; Ward et al . , 2014; Arab Ameri et al . , 2012; Ekbote et al . , 2011; Hemayattalab , 2010; Islam et al . , 2010; Yin et al . , 2010; Lambert et al . , 2008; Zhu et al . , 2008; Ward et al . , 2007; Bass et al . , 2007; Barger-Lux et al . , 2005; Chevalley et al . , 2005b; Winters-Stone and Snow , 2004; Volek et al . , 2003 ) . Table 1 shows the baseline characteristics of the included studies . Of the 43 RCTs , 20 used dietary sources of calcium ( Lu et al . , 2019; Vogel et al . , 2017; Bonjour et al . , 1997; Cadogan et al . , 1997; Cheng et al . , 2005; Du et al . , 2004; Gibbons et al . , 2004; Lau et al . , 2004; Iuliano-Burns et al . , 2003; Nowson et al . , 1997; Ho et al . , 2005; Ma et al . , 2014; Zhang et al . , 2014; Arab Ameri et al . , 2012; Ekbote et al . , 2011; Lambert et al . , 2008; Zhu et al . , 2008; Bass et al . , 2007; Volek et al . , 2003 ) and 23 used calcium supplements ( including calcium , calcium citrate malate , and calcium phosphate ) ( Lloyd et al . , 1993; Khadilkar et al . , 2012; Cameron et al . , 2004; Chevalley et al . , 2005a; Lee et al . , 1995; Lee et al . , 1994; Lloyd et al . , 1996; Matkovic et al . , 2005; Moyer-Mileur et al . , 2003; Prentice et al . , 2005; Rozen et al . , 2003; Specker and Binkley , 2003; Stear et al . , 2003; Courteix et al . , 2005; Johnston et al . , 1992; M\\u00f8lgaard et al . , 2004; Ward et al . , 2014; Hemayattalab , 2010; Islam et al . , 2010; Yin et al . , 2010; Ward et al . , 2007; Barger-Lux et al . , 2005; Winters-Stone and Snow , 2004 ) . The median baseline dietary calcium intake was 714 mg/day; the duration of calcium supplementation intervention did not exceed 2 years in most trials ( 38/43 ) ; and the dose of calcium intervention did not exceed 1000 mg/day in most trials ( 38/43 ) . Of all the included trials , 23 trials were categorized as low risk of bias; 16 , as moderate risk; and 4 , as high risk ( Supplementary file 3 ) . Figure 2 , Figure 2\\u2014source data 1 , Figure 2\\u2014source data 2 , Figure 2\\u2014source data 3 , Figure 2\\u2014source data 4 , Figure 3 , Figure 3\\u2014source data 1 , Figure 3\\u2014source data 2 , Figure 3\\u2014source data 3 , and Figure 3\\u2014source data 4 show the summarized effect estimates . For total body , moderate evidence showed that calcium supplementation significantly improved BMD levels with an SMD of 0 . 330 ( 95% CI: 0 . 163\\u20130 . 496 , p < 0 . 001 ) and slightly improved BMC levels with an SMD of 0 . 149 ( 95% CI: 0 . 006\\u20130 . 291 , p < 0 . 001 ) . At the femoral neck , we found a stronger and moderate protective effect on BMD ( 0 . 627 , 95% CI: 0 . 338\\u20130 . 915 , p < 0 . 001 ) and a small improvement effect on BMC ( 0 . 364 , 95% CI: 0 . 134\\u20130 . 595 , p = 0 . 002 ) . Meanwhile , a slight but significant improvement in BMC was observed for the lumbar spine ( 0 . 163 , 95% CI: 0 . 008\\u20130 . 317 , p = 0 . 039 ) . However , calcium supplementation did not improve the BMD levels at the lumbar spine ( 0 . 090 , 95% CI: \\u22120 . 044 to 0 . 224 , p = 0 . 190 ) or total hip ( 0 . 257 , 95% CI: \\u22120 . 053 to 0 . 566 , p = 0 . 104 ) or the BMC level at the total hip ( 0 . 116 , 95% CI: \\u22120 . 382 to 0 . 614 , p = 0 . 648 ) . Tables 2 and 3 show the results of subgroup analyses . To explore whether the observed effect differed by the age of participants , we divided these participants into two subgroups: prepeak ( <20 years ) and peripeak ( \\u226520\\u201335 years ) , and the results were generally consistent with the findings from the primary analyses . Notably , the improvement effect on both BMD and BMC at the femoral neck ( see Figure 4 ) tended to be stronger in the peripeak subjects than in the prepeak subjects ( 0 . 852 , 95% CI: 0 . 257\\u20131 . 446 vs . 0 . 600 , 95% CI: 0 . 292\\u20130 . 909 [for BMD] and 1 . 045 , 95% CI: 0 . 701\\u20131 . 39 vs . 0 . 249 , 95% CI: 0 . 043\\u20130 . 454 [for BMC] , respectively ) . Subgroup analyses by the duration of calcium supplementation showed that the improvement effects on both BMD and BMC of the femoral neck were stronger in the subgroup with <18 months than in the subgroup with \\u226518 months . However , regarding total body BMD , the effect of calcium supplementation in the subgroup with \\u226518 months duration was slightly greater than that in the other subgroup . Regarding the sex of subjects , we found a stronger beneficial effect on femoral neck BMD and BMC in women-only trials ( 0 . 712 , 95% CI: 0 . 149\\u20131 . 275 , p = 0 . 013; 0 . 742 , 95% CI: 0 . 267\\u20131 . 217 , p = 0 . 002 , respectively ) than in trials including men and women ( 0 . 556 , 95% CI: 0 . 233\\u20130 . 879 , p = 0 . 001; 0 . 195 , 95% CI: \\u22120 . 027 to 0 . 418 , p = 0 . 086 ) . When considering the sources of participants , the improvement effects on femoral neck and total body BMD or on femoral neck and lumbar spine BMC were obviously stronger in Western countries than in Asian countries . Subgroup analyses by the level of dietary calcium intake at baseline showed that , for femoral neck BMD , the beneficial effect was significant only in the lower subgroup receiving <714 mg/day ( 0 . 581 , 95% CI: 0 . 266\\u20130 . 896; p < 0 . 001 ) ; for total body BMD , the beneficial effect was slightly greater in the lower subgroup receiving <714 mg/day ( 0 . 363 , 95% CI: 0 . 127\\u20130 . 599; p = 0 . 003 ) ; for total hip BMD and lumbar spine BMC , however , the beneficial effects were statistically significant in the higher subgroup receiving \\u2265714 mg/day ( 0 . 723 , 95% CI: 0 . 245\\u20131 . 201; p = 0 . 003 and 0 . 2 , 95% CI: 0 . 052\\u20130 . 348; p = 0 . 008 , respectively ) . Subgroup analyses based on calcium supplement dosages demonstrated a statistically significant effect on femoral neck and total body BMD in the lower dose subgroup receiving <1000 mg/day ( 0 . 717 , 95% CI: 0 . 349\\u20131 . 085; p < 0 . 001 and 0 . 392 , 95% CI: 0 . 161\\u20130 . 624; p = 0 . 001 , respectively ) but not in the higher dose subgroup receiving \\u22651000 mg/day . When considering the different sources of calcium , both calcium sources from dietary intake and additional calcium supplements exerted significantly positive effects on femoral neck BMD ( 0 . 728 , 95% CI: 0 . 311\\u20131 . 144 , p < 0 . 001; 0 . 510 , 95% CI: 0 . 101\\u20130 . 919 , p = 0 . 014 ) and total body BMD ( 0 . 290 , 95% CI: 0 . 054\\u20130 . 526 , p = 0 . 016; 0 . 405 , 95% CI: 0 . 195\\u20130 . 615 , p < 0 . 001 ) . For BMCs of the lumbar spine and femoral neck , only calcium supplements other than dietary intake had a significant improvement effect . To explore the longevity of the beneficial effect , we performed subgroup analyses and found that calcium supplementation improved the BMD levels during the follow-up periods after the end of intervention , and the beneficial effect was maintained for at least 1 year after the intervention ( 0 . 933 , 95% CI: 0 . 323\\u20131 . 664 , p = 0 . 004 ) . However , this beneficial effect seemed to disappear when the follow-up period exceeded 2 years . In order to compare the effect of the presence or absence of vitamin D on the effect of calcium supplementation , we divided all the data into two groups and ran the calculations separately . Calcium supplementation with vitamin D showed greater beneficial effects on femoral neck BMD and BMC ( 0 . 758 , 95% CI: 0 . 350\\u20131 . 166 , p < 0 . 001; 0 . 393 , 95% CI: 0 . 067\\u20130 . 719 , p = 0 . 018 ) . However , for BMCs of lumbar spine and total body , as well as total body BMD , only calcium supplementation without vitamin D had a significant improvement effect . Sensitivity analyses including only trials with a low risk of bias ( high quality , see Supplementary file 4 ) showed that the improvement effects on femoral neck BMD and BMC remained statistically significant and stable ( 0 . 356 , 95% CI: 0 . 064\\u20130 . 648 , p = 0 . 017; 0 . 249 , 95% CI: 0 . 043\\u20130 . 454 , p = 0 . 018 ) . The result for total body BMD was also stable ( 0 . 343 , 95% CI: 0 . 098\\u20130 . 588 , p = 0 . 006 ) . However , for lumbar spine and total body BMCs , the positive effect was not statistically significant . For other sites , the results were generally consistent with those of the primary analyses . Additional sensitivity analyses using fixed-effect models ( see Supplementary file 5 ) , performing cumulative meta-analysis ( see Supplementary file 6 ) , and excluding studies had been included in previous meta-analysis ( see Supplementary file 7 ) showed generally consistent results with the primary analyses . Supplementary file 8 shows a summary of the GRADE assessments of the overall certainty of the evidence for the effect of calcium supplementation on bone measurements . The evidence was graded as moderate for all sites . All of these outcomes were downgraded for inconsistency . For femoral neck BMD , it was downgraded because of strongly suspected publication bias , however , it was upgraded due to the effect size was over 0 . 5 . In summary , the outcome of femoral neck BMD was graded as moderate . In general , the heterogeneity between trials was obvious in the analysis for BMD ( p < 0 . 001 , I2 = 86 . 28% ) and slightly smaller for BMC ( p < 0 . 001 , I2 = 79 . 28% ) . The intertrial heterogeneity was significantly distinct across the sites measured . Subgroup analyses and meta-regression analyses suggested that this heterogeneity could be explained partially by differences in age , duration , calcium dosages , types of calcium supplement , supplementation with or without vitamin D , baseline calcium intake levels , sex , and region of participants ( Tables 2 and 3 and Supplementary file 9 ) . Funnel plots , Begg\\u2019s rank correlation , and Egger\\u2019s regression test for each outcome bias are presented in Supplementary file 10 . Publication bias was obvious in the femoral neck BMD . The adjusted effect size analyzed using the trim and fill method also showed a difference from the unadjusted value . Except for the outcome above , no evidence for publication bias was found . The adjusted summary effect size analyzed using the trim and fill method did not show substantial changes as well , which also implies no evidence of publication bias . \\n This meta-analysis comprehensively summarized the evidence for the efficiency of calcium supplementation in young people before the peak of bone mass and at the plateau period . The findings indicated significant improvement effects of calcium supplements on both BMD and BMC , especially on the femoral neck . Numerous recent systematic reviews have concluded that there is no evidence for associations between calcium supplements and reduced risk of fracture or improvement of bone density in people aged over 50 years ( Tai et al . , 2015; Zhao et al . , 2017; Bolland et al . , 2015; Bristow et al . , 2022 ) . Since calcium supplements are unlikely to translate into clinically meaningful reductions in fractures or improvement of bone mass in aged people , we wondered if it is possible to increase bone mass at the peak by administering calcium supplements before the age of reaching the PBM or at the plateau of this peak to prevent osteoporosis and reduce the risk of fractures in later life . To the best of our knowledge , this is the first meta-analysis to focus on age before achieving PBM or age at the plateau of PBM , at which the risk of fracture is extremely low . Why did we do such a meta-analysis ? Instead of traditionally solving problems when they occurred , that is , treating osteoporosis after a patient has developed osteoporosis , our research attempted to explore the effects of preventive intervention before reaching the plateau and before osteoporosis development . Our study suggests that calcium supplementation can significantly boost peak bone content , which can improve bone mass . Since calcium supplementation in elderly individuals occurs late and has no influence , our findings have critical implications for the early prevention of fractures in the elderly population and provide better insights for the current situation of calcium supplementation . Preventive calcium supplementation in young populations is a shift in the window of intervention for osteoporosis , not limited to a certain age group but involving the whole life cycle of bone health . Is there any difference in supplementation of calcium before or after the achievement of the PBM ? We found that calcium supplementation improved the bone mass at the femoral neck in both the prepeak and peripeak subjects; furthermore , it is worth noting that the improvement effect was obviously stronger in the peripeak population ( \\u226520\\u201335 years ) than in the prepeak population ( <20 years ) . Based on our findings and the negative associations of calcium supplements with bone outcomes in aged people from previous studies , one can conclude that young adulthood may be the best intervention window to optimize bone mass , especially the PBM; moreover , our study indicates the importance of calcium supplementation at this age instead of the often-mentioned age groups of children or elderly individuals . The findings of our study provide completely new insight into a novel intervention window in young adulthood to improve bone mass and further prevent osteoporosis and fractures in their late lifespan . To synthesize previously published studies in children , we found a meta-analysis conducted by Winzenberg et al . , 2006 that included 19 studies involving 2859 children and found a small effect on total body BMC and no effect on lumbar spine BMD in children , which was in line with our finding . However , they found no effect on BMD at the femoral neck , which was inconsistent with our result . We therefore performed a sensitivity analysis , excluding all the literature they included , and found that the results of our newly included studies , 28 in total , were generally consistent with the primary results . We also performed a sensitivity analysis incorporating only the studies they pooled and found a statistically significant effect for BMD in the femoral neck and total body , while the results for total body BMC were nonsignificant ( see e Supplementary file 7 ) . These slightly different findings can be interpreted as follows: first , we included more and updated literature; second , they used only endpoint data directly , whereas we used change data , taking into account the difference in baseline conditions; third , we used change data to represent the change before and after calcium supplementation more directly . Another meta-analysis conducted by Huncharek et al . , 2008 included 21 studies involving 3821 subjects and pooled three reports involving subjects with low baseline calcium intake and reported a statistically significant summary of the mean BMC in children . Combining the above published literature with our conclusions , it can be concluded that calcium supplementation is more effective in young adults aged 20\\u201335 years than in children . Although this issue needs to be confirmed in the future , our findings highlight the importance of this intervention window of approximately 10\\u201315 years at the peri-PBM period , which is better than the pre-PBM period . To explore whether there is a difference between dietary calcium intake and calcium supplements , our subgroup analyses suggested that one can obtain this beneficial effect from both calcium sources , including dietary intake and calcium supplements . For BMD at the femoral neck , dietary calcium seemed to exert a better effect than calcium supplements . Similarly , we also found that the improvement effect was statistically significant only in subjects supplied with calcium dosages lower than 1000 mg/day . These findings support the hypothesis that there may be a threshold dose of calcium supplementation; when exceeded , the effect does not increase . Our findings are consistent with the previous research by Prentice , 2002 , which is that no additional benefit is associated with an intake above the currently recommended dose at the population level . The underlying mechanisms are unclear and need to be elucidated in future studies . To explore whether the effect of improving BMD or BMC is due to calcium alone or calcium plus vitamin D , our subgroup analyses found that calcium supplementation with vitamin D had greater beneficial effects on both the femoral neck BMD and BMC than calcium supplementation without vitamin D . However , for both BMD and BMC at the other sites ( including lumbar spine , total hip , and total body ) , the observed effects in the subgroup without vitamin D supplementation appeared to be slightly better than in the subgroup with vitamin D supplementation . Therefore , these results suggested that calcium supplementation alone could improve BMD or BMC , although additional vitamin D supplementation may be beneficial in improving BMD or BMC at the femoral neck . To determine the differences between high dietary intake and low dietary intake of calcium at baseline , our subgroup analyses showed that the improvement effect seemed to be stronger in subjects with high intake at baseline than in those in the lower subgroup . Interestingly , these results were in accordance with the findings of subgroup analyses by population area , which suggested that calcium supplementation was more effective in Western populations , whose level of baseline calcium intake is normally higher than that in Asian countries . However , these findings are likely to be contrary to our common sense , which is , that under normal circumstances , the effects of calcium supplementation should be more obvious in people with lower calcium intake than in those with higher calcium intake . Therefore , this issue needs to be tested and confirmed in future trials . To investigate changes in the effect of calcium supplementation after cessation , our subgroup analysis showed that the effect remained significant 1 year after cessation , particularly at various sites of BMD . For studies with a follow-up period longer than 1 year , we included only two articles: one study Lambert et al . , 2008 with 2 years of follow-up after calcium supplementation was stopped and another study Chevalley et al . , 2005b with 7 years of follow-up . Their results were pooled and showed that the effects of calcium supplementation no longer persisted . The number of studies is too small for us to explore how long the effects of calcium supplementation will last , and well-designed cohort studies are needed in the future . In the meantime , we have found a point to ponder about whether gains can be made when calcium supplementation is restarted after a period of withdrawal and what other changes in the organism remain to be discovered . Several limitations need to be considered . First , there was substantial intertrial heterogeneity in the present analysis , which might be attributed to the differences in baseline calcium intake levels , regions , age , duration , calcium dosages , types of calcium supplement , supplementation with or without vitamin D and sexes according to subgroup and meta-regression analyses . To take heterogeneity into account , we used random-effect models to summarize the effect estimates , which could reduce the impact of heterogeneity on the results to some extent . Second , our research failed to clearly compare the difference between males and females due to the limitation of existing data \\u2013 some studies provided merged data of males and females without males alone . Based on the existing data , the beneficial effect was more obvious when subjects were limited to women only , which needs to be validated in future trials . Third , we found that few of the existing studies focused on the 20- to 35-year age group , which was why there were only three studies of this age group that met our inclusion criteria; although the number was small , our evidence was of high quality , and the results were stable , especially in the femoral neck . We also tried to find mechanisms related to bone metabolism in the age group of 20\\u201335 years , but few studies have focused on this age group; most studies have focused only on mechanisms related to older people or children . Therefore , more high-quality RCTs and studies on the exploration of mechanisms focusing on the 20- to 35-year age group are needed in the future . Finally , as some of the studies did not provide the physical activity levels of the participants , we failed to exclude the effect of physical activity on the results . This study has several strengths . In this first systematic review by meta-analysis to focus on people at the age before achieving PBM and at the age around the peak of bone mass , we comprehensively searched for all of the currently eligible trials and included a total of 7382 participants ( including 3283 calcium supplement users and 4099 controls ) , which added reliability to our findings . Another strength is the high consistency of the results across predesigned subgroup analyses and sensitivity analyses . Additionally , we analyzed both BMD and BMC separately for the different measurement sites rather than using the mean of all combined values to draw conclusions , which has the advantage of obtaining changes in bone indexes at different sites and drawing more accurate conclusions . In conclusion , calcium supplementation can significantly improve BMD and BMC , especially at the femoral neck . Moreover , supplementation in people who are at the plateau of their PBM has a better effect . Although further well-designed RCTs with larger sample sizes are required to verify our findings , we provide a new train of thought regarding calcium supplementation and the evaluation of its effects . In terms of bone health and the full life cycle of a person , the intervention window of calcium supplementation should be advanced to the age around the plateau of PBM , namely , at 20\\u201335 years of age .\",\n          \"Photosystem II ( PSII ) uses the energy from red light to split water and reduce quinone , an energy-demanding process based on chlorophyll a ( Chl-a ) photochemistry . Two types of cyanobacterial PSII can use chlorophyll d ( Chl-d ) and chlorophyll f ( Chl-f ) to perform the same reactions using lower energy , far-red light . PSII from Acaryochloris marina has Chl-d replacing all but one of its 35 Chl-a , while PSII from Chroococcidiopsis thermalis , a facultative far-red species , has just 4 Chl-f and 1 Chl-d and 30 Chl-a . From bioenergetic considerations , the far-red PSII were predicted to lose photochemical efficiency and/or resilience to photodamage . Here , we compare enzyme turnover efficiency , forward electron transfer , back-reactions and photodamage in Chl-f-PSII , Chl-d-PSII , and Chl-a-PSII . We show that: ( i ) all types of PSII have a comparable efficiency in enzyme turnover; ( ii ) the modified energy gaps on the acceptor side of Chl-d-PSII favour recombination via PD1+Phe- repopulation , leading to increased singlet oxygen production and greater sensitivity to high-light damage compared to Chl-a-PSII and Chl-f-PSII; ( iii ) the acceptor-side energy gaps in Chl-f-PSII are tuned to avoid harmful back reactions , favouring resilience to photodamage over efficiency of light usage . The results are explained by the differences in the redox tuning of the electron transfer cofactors Phe and QA and in the number and layout of the chlorophylls that share the excitation energy with the primary electron donor . PSII has adapted to lower energy in two distinct ways , each appropriate for its specific environment but with different functional penalties . \\n Photosystem II ( PSII ) is the water/plastoquinone photo-oxidoreductase , the key energy converting enzyme in oxygenic photosynthesis . The near-universal type of PSII , found in all photosynthetic eukaryotes and in most cyanobacteria , contains 35 chlorophylls a ( Chl-a ) and 2 pheophytins a ( Phe ) . Four of the Chl molecules ( PD1 , PD2 , ChlD1 , and ChlD2 ) and both Phe molecules are located in the reaction centre ( Diner and Rappaport , 2002 ) . The remaining 31 Chl-a in the PSII core constitute a peripheral light-collecting antenna . When antenna chlorophylls are excited by absorbing a photon , they transfer the excitation energy to the primary electron donor , ChlD1 , the red-most chlorophyll in the reaction centre , although it\\u2019s been reported that charge separation from PD1 can occur in a fraction of centres ( Diner and Rappaport , 2002; Holzwarth et al . , 2006; Romero et al . , 2010; Cardona et al . , 2012 ) . The initial charge separation , forming the first radical pair ChlD1+Phe- ( assuming ChlD1 as primary donor ) , is quickly stabilized by the formation of the second radical pair , PD1+Phe- , and then by further electron transfer steps ( Figure 1A ) that lead to the reduction of plastoquinone and the oxidation of water . PSII activity is energy demanding . In Chl-a-PSII , the primary donor absorbs red photons at 680 nm , and this defines the energy available for photochemistry ( 1 . 82 eV ) with a high quantum yield for the forward reactions . The energy stored in the products of the reaction ( reduced plastoquinone and molecular oxygen ) and in the trans-membrane electrochemical gradient is ~1 eV , while the remaining ~0 . 82 eV is released as heat helping to ensure a high quantum yield for the forward reaction and minimize damaging and wasteful side- and back-reactions . The 1 . 82 eV was suggested to be the minimum amount of energy required for an optimum balance of efficiency versus resilience to photodamage , and responsible for explaining the \\u2018red limit\\u2019 ( ~680 nm ) for oxygenic photosynthesis ( N\\u00fcrnberg et al . , 2018; Rutherford et al . , 2012 ) . The first reported case in which the red limit is exceeded was the chlorophyll d ( Chl-d ) -containing cyanobacterium Acaryochloris marina ( A . marina ) ( Miyashita et al . , 1996 ) . Chl-d-PSII contains 34 Chl-d and 1 Chl-a ( proposed to be in the PD1 position Renger and Schlodder , 2008 ) and uses less energy , with the proposed Chl-d primary donor in the ChlD1 position absorbing far-red photons at ~720 nm ( Schlodder et al . , 2007 ) , corresponding to an energy of ~1 . 72 eV ( Figure 1B ) . Recently , it was discovered that certain cyanobacteria use an even more red-shifted pigment , chlorophyll f ( Chl-f ) , in combination with Chl-a ( Chen et al . , 2010; Gan et al . , 2014 ) . When grown in far-red light , these cyanobacteria replace their Chl-a-PSII with Chl-f-PSII , that has far-red specific variants of the core protein subunits ( D1 , D2 , CP43 , CP47 , and PsbH ) and contains ~90% of Chl-a and ~10% of Chl-f ( N\\u00fcrnberg et al . , 2018; Gan et al . , 2014 ) . The Chl-f-PSII from Chroococcidiopsis thermalis PCC7203 ( C . thermalis ) , which contains 30 Chl-a , 4 Chl-f , and 1 Chl-d , was shown to have a long wavelength primary donor ( originally proposed to be either Chl-f or d , in the ChlD1 position N\\u00fcrnberg et al . , 2018 ) absorbing far-red photons at ~720 nm ( Figure 1C ) , the same wavelength as in A . marina ( N\\u00fcrnberg et al . , 2018; Judd et al . , 2020 ) . A recent cryo-EM structure has also argued for ChlD1 being the single Chl-d in the Chl-f-PSII of Synechococcus sp . PCC7335 ( Gisriel et al . , 2022 ) . This suggests that this could be the case also in the Chl-f-PSII of C . thermalis , because of the conservation of the amino acids coordinating ChlD1 in the far-red PSII of the two species . The facultative , long-wavelength species that use Chl-f are thus the second case of oxygenic photosynthesis functioning beyond the red-limit ( N\\u00fcrnberg et al . , 2018 ) , but the layout of their long wavelength pigments is quite different from that of the Chl-d-PSII . Assuming that Chl-a-PSII already functions at an energy red limit ( Rutherford et al . , 2012 ) , the diminished energy in Chl-d-PSII and Chl-f-PSII seems likely to increase the energetic constraints . Thus , if the far-red PSII variants store the same amount of energy in their products and electrochemical gradient , as seems likely , then it was suggested that they should have decreased photochemical efficiency and/or a loss of resilience to photodamage ( N\\u00fcrnberg et al . , 2018; Cotton et al . , 2015; Davis et al . , 2016 ) . These predicted energetic constraints are worth investigating to generate knowledge that could be beneficial for designing strategies aimed at engineering of far-red photosynthesis into other organisms of agricultural or technological interest ( Chen and Blankenship , 2011 ) . Here we report a comparison of the enzyme turnover efficiency , forward reactions , and back-reactions in the three known types of PSII: Chl-a-PSII , and the two far-red types , the Chl-f-PSII from C . thermalis and the Chl-d-PSII from A . marina . To compare the enzymatic properties of the three types of PSII and minimize the effects of physiological differences between strains , isolated membranes rather than intact cells were used . The use of isolated membranes allows the minimization of potential effects due to: ( i ) the transmembrane electric field , which affects forward electron transfer ( Diner and Joliot , 1976 ) and charge recombination ( Joliot and Joliot , 1980 ) , ( ii ) the uncontrolled redox state of the plastoquinone pool in whole cells , which can affect the QB/QB- ratio present in dark-adapted PSII , ( iii ) differences in the size and composition of the phycobilisomes and in their association with PSII , and ( iv ) the presence of photoprotective mechanisms such as excitation energy quenching and scavengers of reactive oxygen species . \\n The electron transfer properties of the three types of PSII were investigated by comparing the decay kinetics of the flash-induced fluorescence in membranes from A . marina , white-light ( WL ) grown C . thermalis and far-red-light ( FR ) grown C . thermalis . When forward electron transfer occurs ( Figure 2A ) , the fluorescence decay comprises three phases ( Crofts and Wraight , 1983; Vass et al . , 1999 ) : the fast phase ( ~0 . 5ms ) is attributed to electron transfer from QA- to QB or QB- and the middle phase ( ~3ms ) is generally attributed to QA- oxidation limited by plastoquinone ( PQ ) entry to an initially empty QB site and/or by QBH2 exiting the site prior to PQ entry ( de Wijn and van Gorkom , 2001 ) . These two phases had comparable time-constants in all samples ( T1=0 . 5\\u20130 . 6 and T2=3 . 5\\u20135ms , Table 1 ) . The fast electron transfer from QA- to the non-heme iron possibly oxidized in a fraction of centres is too fast ( t\\u00bd~50 \\u00b5s ) to be detected here . The slower decay phase is attributed to the charge recombination between QA- and the Mn-cluster mostly in the S2 state ( see section 2 . 2 ) in centres where forward electron transfer to QB/QB- did not occur . This phase was significantly slower in FR C . thermalis ( T3=14 . 3 \\u00b1 4 . 6 s ) than in WL C . thermalis ( T3=5 . 6 \\u00b1 2 . 4 s ) but had a similar amplitude in the two samples ( Figure 2\\u2014figure supplement 1 and Table 1 ) . In A . marina this phase had a bigger amplitude than in the two C . thermalis samples ( Table 1 ) , because it was superimposed to a non-decaying component of the fluorescence , that did not return to the original F0 level even at 100 s after the flash ( Figure 2\\u2014figure supplement 1 ) . This non-decaying component , absent in the two C . thermalis samples , is attributed to centres without a functional Mn-cluster , in which PD1+ is reduced by an electron donor that does not recombine in the minutes timescale ( such as Mn2+ , TyrD , or the ChlZ/Car side-path ) , with the consequence of stabilizing QA- ( Nixon et al . , 1992; Debus et al . , 2000 ) . The fluorescence decay arising from the S2QA- recombination was slower in A . marina ( T3=10 . 8 \\u00b1 2 . 6 s ) than in WL C . thermalis , but its overlap with the non-decaying component made the fit of its time-constant potentially less reliable . Indeed , when the fluorescence decay due to charge recombination was measured in presence of the QB-site inhibitor DCMU ( Figure 2B ) , the decay kinetics were bi-phasic in all samples , and no difference in the major S2QA- recombination phase ( slow phase in Table 1 , ~80% amplitude , T3 ~6\\u20137 s ) was found between A . marina and WL C . thermalis . In contrast , the decay was significantly slower in FR C . thermalis , with the time-constant of the major S2QA- recombination phase ( slow phase in Table 1 , ~80% amplitude , T3=10 . 4 \\u00b1 0 . 8 s ) similar to that measured in the absence of DCMU . The shorter lifetime ( ~0 . 22\\u20131 s ) of the middle decay phase ( amplitude 15\\u201320% ) was compatible with it originating from TyrZ\\u2022 ( H+ ) QA- recombination occurring either in centres lacking an intact Mn-cluster ( Yerkes et al . , 1983 ) or in intact centres before charge separation is fully stabilised , as proposed in Debus et al . , 2000 . The fluorescence decay in WL and FR C . thermalis also had an additional fast phase of small amplitude ( 5\\u20136% ) , attributed to forward electron transfer in centres in which DCMU was not bound ( Lavergne , 1983 ) . Again , the A . marina traces included a non-decaying phase of fluorescence , attributed to centres lacking an intact Mn-cluster . The fluorescence decay kinetics in membranes of Synechocystis sp . PCC6803 ( Synechocystis ) , perhaps the best studied Chl-a containing cyanobacterium , were also measured as an additional control . The kinetics in Synechocystis membranes were comparable to those reported for WL C . thermalis ( Appendix 1 ) . The Synechocystis and A . marina fluorescence decay kinetics measured in membranes here are overall slower than those previously measured in cells ( Cser et al . , 2008 ) . This difference is ascribed to pH and membrane potential effects , as discussed in Appendix 1 , and illustrates the difficulty to use whole cells for such measurements . To conclude , the forward electron transfer rates from QA- to QB/QB- are not significantly different in the three types of PSII . In contrast , the S2QA- recombination is slower in Chl-f-PSII of FR C . thermalis compared to Chl-a-PSII of WL C . thermalis and Chl-d-PSII of A . marina . The efficiency of PSII water oxidation activity can be estimated by the flash-dependent progression through the S-states of the Mn-cluster . This can be measured by thermoluminescence ( TL ) , which arises from radiative recombination of the S2QB- and S3QB- states ( Rutherford et al . , 1982 ) . The TL measured in A . marina , WL C . thermalis , and FR C . thermalis membranes showed similar flash-dependencies in all three types of PSII ( Appendix 2\\u2014figure 1 ) , confirming and extending the earlier report ( N\\u00fcrnberg et al . , 2018 ) . Because the TL data presented some variability between biological replicates ( Appendix 2 ) , additional analyses were performed by polarography and absorption spectroscopy . Figure 3 shows the flash-dependent oxygen evolution measured in A . marina , FR C . thermalis and Synechocystis membranes . The latter were used as a Chl-a-PSII control because the content of PSII in membranes of WL C . thermalis was too low to allow accurate O2 polarography measurements ( Figure 3\\u2014figure supplement 1D ) . As shown by fluorescence , no significant difference in forward electron transfer between the two types of Chl-a-PSII was observed ( Appendix 1 ) , and the use of Synechocystis membranes was therefore considered as a valid control . The measurements were performed using white , red , and far-red flashes . As expected , in dark-adapted samples , with S1 as the majority state ( Table 2 ) , the maximal O2 evolution occurred on the 3rd flash with subsequent maxima at 4 flash intervals . These maxima reflect the occurrence of the S3YZ\\u25cf/S4 to S0 transition in most centres as two water molecules are oxidized , resulting in the release of O2 . This oscillation pattern was the same in all samples and under all excitation conditions , except in Synechocystis membranes illuminated with far-red light , where the slow rise in O2 evolution is due to the weak excitation of Chl-a-PSII by the short wavelength tail of the 730 nm flash . The miss factor , indicating the fraction of PSII centres failing to progress through the S-states after a saturating flash excitation ( Lavergne , 1991; Grabolle and Dau , 2007 and see Discussion ) , was \\u226420% in all the samples except in the Synechocystis sample illuminated with far-red flashes , where it was >80% ( Figure 3D ) . For A . marina , the misses ( 13\\u201317% ) were very similar to those reported earlier ( Shevela et al . , 2006 ) . The misses in FR C . thermalis and in Synechocystis when illuminated with the 613 nm LED were slightly higher ( 17%\\u201320% ) . Nevertheless , these differences , attributed to the combination of the absence of exogenous electron acceptors , and the relatively long and possibly not fully saturating flashes ( Figure 3\\u2014figure supplement 1 ) , were not significant . In order to confirm and expand the results obtained with polarography , we measured the S-state turnover as the flash-induced absorption changes at 291 nm ( Figure 4 ) , that reflect the redox state of the Mn ions in the oxygen evolving complex ( Lavergne , 1991; Boussac et al . , 2004 ) . These measurements were done in the presence of the electron acceptor PPBQ and using single-turnover monochromatic saturating laser flashes . In the case of A . marina , the measurements could be done using membranes , but the membranes of WL and FR C . thermalis could not be used because of their high light-scattering properties in the UV part of the spectrum . In the case of the FR C . thermalis partially purified O2 evolving Chl-f-PSII were made and used for the measurements , while difficulties were encountered in isolating O2 evolving PSII from WL C . thermalis . Therefore , PSII cores from T . elongatus with the D1 isoform PsbA3 ( Sugiura et al . , 2008 ) were used as a Chl-a-PSII control . Among the three D1 present in T . elongatus , PsbA3 has the highest sequence identity with the D1 of Chl-f-PSII in FR C . thermalis ( see Discussion ) . The Chl-f-PSII was illuminated with flashes at wavelengths preferentially absorbed by Chl-a ( 680 nm ) and by long-wavelength chlorophylls ( 720\\u2013750 nm ) ( Figure 4A ) . As expected , maximum absorption decrease ( positive \\u0394I/I , as defined in Materials and methods , section UV transient absorption ) occurred on S2 ( flash 1 , 5 , 9 etc . ) and maximum absorption increase ( negative \\u0394I/I ) on S0 ( flash 3 , 7 , 11 etc . ) ( Lavergne , 1991 ) . No differences could be observed in either the amplitude or the damping of the oscillations between the excitation wavelengths . When using sub-saturating flashes ( ~83% power ) , the damping of the oscillations was the same for all excitation wavelengths ( Figure 4B ) , verifying that the illumination with 100% laser power was saturating at all the wavelengths . The equal amplitude of the oscillations obtained at all excitation wavelengths also indicates that the FR C . thermalis sample used does not contain any detectable Chl-a-PSII contamination . No differences in the oscillation patterns measured in FR C . thermalis Chl-f-PSII cores and in A . marina membranes , flashed at either 680 or 725 nm , were observed ( Figure 4C ) . The PSII of T . elongatus showed a normal S-states progression when using 680 nm excitation , but no oscillation pattern when far-red flashes were used ( Figure 4D ) . For all samples the calculated miss factor was ~10% ( Appendix 3 , discussion based on Styring and Rutherford , 1987; Velthuys and Visser , 1975; Vermaas et al . , 1984; Sugiura et al . , 2004 ) . In conclusion , the data reported here show that the overall efficiency of electron transfer from water to the PQ pool is comparable in all three types of PSII ( independently of the Chl-a-PSII control used ) , as shown by the near-identical flash patterns of thermoluminescence ( Appendix 2 ) and O2 release ( Figure 3 ) , both measured without external electron acceptors . When the S-state turnover was measured by following the absorption of the Mn-cluster in the UV ( Figure 4 ) , the use of artificial electron acceptors and single-turnover saturating flashes allowed us to obtain better resolved flash patterns that were essentially indistinguishable in all three types of PSII and between excitation with visible or far-red light in the case of the Chl-d-PSII and Chl-f-PSII . Charge recombination reactions were investigated by monitoring the thermoluminescence and luminescence emissions . The TL curves in Figure 5A and B show that both Chl-f-PSII and Chl-d-PSII are more luminescent than Chl-a-PSII , with Chl-f-PSII being the most luminescent . These differences , that are much larger than the variability between biological replicates ( Figure 5\\u2014figure supplement 1C and D and Table 3 ) , fit qualitatively with earlier reports ( N\\u00fcrnberg et al . , 2018; Cser et al . , 2008 ) ( see Appendix 4 for more details ) . The high luminescence indicates that in the Chl-d-PSII and Chl-f-PSII there is an increase in radiative recombination , although the causes of this increase are likely to be different between the two photosystems , as detailed in the Discussion . Despite the large difference in TL intensity between the Chl-a-PSII and Chl-f-PSII , the peak temperatures corresponding to the S2QB- and S2QA- recombination were both similar in Chl-a-PSII and Chl-f-PSII . In Chl-d-PSII , the temperature of the S2QB- peak was only slightly lower , while the S2QA- peak was ~15 \\u00b0C lower ( Figure 5\\u2014figure supplement 1A and B and Table 3 ) . Earlier TL reports comparing Chl-d-PSII in A marina cells with Chl-a-PSII in Synechocystis cells also showed that , while the peak position of S2QB- recombination was similar in the two samples , the S2QA- peak position was lower in A . marina ( Cser et al . , 2008 ) , in agreement with the present results in membranes . The peak temperatures measured in cells were lower than those reported here , which can be explained by ( i ) the effect of the transmembrane electric field , as discussed for the fluorescence decay results , and ( ii ) by differences in the heating rates used ( 1 \\u00b0C s\\u20131 here , 0 . 33 \\u00b0C s\\u20131 in Cser et al . , 2008 ) . When performing the same measurements in Synechocystis membranes ( Figure 5\\u2014figure supplement 2A ) , the S2QB- and S2QA- peak positions were comparable to those obtained in the two C . thermalis samples , confirming that the lower S2QA- peak temperature is a specific feature of Chl-d-PSII . The S2QA- recombination in the presence of DCMU was also measured by luminescence decay kinetics at 10 , 20 , and 30\\u00b0C , a range of temperatures that covers those of the S2QA- TL peaks of the three samples . Luminescence decay kinetics were recorded from 570ms for 300 seconds after the flash . In this time-range , the luminescence arises mainly from recombination via the back-reaction of S2QA- ( Goltsev et al . , 2009 ) . The total S2QA- luminescence emission ( Figure 5C ) reflected the intensities of the TL peaks , as expected ( Rutherford and Inoue , 1984 ) , with the order of intensity as follows: Chl-f-PSII>Chl-d-PSII>Chl-a-PSII ( although the variability between replicates made the difference between Chl-a-PSII and Chl-d-PSII less significant than that measured by TL ) . The total emissions did not vary significantly between 10\\u00b0C and 30\\u00b0C , although the decay kinetics were temperature-sensitive ( Appendix 5\\u2014figure 1 ) . The decay components identified by fitting the curves and their significance are discussed further in Appendix 5 , based on Yerkes et al . , 1983; Lavorel and Dennery , 1984; Tyystjarvi and Vass , 2007; Sugiura et al . , 2014 . The luminescence decay attributed to S2QA- recombination was bi-phasic ( Appendix 5\\u2014table 1 ) , with the kinetics of both phases being faster in Chl-d-PSII ( ~3 and~11 s ) than in Chl-a-PSII ( ~4 and~25 s ) , but slower in Chl-f-PSII ( ~9 and~39 s ) . The average S2QA- luminescence decay lifetimes accelerated with increasing temperature in Chl-a-PSII and Chl-f-PSII but were always the fastest in Chl-d-PSII and the slowest in Chl-f-PSII ( Figure 5D ) . The luminescence decay kinetics of the Chl-a-PSII in Synechocystis membranes were similar to those measured in WL C . thermalis ( Figure 5\\u2014figure supplement 2B and C ) , suggesting , as seen with the TL data , that the differences in kinetics observed in the two types of far-red PSII are not due to differences between species . In conclusion , both Chl-f-PSII and Chl-d-PSII show strongly enhanced luminescence , as previously reported ( N\\u00fcrnberg et al . , 2018; Cser and Vass , 2007 ) . However , the Chl-d-PSII differs from the Chl-a-PSII and Chl-f-PSII by having a lower S2QA- TL peak temperature and a faster S2QA- luminescence decay . This indicates that Chl-d-PSII has a smaller energy gap between QA- and Phe compared to Chl-a-PSII and Chl-f-PSII , resulting in: ( i ) less heat required for the electron to be transferred energetically uphill from QA- to Phe ( manifest as lower TL peak temperature ) , and ( ii ) a bigger proportion of S2QA- recombination occurring via repopulation of PD1+Phe- , a route faster than direct PD1+QA- recombination ( manifest as faster luminescence decay kinetics , see also Appendix 5 ) . The lower TL temperature and faster luminescence decay for S2QA- recombination in Chl-d-PSII , but without a marked increase in its QA- decay rate as monitored by fluorescence ( Figure 2 ) , could reflect differences in the competition between radiative and non-radiative recombination pathways in Chl-d-PSII compared to those in Chl-a-PSII and Chl-f-PSII . In contrast , in Chl-f-PSII the energy gap between QA- and Phe does not appear to be greatly affected or could even be larger , as suggested by the slower S2QA- recombination measured by fluorescence ( Figure 2 ) and luminescence ( Figure 3 ) decay . The QB potentials appear to be largely unchanged , as manifested by the similar S2QB- stability in all three types of PSII , with the slightly lower S2QB- TL peak temperature in A . marina probably reflecting the decrease in the energy gap between QA- and Phe . The smaller energy gap between QA- and Phe reported here in A marina is expected to result in enhanced singlet O2 production and hence greater sensitivity to photodamage ( N\\u00fcrnberg et al . , 2018; Cotton et al . , 2015; Johnson et al . , 1995; Vass and Cser , 2009 ) . This was investigated by measuring the rates of 1O2 generation induced by saturating illumination in isolated membranes using histidine as a chemical trap ( Figure 6A , representative traces in Figure 6\\u2014figure supplement 1A-C ) . 1O2 reacts with histidine to form the final oxygenated product , HisO2 , resulting in the consumption of O2 , as measured using the O2 electrode . Without the histidine trap , most 1O2 is thought to be quenched by carotenoids ( Telfer et al . , 1994 ) . When histidine was present in addition to DCMU , the Chl-d-PSII in A . marina membranes showed significant light-induced 1O2 formation . Under the same conditions , little 1O2 formation occurred in Chl-a-PSII or Chl-f-PSII in C . thermalis membranes . Similarly low levels of 1O2 were generated by Chl-a-PSII in Synechocystis membranes ( Figure 6\\u2014figure supplement 1D ) . The His-dependent O2 consumption in A . marina membranes showed the same light intensity dependence as O2 evolution ( Appendix 6\\u2014figure 1B ) , which suggests that 1O2 formation was related to Chl-d-PSII photochemistry . Sodium azide , a 1O2 quencher , suppressed the His-dependent oxygen consumption measured in A . marina in the presence of DCMU and when using the 1O2-generating dye Rose Bengal , confirming that it was due to the production of 1O2 ( Figure 6\\u2014figure supplement 1E and F ) . The strikingly high amount of 1O2 generated by Chl-d-PSII prompted us to perform additional controls . ( i ) To test if the high 1O2 production was related to the intactness of the PSII donor side , Mn was removed from A . marina membranes by Tris-washing . This had little effect on the 1O2 formation with respect to the Mn-containing membranes ( Appendix 6\\u2014figure 2 ) , suggesting that the high 1O2 production in untreated A . marina membranes does not arise specifically from the fraction of centres lacking an intact Mn-cluster that are potentially responsible for the non-decaying fluorescence observed in Figure 2 . ( ii ) The possibility that photosystem I ( PSI ) contributed to the light-induced O2 consumption by reducing oxygen to O2\\u2022- in membranes was tested ( Appendix 6\\u2014figure 3 ) . In the presence of DCMU , PSI-driven O2 reduction mediated by methyl viologen only took place when exogenous electron donors to PSI were provided . This indicates that there is no contribution from PSI-induced O2 reduction in Figure 6A , where exogenous PSI donors are absent . ( iii ) The higher 1O2 production is also seen in A . marina cells compared to FR C . thermalis cells ( Appendix 6\\u2014figure 4A ) , and thus is not an artefact associated with the isolation of membranes ( e . g . damaged photosystems or free chlorophyll ) . WL C . thermalis cells also showed low levels of 1O2 production , similar to those measured in membranes ( Appendix 6\\u2014figure 4B ) . The reliability of the His-trapping method to monitor 1O2 production in intact cyanobacterial cells has been previously demonstrated ( Rehman et al . , 2013 ) . Figure 6B shows the effect of 30 min of saturating illumination ( red light ) on the activity of the Chl-d-PSII , Chl-a-PSII and Chl-f-PSII . The results show that Chl-d-PSII is significantly more susceptible to light induced loss of activity compared to Chl-f-PSII , and to a lesser extent to Chl-a-PSII , and this can be correlated to the higher levels of 1O2 production in Chl-d-PSII . \\n The turnover of the water oxidation cycle is comparably efficient in all three types of PSII , as shown by their near-identical flash patterns in thermoluminescence ( Appendix 2\\u2014figure 1 ) , O2 release ( Figure 3 ) , and UV spectroscopy ( Figure 4 ) . In PSII , a photochemical \\u2018miss factor\\u2019 can be calculated from the damping of the flash patterns of O2 evolution . These misses , which are typically ~10% in Chl-a-PSII , are mainly ascribed to the \\u00b5s to ms recombination of S2TyrZ\\u2022QA- and S3TyrZ\\u2022QA- states ( Grabolle and Dau , 2007 ) . Despite the diminished energy available , the miss factors in both types of far-red PSII were virtually unchanged compared to Chl-a-PSII , which also suggests that the misses have the same origin . If so , the energy gaps between TyrZ and PD1 , and thus their redox potentials , would be essentially unchanged . These conclusions agree with those in earlier work on Chl-d-PSII ( Shevela et al . , 2006 ) and on Chl-f-PSII ( N\\u00fcrnberg et al . , 2018 ) . The similar flash-patterns also indicate that , after the primary charge separation , the electron transfer steps leading to water oxidation must have very similar efficiencies in all three types of PSII , that is , close to 90% , and that there are no major changes affecting the kinetics of forward electron transfer . In the case of Chl-f-PSII , this confirms earlier suggestions based on flash-dependent thermoluminescence measurements ( N\\u00fcrnberg et al . , 2018 ) . Indeed , electron transfer from QA- to QB/QB- , monitored by fluorescence , showed no significant differences in kinetics in the three types of PSII ( Figure 2A ) . The most striking difference between the three types of PSII is that the Chl-d-PSII of A . marina shows a decreased stability of S2QA- , indicated by the lower temperature of its TL peak and the correspondingly faster luminescent decay kinetics ( Figure 5 ) , and consequently a significant increase in 1O2 generation under high light ( Figure 6A ) . This likely corresponds to the decrease in the energy gap between Phe and QA predicted to result from the ~100 meV lower energy available when using light at ~720 nm to do photochemistry ( N\\u00fcrnberg et al . , 2018; Cotton et al . , 2015 ) . This is also supported by the estimates in the literature of the redox potential ( Em ) values of Phe/Phe- and QA/QA- in Mn-containing Chl-d-PSII: compared to Chl-a-PSII , the estimated increase of ~125 mV in the Em of Phe/Phe- is accompanied by an estimated increase of only ~60 mV in the Em of QA/QA- , which implies that a normal energy gap between the excited state of the primary donor ( ChlD1* ) and the first and second radical pairs ( ChlD1+Phe- and PD1+Phe- ) is maintained , but the energy gap between PD1+Phe- and PD1+QA- is significantly decreased ( ~325 meV vs ~385 meV ) ( Allakhverdiev et al . , 2011 ) . The changes in the D1 and D2 proteins of A . marina responsible for the changes in the Em of Phe/Phe- and QA/QA- are currently unknown . Our results indicate that in Chl-d-PSII , the decrease in the energy gap between Phe and QA favours charge recombination by the back-reaction route ( via PD1+Phe- ) , forming the reaction centre chlorophyll triplet state ( Rutherford et al . , 1981 ) , which acts as an efficient sensitizer for 1O2 formation ( Johnson et al . , 1995; Vass and Cser , 2009; Keren et al . , 1995; Keren et al . , 2000 ) . Consequently , the Chl-d-PSII is more sensitive to high light ( Figure 6B ) , reflecting the fact that this long-wavelength form of PSII has evolved in shaded epiphytic environments ( N\\u00fcrnberg et al . , 2018; Miyashita et al . , 1996; Cotton et al . , 2015; Davis et al . , 2016; Murakami et al . , 2004; Miller et al . , 2005; K\\u00fchl et al . , 2005; Mohr et al . , 2010 ) . The increase in the proportion of recombination going via PD1+Phe- in Chl-d-PSII can also result in a higher repopulation of the excited state of the primary donor ( ChlD1* ) , with a consequent increase in radiative decay ( high luminescence ) . In contrast to the Chl-d-PSII , the Chl-f-PSII shows no increased production of 1O2 and no increased sensitivity to high light compared to Chl-a-PSII , in the conditions tested here ( Figure 6 ) . The back-reactions appear to be little different from the Chl-a-PSII except for the more stable ( more slowly recombining ) S2QA- , as seen by fluorescence ( Figure 2 ) and luminescence ( Figure 5 ) decay . These properties may seem unexpected because this type of PSII has the same energy available for photochemistry as the Chl-d-PSII . In the Chl-d-PSII the lower energy of ChlD1* is matched by an increase in the Em of Phe/Phe- . In the Chl-f-PSII of C . thermalis and of the other Chl-f containing species , the Em of Phe/Phe- is also expected to be increased by the presence , in the far-red D1 isoform , of the strong H-bond from Glu130 ( Figure 7 ) , which is characteristic of high-light D1 variants in cyanobacteria ( Sugiura et al . , 2010 ) . In Chl-a-PSII , this change has been reported to induce an increase in the Em of Phe/Phe- between ~15 and~30 mV ( Sugiura et al . , 2010; Merry et al . , 1998 ) : an increase of this size would only partially compensate for the ~100 meV decrease in the energy of ChlD1* in Chl-f-PSII , and this would result in a smaller energy gap between ChlD1* and the first and second radical pairs ChlD1+Phe- and PD1+Phe- . This would favour the repopulation of ChlD1* by back-reaction from PD1+Phe- ( even if the repopulation of PD1+Phe- from the PD1+QA- state did not increase ) , resulting in the higher luminescence of Chl-f-PSII , as proposed earlier ( N\\u00fcrnberg et al . , 2018 ) . Increased decay of the PD1+QA- radical pair via the radiative route could in principle decrease the decay via the triplet route , but the overall small yield of luminescence means that this could be a minor effect . Additionally , the longer lifetime of S2QA- recombination in Chl-f-PSII indicates that the Em of QA/QA- has increased to compensate the up-shift in the Em of Phe/Phe- and to maintain an energy gap between Phe and QA large enough to prevent an increase in PD1+Phe- repopulation and thus in reaction centre chlorophyll triplet formation . This situation occurs in the PsbA3-D1 high light variant of T . elongatus , although the protein changes responsible for the increase in the Em of QA/QA- are not known ( Sugiura et al . , 2010 ) . A slower S2QA- recombination could also arise from an increase in the redox potential of PD1/PD1+ ( Diner et al . , 2001 ) , but this would likely compromise forward electron transfer in Chl-f-PSII by decreasing the driving force for stabilization of ChlD1+Phe- by the formation of PD1+Phe- , if the redox potential of ChlD1/ChlD1+ was not increased accordingly . If this problem were avoided by an equivalent increase in the redox potential of ChlD1/ChlD1+ , this would likely compromise charge separation , as a matching up-shift the ChlD1+/ChlD1* couple would occur , thus decreasing the reductive power of ChlD1* . As a 720 nm pigment , the reducing power of ChlD1* seems likely to be already compromised in Chl-f-PSII . The results here ( Figure 2A ) show forward electron transfer and charge separation appear to be comparable to Chl-a PSII and so it is unlikely that redox tuning of PD1 and ChlD1 is responsible for the stabilisation of the S2QA- recombination . In addition to changes in the redox potentials of Phe and QA , the size and pigment composition of the antennas of Chl-d-PSII and Chl-f-PSII could also contribute to the functional differences reported in the present work . These differences are summarized in Figure 8 . In PSII , two factors will determine the yield of charge separation: ( i ) the relative population of the excited state of the primary donor , ChlD1* , which depends on the dynamics of excitation energy transfer between pigments , and ( ii ) the rate of population of the second radical pair , PD1+Phe- , that is more stable ( less reversible ) than the first radical pair , ChlD1+Phe- . This rate is determined by the rates of the primary charge separation ( forming ChlD1+Phe- ) and of its stabilization by secondary electron transfer ( forming PD1+Phe- ) , and hence by the energetics of these electron transfer steps . In the Chl-a-PSII core , the 37 chlorins absorb light between ~660 and~690 nm and are therefore almost isoenergetic to the ChlD1 primary donor absorbing at 680 nm . Given the small energy differences , there is little driving force for downhill \\u2018funnelling\\u2019 of excitation energy to ChlD1 , making it a \\u2018shallow trap\\u2019 . Different models have been proposed to explain the shallowness of the photochemical trap in Chl-a-PSII . In the trap-limited model , the transfer of excitation between pigments is significantly faster than the electron transfer reactions leading to PD1+Phe- formation , and a near-complete equilibration of the excitation energy is established over all pigments , including ChlD1 , with a distribution that is determined by their individual site energies ( van Mieghem et al . , 1995; Schatz et al . , 1987; Dau and Sauer , 1996 ) . This leads to a low population of ChlD1* ( Table 4 ) , that is diminished as a function of the number of quasi-isoenergetic pigments with which it shares the excitation energy . In the transfer-to-trap limited model , the small driving force for downhill \\u2018funnelling\\u2019 of excitation energy to ChlD1 causes kinetic bottlenecks for excitation energy equilibration between the core antenna complexes CP43 and CP47 and for excitation energy transfer from these antennas to the reaction centre ( Jennings et al . , 2000; Pawlowicz et al . , 2007; Raszewski and Renger , 2008; Kaucikas et al . , 2016 ) . In this scenario , there is not a full equilibration of the excitation energy over all pigments , but the relatively slow and reversible energy transfer from the core antennas to the reaction centre still leads to a relatively low population of ChlD1* . Irrespectively of the differences in the details of the kinetic limitation to photochemical trapping between the two models , the common requirement for establishing a high quantum yield of charge separation is a sufficiently large overall energy gap ( ~160 meV , ) between ChlD1* and PD1+Phe- , that comprises the primary charge separation ( ChlD1* \\u2194 ChlD1+Phe- ) and secondary electron transfer ( ChlD1+Phe- \\u2194 PD1+Phe- ) , as shown in Figure 8 . An energy gap of this magnitude is required to avoid rapid recombination to the excited state ChlD1* , thereby limiting the probability of its dissipation via non-photochemical relaxation to the ground state in the antenna ( Raszewski and Renger , 2008; Schatz et al . , 1988 ) . For Chl-d-PSII the antenna system is comparable to that in Chl-a-PSII: all 34 Chl-d molecules , including the primary donor ChlD1 at ~720 nm , are close in wavelength and thus both systems are expected to have comparable ChlD1* population ( Table 4 ) , irrespective of the rate-limitation model assumed . Chl-a-PSII and Chl-d-PSII should therefore have the same energetic requirements to ensure a sufficiently high yield of charge separation . Given that the energy of ChlD1* is ~100 meV lower in Chl-d-PSII than in Chl-a-PSII , the energy level of the second and more stable radical pair , PD1+Phe- , needs to be decreased by ~100 meV in Chl-d-PSII relative to Chl-a-PSII . This corresponds to the published Em of Phe/Phe- ( Allakhverdiev et al . , 2011 ) and to the kinetic data ( Figures 5 and 6 ) , as detailed in the previous section . In A . marina membranes , additional Chl-d containing antenna proteins , which form supercomplexes with PSII cores , have been reported to increase the Chl-d-PSII antenna size by almost 200% ( Chen et al . , 2005 ) . This will likely result in an increased sharing of the excited state , leading to a diminished population of ChlD1* , and thus a bigger requirement for an energy drop between ChlD1* and PD1+Phe- to ensure efficient charge separation . At the same time , the larger near-isoenergetic antenna could also contribute to its higher luminescence , by increasing the probability of ChlD1* decay via radiative emission with respect to photochemical re-trapping ( Rappaport and Lavergne , 2009 ) . This is similar to what happens in plant PSII , where the yield of photochemical trapping of excitation energy is decreased by 10\\u201315% by the association of the Light Harvesting Complex antennas ( Engelmann et al . , 2005 ) . The pigment layout of Chl-f-PSII is very different from that of Chl-a-PSII and Chl-d-PSII . The 30 Chl-a molecules are energetically separated from ChlD1 , absorbing at 720 nm , by >30 nm ( >3 kBT ) . This means the excitation energy resides predominantly on ChlD1* and on the other 4 far-red pigments . If the equilibration of the excitation energy between the 5 far-red pigments were significantly faster than charge separation , this pigment arrangement would result in a higher probability of populating ChlD1* in Chl-f-PSII than in Chl-a-PSII and Chl-d-PSII ( Table 4 ) . The higher ChlD1* population in Chl-f-PSII could ensure that sufficient yield of charge separation is achieved even when the Em of Phe/Phe- is increased by much less that the 100 meV needed to compensate for the nominally lower energy in ChlD1* . However , thermal equilibration of the excitation energy over the entire antenna in Chl-f-PSII might not occur due to 3 of the 4 long-wavelength antenna chlorophylls absorbing at longer wavelength than ChlD1 . This type of antenna energetics could result in rapid excited state equilibration in each of the three main pigment-protein complexes ( CP43 , CP47 and reaction centre ) , due to rapid energy transfer from Chl-a to Chl-f/d ( with visible light excitation ) followed by slower transfer from the two postulated far-red antenna pools to ChlD1 , leading to a transfer-to-trap limited bottleneck . As a result , the kinetics of excitation energy transfer from the red and far-red antenna to the reaction centre could be more complex than in Chl-a-PSII and Chl-d-PSII , explaining the spread in charge separation kinetics that has been suggested based on ultrafast absorption data ( Zamzam et al . , 2020 ) and the slower excitation energy trapping kinetics measured by time-resolved fluorescence ( Mascoli et al . , 2020 ) . The driving force for charge separation is decreased in Chl-f-PSII also by the smaller energy gap between ChlD1* and PD1+Phe- , estimated to be ~80 meV in Chl-f-PSII compared to ~160 meV in Chl-a-PSII and Chl-d-PSII . This decrease in the energy gap between ChlD1* and PD1+Phe- is necessary in Chl-f-PSII to avoid the increased photosensitivity seen in Chl-d-PSII by maintaining a large energy gap between PD1+Phe- and PD1+QA- ( ~385 meV ) ( Figure 8 ) . Nonetheless , the slower excitation energy transfer and the smaller energy gap between ChlD1* and PD1+Phe- could be partially compensated by the decreased dilution of the excitation energy on ChlD1* arising from the small number of long-wavelength antenna pigments , resulting in only a small loss of trapping efficiency ( Mascoli et al . , 2020 ) and a near-negligible effect on enzyme turnover efficiency ( Figures 2\\u20145 ) . This energetic balancing trick in Chl-f-PSII , which allows both reasonably high enzyme efficiency and high resilience to photodamage ( by limiting recombination via the repopulation of PD1+Phe- ) despite working with 100 meV less energy , comes with a very significant disadvantage: its absorption cross-section at long wavelength is ~7 times smaller than that of the Chl-a-PSII core antenna in visible light . In the case of Chl-f-PSII , evolution therefore seems to have prioritized the minimization of harmful charge recombination , by maintaining a big energy gap between Phe and QA , over light collection and photochemical quantum efficiency . This makes sense as this system has evolved as a facultative survival mechanism , that is not advantageous when visible light is available . It must be noted that in vivo the far-red antenna cross-section of Chl-f-PSII is increased by the presence of red-shifted phycobilisomes , that replace the visible light-absorbing phycobilisomes when the cells are adapted to far-red light ( Gan et al . , 2014 ) . In contrast , Chl-d-PSII seems to have maximized light collection at long wavelengths ( with its full-size far-red antenna ) and maximized the yield of charge separation ( by maintaining the full ChlD1* to PD1+Phe- driving force ) . However , the energy shortfall at long wavelength is lost from the \\u2018energy headroom\\u2019 ( mainly from the transmembrane energy gap between Phe and QA ) that is proposed to minimize harmful charge recombination by buffering the effects of pulses of the trans-membrane electric field associated with fluctuations in light intensity ( Davis et al . , 2016; Davis et al . , 2017 ) . This seems to correspond well to the shaded and stable epiphytic niche that A . marina occupies ( N\\u00fcrnberg et al . , 2018; Miyashita et al . , 1996; Cotton et al . , 2015; Davis et al . , 2016; Murakami et al . , 2004; Miller et al . , 2005; K\\u00fchl et al . , 2005; Mohr et al . , 2010 ) . Chl-d-PSII and Chl-f-PSII have evolved different strategies to do oxygenic photosynthesis in far-red light and have been impacted differently by the decrease in energy available . Understanding how the redox tuning of the electron transfer cofactors and the layout of the far-red pigments determine the trade-off between efficiency and resilience in PSII is a necessary step to inform strategies aimed at using far-red photosynthesis for agricultural and biotechnological applications . The present findings suggest the exchange of the full Chl-a manifold to long-wavelength chlorophylls , as seen in Chl-d-PSII ( A . marina ) , should allow efficient oxygenic photosynthesis , but only under constant shading and stable light conditions: for example , for cultivation under LED light ( vertical farming , etc ) . The more robust , facultative Chl-f PSII has an intrinsically low absorption cross-section in the far red; however , this could be beneficial in a shaded canopy , especially in combination with a suitably designed far-red external antenna . \\n Acaryochloris marina was grown in a modified liquid K-ESM medium containing 14 \\u00b5M iron ( Bailleul et al . , 2008 ) , at 30 \\u00b0C under constant illumination with far-red light ( 750 nm , Epitex; L750-01AU ) at ~30 \\u03bcmol photons m\\u20132 s\\u20131 . Chroococcidiopsis thermalis PCC7203 was grown in liquid BG11 medium ( Stanier et al . , 1979 ) at 30 \\u00b0C , under two illumination conditions: white light at ~30 \\u03bcmol photons m\\u20132 s\\u20131 ( for WL C . thermalis samples ) and far-red light ( 750 nm , Epitex; L750-01AU ) at ~30 \\u03bcmol photons m\\u20132 s\\u20131 ( for FR C . thermalis samples ) . Synechocystis sp . PCC6803 was grown in liquid BG11 medium at 30 \\u00b0C under constant illumination with white light at ~30 \\u03bcmol photons m\\u20132 s\\u20131 . The Thermosynechococcus elongatus \\u0394psbA1 , \\u0394psbA2 deletion mutant ( WT*3 , Sugiura et al . , 2008 ) was grown in liquid DNT medium at 45 \\u00b0C . Membranes were prepared as described in Appendix 7 , frozen in liquid nitrogen and stored at \\u201380 \\u00b0C until use . Partially purified C . thermalis PSII cores retaining oxygen evolution activity were isolated by anion exchange chromatography using a 40 ml DEAE column . The column was equilibrated with 20 mM MES ( 2- ( N-morpholino ) ethanesulfonic acid ) -NaOH pH 6 . 5 , 5 mM CaCl2 , 5 mM MgCl2 and 0 . 03% ( w/v ) \\u03b2-DM ( n-Dodecyl-\\u03b2-D-maltoside ) and elution was done using a linear gradient of MgSO4 from 0 to 200 mM in 100 min ( in the same buffer conditions as those used to equilibrate the column ) , with a flow rate of 4 ml min\\u20131 . Fractions enriched in PSII were pooled , frozen in liquid nitrogen and stored at \\u201380 \\u00b0C . PSII-PsbA3 cores from T . elongatus WT*3 were purified as previously described ( Sugiura et al . , 2014 ) . Flash-induced chlorophyll fluorescence and its subsequent decay were measured with a fast double modulation fluorimeter ( FL 3000 , PSI , Czech Republic ) . Excitation was provided by a saturating 70 \\u03bcs flash at 630 nm and the decay in QA- concentration was probed in the 100 \\u03bcs to 100 s time region using non-actinic measuring pulses following a logarithmic profile as described in Vass et al . , 1999 . The first measuring point was discarded during the data analysis because it contains a light artefact arising from the tail of the saturating flash used for excitation . Details on the analysis of the fluorescence curves ( based on Crofts and Wraight , 1983; Vass et al . , 1999; Cser et al . , 2008 ) are provided in Appendix 7 . Membrane samples were adjusted to a total chlorophyll concentration of 5 \\u03bcg Chl ml\\u20131 in resuspension buffer , pre-illuminated with room light ( provided by a white fluorescent lamp , ~80 \\u03bcmol photons m\\u20132 s\\u20131 ) for 10 s and then kept in the dark on ice until used for measurements . Measurements were performed at 20 \\u00b0C . Where indicated , 20 \\u00b5M DCMU ( 3- ( 3 , 4-dichlorophenyl ) \\u20131 , 1-dimethylurea ) was used . Thermoluminescence curves and luminescence decay kinetics were measured with a laboratory-built apparatus , described in De Causmaecker , 2018 . Membrane samples were diluted in resuspension buffer to a final concentration of 5 \\u00b5g Chl ml\\u20131 in the case of A . marina and FR C . thermalis and of 10 \\u00b5g ml\\u20131 in the case of WL C . thermalis and Synechocystis . The samples were pre-illuminated with room light ( provided by a white fluorescent lamp , ~80 \\u03bcmol photons m\\u20132 s\\u20131 ) for 10 s and then kept in the dark on ice for at least one hour before the measurements . When used , 20 \\u00b5M DCMU was added to the samples before the pre-illumination step . Excitation was provided by single turnover saturating laser flashes ( Continuum Minilite II , frequency doubled to 532 nm , 5 ns FWHM ) . Details on the measurement conditions and on the analysis of the luminescence decay kinetics are provided in Appendix 7 . Oxygen evolution and consumption rates were measured with a Clark-type electrode ( Oxygraph , Hansatech ) at 25 \\u00b0C . Membrane samples were adjusted to a total chlorophyll concentration of 5 \\u03bcg Chl ml\\u20131 . Illumination was provided by a white xenon lamp filtered with a heat filter plus red filter , emitting 600\\u2013700 nm light at 7100 \\u00b5mol photons m\\u20132 s\\u20131 ( Quantitherm light meter , Hansatech ) . When required , the light intensity was reduced by using neutral density filters ( Thorlabs ) . For PSII maximal oxygen evolution rates , 1 mM DCBQ ( 2 , 5-Dichloro-1 , 4-benzoquinone ) and 2 mM potassium ferricyanide were used as an electron acceptor system . Photoinhibitory illumination was performed at room temperature for 30 min with a laboratory-built red LED ( 660 nm , 2600 \\u00b5mol photons m-2 s\\u20131 ) . For histidine-mediated chemical trapping of singlet oxygen , 20 \\u00b5M DCMU , 5 mM L-Histidine and , where specified , 10 mM sodium azide ( NaN3 ) were used . PSI activity was measured as the rate of oxygen consumption in presence of 20 \\u00b5M DCMU and 100 \\u00b5M methyl viologen using 5 mM sodium ascorbate and 50 \\u00b5M TMPD ( N , N , N\\u2032 , N\\u2032-tetramethyl-p-phenylenediamine ) as electron donors . The dye Rose Bengal was used at a final concentration of 0 . 1 \\u00b5M . After all necessary additions , samples were left to equilibrate with air in the measuring chamber for 1 minute , under stirring , to start with similar dissolved O2 concentrations in all measurements . For each measurement , a dark baseline was recorded for 1 min before starting the illumination . Time-resolved oxygen polarography was performed using a custom-made centrifugable static ring-disk electrode assembly of a bare platinum working electrode and silver-ring counter electrode , as previously described ( Dilbeck et al . , 2012 ) . For each measurement , membranes equivalent to 10 \\u00b5g of total chlorophyll were used . Three different light sources were used to induce the S-state transitions: a red LED ( 613 nm ) , a far-red LED ( 730 nm ) and a Xenon flashlamp . Details on the experimental setup and on the lights used are provided in Appendix 7 . Measurements were performed at 20 \\u00b0C . For each measurement , a train of 40 flashes fired at 900ms time interval was given and the flash-induced oxygen-evolution patterns were taken from the maximal O2 signal of each flash and fitted with an extended Kok model with flash-independent miss factor , as described in N\\u00f6ring et al . , 2008 . UV pump-probe absorption measurements were performed using a lab-built Optical Parametric Oscillator ( OPO ) -based spectrophotometer ( Joliot et al . , 1998 ) with a time resolution of 10 ns and a spectral resolution of 2 nm ( see Appendix 7 for details on the setup ) . \\u0394I/I stands for differential absorption , a method that measures the changes in absorption depending on whether or not a sample is subjected to actinic light . Samples were diluted in resuspension buffer to a final concentration of 25 \\u00b5g Chl ml\\u20131 for isolated C . thermalis and T . elongatus PSII cores and 40 \\u00b5g Chl ml\\u20131 for A . marina membranes . Samples were pre-illuminated with room light ( provided by a white fluorescent lamp , ~80 \\u03bcmol photons m\\u20132 s\\u20131 ) for 10 s and then kept in the dark on ice for at least one hour before the measurements . 100 \\u00b5M PPBQ ( Phenyl-p-benzoquinone ) was added just before the measurements . The sample was refreshed between each train of flashes . For each measurement , a train of 20 flashes ( 6 ns FWHM ) fired at 300ms time interval was given , and absorption changes measured at 100ms after each flash . The data were fitted according to Lavergne , 1991; Lavorel , 1976 .\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"headings\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"keywords\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"id\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 142,\n        \"samples\": [\n          \"elife-78526-v1\",\n          \"elife-79002-v1\",\n          \"elife-79890-v3\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"biomistral_summary\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 142,\n        \"samples\": [\n          \" GENESPACE is a tool for inferring syntenic relationships between genomes and identifying paralogs and orthologs. It uses the DIAMOND2 algorithm to find reciprocal best hits between genomes, and then uses MCScanX to identify syntenic regions. The tool also allows for the identification of tandem arrays and the calculation of gene rank order. The output of GENESPACE is a tab-delimited text file containing information about each gene, including its chromosome, start and end positions, and gene rank order. The tool can also be used to create a wide-formatted data table containing information about each gene by genome.\",\n          \" The study aimed to investigate the effect of calcium supplementation on bone mass in young people before achieving peak bone mass (PBM) and at the plateau of PBM. The authors conducted a systematic review and meta-analysis of 43 randomized controlled trials (RCTs) that included 3283 calcium supplement users and 4099 controls. The results showed that calcium supplementation significantly improved bone mineral density (BMD) and bone mineral content (BMC) at the femoral neck, lumbar spine, total hip, and total body. The effect was more obvious in the peri-PBM period than in the pre-PBM period. The authors also found that calcium supplementation with vitamin D had greater beneficial effects on BMD and BMC than calcium supplementation without vitamin D. The authors concluded that calcium supplementation can significantly improve bone mass in young people before achieving PBM and at the plateau of PBM, and the intervention window of calcium supplementation should be advanced to the age around the plateau of PBM, namely, at 20\\u201335 years of age.\",\n          \" The article describes the functional differences between three types of photosystem II (PSII) from different cyanobacteria species. The authors found that the far-red PSII from A. marina has a higher luminescence than the blue-green PSII from T. elongatus and the green PSII from C. thermalis. They also found that the far-red PSII from A. marina has a higher yield of charge separation than the blue-green PSII from T. elongatus and the green PSII from C. thermalis. The authors suggest that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. They also suggest that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. The authors also found that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. They also suggest that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. The authors also found that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. The authors also found that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. The authors also found that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. The authors also found that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. The authors also found that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. The authors also found that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. The authors also found that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. The authors also found that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. The authors also found that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. The authors also found that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. The authors also found that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. The authors also found that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. The authors also found that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. The authors also found that the far-red PSII from A. marina has a higher energy gap between the excited state of the reaction center and the ground state of the electron acceptor, which allows for a higher yield of charge separation. The authors also found that the far-red PSII from A. marina has a higher energy gap between the excited state of the\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}",
       "type": "dataframe",
       "variable_name": "test_elife_df"
      },
      "text/html": [
       "\n",
       "  <div id=\"df-6b3ccb50-7a6d-4c00-a2ee-8d7396ab2baf\" class=\"colab-df-container\">\n",
       "    <div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>article</th>\n",
       "      <th>headings</th>\n",
       "      <th>keywords</th>\n",
       "      <th>id</th>\n",
       "      <th>biomistral_summary</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Acylation of diverse carbohydrates occurs acro...</td>\n",
       "      <td>[Abstract, Introduction, Results and discussio...</td>\n",
       "      <td>[biochemistry and chemical biology, computatio...</td>\n",
       "      <td>elife-81547-v1</td>\n",
       "      <td>The OafB protein is a member of the acyltrans...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Honey bee ecology demands they make both rapid...</td>\n",
       "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
       "      <td>[computational and systems biology]</td>\n",
       "      <td>elife-86176-v2</td>\n",
       "      <td>The authors present a novel sequential sampli...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Biguanides , including the world’s most prescr...</td>\n",
       "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
       "      <td>[genetics and genomics]</td>\n",
       "      <td>elife-82210-v1</td>\n",
       "      <td>The study by Beydoun et al. ( 2021 ) demonstr...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Ecological relationships between bacteria medi...</td>\n",
       "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
       "      <td>[microbiology and infectious disease, ecology]</td>\n",
       "      <td>elife-83152-v2</td>\n",
       "      <td>The manuscript \"Universality of microbial tax...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Gamma oscillations are believed to underlie co...</td>\n",
       "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
       "      <td>[neuroscience]</td>\n",
       "      <td>elife-83044-v2</td>\n",
       "      <td>The article describes a study on the relation...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>\n",
       "    <div class=\"colab-df-buttons\">\n",
       "\n",
       "  <div class=\"colab-df-container\">\n",
       "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-6b3ccb50-7a6d-4c00-a2ee-8d7396ab2baf')\"\n",
       "            title=\"Convert this dataframe to an interactive table.\"\n",
       "            style=\"display:none;\">\n",
       "\n",
       "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
       "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
       "  </svg>\n",
       "    </button>\n",
       "\n",
       "  <style>\n",
       "    .colab-df-container {\n",
       "      display:flex;\n",
       "      gap: 12px;\n",
       "    }\n",
       "\n",
       "    .colab-df-convert {\n",
       "      background-color: #E8F0FE;\n",
       "      border: none;\n",
       "      border-radius: 50%;\n",
       "      cursor: pointer;\n",
       "      display: none;\n",
       "      fill: #1967D2;\n",
       "      height: 32px;\n",
       "      padding: 0 0 0 0;\n",
       "      width: 32px;\n",
       "    }\n",
       "\n",
       "    .colab-df-convert:hover {\n",
       "      background-color: #E2EBFA;\n",
       "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
       "      fill: #174EA6;\n",
       "    }\n",
       "\n",
       "    .colab-df-buttons div {\n",
       "      margin-bottom: 4px;\n",
       "    }\n",
       "\n",
       "    [theme=dark] .colab-df-convert {\n",
       "      background-color: #3B4455;\n",
       "      fill: #D2E3FC;\n",
       "    }\n",
       "\n",
       "    [theme=dark] .colab-df-convert:hover {\n",
       "      background-color: #434B5C;\n",
       "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
       "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
       "      fill: #FFFFFF;\n",
       "    }\n",
       "  </style>\n",
       "\n",
       "    <script>\n",
       "      const buttonEl =\n",
       "        document.querySelector('#df-6b3ccb50-7a6d-4c00-a2ee-8d7396ab2baf button.colab-df-convert');\n",
       "      buttonEl.style.display =\n",
       "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
       "\n",
       "      async function convertToInteractive(key) {\n",
       "        const element = document.querySelector('#df-6b3ccb50-7a6d-4c00-a2ee-8d7396ab2baf');\n",
       "        const dataTable =\n",
       "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
       "                                                    [key], {});\n",
       "        if (!dataTable) return;\n",
       "\n",
       "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
       "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
       "          + ' to learn more about interactive tables.';\n",
       "        element.innerHTML = '';\n",
       "        dataTable['output_type'] = 'display_data';\n",
       "        await google.colab.output.renderOutput(dataTable, element);\n",
       "        const docLink = document.createElement('div');\n",
       "        docLink.innerHTML = docLinkHtml;\n",
       "        element.appendChild(docLink);\n",
       "      }\n",
       "    </script>\n",
       "  </div>\n",
       "\n",
       "\n",
       "<div id=\"df-a8735176-25d2-48c0-9f35-3133b7fff6ae\">\n",
       "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-a8735176-25d2-48c0-9f35-3133b7fff6ae')\"\n",
       "            title=\"Suggest charts\"\n",
       "            style=\"display:none;\">\n",
       "\n",
       "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
       "     width=\"24px\">\n",
       "    <g>\n",
       "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
       "    </g>\n",
       "</svg>\n",
       "  </button>\n",
       "\n",
       "<style>\n",
       "  .colab-df-quickchart {\n",
       "      --bg-color: #E8F0FE;\n",
       "      --fill-color: #1967D2;\n",
       "      --hover-bg-color: #E2EBFA;\n",
       "      --hover-fill-color: #174EA6;\n",
       "      --disabled-fill-color: #AAA;\n",
       "      --disabled-bg-color: #DDD;\n",
       "  }\n",
       "\n",
       "  [theme=dark] .colab-df-quickchart {\n",
       "      --bg-color: #3B4455;\n",
       "      --fill-color: #D2E3FC;\n",
       "      --hover-bg-color: #434B5C;\n",
       "      --hover-fill-color: #FFFFFF;\n",
       "      --disabled-bg-color: #3B4455;\n",
       "      --disabled-fill-color: #666;\n",
       "  }\n",
       "\n",
       "  .colab-df-quickchart {\n",
       "    background-color: var(--bg-color);\n",
       "    border: none;\n",
       "    border-radius: 50%;\n",
       "    cursor: pointer;\n",
       "    display: none;\n",
       "    fill: var(--fill-color);\n",
       "    height: 32px;\n",
       "    padding: 0;\n",
       "    width: 32px;\n",
       "  }\n",
       "\n",
       "  .colab-df-quickchart:hover {\n",
       "    background-color: var(--hover-bg-color);\n",
       "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
       "    fill: var(--button-hover-fill-color);\n",
       "  }\n",
       "\n",
       "  .colab-df-quickchart-complete:disabled,\n",
       "  .colab-df-quickchart-complete:disabled:hover {\n",
       "    background-color: var(--disabled-bg-color);\n",
       "    fill: var(--disabled-fill-color);\n",
       "    box-shadow: none;\n",
       "  }\n",
       "\n",
       "  .colab-df-spinner {\n",
       "    border: 2px solid var(--fill-color);\n",
       "    border-color: transparent;\n",
       "    border-bottom-color: var(--fill-color);\n",
       "    animation:\n",
       "      spin 1s steps(1) infinite;\n",
       "  }\n",
       "\n",
       "  @keyframes spin {\n",
       "    0% {\n",
       "      border-color: transparent;\n",
       "      border-bottom-color: var(--fill-color);\n",
       "      border-left-color: var(--fill-color);\n",
       "    }\n",
       "    20% {\n",
       "      border-color: transparent;\n",
       "      border-left-color: var(--fill-color);\n",
       "      border-top-color: var(--fill-color);\n",
       "    }\n",
       "    30% {\n",
       "      border-color: transparent;\n",
       "      border-left-color: var(--fill-color);\n",
       "      border-top-color: var(--fill-color);\n",
       "      border-right-color: var(--fill-color);\n",
       "    }\n",
       "    40% {\n",
       "      border-color: transparent;\n",
       "      border-right-color: var(--fill-color);\n",
       "      border-top-color: var(--fill-color);\n",
       "    }\n",
       "    60% {\n",
       "      border-color: transparent;\n",
       "      border-right-color: var(--fill-color);\n",
       "    }\n",
       "    80% {\n",
       "      border-color: transparent;\n",
       "      border-right-color: var(--fill-color);\n",
       "      border-bottom-color: var(--fill-color);\n",
       "    }\n",
       "    90% {\n",
       "      border-color: transparent;\n",
       "      border-bottom-color: var(--fill-color);\n",
       "    }\n",
       "  }\n",
       "</style>\n",
       "\n",
       "  <script>\n",
       "    async function quickchart(key) {\n",
       "      const quickchartButtonEl =\n",
       "        document.querySelector('#' + key + ' button');\n",
       "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
       "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
       "      try {\n",
       "        const charts = await google.colab.kernel.invokeFunction(\n",
       "            'suggestCharts', [key], {});\n",
       "      } catch (error) {\n",
       "        console.error('Error during call to suggestCharts:', error);\n",
       "      }\n",
       "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
       "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
       "    }\n",
       "    (() => {\n",
       "      let quickchartButtonEl =\n",
       "        document.querySelector('#df-a8735176-25d2-48c0-9f35-3133b7fff6ae button');\n",
       "      quickchartButtonEl.style.display =\n",
       "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
       "    })();\n",
       "  </script>\n",
       "</div>\n",
       "\n",
       "    </div>\n",
       "  </div>\n"
      ],
      "text/plain": [
       "                                             article  \\\n",
       "0  Acylation of diverse carbohydrates occurs acro...   \n",
       "1  Honey bee ecology demands they make both rapid...   \n",
       "2  Biguanides , including the world’s most prescr...   \n",
       "3  Ecological relationships between bacteria medi...   \n",
       "4  Gamma oscillations are believed to underlie co...   \n",
       "\n",
       "                                            headings  \\\n",
       "0  [Abstract, Introduction, Results and discussio...   \n",
       "1  [Abstract, Introduction, Results, Discussion, ...   \n",
       "2  [Abstract, Introduction, Results, Discussion, ...   \n",
       "3  [Abstract, Introduction, Results, Discussion, ...   \n",
       "4  [Abstract, Introduction, Results, Discussion, ...   \n",
       "\n",
       "                                            keywords              id  \\\n",
       "0  [biochemistry and chemical biology, computatio...  elife-81547-v1   \n",
       "1                [computational and systems biology]  elife-86176-v2   \n",
       "2                            [genetics and genomics]  elife-82210-v1   \n",
       "3     [microbiology and infectious disease, ecology]  elife-83152-v2   \n",
       "4                                     [neuroscience]  elife-83044-v2   \n",
       "\n",
       "                                  biomistral_summary  \n",
       "0   The OafB protein is a member of the acyltrans...  \n",
       "1   The authors present a novel sequential sampli...  \n",
       "2   The study by Beydoun et al. ( 2021 ) demonstr...  \n",
       "3   The manuscript \"Universality of microbial tax...  \n",
       "4   The article describes a study on the relation...  "
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test_elife_df[\"biomistral_summary\"] = test_elife_df[\"article\"].apply(lambda text: lay_summarize(text))\n",
    "test_elife_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "id": "9rynahS7ocCN"
   },
   "outputs": [],
   "source": [
    "def write_to_txt(df, output_path, output_file_txt):\n",
    "  # write the baseline_summary column to txt file\n",
    "  txt_df = df['biomistral_summary']\n",
    "  txt_df.to_csv(output_path+output_file_txt,\n",
    "                index=False,\n",
    "                header=False,\n",
    "                sep=\"\\n\"\n",
    "              )\n",
    "  print(f\"Output {output_file_txt} completed\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "id": "Bm2cY8PLpdDb",
    "outputId": "7ba9d01c-9990-483e-97f2-ff9b8151d520"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Output plos.txt completed\n",
      "Output elife.txt completed\n"
     ]
    }
   ],
   "source": [
    "output_path = '/content/drive/MyDrive/BioLaySumm2024_main/model_outputs/BioMistral-7B-DARE-4bit/mini_dev/'\n",
    "output_file_txt = \"plos.txt\"\n",
    "\n",
    "write_to_txt(mini_plos_df, output_path, output_file_txt)\n",
    "\n",
    "output_file_txt = \"elife.txt\"\n",
    "\n",
    "write_to_txt(mini_elife_df, output_path, output_file_txt)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "id": "50HCvthU6r3K",
    "outputId": "21af65f7-81b9-43c1-c317-e0f8e80a472d"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Output plos.txt completed\n",
      "Output elife.txt completed\n"
     ]
    }
   ],
   "source": [
    "output_path = '/content/drive/MyDrive/BioLaySumm2024_main/model_outputs/BioMistral-7B-DARE-4bit/test/'\n",
    "output_file_txt = \"plos.txt\"\n",
    "\n",
    "write_to_txt(test_plos_df, output_path, output_file_txt)\n",
    "\n",
    "output_file_txt = \"elife.txt\"\n",
    "\n",
    "write_to_txt(test_elife_df, output_path, output_file_txt)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "id": "JBMzrPMZdXF9",
    "outputId": "5f35022e-ac39-4c84-edbc-3e404bd828e8"
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Processed prompts: 100%|██████████| 1/1 [00:19<00:00, 19.32s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "Gamma oscillations are believed to underlie cognitive processes by shaping the formation of transient neuronal partnerships on a millisecond scale . These oscillations are coupled to the phase of breathing cycles in several brain areas , possibly reflecting local computations driven by sensory inputs sampled at each breath . Here , we investigated the mechanisms and functions of gamma oscillations in the piriform ( olfactory ) cortex of awake mice to understand their dependence on breathing and how they relate to local spiking activity . Mechanistically , we find that respiration drives gamma oscillations in the piriform cortex , which correlate with local feedback inhibition and result from recurrent connections between local excitatory and inhibitory neuronal populations . Moreover , respiration-driven gamma oscillations are triggered by the activation of mitral/tufted cells in the olfactory bulb and are abolished during ketamine/xylazine anesthesia . Functionally , we demonstrate that they locally segregate neuronal assemblies through a winner-take-all computation leading to sparse odor coding during each breathing cycle . Our results shed new light on the mechanisms of gamma oscillations , bridging computation , cognition , and physiology . \n",
      " Since the pioneer works of Adrian , 1942 and Bressler and Freeman , 1980 , gamma oscillations have been one of the most studied brain rhythms ( Bastos et al . , 2020; Bastos et al . , 2015; Bragin et al . , 1995; Buzsáki and Wang , 2012; Csicsvari et al . , 2003; Fries et al . , 2007; Gray et al . , 1989; Sirota et al . , 2008; Vinck et al . , 2010; Womelsdorf et al . , 2012; Womelsdorf et al . , 2006 ) . Gamma is believed to be critical for a variety of cognitive functions such as sensory processing ( Fries et al . , 2001 ) , memory ( Fernández-Ruiz et al . , 2021 ) , navigation ( Colgin et al . , 2009 ) , and conscious awareness ( Rodriguez et al . , 1999 ) . At the cellular scale , gamma rhythms modulate spiking activity , shaping the formation of transient neuronal partnerships , the so-called cell assemblies ( Buzsáki , 2010 ) . Computational models show that gamma depends on local inhibitory-inhibitory or excitatory-inhibitory interactions ( Tort et al . , 2007; Wang and Rinzel , 1992 ) and ultimately emerges from synchronous inhibitory postsynaptic potentials ( Buzsáki and Wang , 2012 ) . However , despite the initial evidence for their underlying principles being general , gamma oscillations are not a monolithic entity but actually encompass a diversity of rhythms observed experimentally ( Lopes-Dos-Santos et al . , 2018; Scheffer-Teixeira et al . , 2012; Schomburg et al . , 2014; Zhong et al . , 2017 ) . This warrants studying the mechanisms and functions of these oscillations in specific brain regions in vivo . Gamma oscillations usually appear nested within slower rhythms , a phenomenon known as cross-frequency coupling , in which the amplitude of gamma waxes and wanes depending on the phase of a slow oscillation ( Canolty and Knight , 2010; Lisman and Jensen , 2013 ) . Important examples are the coupling of specific gamma sub-bands to the hippocampal theta rhythm ( Cavelli et al . , 2020; Sirota et al . , 2008; Tort et al . , 2009; Tort et al . , 2008 ) or to the phase of breathing cycles ( Cavelli et al . , 2018; Ito et al . , 2014; Zhong et al . , 2017 ) . Regarding the latter , it is worth noting that respiration-entrained brain rhythms depend on nasal airflow and are not a consequence of the respiratory pattern generation in the brainstem ( Lockmann et al . , 2016; Moberly et al . , 2018; Yanovsky et al . , 2014 ) . Therefore , it seems likely that respiration-entrained gamma activity arises from local computations driven by sensory inputs sampled at each breath and thus plays a major role in cognition . A promising area to study this hypothesis is the piriform cortex ( PCx ) , which constitutes the primary olfactory area in the rodent brain ( Bolding and Franks , 2018a; Stettler and Axel , 2009 ) and exhibits prominent gamma oscillations ( Bressler and Freeman , 1980; Courtiol et al . , 2019; Freeman , 1960; Freeman and Skarda , 1985; Kay et al . , 2009; Kay and Freeman , 1998; Litaudon et al . , 2008; Mori et al . , 2013; Vanderwolf , 2000 ) . Most of our understanding of piriform oscillations comes from the studies of Walter Freeman in the 20th century ( Barrie et al . , 1996; Eeckman and Freeman , 1990; Freeman , 1968; Freeman , 1960; Freeman , 1959 ) . Freeman characterized gamma activity in terms of its topography , frequency range , and relationship to unitary activity and behavior . These observations led him to hypothesize that these oscillations constitute a fundamental sensory processing component that emerges from an excitatory-inhibitory feedback loop . However , despite his influential insights , Freeman’s conjectures could not be conclusively tested due to the technological limitations of his time . Thus , we still lack compelling experimental demonstrations of how gamma generation depends on the interactions between the different piriform neuronal subpopulations or how gamma relates to odor representations encoded as cell assemblies . Therefore , the mechanisms of gamma oscillations in the PCx and their functional role in olfaction need to be studied under the lens of modern experimental and analytical tools ( Courtiol et al . , 2019; Kay et al . , 2009; Mori et al . , 2013 ) . In this report , we study gamma oscillations in the PCx of the awake mouse . We took advantage of modern genetic tools , which accurately identify neuronal populations and precisely modify the local connectivity of the PCx , thus enabling an unprecedented study of the mechanisms and functions of its oscillatory activity . We found that respiration drives gamma oscillations in this region , which derive from feedback inhibition and depend on recurrent connections between local excitatory and feedback inhibitory populations . This loop is triggered by the projection of mitral/tufted cells in the olfactory bulb onto the principal cells of the PCx . As functional consequences , we show that respiration-driven gamma oscillations determine odor-assembly representations through a winner-take-all computation taking place within breathing cycles . \n",
      " To understand the mechanisms and functions of gamma oscillations and their relationship with respiration in the mouse brain , we analyzed local field potentials ( LFP ) from the PCx recorded simultaneously with the respiration signal ( Figure 1A ) during odorless cycles , hereafter referred to as ‘spontaneous’ activity . The dataset was collected by Bolding and Franks , 2018a and generously made available through CRCNS ( http://crcns . org , pcx-1 dataset ) . Figure 1B depicts the LFP and respiration signal from a representative animal; notice that low-gamma oscillations ( 30–60 Hz ) emerge following inhalation start . These low-gamma oscillations , already evident in raw recordings , are part of a larger gamma peak ( 30–100 Hz ) in the LFP power spectrum ( Figure 1C ) , reflecting a true oscillation ( Yuval-Greenberg et al . , 2008 ) . Consistent with Figure 1B , only the low-gamma sub-band couples to the respiration cycle across animals , as its amplitude is modulated by both a 2–3 Hz LFP rhythm coherent to respiration ( Figure 1D top panel; see also Figure 1—figure supplements 1–3 ) and by respiration itself ( Figure 1D bottom panel ) . Thus , respiration entrains low-gamma oscillations in the PCx . To further characterize the interaction between respiration and low-gamma oscillations in the PCx , we performed directionality analyses ( Figure 1E and Figure 1—figure supplement 4 ) . The gamma amplitude envelope showed a peak ~200 ms following the inhalation start , coinciding with a large positive cross-correlation peak between these signals , which suggests that respiration causes gamma . Consistent with these results , time-domain Granger causality was significantly higher in the respiration→gamma direction than in the opposite one ( t ( 12 ) = 9 . 82 , p<10–6 ) . Together , these results show that respiration drives low-gamma oscillations in the PCx . We analyzed the contribution of the different PCx neuronal populations to network low-gamma oscillations . First , we performed a current source density analysis ( Figure 1F ) , which revealed that these oscillations are generated locally within the piriform circuit and show a phase reversal near layer 2 ( where pyramidal cells are located ) . Next , we classified single units according to the expression of a light-sensitive channelrhodopsin coupled to the vesicular GABA transporter ( VGAT ) . This allowed us to discriminate between VGAT- principal cells and VGAT+ inhibitory interneurons . Additionally , VGAT+ neurons were further classified into feedback inhibitory interneurons ( FBI ) and feedforward interneurons ( FFI ) according to their location relative to the principal cell layer ( FFI are located at layer 1 while FBI tend to be located within the cell layer 2/3; Bolding and Franks , 2018a; Figure 1—figure supplement 5 ) . Upon averaging the activity of each neuronal subpopulation , we found that the time course of FBI firing rate changes correlates with the amplitude of low-gamma oscillations in time ( Figure 1G ) . In contrast , principal cells and FFI spike earlier within the respiratory cycle and return to baseline during the gamma amplitude peak ( Figure 1G ) . Notice further that the gamma peak coincides with principal cell inhibition , as evidenced by their firing rate decrease . Moreover , by computing spike-triggered gamma envelope averages , we confirmed that gamma oscillations closely follow FBI spiking ( Figure 1G inset ) . Therefore , we conclude that respiration-driven low-gamma oscillations in the PCx arise from feedback inhibition . We next studied the circuit mechanisms responsible for the respiration-driven gamma oscillations in the PCx . To that end , we analyzed spontaneous PCx LFPs following the selective expression of the tetanus toxin light chain in principal cells of a targeted hemisphere ( TeLC ipsi; Figure 2A ) . Under this approach , TeLC expression blocks excitatory synaptic transmission without affecting cellular excitability ( Bolding and Franks , 2018a ) , allowing us to study the local computations underlying the respiration-driven gamma . Figure 2B shows that TeLC ipsi LFPs had a large significant reduction in low-gamma oscillations with respect to either control mice ( t ( 19 ) = 4 . 11 , p<0 . 001 ) or the contralateral PCx ( TeLC contra , not infected; t ( 12 ) = 2 . 99 , p=0 . 0055 ) . Moreover , TeLC ipsi LFPs also showed a significant decrease in respiration-gamma coupling compared to control ( t ( 19 ) = 5 . 87 , p<10–5 ) or TeLC contra LFPs ( t ( 12 ) = 3 . 10 , p=0 . 0045 ) ( Figure 2C and D ) , despite the respiration signal still reaching the PCx ( note the dotted white traces in Figure 2C showing LFP-Resp coherence ) . These results demonstrate that gamma oscillations and their coupling to respiration depend on local recurrent excitatory connections within the PCx . Noteworthy , respiration-driven low-gamma oscillations also depend on the cognitive state since ketamine/xylazine anesthesia abolishes them ( Figure 2—figure supplement 1 , Appendix 1 ) . After confirming that respiration-driven gamma oscillations depend on recurrent connections formed by principal cells , we asked how PCx inputs affect gamma generation . We expected mitral/tufted cell activation in the olfactory bulb ( OB ) to trigger similar low-gamma oscillations since these projections convey the respiratory inputs to the PCx ( Pashkovski et al . , 2020 ) . Consistently , optogenetic activation of the OB ( Thy-Control ) triggered piriform low-gamma oscillations , which matched the laser time course ( Figure 3A , top panel ) . Interestingly , TeLC ipsi LFPs showed almost no gamma activity following laser onset ( Figure 3A , middle panel ) , while TeLC contra LFPs still exhibited gamma activity upon stimulation of the OB ( Figure 3A , bottom panel ) . Analyzing the group response , we found a significant reduction of low-gamma activity following the laser onset in TeLC ipsi LFPs compared to control ( t ( 17 ) = 5 . 12 , p<0 . 0001 ) or TeLC contra LFPs ( t ( 19 ) = 3 . 38 , p=0 . 0015 ) ( Figure 3B and C ) , further confirming the importance of recurrent connections for gamma generation . Moreover , low-gamma power in control and TeLC contra LFPs increased with laser intensity , while it remained constant in TeLC ipsi LFPs ( Figure 3D ) . Nevertheless , it should be noted that the contralateral TeLC hemisphere showed lower amplitude gamma oscillations following light stimulation than control recordings , though whether this gamma difference is related to an impaired network interplay between both hemispheres or to genetic differences between mouse lines remains to be determined . In any event , these results experimentally prove two critical facts about respiration-driven piriform gamma oscillations . First , that respiration drives PCx low-gamma oscillations mediated by OB projections from mitral/tufted cells . Second , feedforward interneurons do not generate low gamma , which necessarily requires the principal cells to excite local feedback interneurons . Having studied the mechanisms of spontaneous respiration-driven low-gamma oscillations in awake mice , we next analyzed their behavior during odor sampling ( Figure 4 ) . First , we noted that odors elicited large beta oscillations ( 10–30 Hz ) , which have been widely associated with olfactory processing ( Lepousez and Lledo , 2013; Martin et al . , 2006; Poo and Isaacson , 2009 ) . Interestingly , the amplitude of gamma oscillations was not affected by odor delivery , but the duration of gamma activity increased notoriously ( Figure 4B ) . Of note , the increase in gamma duration specifically depended on odor stimulation and not on longer breaths as previously suggested by Vanderwolf , 2000 , since longer odorless cycles did not prolong the induced gamma activity ( Figure 4—figure supplement 1 ) . Notably , beta oscillations occurred 100 ms before gamma onset , and , moreover , showed a consistent phase resetting during each sniff cycle ( Figure 4C and D top ) , resulting in a large amplitude envelope of the inhalation-trigged average of beta-filtered LFPs ( Figure 4D middle ) , which is to say that beta was evoked at each cycle . The unfamiliar reader is referred to Tallon-Baudry and Bertrand , 1999 for a discussion about evoked vs . induced oscillations . In short , an oscillatory activity that shows up in the average filtered trace is said to be evoked since this requires phase consistency ( or ‘resetting’ ) following each stimulus . On the other hand , oscillations that increase in amplitude following each stimulus ( in our case , an inhalation ) , but that exhibit phase jitters from trial to trial , cannot be properly detected in the averaged trace due to peak-trough cancellations across trials . Such oscillations , referred to as ‘induced , ’ can only be detected by inspecting the average across all individual trial amplitudes ( or spectrogram; see Tallon-Baudry and Bertrand , 1999 ) . This is the case of piriform low-gamma oscillations ( Figure 4D bottom ) since they increase following each inhalation but exhibit time jitters in peak activity from cycle to cycle and no phase resetting ( Figure 4D top ) . That is , our results show that gamma oscillations are not evoked but induced at each sniff cycle , contrasting therefore , with the evoked beta oscillations . We also investigated how beta and gamma responses depended on odor identity and concentration ( Figure 4E ) . Different odorants triggered similar beta and gamma responses ( Figure 4E top ) . The amplitude of beta oscillations depended on concentration ( F ( 3 , 12 ) =5 . 84 , p=0 . 01 ) , while gamma amplitude did not though its variability increased ( F ( 3 , 12 ) =2 . 89 , p=0 . 079; Figure 4E bottom ) . Importantly , when comparing TeLC-infected hemispheres with the contralateral ones during odor delivery , we found that only odor-induced gamma oscillations depended on the local piriform recurrent connections , while beta oscillations were still present in infected animals ( Figure 4F ) . These results suggest that beta oscillations do not relate to local piriform computations and are likely of OB postsynaptic origin . Next , we studied how gamma oscillations influenced piriform spiking patterns during odor processing . First , we compared the firing rate of each piriform neuronal subtype during odorless and odor cycles ( Figure 5A ) . Notably , feedback interneurons showed the most pronounced changes during odor cycles , substantially increasing their firing rate and prolonging their spiking period above baseline , thus mirroring the increase in gamma duration . Nonetheless , we observed that the increased FBI spiking ( ~100 ms ) preceded gamma amplitude increase ( ~200 ms; c . f . Figures 4B and 5A ) . This result may be related to the time required for different FBIs to synchronize their activity and generate gamma , as observed in computational models of gamma generation ( Wang and Buzsáki , 1996 ) . We then studied how gamma oscillations influence principal cell spiking during odor delivery , pooling together all recorded principal cells and analyzing their spiking as a function of the respiration phase . We found that the respiration phase modulated principal cell spiking within breathing cycles ( Figure 5B ) , though the preferred spiking phase differed across neurons . Interestingly , while a large proportion of cells were inhibited during the same respiration phase as the maximal gamma amplitude , some cells increased their spiking coincidently with the gamma peak . The respiratory phase preference was stable throughout the recording session ( Figure 5C , top panel; t ( 857 ) = –0 . 18 , p=0 . 57 ) , and respiratory modulation of principal cell spiking did not change between odor and odorless cycles ( Figure 5C , middle panel; t ( 857 ) = –0 . 57 , p=0 . 71 ) . Remarkably , spike-gamma phase coupling increased during the processing of odors ( Figure 5C , bottom panel; t ( 857 ) = 9 . 49 , p<10–19 ) , suggesting that these prolonged oscillations play a role in shaping odor coding . Consistently , we found that odor context determined which cells fired during the gamma oscillation ( Figure 5D ) . We further confirmed this observation by measuring the spiking specificity to odors , which closely followed the gamma envelope ( Figure 5E ) . Thus , these results demonstrate that gamma inhibition shapes single-cell responses during olfaction . The gamma spiking specificity supports the conjecture that respiration-driven gamma oscillations could mediate odor assembly representations . To further study this possibility , we analyzed cell assembly compositions for each odor by measuring the contribution of each principal cell to the first independent component ( IC ) of the population response ( El-Gaby et al . , 2021; Lopes-dos-Santos et al . , 2013; Trouche et al . , 2016 ) . We found highly skewed distributions for each odor , where only a small fraction of neurons showed a strong positive contribution to the 1st IC , hereafter referred to as winner cells , while the vast majority showed low weights , referred to as loser cells ( Figure 6A and B ) . Notably , the few winning neurons determining the 1st IC activity changed from odor to odor ( Figure 6A ) ; in other words , there was strong orthogonality in the 1st IC weight distribution across different odors . Consistent with this , the 1st IC weights were not significantly correlated between odors ( corrected by multiple comparisons ) ( Figure 6C ) . We note that a similar odor separation was achieved when computing ICA on all odors together and analyzing the top six independent components ( Figure 6—figure supplement 1 ) . Interestingly , the losing cells were significantly more phase-locked to the gamma phase ( Figure 6D top; t ( 14 ) = 5 . 88 , p<10–4 ) , consistent with them being actively inhibited during the oscillation . There were no differences between the preferred gamma phase for winners and losers ( Figure 6D bottom; t ( 14 ) = 0 . 05 , p=0 . 95 ) . Moreover , when analyzing the spiking time course of winner and loser cells , we found that both groups tended to spike immediately following inhalation start; nevertheless , once feedback inhibition was recruited , loser cells spiked below their average while the winner cells continued spiking above baseline ( Figure 6E ) . Notably , loser cell inhibition correlated with gamma activity , which suggests that sparse odor assembly representations emerge from a winner-take-all process mediated by inhibitory gamma oscillations . Because loser cell inhibition correlated with low gamma activity , we expected the winner-take-all process to be affected in TeLC recordings . Thus , we analyzed cell assemblies in both infected and contralateral hemispheres . We found that the TeLC-infected piriform cortex showed significantly less skewed distributions ( t ( 5 ) = –3 . 08 , p=0 . 013 ) with a decreased number of losing cells ( t ( 5 ) = –2 . 10 , p=0 . 044 ) , which were significantly less inhibited than their contralateral counterparts ( t ( 5 ) = –2 . 22 , p=0 . 038; Figure 6F and G ) . Furthermore , the reduced inhibition of losing cells was associated with a high correlation of assembly weights across odors for the affected hemisphere ( t ( 5 ) = 2 . 63 , p=0 . 023; Figure 6H ) , suggesting that gamma oscillations are necessary for the segregation of odor-selective assemblies . Because a winner-take-all process selects a single odor representation while actively suppressing others , it effectively implements an XOR logic gate . As a consequence , we would expect odor decoding accuracy to increase during gamma inhibition . Following this reasoning , we first analyzed the correlation of population spiking vectors in response to different odorants and found that correlations significantly dropped during the gamma peak ( Gamma vs . Pre: t ( 14 ) = –3 . 32 , p=0 . 002; Gamma vs . Post: t ( 14 ) = –3 . 73 , p=0 . 001; Figure 7A ) , confirming that odor representations diverge the most during gamma inhibition . We next tested odor decoding from the principal cell activity by training a supervised linear classifier using population spiking ( i . e . spike vectors ) at different respiratory cycle phases . Consistent with a winner-take-all mechanism , we found that variations in odor decoding accuracy matched the low-gamma activity time course ( Figure 7B and C ) ; there was a significant correlation between these variables ( r=0 . 45 , p<10–12 ) . Importantly , decoding accuracy significantly increased around the gamma peak compared to time windows before or after it ( Gamma vs . Pre: t ( 14 ) = 9 . 47 , p<10–7; Gamma vs . Post: t ( 14 ) = 7 . 56 , p<10–5; see also Figure 7—figure supplement 1 ) . This result was robust among different classification algorithms ( Figure 7—figure supplement 1 ) . In addition , odor decoding accuracy significantly fluctuated within the gamma cycle ( Figure 7D ) , confirming that gamma inhibition directly modulates odor processing . Finally , when analyzing the TeLC experiments , we found that decoding accuracy significantly decreased in the 150–250 ms ‘gamma’ time window in the infected hemisphere compared to the contralateral one ( t ( 5 ) = –2 . 41 , p=0 . 03 ) . In all , these findings demonstrate that respiration-driven gamma oscillations provide an optimal temporal window for olfaction . \n",
      " The present report demonstrates the mechanisms and functions of gamma oscillations in the PCx of awake mice . We show that respiratory inputs to this cortex elicit large feedback inhibitory low-gamma oscillations that aid odor coding . Our work provides critical information for understanding gamma oscillations in the olfactory system . First , the results support the excitatory-inhibitory models of gamma generation ( Buzsáki and Wang , 2012; Fisahn et al . , 1998; Freeman , 1964 ) , highlighting the role of excitatory neurons in initiating the gamma cycle by driving feedback interneuron spiking . Our findings thus experimentally demonstrate several of Walter Freeman’s hypotheses regarding gamma generation ( Bressler and Freeman , 1980; Eeckman and Freeman , 1990; Freeman , 1968 ) , and reveal similarities with the mechanisms generating gamma in the OB ( Fukunaga et al . , 2014; Fukunaga et al . , 2012; Kay et al . , 2009; Lepousez and Lledo , 2013; Mori et al . , 2013 ) . Second , we demonstrate that the PCx low-gamma oscillations are the extracellular correlates of a winner-take-all process , thus confirming previous theoretical conjectures ( de Almeida et al . , 2009 ) and linking this brain oscillation to an algorithmic operation ( an XOR logic gate ) . Third , the winner-take-all interpretation provides a direct relationship between gamma oscillations and cell assemblies . Under this scenario , the low-gamma feedback inhibition is critical for segregating cell assemblies and generating a sparse , orthogonal odor representation . Concomitantly , the assembly recruitment would depend on OB projections determining which winner cells ‘escape’ gamma inhibition , highlighting the relevance of the OB-PCx interplay for olfaction ( Chae et al . , 2022; Otazu et al . , 2015 ) . An important feature of the respiration-driven low-gamma oscillations is that the winners triggering them change according to odor context ( Figure 6A ) , a process likely dependent on odor-encoding mitral cell assemblies from the OB ( Chae et al . , 2022 ) . Thus , the involvement of different cells and spiking dynamics would make the recruitment of feedback inhibition a non-homogenous process across respiration cycles , leading to small time jitters in peak gamma activity . As an empirical end result , the observed low-gamma oscillations are not phase-locked from cycle to cycle; in other words , PCx low-gamma power is induced , but not evoked ( Tallon-Baudry and Bertrand , 1999 ) , at each respiratory cycle . Our results , nonetheless , do not exclude a role for OB gamma in triggering piriform oscillations , as the OB output may arise from similar gamma-dependent winner-take-all processes in that area . Nevertheless , our results make it less likely that piriform gamma is simply inherited from the OB , given that we observe ( 1 ) a local current source , ( 2 ) a tight correlation with FBI spiking , and ( 3 ) that gamma disappears when local recurrent connections are abolished , despite the OB remaining intact . In contrast , beta oscillations were evoked by the respiratory input to the PCx , leading to a prominent phase resetting ~100 ms after the inhalation start ( Figure 4 ) . Further consistent with these results , respiration-triggered beta oscillations did not disappear in the TeLC condition and were close in time to excitatory and FFI spiking ( c . f . Figures 1 , 4 and 5 ) . Interestingly , a previous report showed that odors evoke PCx beta oscillations under urethane anesthesia , which , as here , occurred before global inhibition ( Poo and Isaacson , 2009 ) . Based on this , a role for piriform beta in odor representations has been proposed; nevertheless , here we show that beta oscillations are likely inherited and do not require local computations in the PCx , i . e . , they are not locally generated . The study of the role of the PCx in olfaction has recently been boosted by selective genetic techniques , allowing researchers to examine how the individual cellular components relate to olfaction ( Franks et al . , 2011; Nagappan and Franks , 2021 ) . The PCx , also known as the olfactory cortex , receives inputs from the OB and constructs a robust odor representation ( Bolding et al . , 2020 ) . This representation can be inferred from the population activity and has two major features . On the one hand , it separates odor identity from its concentration ( Bolding and Franks , 2018a; Bolding and Franks , 2017; Roland et al . , 2017 ) . This process depends on large-scale piriform inhibition ( Bolding and Franks , 2018a ) and thus correlates with low-gamma oscillations . On the other hand , the representation groups odors into perceptual categories , not necessarily related to the physical structure of the odor molecules ( Pashkovski et al . , 2020 ) . Surprisingly , these odor assembly representations are not fixed but change with time ( Schoonover et al . , 2021 ) , which might be a particular property of unstructured cortices such as the piriform cortex ( Stettler and Axel , 2009 ) . In light of this evidence , spike-field synchronization might be an essential coding strategy , as neural oscillations might provide an internal signal in which assembly representations can be organized and structured . Thus , we hypothesize that low-gamma oscillations provide a critical time window for odor assembly representations to be formed and modified through winner-take-all computations . The role of neuronal oscillations in shaping spiking patterns has been widely discussed ( Buzsáki , 2010 ) . Among other functions , oscillations are thought to provide syntactical blocks in which spiking activity can be parsed into meaningful words . A reader ( receiver ) area can then decode such neuronal representations by following these syntactic rules . In favor of this hypothesis , our decoding analysis shows that , for a reader neuron , optimal odor decoding occurs during the gamma peak within breathing cycles . A likely explanation for this scenario – based on our results – is that by suppressing the activity of competing representations , gamma oscillations increase the signal-to-background ratio and allow for optimal decoding . Under this framework , maximal decoding occurs after and not during the peak firing rate , coinciding with peak gamma activity and synchronous inhibition . Thus , the inhibited neurons , despite being the most phase-locked to gamma , would not directly encode odor information , while the neurons that escape gamma inhibition would do . Interestingly , and further consistent with our results , a recent study in humans showed that the accurate perception and identification of odors depend on piriform gamma oscillations ( Yang et al . , 2022 ) . Moreover , the behavioral response to odors also seems to depend on piriform gamma , and its suppression induces depressive-like behaviors in mice ( Li et al . , 2022 ) . Therefore , respiratory-triggered gamma oscillations might constitute a fundamental building block for neuronal communication in the olfactory system . Several reports show that the PCx is the source of gamma activity for widespread brain regions , including the striatum ( Carmichael et al . , 2017 ) and the anterior limbic system ( Carmichael et al . , 2019 ) . Although the authors largely attribute such oscillations to piriform volume conduction , recent evidence shows that olfactory gamma synchronizes distant regions as well ( Li et al . , 2022 ) . Based on our results , we postulate that genuine respiration-entrained gamma oscillations in downstream regions are triggered by the output of the winning piriform assemblies , leading to new local gamma winner-take-all processes . This provides a new look into long-range gamma synchronization , emerging naturally if winners generating gamma in one area consistently trigger winners generating gamma in a downstream area . Such a mechanism closely agrees with the theory and experiments reported by Schneider et al . , 2021 , which suggest that oscillatory power and connectivity are the main drivers underlying inter-areal coherence . A central element to this directional view of interregional synchronization is gamma coupling to respiration , as the slow oscillation ensures a flow of information from the most primary olfactory areas ( OB , PCx ) to the higher limbic areas ( prefrontal cortex , amygdala ) . In accordance , a recent report showed that respiration-related brain oscillations drive sparse assemblies in the prefrontal cortex ( Folschweiller and Sauer , 2022 ) . Notably , these authors also found that inhibitory recruitment by assembly members was key for assembly segregation , suggesting a potential role for gamma in that area . Hence , the directional view of synchronization across regions places a major role in cross-frequency interactions driving communication , highlighting the relevance of interregional synchrony modulation by slower rhythms , for instance , theta ( González et al . , 2020 ) or respiration itself ( Cavelli et al . , 2020; González et al . , 2023 ) . In light of our findings , we propose the following model to understand the relationship between spiking activity , neural oscillations , and odor coding in the piriform cortex: ( 1 ) Following each sniff , olfactory information is processed in the OB and is transmitted to the piriform circuit through mitral/tufted cell spiking . These synchronous postsynaptic potentials cause the field beta oscillation , whose amplitude depends on the afferent volley . ( 2 ) The principal cells that better encode the olfactory stimulus excite feedback interneurons . ( 3 ) Feedback interneurons inhibit competing principal cells , causing the field gamma oscillation . ( 4 ) This winner-take-all process segregates cell assemblies and dictates a sparse piriform odor representation . Note that because the piriform cortex normalized odorant concentration ( Bolding and Franks , 2018a ) , the amplitude of gamma oscillations does not change . Hence , gamma oscillations provide an optimal temporal window to decode odor . \n",
      " We analyzed recordings of the PCx generously made available by Bolding and Franks through the Collaborative Research in Computational Neuroscience data-sharing website ( http://crcns . org , pcx-1 dataset ) . The SIMUL , TeLC-PCx , THY , TeLC-THY , and VGAT experiments were used . Detailed descriptions of the experimental procedures can be found in previous publications ( Bolding et al . , 2020; Bolding and Franks , 2018a; Bolding and Franks , 2017 ) . All protocols were approved by the Institutional Animal Care and Use Committee of Duke University . Below we describe the analytical methods employed by us , and , for convenience , also the relevant experimental procedures from the original publications . Adult mice were employed ( >P60 , 20–24 g ) , and housed in single cages on a normal light-dark cycle . For the Cre-dependent TeLC group , offspring of Emx1Cre/Cre breeding pairs were obtained from The Jackson Laboratory ( 005628 ) . For the optogenetic experiments , the mice employed were: adult Thy1ChR2/ChR2-YFP , line 18 ( Thy1-COP4/EYFP , Jackson Laboratory , 007612 ) and VGAT ChR2/+ , line 8 ( Slc32a1-COP4*H134R/EYFP , Jackson Laboratory , 014548 ) . For the combined optogenetics and TeLC expression experiments , adult offspring of Emx1Cre/Cre mice crossed with Thy1ChR2/ChR2-YFP mice were employed . For the TeLC experiments , AAV5-DIO-TeLC-GFP was expressed under CBA control ( 6/7 mice ) or synapsin ( 1/7 mice ) , whose effects were similar and pooled together . Three 500 nL injections in the PCx ( AP , ML , DV:+1 . 8 , 2 . 7 , 3 . 85;+0 . 5 , 3 . 5 , 3 . 8; –1 . 5 , 3 . 9 , 4 . 2; DV measured from brain surface ) were employed to achieve TeLC expression . All recordings took place ~14 days after the promoter injection . All viruses were obtained from the University of North Carolina-Chapel Hill ( UNC Vector Core ) . The electrophysiological signals were recorded using 32-site polytrode acute probes ( A1x32-Poly3-5mm- 25 s-177 , Neuronexus ) with an A32-OM32 adaptor ( Neuronexus ) through a Cereplex digital headstage ( Blackrock Microsystems ) . For the optogenetic identification of GABAergic cells , a fiber-attached polytrode probe was employed ( A1x32-Poly3-5mm-25s-177-OA32LP , Neuronexus ) . Data were acquired at 30 kHz , unfiltered , employing a Cerebus multichannel data acquisition system ( BlackRock Microsystems ) . Respiration and experimental events were acquired at 2 kHz by analog inputs of the Cerebus system . The respiration signal was measured employing a microbridge mass airflow sensor ( Honeywell AWM3300V ) , which was positioned opposite to the animal’s nose . Inhalation generated a negative airflow and thus negative changes in the voltage of the sensor output . A Patchstar Micromanipulator ( Scientifica ) was employed to position the recording probe in the anterior PCx ( 1 . 32 mm anterior and 3 . 8 mm lateral from bregma ) . Recordings were targeted 3 . 5–4 mm ventral from the brain surface at this position , and were further adjusted according to the LFP and spiking activity monitored online . The electrode sites spanned 275 µm along the dorsal-ventral axis . The probe was lowered until an intense spiking band was found , which covered 30–40% of electrode sites near the correct ventral coordinate , thus reflecting the piriform layer II . For the optogenetic experiments stimulating OB cells in Thy1ChR2/ChR2-YFP mice , the optic fiber was placed <500 µm above the OB dorsal surface . Spyking-Circus software was employed to isolate individual units ( https://github . com/spyking-circus ) . All clusters which had more than 1% of ISIs violating the refractory period ( <2 ms ) or appearing contaminated were manually removed . Units that showed both similar waveforms and coordinated refractory periods were merged into a single cluster . The unit position was characterized as the mean electrode position ( across electrodes ) weighted by the amplitude of the unit waveform on each electrode . Stimuli consisted of monomolecular odorants diluted in mineral oil . These were hexanal , ethyl butyrate , ethyl acetate , 2-hexanone , isoamyl acetate , and ethyl tiglate . Odors were presented for one second through an olfactometer controlled by MATLAB scripts and repeated every ten seconds . For all analyses , we used Python 3 with numpy ( https://numpy . org/ ) , scipy ( https://docs . scipy . org/ ) , matplotlib ( https://matplotlib . org/ ) , sklearn ( https://scikit-learn . org/stable/ ) , and statsmodel ( https://www . statsmodels . org/stable/index . html ) libraries . The codes to reproduce all seven figures are freely available at: https://github . com/joaqgonzar/Gamma_Oscillations_PCx; ( copy archived at swh:1:rev:3e01d6b0112e2739fbfb6d0fef2be95f2d48ebd5 ) ( Gonzalez , 2023 ) . Raw LFPs were decimated to a sampling rate of 2000 Hz to match the respiration signal sampling rate , employing the decimate scipy function . This function first low-pass filters the 30 kHz raw data and then downsamples it , avoiding aliasing . For each animal , we used the same channel ( either channel 17 or 16 depending on the headstage configuration ) for all analyses in the SIMUL and TeLC-PCx datasets , though similar results are obtained for the rest of the channels given their high LFP redundancy ( Figure 1—figure supplement 3 ) . Channel 28 was employed for the VGAT datasets . We analyzed both odor-related and spontaneous activity; for the former , we selected all inhalations occurring within one-second following odor delivery , and the latter was obtained as all awake periods without odor delivery . To analyze the LFP spectra , we computed Welch’s modified periodogram using welch scipy function . Specifically , we employed a 1 s moving window with half a window overlap , setting the numerical frequency resolution to 0 . 1 Hz ( by setting the nfft parameter to 10 times the sampling rate ) . All spectra were whitened by multiplying each power value by its associated frequency , thus eliminating the 1 /f trend . For Figure 1B , we computed the wavelet transform ( cwt scipy function ) of the LFP signal using a Morlet wavelet with 0 . 1 Hz resolution . The triggered spectrograms were obtained first by selecting all 500 ms LFP windows following inhalation start ( time stamps provided in the dataset , see Bolding and Franks , 2018a for further details ) and then computing the spectrogram using the spectrogram scipy function . We used 40 ms windows with 82% overlap and a frequency resolution of 0 . 1 Hz . After computing each individual spectrogram , we whitened ( i . e . multiplied by f ) and averaged them to yield the mean spectrogram shown in Figures 1G and 3A . We computed phase-amplitude comodulograms following the methods proposed by Tort et al . , 2010 . We first band-pass filtered the LFP/respiration signal using the eegfilt function ( Delorme and Makeig , 2004 ) adapted for Python 3 ( available at https://github . com/joaqgonzar/Gamma_Oscillations_PCx ( copy archived at swh:1:rev:3e01d6b0112e2739fbfb6d0fef2be95f2d48ebd5 ) , Gonzalez , 2023 ) . We filtered between 20–130 Hz in 10 Hz steps to obtain the higher frequency components , and between 1 and 8 Hz in 1 Hz steps to obtain the slower frequency components . The phase ( angle ) and amplitude time series of the filtered signals were estimated from their analytical representation based on the Hilbert transform . We then binned the phase time series into 18 bins and computed the mean amplitude of the fast signal for each bin . The amount of phase-amplitude coupling was estimated through the modulation index ( MI ) metric ( Tort et al . , 2010 ) , MI = ( Hmax-H ) /Hmax , where Hmax is the maximum possible Shannon entropy for a given distribution ( log ( number of phase bins ) ) and H is the actual entropy of the amplitude distribution . To study the directionality between the gamma envelope and respiration signal , we employed three different strategies: ( 1 ) we averaged the gamma envelope using the inhalation start as a trigger; ( 2 ) we computed the cross-correlogram , using the correlate scipy function , between the gamma envelope and the respiration signal; ( 3 ) we computed Granger Causality estimates , using the grangercausalitytest stats model function , with a 10-order VAR model and employing the log ( F-stat ) as the Granger Causality magnitude ( Geweke , 1982 ) . We first obtained gamma averages by filtering the LFP signal between 30–50 Hz; the amplitude peaks were then identified and used for averaging 100 ms epochs surrounding these timestamps . CSD analysis was obtained by -A+2B-C for adjacent probe sites . To study induced power , we ( 1 ) filtered the recordings ( beta: 10–20 Hz , low gamma: 30–60 Hz ) , ( 2 ) obtained their amplitude through the Hilbert transform , and ( 3 ) computed the inhalation-triggered average . To study evoked power , we ( 1 ) filtered the recordings , ( 2 ) obtained the inhalation-triggered average , and then ( 3 ) estimated the amplitude of the average signal through the Hilbert transform . For obtaining the phase-resetting index of the filtered signals , we used the same procedure as for computing the inter-trial coherence ( Makeig et al . , 2004 ) using 500 ms windows following inhalation start as a ‘trial . ’ All cells were classified de-novo following the protocol described by Bolding and Franks , 2018a . A Wilcoxon rank-sum test was employed to determine laser responsiveness; p<0 . 001 was used to detect VGAT+ neurons . The distinction between FFI and FBI was made according to their location on the probe relative to the principal cell layer ( FFI are located at layer 1 while FBI tend to be located within the cell layer 2/3; Bolding and Franks , 2018a; Figure 1—figure supplement 5 ) . The spike times were rounded to match the LFP resolution ( 0 . 5 ms or 2000 Hz sampling rate ) . After that , we convolved the spike times of each unit with a 10 ms standard deviation Gaussian kernel employing the convolve scipy function , in close similarity with the original publication , which gave rise to a smoothed spiking activity . For the normalized responses ( Figure 1G ) , the spiking response of a neuronal subpopulation ( i . e . excitatory , FFI , or FBI neurons ) was normalized by dividing the firing rate by its average across time bins . We defined the odor specificity index for single cell responses following McNaughton et al . , 1983 as SI = max ( Fx ) / ( Fa +Fb + Fc +Fd + Fe +Ff ) , where F represents firing rate , the subscripts a-f represent different odors , and max ( Fx ) is the firing rate for the odor that generates the largest response . The specificity index time course was obtained using 100 ms sliding windows ( 62 . 5% overlap ) following inhalation start . We employed the ICA sklearn function on concatenated time series of the smoothed principal cell spiking activity using all 1000 ms windows following inhalation start during odor presentation ( 0 . 3% v . /v . concentration ) . Each odor was treated separately . We analyzed only the first independent component , setting n_components = 1 . Winners and losers were identified according to the weight each neuron exerted on the 1st principal component . Namely , neurons with a weight above the 95th percentile were labeled as winners , while neurons with a more negative weight than the IC mean as losers . For representation purposes ( Figure 6B ) , large positive and negative weights were truncated at 0 . 15 and –0 . 05 , respectively . The same procedures were employed for the assembly analyses on TeLC recordings , except that all cells were employed due to the lack of opto-tagging . We note that similar results were obtained in control recordings if all available neurons were included , likely due to excitatory cells far outnumbering inhibitory interneurons . To assess the coupling between spikes and the gamma phase , the mean-vector length of the spiking gamma phases was calculated as MVL=‖1Nk ∑k=1Nke−1φk‖ , where φk is the gamma phase associated to the k-th spike , and Nk is the total number of spikes . The average gamma spiking angle ( stats . circmean ) was computed to obtain the preferred phase . For the spike-triggered amplitude envelope averages , we first obtained the gamma amplitude envelope using the Hilbert transform , and then we computed the averages using –150–300 ms windows surrounding each spike . To infer the similarity/dissimilarity in the population response to the different odors , we analyzed the population vector correlations , defined as the Pearson correlation ( pearsonr scipy function ) measured using paired data from the same neurons to two odors . Correlations were computed between spiking vectors for each odor; the correlation time course shown in Figure 7A bottom was obtained using 100 ms sliding windows ( 62 . 5% overlap ) following inhalation start . For statistics , we averaged the correlation across odors and compared them among three-time windows of interest: from 0 to 100 ms ( Pre ) , from 150 to 250 ms ( Gamma ) , and from 525 to 625 ms ( Post ) . We employed a supervised linear classifier to decode the odor identity ( 0 . 3% v . /v . concentration ) from the population spiking vectors . The classification algorithm was supplied with the average spiking of each principal cell in 100 ms sliding windows following the inhalation start . The classification algorithm was a linear support vector machine with a stochastic gradient descent optimization , implemented using the sgdclassifier sklearn function . Each mouse was trained and tested separately; for each time window , the training sample consisted of 2/3 of the data , and the test sample consisted of the remaining third . The division of training/test samples was randomized in order to avoid odor bias , repeated 100 times , and the results of the repetitions averaged to obtain the final decoding value . For the statistical comparisons , we used the same 100 ms time windows employed in the population vector correlations ( i . e . Pre , Gamma , and Post ) . To verify the robustness of the results , we also employed a linear discriminant analysis and a k-nearest neighbor classifier , which rendered similar results to the sgdclassifier algorithm ( Figure 7—figure supplement 1 ) . We show data as either mean ± SEM or regular boxplots showing the median , 1st , 3rd quartiles , and the distribution range without outliers . We employed paired and unpaired t-test to compare between groups . To compare among odor concentrations , we employed repeated measures ANOVA . We set p<0 . 05 to be considered significant ( in the figure panels , * , ** and *** denote p<0 . 05 , p<0 . 01 , and p<0 . 001 , respectively ) . For the 1st IC weight correlations , we employed Pearson r correlation and corrected the statistical threshold through the Bonferroni method for multiple comparisons . All the data employed is freely available at http://dx . doi . org/10 . 6080/K00C4SZB . Previously Published Datasets: Bolding and Franks , 2018b Collaborative Research in Computational Neuroscience ( http://crcns . org/ ) . ‘Simultaneous extracellular recordings from mice OB and PCx and respiration data in response to odor stimuli and optogenetic stimulation of OB ( https://crcns . org/data-sets/pcx/pcx-1 ) . ’\n",
      "\n",
      "\n",
      " The article describes a study on the relationship between respiration and gamma oscillations in the piriform cortex (PCx) of mice. The authors employed a combination of optogenetic and electrophysiological techniques to investigate the role of inhibitory neurons in gamma oscillations and their relationship with respiration. They found that gamma oscillations are driven by inhibitory neurons and that the phase of gamma oscillations is reset by inhalation. The authors also found that gamma oscillations are involved in the encoding of odor identity and concentration. The study provides new insights into the relationship between respiration and gamma oscillations in the PCx and their role in odor processing.\n",
      "\n",
      "\n",
      "714\n",
      "105\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "\n",
    "s = test_elife_df[\"article\"].iloc[4]\n",
    "\n",
    "laysum = lay_summarize(s)\n",
    "\n",
    "print('\\n')\n",
    "print(s)\n",
    "print('\\n')\n",
    "print(laysum)\n",
    "print('\\n')\n",
    "print(len(laysum))\n",
    "print(len(laysum.split()))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "id": "MVZLpnH3fOJ9",
    "outputId": "8af51216-fae2-4609-c5b8-194de5fee748"
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Processed prompts: 100%|██████████| 1/1 [00:18<00:00, 18.48s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "Gamma oscillations are believed to underlie cognitive processes by shaping the formation of transient neuronal partnerships on a millisecond scale . These oscillations are coupled to the phase of breathing cycles in several brain areas , possibly reflecting local computations driven by sensory inputs sampled at each breath . Here , we investigated the mechanisms and functions of gamma oscillations in the piriform ( olfactory ) cortex of awake mice to understand their dependence on breathing and how they relate to local spiking activity . Mechanistically , we find that respiration drives gamma oscillations in the piriform cortex , which correlate with local feedback inhibition and result from recurrent connections between local excitatory and inhibitory neuronal populations . Moreover , respiration-driven gamma oscillations are triggered by the activation of mitral/tufted cells in the olfactory bulb and are abolished during ketamine/xylazine anesthesia . Functionally , we demonstrate that they locally segregate neuronal assemblies through a winner-take-all computation leading to sparse odor coding during each breathing cycle . Our results shed new light on the mechanisms of gamma oscillations , bridging computation , cognition , and physiology . \n",
      " Since the pioneer works of Adrian , 1942 and Bressler and Freeman , 1980 , gamma oscillations have been one of the most studied brain rhythms ( Bastos et al . , 2020; Bastos et al . , 2015; Bragin et al . , 1995; Buzsáki and Wang , 2012; Csicsvari et al . , 2003; Fries et al . , 2007; Gray et al . , 1989; Sirota et al . , 2008; Vinck et al . , 2010; Womelsdorf et al . , 2012; Womelsdorf et al . , 2006 ) . Gamma is believed to be critical for a variety of cognitive functions such as sensory processing ( Fries et al . , 2001 ) , memory ( Fernández-Ruiz et al . , 2021 ) , navigation ( Colgin et al . , 2009 ) , and conscious awareness ( Rodriguez et al . , 1999 ) . At the cellular scale , gamma rhythms modulate spiking activity , shaping the formation of transient neuronal partnerships , the so-called cell assemblies ( Buzsáki , 2010 ) . Computational models show that gamma depends on local inhibitory-inhibitory or excitatory-inhibitory interactions ( Tort et al . , 2007; Wang and Rinzel , 1992 ) and ultimately emerges from synchronous inhibitory postsynaptic potentials ( Buzsáki and Wang , 2012 ) . However , despite the initial evidence for their underlying principles being general , gamma oscillations are not a monolithic entity but actually encompass a diversity of rhythms observed experimentally ( Lopes-Dos-Santos et al . , 2018; Scheffer-Teixeira et al . , 2012; Schomburg et al . , 2014; Zhong et al . , 2017 ) . This warrants studying the mechanisms and functions of these oscillations in specific brain regions in vivo . Gamma oscillations usually appear nested within slower rhythms , a phenomenon known as cross-frequency coupling , in which the amplitude of gamma waxes and wanes depending on the phase of a slow oscillation ( Canolty and Knight , 2010; Lisman and Jensen , 2013 ) . Important examples are the coupling of specific gamma sub-bands to the hippocampal theta rhythm ( Cavelli et al . , 2020; Sirota et al . , 2008; Tort et al . , 2009; Tort et al . , 2008 ) or to the phase of breathing cycles ( Cavelli et al . , 2018; Ito et al . , 2014; Zhong et al . , 2017 ) . Regarding the latter , it is worth noting that respiration-entrained brain rhythms depend on nasal airflow and are not a consequence of the respiratory pattern generation in the brainstem ( Lockmann et al . , 2016; Moberly et al . , 2018; Yanovsky et al . , 2014 ) . Therefore , it seems likely that respiration-entrained gamma activity arises from local computations driven by sensory inputs sampled at each breath and thus plays a major role in cognition . A promising area to study this hypothesis is the piriform cortex ( PCx ) , which constitutes the primary olfactory area in the rodent brain ( Bolding and Franks , 2018a; Stettler and Axel , 2009 ) and exhibits prominent gamma oscillations ( Bressler and Freeman , 1980; Courtiol et al . , 2019; Freeman , 1960; Freeman and Skarda , 1985; Kay et al . , 2009; Kay and Freeman , 1998; Litaudon et al . , 2008; Mori et al . , 2013; Vanderwolf , 2000 ) . Most of our understanding of piriform oscillations comes from the studies of Walter Freeman in the 20th century ( Barrie et al . , 1996; Eeckman and Freeman , 1990; Freeman , 1968; Freeman , 1960; Freeman , 1959 ) . Freeman characterized gamma activity in terms of its topography , frequency range , and relationship to unitary activity and behavior . These observations led him to hypothesize that these oscillations constitute a fundamental sensory processing component that emerges from an excitatory-inhibitory feedback loop . However , despite his influential insights , Freeman’s conjectures could not be conclusively tested due to the technological limitations of his time . Thus , we still lack compelling experimental demonstrations of how gamma generation depends on the interactions between the different piriform neuronal subpopulations or how gamma relates to odor representations encoded as cell assemblies . Therefore , the mechanisms of gamma oscillations in the PCx and their functional role in olfaction need to be studied under the lens of modern experimental and analytical tools ( Courtiol et al . , 2019; Kay et al . , 2009; Mori et al . , 2013 ) . In this report , we study gamma oscillations in the PCx of the awake mouse . We took advantage of modern genetic tools , which accurately identify neuronal populations and precisely modify the local connectivity of the PCx , thus enabling an unprecedented study of the mechanisms and functions of its oscillatory activity . We found that respiration drives gamma oscillations in this region , which derive from feedback inhibition and depend on recurrent connections between local excitatory and feedback inhibitory populations . This loop is triggered by the projection of mitral/tufted cells in the olfactory bulb onto the principal cells of the PCx . As functional consequences , we show that respiration-driven gamma oscillations determine odor-assembly representations through a winner-take-all computation taking place within breathing cycles . \n",
      " To understand the mechanisms and functions of gamma oscillations and their relationship with respiration in the mouse brain , we analyzed local field potentials ( LFP ) from the PCx recorded simultaneously with the respiration signal ( Figure 1A ) during odorless cycles , hereafter referred to as ‘spontaneous’ activity . The dataset was collected by Bolding and Franks , 2018a and generously made available through CRCNS ( http://crcns . org , pcx-1 dataset ) . Figure 1B depicts the LFP and respiration signal from a representative animal; notice that low-gamma oscillations ( 30–60 Hz ) emerge following inhalation start . These low-gamma oscillations , already evident in raw recordings , are part of a larger gamma peak ( 30–100 Hz ) in the LFP power spectrum ( Figure 1C ) , reflecting a true oscillation ( Yuval-Greenberg et al . , 2008 ) . Consistent with Figure 1B , only the low-gamma sub-band couples to the respiration cycle across animals , as its amplitude is modulated by both a 2–3 Hz LFP rhythm coherent to respiration ( Figure 1D top panel; see also Figure 1—figure supplements 1–3 ) and by respiration itself ( Figure 1D bottom panel ) . Thus , respiration entrains low-gamma oscillations in the PCx . To further characterize the interaction between respiration and low-gamma oscillations in the PCx , we performed directionality analyses ( Figure 1E and Figure 1—figure supplement 4 ) . The gamma amplitude envelope showed a peak ~200 ms following the inhalation start , coinciding with a large positive cross-correlation peak between these signals , which suggests that respiration causes gamma . Consistent with these results , time-domain Granger causality was significantly higher in the respiration→gamma direction than in the opposite one ( t ( 12 ) = 9 . 82 , p<10–6 ) . Together , these results show that respiration drives low-gamma oscillations in the PCx . We analyzed the contribution of the different PCx neuronal populations to network low-gamma oscillations . First , we performed a current source density analysis ( Figure 1F ) , which revealed that these oscillations are generated locally within the piriform circuit and show a phase reversal near layer 2 ( where pyramidal cells are located ) . Next , we classified single units according to the expression of a light-sensitive channelrhodopsin coupled to the vesicular GABA transporter ( VGAT ) . This allowed us to discriminate between VGAT- principal cells and VGAT+ inhibitory interneurons . Additionally , VGAT+ neurons were further classified into feedback inhibitory interneurons ( FBI ) and feedforward interneurons ( FFI ) according to their location relative to the principal cell layer ( FFI are located at layer 1 while FBI tend to be located within the cell layer 2/3; Bolding and Franks , 2018a; Figure 1—figure supplement 5 ) . Upon averaging the activity of each neuronal subpopulation , we found that the time course of FBI firing rate changes correlates with the amplitude of low-gamma oscillations in time ( Figure 1G ) . In contrast , principal cells and FFI spike earlier within the respiratory cycle and return to baseline during the gamma amplitude peak ( Figure 1G ) . Notice further that the gamma peak coincides with principal cell inhibition , as evidenced by their firing rate decrease . Moreover , by computing spike-triggered gamma envelope averages , we confirmed that gamma oscillations closely follow FBI spiking ( Figure 1G inset ) . Therefore , we conclude that respiration-driven low-gamma oscillations in the PCx arise from feedback inhibition . We next studied the circuit mechanisms responsible for the respiration-driven gamma oscillations in the PCx . To that end , we analyzed spontaneous PCx LFPs following the selective expression of the tetanus toxin light chain in principal cells of a targeted hemisphere ( TeLC ipsi; Figure 2A ) . Under this approach , TeLC expression blocks excitatory synaptic transmission without affecting cellular excitability ( Bolding and Franks , 2018a ) , allowing us to study the local computations underlying the respiration-driven gamma . Figure 2B shows that TeLC ipsi LFPs had a large significant reduction in low-gamma oscillations with respect to either control mice ( t ( 19 ) = 4 . 11 , p<0 . 001 ) or the contralateral PCx ( TeLC contra , not infected; t ( 12 ) = 2 . 99 , p=0 . 0055 ) . Moreover , TeLC ipsi LFPs also showed a significant decrease in respiration-gamma coupling compared to control ( t ( 19 ) = 5 . 87 , p<10–5 ) or TeLC contra LFPs ( t ( 12 ) = 3 . 10 , p=0 . 0045 ) ( Figure 2C and D ) , despite the respiration signal still reaching the PCx ( note the dotted white traces in Figure 2C showing LFP-Resp coherence ) . These results demonstrate that gamma oscillations and their coupling to respiration depend on local recurrent excitatory connections within the PCx . Noteworthy , respiration-driven low-gamma oscillations also depend on the cognitive state since ketamine/xylazine anesthesia abolishes them ( Figure 2—figure supplement 1 , Appendix 1 ) . After confirming that respiration-driven gamma oscillations depend on recurrent connections formed by principal cells , we asked how PCx inputs affect gamma generation . We expected mitral/tufted cell activation in the olfactory bulb ( OB ) to trigger similar low-gamma oscillations since these projections convey the respiratory inputs to the PCx ( Pashkovski et al . , 2020 ) . Consistently , optogenetic activation of the OB ( Thy-Control ) triggered piriform low-gamma oscillations , which matched the laser time course ( Figure 3A , top panel ) . Interestingly , TeLC ipsi LFPs showed almost no gamma activity following laser onset ( Figure 3A , middle panel ) , while TeLC contra LFPs still exhibited gamma activity upon stimulation of the OB ( Figure 3A , bottom panel ) . Analyzing the group response , we found a significant reduction of low-gamma activity following the laser onset in TeLC ipsi LFPs compared to control ( t ( 17 ) = 5 . 12 , p<0 . 0001 ) or TeLC contra LFPs ( t ( 19 ) = 3 . 38 , p=0 . 0015 ) ( Figure 3B and C ) , further confirming the importance of recurrent connections for gamma generation . Moreover , low-gamma power in control and TeLC contra LFPs increased with laser intensity , while it remained constant in TeLC ipsi LFPs ( Figure 3D ) . Nevertheless , it should be noted that the contralateral TeLC hemisphere showed lower amplitude gamma oscillations following light stimulation than control recordings , though whether this gamma difference is related to an impaired network interplay between both hemispheres or to genetic differences between mouse lines remains to be determined . In any event , these results experimentally prove two critical facts about respiration-driven piriform gamma oscillations . First , that respiration drives PCx low-gamma oscillations mediated by OB projections from mitral/tufted cells . Second , feedforward interneurons do not generate low gamma , which necessarily requires the principal cells to excite local feedback interneurons . Having studied the mechanisms of spontaneous respiration-driven low-gamma oscillations in awake mice , we next analyzed their behavior during odor sampling ( Figure 4 ) . First , we noted that odors elicited large beta oscillations ( 10–30 Hz ) , which have been widely associated with olfactory processing ( Lepousez and Lledo , 2013; Martin et al . , 2006; Poo and Isaacson , 2009 ) . Interestingly , the amplitude of gamma oscillations was not affected by odor delivery , but the duration of gamma activity increased notoriously ( Figure 4B ) . Of note , the increase in gamma duration specifically depended on odor stimulation and not on longer breaths as previously suggested by Vanderwolf , 2000 , since longer odorless cycles did not prolong the induced gamma activity ( Figure 4—figure supplement 1 ) . Notably , beta oscillations occurred 100 ms before gamma onset , and , moreover , showed a consistent phase resetting during each sniff cycle ( Figure 4C and D top ) , resulting in a large amplitude envelope of the inhalation-trigged average of beta-filtered LFPs ( Figure 4D middle ) , which is to say that beta was evoked at each cycle . The unfamiliar reader is referred to Tallon-Baudry and Bertrand , 1999 for a discussion about evoked vs . induced oscillations . In short , an oscillatory activity that shows up in the average filtered trace is said to be evoked since this requires phase consistency ( or ‘resetting’ ) following each stimulus . On the other hand , oscillations that increase in amplitude following each stimulus ( in our case , an inhalation ) , but that exhibit phase jitters from trial to trial , cannot be properly detected in the averaged trace due to peak-trough cancellations across trials . Such oscillations , referred to as ‘induced , ’ can only be detected by inspecting the average across all individual trial amplitudes ( or spectrogram; see Tallon-Baudry and Bertrand , 1999 ) . This is the case of piriform low-gamma oscillations ( Figure 4D bottom ) since they increase following each inhalation but exhibit time jitters in peak activity from cycle to cycle and no phase resetting ( Figure 4D top ) . That is , our results show that gamma oscillations are not evoked but induced at each sniff cycle , contrasting therefore , with the evoked beta oscillations . We also investigated how beta and gamma responses depended on odor identity and concentration ( Figure 4E ) . Different odorants triggered similar beta and gamma responses ( Figure 4E top ) . The amplitude of beta oscillations depended on concentration ( F ( 3 , 12 ) =5 . 84 , p=0 . 01 ) , while gamma amplitude did not though its variability increased ( F ( 3 , 12 ) =2 . 89 , p=0 . 079; Figure 4E bottom ) . Importantly , when comparing TeLC-infected hemispheres with the contralateral ones during odor delivery , we found that only odor-induced gamma oscillations depended on the local piriform recurrent connections , while beta oscillations were still present in infected animals ( Figure 4F ) . These results suggest that beta oscillations do not relate to local piriform computations and are likely of OB postsynaptic origin . Next , we studied how gamma oscillations influenced piriform spiking patterns during odor processing . First , we compared the firing rate of each piriform neuronal subtype during odorless and odor cycles ( Figure 5A ) . Notably , feedback interneurons showed the most pronounced changes during odor cycles , substantially increasing their firing rate and prolonging their spiking period above baseline , thus mirroring the increase in gamma duration . Nonetheless , we observed that the increased FBI spiking ( ~100 ms ) preceded gamma amplitude increase ( ~200 ms; c . f . Figures 4B and 5A ) . This result may be related to the time required for different FBIs to synchronize their activity and generate gamma , as observed in computational models of gamma generation ( Wang and Buzsáki , 1996 ) . We then studied how gamma oscillations influence principal cell spiking during odor delivery , pooling together all recorded principal cells and analyzing their spiking as a function of the respiration phase . We found that the respiration phase modulated principal cell spiking within breathing cycles ( Figure 5B ) , though the preferred spiking phase differed across neurons . Interestingly , while a large proportion of cells were inhibited during the same respiration phase as the maximal gamma amplitude , some cells increased their spiking coincidently with the gamma peak . The respiratory phase preference was stable throughout the recording session ( Figure 5C , top panel; t ( 857 ) = –0 . 18 , p=0 . 57 ) , and respiratory modulation of principal cell spiking did not change between odor and odorless cycles ( Figure 5C , middle panel; t ( 857 ) = –0 . 57 , p=0 . 71 ) . Remarkably , spike-gamma phase coupling increased during the processing of odors ( Figure 5C , bottom panel; t ( 857 ) = 9 . 49 , p<10–19 ) , suggesting that these prolonged oscillations play a role in shaping odor coding . Consistently , we found that odor context determined which cells fired during the gamma oscillation ( Figure 5D ) . We further confirmed this observation by measuring the spiking specificity to odors , which closely followed the gamma envelope ( Figure 5E ) . Thus , these results demonstrate that gamma inhibition shapes single-cell responses during olfaction . The gamma spiking specificity supports the conjecture that respiration-driven gamma oscillations could mediate odor assembly representations . To further study this possibility , we analyzed cell assembly compositions for each odor by measuring the contribution of each principal cell to the first independent component ( IC ) of the population response ( El-Gaby et al . , 2021; Lopes-dos-Santos et al . , 2013; Trouche et al . , 2016 ) . We found highly skewed distributions for each odor , where only a small fraction of neurons showed a strong positive contribution to the 1st IC , hereafter referred to as winner cells , while the vast majority showed low weights , referred to as loser cells ( Figure 6A and B ) . Notably , the few winning neurons determining the 1st IC activity changed from odor to odor ( Figure 6A ) ; in other words , there was strong orthogonality in the 1st IC weight distribution across different odors . Consistent with this , the 1st IC weights were not significantly correlated between odors ( corrected by multiple comparisons ) ( Figure 6C ) . We note that a similar odor separation was achieved when computing ICA on all odors together and analyzing the top six independent components ( Figure 6—figure supplement 1 ) . Interestingly , the losing cells were significantly more phase-locked to the gamma phase ( Figure 6D top; t ( 14 ) = 5 . 88 , p<10–4 ) , consistent with them being actively inhibited during the oscillation . There were no differences between the preferred gamma phase for winners and losers ( Figure 6D bottom; t ( 14 ) = 0 . 05 , p=0 . 95 ) . Moreover , when analyzing the spiking time course of winner and loser cells , we found that both groups tended to spike immediately following inhalation start; nevertheless , once feedback inhibition was recruited , loser cells spiked below their average while the winner cells continued spiking above baseline ( Figure 6E ) . Notably , loser cell inhibition correlated with gamma activity , which suggests that sparse odor assembly representations emerge from a winner-take-all process mediated by inhibitory gamma oscillations . Because loser cell inhibition correlated with low gamma activity , we expected the winner-take-all process to be affected in TeLC recordings . Thus , we analyzed cell assemblies in both infected and contralateral hemispheres . We found that the TeLC-infected piriform cortex showed significantly less skewed distributions ( t ( 5 ) = –3 . 08 , p=0 . 013 ) with a decreased number of losing cells ( t ( 5 ) = –2 . 10 , p=0 . 044 ) , which were significantly less inhibited than their contralateral counterparts ( t ( 5 ) = –2 . 22 , p=0 . 038; Figure 6F and G ) . Furthermore , the reduced inhibition of losing cells was associated with a high correlation of assembly weights across odors for the affected hemisphere ( t ( 5 ) = 2 . 63 , p=0 . 023; Figure 6H ) , suggesting that gamma oscillations are necessary for the segregation of odor-selective assemblies . Because a winner-take-all process selects a single odor representation while actively suppressing others , it effectively implements an XOR logic gate . As a consequence , we would expect odor decoding accuracy to increase during gamma inhibition . Following this reasoning , we first analyzed the correlation of population spiking vectors in response to different odorants and found that correlations significantly dropped during the gamma peak ( Gamma vs . Pre: t ( 14 ) = –3 . 32 , p=0 . 002; Gamma vs . Post: t ( 14 ) = –3 . 73 , p=0 . 001; Figure 7A ) , confirming that odor representations diverge the most during gamma inhibition . We next tested odor decoding from the principal cell activity by training a supervised linear classifier using population spiking ( i . e . spike vectors ) at different respiratory cycle phases . Consistent with a winner-take-all mechanism , we found that variations in odor decoding accuracy matched the low-gamma activity time course ( Figure 7B and C ) ; there was a significant correlation between these variables ( r=0 . 45 , p<10–12 ) . Importantly , decoding accuracy significantly increased around the gamma peak compared to time windows before or after it ( Gamma vs . Pre: t ( 14 ) = 9 . 47 , p<10–7; Gamma vs . Post: t ( 14 ) = 7 . 56 , p<10–5; see also Figure 7—figure supplement 1 ) . This result was robust among different classification algorithms ( Figure 7—figure supplement 1 ) . In addition , odor decoding accuracy significantly fluctuated within the gamma cycle ( Figure 7D ) , confirming that gamma inhibition directly modulates odor processing . Finally , when analyzing the TeLC experiments , we found that decoding accuracy significantly decreased in the 150–250 ms ‘gamma’ time window in the infected hemisphere compared to the contralateral one ( t ( 5 ) = –2 . 41 , p=0 . 03 ) . In all , these findings demonstrate that respiration-driven gamma oscillations provide an optimal temporal window for olfaction . \n",
      " The present report demonstrates the mechanisms and functions of gamma oscillations in the PCx of awake mice . We show that respiratory inputs to this cortex elicit large feedback inhibitory low-gamma oscillations that aid odor coding . Our work provides critical information for understanding gamma oscillations in the olfactory system . First , the results support the excitatory-inhibitory models of gamma generation ( Buzsáki and Wang , 2012; Fisahn et al . , 1998; Freeman , 1964 ) , highlighting the role of excitatory neurons in initiating the gamma cycle by driving feedback interneuron spiking . Our findings thus experimentally demonstrate several of Walter Freeman’s hypotheses regarding gamma generation ( Bressler and Freeman , 1980; Eeckman and Freeman , 1990; Freeman , 1968 ) , and reveal similarities with the mechanisms generating gamma in the OB ( Fukunaga et al . , 2014; Fukunaga et al . , 2012; Kay et al . , 2009; Lepousez and Lledo , 2013; Mori et al . , 2013 ) . Second , we demonstrate that the PCx low-gamma oscillations are the extracellular correlates of a winner-take-all process , thus confirming previous theoretical conjectures ( de Almeida et al . , 2009 ) and linking this brain oscillation to an algorithmic operation ( an XOR logic gate ) . Third , the winner-take-all interpretation provides a direct relationship between gamma oscillations and cell assemblies . Under this scenario , the low-gamma feedback inhibition is critical for segregating cell assemblies and generating a sparse , orthogonal odor representation . Concomitantly , the assembly recruitment would depend on OB projections determining which winner cells ‘escape’ gamma inhibition , highlighting the relevance of the OB-PCx interplay for olfaction ( Chae et al . , 2022; Otazu et al . , 2015 ) . An important feature of the respiration-driven low-gamma oscillations is that the winners triggering them change according to odor context ( Figure 6A ) , a process likely dependent on odor-encoding mitral cell assemblies from the OB ( Chae et al . , 2022 ) . Thus , the involvement of different cells and spiking dynamics would make the recruitment of feedback inhibition a non-homogenous process across respiration cycles , leading to small time jitters in peak gamma activity . As an empirical end result , the observed low-gamma oscillations are not phase-locked from cycle to cycle; in other words , PCx low-gamma power is induced , but not evoked ( Tallon-Baudry and Bertrand , 1999 ) , at each respiratory cycle . Our results , nonetheless , do not exclude a role for OB gamma in triggering piriform oscillations , as the OB output may arise from similar gamma-dependent winner-take-all processes in that area . Nevertheless , our results make it less likely that piriform gamma is simply inherited from the OB , given that we observe ( 1 ) a local current source , ( 2 ) a tight correlation with FBI spiking , and ( 3 ) that gamma disappears when local recurrent connections are abolished , despite the OB remaining intact . In contrast , beta oscillations were evoked by the respiratory input to the PCx , leading to a prominent phase resetting ~100 ms after the inhalation start ( Figure 4 ) . Further consistent with these results , respiration-triggered beta oscillations did not disappear in the TeLC condition and were close in time to excitatory and FFI spiking ( c . f . Figures 1 , 4 and 5 ) . Interestingly , a previous report showed that odors evoke PCx beta oscillations under urethane anesthesia , which , as here , occurred before global inhibition ( Poo and Isaacson , 2009 ) . Based on this , a role for piriform beta in odor representations has been proposed; nevertheless , here we show that beta oscillations are likely inherited and do not require local computations in the PCx , i . e . , they are not locally generated . The study of the role of the PCx in olfaction has recently been boosted by selective genetic techniques , allowing researchers to examine how the individual cellular components relate to olfaction ( Franks et al . , 2011; Nagappan and Franks , 2021 ) . The PCx , also known as the olfactory cortex , receives inputs from the OB and constructs a robust odor representation ( Bolding et al . , 2020 ) . This representation can be inferred from the population activity and has two major features . On the one hand , it separates odor identity from its concentration ( Bolding and Franks , 2018a; Bolding and Franks , 2017; Roland et al . , 2017 ) . This process depends on large-scale piriform inhibition ( Bolding and Franks , 2018a ) and thus correlates with low-gamma oscillations . On the other hand , the representation groups odors into perceptual categories , not necessarily related to the physical structure of the odor molecules ( Pashkovski et al . , 2020 ) . Surprisingly , these odor assembly representations are not fixed but change with time ( Schoonover et al . , 2021 ) , which might be a particular property of unstructured cortices such as the piriform cortex ( Stettler and Axel , 2009 ) . In light of this evidence , spike-field synchronization might be an essential coding strategy , as neural oscillations might provide an internal signal in which assembly representations can be organized and structured . Thus , we hypothesize that low-gamma oscillations provide a critical time window for odor assembly representations to be formed and modified through winner-take-all computations . The role of neuronal oscillations in shaping spiking patterns has been widely discussed ( Buzsáki , 2010 ) . Among other functions , oscillations are thought to provide syntactical blocks in which spiking activity can be parsed into meaningful words . A reader ( receiver ) area can then decode such neuronal representations by following these syntactic rules . In favor of this hypothesis , our decoding analysis shows that , for a reader neuron , optimal odor decoding occurs during the gamma peak within breathing cycles . A likely explanation for this scenario – based on our results – is that by suppressing the activity of competing representations , gamma oscillations increase the signal-to-background ratio and allow for optimal decoding . Under this framework , maximal decoding occurs after and not during the peak firing rate , coinciding with peak gamma activity and synchronous inhibition . Thus , the inhibited neurons , despite being the most phase-locked to gamma , would not directly encode odor information , while the neurons that escape gamma inhibition would do . Interestingly , and further consistent with our results , a recent study in humans showed that the accurate perception and identification of odors depend on piriform gamma oscillations ( Yang et al . , 2022 ) . Moreover , the behavioral response to odors also seems to depend on piriform gamma , and its suppression induces depressive-like behaviors in mice ( Li et al . , 2022 ) . Therefore , respiratory-triggered gamma oscillations might constitute a fundamental building block for neuronal communication in the olfactory system . Several reports show that the PCx is the source of gamma activity for widespread brain regions , including the striatum ( Carmichael et al . , 2017 ) and the anterior limbic system ( Carmichael et al . , 2019 ) . Although the authors largely attribute such oscillations to piriform volume conduction , recent evidence shows that olfactory gamma synchronizes distant regions as well ( Li et al . , 2022 ) . Based on our results , we postulate that genuine respiration-entrained gamma oscillations in downstream regions are triggered by the output of the winning piriform assemblies , leading to new local gamma winner-take-all processes . This provides a new look into long-range gamma synchronization , emerging naturally if winners generating gamma in one area consistently trigger winners generating gamma in a downstream area . Such a mechanism closely agrees with the theory and experiments reported by Schneider et al . , 2021 , which suggest that oscillatory power and connectivity are the main drivers underlying inter-areal coherence . A central element to this directional view of interregional synchronization is gamma coupling to respiration , as the slow oscillation ensures a flow of information from the most primary olfactory areas ( OB , PCx ) to the higher limbic areas ( prefrontal cortex , amygdala ) . In accordance , a recent report showed that respiration-related brain oscillations drive sparse assemblies in the prefrontal cortex ( Folschweiller and Sauer , 2022 ) . Notably , these authors also found that inhibitory recruitment by assembly members was key for assembly segregation , suggesting a potential role for gamma in that area . Hence , the directional view of synchronization across regions places a major role in cross-frequency interactions driving communication , highlighting the relevance of interregional synchrony modulation by slower rhythms , for instance , theta ( González et al . , 2020 ) or respiration itself ( Cavelli et al . , 2020; González et al . , 2023 ) . In light of our findings , we propose the following model to understand the relationship between spiking activity , neural oscillations , and odor coding in the piriform cortex: ( 1 ) Following each sniff , olfactory information is processed in the OB and is transmitted to the piriform circuit through mitral/tufted cell spiking . These synchronous postsynaptic potentials cause the field beta oscillation , whose amplitude depends on the afferent volley . ( 2 ) The principal cells that better encode the olfactory stimulus excite feedback interneurons . ( 3 ) Feedback interneurons inhibit competing principal cells , causing the field gamma oscillation . ( 4 ) This winner-take-all process segregates cell assemblies and dictates a sparse piriform odor representation . Note that because the piriform cortex normalized odorant concentration ( Bolding and Franks , 2018a ) , the amplitude of gamma oscillations does not change . Hence , gamma oscillations provide an optimal temporal window to decode odor . \n",
      " We analyzed recordings of the PCx generously made available by Bolding and Franks through the Collaborative Research in Computational Neuroscience data-sharing website ( http://crcns . org , pcx-1 dataset ) . The SIMUL , TeLC-PCx , THY , TeLC-THY , and VGAT experiments were used . Detailed descriptions of the experimental procedures can be found in previous publications ( Bolding et al . , 2020; Bolding and Franks , 2018a; Bolding and Franks , 2017 ) . All protocols were approved by the Institutional Animal Care and Use Committee of Duke University . Below we describe the analytical methods employed by us , and , for convenience , also the relevant experimental procedures from the original publications . Adult mice were employed ( >P60 , 20–24 g ) , and housed in single cages on a normal light-dark cycle . For the Cre-dependent TeLC group , offspring of Emx1Cre/Cre breeding pairs were obtained from The Jackson Laboratory ( 005628 ) . For the optogenetic experiments , the mice employed were: adult Thy1ChR2/ChR2-YFP , line 18 ( Thy1-COP4/EYFP , Jackson Laboratory , 007612 ) and VGAT ChR2/+ , line 8 ( Slc32a1-COP4*H134R/EYFP , Jackson Laboratory , 014548 ) . For the combined optogenetics and TeLC expression experiments , adult offspring of Emx1Cre/Cre mice crossed with Thy1ChR2/ChR2-YFP mice were employed . For the TeLC experiments , AAV5-DIO-TeLC-GFP was expressed under CBA control ( 6/7 mice ) or synapsin ( 1/7 mice ) , whose effects were similar and pooled together . Three 500 nL injections in the PCx ( AP , ML , DV:+1 . 8 , 2 . 7 , 3 . 85;+0 . 5 , 3 . 5 , 3 . 8; –1 . 5 , 3 . 9 , 4 . 2; DV measured from brain surface ) were employed to achieve TeLC expression . All recordings took place ~14 days after the promoter injection . All viruses were obtained from the University of North Carolina-Chapel Hill ( UNC Vector Core ) . The electrophysiological signals were recorded using 32-site polytrode acute probes ( A1x32-Poly3-5mm- 25 s-177 , Neuronexus ) with an A32-OM32 adaptor ( Neuronexus ) through a Cereplex digital headstage ( Blackrock Microsystems ) . For the optogenetic identification of GABAergic cells , a fiber-attached polytrode probe was employed ( A1x32-Poly3-5mm-25s-177-OA32LP , Neuronexus ) . Data were acquired at 30 kHz , unfiltered , employing a Cerebus multichannel data acquisition system ( BlackRock Microsystems ) . Respiration and experimental events were acquired at 2 kHz by analog inputs of the Cerebus system . The respiration signal was measured employing a microbridge mass airflow sensor ( Honeywell AWM3300V ) , which was positioned opposite to the animal’s nose . Inhalation generated a negative airflow and thus negative changes in the voltage of the sensor output . A Patchstar Micromanipulator ( Scientifica ) was employed to position the recording probe in the anterior PCx ( 1 . 32 mm anterior and 3 . 8 mm lateral from bregma ) . Recordings were targeted 3 . 5–4 mm ventral from the brain surface at this position , and were further adjusted according to the LFP and spiking activity monitored online . The electrode sites spanned 275 µm along the dorsal-ventral axis . The probe was lowered until an intense spiking band was found , which covered 30–40% of electrode sites near the correct ventral coordinate , thus reflecting the piriform layer II . For the optogenetic experiments stimulating OB cells in Thy1ChR2/ChR2-YFP mice , the optic fiber was placed <500 µm above the OB dorsal surface . Spyking-Circus software was employed to isolate individual units ( https://github . com/spyking-circus ) . All clusters which had more than 1% of ISIs violating the refractory period ( <2 ms ) or appearing contaminated were manually removed . Units that showed both similar waveforms and coordinated refractory periods were merged into a single cluster . The unit position was characterized as the mean electrode position ( across electrodes ) weighted by the amplitude of the unit waveform on each electrode . Stimuli consisted of monomolecular odorants diluted in mineral oil . These were hexanal , ethyl butyrate , ethyl acetate , 2-hexanone , isoamyl acetate , and ethyl tiglate . Odors were presented for one second through an olfactometer controlled by MATLAB scripts and repeated every ten seconds . For all analyses , we used Python 3 with numpy ( https://numpy . org/ ) , scipy ( https://docs . scipy . org/ ) , matplotlib ( https://matplotlib . org/ ) , sklearn ( https://scikit-learn . org/stable/ ) , and statsmodel ( https://www . statsmodels . org/stable/index . html ) libraries . The codes to reproduce all seven figures are freely available at: https://github . com/joaqgonzar/Gamma_Oscillations_PCx; ( copy archived at swh:1:rev:3e01d6b0112e2739fbfb6d0fef2be95f2d48ebd5 ) ( Gonzalez , 2023 ) . Raw LFPs were decimated to a sampling rate of 2000 Hz to match the respiration signal sampling rate , employing the decimate scipy function . This function first low-pass filters the 30 kHz raw data and then downsamples it , avoiding aliasing . For each animal , we used the same channel ( either channel 17 or 16 depending on the headstage configuration ) for all analyses in the SIMUL and TeLC-PCx datasets , though similar results are obtained for the rest of the channels given their high LFP redundancy ( Figure 1—figure supplement 3 ) . Channel 28 was employed for the VGAT datasets . We analyzed both odor-related and spontaneous activity; for the former , we selected all inhalations occurring within one-second following odor delivery , and the latter was obtained as all awake periods without odor delivery . To analyze the LFP spectra , we computed Welch’s modified periodogram using welch scipy function . Specifically , we employed a 1 s moving window with half a window overlap , setting the numerical frequency resolution to 0 . 1 Hz ( by setting the nfft parameter to 10 times the sampling rate ) . All spectra were whitened by multiplying each power value by its associated frequency , thus eliminating the 1 /f trend . For Figure 1B , we computed the wavelet transform ( cwt scipy function ) of the LFP signal using a Morlet wavelet with 0 . 1 Hz resolution . The triggered spectrograms were obtained first by selecting all 500 ms LFP windows following inhalation start ( time stamps provided in the dataset , see Bolding and Franks , 2018a for further details ) and then computing the spectrogram using the spectrogram scipy function . We used 40 ms windows with 82% overlap and a frequency resolution of 0 . 1 Hz . After computing each individual spectrogram , we whitened ( i . e . multiplied by f ) and averaged them to yield the mean spectrogram shown in Figures 1G and 3A . We computed phase-amplitude comodulograms following the methods proposed by Tort et al . , 2010 . We first band-pass filtered the LFP/respiration signal using the eegfilt function ( Delorme and Makeig , 2004 ) adapted for Python 3 ( available at https://github . com/joaqgonzar/Gamma_Oscillations_PCx ( copy archived at swh:1:rev:3e01d6b0112e2739fbfb6d0fef2be95f2d48ebd5 ) , Gonzalez , 2023 ) . We filtered between 20–130 Hz in 10 Hz steps to obtain the higher frequency components , and between 1 and 8 Hz in 1 Hz steps to obtain the slower frequency components . The phase ( angle ) and amplitude time series of the filtered signals were estimated from their analytical representation based on the Hilbert transform . We then binned the phase time series into 18 bins and computed the mean amplitude of the fast signal for each bin . The amount of phase-amplitude coupling was estimated through the modulation index ( MI ) metric ( Tort et al . , 2010 ) , MI = ( Hmax-H ) /Hmax , where Hmax is the maximum possible Shannon entropy for a given distribution ( log ( number of phase bins ) ) and H is the actual entropy of the amplitude distribution . To study the directionality between the gamma envelope and respiration signal , we employed three different strategies: ( 1 ) we averaged the gamma envelope using the inhalation start as a trigger; ( 2 ) we computed the cross-correlogram , using the correlate scipy function , between the gamma envelope and the respiration signal; ( 3 ) we computed Granger Causality estimates , using the grangercausalitytest stats model function , with a 10-order VAR model and employing the log ( F-stat ) as the Granger Causality magnitude ( Geweke , 1982 ) . We first obtained gamma averages by filtering the LFP signal between 30–50 Hz; the amplitude peaks were then identified and used for averaging 100 ms epochs surrounding these timestamps . CSD analysis was obtained by -A+2B-C for adjacent probe sites . To study induced power , we ( 1 ) filtered the recordings ( beta: 10–20 Hz , low gamma: 30–60 Hz ) , ( 2 ) obtained their amplitude through the Hilbert transform , and ( 3 ) computed the inhalation-triggered average . To study evoked power , we ( 1 ) filtered the recordings , ( 2 ) obtained the inhalation-triggered average , and then ( 3 ) estimated the amplitude of the average signal through the Hilbert transform . For obtaining the phase-resetting index of the filtered signals , we used the same procedure as for computing the inter-trial coherence ( Makeig et al . , 2004 ) using 500 ms windows following inhalation start as a ‘trial . ’ All cells were classified de-novo following the protocol described by Bolding and Franks , 2018a . A Wilcoxon rank-sum test was employed to determine laser responsiveness; p<0 . 001 was used to detect VGAT+ neurons . The distinction between FFI and FBI was made according to their location on the probe relative to the principal cell layer ( FFI are located at layer 1 while FBI tend to be located within the cell layer 2/3; Bolding and Franks , 2018a; Figure 1—figure supplement 5 ) . The spike times were rounded to match the LFP resolution ( 0 . 5 ms or 2000 Hz sampling rate ) . After that , we convolved the spike times of each unit with a 10 ms standard deviation Gaussian kernel employing the convolve scipy function , in close similarity with the original publication , which gave rise to a smoothed spiking activity . For the normalized responses ( Figure 1G ) , the spiking response of a neuronal subpopulation ( i . e . excitatory , FFI , or FBI neurons ) was normalized by dividing the firing rate by its average across time bins . We defined the odor specificity index for single cell responses following McNaughton et al . , 1983 as SI = max ( Fx ) / ( Fa +Fb + Fc +Fd + Fe +Ff ) , where F represents firing rate , the subscripts a-f represent different odors , and max ( Fx ) is the firing rate for the odor that generates the largest response . The specificity index time course was obtained using 100 ms sliding windows ( 62 . 5% overlap ) following inhalation start . We employed the ICA sklearn function on concatenated time series of the smoothed principal cell spiking activity using all 1000 ms windows following inhalation start during odor presentation ( 0 . 3% v . /v . concentration ) . Each odor was treated separately . We analyzed only the first independent component , setting n_components = 1 . Winners and losers were identified according to the weight each neuron exerted on the 1st principal component . Namely , neurons with a weight above the 95th percentile were labeled as winners , while neurons with a more negative weight than the IC mean as losers . For representation purposes ( Figure 6B ) , large positive and negative weights were truncated at 0 . 15 and –0 . 05 , respectively . The same procedures were employed for the assembly analyses on TeLC recordings , except that all cells were employed due to the lack of opto-tagging . We note that similar results were obtained in control recordings if all available neurons were included , likely due to excitatory cells far outnumbering inhibitory interneurons . To assess the coupling between spikes and the gamma phase , the mean-vector length of the spiking gamma phases was calculated as MVL=‖1Nk ∑k=1Nke−1φk‖ , where φk is the gamma phase associated to the k-th spike , and Nk is the total number of spikes . The average gamma spiking angle ( stats . circmean ) was computed to obtain the preferred phase . For the spike-triggered amplitude envelope averages , we first obtained the gamma amplitude envelope using the Hilbert transform , and then we computed the averages using –150–300 ms windows surrounding each spike . To infer the similarity/dissimilarity in the population response to the different odors , we analyzed the population vector correlations , defined as the Pearson correlation ( pearsonr scipy function ) measured using paired data from the same neurons to two odors . Correlations were computed between spiking vectors for each odor; the correlation time course shown in Figure 7A bottom was obtained using 100 ms sliding windows ( 62 . 5% overlap ) following inhalation start . For statistics , we averaged the correlation across odors and compared them among three-time windows of interest: from 0 to 100 ms ( Pre ) , from 150 to 250 ms ( Gamma ) , and from 525 to 625 ms ( Post ) . We employed a supervised linear classifier to decode the odor identity ( 0 . 3% v . /v . concentration ) from the population spiking vectors . The classification algorithm was supplied with the average spiking of each principal cell in 100 ms sliding windows following the inhalation start . The classification algorithm was a linear support vector machine with a stochastic gradient descent optimization , implemented using the sgdclassifier sklearn function . Each mouse was trained and tested separately; for each time window , the training sample consisted of 2/3 of the data , and the test sample consisted of the remaining third . The division of training/test samples was randomized in order to avoid odor bias , repeated 100 times , and the results of the repetitions averaged to obtain the final decoding value . For the statistical comparisons , we used the same 100 ms time windows employed in the population vector correlations ( i . e . Pre , Gamma , and Post ) . To verify the robustness of the results , we also employed a linear discriminant analysis and a k-nearest neighbor classifier , which rendered similar results to the sgdclassifier algorithm ( Figure 7—figure supplement 1 ) . We show data as either mean ± SEM or regular boxplots showing the median , 1st , 3rd quartiles , and the distribution range without outliers . We employed paired and unpaired t-test to compare between groups . To compare among odor concentrations , we employed repeated measures ANOVA . We set p<0 . 05 to be considered significant ( in the figure panels , * , ** and *** denote p<0 . 05 , p<0 . 01 , and p<0 . 001 , respectively ) . For the 1st IC weight correlations , we employed Pearson r correlation and corrected the statistical threshold through the Bonferroni method for multiple comparisons . All the data employed is freely available at http://dx . doi . org/10 . 6080/K00C4SZB . Previously Published Datasets: Bolding and Franks , 2018b Collaborative Research in Computational Neuroscience ( http://crcns . org/ ) . ‘Simultaneous extracellular recordings from mice OB and PCx and respiration data in response to odor stimuli and optogenetic stimulation of OB ( https://crcns . org/data-sets/pcx/pcx-1 ) . ’\n",
      "\n",
      "\n",
      " The article describes a study on the relationship between respiration and gamma oscillations in the piriform cortex (PCx) of mice. The authors employed a combination of optogenetic and electrophysiological techniques to investigate the role of inhibitory neurons in gamma oscillations and their relationship with respiration. They found that gamma oscillations are driven by inhibitory neurons and that the phase of gamma oscillations is reset by inhalation. The authors also found that gamma oscillations are involved in the encoding of odor identity and concentration. The study provides new insights into the relationship between respiration and gamma oscillations in the PCx and their role in odor processing.\n",
      "\n",
      "\n",
      "714\n",
      "105\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "# prompt = f\"[INST] Simplify and summarize in around 300 words:  {s} [/INST]\"\n",
    "\n",
    "prompt = f\"[INST] Simplify technical terms to layman's terms and summarize in around 300 words:  {s} [/INST]\"\n",
    "\n",
    "\n",
    "sampling_params = SamplingParams(temperature=0.8, top_p=0.05, max_tokens=1024)\n",
    "\n",
    "output = llm.generate(prompt, sampling_params)\n",
    "\n",
    "# output[0].outputs[0].text\n",
    "\n",
    "laysum = output[0].outputs[0].text\n",
    "print('\\n')\n",
    "print(s)\n",
    "print('\\n')\n",
    "print(laysum)\n",
    "print('\\n')\n",
    "print(len(laysum))\n",
    "print(len(laysum.split()))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "id": "CT3D9ztMwXaL"
   },
   "outputs": [],
   "source": [
    "# output = llm.generate('what is the best LLM model out there? Answer in 1 sentence?')\n",
    "# output"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "id": "Pbkl5Vls-thT"
   },
   "outputs": [],
   "source": [
    "# s = 'The evolutionary origins of the hypoxia-sensitive cells that trigger amniote respiratory reflexes – carotid body glomus cells , and ‘pulmonary neuroendocrine cells’ ( PNECs ) - are obscure . Homology has been proposed between glomus cells , which are neural crest-derived , and the hypoxia-sensitive ‘neuroepithelial cells’ ( NECs ) of fish gills , whose embryonic origin is unknown . NECs have also been likened to PNECs , which differentiate in situ within lung airway epithelia . Using genetic lineage-tracing and neural crest-deficient mutants in zebrafish , and physical fate-mapping in frog and lamprey , we find that NECs are not neural crest-derived , but endoderm-derived , like PNECs , whose endodermal origin we confirm . We discover neural crest-derived catecholaminergic cells associated with zebrafish pharyngeal arch blood vessels , and propose a new model for amniote hypoxia-sensitive cell evolution: endoderm-derived NECs were retained as PNECs , while the carotid body evolved via the aggregation of neural crest-derived catecholaminergic ( chromaffin ) cells already associated with blood vessels in anamniote pharyngeal arches . '\n",
    "# s"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "id": "rXHvUL9G-0Zu"
   },
   "outputs": [],
   "source": [
    "# output = llm.generate(f\"Simplify and summarize in less than 300 words: {s}\")\n",
    "# output"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "id": "4nEmjV2SDZtp"
   },
   "outputs": [],
   "source": [
    "# from vllm import SamplingParams\n",
    "\n",
    "# prompt = f\"[INST] Simplify and summarize in 200 to 300 words:  {s} [/INST]\"\n",
    "\n",
    "# sampling_params = SamplingParams(temperature=0.8, top_p=0.05, max_tokens=1024)\n",
    "\n",
    "# output = llm.generate(prompt, sampling_params)\n",
    "# output[0].outputs[0].text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "id": "GV1NZQ0xKbWL"
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "accelerator": "GPU",
  "colab": {
   "gpuType": "A100",
   "machine_shape": "hm",
   "provenance": []
  },
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.6"
  },
  "widgets": {
   "application/vnd.jupyter.widget-state+json": {
    "01a5e107d3ab486285ac7291373ff263": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HTMLModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HTMLModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HTMLView",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_59885a813497441982d7bfdf4121d569",
      "placeholder": "​",
      "style": "IPY_MODEL_6f00d887e2e745b3ad8ca604ae375c93",
      "value": "tokenizer.json: 100%"
     }
    },
    "0d9a4fb9bb9b442d85b72a171b63923e": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "DescriptionStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "DescriptionStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "description_width": ""
     }
    },
    "1038eaed83ea4602907f27edb8195579": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "17017b487ab547edbb8725a5063ea513": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HBoxModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HBoxModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HBoxView",
      "box_style": "",
      "children": [
       "IPY_MODEL_f2f902693cd84cf09fc18de13e00c47f",
       "IPY_MODEL_197b7d783d2f4e71b29b727d1ed62b29",
       "IPY_MODEL_e64bbd19f5ea4a449da4ef1a41609aac"
      ],
      "layout": "IPY_MODEL_cb0e7ab19b3f43f3898ec41d3d556ebd"
     }
    },
    "197b7d783d2f4e71b29b727d1ed62b29": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "FloatProgressModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "FloatProgressModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "ProgressView",
      "bar_style": "success",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_1038eaed83ea4602907f27edb8195579",
      "max": 414,
      "min": 0,
      "orientation": "horizontal",
      "style": "IPY_MODEL_d86bf62213914f25b670d5947ee155bf",
      "value": 414
     }
    },
    "19c778861e0d48a79a30beab55e55c0c": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "ProgressStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "ProgressStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "bar_color": null,
      "description_width": ""
     }
    },
    "1abee22e1cdc4a3caecf23a6296edcf3": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "DescriptionStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "DescriptionStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "description_width": ""
     }
    },
    "1aef87e388514463882585f15132b860": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HTMLModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HTMLModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HTMLView",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_704fa07cdd71433ab1841ab4173da566",
      "placeholder": "​",
      "style": "IPY_MODEL_8ce234f295d04391a3d733fb4942076c",
      "value": " 1.47k/1.47k [00:00&lt;00:00, 104kB/s]"
     }
    },
    "2584983e901747ff8cf494c088f707be": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "DescriptionStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "DescriptionStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "description_width": ""
     }
    },
    "2687cea98b134dd8b25a72806cd39cdc": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "309e3dd29f7f4b45866463ce15a1ad3c": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HTMLModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HTMLModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HTMLView",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_5ad2b60833214d73a69119a68f409097",
      "placeholder": "​",
      "style": "IPY_MODEL_f3f8d26bf1c245a49622c88f953f7617",
      "value": "config.json: 100%"
     }
    },
    "30bab861b7da44729281c99b2e7d8267": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "DescriptionStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "DescriptionStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "description_width": ""
     }
    },
    "37ff89115c024be99ca2fc0ca9c70fdd": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HBoxModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HBoxModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HBoxView",
      "box_style": "",
      "children": [
       "IPY_MODEL_01a5e107d3ab486285ac7291373ff263",
       "IPY_MODEL_387669df637d43b99931d527f0c49b05",
       "IPY_MODEL_bd535d3ace9e483593cfa14bf0d33b86"
      ],
      "layout": "IPY_MODEL_3cb4338afd3747858caffa421b13ab49"
     }
    },
    "387669df637d43b99931d527f0c49b05": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "FloatProgressModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "FloatProgressModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "ProgressView",
      "bar_style": "success",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_97065d64d29a43aaa3643030365de255",
      "max": 1795303,
      "min": 0,
      "orientation": "horizontal",
      "style": "IPY_MODEL_f2fa150d03de4155a8ba87b4e1d316ca",
      "value": 1795303
     }
    },
    "3cb4338afd3747858caffa421b13ab49": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "3d2bde8aaf4e4fe1ace0cab205b6f75e": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "49f9399cacc840c78099a93210ac9bf4": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "4b39b985c3d44aec8dc71f325eabf747": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "4ce382551bd943d9abe342d704a382fb": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "50e886e23b22415fbeb49922005d9ee6": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "5314d8a147af46e09b6869ded9051216": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HTMLModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HTMLModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HTMLView",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_61fc01da51104089b6bdd40a842a0967",
      "placeholder": "​",
      "style": "IPY_MODEL_6e5de86d7f7a43f78ec2505554a2b6c9",
      "value": "tokenizer_config.json: 100%"
     }
    },
    "53838eef853144f1adec46e15e4b9c0d": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "FloatProgressModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "FloatProgressModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "ProgressView",
      "bar_style": "success",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_4b39b985c3d44aec8dc71f325eabf747",
      "max": 493443,
      "min": 0,
      "orientation": "horizontal",
      "style": "IPY_MODEL_e0b42c898ef64d80865220e25ef7e0f1",
      "value": 493443
     }
    },
    "59885a813497441982d7bfdf4121d569": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "5ad2b60833214d73a69119a68f409097": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "5f0f813cce344332886986c894c884d9": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "61fc01da51104089b6bdd40a842a0967": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "694a781667bb4f6e8fbe032013a4d2c5": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "6a854e571d004965bcfc439b1c12b318": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "6e5de86d7f7a43f78ec2505554a2b6c9": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "DescriptionStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "DescriptionStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "description_width": ""
     }
    },
    "6f00d887e2e745b3ad8ca604ae375c93": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "DescriptionStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "DescriptionStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "description_width": ""
     }
    },
    "704fa07cdd71433ab1841ab4173da566": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "73366f1b94a940ac8c5c7854dbc88936": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "ProgressStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "ProgressStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "bar_color": null,
      "description_width": ""
     }
    },
    "785ddc81d4894b7f8251537d13ae8ad1": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "85a557d925a6438482d609203bb39c6d": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HBoxModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HBoxModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HBoxView",
      "box_style": "",
      "children": [
       "IPY_MODEL_5314d8a147af46e09b6869ded9051216",
       "IPY_MODEL_dcdf8a591dbe40f1b1478ab2c2159ee7",
       "IPY_MODEL_1aef87e388514463882585f15132b860"
      ],
      "layout": "IPY_MODEL_694a781667bb4f6e8fbe032013a4d2c5"
     }
    },
    "8ce234f295d04391a3d733fb4942076c": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "DescriptionStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "DescriptionStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "description_width": ""
     }
    },
    "8e64858c8fe443e8827da5e50491409a": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "9371f34b638e43ffa0794cde1b317b5f": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "DescriptionStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "DescriptionStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "description_width": ""
     }
    },
    "97065d64d29a43aaa3643030365de255": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "98019547f22741bd9d2a7ef2778182ac": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HTMLModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HTMLModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HTMLView",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_5f0f813cce344332886986c894c884d9",
      "placeholder": "​",
      "style": "IPY_MODEL_ff3056d47fbb44b69409e541c662e90f",
      "value": " 887/887 [00:00&lt;00:00, 79.1kB/s]"
     }
    },
    "a00cacfa691642eeb6db41f0e18a6a49": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HTMLModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HTMLModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HTMLView",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_ed2e74f10eb04675a70ab0b84c93b5e8",
      "placeholder": "​",
      "style": "IPY_MODEL_2584983e901747ff8cf494c088f707be",
      "value": " 493k/493k [00:00&lt;00:00, 25.2MB/s]"
     }
    },
    "a37d9e65fe5c4471a962ad3b87ee67bd": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "FloatProgressModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "FloatProgressModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "ProgressView",
      "bar_style": "success",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_4ce382551bd943d9abe342d704a382fb",
      "max": 887,
      "min": 0,
      "orientation": "horizontal",
      "style": "IPY_MODEL_73366f1b94a940ac8c5c7854dbc88936",
      "value": 887
     }
    },
    "a8cc64bfa5974e039d0074dcce35936f": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HTMLModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HTMLModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HTMLView",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_f9d4d832cd204b538955e6ddce50f3dc",
      "placeholder": "​",
      "style": "IPY_MODEL_30bab861b7da44729281c99b2e7d8267",
      "value": "model.safetensors: 100%"
     }
    },
    "aa47ecbc800642b48a60a097eca3624d": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "ProgressStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "ProgressStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "bar_color": null,
      "description_width": ""
     }
    },
    "ac795b97844c4f35b3abafafef245c8a": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "DescriptionStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "DescriptionStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "description_width": ""
     }
    },
    "b170e6cd5beb423fb7018f653992fb48": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HTMLModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HTMLModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HTMLView",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_c0d9dfde7cf842e4a8cc4123567f4899",
      "placeholder": "​",
      "style": "IPY_MODEL_9371f34b638e43ffa0794cde1b317b5f",
      "value": "tokenizer.model: 100%"
     }
    },
    "b7947e3af0534201a87761238e847927": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HBoxModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HBoxModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HBoxView",
      "box_style": "",
      "children": [
       "IPY_MODEL_309e3dd29f7f4b45866463ce15a1ad3c",
       "IPY_MODEL_a37d9e65fe5c4471a962ad3b87ee67bd",
       "IPY_MODEL_98019547f22741bd9d2a7ef2778182ac"
      ],
      "layout": "IPY_MODEL_49f9399cacc840c78099a93210ac9bf4"
     }
    },
    "bd535d3ace9e483593cfa14bf0d33b86": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HTMLModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HTMLModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HTMLView",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_8e64858c8fe443e8827da5e50491409a",
      "placeholder": "​",
      "style": "IPY_MODEL_ac795b97844c4f35b3abafafef245c8a",
      "value": " 1.80M/1.80M [00:00&lt;00:00, 7.55MB/s]"
     }
    },
    "c0d9dfde7cf842e4a8cc4123567f4899": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "c18b950df3594081abfd5dde36c10fda": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HBoxModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HBoxModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HBoxView",
      "box_style": "",
      "children": [
       "IPY_MODEL_b170e6cd5beb423fb7018f653992fb48",
       "IPY_MODEL_53838eef853144f1adec46e15e4b9c0d",
       "IPY_MODEL_a00cacfa691642eeb6db41f0e18a6a49"
      ],
      "layout": "IPY_MODEL_fe121ca81f6146a39fa847f16bc60368"
     }
    },
    "cb0e7ab19b3f43f3898ec41d3d556ebd": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "d2cd1fcf5dc24d57888778c24868dc1b": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HBoxModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HBoxModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HBoxView",
      "box_style": "",
      "children": [
       "IPY_MODEL_a8cc64bfa5974e039d0074dcce35936f",
       "IPY_MODEL_e4b0ca5ce07c437c9393a935aa207b7b",
       "IPY_MODEL_f7cd9daf7bb845cca8d69b3e0bd47dcb"
      ],
      "layout": "IPY_MODEL_3d2bde8aaf4e4fe1ace0cab205b6f75e"
     }
    },
    "d86bf62213914f25b670d5947ee155bf": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "ProgressStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "ProgressStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "bar_color": null,
      "description_width": ""
     }
    },
    "dcdf8a591dbe40f1b1478ab2c2159ee7": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "FloatProgressModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "FloatProgressModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "ProgressView",
      "bar_style": "success",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_2687cea98b134dd8b25a72806cd39cdc",
      "max": 1467,
      "min": 0,
      "orientation": "horizontal",
      "style": "IPY_MODEL_19c778861e0d48a79a30beab55e55c0c",
      "value": 1467
     }
    },
    "e0b42c898ef64d80865220e25ef7e0f1": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "ProgressStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "ProgressStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "bar_color": null,
      "description_width": ""
     }
    },
    "e4b0ca5ce07c437c9393a935aa207b7b": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "FloatProgressModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "FloatProgressModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "ProgressView",
      "bar_style": "success",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_50e886e23b22415fbeb49922005d9ee6",
      "max": 4150880232,
      "min": 0,
      "orientation": "horizontal",
      "style": "IPY_MODEL_aa47ecbc800642b48a60a097eca3624d",
      "value": 4150880232
     }
    },
    "e557dfec8afc4859be1077a50183da6d": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "e64bbd19f5ea4a449da4ef1a41609aac": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HTMLModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HTMLModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HTMLView",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_e557dfec8afc4859be1077a50183da6d",
      "placeholder": "​",
      "style": "IPY_MODEL_1abee22e1cdc4a3caecf23a6296edcf3",
      "value": " 414/414 [00:00&lt;00:00, 30.2kB/s]"
     }
    },
    "ed2e74f10eb04675a70ab0b84c93b5e8": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "f2f902693cd84cf09fc18de13e00c47f": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HTMLModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HTMLModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HTMLView",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_785ddc81d4894b7f8251537d13ae8ad1",
      "placeholder": "​",
      "style": "IPY_MODEL_fdbccef9fd934bf485a6eba4f43fed9a",
      "value": "special_tokens_map.json: 100%"
     }
    },
    "f2fa150d03de4155a8ba87b4e1d316ca": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "ProgressStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "ProgressStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "bar_color": null,
      "description_width": ""
     }
    },
    "f3f8d26bf1c245a49622c88f953f7617": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "DescriptionStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "DescriptionStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "description_width": ""
     }
    },
    "f7cd9daf7bb845cca8d69b3e0bd47dcb": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "HTMLModel",
     "state": {
      "_dom_classes": [],
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "HTMLModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/controls",
      "_view_module_version": "1.5.0",
      "_view_name": "HTMLView",
      "description": "",
      "description_tooltip": null,
      "layout": "IPY_MODEL_6a854e571d004965bcfc439b1c12b318",
      "placeholder": "​",
      "style": "IPY_MODEL_0d9a4fb9bb9b442d85b72a171b63923e",
      "value": " 4.15G/4.15G [00:16&lt;00:00, 388MB/s]"
     }
    },
    "f9d4d832cd204b538955e6ddce50f3dc": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "fdbccef9fd934bf485a6eba4f43fed9a": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "DescriptionStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "DescriptionStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "description_width": ""
     }
    },
    "fe121ca81f6146a39fa847f16bc60368": {
     "model_module": "@jupyter-widgets/base",
     "model_module_version": "1.2.0",
     "model_name": "LayoutModel",
     "state": {
      "_model_module": "@jupyter-widgets/base",
      "_model_module_version": "1.2.0",
      "_model_name": "LayoutModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "LayoutView",
      "align_content": null,
      "align_items": null,
      "align_self": null,
      "border": null,
      "bottom": null,
      "display": null,
      "flex": null,
      "flex_flow": null,
      "grid_area": null,
      "grid_auto_columns": null,
      "grid_auto_flow": null,
      "grid_auto_rows": null,
      "grid_column": null,
      "grid_gap": null,
      "grid_row": null,
      "grid_template_areas": null,
      "grid_template_columns": null,
      "grid_template_rows": null,
      "height": null,
      "justify_content": null,
      "justify_items": null,
      "left": null,
      "margin": null,
      "max_height": null,
      "max_width": null,
      "min_height": null,
      "min_width": null,
      "object_fit": null,
      "object_position": null,
      "order": null,
      "overflow": null,
      "overflow_x": null,
      "overflow_y": null,
      "padding": null,
      "right": null,
      "top": null,
      "visibility": null,
      "width": null
     }
    },
    "ff3056d47fbb44b69409e541c662e90f": {
     "model_module": "@jupyter-widgets/controls",
     "model_module_version": "1.5.0",
     "model_name": "DescriptionStyleModel",
     "state": {
      "_model_module": "@jupyter-widgets/controls",
      "_model_module_version": "1.5.0",
      "_model_name": "DescriptionStyleModel",
      "_view_count": null,
      "_view_module": "@jupyter-widgets/base",
      "_view_module_version": "1.2.0",
      "_view_name": "StyleView",
      "description_width": ""
     }
    }
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
