{
  "cells": [
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "UPTz0bb4vaDW",
        "outputId": "6c2a8807-2581-4367-d00c-e6c7d05eefaf"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting vllm==0.3.3\n",
            "  Downloading vllm-0.3.3-cp310-cp310-manylinux1_x86_64.whl (44.3 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m44.3/44.3 MB\u001b[0m \u001b[31m9.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting ninja (from vllm==0.3.3)\n",
            "  Downloading ninja-1.11.1.1-py2.py3-none-manylinux1_x86_64.manylinux_2_5_x86_64.whl (307 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m307.2/307.2 kB\u001b[0m \u001b[31m19.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: psutil in /usr/local/lib/python3.10/dist-packages (from vllm==0.3.3) (5.9.5)\n",
            "Collecting ray>=2.9 (from vllm==0.3.3)\n",
            "  Downloading ray-2.10.0-cp310-cp310-manylinux2014_x86_64.whl (65.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m65.1/65.1 MB\u001b[0m \u001b[31m7.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: sentencepiece in /usr/local/lib/python3.10/dist-packages (from vllm==0.3.3) (0.1.99)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.10/dist-packages (from vllm==0.3.3) (1.25.2)\n",
            "Collecting torch==2.1.2 (from vllm==0.3.3)\n",
            "  Downloading torch-2.1.2-cp310-cp310-manylinux1_x86_64.whl (670.2 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m670.2/670.2 MB\u001b[0m \u001b[31m2.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: transformers>=4.38.0 in /usr/local/lib/python3.10/dist-packages (from vllm==0.3.3) (4.38.2)\n",
            "Collecting xformers==0.0.23.post1 (from vllm==0.3.3)\n",
            "  Downloading xformers-0.0.23.post1-cp310-cp310-manylinux2014_x86_64.whl (213.0 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m213.0/213.0 MB\u001b[0m \u001b[31m5.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting fastapi (from vllm==0.3.3)\n",
            "  Downloading fastapi-0.110.1-py3-none-any.whl (91 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m91.9/91.9 kB\u001b[0m \u001b[31m11.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting uvicorn[standard] (from vllm==0.3.3)\n",
            "  Downloading uvicorn-0.29.0-py3-none-any.whl (60 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m60.8/60.8 kB\u001b[0m \u001b[31m8.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: pydantic>=2.0 in /usr/local/lib/python3.10/dist-packages (from vllm==0.3.3) (2.6.4)\n",
            "Requirement already satisfied: prometheus-client>=0.18.0 in /usr/local/lib/python3.10/dist-packages (from vllm==0.3.3) (0.20.0)\n",
            "Collecting pynvml==11.5.0 (from vllm==0.3.3)\n",
            "  Downloading pynvml-11.5.0-py3-none-any.whl (53 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m53.1/53.1 kB\u001b[0m \u001b[31m7.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: triton>=2.1.0 in /usr/local/lib/python3.10/dist-packages (from vllm==0.3.3) (2.2.0)\n",
            "Collecting outlines>=0.0.27 (from vllm==0.3.3)\n",
            "  Downloading outlines-0.0.37-py3-none-any.whl (83 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m83.6/83.6 kB\u001b[0m \u001b[31m11.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting cupy-cuda12x==12.1.0 (from vllm==0.3.3)\n",
            "  Downloading cupy_cuda12x-12.1.0-cp310-cp310-manylinux2014_x86_64.whl (83.0 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m83.0/83.0 MB\u001b[0m \u001b[31m6.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: fastrlock>=0.5 in /usr/local/lib/python3.10/dist-packages (from cupy-cuda12x==12.1.0->vllm==0.3.3) (0.8.2)\n",
            "Requirement already satisfied: filelock in /usr/local/lib/python3.10/dist-packages (from torch==2.1.2->vllm==0.3.3) (3.13.4)\n",
            "Requirement already satisfied: typing-extensions in /usr/local/lib/python3.10/dist-packages (from torch==2.1.2->vllm==0.3.3) (4.11.0)\n",
            "Requirement already satisfied: sympy in /usr/local/lib/python3.10/dist-packages (from torch==2.1.2->vllm==0.3.3) (1.12)\n",
            "Requirement already satisfied: networkx in /usr/local/lib/python3.10/dist-packages (from torch==2.1.2->vllm==0.3.3) (3.3)\n",
            "Requirement already satisfied: jinja2 in /usr/local/lib/python3.10/dist-packages (from torch==2.1.2->vllm==0.3.3) (3.1.3)\n",
            "Requirement already satisfied: fsspec in /usr/local/lib/python3.10/dist-packages (from torch==2.1.2->vllm==0.3.3) (2023.6.0)\n",
            "Collecting nvidia-cuda-nvrtc-cu12==12.1.105 (from torch==2.1.2->vllm==0.3.3)\n",
            "  Using cached nvidia_cuda_nvrtc_cu12-12.1.105-py3-none-manylinux1_x86_64.whl (23.7 MB)\n",
            "Collecting nvidia-cuda-runtime-cu12==12.1.105 (from torch==2.1.2->vllm==0.3.3)\n",
            "  Using cached nvidia_cuda_runtime_cu12-12.1.105-py3-none-manylinux1_x86_64.whl (823 kB)\n",
            "Collecting nvidia-cuda-cupti-cu12==12.1.105 (from torch==2.1.2->vllm==0.3.3)\n",
            "  Using cached nvidia_cuda_cupti_cu12-12.1.105-py3-none-manylinux1_x86_64.whl (14.1 MB)\n",
            "Collecting nvidia-cudnn-cu12==8.9.2.26 (from torch==2.1.2->vllm==0.3.3)\n",
            "  Using cached nvidia_cudnn_cu12-8.9.2.26-py3-none-manylinux1_x86_64.whl (731.7 MB)\n",
            "Collecting nvidia-cublas-cu12==12.1.3.1 (from torch==2.1.2->vllm==0.3.3)\n",
            "  Using cached nvidia_cublas_cu12-12.1.3.1-py3-none-manylinux1_x86_64.whl (410.6 MB)\n",
            "Collecting nvidia-cufft-cu12==11.0.2.54 (from torch==2.1.2->vllm==0.3.3)\n",
            "  Using cached nvidia_cufft_cu12-11.0.2.54-py3-none-manylinux1_x86_64.whl (121.6 MB)\n",
            "Collecting nvidia-curand-cu12==10.3.2.106 (from torch==2.1.2->vllm==0.3.3)\n",
            "  Using cached nvidia_curand_cu12-10.3.2.106-py3-none-manylinux1_x86_64.whl (56.5 MB)\n",
            "Collecting nvidia-cusolver-cu12==11.4.5.107 (from torch==2.1.2->vllm==0.3.3)\n",
            "  Using cached nvidia_cusolver_cu12-11.4.5.107-py3-none-manylinux1_x86_64.whl (124.2 MB)\n",
            "Collecting nvidia-cusparse-cu12==12.1.0.106 (from torch==2.1.2->vllm==0.3.3)\n",
            "  Using cached nvidia_cusparse_cu12-12.1.0.106-py3-none-manylinux1_x86_64.whl (196.0 MB)\n",
            "Collecting nvidia-nccl-cu12==2.18.1 (from torch==2.1.2->vllm==0.3.3)\n",
            "  Downloading nvidia_nccl_cu12-2.18.1-py3-none-manylinux1_x86_64.whl (209.8 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m209.8/209.8 MB\u001b[0m \u001b[31m2.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting nvidia-nvtx-cu12==12.1.105 (from torch==2.1.2->vllm==0.3.3)\n",
            "  Using cached nvidia_nvtx_cu12-12.1.105-py3-none-manylinux1_x86_64.whl (99 kB)\n",
            "Collecting triton>=2.1.0 (from vllm==0.3.3)\n",
            "  Downloading triton-2.1.0-0-cp310-cp310-manylinux2014_x86_64.manylinux_2_17_x86_64.whl (89.2 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m89.2/89.2 MB\u001b[0m \u001b[31m8.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting nvidia-nvjitlink-cu12 (from nvidia-cusolver-cu12==11.4.5.107->torch==2.1.2->vllm==0.3.3)\n",
            "  Using cached nvidia_nvjitlink_cu12-12.4.127-py3-none-manylinux2014_x86_64.whl (21.1 MB)\n",
            "Collecting interegular (from outlines>=0.0.27->vllm==0.3.3)\n",
            "  Downloading interegular-0.3.3-py37-none-any.whl (23 kB)\n",
            "Collecting lark (from outlines>=0.0.27->vllm==0.3.3)\n",
            "  Downloading lark-1.1.9-py3-none-any.whl (111 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m111.7/111.7 kB\u001b[0m \u001b[31m17.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: nest-asyncio in /usr/local/lib/python3.10/dist-packages (from outlines>=0.0.27->vllm==0.3.3) (1.6.0)\n",
            "Requirement already satisfied: cloudpickle in /usr/local/lib/python3.10/dist-packages (from outlines>=0.0.27->vllm==0.3.3) (2.2.1)\n",
            "Collecting diskcache (from outlines>=0.0.27->vllm==0.3.3)\n",
            "  Downloading diskcache-5.6.3-py3-none-any.whl (45 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m45.5/45.5 kB\u001b[0m \u001b[31m6.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: scipy in /usr/local/lib/python3.10/dist-packages (from outlines>=0.0.27->vllm==0.3.3) (1.11.4)\n",
            "Requirement already satisfied: numba in /usr/local/lib/python3.10/dist-packages (from outlines>=0.0.27->vllm==0.3.3) (0.58.1)\n",
            "Requirement already satisfied: joblib in /usr/local/lib/python3.10/dist-packages (from outlines>=0.0.27->vllm==0.3.3) (1.4.0)\n",
            "Requirement already satisfied: referencing in /usr/local/lib/python3.10/dist-packages (from outlines>=0.0.27->vllm==0.3.3) (0.34.0)\n",
            "Requirement already satisfied: jsonschema in /usr/local/lib/python3.10/dist-packages (from outlines>=0.0.27->vllm==0.3.3) (4.19.2)\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.10/dist-packages (from outlines>=0.0.27->vllm==0.3.3) (2.31.0)\n",
            "Requirement already satisfied: annotated-types>=0.4.0 in /usr/local/lib/python3.10/dist-packages (from pydantic>=2.0->vllm==0.3.3) (0.6.0)\n",
            "Requirement already satisfied: pydantic-core==2.16.3 in /usr/local/lib/python3.10/dist-packages (from pydantic>=2.0->vllm==0.3.3) (2.16.3)\n",
            "Requirement already satisfied: click>=7.0 in /usr/local/lib/python3.10/dist-packages (from ray>=2.9->vllm==0.3.3) (8.1.7)\n",
            "Requirement already satisfied: msgpack<2.0.0,>=1.0.0 in /usr/local/lib/python3.10/dist-packages (from ray>=2.9->vllm==0.3.3) (1.0.8)\n",
            "Requirement already satisfied: packaging in /usr/local/lib/python3.10/dist-packages (from ray>=2.9->vllm==0.3.3) (24.0)\n",
            "Requirement already satisfied: protobuf!=3.19.5,>=3.15.3 in /usr/local/lib/python3.10/dist-packages (from ray>=2.9->vllm==0.3.3) (3.20.3)\n",
            "Requirement already satisfied: pyyaml in /usr/local/lib/python3.10/dist-packages (from ray>=2.9->vllm==0.3.3) (6.0.1)\n",
            "Requirement already satisfied: aiosignal in /usr/local/lib/python3.10/dist-packages (from ray>=2.9->vllm==0.3.3) (1.3.1)\n",
            "Requirement already satisfied: frozenlist in /usr/local/lib/python3.10/dist-packages (from ray>=2.9->vllm==0.3.3) (1.4.1)\n",
            "Requirement already satisfied: huggingface-hub<1.0,>=0.19.3 in /usr/local/lib/python3.10/dist-packages (from transformers>=4.38.0->vllm==0.3.3) (0.20.3)\n",
            "Requirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.10/dist-packages (from transformers>=4.38.0->vllm==0.3.3) (2023.12.25)\n",
            "Requirement already satisfied: tokenizers<0.19,>=0.14 in /usr/local/lib/python3.10/dist-packages (from transformers>=4.38.0->vllm==0.3.3) (0.15.2)\n",
            "Requirement already satisfied: safetensors>=0.4.1 in /usr/local/lib/python3.10/dist-packages (from transformers>=4.38.0->vllm==0.3.3) (0.4.2)\n",
            "Requirement already satisfied: tqdm>=4.27 in /usr/local/lib/python3.10/dist-packages (from transformers>=4.38.0->vllm==0.3.3) (4.66.2)\n",
            "Collecting starlette<0.38.0,>=0.37.2 (from fastapi->vllm==0.3.3)\n",
            "  Downloading starlette-0.37.2-py3-none-any.whl (71 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m71.9/71.9 kB\u001b[0m \u001b[31m11.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting h11>=0.8 (from uvicorn[standard]->vllm==0.3.3)\n",
            "  Downloading h11-0.14.0-py3-none-any.whl (58 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m58.3/58.3 kB\u001b[0m \u001b[31m8.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting httptools>=0.5.0 (from uvicorn[standard]->vllm==0.3.3)\n",
            "  Downloading httptools-0.6.1-cp310-cp310-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (341 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m341.4/341.4 kB\u001b[0m \u001b[31m42.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting python-dotenv>=0.13 (from uvicorn[standard]->vllm==0.3.3)\n",
            "  Downloading python_dotenv-1.0.1-py3-none-any.whl (19 kB)\n",
            "Collecting uvloop!=0.15.0,!=0.15.1,>=0.14.0 (from uvicorn[standard]->vllm==0.3.3)\n",
            "  Downloading uvloop-0.19.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.4 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m3.4/3.4 MB\u001b[0m \u001b[31m93.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting watchfiles>=0.13 (from uvicorn[standard]->vllm==0.3.3)\n",
            "  Downloading watchfiles-0.21.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.3 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m1.3/1.3 MB\u001b[0m \u001b[31m54.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting websockets>=10.4 (from uvicorn[standard]->vllm==0.3.3)\n",
            "  Downloading websockets-12.0-cp310-cp310-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (130 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m130.2/130.2 kB\u001b[0m \u001b[31m20.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: anyio<5,>=3.4.0 in /usr/local/lib/python3.10/dist-packages (from starlette<0.38.0,>=0.37.2->fastapi->vllm==0.3.3) (3.7.1)\n",
            "Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from jinja2->torch==2.1.2->vllm==0.3.3) (2.1.5)\n",
            "Requirement already satisfied: attrs>=22.2.0 in /usr/local/lib/python3.10/dist-packages (from jsonschema->outlines>=0.0.27->vllm==0.3.3) (23.2.0)\n",
            "Requirement already satisfied: jsonschema-specifications>=2023.03.6 in /usr/local/lib/python3.10/dist-packages (from jsonschema->outlines>=0.0.27->vllm==0.3.3) (2023.12.1)\n",
            "Requirement already satisfied: rpds-py>=0.7.1 in /usr/local/lib/python3.10/dist-packages (from jsonschema->outlines>=0.0.27->vllm==0.3.3) (0.18.0)\n",
            "Requirement already satisfied: llvmlite<0.42,>=0.41.0dev0 in /usr/local/lib/python3.10/dist-packages (from numba->outlines>=0.0.27->vllm==0.3.3) (0.41.1)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests->outlines>=0.0.27->vllm==0.3.3) (3.3.2)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests->outlines>=0.0.27->vllm==0.3.3) (3.6)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests->outlines>=0.0.27->vllm==0.3.3) (2.0.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests->outlines>=0.0.27->vllm==0.3.3) (2024.2.2)\n",
            "Requirement already satisfied: mpmath>=0.19 in /usr/local/lib/python3.10/dist-packages (from sympy->torch==2.1.2->vllm==0.3.3) (1.3.0)\n",
            "Requirement already satisfied: sniffio>=1.1 in /usr/local/lib/python3.10/dist-packages (from anyio<5,>=3.4.0->starlette<0.38.0,>=0.37.2->fastapi->vllm==0.3.3) (1.3.1)\n",
            "Requirement already satisfied: exceptiongroup in /usr/local/lib/python3.10/dist-packages (from anyio<5,>=3.4.0->starlette<0.38.0,>=0.37.2->fastapi->vllm==0.3.3) (1.2.0)\n",
            "Installing collected packages: ninja, websockets, uvloop, triton, python-dotenv, pynvml, nvidia-nvtx-cu12, nvidia-nvjitlink-cu12, nvidia-nccl-cu12, nvidia-curand-cu12, nvidia-cufft-cu12, nvidia-cuda-runtime-cu12, nvidia-cuda-nvrtc-cu12, nvidia-cuda-cupti-cu12, nvidia-cublas-cu12, lark, interegular, httptools, h11, diskcache, cupy-cuda12x, watchfiles, uvicorn, starlette, nvidia-cusparse-cu12, nvidia-cudnn-cu12, nvidia-cusolver-cu12, fastapi, torch, ray, xformers, outlines, vllm\n",
            "  Attempting uninstall: triton\n",
            "    Found existing installation: triton 2.2.0\n",
            "    Uninstalling triton-2.2.0:\n",
            "      Successfully uninstalled triton-2.2.0\n",
            "  Attempting uninstall: cupy-cuda12x\n",
            "    Found existing installation: cupy-cuda12x 12.2.0\n",
            "    Uninstalling cupy-cuda12x-12.2.0:\n",
            "      Successfully uninstalled cupy-cuda12x-12.2.0\n",
            "  Attempting uninstall: torch\n",
            "    Found existing installation: torch 2.2.1+cu121\n",
            "    Uninstalling torch-2.2.1+cu121:\n",
            "      Successfully uninstalled torch-2.2.1+cu121\n",
            "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
            "torchaudio 2.2.1+cu121 requires torch==2.2.1, but you have torch 2.1.2 which is incompatible.\n",
            "torchtext 0.17.1 requires torch==2.2.1, but you have torch 2.1.2 which is incompatible.\n",
            "torchvision 0.17.1+cu121 requires torch==2.2.1, but you have torch 2.1.2 which is incompatible.\u001b[0m\u001b[31m\n",
            "\u001b[0mSuccessfully installed cupy-cuda12x-12.1.0 diskcache-5.6.3 fastapi-0.110.1 h11-0.14.0 httptools-0.6.1 interegular-0.3.3 lark-1.1.9 ninja-1.11.1.1 nvidia-cublas-cu12-12.1.3.1 nvidia-cuda-cupti-cu12-12.1.105 nvidia-cuda-nvrtc-cu12-12.1.105 nvidia-cuda-runtime-cu12-12.1.105 nvidia-cudnn-cu12-8.9.2.26 nvidia-cufft-cu12-11.0.2.54 nvidia-curand-cu12-10.3.2.106 nvidia-cusolver-cu12-11.4.5.107 nvidia-cusparse-cu12-12.1.0.106 nvidia-nccl-cu12-2.18.1 nvidia-nvjitlink-cu12-12.4.127 nvidia-nvtx-cu12-12.1.105 outlines-0.0.37 pynvml-11.5.0 python-dotenv-1.0.1 ray-2.10.0 starlette-0.37.2 torch-2.1.2 triton-2.1.0 uvicorn-0.29.0 uvloop-0.19.0 vllm-0.3.3 watchfiles-0.21.0 websockets-12.0 xformers-0.0.23.post1\n"
          ]
        }
      ],
      "source": [
        "!pip install vllm==0.3.3"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 61,
      "metadata": {
        "id": "j1CKCFulLClk"
      },
      "outputs": [],
      "source": [
        "from vllm import LLM\n",
        "from vllm import SamplingParams\n",
        "import pandas as pd"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 62,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "k61J0LDGOdu9",
        "outputId": "c237259b-9e82-402c-9057-bd393294f605"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount(\"/content/drive\", force_remount=True).\n"
          ]
        }
      ],
      "source": [
        "from google.colab import drive\n",
        "\n",
        "drive.mount(\"/content/drive\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 63,
      "metadata": {
        "id": "JetwnNLoNgYA"
      },
      "outputs": [],
      "source": [
        "import pandas as pd\n",
        "\n",
        "train_filepath = \"/content/drive/MyDrive/biolaysumm2024_data/\"\n",
        "train_plos_filename = \"PLOS_train.jsonl\"\n",
        "train_elife_filename = \"eLife_train.jsonl\"\n",
        "\n",
        "def read_jsonl(filepath, filename):\n",
        "    df = pd.read_json(filepath + filename, orient=\"records\", lines=True)\n",
        "    return df"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 64,
      "metadata": {
        "id": "cOsKIZz0T_Ce"
      },
      "outputs": [],
      "source": [
        "train_plos_df = read_jsonl(train_filepath, train_plos_filename)\n",
        "train_elife_df = read_jsonl(train_filepath, train_elife_filename)\n",
        "\n",
        "mini_plos_df = read_jsonl(mini_dev_filepath, mini_dev_plos_filename)\n",
        "mini_elife_df = read_jsonl(mini_dev_filepath, mini_dev_elife_filename)\n",
        "\n",
        "# full_plos_df = read_jsonl(full_dev_filepath, full_dev_plos_filename)\n",
        "# full_elife_df = read_jsonl(full_dev_filepath, full_dev_elife_filename)\n",
        "\n",
        "test_plos_df = read_jsonl(test_filepath, test_plos_filename)\n",
        "test_elife_df = read_jsonl(test_filepath, test_elife_filename)"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [],
      "metadata": {
        "id": "npgpq_WgyOJF"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import pandas as pd\n",
        "import random\n",
        "\n",
        "random.seed(42)\n",
        "torch.manual_seed(42)\n",
        "\n",
        "plos_sample = train_plos_df.sample(5)\n",
        "elife_sample = train_elife_df.sample(5)\n",
        "\n",
        "combined_samples = pd.concat([plos_sample, elife_sample]).sample(frac=1).reset_index(drop=True)\n",
        "\n",
        "print(combined_samples)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "-goqrD2Mthgw",
        "outputId": "19e2bddb-9985-4ecf-864d-35ae18ccc53a"
      },
      "execution_count": 65,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "                                         lay_summary  \\\n",
            "0  Embryonic stem cells and induced pluripotent s...   \n",
            "1  During an animal’s lifetime , many of its cell...   \n",
            "2  Parasitic helminths are inducers of chronic di...   \n",
            "3  Genes encode instructions to make proteins and...   \n",
            "4  Rough-skinned newts produce tetrodotoxin or TT...   \n",
            "5  DNA carries the genetic information that is es...   \n",
            "6  Interferon Regulatory Factor 3 ( IRF3 ) is an ...   \n",
            "7  Inside cells , proteins are produced by comple...   \n",
            "8  Plasmacytoid dendritic cells ( pDC ) are innat...   \n",
            "9  Gene action determines how mutations affect ph...   \n",
            "\n",
            "                                             article  \\\n",
            "0  Embryonic stem cells and induced pluripotent s...   \n",
            "1  Mesenchymal ( lamellipodial ) migration is het...   \n",
            "2  Alveolar echinococcosis , caused by Echinococc...   \n",
            "3  Errors during transcription may play an import...   \n",
            "4  Rough-skinned newts ( Taricha granulosa ) use ...   \n",
            "5  The Dna2 nuclease-helicase maintains genomic i...   \n",
            "6  Innate immunity is the first line of defense a...   \n",
            "7  Using cryo-electron microscopy ( cryo-EM ) , w...   \n",
            "8  Plasmacytoid dendritic cells ( pDC ) are innat...   \n",
            "9  The genetic component of complex disease risk ...   \n",
            "\n",
            "                                            headings  \\\n",
            "0  [Abstract, Introduction, Results, Discussion, ...   \n",
            "1  [Abstract, Introduction, Results, Discussion, ...   \n",
            "2  [Abstract, Introduction, Materials and Methods...   \n",
            "3                  [Abstract, Materials and methods]   \n",
            "4  [Abstract, Introduction, Results, Discussion, ...   \n",
            "5  [Abstract, Introduction, Results and discussio...   \n",
            "6  [Abstract, Introduction, Results, Discussion, ...   \n",
            "7  [Abstract, Introduction, Results, Discussion, ...   \n",
            "8  [Abstract, Introduction, Results, Discussion, ...   \n",
            "9  [Abstract, Introduction, Results and Discussio...   \n",
            "\n",
            "                                            keywords                    id  \n",
            "0  [mechanisms of signal transduction, cell diffe...  journal.pgen.1003112  \n",
            "1  [computational and systems biology, cell biolo...        elife-11384-v1  \n",
            "2  [medicine, immune cells, clinical immunology, ...  journal.pntd.0001516  \n",
            "3  [short report, computational and systems biology]        elife-09945-v2  \n",
            "4                             [evolutionary biology]        elife-53898-v1  \n",
            "5  [chromosomes and gene expression, structural b...        elife-09832-v3  \n",
            "6                                                 []  journal.ppat.1004779  \n",
            "7      [structural biology and molecular biophysics]        elife-60482-v2  \n",
            "8  [medicine and health sciences, lysosomes, vesi...  journal.ppat.1005553  \n",
            "9  [genome-wide association studies, deletion mut...  journal.pgen.1006573  \n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "def build_text_with_headings(item):\n",
        "    \"\"\"\n",
        "    return the article text with heading embedded\n",
        "    \"\"\"\n",
        "\n",
        "    result = \"\"\n",
        "    paras = item[\"article\"].split(\"\\n\")\n",
        "    keywords = ', '.join(item[\"keywords\"])\n",
        "    keywords = f\"## Keywords: {keywords}\"\n",
        "    result += keywords + \"\\n\"\n",
        "\n",
        "    headings = item[\"headings\"]\n",
        "    if len(paras) != len(headings):\n",
        "        print(\"Error, not matching length\")\n",
        "        return item[\"article\"]\n",
        "    else:\n",
        "        for (heading, paragraph) in zip(headings, paras):\n",
        "            result += f\"## {heading}\\n{paragraph}\\n\\n\"\n",
        "\n",
        "    return result"
      ],
      "metadata": {
        "id": "v6zOFuah3QxP"
      },
      "execution_count": 66,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "import pandas as pd\n",
        "\n",
        "def get_unique_headings(df, headings_column):\n",
        "    all_headings = [heading for sublist in df[headings_column] for heading in sublist]\n",
        "    unique_headings = set(all_headings)\n",
        "    return unique_headings\n",
        "\n",
        "unique_headings = get_unique_headings(combined_samples, 'headings')\n",
        "\n",
        "print(\"Unique Headings:\", unique_headings)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "BYv8IifdJ_X1",
        "outputId": "4844146b-a96f-44dd-f0d9-3be7d327e025"
      },
      "execution_count": 67,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Unique Headings: {'Materials and methods', 'Results and discussion', 'Introduction', 'Conclusions', 'Materials and Methods', 'Abstract', 'Results', 'Results and Discussion', 'Discussion'}\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "combined_samples['processed_text'] = combined_samples.apply(lambda row: build_text_with_headings(row), axis=1)"
      ],
      "metadata": {
        "id": "dhuw8zvRD7cL"
      },
      "execution_count": 68,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "def split_into_sections(text, unique_headings):\n",
        "    unique_headings = {heading.replace('\\xa0', ' ') for heading in unique_headings}\n",
        "\n",
        "    section_dict = {heading: \"\" for heading in unique_headings}\n",
        "\n",
        "    sections = text.split(\"\\n\\n\")\n",
        "    current_heading = None\n",
        "\n",
        "    for section in sections:\n",
        "        section_clean = section.strip().replace('\\xa0', ' ')\n",
        "        for heading in unique_headings:\n",
        "            heading_marker = f\"## {heading}\"\n",
        "            if section_clean.startswith(heading_marker):\n",
        "                current_heading = heading\n",
        "                section_text = section_clean[len(heading_marker):].strip()\n",
        "                section_dict[heading] += section_text + \"\\n\\n\"\n",
        "                break\n",
        "        else:\n",
        "            if current_heading:\n",
        "                section_dict[current_heading] += section + \"\\n\\n\"\n",
        "\n",
        "    return section_dict"
      ],
      "metadata": {
        "id": "lXdiw_nFKJGO"
      },
      "execution_count": 69,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "def split_text_by_headings(text):\n",
        "    lines = text.split(\"\\n\")\n",
        "    split_sections = {}\n",
        "    current_heading = None\n",
        "    section_content = []\n",
        "\n",
        "    for line in lines:\n",
        "        if line.startswith(\"##\"):\n",
        "            if current_heading is not None:\n",
        "                split_sections[current_heading] = \"\\n\".join(section_content).strip()\n",
        "                section_content = []\n",
        "\n",
        "            current_heading = line[2:].strip()\n",
        "        else:\n",
        "            section_content.append(line)\n",
        "\n",
        "    if current_heading is not None and section_content:\n",
        "        split_sections[current_heading] = \"\\n\".join(section_content).strip()\n",
        "\n",
        "    return split_sections\n"
      ],
      "metadata": {
        "id": "54akv_pYKjRg"
      },
      "execution_count": 70,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "combined_samples['split_sections'] = combined_samples['processed_text'].apply(split_text_by_headings)\n",
        "\n",
        "unique_headings = {'Abstract', 'Introduction', 'Results', 'Discussion', 'Methods', 'Materials and Methods', 'Results and Discussion', 'Materials and methods'}\n",
        "\n",
        "for heading in unique_headings:\n",
        "    combined_samples[heading] = combined_samples['split_sections'].apply(lambda sections: sections.get(heading, \"\"))"
      ],
      "metadata": {
        "id": "6Phg56GuL4hy"
      },
      "execution_count": 71,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "combined_samples"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "QiM552TyL8ci",
        "outputId": "6d1fa2fa-e98d-446e-af88-b638835184f9"
      },
      "execution_count": 72,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "                                         lay_summary  \\\n",
              "0  Embryonic stem cells and induced pluripotent s...   \n",
              "1  During an animal’s lifetime , many of its cell...   \n",
              "2  Parasitic helminths are inducers of chronic di...   \n",
              "3  Genes encode instructions to make proteins and...   \n",
              "4  Rough-skinned newts produce tetrodotoxin or TT...   \n",
              "5  DNA carries the genetic information that is es...   \n",
              "6  Interferon Regulatory Factor 3 ( IRF3 ) is an ...   \n",
              "7  Inside cells , proteins are produced by comple...   \n",
              "8  Plasmacytoid dendritic cells ( pDC ) are innat...   \n",
              "9  Gene action determines how mutations affect ph...   \n",
              "\n",
              "                                             article  \\\n",
              "0  Embryonic stem cells and induced pluripotent s...   \n",
              "1  Mesenchymal ( lamellipodial ) migration is het...   \n",
              "2  Alveolar echinococcosis , caused by Echinococc...   \n",
              "3  Errors during transcription may play an import...   \n",
              "4  Rough-skinned newts ( Taricha granulosa ) use ...   \n",
              "5  The Dna2 nuclease-helicase maintains genomic i...   \n",
              "6  Innate immunity is the first line of defense a...   \n",
              "7  Using cryo-electron microscopy ( cryo-EM ) , w...   \n",
              "8  Plasmacytoid dendritic cells ( pDC ) are innat...   \n",
              "9  The genetic component of complex disease risk ...   \n",
              "\n",
              "                                            headings  \\\n",
              "0  [Abstract, Introduction, Results, Discussion, ...   \n",
              "1  [Abstract, Introduction, Results, Discussion, ...   \n",
              "2  [Abstract, Introduction, Materials and Methods...   \n",
              "3                  [Abstract, Materials and methods]   \n",
              "4  [Abstract, Introduction, Results, Discussion, ...   \n",
              "5  [Abstract, Introduction, Results and discussio...   \n",
              "6  [Abstract, Introduction, Results, Discussion, ...   \n",
              "7  [Abstract, Introduction, Results, Discussion, ...   \n",
              "8  [Abstract, Introduction, Results, Discussion, ...   \n",
              "9  [Abstract, Introduction, Results and Discussio...   \n",
              "\n",
              "                                            keywords                    id  \\\n",
              "0  [mechanisms of signal transduction, cell diffe...  journal.pgen.1003112   \n",
              "1  [computational and systems biology, cell biolo...        elife-11384-v1   \n",
              "2  [medicine, immune cells, clinical immunology, ...  journal.pntd.0001516   \n",
              "3  [short report, computational and systems biology]        elife-09945-v2   \n",
              "4                             [evolutionary biology]        elife-53898-v1   \n",
              "5  [chromosomes and gene expression, structural b...        elife-09832-v3   \n",
              "6                                                 []  journal.ppat.1004779   \n",
              "7      [structural biology and molecular biophysics]        elife-60482-v2   \n",
              "8  [medicine and health sciences, lysosomes, vesi...  journal.ppat.1005553   \n",
              "9  [genome-wide association studies, deletion mut...  journal.pgen.1006573   \n",
              "\n",
              "                                      processed_text  \\\n",
              "0  ## Keywords: mechanisms of signal transduction...   \n",
              "1  ## Keywords: computational and systems biology...   \n",
              "2  ## Keywords: medicine, immune cells, clinical ...   \n",
              "3  ## Keywords: short report, computational and s...   \n",
              "4  ## Keywords: evolutionary biology\\n## Abstract...   \n",
              "5  ## Keywords: chromosomes and gene expression, ...   \n",
              "6  ## Keywords: \\n## Abstract\\nInnate immunity is...   \n",
              "7  ## Keywords: structural biology and molecular ...   \n",
              "8  ## Keywords: medicine and health sciences, lys...   \n",
              "9  ## Keywords: genome-wide association studies, ...   \n",
              "\n",
              "                                      split_sections  \\\n",
              "0  {'Keywords: mechanisms of signal transduction,...   \n",
              "1  {'Keywords: computational and systems biology,...   \n",
              "2  {'Keywords: medicine, immune cells, clinical i...   \n",
              "3  {'Keywords: short report, computational and sy...   \n",
              "4  {'Keywords: evolutionary biology': '', 'Abstra...   \n",
              "5  {'Keywords: chromosomes and gene expression, s...   \n",
              "6  {'Keywords:': '', 'Abstract': 'Innate immunity...   \n",
              "7  {'Keywords: structural biology and molecular b...   \n",
              "8  {'Keywords: medicine and health sciences, lyso...   \n",
              "9  {'Keywords: genome-wide association studies, d...   \n",
              "\n",
              "                               Materials and methods Methods  \\\n",
              "0                                                              \n",
              "1  We have integrated a comprehensive and unique ...           \n",
              "2                                                              \n",
              "3  Much existing RNA-seq data is available as bam...           \n",
              "4  Adult male rough-skinned newts ( Taricha granu...           \n",
              "5  Full-length mouse Dna2 was cloned into a pFast...           \n",
              "6                                                              \n",
              "7  E . coli 70S ribosome purification ( Travin et...           \n",
              "8                                                              \n",
              "9                                                              \n",
              "\n",
              "                                        Introduction  \\\n",
              "0  Embryonic stem cells and induced pluripotent s...   \n",
              "1  Cell migration is a profoundly heterogeneous p...   \n",
              "2  The metacestode larval stage of the fox-tapewo...   \n",
              "3                                                      \n",
              "4  Coevolutionary interactions among species are ...   \n",
              "5  Dna2 has nuclease and helicase activities and ...   \n",
              "6  Toxoplasma gondii , an obligate intracellular ...   \n",
              "7  The ribosome performs the crucial task of tran...   \n",
              "8  Type I interferon ( IFN ) plays a dichotomous ...   \n",
              "9  Risk for complex diseases in humans , such as ...   \n",
              "\n",
              "                               Materials and Methods  \\\n",
              "0  ES cells were generally maintained in NDiff N2...   \n",
              "1                                                      \n",
              "2  All experiments were carried out in accordance...   \n",
              "3                                                      \n",
              "4                                                      \n",
              "5                                                      \n",
              "6  H196 , H1048 , HME , and HEK293 cell lines wer...   \n",
              "7                                                      \n",
              "8  PBMCs were separated on Ficoll-Hypaque ( Amers...   \n",
              "9  Using the fwdpp template library v0 . 2 . 8 [8...   \n",
              "\n",
              "                                            Abstract  \\\n",
              "0  Embryonic stem cells and induced pluripotent s...   \n",
              "1  Mesenchymal ( lamellipodial ) migration is het...   \n",
              "2  Alveolar echinococcosis , caused by Echinococc...   \n",
              "3  Errors during transcription may play an import...   \n",
              "4  Rough-skinned newts ( Taricha granulosa ) use ...   \n",
              "5  The Dna2 nuclease-helicase maintains genomic i...   \n",
              "6  Innate immunity is the first line of defense a...   \n",
              "7  Using cryo-electron microscopy ( cryo-EM ) , w...   \n",
              "8  Plasmacytoid dendritic cells ( pDC ) are innat...   \n",
              "9  The genetic component of complex disease risk ...   \n",
              "\n",
              "                                             Results  \\\n",
              "0  To identify the programme of genes involved in...   \n",
              "1  To enable the detection of discrete mesenchyma...   \n",
              "2  The morphology of the three different E . mult...   \n",
              "3                                                      \n",
              "4  To investigate whether bacterial symbionts pro...   \n",
              "5                                                      \n",
              "6  To investigate whether the type I IFN system r...   \n",
              "7  We determined the structure of the E . coli 70...   \n",
              "8  We hypothesized that HIV envelope protein inte...   \n",
              "9                                                      \n",
              "\n",
              "                              Results and Discussion  \\\n",
              "0                                                      \n",
              "1                                                      \n",
              "2                                                      \n",
              "3                                                      \n",
              "4                                                      \n",
              "5                                                      \n",
              "6                                                      \n",
              "7                                                      \n",
              "8                                                      \n",
              "9  As in [36] , we simulate a 100 kilobase region...   \n",
              "\n",
              "                                          Discussion  \n",
              "0  The derivation of pluripotent iPS cells and th...  \n",
              "1  We here present an integrated analytical appro...  \n",
              "2  As typical in the case of helminth infections ...  \n",
              "3                                                     \n",
              "4  In this study , we found that bacterial isolat...  \n",
              "5                                                     \n",
              "6  We have uncovered a new signaling pathway , PI...  \n",
              "7  High-resolution cryo-EM maps are now on the cu...  \n",
              "8  Although many receptors and signaling pathways...  \n",
              "9                                                     "
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-80e6f78b-570f-417f-a4fa-82385fbbf051\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>lay_summary</th>\n",
              "      <th>article</th>\n",
              "      <th>headings</th>\n",
              "      <th>keywords</th>\n",
              "      <th>id</th>\n",
              "      <th>processed_text</th>\n",
              "      <th>split_sections</th>\n",
              "      <th>Materials and methods</th>\n",
              "      <th>Methods</th>\n",
              "      <th>Introduction</th>\n",
              "      <th>Materials and Methods</th>\n",
              "      <th>Abstract</th>\n",
              "      <th>Results</th>\n",
              "      <th>Results and Discussion</th>\n",
              "      <th>Discussion</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>Embryonic stem cells and induced pluripotent s...</td>\n",
              "      <td>Embryonic stem cells and induced pluripotent s...</td>\n",
              "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
              "      <td>[mechanisms of signal transduction, cell diffe...</td>\n",
              "      <td>journal.pgen.1003112</td>\n",
              "      <td>## Keywords: mechanisms of signal transduction...</td>\n",
              "      <td>{'Keywords: mechanisms of signal transduction,...</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td>Embryonic stem cells and induced pluripotent s...</td>\n",
              "      <td>ES cells were generally maintained in NDiff N2...</td>\n",
              "      <td>Embryonic stem cells and induced pluripotent s...</td>\n",
              "      <td>To identify the programme of genes involved in...</td>\n",
              "      <td></td>\n",
              "      <td>The derivation of pluripotent iPS cells and th...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>During an animal’s lifetime , many of its cell...</td>\n",
              "      <td>Mesenchymal ( lamellipodial ) migration is het...</td>\n",
              "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
              "      <td>[computational and systems biology, cell biolo...</td>\n",
              "      <td>elife-11384-v1</td>\n",
              "      <td>## Keywords: computational and systems biology...</td>\n",
              "      <td>{'Keywords: computational and systems biology,...</td>\n",
              "      <td>We have integrated a comprehensive and unique ...</td>\n",
              "      <td></td>\n",
              "      <td>Cell migration is a profoundly heterogeneous p...</td>\n",
              "      <td></td>\n",
              "      <td>Mesenchymal ( lamellipodial ) migration is het...</td>\n",
              "      <td>To enable the detection of discrete mesenchyma...</td>\n",
              "      <td></td>\n",
              "      <td>We here present an integrated analytical appro...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>Parasitic helminths are inducers of chronic di...</td>\n",
              "      <td>Alveolar echinococcosis , caused by Echinococc...</td>\n",
              "      <td>[Abstract, Introduction, Materials and Methods...</td>\n",
              "      <td>[medicine, immune cells, clinical immunology, ...</td>\n",
              "      <td>journal.pntd.0001516</td>\n",
              "      <td>## Keywords: medicine, immune cells, clinical ...</td>\n",
              "      <td>{'Keywords: medicine, immune cells, clinical i...</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td>The metacestode larval stage of the fox-tapewo...</td>\n",
              "      <td>All experiments were carried out in accordance...</td>\n",
              "      <td>Alveolar echinococcosis , caused by Echinococc...</td>\n",
              "      <td>The morphology of the three different E . mult...</td>\n",
              "      <td></td>\n",
              "      <td>As typical in the case of helminth infections ...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>Genes encode instructions to make proteins and...</td>\n",
              "      <td>Errors during transcription may play an import...</td>\n",
              "      <td>[Abstract, Materials and methods]</td>\n",
              "      <td>[short report, computational and systems biology]</td>\n",
              "      <td>elife-09945-v2</td>\n",
              "      <td>## Keywords: short report, computational and s...</td>\n",
              "      <td>{'Keywords: short report, computational and sy...</td>\n",
              "      <td>Much existing RNA-seq data is available as bam...</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td>Errors during transcription may play an import...</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>Rough-skinned newts produce tetrodotoxin or TT...</td>\n",
              "      <td>Rough-skinned newts ( Taricha granulosa ) use ...</td>\n",
              "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
              "      <td>[evolutionary biology]</td>\n",
              "      <td>elife-53898-v1</td>\n",
              "      <td>## Keywords: evolutionary biology\\n## Abstract...</td>\n",
              "      <td>{'Keywords: evolutionary biology': '', 'Abstra...</td>\n",
              "      <td>Adult male rough-skinned newts ( Taricha granu...</td>\n",
              "      <td></td>\n",
              "      <td>Coevolutionary interactions among species are ...</td>\n",
              "      <td></td>\n",
              "      <td>Rough-skinned newts ( Taricha granulosa ) use ...</td>\n",
              "      <td>To investigate whether bacterial symbionts pro...</td>\n",
              "      <td></td>\n",
              "      <td>In this study , we found that bacterial isolat...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>DNA carries the genetic information that is es...</td>\n",
              "      <td>The Dna2 nuclease-helicase maintains genomic i...</td>\n",
              "      <td>[Abstract, Introduction, Results and discussio...</td>\n",
              "      <td>[chromosomes and gene expression, structural b...</td>\n",
              "      <td>elife-09832-v3</td>\n",
              "      <td>## Keywords: chromosomes and gene expression, ...</td>\n",
              "      <td>{'Keywords: chromosomes and gene expression, s...</td>\n",
              "      <td>Full-length mouse Dna2 was cloned into a pFast...</td>\n",
              "      <td></td>\n",
              "      <td>Dna2 has nuclease and helicase activities and ...</td>\n",
              "      <td></td>\n",
              "      <td>The Dna2 nuclease-helicase maintains genomic i...</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>Interferon Regulatory Factor 3 ( IRF3 ) is an ...</td>\n",
              "      <td>Innate immunity is the first line of defense a...</td>\n",
              "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
              "      <td>[]</td>\n",
              "      <td>journal.ppat.1004779</td>\n",
              "      <td>## Keywords: \\n## Abstract\\nInnate immunity is...</td>\n",
              "      <td>{'Keywords:': '', 'Abstract': 'Innate immunity...</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td>Toxoplasma gondii , an obligate intracellular ...</td>\n",
              "      <td>H196 , H1048 , HME , and HEK293 cell lines wer...</td>\n",
              "      <td>Innate immunity is the first line of defense a...</td>\n",
              "      <td>To investigate whether the type I IFN system r...</td>\n",
              "      <td></td>\n",
              "      <td>We have uncovered a new signaling pathway , PI...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>Inside cells , proteins are produced by comple...</td>\n",
              "      <td>Using cryo-electron microscopy ( cryo-EM ) , w...</td>\n",
              "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
              "      <td>[structural biology and molecular biophysics]</td>\n",
              "      <td>elife-60482-v2</td>\n",
              "      <td>## Keywords: structural biology and molecular ...</td>\n",
              "      <td>{'Keywords: structural biology and molecular b...</td>\n",
              "      <td>E . coli 70S ribosome purification ( Travin et...</td>\n",
              "      <td></td>\n",
              "      <td>The ribosome performs the crucial task of tran...</td>\n",
              "      <td></td>\n",
              "      <td>Using cryo-electron microscopy ( cryo-EM ) , w...</td>\n",
              "      <td>We determined the structure of the E . coli 70...</td>\n",
              "      <td></td>\n",
              "      <td>High-resolution cryo-EM maps are now on the cu...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>Plasmacytoid dendritic cells ( pDC ) are innat...</td>\n",
              "      <td>Plasmacytoid dendritic cells ( pDC ) are innat...</td>\n",
              "      <td>[Abstract, Introduction, Results, Discussion, ...</td>\n",
              "      <td>[medicine and health sciences, lysosomes, vesi...</td>\n",
              "      <td>journal.ppat.1005553</td>\n",
              "      <td>## Keywords: medicine and health sciences, lys...</td>\n",
              "      <td>{'Keywords: medicine and health sciences, lyso...</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td>Type I interferon ( IFN ) plays a dichotomous ...</td>\n",
              "      <td>PBMCs were separated on Ficoll-Hypaque ( Amers...</td>\n",
              "      <td>Plasmacytoid dendritic cells ( pDC ) are innat...</td>\n",
              "      <td>We hypothesized that HIV envelope protein inte...</td>\n",
              "      <td></td>\n",
              "      <td>Although many receptors and signaling pathways...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td>Gene action determines how mutations affect ph...</td>\n",
              "      <td>The genetic component of complex disease risk ...</td>\n",
              "      <td>[Abstract, Introduction, Results and Discussio...</td>\n",
              "      <td>[genome-wide association studies, deletion mut...</td>\n",
              "      <td>journal.pgen.1006573</td>\n",
              "      <td>## Keywords: genome-wide association studies, ...</td>\n",
              "      <td>{'Keywords: genome-wide association studies, d...</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td>Risk for complex diseases in humans , such as ...</td>\n",
              "      <td>Using the fwdpp template library v0 . 2 . 8 [8...</td>\n",
              "      <td>The genetic component of complex disease risk ...</td>\n",
              "      <td></td>\n",
              "      <td>As in [36] , we simulate a 100 kilobase region...</td>\n",
              "      <td></td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-80e6f78b-570f-417f-a4fa-82385fbbf051')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-80e6f78b-570f-417f-a4fa-82385fbbf051 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-80e6f78b-570f-417f-a4fa-82385fbbf051');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-6592779d-4dee-4c07-a250-bbb35328105e\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-6592779d-4dee-4c07-a250-bbb35328105e')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-6592779d-4dee-4c07-a250-bbb35328105e button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "  <div id=\"id_5235dda3-fc06-448e-af1d-42d6900cdbc6\">\n",
              "    <style>\n",
              "      .colab-df-generate {\n",
              "        background-color: #E8F0FE;\n",
              "        border: none;\n",
              "        border-radius: 50%;\n",
              "        cursor: pointer;\n",
              "        display: none;\n",
              "        fill: #1967D2;\n",
              "        height: 32px;\n",
              "        padding: 0 0 0 0;\n",
              "        width: 32px;\n",
              "      }\n",
              "\n",
              "      .colab-df-generate:hover {\n",
              "        background-color: #E2EBFA;\n",
              "        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "        fill: #174EA6;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate {\n",
              "        background-color: #3B4455;\n",
              "        fill: #D2E3FC;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate:hover {\n",
              "        background-color: #434B5C;\n",
              "        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "        fill: #FFFFFF;\n",
              "      }\n",
              "    </style>\n",
              "    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('combined_samples')\"\n",
              "            title=\"Generate code using this dataframe.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "    <script>\n",
              "      (() => {\n",
              "      const buttonEl =\n",
              "        document.querySelector('#id_5235dda3-fc06-448e-af1d-42d6900cdbc6 button.colab-df-generate');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      buttonEl.onclick = () => {\n",
              "        google.colab.notebook.generateWithVariable('combined_samples');\n",
              "      }\n",
              "      })();\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "combined_samples",
              "summary": "{\n  \"name\": \"combined_samples\",\n  \"rows\": 10,\n  \"fields\": [\n    {\n      \"column\": \"lay_summary\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 10,\n        \"samples\": [\n          \"Plasmacytoid dendritic cells ( pDC ) are innate immune cells that are specialized to produce type I interferon ( IFN ) and to activate adaptive immune responses . Although IFN is an anti-viral cytokine , it may contribute more to pathogenesis than to protection during chronic viral infections , including chronic HIV infection . pDC sense HIV to produce abundant IFN but minimal NF- \\u03baB\\u2013dependent production of TNF\\u03b1 and minimal up-regulation of co-stimulatory molecules , suggesting that HIV promotes pDC to become interferon producing cells ( IPC ) rather than antigen presenting cells ( APC ) . Here , we use florescent HIV virions pseudotyped with influenza hemagglutinin ( HA ) envelope and a cell system expressing CD4 molecules with modified intracellular trafficking . We found that HIV virions pseudotyped with HA stimulate pDC to mature , similar to influenza-stimulated pDC , and traffic intracellularly similarly to influenza . We also find that CD4-mediated intracellular trafficking guides HIV trafficking and downstream signaling . Our study presents new and important findings which demonstrate that divergent HIV sensing by pDC to produce IFN , rather than to become mature antigen presenting cells , is mediated specifically by CD4-HIV envelope interactions .\",\n          \"During an animal\\u2019s lifetime , many of its cells will move from one location in the body to another . For example , skin cells can migrate to repair wounds . Prior to migration , cells are usually attached to a scaffold called the extracellular-matrix , which helps to hold them in a particular location within a tissue . Individual cells can move in different ways . During a type of movement called mesenchymal migration , the front end of a cell grows outwards and attaches to a different section of the matrix . The rear of the cell is pulled forward and it detaches from the matrix and retracts , which allows the entire cell to move forward . In contrast , during amoeboid migration , the moving cells are only loosely attached to the matrix and move by gliding . There are large variations in how cells move and they can adopt modes that lie between the two extremes of mesenchymal and amoeboid migration . They can also switch between modes depending on their requirements . Shafqat-Abbasi et al . developed a method to analyse how individual human lung cancer cells move . The method uses software to collect data on cell shape , speed of movement and other features from microscopy images of the migrating cells . The experiments reveal that the cells adopt two distinct migration modes , which Shafqat-Abbasi et al . termed 'Discontinuous' and 'Continuous' . The majority of cells migrated in the Discontinuous mode , in which cells moved in many different directions . This was caused by a lack of coordination between the outgrowth of the front end of the cell , and the retraction of the back from the matrix . In contrast , in the cells that migrated using the Continuous mode , an outgrowth consistently led to a retraction , which enabled cells to move in one direction . Further experiments revealed that the mode of migration used by the cells is affected by how tightly they are bound to the extracellular-matrix , and the mechanical forces generated inside the cells to drive the movement . Shafqat-Abbasi et al . \\u2019s method provides an analytical toolbox that other researchers can use to study the mesenchymal migration of animal cells .\",\n          \"DNA carries the genetic information that is essential for organisms to survive and reproduce . It is made of two strands that twist together to form a double helix . However , these strands can be damaged when the DNA is copied before a cell divides , or by exposure to radiation or hazardous chemicals . To prevent this damage from causing serious harm to an organism , cells activate processes that rapidly repair the damaged DNA . \\u201cHomologous recombination\\u201d is one way in which cells can repair damage that has caused both strands of the DNA to break in a particular place . In the first step , several enzymes trim one of the two DNA strands at each broken end to leave single stranded \\u201ctails\\u201d . Dna2 is one enzyme that is involved in making these tails , but it can only bind to single-stranded DNA so it only acts after another enzyme has made some initial cuts . The exposed single stranded DNA then searches for an intact copy of itself elsewhere in the genome , which promotes its repair . It is important that only one of the two DNA strands is trimmed at each end otherwise the repair will fail . A protein called Rpa is bound to the DNA and is required for Dna2 to correctly trim the DNA . However , it is not clear exactly how Rpa2 regulates Dna2 . Zhou et al . used a technique called X-ray crystallography to analyze the three-dimensional structures of Dna2 when it is bound to single stranded DNA and when it is bound to Rpa . The experiments show that Dna2 adopts a cylindrical shape with a tunnel through which the single-stranded DNA passes through . The region of Dna2 that is capable of trimming DNA \\u2013 which is called the nuclease domain \\u2013 is embedded within the tunnel . The entrance to the tunnel is too narrow to allow double-stranded DNA to enter , so this explains why Dna2 can only act on double-stranded DNA that already has a small single-stranded section at the end . Inside the tunnel , Dna2 displaces Rpa from one of the strands , which allows Dna2 to trim the DNA . However , other molecules of Rpa remain firmly bound to the other strand to protect it from Dna2 . These enzymes also act in a similar way to trim DNA before it is copied in preparation for cell division . Zhou et al . \\u2019s findings provide an explanation for how Rpa determines which strand of DNA is trimmed by Dna2 . Further work is needed to understand how Dna2 and Rpa work with other enzymes to trim DNA .\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"article\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 10,\n        \"samples\": [\n          \"Plasmacytoid dendritic cells ( pDC ) are innate immune cells that sense viral nucleic acids through endosomal Toll-like receptor ( TLR ) 7/9 to produce type I interferon ( IFN ) and to differentiate into potent antigen presenting cells ( APC ) . Engagement of TLR7/9 in early endosomes appears to trigger the IRF7 pathway for IFN production whereas engagement in lysosomes seems to trigger the NF-\\u03baB pathway for maturation into APC . We showed previously that HIV-1 ( HIV ) localizes predominantly to early endosomes , not lysosomes , and mainly stimulate IRF7 rather than NF-\\u03baB signaling pathways in pDC . This divergent signaling may contribute to disease progression through production of pro-apoptotic and pro-inflammatory IFN and inadequate maturation of pDCs . We now demonstrate that HIV virions may be re-directed to lysosomes for NF-\\u03baB signaling by either pseudotyping HIV with influenza hemagglutinin envelope or modification of CD4 mediated-intracellular trafficking . These data suggest that HIV envelope-CD4 receptor interactions drive pDC activation toward an immature IFN producing phenotype rather than differentiation into a mature dendritic cell phenotype . \\n Type I interferon ( IFN ) plays a dichotomous role in chronic viral infections such as Human Immunodeficiency Virus-1 ( HIV ) , contributing to the control of viral replication during the earliest stages of infection , yet fueling disease progression by activating target cells for infection , decreasing antiviral gene expression , enabling infection with increased reservoir size , and accelerating CD4 T-cell loss [1\\u20138] . Plasmacytoid dendritic cells ( pDC ) are thought to play a significant role in IFN responses during HIV infection , arriving rapidly at sites of mucosal transmission [4] and relocating from blood to lymphoid tissues where they produce pro-apoptotic and pro-inflammatory IFN [9\\u201311] . Cellular mechanisms underlying HIV-stimulated IFN production by pDC are only partially understood . We have previously shown that abundant IFN is produced by pDC upon HIV stimulation through endosomal recognition of genomic RNA by TLR7 . This response requires the presence of HIV envelope protein on viral particles , interactions between CD4 and the viral envelope protein , HIV endocytosis and endosomal acidification; however , co-receptor usage , viral fusion and viral replication are not required [12 , 13] . Cell-to-cell infection seems to amplify pDC responses to HIV , however precise mechanisms underlying differences between cell-free and cell-to-cell pDC activation are not clearly defined [14] . We and others have shown that pDC are highly resistant to HIV infection , and this block to replication is IFN-independent [15 , 16] . In addition to IFN production , pDC can act as antigen-presenting cells ( APC ) to activate T-cell\\u2013mediated adaptive immune responses [17\\u201321] . Acquisition of an APC phenotype requires specific signals that are distinct from the signals that induce large amounts of IFN . We have previously shown that HIV stimulated pDC express low levels of the co-stimulatory molecule CD86 and express Indoleamine 2 , 3-dioxygenase ( IDO ) , a potent inducer of regulatory T cells , indicating that they do not differentiate into mature APC and fail to stimulate potent T cell responses [22 , 23] . However , pDC can differentiate into APC with influenza virus or the synthetic TLR7 agonist R837 and are able to cross-present antigens from HIV-1-infected apoptotic cells to HIV-specific CD8+ T lymphocytes , demonstrating that pDC do not have an intrinsic defect in presentation of HIV antigens , but rather that sensing of HIV does not provide the signals that are required for efficient differentiation of pDC into APC [17] . pDC sense single stranded RNA or unmethylated DNA containing Cytosine\\u2013Guanosine dinucleotides ( CpG ) through Toll-like receptors ( TLR ) 7 and 9 , respectively , located in endosomal compartments . Both TLR7 and TLR9 signal through the adapter protein myeloid differentiation primary response gene 88 ( MyD88 ) . Downstream IFN signaling occurs in response to activation of IFN genes through phosphorylation of interferon regulatory factor 7 ( IRF7 ) , whereas downstream signaling of nuclear factor kappa-light-chain-enhancer of activated B cells ( NF-\\u03baB ) leads to the transcriptional activation of proinflammatory kinases and upregulation of MHC and co-stimulatory molecules necessary for maturation into APC . [12 , 24] . The functional response of pDC to pathogens is flexible . As posited by the spatiotemporal model of pDC sensing [25] , differential pDC activation is likely related to the subcellular location where the TLR senses the pathogen . Thus , engagement of TLR 7/9 in the early endosomes of pDC preferentially triggers the IRF7 signal cascade , leading to type I IFN responses; whereas engagement of TLR7/9 in lysosomes preferentially triggers the NF-\\u03baB signal cascade , leading to the production of proinflammatory cytokines TNF\\u03b1 and IL6 , upregulation of co-stimulatory molecules , and an APC phenotype [25 , 26] . Differential trafficking and therefore sensing of synthetic TLR9-activating CpGs is attributed to sequence-related secondary and tertiary structural features of the CpGs . CpGs which contain phosphodiester backbones and palindromic motifs ( CpGA ) form multimeric complexes and traffic to early endosomes for IRF7 signaling whereas CpGs which contain phosphorothioate backbones and lack palindromic motifs ( CpGB ) traffic as monomers to lysosomes for NF-\\u03baB signaling . Intermediate CpGs ( CpGC ) combine structural elements of both CpGA and CpGB , traffic to both compartments , and stimulate both IRF7 and NF-\\u03baB signaling [27\\u201329] . While the spatiotemporal model of pDC sensing has been most clearly evaluated using synthetic TLR9 agonists ( CpG ) , we have shown that the model also applies to HIV and TLR7 , whereby HIV traffics to early endosomes in pDC , activating IRF7 signaling rather than NF-\\u03baB signaling [22 , 23] . The upstream events that determine activation of each of these pathways , and in particular , HIV virion trafficking in pDC , are currently unknown , however , prior studies suggest that HIV envelope may play a major role [13 , 30 , 31] . Here we demonstrate that HIV trafficking and pDC phenotype is predominantly determined by envelope-CD4 interaction , such that manipulation of HIV envelope or CD4 intracellular trafficking enables modulation of divergent sensing of HIV . \\n We hypothesized that HIV envelope protein interactions with cell surface CD4 determine the intracellular trafficking of HIV and the resultant signaling in pDC , based on the spatiotemporal model of TLR signaling [25] . To test this , we replaced HIV envelope protein with envelope protein from a virus that activates pDC to differentiate into mature pDC , namely influenza virus . Influenza virus hemagglutinin envelope glycoprotein ( HA ) binds to sialic acids on the cell surface to trigger clathrin-dependent endocytosis [32] . We pseudotyped HIV virions with influenza hemagglutinin glycoprotein envelope ( HA-HIV ) and first compared the functional response of purified pDC to HIV , influenza , and HA-HIV , in terms of magnitude and kinetics of TNF\\u03b1 ( TNF ) and IFN production . TNF is produced downstream of NF-\\u03baB signaling and IFN is produced downstream of IRF7 signaling . TNF and IFN were measured by intracellular cytokine staining ( ICS ) at 30 minutes , 2 hours , 6 hours , and 12 hours and in the culture medium by cytokine bead array ( CBA ) and ELISA , respectively , at 2 hours , 6 hours , and 12\\u201324 hours . As previously demonstrated [22] , the response of pDC to HIV was characterized by delayed IFN and TNF responses with IFN predominating at later time points . In comparison , both HA-HIV and influenza stimulated pDC to rapidly produce TNF within 30 minutes- 2 hours , an effect which plateaued by 12\\u201324 hours . Influenza also stimulated early IFN secretion , within 2 hours . HA-HIV induced IFN secretion , albeit at lower levels than Flu itself ( Fig 1A and 1B ) , possibly due to faster trafficking kinetics and early global cytokine shutdown ( as evidenced by earlier IFN and TNF shutdown compared to Flu in ICS Fig 1A ) . Strikingly , TNF was always produced antecedent to IFN , as evidenced by ICS staining , and as has been observed in murine pDC [33] ( Fig 1A ) . After overnight incubation of pDC , HIV stimulated minimal upregulation of CD86 and HLA-DR while HA-HIV and influenza stimulated strong upregulation of CD86 and HLA-DR expression , providing further evidence that HA-HIV and influenza activate NF-\\u03baB signaling/maturation pathways in pDC while HIV does not ( Fig 1C ) . Similar maturation and IFN effects were seen whether pDC were stimulated with X4 lab strain , MN or HIV backbone pNL43-\\u0394Env-vpr+-luc+ pseudotyped with X5 HIV envelopes ( JRFL , REJO , JOTO ) as compared to HIV backbone pNL43-\\u0394Env-vpr+-luc+ pseudotyped with hemagglutinin envelopes H1 and H5 ( S1A Fig ) . A functional measure of pDC maturation is to test whether cells become refractory to re-stimulation by TLR agonists , known as TLR tolerance . We had previously shown that HIV-activated pDC maintain an immature phenotype and are not refractory to re-stimulation to produce IFN . This effect was not due to activation of pDC that had failed to become activated during the previous overnight incubation [22] . We therefore compared the effects of HIV , influenza , HA-HIV , and CpGB , a potent pDC TLR9 maturation stimulus , to inhibit re-stimulation , as a marker of complete pDC maturation . As compared to HIV , HA-HIV inhibited re-stimulation of pDC similarly to influenza and CpGB , thus signifying that HIV pseudotyped with HA matured pDC fully ( Fig 1D ) . Overall , swapping HIV envelope with influenza envelope induced a mature pDC phenotype similar to that induced by influenza activation . Notably , MN HIV , a CXCR4 lab strain of HIV was used in these experiments . However , HIV ( pNL43-\\u0394Env-vpr+-luc+ ) pseudotyped with envelopes JRFL , REJO , or JOTO ( all R5-tropic ) all stimulate pDC to produce IFN , and not to mature , as expected since co-receptor usage is not essential for pDC sensing of HIV ( S1A and S1B Fig ) [12] . Because HA-HIV and influenza similarly stimulated pDC to mature , we sought to investigate whether HA-HIV traffics similarly to influenza in pDC . HIV virions ( pNL43-\\u0394Env-vpr+-luc+ ) pseudotyped with HA and packaging green florescent protein ( GFP ) and HIV virions ( pNL43-\\u0394Env-vpr+-luc+ ) pseudotyped with JRFL ( R5 envelope ) and packaging GFP were generated , using eGFP-Vpr plasmids , as previously described [22] . Influenza packaging GFP were also generated , as previously described [34 , 35] . We found that HA-HIV , similarly to influenza and unlike HIV itself , rapidly trafficked to lysosomes by 30 minutes as evidenced by co-localization with Lysotracker , a dye that traffics to these organelles ( Fig 2A and 2C ) . Both influenza and HA-HIV extensively co-localized with Lysotracker at 30 minutes and 2\\u20134 hours , whereas HIV was barely visible inside the cell at these early time points . After overnight incubation HIV was well visualized inside the cell , but did not co-localize well with lysotracker . Influenza and HA-HIV still seemed to co-localize with lysotracker even though the florescent signal was faded , likely due to lysosomal degradation of the virions ( S2A\\u2013S2D Fig ) . We confirmed that HIV traffics to early endosomal ( EEA1 ) compartments by 18 hours as previously shown [22] , whereas influenza and HA-HIV traffic significantly less to these compartments and the green signal is faded at 18 hours ( Fig 2B and 2D ) . Thus , the nature of the viral envelope seems crucial to determining trafficking of virions in pDC , and for the downstream signaling pathways activated in different intracellular compartments . Our results indicate that viral envelope protein dictates early intracellular trafficking of virions in pDC , suggesting that trafficking is directed by interaction of HIV envelope protein with its cognate receptor . In pDC , sensing of HIV involves CD4-mediated endocytosis [12] . CD4 is a type I integral membrane glycoprotein that can be internalized through clathrin-mediated endocytosis . The intracellular tail of CD4 displays motifs important for its internalization: a dileucine motif that allows interaction with the clathrin adaptor 2 ( AP-2 ) [36] and an adjacent serine , whose phosphorylation augments the affinity of the dileucine motif for AP-2 [36] , thereby regulating CD4 endocytosis . In cells of macrophage-monocyte lineage , CD4 is constitutively endocytosed at low levels through clathrin-coated pits to early and recycling endosomes [37] , as CD4 is serine-phosphorylated to some extent even in unstimulated cells [36] . We first examined whether CD4 itself and CD4-associated targeting motifs are responsible for the predominant localization of HIV in early endosomes . We used HEK 293 T ( HEK ) cells and HEK reporter cells as a model because HEK cells do not express CD4 under native conditions and therefore manipulation of CD4 trafficking and viral-CD4 interactions can be studied more clearly . Moreover , due to technical limitations , it was not possible to transfect or transduce primary pDCs or the Gen2 . 2 pDC cell line to undertake these studies . We engineered hybrid CD4 molecules , mutating its intracytoplasmic tail or swapping it with the intracellular domain of CD205 ( DEC205 ) and Lamp-1 ( Fig 3A ) . CD205 and Lamp-1 contain distinct lysosome-targeting motifs in their intracytoplasmic tail that induce constitutive targeting to late endosomes/lysosomes [38\\u201340] . DEC-205 expresses the coated-pit internalization sequence ( FSSVRY ) and lysosome-targeting motif ( EDE ) , whereas Lamp-1 expresses the lysosomal targeting motif ( YQTI ) . Several mutants were tested , as visualized in Fig 3A: ( 1 ) . CD4-WT ( wild type CD4 ) , ( 2 ) . CD4-STOP ( lacking the cytoplasmic domain ) , ( 3 ) . CD4-DEC ( replacing the CD4 cytoplasmic domain with the DEC-205 cytoplasmic domain to shuttle CD4 to the lysosomes ) , and ( 4 ) . CD4-LAMP ( replacing the CD4 cytoplasmic domain with the Lamp-1 cytoplasmic domain to shuttle CD4 to the lysosomes ) . The CD4 mutant sequences were introduced into lentiviral vectors for stable transduction of HEK cells for microscopy , and HEK-Blue hTLR7-expressing cells to measure NF-\\u03baB activation by HIV . HEK-Blue hTLR7 co-express human TLR7 and an NF-\\u03baB inducible secreted embryonic alkaline phosphatase ( SEAP ) reporter gene . CD4 expression was maintained in the presence of puromycin , and CD4 expression across cell lines was uniform at 65\\u201375% . For these experiments we used JRFL HIV packaging GFP- HIV Gag-iGFP ( GFP HIV ) to track HIV intracellular trafficking . Following incubation with HIV for 2 to 4 hours , CD4-expressing cells bound and endocytosed HIV efficiently , as shown in Fig 3C . This time point was chosen because HIV was not visualized well before 2 hours , and the fluorescent signal was faded after overnight incubation . The main path of viral entry in CD4-expressing HEK cells is CD4-mediated endocytosis as CD4 blockade completely abrogated HIV uptake ( Fig 3B ) . HIV co-localized extensively with CD4 , whatever CD4 construct the cells expressed , in the range of 65% to 80% co-localization per cell , as measured by single cell Mander\\u2019s coefficient ( Fig 3D ) . HEK cells which are not expressing CD4 do not take up HIV as represented by the ( - ) condition in representative images ( Fig 3C ) . Although the intracellular distribution pattern appeared different between the different CD4 constructs , with CD4-WT and CD4-STOP appearing more cell-surface associated and CD4-LAMP and CD4-DEC appearing more internal compartment associated , the overall fluorescence intensities of cell-associated HIV were comparable . To better characterize the intracellular localization of GFP- HIV in HEK cells expressing CD4 hybrids , cells were exposed to GFP-HIV for 2\\u20134 hours , then fixed and stained for the early endosomal marker EEA1 , the recycling endosomal marker transferrin receptor ( TfR ) , or the lysosomal marker LAMP1 . As an additional lysosomal marker , after 2\\u20134 hours of GFP-HIV incubation , lysotracker was added to culture media , and live imaging was performed . Internalized HIV trafficked predominantly to early EEA1+ and TfR+ endosomes in CD4-WT , similar to trafficking of HIV in pDC ( Fig 4 ) . However , HIV trafficked mainly to LAMP-1+ and lysotracker+ lysosomes in CD4-DEC and CD4-LAMP ( Fig 5 ) . Surprisingly HIV trafficked to EEA1+ compartments in CD4-STOP instead of remaining on the cell surface , but did not traffic predominantly to TfR ( Fig 4A\\u20134D ) . As endosomes containing EEA1 ultimately direct trafficking to the degradative machinery of the cell whereas endosomes containing TfR do not [41] , it is likely that some of the mutant CD4-STOP proteins are targeted for degradation to some extent . Overall , these data are consistent with the expected trafficking patterns of each CD4 construct , and indicate that replacing the intracellular tail of CD4 with those of CD205 or LAMP-1 targets CD4 and HIV to the late endosomes/ lysosomes , demonstrating that targeting motifs in CD4 drive the intracellular localization of HIV via CD4-mediated endocytosis . We next examined the functional consequences of CD4 altered trafficking . The lentiviral constructs ( Fig 3A ) were transduced into HEK-Blue cells to measure NF-\\u03baB activation . Notably this cell type is not optimized to produce IFN , therefore only the NF-\\u03baB signaling arm was tested in this system . R848 , a TLR7/8 agonist , was used as a positive control and strong NF-\\u03baB activator that rapidly accumulates in late endosomes [42] . The TLR9 agonist CpGB was used as a negative control in these TLR7-expressing cells and did not induce NF- \\u03baB activation in any condition ( Fig 6 ) . Whereas HIV did not induce NF- \\u03baB activation in CD4-WT and CD4-STOP expressing cells , it induced NF-\\u03baB activation in CD4-DEC and CD4-LAMP expressing cells . These results support the spatiotemporal model of cell signaling , whereby partition into early or late endosomes regulates IFN vs NF-\\u03baB signaling [25] . HIV traffics to early endosomes in pDC and in CD4-WT expressing cells , and induces weak NF-\\u03baB signaling , whereas retargeting CD4 to lysosomes allows for NF-\\u03baB activation by HIV . We also examined more precisely why native CD4 delivers HIV into early endosomes . Endocytosis and intracellular CD4 trafficking is dependent on a dileucine motif in its intracellular domain , and is regulated by phosphorylation of an adjacent Serine ( Ser408 ) . Phosphorylation of Ser408 , as occurs with phorbol ester ( PMA ) , dramatically enhances CD4 delivery from the cell surface to the lysosomes [43] . In contrast to PMA stimulation , HIV activation of human pDCs does not cause marked CD4 internalization at early timepoints . Whether pDC are unstimulated or stimulated with HIV , CD4 internalization is grossly unchanged , whereas PMA stimulation , in the absence or presence of HIV , causes internalization at early timepoints ( Fig 7 ) . To investigate whether CD4 phosphorylation on Ser408 targets CD4 and HIV to late endosomes/ lysosomes , we generated two mutations of the serine residue: one to alanine to abrogate phosphorylation ( CD4-SA ) , and one to the phospho-mimic Glutamic Acid ( CD4-SE ) , and produced HEK cells stably expressing these CD4 mutants . Intracellular localization of HIV and CD4 was monitored by microscopy . As shown in Fig 8 , HIV co-localized with CD4 and appeared less membrane associated in CD4-SE cells , whereas it remained predominantly on the surface of CD4-SA cells . In CD4-SA cells , intracellular HIV accumulated in EEA1+ and TfR+ compartments ( Fig 9 ) , similarly to WT-CD4 cells ( Fig 4 ) , whereas HIV accumulated in Lamp-1+ and lysotracker+ compartments in CD4-SE cells ( Fig 10 ) , similarly to CD4-DEC cells and CD4-Lamp cells ( Fig 5 ) . To test the functional consequences , we transduced HEK-Blue cells with CD4-SA and CD4-SE , as compared to CD4-WT , to measure NF-\\u03baB activation , as above . Whereas HIV did not induce NF-\\u03baB activation in CD4-WT and CD4-SA expressing cells , it induced NF-\\u03baB activation in CD4-SE expressing cells ( Fig 11 ) . Thus , CD4 phosphorylation on Ser408 appears to target CD4 and HIV to late endosomes/ lysosomes , whereas it is routed to early endosomes in its absence ( CD4-SA and CD4-WT ) . Poor specificity of available anti-phospho-Ser408 CD4 antibodies precluded the possibility of studying the phosphorylation state of Ser408 CD4 in primary pDCs after HIV activation . While our data strongly suggest that HIV trafficking and subsequent immune signaling in pDC is driven by HIV envelope/CD4 interactions , we tested an alternative hypothesis of pDC signaling , that the strength of TLR signaling in the early sorting endosomes determines the trafficking and subsequent signaling of TLR agonists . Previous work showed that TLR signaling in other cell types accelerates endosomal maturation through TLR-induced p38 mitogen-activated protein kinase ( p38 ) signaling , as the absence of a TLR/MyD88 signal diminishes phagosome maturation [44] . TLR activation also activates lysosomal function in myeloid DC , which could further influence TLR signaling pathways [45 , 46] . According to this model , strong TLR triggering in the early endosome would accelerate endosomal maturation , and endocytosed viruses or oligonucleotides would rapidly reach late endosomes for NF-\\u03baB signaling [25 , 33] . Using murine pDC differentiated from TLR7-/- and TLR9-/- bone marrow , we tested this hypothesis . As HIV cannot be used in murine immune systems , we tested TLR9 agonists , focusing on CpGB . CpGB has been shown to rapidly traffic to late endosomes/ lysosomes in mouse and human pDC [25 , 26] , and it induces strong NF-\\u03baB signaling and pDC maturation [22] . According to this model where TLR signaling induces endosomal maturation , CpGB would traffic rapidly to lysosomes in wild type ( WT ) or TLR7-/- pDC , but would fail to traffic to lysosomes in murine TLR9-/- pDC . Because MyD88 knockout DCs lacking TLR signaling have been described as having an altered constitutive rate of endosomal maturation [44 , 47] , we also constructed a control oligonucleotide lacking stimulatory CpG motifs ( due to inversion of the CpG motif into a GpC dinucleotide ) which lacks TLR triggering activity . Wild type , TLR7-/- , and TLR9-/- murine pDC were derived from bone marrow with Flt3 ligand , as previously described [48] and purified >85% ( S3A Fig ) . pDCs were incubated with FAM-labeled CpGB or nonactivating FAM-labeled GpC control and were imaged by flow cytometry and live microscopy . We first confirmed the specificity of the various pDC TLR knock out cells by evaluating the expression of the maturation molecule CD86 after overnight incubation of wild type , TLR9-/- , and TLR7-/- pDC with TLR7 agonist R848 and TLR9 agonist FAM-CpGB . As expected , R848 matured WT and TLR9-/- pDC but did not mature TLR7-/- pDC , whereas FAM-CpGB matured WT and TLR7-/- pDC but did not mature TLR9-/- pDC . pDC stimulated with FAM-GpC did not mature WT pDC due to the lack of CpG immunostimulatory motifs ( S3B Fig ) . We then monitored trafficking of FAM-CpGB to lysosomes in WT , TLR7-/- and TLR9-/- pDC by microscopy , by measuring co-localization with Lysotracker Red . Across a z stack spanning the midplane of the cell , we observed that both FAM-CpGB in WT , TLR7-/- and TLR9-/- pDC ( S4A and S4C Fig ) , and FAM-GpC in WT pDC ( S4B and S4D Fig ) , rapidly and extensively trafficked to lysosomes within 15-20min . Thus , TLR activation did not affect intracellular trafficking of TLR agonists in murine pDC , suggesting that an alternative model also exists for human pDC . Altogether , these data demonstrate that divergent HIV-1 sensing by pDC is mediated by CD4-HIV envelope interactions . \\n Although many receptors and signaling pathways have been shown to modulate TLR signaling pathways in pDC [49] , it is likely that the functional outcome of TLR signaling is determined upstream and early on . The spatiotemporal model of TLR signaling [25] was proposed to account for the functional flexibility of pDC in response to TLR signaling , and argues that the surface or intracellular localization of TLR signaling initiation determines which downstream signaling pathway is triggered , with differing functional outcomes [50] . This is because each compartment is associated with different adaptors and signaling platforms , specialized in inflammatory cytokine secretion and NF-\\u03baB activation , or signaling through IRF7 and IFN production [25 , 33 , 50] . As HIV steadily traffics to early endosomes in pDC , a compartment associated with IFN signaling [25] , our goal was to understand how early HIV trafficking is regulated in pDC and how it affects pDC functional response . We studied whether HIV trafficking in pDC involves envelope-receptor interaction and targeting signals in endocytic receptors . A hybrid virus was constructed , where HIV envelope was replaced by influenza hemagglutinin envelope , while maintaining all other HIV structural components unchanged ( HA-HIV ) . In contrast to HIV , influenza is rapidly endocytosed by pDC and triggers a strong NF- \\u03baB activation , secretion of inflammatory cytokines , and maturation of pDC [22 , 51] . Strikingly , HA-HIV was rapidly routed to late endosomes/ lysosomes in pDC , contrary to HIV with its native envelope . Furthermore , it induced early secretion of inflammatory cytokines and strong pDC maturation , in a manner and kinetics similar to influenza virus . This demonstrates that virus envelope directly determines HIV trafficking and pDC phenotype . Despite different structural components and nucleic acids , HA-HIV and influenza induced a similar functional response in pDC , which strengthens the importance of viral envelope in determining pDC phenotype . This latter extended to the unresponsiveness of pDC to further stimulation , whereas HIV stimulated pDC could be re-stimulated to produce IFN . HA-HIV triggers IFN secretion , although at lower levels than Flu itself . This may be due to kinetics differences in trafficking to late endosomes and activation of negative signaling pathways or exhaustion . Trafficking to late endosomes , NF-\\u03baB activation and pDC maturation correlates with a state of refractoriness , likely established early during stimulation , and already evidenced by a global cytokine shutdown after a few hours . As shown in Fig 1A , this shutdown occurs earlier for HA-HIV than for Flu , at the time when IFN secretion is starting to be amplified . It is likely HA-HIV traffics significantly faster to late endosomes and triggers early cytokine shutdown . Another example of HIV pseudotyping is VSV-G pseudotyped HIV , where VSV-G from vesicular stomatitis virus is used to allow HIV uptake and infection of many cellular subtypes . The putative receptor for VSV-G has been recently suggested to be LDL receptor [52] , which traffics mainly to recycling endosomes but rarely to lysosomes . In accordance with this trafficking pattern and localization in early endosomes , VSV-G-pseudotyped HIV behaves mostly like HIV with a native envelope to trigger high levels of IFN but little pDC maturation [12] . In addition to demonstrating that viral envelope determines HIV localization and pDC phenotype , HA-pseudotyped HIV may also provide a tool to study HIV antigen presentation and vaccine design [53] , as it enhances expression of MHC and co-stimulatory molecules on pDC , and influenza itself triggers a developmental program suited for antigen presentation in pDC [21 , 54] . If the nature of the viral envelope dictates HIV trafficking in pDC , it may be due to intracellular targeting motifs present in the viral receptor ( s ) [38] . HIV is mainly taken up through CD4 in pDC , and we tested whether altering the intracytoplasmic domain of CD4 would affect HIV trafficking and TLR signaling . Indeed , we observed that swapping CD4 intracytoplasmic domain for DEC205 or Lamp1 intracytoplasmic domain dramatically re-routed HIV into late endosomes/ lysosomes in CD4 expressing cells , whereas intracellular HIV was localized predominantly in early endosomes in cells expressing native CD4 . DEC205 and Lamp1 contain lysosomal targeting motifs which are likely responsible for constitutive CD4 and HIV targeting to late compartments . In addition , redistribution of HIV to late endosomes was accompanied by activation of NF-\\u03baB , not observed when HIV accumulates in early endosomes , again consistent with the spatiotemporal model of TLR signaling . CD4 contains a dileucine motif in its intracytoplasmic domain , which is essential for CD4 endocytosis [36] . In addition , two adjacent Serines , Ser408 and Ser415 , can be phosphorylated and impact CD4 endocytosis and distribution . Completely deleting these motifs by removing the whole CD4 intracytoplasmic domain indeed almost completely abrogated CD4-mediated HIV endocytosis . However , phosphorylation of Ser408 not only enhances CD4 endocytosis [36] , but also redirects CD4 to lysosomal compartments [43] . Ser408 phosphorylation enhances its association with clathrin Adaptor protein AP-1 and AP-2 [36] . In our experiments , mutating Ser408 to Glutamic acid ( CD4-SE ) to mimic Ser408 phosphorylation , induced a complete redistribution of CD4 and HIV into lysosomes ( Fig 10 ) suggesting that in pDC , HIV traffics by default to recycling endosomes due to the CD4 dileucine motif , in the absence of Ser408 phosphorylation . These results are supported by our CD4 internalization studies where we found that HIV-activation of pDC does not seem to alter CD4 internalization , as compared to PMA-activation . Similarly , in HIV infected cells , Nef triggers endocytosis and degradation of CD4 through a dileucine based motif [55 , 56] and CD4 endocytosis and targeting to lysosomes are encoded in different regions of the Nef protein [56 , 57] . The relatively stable localization of HIV in early endosomes , observed for as long as 18 to 24h , remains unexplained . Furthermore , we observed strong co-localization of CD4 with HIV throughout the course of the study . Although the interaction between CD4 and HIV envelope may be stable enough to maintain CD4-HIV co-localization for a prolonged period of time , other explanations are possible . A recent study described in detail how endocytosis of HIV is coupled to dynamin-dependent endocytosis and partial fusion with plasma and endosomal membrane [58] , which may tether HIV envelope and CD4 to the endosomal membrane in the absence of fusion . Furthermore , pDC possess specialized large perinuclear intracellular stores of MHC-I molecules , with characteristics of recycling endosomes in immature pDC , that can be used as sites for rapid MHC-I loading and peptide presentation [21] . These intracellular stores may represent the stable compartment in which HIV accumulates in pDC , and prolonged localization in these early recycling endosomes may ultimately have important consequences for HIV antigen presentation . pDC may harbor HIV in these structures until activated by a maturation stimulus . Indeed , pDC are capable of HIV antigen cross-presentation [17] , and cross-presentation is strongly enhanced by maturation-inducing stimuli [59 , 60] . Upon influenza activation , stored MHC-I molecules are translocated to the cell surface for efficient cross-presentation by pDC [21] , indicating that the process of maturation drives antigen presentation . On the other hand , the non-acidic environment and limited access to MHC-II compartments may prevent efficient MHC-II peptide generation and association with MHC-II molecules . In addition , MHC-II clustering and antigen presentation by pDC is dependent on NF-\\u03baB [51 , 61] , which HIV weakly induces due to localization in early endosomes . The lack of pDC maturation induced by HIV might prevent effective cross-presentation and MHC-II restricted presentation , due to localization in early endosomes and weak NF-\\u03baB activation . Thus , the particular compartmentalization of HIV can affect HIV antigen presentation . As shown here , HA-pseudotyped HIV , which traffics to late endosomes and activates NF- \\u03baB , may serve as a tool to enhance cross-presentation of HIV antigens by pDC . We also tested whether TLR signaling itself alters maturation of endocytic compartments in pDC as was previously demonstrated in the case of murine macrophages , where TLR/ MyD88 signaling induced marked phagosome maturation , possibly through p38 MAP kinase activation [44] . We tested this model in murine pDC , and compared trafficking of CpGB in WT , TLR7-/- or TLR9-/- pDC . We did not observe any difference in trafficking of CpGB in these conditions , and internalized CpGB was rapidly found within late endosomes/ lysosomes whether TLR signaling had occurred or not . As MyD88-/- cells have altered endosomal maturation [47] , we also used a non-stimulatory type B oligonucleotide ( lacking immunostimulatory CpG motifs ) in WT pDC . However , its trafficking was identical to the stimulatory CpGB oligonucleotide . Furthermore , as described previously [44 , 62] , the lack of TLR signaling , as with nonstimulatory GpC , decreased the speed of CpG endocytosis ( S4E Fig ) , however all oligonucleotides localized identically . These data argue against the above model , whereby rate of endocytosis and/ or agonist potency drive its intracellular localization , and establish trafficking as independent of TLR signaling in pDC [47 , 63] . Using purified human pDCs and viruses , we demonstrate that HIV trafficking in pDC at early timepoints is determined by the initial envelope-receptor ( CD4 ) interaction , and is regulated by receptor targeting motifs . Whether this is still valid in the case of cell-associated virus , this remains to be determined . Engineering of viral envelope for increased pDC maturation and antigen presentation ( e . g . HA-HIV ) or for increased IFN secretion [64] may prove useful for vaccine design and modulation of chronic immune activation in HIV disease . \\n PBMCs were separated on Ficoll-Hypaque ( Amersham Biosciences ) from buffy coats ( New York Blood Center ) . pDC were purified by BDCA-4 magnetic bead separation ( Miltenyi Biotec ) as described previously [12] , with a purity ranging from 80 to 95% . Cells were cultured in RPMI 1640 Glutamax ( Invitrogen ) with 5% PHS ( Innovative Research , MI ) , gentamycin , and HEPES . HIV-1MN ( X4-tropic ) were produced at the AIDS Vaccine Program , National Cancer Institute as previously described [12 , 23 , 65] . Plasmids encoding JRFL HIV-1 envelope , JOTO HIV-1 envelope , REJO HIV-1 envelope , pNL43-\\u0394Env-vpr+-luc+ and pCAGGS ( human airway trypsin-like protease to cleave HA0 to HA1 and HA2 ) were provided by Carol Weiss ( FDA , Silver Spring , MD ) , plasmids encoding CMV/R Influenza H1 ( A/PR8/8/34 ) ( VRC 7702 ) , CMV/R influenza A/PR/8/1934 NA ( VRC 9776 ) , CMV/8R A/Thailand/1Kan-1/2004 H5 ( VRC 7705 ) , and CMV/8R A/Thailand/1Kan-1/2004 NA ( VRC 7708 ) , and were provided by Gary Nabel and Chih-Jen Wei ( Vaccine Research Center , NIH , Bethesda , MD ) . Plasmids encoding vpr-gfp and MN HIV pNL4-3 \\u0394vpr were obtained from David Ott and Jeffrey Lifson ( AIDS Vaccine Program , Frederick , MD ) . Plasmids encoding HIV Gag-iGFP JRFL were obtained from Benjamin Chen ( Mount Sinai School of Medicine , NY , NY ) . The influenza hemagglutinin viral pseudotypes were generated by calcium phosphate co-transfection of 3 . 0x106 HEK cells in a 10cm2 dish with 10ug HIV core ( pNL43-\\u0394Env-vpr+-luc+ ) , 400ng hemagglutinin envelope ( VRC 7702 , VRC 9776 , VRC 7705 , or VRC 7708 ) , 100ng PR8 NA ( VRC9776 ) , and 100ng HAT pCAGGS , with media change after 6 hours , and viral harvest at 48 hours . HIV pseudotypes were generated by calcium phosphate co-transfection of 3 . 0x106 HEK cells in a 10cm2 dish with 10ug HIV core ( pNL43-\\u0394Env-vpr+-luc+ ) and 6ug Env in pCDNA3 . 1 ( e . g . REJO , JOTO , JRFL ) . HIV Gag-iGFP is a full-length molecular clone of HIV derived from pNL4-3 that packages GFP inserted into the Gag protein between the MA and CA domains of Gag , with JRFL Env cloned into the place of the NL4-3 Env . To generate Gag-iGFP virions for CD4-expressing HEK experiments , 20ug of plasmid was transfected using the calcium phosphate method , with media change at 6 hours and transfection for 48 hours . For all viruses , transfection supernatants were filtered through a 0 . 45uM membrane , pelleted through a 20% sucrose cushion at 25 , 000g for 2hrs at 4\\u00b0C . Pelleted viruses were resuspended in PBS , aliquoted , and stored at -80\\u00b0C until use . HIV virions were quantified using p24 ELISA ( AIDS Vaccine Program ) and HA-HIV viruses were quantified using turkey hemagglutination inhibition assay for hemagglutination unit ( HAU ) as well as p24 ELISA . PR8 influenza was provided by David Levy ( NYU ) and influenza packaging GFP was provided by Jesse Bloom ( Fred Hutchinson Cancer Research Center , Seattle , WA ) as previously described [34 , 35] and were quantified by using turkey hemagglutination inhibition assay for hemagglutination unit . Purified pDC were stimulated at 50 , 000 cells/100uL media at 37\\u00b0 C with 5% CO2 with: MN HIV 300ng ( AIDS Vaccine Program , National Cancer Institute ) , HIV pseudotyped HIV envelopes 300ng , type B CPG oligodeoxyribonucleotide ( CpGB ) 2 ug 5\\u2019 T*C*G*T*C*G*T*T*T*T*G*T*C*G*T*T*T*T*G *T*C*G*T*T*-3\\u2019 where asterisks indicate a phosphorothioate bond ( IDT ) , Resiquimod ( R848 ) 10\\u03bcM ( 3M Corporation , St . Paul , MN ) , influenza virus PR8 HAU 10 heat inactivated at 56\\u00b0C for 30 minutes in a water bath , or HA-HIV HAU 10 . For intracellular staining , brefeldin A was added after 30 minutes , 2 hours , 6 hours , or 12 hours; at 24 hours cells were washed , fixed , and stained with PE-conjugated CD123 PE ( BD Biosciences ) , APC-conjugated TNF\\u03b1 ( eBioscience ) , and FITC-conjugated IFN-\\u03b1 ( BD Biosciences ) in 0 . 05% saponin , and analyzed by FACS . Following incubation of pDCs ( 50 , 000 cells/100uL ) with HIV MN and HIV JRFL , influenza , and HA-HIV , culture supernatants were tested for IFN and TNF by IFN\\u03b1 by ELISA ( PBL Interferon Source ) and human inflammatory kit CBA ( Abcam ) , respectively , following manufacturer instructions . For surface staining after overnight incubation , cells were stained with CD123 PE , CD86 APC , and HLA-DR PerCP ( BD Pharmigen ) , washed , fixed with 4%PFA , and analyzed by FACS . For IFN\\u03b1 restimulation experiment , pDCs were incubated overnight with influenza , HIV , HA-HIV , or CpGB , washed , and then incubated again with the same agonists . Culture supernatants were tested after the first and second overnight incubation for IFN\\u03b1 by ELISA ( PBL Interferon Source ) . CD4 expression plasmid ( pcDNAI ) was provided by Nathaniel Landau ( NYU , NY , NY ) . It was used as a template to mutate CD4 by overlapping PCR . Mutated CD4 were then inserted into pLenti vectors , and lentiviruses were produced using Mirus TransIT co-transfection of 3 . 0x106 HEK cells in a 10cm2 dish for 72 hours with 1 . 45 ug VSVg , 2 . 05 ug RSR-Rev , 2 . 9 ug pMDL plasmids ( provided by Dr Landau ) and 8 . 7 ug pLenti-CD4 constructs ( WT , STOP , DEC , LAMP , SE , SA ) . CD4-STOP was encoded of the first 425 amino-acids , including the transmembrane domain of CD4 . CD4-DEC contained the full extracellular domain of CD4 ( 397 amino acid in the immature form ) and the transmembrane and intracytoplasmic domain of DEC205 ( 56 amino acids ) . CD4-Lamp encoded the full extracellular and transmembrane domain of CD4 , and 12 amino acids of Lamp intracytoplasmic domain . CD4-SE and CD4-SA was identical to CD4 WT , except for a mutation of Serine 408 ( mature form ) for glutamate and alanine , respectively . CD4-expressing HEK cell lines were generated by infecting 3 . 0x106 HEK cells in a 10cm2 dish with lentivirus transfection supernatants and CD4 expression was maintained in the presence of puromycin . C57BL/6 wild type , TLR7-/- and TLR9-/- double-knockout mouse femurs were provided by B . Pulendran , Emory University . Mice were maintained in specific-pathogen-free conditions at the Emory Vaccine Center vivarium in accordance with all animal protocols reviewed and approved by the Institute Animal Care and Use Committee of Emory University . Bone marrow cells were isolated by flushing femurs with PBS supplemented with 2% heat inactivated FBS . BM cells were resuspended in Tris-ammonium chloride at room temperature for 1 minute to lyse RBC , washed , then cultured in RPMI 1640 Glutamax ( Invitrogen ) with 10%FBS , 1nM sodium pyruvate , 10mM HEPES buffer , 100 units/mL penicillin , 100ug/mL streptomycin , 2mM L-glutamine , 1% MEM nonessential amino acids . BM cells were cultured for 8 days at 1 \\u00d7 106 cells/ml in 24-well plates in culture medium supplemented with 200 ng/ml recombinant murine Flt-3 ligand ( Peprotech ) as previously described [48] . After 8 days pDCs were purified from BM cells using the mouse plasmacytoid dendritic cell isolation kit II ( Miltenyi ) with a purity ranging from 85 to 95% and cells were phenotyped by CD11c PerCP and PDCA1 APC ( BD Pharmigen ) . Purified murine pDC ( WT , TLR7-/- , and TLR9-/- ) were stimulated overnight at 50 , 000 cells/100uL media at 37\\u00b0C with 5% CO2 with R848 10\\u03bcM ( 3M ) , 5\\u2019FAM-CpGB 5\\u2019 TCGTCGTTTTGTCGTTTTGTCGTT-3\\u2019 2 ug ( IDT ) , or 5\\u2019FAM-GpC 5\\u2019-TGCTGCTTTTGTGCTTTTGTGCTT-3' 2ug ( IDT ) , both with phosphodiester backbones , and were stained with CD11c PerCP , PDCA1 APC , and CD86 PE ( BD Pharmigen ) . Purified murine pDC ( 200 , 000 cells/200uL ) were stimulated in 0 . 01% Poly L-lysine ( Sigma ) coated 8 chamber polystyrene vessel tissue culture treated glass slides ( CultureSlides BD Falcon ) in 10% FBS culture media ( 200uL ) with FAM-CpGB 2ug or FAM-GpC 2ug and stained with 1uM lysotracker ( Invitrogen ) . Cells were imaged by live microscopy for one hour using the Advanced Precision PersonalDV Imaging system , with temperature ( 37\\u00b0C ) and C02 ( 5% ) humidity control , at a size of 512 X 512 pixels and a bit depth of 16 using a 60X , 1 . 4 N . A . oil objective lens . Images were deconvoluted using the DeltaVision deconvolution system and analyzed using ImageJ . Purified human pDC ( 50 , 000 cells/50uL ) were stimulated in Fibronectin ( Corning ) coated Ibidi imaging chambers \\u03bc-Slide VI0 . 4 ( Ibidi , Madison , WI ) in 5% PHS culture media ( 200uL ) with GFP PR8 influenza 10 HAU , GFP HA-HIV ( 10HAU/500ng p24 ) or GFP JRFL pseudotyped HIV 500ng p24 and stained with 1uM lysotracker ( Invitrogen ) . Live imaging was carried out after 30 minutes , 2\\u20134 hours , or after overnight stimulation of cells , using a Yokogawa CSU-X1 spinning disk mounted on a Zeiss AxioObserver Z1 and controlled by MetaMorph under conditions that were reproduced across all experiments . 488nm and 561nm laser lines were generated by a Prairie Technologies Aurora solid state laser and fluorescence and brightfield ( phase ) images were captured with a Hamamatsu EM-CCD C9100 digital camera set at an EM gain of 11 MHz , a size of 512 X 512 pixels and a bit depth of 16 using a 63X , 1 . 4 N . A . oil objective lens . Temperature ( 37\\u00b0C ) , C02 ( 5% ) , and humidity were controlled using a Tokai Hit incubator . CD4 expressing HEK cells were incubated ( 200 , 000 cells/200 \\u03bcl ) with purified NA/LE mouse anti-human CD4 ( 10 \\u03bcg/ml; BD Pharmigen ) or isotype control purified NA/LE mouse IgG1\\u03ba for 30 minutes . HIV Gag-iGFP was then added , and cells were placed back in culture for 18 hours . Cells were washed , fixed with 4% PFA , and analyzed by FACS . Purified human pDC ( 100 , 000 ) were incubated for 18 hours with GFP PR8 influenza 10 HAU , GFP HA-HIV ( 10HAU/500ng p24 ) or GFP JRFL pseudotyped HIV ( HIV Gag-iGFP JRFL ) 500ng p24 in 8 chamber polystyrene vessel tissue culture treated glass slides ( CultureSlides BD Falcon ) in 10% FBS ( Gibco ) culture media ( 200uL ) then cells were washed x2 , fixed with 4% paraformaldehyde , permeabilized with 0 . 1% Triton X-100 , blocked with 0 . 5% BSA in PBS , stained with mouse anti-EEA1 ( 0 . 5 ug; BD biosciences ) at 4\\u00b0C overnight , washed x 2 then stained with donkey anti-mouse TRITC ( Jackson Immunoresearch ) for one hour at RT , then washed , dried , and mounted in DAPI Anti-fade GOLD ( Vector labs ) . Alternatively pDCs were incubated with media , phorbol 12-myristate 13-acetate 100ng/mL ( PMA ) , HIV , or PMA and HIV for 4 hours then cells were washed x2 , fixed with 4% paraformaldehyde , permeabilized with 0 . 1% Triton X-100 , blocked with 0 . 5% BSA in PBS , and then stained with CD4 PE ( BD Pharmigen ) for one hour , washed x2 , dried , and mounted in DAPI Anti-fade GOLD ( Vector labs ) . HEK cells transduced with CD4 mutants ( 25 , 000 ) were incubated overnight in 8 chamber polystyrene vessel tissue culture treated glass slides ( CultureSlides BD Falcon ) in 10% FBS ( Gibco ) culture media ( 200uL ) . iGFP HIV 500ng was added to culture media for 2\\u20134 hours then cells were washed x2 , fixed with 4% paraformaldehyde , permeabilized with 0 . 1% Triton X-100 , blocked with 0 . 5% BSA in PBS , and then stained with CD4 PE ( BD Pharmigen ) for one hour . Alternatively , cells were stained with mouse anti-EEA1 ( 0 . 5 ug; BD biosciences ) , mouse anti-transferrin receptor ( 0 . 5ug; Invitrogen ) , mouse anti LAMP-1 H4A3 ( 0 . 5ug; Developmental Studies Hybridoma Bank University of Iowa ) at 4\\u00b0C overnight , washed x 2 then stained with donkey anti-mouse TRITC ( Jackson Immunoresearch ) for one hour at RT , then washed , dried , and mounted in DAPI Anti-fade GOLD ( Vector labs ) . Fixed cells were imaged using a Zeiss LSM 880 confocal microscope configured to generate laser lines at 488nm and 561nm , as well as transmitted light . Images were scanned using a 100X , 1 . 46 N . A . oil objective lens at a size of 1024 X 1024 pixels and a bit depth of 12 . Images were captured using sequential ( multitrack ) acquisition to avoid bleedthrough of signal between channels . Colocalization of fluorescently labeled ligands ( green ) with endosomal markers or CD4 receptor ( red ) were analyzed quantitatively using JACoP plugin on ImageJ software or MetaMorph colocalization analysis . Image pair channel files of single image mid-planes were opened in as separate 16-bit ( grey scale ) image files . Individual cell regions were identified using the corresponding brightfield image , images were thresholded , and colocalization analysis was performed using the MetaMorph \\u201cmeasure colocalization\\u201d application , which measures the localization correlation between corresponding pixels in two paired images and provides the Manders correlation [66] . 50\\u2013100 cells were analyzed to generate the individual coefficients and data was plotted using GraphPad Prism software . CD4-expressing HEK-Blue hTLR cell lines were generated by infecting 3 . 0x106 HEK-Blue hTLR cells in a 10cm2 dish with CD4 lentivirus transfection supernatants . Stable CD4 expression of cell lines ( WT , STOP , DEC , LAMP , SA , and SE ) were cultured in the presence of puromycin and CD4 expression was verified by FACS and microscopy . 200 , 000 cells were placed in HEK-blue culture media overnight ( 200uL ) with CpGB , HIV , or R848 and NF-KB activation was measured using the HEK-blue detection system , with blue color depicting NF-\\u03baB activation , as measured by ELISA according to manufacturer specification ( Invivogen ) . Statistical significance was determined by the unpaired Student\\u2019s t test and analysis of variance .\",\n          \"Mesenchymal ( lamellipodial ) migration is heterogeneous , although whether this reflects progressive variability or discrete , 'switchable' migration modalities , remains unclear . We present an analytical toolbox , based on quantitative single-cell imaging data , to interrogate this heterogeneity . Integrating supervised behavioral classification with multivariate analyses of cell motion , membrane dynamics , cell-matrix adhesion status and F-actin organization , this toolbox here enables the detection and characterization of two quantitatively distinct mesenchymal migration modes , termed 'Continuous' and 'Discontinuous' . Quantitative mode comparisons reveal differences in cell motion , spatiotemporal coordination of membrane protrusion/retraction , and how cells within each mode reorganize with changed cell speed . These modes thus represent distinctive migratory strategies . Additional analyses illuminate the macromolecular- and cellular-scale effects of molecular targeting ( fibronectin , talin , ROCK ) , including 'adaptive switching' between Continuous ( favored at high adhesion/full contraction ) and Discontinuous ( low adhesion/inhibited contraction ) modes . Overall , this analytical toolbox now facilitates the exploration of both spontaneous and adaptive heterogeneity in mesenchymal migration . \\n Cell migration is a profoundly heterogeneous phenomenon . Indeed , cells can adopt several substantially different migration modalities , including multicellular , amoeboid , and mesenchymal ( also termed lamellipodial or lamellipodial-driven ) migration , which can all be utilized by a broad range of cell types , as well as lobopodial migration , which has been observed specifically in fibroblasts ( Friedl and Wolf , 2010; Sahai , 2005; Petrie and Yamada , 2015; Petrie et al . , 2014; Welch , 2015; Friedl and Alexander , 2011 ) . These migration modes represent 'prespecified' cellular configurations ( i . e . cell states ) that are favored under particular conditions ( Friedl , 2004 ) . Switch-like conversion between these distinct modes is therefore part of the plastic , adaptive/compensatory response of cells to either environmental modulation ( Liu et al . , 2015; Starke et al . , 2014; Ruprecht et al . , 2015 ) or molecular targeting ( Sahai et al . , 2007; Sanz-Moreno et al . , 2008; Somlyo et al . , 2003; Wolf , 2003 ) . At a finer scale , heterogeneity is also evident within these migration modes , arising either stochastically or as an adaptive response to changing cues ( Geiger et al . , 2009; L\\u00e4mmermann and Sixt , 2009; Lock et al . , 2014; Winograd-Katz et al . , 2009 ) . Yet , partly due to a lack of adequate quantification , it remains unclear to what extent variation within modes occurs either progressively along a continuum or in a switch-like manner between as yet undefined intra-modal subpopulations . Specifically , in the case of amoeboid migration , three discrete sub-modalities have been observed , and these can co-exist under individual conditions ( Welch , 2015; L\\u00e4mmermann and Sixt , 2009; Yoshida , 2006 ) . By contrast , potentially distinct styles of mesenchymal ( lamellipodial ) migration , including keratocyte-like ( Barnhart et al . , 2015; Keren et al . , 2008 ) and fibroblast-like ( Abercrombie et al . , 1977; Theisen et al . , 2012 ) migration , have been described as arising largely in separate cell types or conditions . Therefore , despite some early suggestions ( Lewis et al . , 1982 ) , it has remained uncertain to what extent divergent sub-modalities of mesenchymal migration spontaneously emerge in parallel\\u00a0within uniform cell populations and conditions , and whether these modes are truly quantitatively distinct , or instead represent extremes in a broad phenotypic continuum . These questions are important because each discrete migration mode , with its unique internal logic and dependencies , may respond differently to altered regulatory cues \\u2013 thereby defining a multifaceted adaptive response . Such responses are exemplified by the seminal observation that amoeboid motility provides a compensatory escape route for tumor cells when mesenchymal migration is blocked via inhibition of proteolytic mechanisms or by confinement ( Wolf , 2003 ) . Identifying and characterizing such distinct modes within individual cell types and/or conditions may therefore be key to understanding heterogeneity in response to experimental or even clinical interventions . Unlike amoeboid migration , which requires little or no cell-matrix adhesion ( L\\u00e4mmermann et al . , 2008 ) , mesenchymal migration demands cell adhesion to the extracellular matrix ( ECM ) for effective force application ( Friedl and Wolf , 2010 ) . This adhesion is achieved through the interaction of integrin-mediated cell-matrix adhesion complexes ( CMACs ) with ECM ligands , while applied forces are generated by the F-actin cytoskeleton and actomyosin system ( Geiger et al . , 2009; Humphries et al . , 2015 ) . This arrangement defines the ECM \\u2013 adhesion \\u2013 F-actin axis that directly mediates mesenchymal migration ( Parsons et al . , 2010; Schwartz , 2010 ) . Integrins are central to the ECM \\u2013 adhesion \\u2013 F-actin axis and are both anchored and activated by binding through their extracellular domains to ECM ligands , such as collagen , laminin , and fibronectin ( Lewis et al . , 1982 ) . The composition , topology , and density of ECM ligands therefore play key roles in mesenchymal migration ( Geiger et al . , 2009 ) . Through interaction with integrin tails , talin acts a critical intracellular regulator of integrin activation ( Calderwood et al . , 1999; Kiss et al . , 2015; Moser et al . , 2009; Tadokoro , 2003 ) , and is also one of several CMAC proteins linking integrins to the F-actin cytoskeleton ( Schwarz and Gardel , 2012 ) . This enables cellular force application from F-actin , through CMACs , to the ECM , thereby driving membrane protrusion/retraction and cell translocation ( Small and Resch , 2005 ) . Furthermore , CMACs function both up- and downstream of the small GTPases Rac1 and RhoA ( Raftopoulou and Hall , 2004 ) , which modulate both F-actin polymerization and actomyosin contractility , the latter through the Rho kinase ( ROCK ) -mediated regulation of non-muscle myosin II ( Ridley , 2003; Riento and Ridley , 2003 ) . In fact , CMACs function as hubs for bi-directional chemical and mechanical information transduction across the plasma membrane , providing command and control of mesenchymal migration ( Hynes , 2002; Lock et al . , 2008 ) . Given the pivotal roles of CMACs and F-actin , their assessment can efficiently provide a broad estimation of how the complex machinery underlying mesenchymal migration is organized ( Lock et al . , 2014; Gardel et al . , 2010; Gupton and Waterman-Storer , 2006; Kim and Wirtz , 2013 ) . Such an approach , generally based on quantitative imaging , is especially effective when multiscale data capturing both cell behavior ( migration ) and organization ( e . g . CMAC and F-actin status ) is derived simultaneously on a per cell basis . This facilitates the leveraging of natural or induced heterogeneity to define: i ) the statistical structure of variation ( e . g . progressive or discrete ) within and between cell populations , as well as; ii ) key trends , dependencies and relationships within the cell migration system ( Lock et al . , 2014; Keren et al . , 2008; Kiss et al . , 2015; Kowalewski et al . , 2015; Ku et al . , 2012; Lee et al . , 2015; Lock and Str\\u00f6mblad , 2010 ) . Here , we extend on this approach by employing a tailored analytical toolbox to detect two quantitatively distinct sub-modalities of mesenchymal migration , thus illuminating the discrete nature of variation within this general migration archetype . We employ this unique suite of analytical tools to further characterize key aspects of the behavior , organization , and regulation of these divergent migration strategies . Collectively , this study now provides conceptual and practical capabilities to the cell migration research community , while also highlighting the importance of distinguishing individual mesenchymal migration modes as a vital precursor to understanding mesenchymal migration as a whole . \\n To enable the detection of discrete mesenchymal migration sub-modalities , and thereafter , the comprehensive analysis of cellular organization , regulation , and adaptation underlying each mode , we have integrated a unique combination of analytical tools . These are now made freely available as a Matlab Toolbox: 'The Cell Adhesion and Migration Analysis Toolbox' \\u2013 along with the raw quantitative data underlying this study , sample image data to aid implementation , and explanatory documentation ( see doi:10 . 5061/dryad . 9jh6m ) . The features included in this Toolbox are also specified briefly in 'Materials and methods' . H1299 ( non-small cell lung carcinoma ) cells stably expressing EGFP-paxillin ( marker for CMACs ) and RubyRed-LifeAct ( marker for F-actin ) ( H1299 P/L cells [Lock et al . , 2014] ) were imaged via confocal microscopy on glass coated with 2 . 5 \\u00b5g/ml fibronectin ( FN ) . H1299 P/L cells moved individually and exclusively via mesenchymal ( lamellipodial ) migration . Nonetheless , we observed and classified two qualitatively different migration modes emerging within the clonally derived H1299 P/L cell population . We termed these migration modes 'Discontinuous' and 'Continuous' , reflecting their contrasting stepwise and smooth motion , respectively ( Figure 1 ) . Specifically , cells migrating in the Discontinuous mode cycle in sequence through at least three recognizable stages of movement , which we term 'lateral protrusion' , 'polarization' , and 'tail retraction' , as described in Figure 1A ( also see Video 1 ) . These stages produce dramatic changes in cell morphology and frequent changes in migratory direction . Such directional changes occur at least in part because lateral protrusions typically develop at approximately 90\\u00b0 to the axis of the preceding tail retraction . Overall , the stepwise nature of Discontinuous migration is highly reminiscent of previous descriptions of fibroblast-like migration ( Theisen et al . , 2012; Abercrombie et al . , 1970; Chen , 1981 ) . By contrast , cells migrating in the Continuous mode move progressively , with less frequent changes in cell morphology and motile direction , in a manner analogous to classical keratocyte-like migration ( Figure 1B and Video 2 ) ( Keren et al . , 2008 ) . Importantly , these migration modes emerge spontaneously in parallel under uniform experimental conditions within a clonally\\u00a0derived cell line ( Figure 1\\u2014figure supplement 1 and Video 3 ) . It is noteworthy that cells may traverse the same underlying substrate regions while stably occupying distinct modes ( see Video 3 ) , indicating that local environmental inconsistencies are not the principal cause of differential mode identity . Yet , inter-modal conversion is possible , occurring bi-directionally between modes ( Figure 1\\u2014figure supplement 2 and Videos 4 and 5 ) , indicating their profound plasticity . 10 . 7554/eLife . 11384 . 003Figure 1 . Classification of 'Discontinuous' and 'Continuous' modes of mesenchymal ( lamellipodial ) migration . ( A , B ) Confocal image sequences of individual H1299 cells stably expressing EGFP-paxillin ( cell-matrix adhesion complex [CMAC] marker; upper image panels ) and RubyRed-LifeAct ( F-actin marker; lower image panels ) migrating in either Discontinuous ( A ) or Continuous ( B ) modalities on fibronectin ( FN ) -coated ( 2 . 5 \\u00b5g/ml ) glass . Images are displayed with gray-scale inverted . Numbers denote time ( min ) . Schematics in blue ( Discontinuous , A ) and orange ( Continuous , B ) depict typical cell morphology changes associated with each migration mode . Note stepwise cycles of: lateral protrusion ( in directions of dashed arrows ) ; cell polarization; and tail retraction ( in direction of solid arrows ) that recur during Discontinuous migration , and that lateral protrusion tends to occur at 90\\u00b0 to the preceding tail retraction . During Continuous migration , cell morphology is relatively stable , with many small protrusion ( open arrowheads ) and retraction ( closed arrowheads ) events producing smooth movement . Movies corresponding to cells shown in A and B are available in supporting material ( Videos 1 , 2 ) . ( C ) Quantification of frequencies of Discontinuous and Continuous modes . ( D ) Box plots of cell speed ( \\u00b5m/sec ) per migration mode . Boxplots show median values and inter-quartile ranges ( IQR , 25% to 75% ) . Notches indicate median +/\\u2212 1 . 57 * IQR/\\u221an ( approximates 95% confidence interval of the median , n = number of cell observations , see 'Materials and methods' ) . Statistical discernibility assessed by Wilcoxon rank sum test , p < 0 . 001 . ( E ) Cell trajectories in each mode were assessed via mean squared displacement ( MSD ) analysis and divided into quintiles ( 20% bins ) according to their migration cefficient ( related to speed of movement ) . Median values +/\\u22121 . 57 * IQR/\\u221an ( n = number of cell observations ) of a second MSD measure , persistence time ( related to migration direction stability ) , are plotted per migration coefficient quintile . Number of observations per quintile: Discontinuous = ~ 402; Continuous = ~ 95 . ( F ) A table summarizes results from visual inspection of several cell lines migrating on 2 . 5 \\u00b5g/ml FN , confirming the emergence of either one or both Discontinuous and Continuous migration modes in these cell types . Example image sequences are presented in Figure 1\\u2014figure supplement 3 . ( G ) A table summarizes results from visual inspection of H1299 cells migrating on several different extracellular matrix ligands , confirming the emergence of both migration modes under each condition . Example image sequences are presented in Figure 1\\u2014figure supplement 4 . Scale bars = 20 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00310 . 7554/eLife . 11384 . 004Figure 1\\u2014figure supplement 1 . Discontinuous and Continuous migration modes emerge in parallel under uniform conditions . A confocal image sequence showing the parallel emergence of Discontinuous and Continuous migration modes in H1299 P/L cells labeled with a far red membrane-dye on 5 \\u00b5g/ml fibronectin-coated glass . ( A ) De-noised ( left ) and segmented ( right ) images show a selected region from within a 4 x 4 image montage . Based on observed cell dynamics ( see Video 3 ) , cells were classified as in either the Discontinuous ( e . g . cell in blue box ) or Continuous mode ( e . g . cell in red box ) . ( B ) Stereotypical Discontinuous mode behaviors are displayed in a series of time points ( numbers denote minutes ) following the cell from the blue box in A . Specifically: Tail Retraction is denoted by solid arrows; Lateral Protrusion and Polarization are denoted by dashed double headed arrows; ( C ) The relatively stable morphology of Continuous mode migration is shown by the cell from the red box in A . It is noteworthy that both highlighted cells sequentially traverse the same region of the coated substrate while maintaining distinct migration modes ( see Video 3 ) , indicating that potential inconsistencies in the local environment could not be responsible for the parallel emergence of distinct modes . Scale bars = 10 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00410 . 7554/eLife . 11384 . 005Figure 1\\u2014figure supplement 2 . Discontinuous and Continuous migration modes can spontaneously inter-convert ( mode switching ) . Confocal image sequences show the spontaneous inter-conversion between Discontinuous and Continuous migration modes by H1299 P/L cells labeled with a far red membrane-dye on 5 \\u00b5g/ml fibronectin-coated glass . ( A ) De-noised ( upper ) and segmented ( lower ) time sequences ( numbers denote minutes ) of a single cell spontaneously transitioning ( switching ) from the Continuous migration mode to the Discontinuous migration mode ( see tracked cell in Video 4 ) . The approximate timing of the Mode Transition is indicated . ( B ) De-noised ( upper ) and segmented ( lower ) time sequences of a single cell spontaneously transitioning ( switching ) from the Discontinuous migration mode to the Continuous migration mode ( see tracked cell in Video 5 ) . The approximate timing of the Mode Transition is indicated . Scale bars = 10 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00510 . 7554/eLife . 11384 . 006Figure 1\\u2014figure supplement 3 . Discontinuous and/or Continuous migration modes recur in multiple cell types . Confocal image sequences exemplifying Discontinuous and/or Continuous migration on 2 . 5 \\u00b5g/ml fibronectin-coated glass by several cell types , including: BT549 ( breast epithelial [ductal carcinoma] ) ; MDA-MB-231 ( breast epithelial [adenocarcinoma] ) ; Hep-3 ( hepatocellular carcinoma ) ; and HS578T ( breast epithelial [carcinoma] ) . Cells were labeled with a membrane dye and imaged at 5-min intervals . Images are displayed with gray-scale inverted . Numbers denote time ( min ) . Movies corresponding to image sequences ( in the order shown ) are available in supporting material ( Videos 6\\u201311 , respectively ) . Scale bars = 20 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00610 . 7554/eLife . 11384 . 007Figure 1\\u2014figure supplement 4 . Discontinuous and Continuous migration modes recur in H1299 cells adhering to multiple extracellular matrix ligands . Confocal image sequences exemplifying H1299 cell migration in Discontinuous and Continuous modes on glass coated with various purified extracellular matrix ligands , including: fibronectin ( FN ) ; Laminin; Collagen Type 1; and Vitronectin . Cells were labeled with a membrane dye and imaged at 5-min intervals . Images are displayed with gray-scale inverted . Numbers denote time ( min ) . Schematics of sequences on FN detail the characteristic movement and morphology changes associated with each mode , as shown in Figure 1 . See movies corresponding to image sequences ( in the order shown but excluding fibronectin sequences; as this is shown in Videos 1 , 2 ) ( Videos 12\\u201317 , respectively ) . Scale bars = 20 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00710 . 7554/eLife . 11384 . 008Video 1 . H1299 P/L cell migration in the Discontinuous mode . High-resolution multiscale imaging of a single H1299 P/L cell expressing EGFP-paxillin ( green , CMAC marker ) and RubyRed-LifeAct ( red , F-actin marker ) during migration in the Discontinuous mode on 2 . 5 \\u00b5g/ml fibronectin . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00810 . 7554/eLife . 11384 . 009Video 2 . H1299 P/L cell migration in the Continuous mode . High-resolution multiscale imaging of a single H1299 P/L cell expressing EGFP-paxillin ( green , CMAC marker ) and RubyRed-LifeAct ( red , F-actin marker ) during migration in the Continuous mode on 2 . 5 \\u00b5g/ml fibronectin . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00910 . 7554/eLife . 11384 . 010Video 3 . Parallel emergence of Discontinuous and Continuous modes in H1299 P/L cells . Low-resolution imaging of H1299 P/L cells labeled with a membrane dye shows the parallel emergence of both Discontinuous ( e . g . dark blue outline , trajectory shown ) and Continuous ( e . g . light blue outline , trajectory shown ) modes under uniform conditions . Cells have been segmented and tracked to highlight morphologies and trajectories . Note that two cells in the Continuous mode ( yellow and lilac outlines ) pass through the substrate region traversed by the Discontinuous mode cell ( dark blue ) , yet these cells remain in the Continuous mode . This implies that these migration modes are not simply determined by ( possible ) local variations in , for example , ECM substrate ( 5 \\u00b5g/ml fibronectin ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01010 . 7554/eLife . 11384 . 011Video 4 . Migration mode transition from Continuous to Discontinuous motility . Low-resolution imaging of H1299 P/L cells labeled with a membrane dye , during random migration on 5 \\u00b5g/ml fibronectin . Cells have been segmented and tracked to highlight morphologies and trajectories . Note that the cell with light green outline ( trajectory shown ) transitions from Continuous to Discontinuous migration during the course of imaging . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01110 . 7554/eLife . 11384 . 012Video 5 . Migration mode transition from Discontinuous to Continuous motility . Low-resolution imaging of H1299 P/L cells labeled with a membrane dye , during random migration on 5 \\u00b5g/ml fibronectin . Cells have been segmented and tracked to highlight morphologies and trajectories . Note that the cell with light green outline ( trajectory shown ) transitions from Discontinuous to Continuous migration during the course of imaging . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 012 Under the conditions described above , ~80% of observations showed cells migrating via the Discontinuous mode , with the remaining 20% moving in the Continuous mode ( Figure 1C , see 'Materials and methods' for sample numbers relating to all experimental data ) . Automated cell tracking ( see 'Materials and methods' , [Lock et al . , 2014] ) revealed that cells in the Discontinuous mode migrate significantly faster than during Continuous migration ( Figure 1D ) . By applying mean squared displacement analysis of cell trajectories to assess measures related to cell speed ( migration coefficient ) and directionality ( persistence time ) , we found that Discontinuous migration is less directionally stable at any given speed ( Figure 1E ) . However , both modes of migration show a positive correspondence between speed and directionality , as recently reported ( Maiuri et al . , 2015 ) . Importantly , an extensive comparison of migratory behaviors in several additional cell lines ( Figure 1F , Figure 1\\u2014figure supplement 3 and Videos 6\\u201311 ) and during H1299 cell migration on alternative extracellular matrix ligands ( Figure 1G , Figure 1\\u2014figure supplement 4 and Videos 12\\u201317 ) indicated that Discontinuous and Continuous migration modes are consistently recurring phenomena . 10 . 7554/eLife . 11384 . 013Video 6 . BT549 cell migration in the Discontinuous mode . A single BT549 cell migrating in the Discontinuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01310 . 7554/eLife . 11384 . 014Video 7 . BT549 cell migration in the Continuous mode . A single BT549 cell migrating in the Continuous mode ( upper centre cell ) . Cropped from a larger image montage ( note , montage stitching can cause observable intensity boundaries in image; however , these have no effect on interpretation ) . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are\\u00a0not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01410 . 7554/eLife . 11384 . 015Video 8 . MDA-MB-231 cell migration in the Discontinuous mode . A single MDA-MB-231 cell migrating in the Discontinuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01510 . 7554/eLife . 11384 . 016Video 9 . MDA-MB-231 cell migration in the Continuous mode . MDA-MB-231 cells migrating in the Continuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01610 . 7554/eLife . 11384 . 017Video 10 . Hep3 cell migration in the Discontinuous mode . Hep3 cells migrating in the Discontinuous mode . Cropped from a larger image montage ( note , montage stitching can cause observable intensity boundaries in image; however , these have no effect on interpretation ) . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data is not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01710 . 7554/eLife . 11384 . 018Video 11 . HS578T cell migration in the Continuous mode . A single HS578T cell migrating in the Continuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01810 . 7554/eLife . 11384 . 019Video 12 . H1299 P/L cell migration in the Discontinuous mode on laminin . H1299 P/L cells migrating in the Discontinuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 \\u00b5g/ml laminin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01910 . 7554/eLife . 11384 . 020Video 13 . H1299 P/L cell migration in the Continuous mode on laminin . A single H1299 P/L cell migrating in the Continuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 50 \\u00b5g/ml laminin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 02010 . 7554/eLife . 11384 . 021Video 14 . H1299 P/L cell migration in the Discontinuous mode on collagen type 1 . H1299 P/L cells migrating in the Discontinuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 \\u00b5g/ml collagen type 1 . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 02110 . 7554/eLife . 11384 . 022Video 15 . H1299 P/L cell migration in the Continuous mode on collagen type 1 . H1299 P/L cells migrating in both Discontinuous and Continuous modes . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 5 \\u00b5g/ml collagen type 1 . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 02210 . 7554/eLife . 11384 . 023Video 16 . H1299 P/L cell migration in the Discontinuous mode on vitronectin . A single H1299 P/L cell migrating in the Discontinuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 1 \\u00b5g/ml vitronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 02310 . 7554/eLife . 11384 . 024Video 17 . H1299 P/L cell migration in the Continuous mode on vitronectin . A single H1299 P/L cell migrating in the Continuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 1 \\u00b5g/ml vitronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 024 Cell migration is the product of membrane dynamics that can be divided into protrusive and retractive processes ( Ridley , 2003 ) . We quantitatively compared the spatial and temporal dynamics of membrane protrusions and retractions between and within each migration mode based on defining membrane dynamics over the minimal imaging interval of 5 min ( Figure 2 ) ( Kowalewski et al . , 2015 ) . This revealed that protrusions share similar size ( area ) distributions in both modes ( Figure 2A ) , while retraction events are more extreme in size during Discontinuous migration , that\\u00a0is , more frequently very small or very large ( Figure 2B ) . When compared within\\u00a0each mode , membrane retractions have a broader size distribution than protrusions in the Continuous mode ( Figure 2C ) , although this is far more striking in the Discontinuous mode ( Figure 2D ) . To investigate membrane dynamics operating over longer time-scales , we calculated the probabilities of protrusions and retractions of a given size based on net cell shape/position changes occurring over intervals of between 1 and 15 image frames ( 5 to 75 min ) . This revealed a relatively unstructured pattern of dynamics in Continuous cells , although retractions tended to be smaller and protrusions larger ( Figure 2E ) . In contrast , Discontinuous migration consistently displayed a wide distribution of retraction sizes ( very small and very large ) , with protrusion sizes uniformly moderate ( Figure 2F ) . Notably , the relatively high probability of very large retraction events in Discontinuous cells corresponds with their observed tendency to undergo dramatic tail retraction events . Collectively , these results explain the spatial characteristics underlying stepwise or smooth cell motion during migration in Discontinuous or Continuous modes , respectively . 10 . 7554/eLife . 11384 . 025Figure 2 . Membrane protrusion and retraction dynamics are temporally decoupled and less coordinated during Discontinuous migration . ( A ) Probability distribution of membrane protrusion sizes ( \\u00b5m2 ) per 5-min interval during Discontinuous ( blue ) and Continuous ( orange ) migration . Subtraction of the Continuous from Discontinuous protrusion size probability distribution ( inset ) reveals no substantial or structured difference in these distributions ( probability difference , Y axis ) . ( B ) Probability distribution of membrane retraction sizes ( \\u00b5m2 ) per 5-min interval during Discontinuous ( blue ) or Continuous ( orange ) migration . Subtraction of the Continuous from Discontinuous retraction size probability distribution ( inset ) reveals that retraction sizes tend to be more extreme ( frequently small or large , rarely moderate ) in cells during Discontinuous migration ( probability difference , Y axis ) . ( C ) Subtraction of the protrusion from retraction size probability distribution ( 5 min intervals only ) during Continuous migration reveals little structure in probability differences . ( D ) Subtraction of the protrusion from retraction size probability distribution ( 5 min intervals only ) during Discontinuous migration reveals that retractions in this mode are consistently more extreme in size ( frequently small or large , rarely moderate ) than protrusions . ( E ) Surface plotting of probability differences ( retraction minus protrusion probability , per size ) over various time windows ( 1 to 15 frames; 5 to 75 min ) in cells during Continuous migration . Surface color-coding indicates where protrusions ( blue ) or retractions ( red ) are more common at a particular size . ( F ) Surface plotting of probability differences ( retraction minus protrusion probability , per size ) over various time windows ( 1 to 15 frames; 5 to 75 min ) in cells during Discontinuous migration . ( G ) Signal cross-correlation was calculated between protrusion size and retraction size fluctuations per cell during Continuous migration . Mean cross-correlation values ( Y axis , red = positive , blue = negative ) plotted as a surface , per time lag ( -12 to 12 frames , negative values indicate protrusion leads retraction , positive values indicate retraction leads protrusion ) , per time window ( ranging from 1 to 15 frames; 5 to 75 min ) . ( H ) Signal cross-correlation was calculated between protrusion size and retraction size fluctuations per cell during Discontinuous migration . Mean cross-correlation values ( Y axis , red = positive , blue = negative ) plotted as a surface , per time lag ( -12 to 12 frames , -ve values indicate protrusion leads retraction , +ve values indicate retraction leads protrusion ) , per time window ( ranging from 1 to 15 time points; 5 to 75 min ) . ( I ) Mean ( over all lags , per time window ) absolute cross-correlation values for Continuous ( orange ) and Discontinuous ( blue ) migration , /- 95% confidence intervals , n = number of cells ( see Materials and Methods ) . Statistical discernibility assessed by Friedman testing , p = 0 . 0012 . ( J ) Cells in each migration mode were divided into quintiles ( 20% bins ) based on instantaneous cell speed , and median protrusion areas /- 1 . 57 * interquartile range ( IQR , 25% to 75% ) / \\u221an ( approximates 95% confidence interval of the median , n = number of cell observations , see Materials and Methods ) were calculated per quintile . ( K ) Median retraction areas /- 1 . 57 * IQR/ / \\u221an ( n = number of observations ) were calculated per cell speed quintile . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 025 To understand the temporal dynamics underlying differences in migration mode behavior , we assessed the cross-correlation between protrusion and retraction size signals over time . This revealed strong temporal coordination ( high cross-correlation ) between protrusion and retraction events during Continuous migration , with a single cross-correlation peak around the -1 time-lag . This indicated that protrusion consistently led retraction , but that these dynamics are tightly coupled in time ( Figure 2G ) . This was true regardless of the time frame over which membrane dynamics were defined . In marked contrast , two cross-correlation peaks were apparent in the Discontinuous mode , with one peak stable around the -1 time-lag ( tightly coupled dynamics ) , and a second peak temporally offset with positive lags ( Figure 2H ) . This offset was dependent on the time frame used to define membrane dynamics , but clearly indicates a population of temporally decoupled membrane dynamics , wherein retraction leads protrusion events that are significantly delayed . This confirmed the visual impression of Discontinuous migration as a cyclical , stepwise process with asynchronous retraction ( i . e . tail retraction stage ) preceding protrusion ( i . e . lateral protrusion stage ) . In contrast , the morphological stability observed during Continuous migration is a consequence of the tight coupling of protrusion and retraction , producing motility without large-scale , transient size/shape changes . Visual impressions also suggested the Discontinuous migration mode to be less ordered/coordinated than the Continuous mode . The significantly lower average ( combining all time-lags ) protrusion-retraction cross-correlation values during Discontinuous migration ( Figure 2I ) confirmed reduced coordination of membrane dynamics in this migration mode . To complete our analysis of how membrane dynamics differentially contribute to migration in each mode , we assessed the relationship between cell speed and the size of membrane protrusions ( Figure 2J ) or retractions ( Figure 2K ) . Both modes exhibited similar dependencies . Remarkably , cell speed appears in each case to depend only slightly on the size of membrane protrusions , instead being largely determined by retraction size . The data described above detail significant quantitative differences between both the cell motion and membrane dynamics comprising each migration mode behavior . Yet , how these divergent behaviors emerge from the organization of underlying macromolecular machineries remains unclear . We therefore extended our image analyses ( see Materials and Methods and [Lock et al . , 2014] ) to derive a multivariate dataset of 55 organizational features ( detailed in Figure 3\\u2014figure supplement 1 ) defining cellular-scale morphology as well as the state ( e . g . size , number , density , morphology , localization ) and dynamics ( e . g . motion , stability , rates of area/density change ) of critical macromolecular-scale machineries driving cell migration , namely , cell-matrix adhesion complexes ( CMACs; demarcated by EGFP-paxillin ) and F-actin ( demarcated by RubyRed-LifeAct ) . This multiscale organizational data was measured per cell , per time-point , thereby complementing corresponding measures of migration behavior from the same individual cells ( Figure 3\\u2014figure supplement 1 ) . Such integrated data allowed exploration of the organizational states that give rise to particular cell migration behaviors , based on information embedded in natural cell heterogeneity ( Lock et al . , 2014; Lock and Str\\u00f6mblad , 2010 ) . To first understand how much overlap exists between the organizational states underpinning Discontinuous and Continuous migration modes , we performed supervised clustering ( canonical vectors analysis , CVA ) of all cell observations based on their organizational features only ( Figure 3A ) . Strikingly , the two migration mode populations showed almost no overlap , indicating that their underlying patterns of organization can be quantitatively distinguished . We next visualized these organizational states via parallel coordinate mapping of 15 key organizational features ( Figure 3B ) . This revealed specific instances of dissimilarity in feature values as well as an overview of the collective organizational signatures associated with each mode . To estimate the contribution of specific features to the divergence between modes , we ranked organizational features by the coefficient values assigned to them within the first canonical vector of the CVA ( Figure 3C ) . We then compared feature value distributions between modes for a subset of these top-ranked features , finding highly significant differences in all cases ( Figure 3D\\u2013K ) . These analyses help to define the spectrum of differences in organization that spontaneously arise in parallel with the emergence of Discontinuous and Continuous migration mode behaviors . For instance , cells in the Discontinuous mode are: far smaller; less round ( more protrusive ) ; have fewer , smaller , less dense CMACs ( based on EGFP-paxillin intensity ) with less F-actin association ( based on background-subtracted RubyRed-LifeAct intensity [Li et al . , 2010] ) and shorter lifetimes ( less stable ) , resulting in a drastically decreased total adhesion area . 10 . 7554/eLife . 11384 . 026Figure 3 . Discontinuous and Continuous migration modes reflect quantitatively distinct cell states with unique signatures of underlying organization . ( A ) Canonical vectors analysis ( CVA ) -based multivariate clustering ( based on all 55 organizational features ( see Figure 3\\u2014figure supplement 1 ) , canonical vectors ( CVs ) 1 and 2 displayed , percentages indicate proportion of total variance per CV ) of cell observations ( see Materials and Methods ) during Discontinuous ( blue ) or Continuous ( orange ) migration show modes to be quantitatively distinct . Large circles with black outlines indicate population centers of mass . ( B ) Parallel coordinate mapping of key organizational feature values ( per cell observation ) detail the multivariate signatures associated with cells during Discontinuous ( light blue ) or Continuous ( light orange ) migration . Points of difference between these multivariate signatures are emphasized by plotting of median values for Discontinuous ( blue ) and Continuous ( orange ) modes . Wilcoxon rank sum testing ( per feature ) assessed statistically discernable differences ( p < 0 . 001 ) , except where shown in gray italics . ( C ) A list of organizational features ranked by their contribution ( coefficient values in canonical vector 1 , which contains 99% of total variance ) to the separation of migration modes in the canonical vectors space shown in ( A ) . Gray backgrounds highlight features for which value distributions are compared between Discontinuous ( blue ) and Continuous ( orange ) modes in ( D-K ) . ( D-K ) Features compared indicated on boxplot Y-axes . Boxplots show median values and inter-quartile ranges ( IQR , 25% to 75% ) . Notches indicate median /- 1 . 57 * IQR/ / \\u221an ( approximates 95% confidence interval of the median , n = number of cell observations , as in ( A ) . P values reflect Wilcoxon rank sum testing as in ( B ) . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 02610 . 7554/eLife . 11384 . 027Figure 3\\u2014figure supplement 1 . Definition of organizational and behavioral features . This table defines the names , relevant measurement units , and a brief description of quantitative features derived through multiscale image analysis . Features are numbered 1-60 in correspondence with the order and numbering presented in Spearman's correlation heatmaps in Figure 4 . Note , of these 60 features , 55 define Organizational features ( light gray , used in multivariate clustering analyses ) , while 5 define the Behavioral features assessed herein ( dark gray ) . Additional terms are defined above relevant columns . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 027 Having defined a spectrum of differences in key organizational features underlying migration modes , we next explored how the correlative relationships between these features might also vary . To this end , we mapped the complete network ( including both organizational and behavioral features ) of pairwise inter-feature Spearman\\u2019s correlation coefficients ( rs ) for both Discontinuous ( Figure 4A ) and Continuous ( Figure 4B ) modes . Visual inspection of the heatmaps presented in 4A and 4B suggests that , while some differences in correlations are apparent , general patterns of correspondence are well preserved regardless of migration mode . This is emphasized by calculation of absolute differences in rs values between modes ( Figure 4C ) , wherein the majority of relationships appear virtually unchanged . Indeed , only a very limited number of inter-feature relationships change by more than 0 . 4 rs ( Figure 4D ) . This suggests that changes in inter-feature correlations are unexpectedly selective , given the magnitude of changes in the actual feature values themselves ( Figure 3 ) . To assess this , we compared the observed distribution of absolute differences in rs values to that which would be expected if changes arose randomly from the rs distributions depicted in Figure 4A and 4B ( Figure 4E ) . This confirmed that observed changes are much smaller and less frequent than randomly expected , thus supporting the proposal that reconfiguration of correlative relationships is highly selective . 10 . 7554/eLife . 11384 . 028Figure 4 . Differences in inter-feature relationships between Discontinuous and Continuous migration modes appear highly selective . ( A , B ) Heatmaps of Spearman's rank correlation coefficients ( rs ) for all pairwise combinations between 60 features ( 55 organizational and 5 behavioral ) , derived from cells during ( A ) Discontinuous or ( B ) Continuous migration . Individual correlation coefficient values are color-coded as depicted in the color bar ( blue = negative; red = positive; green = near zero ) . Numbers on X and Y axes ( 1 to 60 ) correspond to the identities of features , as defined in Feature Number column of Figure 3\\u2014figure supplement 1 . ( C ) A heatmap summarizes absolute differences in correlation coefficient values , per feature pair , between Discontinuous and Continuous migration modes . Difference values are color-coded as depicted in the color bar ( blue = no difference; red = large difference ) . ( D ) A histogram shows the frequency distribution of absolute differences in correlation coefficient values between Continuous and Discontinuous migration modes . ( E ) A plot of cumulative distribution functions ( CDFs ) comparing the observed distribution of absolute differences in correlation coefficient values ( between Discontinuous and Continuous modes as in [C] and [D] , black line ) and the differences in coefficient values obtained following randomized permutation of Spearman's correlation pairs ( gray line ) . The permuted distribution ( repeated 100 times for all relationships , all values included in CDF ) shows the expected distribution of coefficient differences if inter-feature correlation changes occur randomly . Comparison of observed and permuted CDFs suggests that observed differences in inter-feature Spearman's correlation values are far more selective than randomly expected . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 028 We next analyzed how organizational features correlated to cell speed in each migration mode . As exemplified in Figure 5A , and more extensively in Figure 5B , we found that a proportion of such correlations were essentially equivalent between modes ( e . g . Cell Area vs Cell Speed rs = negative in both modes; Cell Compactness \\u2212 Cell Speed rs = positive in both modes ) . However , a large number of cell speed correlations were detectable in only one of the two modes ( e . g . Median [CMAC Distance to Border] per Cell vs Cell Speed rs = positive only in Continuous mode; Median [CMAC Area] per Cell vs Cell Speed rs = positive only in Continuous mode ) . Even more salient , a substantial number of correlations to Cell Speed were inverted between modes ( e . g . Median [CMAC Lifetime] per Cell vs Cell Speed rs = negative in Discontinuous mode , positive in Continuous mode; Median [CMAC Average Trailing Edge Speed] per Cell vs Cell Speed rs = positive in Discontinuous mode , negative in Continuous mode ) . Such inversions in organizational feature-to-Cell Speed relationships hint at profound differences in the mechanisms controlling each mode . We also note that , more broadly , most organizational feature-to-Cell Speed correlations were both stronger and more positive during migration in the Continuous mode ( Figure 5C ) . This suggests that Continuous mode migration is more directly dependent on the state of CMAC and F-actin machineries , while alternative machineries may play more dominant roles during Discontinuous migration . 10 . 7554/eLife . 11384 . 029Figure 5 . Cell speed variation is coupled to distinct patterns of remodeling in underlying features and inter-feature relationships within each migration mode . ( A ) A selection of Spearman's correlation coefficient values ( rs ) for organizational feature relationships to cell speed are depicted ( color-coded as in color bar ) . Values reflect medians of populations values . Correlations are considered non-significant ( ns ) if zero is included in the range of the median +/\\u22121 . 57 * inter-quartile range ( IQR , 25% to 75% ) \\u221an ( approximates 95% confidence interval of the median , n = number of cell observations , see 'Materials and methods' ) . This highlights correlations to cell speed that: are equivalent between modes ( e . g . Cell Area , negative correlation in both modes ) ; exist in only one mode ( e . g . Median [cell-matrix adhesion complex ( CMAC ) Area] per Cell , ns in Discontinuous vs positive in Continuous ) ; or are opposite between modes ( e . g . Median [CMAC Lifetime] per Cell , negative in Discontinuous vs positive in Continuous ) . Examples in A are linked to an extensive analysis of cell speed-to-organizational feature correlations in B and C . ( B ) Parallel coordinate mapping of median cell speed-to-organizational feature correlations in Discontinuous ( blue ) and Continuous ( orange ) modes . Relationships are categorized by correlation value similarity or difference between modes , as indicated by the orange;\\u00a0C' ( denotes Continuous ) and blue 'D' ( denotes Discontinuous ) followed by '\\u2212'; ( negative rs ) or\\u00a0'+' ( positive rs ) or 'ns' ( non-significant rs ) . ( C ) Bar graphs depict the magnitude of differences in median cell speed-to-organizational feature correlation values from the Discontinuous to the Continuous mode . Correlations tend to be stronger and more positive in cells during Continuous migration . ( D ) Canonical vectors analysis ( CVA ) -based multivariate clustering ( using all 55 organizational features , canonical vectors [CV] 1 , 2 , and 3 displayed , percentages indicate proportion of total variance per CV ) of cell observations during: slow Discontinuous ( 0\\u201333 . 33% of Discontinuous migration speed values , blue ) ; medium Discontinuous ( 33 . 34\\u201366 . 66% , cyan ) ; fast Discontinuous ( 66 . 67\\u2013100% , green ) ; slow Continuous ( 0\\u201333 . 33% of Continuous migration speed values , yellow ) ; medium Continuous ( 33 . 34\\u201366 . 66% , orange ) ; or fast Continuous ( 66 . 67\\u2013100% , red ) migration . Two orientations of the same three-dimensional clustering are depicted ( upper and lower ) , revealing the separate clustering of each migration mode . Within each mode , progressive differences in cell speed correspond to similarly progressive variations in the position of observations within the multivariate organizational feature ( or cell state ) space . These speed-dependent trends in clustering define trajectories in the cell state space along which cells evolve as cell speed changes . Remarkably , these trajectories are completely distinct when comparing Discontinuous ( dashed lines ) and Continuous ( solid lines ) migration modes . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 029 We have thus far detailed a variety of specific differences between Discontinuous and Continuous migration modes in terms of behavior ( Figure 1 and Figure 2 ) , underlying organizational features ( Figure 3 ) and individual inter-feature correlations ( Figure 4 and Figure 5A\\u2013C ) . We next explored how these differences might translate into systemic dissimilarities in the coupling of cell behavior ( i . e . cell migration speed ) and cell organization within each mode . To this end , we first grouped cell observations according to both migration mode and tertiles of cell speed ( 0\\u221233% [slow]; 33\\u221266% [medium]; 66\\u2212100% [fast] ) . We then performed supervised CVA clustering of these groups ( Figure 5D ) . This enabled assessment of how cells in each mode cluster in the multivariate cell state space , but also of the trajectories defined through cell state space given variability in cell migration speed ( as described previously , [Lock et al . , 2014] ) within each mode . As also observed in Figure 3A ( based on similar but not equivalent clustering ) , we found that cells in Discontinuous or Continuous modes cluster separately . However , in a striking extension of this result , we also found that variability in migration speed within each mode corresponded to completely independent trajectories in cell state space . This means that different patterns of organizational remodeling are associated with varying cell speed within each mode . Thus , profound differences exist between modes in terms of the underlying strategies that produce and control cell migration speed . We have demonstrated how specific organizational feature values diverge in conjunction with the spontaneous emergence of Discontinuous and Continuous migration modes ( Figure 3 ) . Yet , it remains unclear whether these features are tightly coupled to migration mode identity , or only loosely correlated with these behaviors . We therefore assessed whether directed modulation of these features could drive corresponding and predictable changes in the frequency equilibrium between migration modes . To test for such bottom-up regulation , we applied a selection of targeted molecular perturbations for which we and others have established prior knowledge regarding their effects on the organizational features in question . These perturbations included: ECM ligand ( FN ) concentration modulation ( Gupton and Waterman-Storer , 2006 ) ; talin 1 depletion ( Kiss et al . , 2015 ) ; and ROCK inhibition-mediated reduction of actomyosin contractility ( Lock et al . , 2014; Kim and Wirtz , 2013; Hern\\u00e1ndez-Varas et al . , 2015 ) ( Figure 6A\\u2013C ) . Images exemplifying perturbation effects on cell , CMAC and F-actin morphology are presented in Figure 6\\u2014figure supplement 1 . Notably , these molecular perturbations collectively target four core regulatory mechanisms around the ECM \\u2013 adhesion \\u2013 F-actin axis , including integrin ligation; integrin activation/clustering; integrin-F-actin linkage; and actomyosin contractility ( Figure 6A ) . Given the pivotal importance of these mechanisms for mesenchymal migration , the effects of the specific molecular perturbations applied herein may be somewhat predictive of a wide array of related regulatory mechanisms . 10 . 7554/eLife . 11384 . 030Figure 6 . Cell-matrix adhesion and actomyosin contractility control the equilibrium between Continuous and Discontinuous migration modes . ( A ) Schematic summary of molecular mechanisms ( green ) associated with the ECM \\u2013 adhesion \\u2013 F-actin axis ( gray ) targeted here for perturbation to modulate implicated organizational features and , potentially , corresponding migration mode frequencies . Specifically , altering ECM ligand density ( fibronectin concentration , [FN] ) impacts mechanism 1 ) - integrin ligation and subsequent cell-matrix adhesion complex ( CMAC ) formation and maturation . RNAi-mediated knockdown of talin 1 limits both mechanism 2 ) - integrin activation and mechanism 3 ) - integrin-F-actin linkage , thereby also affecting CMAC formation , maturation and stability . Y-27632-mediated inhibition of ROCK disrupts mechanism 4 ) - actomyosin contractility , affecting F-actin dynamics and CMAC maturation . Images exemplifying the effects of each molecular perturbation are presented in Figure 6\\u2014figure supplement 1 . ( B ) Boxplots summarizing the response of the same selection of organizational features to: ECM ligand modulation ( 2 . 5 \\u00b5g/ml FN vs 10 \\u00b5g/ml FN , upper row ) ; talin 1 RNAi ( control siRNA vs talin 1 siRNA , 5 \\u00b5g/ml FN , middle row ) ; or ROCK inhibition ( DMSO vs 6 \\u00b5M Y-27632 , 5 \\u00b5g/ml FN , lower row ) . All boxplots in B show median values per condition and inter-quartile ranges ( IQR , 25% to 75% ) . Notches show the median +/\\u22121 . 57 * IQR/ / \\u221an ( approximates 95% confidence interval of the median , n = number of cell observations , see 'Materials and methods' ) . In each case , statistically discernable differences were assessed by Wilcoxon rank sum testing , with resulting p values <0 . 001 . ( C ) Migration mode frequency responses to each perturbation are depicted ( Discontinuous , blue; Continuous , orange ) . Additional conditions were included for ROCK inhibition ( low panel ) , showing a progressive response to 1 \\u00b5M , 2 \\u00b5M , and 6 \\u00b5M Y-27632 as compared to DMSO vehicle control . Note: in addition to depicting specific perturbation-dependent trends , the matrix of results presented in B and C is used , together with spontaneous feature variations depicted in Figure 3D , E , G\\u2013I , to logically parse organizational features that are consistently coupled , or just occasionally correlated , with migration mode identity . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 03010 . 7554/eLife . 11384 . 031Figure 6\\u2014figure supplement 1 . Comparison of cell , adhesion complex and F-actin morphologies following perturbations targeting the ECM \\u2013 adhesion \\u2013 F-actin axis . Confocal images of H1299 cells comparing: ( A ) low concentration ( 2 . 5 \\u00b5g/ml ) fibronectin ( FN ) -coated glass ( left ) vs high concentration ( 10 \\u00b5g/ml ) FN-coated glass ( right ) ; ( B ) control siRNA treatment ( left ) vs talin 1 siRNA treatment ( right ) ; and ( C ) control ( DMSO ) treatment ( left ) vs ROCK inhibitor ( Y-27632 , 6 \\u00b5M ) treatment ( right ) . EGFP-paxillin ( upper panels ) and RubyRed-LifeAct ( lower panels ) signals are shown to demarcate cell-matrix adhesion complexes ( CMACs ) and F-actin , respectively . Images are displayed with gray-scale inverted . ( D ) Talin 1 knockdown was confirmed by immunoblotting . Scale bars = 20 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 031 Importantly , we found that Continuous mesenchymal migration occurred upon high adhesion ( high FN concentration ) and with unperturbed actomyosin contractility , while Discontinuous migration occurred under conditions of low adhesion ( low FN concentration ) and inhibited contractility ( ROCK-inhibitor ) . Talin knock-down promoted Discontinuous migration ( Figure 6C ) . We then compared: i ) spontaneous differences in feature values between Continuous and Discontinuous migration modes ( displayed in Figure 3D , E , G and I ) ; ii ) perturbation-induced differences in the same features ( Figure 6B ) ; and iii ) perturbation effects on migration mode frequencies ( Figure 6C ) . From this matrix of results , we were able to determine whether a logical coherence was preserved over all these measures . For example , median CMAC Area was spontaneously higher in cells during Continuous migration ( Figure 3I ) , and increasing ECM ligand concentration ( FN ) causes higher CMAC Area values ( Figure 6B , upper left panel ) . It is therefore logically consistent ( if CMAC Area and migration mode behaviors are functionally coupled ) that raising ECM ligand concentration increases the frequency of the Continuous migration mode ( Figure 6C , upper panel ) . In fact , although the applied perturbations produce differing effects on CMAC Area ( increasing or decreasing ) , migration mode frequencies show logically consistent responses in each case . CMAC Area therefore appears likely to be closely coupled to migration mode identity , since there is no evidence that CMAC Area and migration mode identity could be uncoupled . Such coherence was also observed for CMAC Lifetime and Cell Compactness , thus similarly supporting their functional coupling to migration mode determination . In contrast , while the modulation of either ECM ligand concentration or talin 1 expression caused effects on Cell Area and CMAC Number that were consistent with their parallel effects on mode frequencies , ROCK inhibition caused contradictory results: CMAC number and Cell Area were both higher in cells in the Continuous mode ( Figure 3H ) and were increased following ROCK inhibition ( Figure 6B , lower right panel ) , yet ROCK-inhibition caused a decreased frequency of the Continuous mode ( Figure 6C , lower panel ) . These results imply that , although Cell Area and CMAC Number spontaneously correlate with migration mode identities under the original experimental condition , this correspondence can be uncoupled . These organizational features are therefore unlikely to be causal of , or caused by , migration mode identity determination . Taken together , the matrix of results presented delineates organizational features ( CMAC Area , CMAC Lifetime , Cell Compactness ) that may be directly coupled with migration mode determination . More definitively , these results effectively exclude Cell Area and CMAC Number from being ( linearly ) causally linked to this process . Furthermore , these results demonstrate the potent regulatory influence of the specific molecular components fibronectin , talin 1 and ROCK , on migration mode determination . These perturbations also allows us to conclude more broadly that cell-matrix adhesion complexes and actomyosin contractility play key roles in shaping Discontinuous and Continuous mode frequencies . The molecular perturbations described above changed both cellular and macromolecular organization , as well as the frequency equilibrium between Discontinuous and Continuous migration modes . However , it remained unclear to what extent perturbation effects on organizational features may reflect: a ) mode frequency changes ( inter-modal 'adaptive switching' ) , as opposed to b ) organizational remodeling of the modes themselves ( intra-modal 'adaptive stretching' ) . To address the balance between these two adaptive responses , we leveraged data from the comparison of low ( 2 . 5 \\u00b5g/ml ) and high ( 10 \\u00b5g/ml ) FN concentrations displayed in Figure 6 . Using only organizational features , we performed principal component analysis ( PCA ) -based unsupervised clustering of cell observations grouped by both FN concentration and migration mode ( Figure 7A ) . Surprisingly , this revealed that the distances ( in PCA-defined cell state space ) between Discontinuous and Continuous migration modes were far greater than the distances induced within each mode by FN concentration variation . Comparing the heterogeneity of individual cells over time ( Figure 7B and C , Figure 7\\u2014figure supplement 1 ) and the distances between migration mode centers of mass confirmed the substantial nature of average organizational differences between modes ( equal to or greater than the maximal span of variation observed over 8 hr in the individual cells shown in Figure 7B\\u2013C ) . To more intuitively visualize how changing FN concentration shifts the distribution of cell organization within the PCA-defined cell state space ( and between modes ) , we calculated the probability density distributions of cell observations in each mode , for each experimental condition ( Figure 7D\\u2013E ) . This revealed that cell observations in each mode populate high probability-density 'valleys' that are at least partially separated by low probability-density 'ridges' . This visualization also emphasized that the major response to FN modulation was a substantial rebalancing of cell organizational states from one valley ( migration mode ) to the other . Collectively , these observations demonstrate that switching ( between modes ) dominates over stretching ( remodeling within modes ) as an adaptive response to FN concentration modulation . The conceptual relation between these two mechanisms of adaptation in cell state space is schematized in Figure 7F . 10 . 7554/eLife . 11384 . 032Figure 7 . Adaptive switching between migration modes is the dominant perturbation response rather than intra-modal remodeling . ( A ) Principal component analysis ( PCA ) -based clustering of cell observations ( based on all 55 organizational features ) , principal components ( PC ) 1 and 2 displayed , percentages indicate proportion of total variance per PC , color-coded by migration mode and experimental condition: Discontinuous mode on 2 . 5 \\u00b5g/ml FN ( dark blue ) ; Discontinuous mode on 10 \\u00b5g/ml FN ( light blue ) ; Continuous mode on 2 . 5 \\u00b5g/ml FN ( red ) ; Continuous mode on 10 \\u00b5g/ml FN ( orange ) . Large black outlined circles indicate population centers of mass . Note that , despite FN variation exerting large effects on cell organization , differences between migration modes are much larger that differences between conditions within the same mode . ( B ) PCA clustering as in ( A ) , overlaid by the heterogeneous trajectories over time of 4 cells ( I\\u2013IV ) within the PCA-based multivariate state space . Cells I\\u2013IV belong to the spatially corresponding cell populations defined in ( A ) and are relatively constrained within their respective regions of the cell state space . ( C ) Single time point confocal images of EGFP-paxillin from Cells I\\u2013IV from ( B ) . Images are displayed with gray-scale inverted . Corresponding image sequence selections are presented in Figure 7\\u2014figure supplement 1 . Scale bars = 20 \\u00b5m . ( D ) A probability density map based on the coordinates in PCA space of cell observations from the 2 . 5 \\u00b5g/ml FN condition in ( A ) ( view orientation relative to A is rotated 180 degrees ) . Cells in the Discontinuous mode ( dark blue ) define a deep probability valley that partially overlaps with but is largely distinct from a shallow probability valley defined by cells in the Continuous mode ( red ) . The sum of probabilities equals 1 . ( E ) A probability density map based on the coordinates in PCA space of cell observations from the 10 \\u00b5g/ml FN condition in ( A ) ( view orientation relative to A is rotated 180 degrees ) . Cells in the Continuous mode ( orange ) define a deep probability valley that overlaps with but is partly distinct from a shallow probability valley defined by cells in the Discontinuous mode ( light blue ) . The sum of probabilities equals 1 . Centers of mass for each population in ( D ) and ( E ) are indicated by capped vertical lines of matching colors . Arrows in ( D ) and ( E ) signify how decreasing or increasing FN concentration , respectively , causes 'switching' from one probability valley ( migration mode ) to the other . ( F ) Schematic summary of the dual adaptive strategies employed by cells , with respect to mesenchymal migration . Modulation of intracellular or extracellular conditions can cause the remodeling of cellular and macromolecular organization locally within a given mode ( 'adaptive stretching' ) . However , the much larger and more frequent response to such perturbations is 'adaptive switching' between discrete migration modes , resulting in substantial rebalancing of Discontinuous and Continuous migration mode frequencies . ( G ) Dendogram ( qualitative ) indicating the major recognized archetypes of cell migration ( top row ) and the new modalities of Continuous and Discontinuous mesenchymal migration described herein ( bottom row ) . Italicized names in brackets correspond to similar/analogous migration modes/terms . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 03210 . 7554/eLife . 11384 . 033Figure 7\\u2014figure supplement 1 . Confocal image sequences of cells during Discontinuous and Continuous mode migration on glass coated with low or high fibronectin concentrations . Confocal image sequences of H1299 cells during migration on low concentration ( 2 . 5 \\u00b5g/ml ) fibronectin ( FN ) -coated glass ( A and B ) or\\u00a0high concentration ( 10 \\u00b5g/ml ) FN-coated glass ( C and D ) . EGFP-paxillin ( upper panels ) and RubyRed-LifeAct ( lower panels ) signals are shown to demarcate cell-matrix adhesion complexes ( CMACs ) and F-actin , respectively . Images are displayed with gray-scale inverted . Numbers denote time ( min ) . Scale bars = 20 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 033 \\n We here present an integrated analytical approach and associated tools , designed to interrogate both spontaneous and induced heterogeneity in single-cell mesenchymal ( lamellipodial ) migration . By deploying these tools , we quantitatively demonstrate that mesenchymal migration is composed of two distinct sub-modalities , each with specific behavioral , organizational , and regulatory characteristics . This marks an important advance that may enhance future investigations of the mesenchymal migration archetype . In particular , interpretations of natural and/or experimentally induced variability will likely become more coherent and precise if data are disaggregated according to Discontinuous or Continuous modes , rather than being unintentionally aggregated across cells with fundamentally different properties and dependencies . Recognition of these modes may therefore facilitate a more accurate understanding of the heterogeneity , adaptability , and regulation of mesenchymal migration as a whole . Our initial classification of cells into Discontinuous and Continuous migration modes was based on observed behavioral differences . Crucially , the same distinctive behaviors emerged in parallel within a variety of cell lines and under a spectrum of conditions , thus supporting the robustness and broad relevance of these behaviors . Yet , given the similarity of Continuous and Discontinuous migrations to previous descriptions of keratocyte-like ( Barnhart et al . , 2015; Keren et al . , 2008 ) or fibroblast-like motility ( Abercrombie et al . , 1977; Theisen et al . , 2012 ) , respectively , the existence of these behaviors per\\u00a0se\\u00a0is not novel . Indeed , early observations even qualitatively suggest the co-existence of similar behaviors ( Lewis et al . , 1982 ) . However , we now present quantitative evidence of the spontaneous , parallel emergence of two distinct yet inter-convertible mesenchymal migration modes , wherein divergent couplings exist between behavioral ( speed ) and organizational characteristics . We thereby provide new insights into the discontinuous structure of heterogeneity within the broad mesenchymal migration archetype . Indeed , by differentiating between progressive ( intra-modal ) and discrete ( inter-modal ) forms of variation , we gained the capacity to compare the roles that these two mechanisms play in adaptive responses to perturbation . Surprisingly , this revealed that adaptive switching between modes is the dominant response to perturbation ( FN concentration modulation ) , with adaptive stretching ( remodelling ) within modes playing only a minor role . The quantitative distinction between Discontinuous and Continuous migration modes is based on systemic differences detected via multivariate clustering of data following visual classification . Importantly , given that cells were classified based on perceived differences in their behavioral dynamics , we excluded behavioral features and focused only on organizational features ( predominantly related to CMAC and F-actin status ) when addressing the question of separation between the modes . As a result , at least four lines of evidence support the discrete nature of Discontinuous and Continuous migration modes . First , supervised multivariate clustering ( canonical vectors analysis , CVA ) of mode-classified cell observations indicated near complete separation between Discontinuous and Continuous modes . The systemic organization underlying these migration modes is therefore , to a very large extent , quantitatively distinct . Second , although separation between modes was not complete when cell observations were clustered in unsupervised PCA space , the topology of probability-density distributions was consistent with the existence of subpopulations in cell state space . Precisely , the high probability-density 'valleys' near the center of mass of each mode contrast with the low probability-density 'ridges' at the boundaries between modes . Importantly , these probability-density distributions are not predetermined outcomes of such PCA clustering , since PCA is unsupervised with respect to mode identity . In fact , these topologies are suggestive of a system comprised of two attractor states ( Huang and Ingber , 2000 ) , whose basins of attraction diverge on either side of a set of potentially unstable or unfavorable configurations . The low probability of these boundary configurations effectively divides mesenchymal migration into two independent sub-modalities . Such attractor states have been described in relation to , for example: gene expression during development and tumorigenesis ( Huang et al . , 2009; Huang , 2009 ) ; cell signaling during motility ( Kim et al . , 2015 ) ; and even in the force-coupled dynamics of CMACs ( Hern\\u00e1ndez-Varas et al . , 2015 ) . Moreover , the attractor state concept appears analogous to the concept of prespecification originally proposed by Friedl to explain the robust , recurrent and yet switchable characteristics of mesenchymal and amoeboid migration modes ( Friedl , 2004 ) . Interestingly , the limited remodelling within Discontinuous and Continuous modes in response to FN modulation ( as opposed to mode switching ) suggests that these modes may be relatively inflexible , providing additional support for their proposed status as prespecified attractor states . Third , further indicating the discrete nature of Discontinuous and Continuous migration modes , CVA clustered cell observations grouped by both migration mode and cell speed were again largely distinct , supporting the validity of the original CVA clustering result . In addition , this approach also provided a fourth compelling line of evidence by revealing completely independent trajectories through cell state space defined by cells migrating at different speeds within each mode . Remarkably , this indicates that changes in cell speed within each mode were coupled to substantially different patterns of organizational remodelling . These systemic differences were composed in part by specific disparities in Spearman\\u2019s correlations between individual organizational features and cell speed within each mode . While a variety of differences were observed , most striking were instances where correlations to cell speed were inverted between modes , such as in the case of adhesion stability ( CMAC lifetime ) \\u2013 a critical parameter previously linked to causal regulation of cell speed ( Lock et al . , 2014 ) . Interestingly , when comparing modes , the independent patterns of cell speed-coupled reorganization arose despite only subtle differences across the entire network of inter-feature Spearman\\u2019s correlations . Indeed , these differences in correlation values were far more limited than would be expected at random , suggesting that a select few features and relationships may control migration mode identity . Taking into account each of these lines of evidence , we conclude that Discontinuous and Continuous migration modes are representative of more than simply cells with different organizational feature values . We propose that these modes in fact represent divergent , yet co-existing strategies of mesenchymal migration . Despite strongly embracing quantitative imaging-based research ( Lock and Str\\u00f6mblad , 2010; Le D\\u00e9v\\u00e9dec et al . , 2010; Masuzzo and Martens , 2015 ) , the cell migration field has seen remarkably little application of automated approaches to the classification of migrating cells . This is in contrast to the study of mitosis , where automated classification has already proven to be a powerful research tool ( Neumann et al . , 2010; Schmitz et al . , 2010 ) . It is likely that this reflects differences in the degree and structure of heterogeneity in these biological processes . For example , mitosis is a relatively stereotyped process - both morphologically and particularly in the terms of the temporal ordering of events . Both characteristics can increase the accuracy of automated classification approaches ( Held et al . , 2010 ) . While cell migration is also more ( e . g . Discontinuous mode ) or less ( e . g . Continuous mode ) a globally time-ordered process , the underlying events appear far less stereotypical . Given these challenges , we have employed a blinded manual classification approach in this study . However , we note that the CVA clustering results \\u2013 even excluding behavioral features ( as herein ) \\u2013 provide evidence that automated classification of migration modes may be feasible in future . This may significantly enhance subsequent quantitative analyses of mode organization , behavior , and regulation . Such automated classification may be in part based on our detailed spatiotemporal analysis of protrusive and retractive membrane dynamics within each mode . This revealed that the major differences between modes were: the size of membrane retraction events; the order and temporal coupling of protrusion and retraction events; and the coordination displayed between protrusive and retractive dynamics . Specifically , cells in the Discontinuous mode produced many very small and occasional very large retraction events , with these retractions tending to lead protrusion , reversing the order detected in Continuous cells . A large time delay also exists between retraction and subsequent protrusion dynamics in Discontinuous cells , and these dynamics are less coordinated ( lower cross-correlation ) . Such substantial differences in membrane dynamics ( especially the inversion of protrusion and retraction orders ) further support the distinct nature of Discontinuous and Continuous migration modes . Furthermore , these analyses both confirmed and explained the visual impressions that originally differentiated Discontinuous from Continuous migration . In addition , given the strong correspondence between retraction ( but not protrusion ) size and cell speed , these analyses now add migration mode and cell speed to motility initiation as aspects of migration where membrane retraction appears to be the deterministic membrane process ( Barnhart et al . , 2015; Kowalewski et al . , 2015; Cramer , 2010 ) . Having quantitatively established the discrete nature and behavioral characteristics of Discontinuous and Continuous migration modes , we sought to understand how their frequency balance is regulated . Importantly , to select candidate molecular regulators , we first considered the differences in organizational feature values that arose spontaneously between the modes . Then , given existing knowledge on how targeting of particular molecular components ( FN concentration [Gupton and Waterman-Storer , 2006]; talin 1 expression [Kiss et al . , 2015]; ROCK signaling [Lock et al . , 2014; Kim and Wirtz , 2013; Hern\\u00e1ndez-Varas et al . , 2015] ) impact these same features , we designed a series of perturbations intended to drive specific changes in these features , while also monitoring the behavioral balance between modes . This demonstrated that each of these molecular factors plays a key role in shaping the equilibrium between modes . Furthermore , these molecular targets also regulate generic characteristics of the pivotal ECM \\u2013 adhesion \\u2013 F-actin axis , including: integrin ligation ( FN concentration ) ; integrin activation and clustering ( talin expression ) ; integrin \\u2013 F-actin linkage ( talin expression ) ; and actomyosin contractility ( ROCK signaling ) . Hence , the effects of these specific perturbations may also be illustrative of how generalized regulatory mechanisms impact the migration mode equilibrium . Taking this perspective , we observe that while Continuous migration arises under conditions of high adhesion and full contractility , Discontinuous migration is preferred given low adhesion and inhibited contractility . It is noteworthy that both modes emerge under conditions that are distinct from those that favour amoeboid migration ( i . e . low adhesion and high contractility ) . Therefore , the switch between Continuous and Discontinuous ( mesenchymal ) migration modes clearly contrasts to the previously identified conversion between mesenchymal and amoeboid migration . This emphasizes the multifaceted adaptive capacity of migrating cells , which likely expands the contexts within which efficient migration is possible . This plasticity is highlighted by the growing variety of distinct migration modalities now recognized ( Figure 7G ) . Finally , by viewing differences in organizational feature values ( both spontaneous and perturbation-induced ) and mode frequencies as a logical matrix , we could identify organizational features that are coupled to migration mode identity under all observed conditions , as well as features that can be explicitly uncoupled . This provides a first step in parsing correlated features from those with significant causal influence over migration mode identity . Specifically , it is notable that CMAC Area and CMAC Lifetime were previously indicated to causally influence cell migration speed ( Lock et al . , 2014; Kim and Wirtz , 2013 ) , while Cell Compactness was causally downstream of cell speed ( Lock et al . , 2014 ) . Thus , the evidence that these features may be functionally coupled to the determination of migration mode identity is consistent with them being integral to migratory regulation . Conversely , we previously found that Cell Area showed no causal relationship to cell speed ( Lock et al . , 2014 ) , and this is again consistent with it being functionally uncoupled from the process of mode identity determination . Overall , three types of insight were revealed through these perturbation-based analyses , including: the regulatory roles of specific molecules ( fibronectin , talin and ROCK ) ; the putative influence of generic regulatory mechanisms; and the nature of links ( coupled vs uncoupled ) between migration mode identity and commonly measured macromolecular features . Nonetheless , much remains to be learned about the proximal and distal determinants of these migration modes . In conclusion , we have presented and applied a quantitative , imaging-based analytic approach to explore the heterogeneity that naturally emerges within the mesenchymal ( lamellipodial ) archetype of cell migration . We thereby characterized two quantitatively distinct migration modes , here termed Continuous and Discontinuous migrations , that co-exist within the broad mesenchymal migration archetype . We compared these migration modes in terms of: their motion and behavioral dynamics; the specific and systemic organization of key machineries ( CMACs and F-actin ) underlying their motility , and; differences in how behavior ( migration speed ) and organization co-evolve within each mode . Through targeted molecular perturbations , we defined both specific molecular and general mechanisms of bottom-up control over migration mode determination , while also beginning to parse potentially functional ( coupled ) relationships ( between features and modes ) from those that are correlated but non-functional ( uncoupled ) . Finally , our systemic analysis of adaptive responses to perturbation has revealed how discrete mechanisms ( inter-modal adaptive switching ) dominate over progressive remodeling ( intra-modal adaptive stretching ) . This study therefore emphasize the importance of distinguishing and comprehending Discontinuous and Continuous migration modes as a necessary precursor to understanding mesenchymal migration in its totality , while simultaneously providing tools and approaches that enable this endeavor . \\n We have integrated a comprehensive and unique combination of analytical capabilities and made them freely available as a Matlab toolbox: 'The Cell Adhesion and Migration Analysis Toolbox' \\u2013 along with the raw quantitative data underlying this study , sample image data to aid implementation , and explanatory documentation ( see doi:10 . 5061/dryad . 9jh6m ) . Briefly , this toolbox includes the following features utilized in this study: Data import and preprocessing based on images and extracted variables ( see Quantitative Image Analysis , below ) ; Smoothing of cell and adhesion trajectories; Membrane protrusion and retraction dynamics extraction; Migration mode classification; Mean square displacement ( MSD ) of cell trajectories; Variable selection for uni- and multivariate analyses; Univariate statitstical analysis; Variable correlation based heat maps; Canonical vector analysis ( CVA ) ; Principal component analysis ( PCA ) ; and two dimensional kernel density estimation . A more detailed description is available within the Matlab Toolbox . All cell lines were acquired directly from the American Type Culture Collection ( ATCC , Manassas , VA ) , and therefore were not further authenticated . The cell lines used herein are not members of the ATCC list of commonly misidentified cell lines . H1299 ( human non-small cell lung carcinoma , ATCC; ATCC# CRL-5803 , mycoplasma negative ) cells stably expressing EGFP-Paxillin and RubyRed-LifeAct , termed H1299 P/L cells , were established and maintained in RPMI 1640 medium ( Gibco \\u2013 Thermo Fisher Scientific , Waltham , MA ) containing 400 \\u00b5g/ml geneticin ( G-418 sulfate , Gibco ) with 10% fetal bovine serum ( Gibco ) and 1 mM glutamine , as described previously ( Lock et al . , 2014; Kiss et al . , 2015 ) . BT549 ( ductal breast carcinoma , ATCC; ATCC# HTB-122 , mycoplasma negative ) cells were maintained in RPMI 1640 medium ( Gibco ) containing 10% fetal bovine serum ( Gibco ) and 1 mM glutamine . MDA-MB-231 ( adenocarcinoma , ATCC; ATCC# HTB-26 , mycoplasma negative ) , Hep-3 ( hepatocellular carcinoma , ATCC; ATCC# HB-8064 , mycoplasma negative ) and Hs578T ( breast carcinoma , ATCC; ATCC# HTB-126 , mycoplasma negative ) cells were maintained in DMEM ( Gibco ) containing 10% fetal bovine serum ( Gibco ) . All cells were incubated at 37\\u00b0C in 5% CO2 . In preparation for imaging , 96-well optical glass-bottomed plates ( Zell-Kontakt , N\\u00f6rten-Hardenberg , Germany ) were coated with ECM ligands , including either collagen type 1 ( Life Technologies \\u2013\\u00a0Thermo Fisher Scientific ) , laminin ( Sigma-Aldrich , St . Louis , MO ) , fibronectin or vitronectin ( both purified from human plasma as described previously [Smilenov et al . , 1992; Yatohgo et al . , 1988] ) . Coating was performed for 2 hr at 37\\u00b0C followed by blocking with 1% heat-denatured bovine serum albumin ( Sigma-Aldrich ) for 1 hr at 37\\u00b0C . ECM ligand coating concentrations were 10 \\u00b5g/ml except where otherwise indicated ( i . e . where fibronectin concentration was varied ) . RNAi-depletion of talin 1 was performed 48 hr prior to imaging or immunoblotting ( with anti-talin [8d4 , 1:500 , Sigma-Aldrich] and anti-tubulin [DM1A , 1:2000 , Fisher Scientific] antibodies as described previously [Kiss et al . , 2015] ) using the following oligonucleotide sequence: ( 5\\u2019 -GAA GAU GGU UGG CGG CAUU- 3\\u2019 ) ( synthesized by GenePharma , Shanghai , P . R . China ) . A non-targeting oligonucleotide sequence was used as control: ( 5\\u2019 -GCG CGC UUU GUA GGA UUCG- 3\\u2019 ) . Transfection of 2 x 104 cells was performed in 24-well plates using 20 pmol of siRNA and 2 \\u00b5l RNAiMAX ( Invitrogen \\u2013 Thermo Fisher Scientific ) according to the manufacturers instructions . Inhibition of Rho kinase ( ROCK ) was performed starting 1 hr prior to imaging using the Y-27632 inhibitor ( Sigma-Aldrich ) at 1 \\u00b5M , 2 \\u00b5M , or 6 \\u00b5M . Di-methyl sulfoxide ( DMSO , Sigma-Aldrich ) was used as control . Live-cell imaging was initiated 2 hr after replating of 2 x 104 cells per well into ECM-coated 96-well glass-bottomed plates ( see coating details above ) . High-resolution , multiscale imaging ( of CMACs , F-actin and cell migration ) was performed on a Nikon A1R confocal microscope using a PlanApo VC 60X/1 . 4 NA oil-immersion objective ( Nikon , Amsterdam , Netherlands ) . Images were acquired for 8\\u201310 hr at 5 min intervals with a pixel resolution of 0 . 21 \\u00b5m . Long-term ( 16 hr at 5-min intervals ) , low resolution imaging to monitor for inter-modal transitions was performed on a Nikon A1R confocal microscope using a Plan Apo \\u03b3 20X/0 . 75 NA air objective and resonance scanner , producing a pixel resolution of 0 . 82 \\u00b5m . 3\\u00d73 or 4\\u00d74 image montages were acquired and stitched . Prior to imaging , cells were labeled at 1:5000 with a far-red membrane dye ( Cell Mask Deep Red , Thermo Fisher Scientific ) for 2 hr and throughout imaging . Normal culture medium minus fetal bovine serum was used during all imaging , and cells were maintained at 37\\u00b0C in 5% CO2 . Multiscale image sequences ( Nikon ND2 files ) were analyzed using PAD software ( Digital Cell Imaging Laboratories , Keerbergen , Belgium ) , as described previously ( Lock et al . , 2014 ) . Briefly , cell boundaries were detected using the RubyRed-LifeAct signal . Cells touching the image border in any given time frame were excluded . Within each segmented cell , CMACs greater than 0 . 05 \\u00b5m2 in area were segmented based on EGFP-Paxillin signal . Segmented cells and CMACs were then tracked based on nearest neighbor analysis . Cell trajectories were smoothed using smoothing splines . CMACs were excluded where tracking did not contain at least three consecutive time points . CMACs present in the first and last frames of each image sequence were excluded from the calculation of CMAC Lifetime . All segmentation and tracking was manually inspected and iteratively optimized through parameter tuning . Quantitative features characterizing cell , CMAC , and CMAC-associated F-actin properties were then extracted ( Figure 3\\u2014figure supplement 1 ) . CMAC intensity values per channel were corrected by subtraction of mean intensity values in a 1 \\u00b5m radius around the CMAC , excluding other segmented CMACs . CMAC intensity data were further standardized per experimental repeat relative to the median value of CMAC intensities in the size range between 0 . 15 and 0 . 2 \\u00b5m2 within the low FN no INH condition . Cell motion was characterized by instantaneous cell speed or via analysis of mean squared displacement ( MSD ) . Mean square displacement ( MSD ) was calculated using a moving window as previously described ( Khorshidi et al . , 2011 ) . The window size was set to 24 frames ( 2 hr ) . The first 12 time lags ( tlag ) , up to 1 hr , of the MSD were fitted to Furth\\u2019s formula: MSD=4M ( tlag\\u2212tp ( 1\\u2212e\\u2212tlag/tp ) ) , thus measuring the migration coefficient M and the persistence time tp at each time point . The observations were divided into quintiles of migration coefficient and within each such quintile the mean persistence time for each trajectory was calculated . For visualization of parallel mode emergence ( Figure 1\\u2014figure supplement 1 and Video 3 ) and mode transitions ( Figure 1\\u2014figure supplement 2 and Videos 4 , 5 ) , long-term low-resolution image data was segmented and tracked ( following advanced denoising , power 15 ) using a custom general analysis pipeline in NIS-Elements ( v4 . 30 , Nikon ) . Collectively , data sets for comparison of fibronectin concentration , talin 1 expression and ROCK inhibition had structures as follows: 2 . 5 \\u00b5g/ml FN ( 3 experimental repeats , 34 cells , 2525 cell observations , 29651 CMAC observations ) vs 10 \\u00b5g/ml FN ( 19 experimental repeats , 118 cells , 6528 cell observations , 213406 CMAC observations ) ; talin 1 siRNA ( 4 experimental repeats , 101 cells , 6263 cell observations , 81675 CMAC observations ) vs control siRNA ( 4 experimental repeats , 51 cells , 3154 cell observations , 131679 CMAC observations ) ; High-resolution , multiscale imaging of 6\\u00a0\\u00b5M Y27632 ( ROCK inhibitor , 4 experimental repeats , 17 cells , 985 cell observations , 44647 CMAC observations ) vs DMSO control ( 10 experimental repeats , 34 cells , 2666 cell observations , 86794 CMAC observations ) ; long-term , low resolution imaging of 6\\u00b5M Y27632 ( 4 experimental repeats , 73 cells , 4478 cell observations ) vs 2 \\u00b5M Y27632 ( 4 experimental repeats , 71 cells , 5198 cell observations ) vs 1 \\u00b5M Y27632 ( 4 experimental repeats , 69 cells , 4885 cell observations ) vs DMSO control ( 4 experimental repeats , 83 cells , 5968 cell observations ) . Note that the term 'experimental repeats' refers to independent biological repeats , not technical replicates . Data sample sizes were selected based on prior experience from related multiscale analyses in several previous studies ( Lock et al . , 2014 ) . Samples were collected and analyzed without iteration , that\\u00a0is , all relevant available data were used and sample sizes were not altered to increase or decrease statistical power . Cells were classified into Discontinuous or Continuous migration modes according to the behavioral criteria described in Figure 1 based on visual inspection and optimization between two individuals , using a custom platform established in Matlab ( vR2013b , The Mathworks , Natick , MA ) . Single cells were assessed at each time point within each image sequence ( inter-mode transitions , though infrequent , were assessed ) . All data presented herein were assessed concurrently and were both computationally blinded and randomized with respect to experimental conditions . Dead cells and cells in sustained contact were classified as null and excluded from subsequent analyses . Membrane protrusion and retraction dynamics were quantified as described previously ( Kowalewski et al . , 2015 ) . Briefly , consecutive segmented images were compared to define protrusions ( new pixels in cell segment compared to previous frame ) and retractions ( pixels lost from cell segment compared to previous frame ) . To explore dynamics over different time frames , images separated by different intervals were assessed using the same criteria , establishing time window samplings of between 5 min ( 1 frame ) and 75 min ( 15 frames ) . Subtraction of protrusion and retraction size probability distributions was performed as described previously for other feature probability distributions ( Kiss et al . , 2015 ) . Analysis of cross-correlation between protrusion and retraction areas over time was performed on a per cell sequence basis using the Matlab function xcov . Missing values were replaced by the mean value of protrusion/retraction area for that cell trajectory . Time lags of between \\u221260 min ( \\u221212 frames ) and 60 min ( +12 frames ) were assessed , with negative lags indicating that protrusion dynamics lead retraction dynamics , and positive lags indicating that retraction dynamics lead protrusion dynamics . To summarize the mean absolute cross-correlation per migration mode , cross-correlation values at all lags ( \\u221260 to 60 min; \\u221212 to 12 frames ) were averaged for each time window ( 5 min to 75 min; 1 to 15 frames ) , per cell , and then averaged per condition . Nienty-five\\u00a0percent confidence intervals were calculated per time window size . Friedman testing based on distinct cell numbers defined statistically discernable differences in mean protrusion-retraction cross-correlation between modes across the sampled time windows . All membrane dynamics analyses were performed in Matlab ( vR2013b , The Mathworks ) . Supervised multivariate clustering via canonical vectors analysis ( CVA ) was performed based on decompositions of the between-group and within-group covariance matrices following eigenvalue decomposition , as previously described ( Lock et al . , 2014 ) . Feature selection via CVA was performed by ranking absolute feature weighting coefficients from the first canonical vector ( CV1 ) . Unsupervised multivariate clustering using principal component analysis ( PCA ) was performed using singular value decomposition of the normalized data matrix . Probability density surfaces were calculated using a Gaussian kernel density estimation of significantly smaller scale that the overall mode distributions , and hence does not dictate the 'valley' shape of these distributions \\u2013 rather this is a reflection of the true value distribution . Spearman\\u2019s rank correlation coefficients were calculated based on feature values per cell observation . Estimation of the expected distribution of random absolute value changes in Spearman\\u2019s correlation between modes was performed by permuting the identity of correlation values in one mode while holding the other constant . This was repeated 100 times and the cumulative distribution function of all differences is displayed . Pairwise testing for differences in feature distributions ( boxplots , parallel coordinates ) was performed using the Wilcoxon rank sum test ( equivalent to Mann-Whitney test ) given all cell observations . Analyses and visualizations described above were performed in Matlab ( The Mathworks ) . Parallel coordinates-based visualization of selected feature values in each migration mode was generated using Knime ( v2 . 12 . 00 , KNIME . com , Zurich , Switzerland ) . Parallel coordinates-based comparison of Spearman\\u2019s correlation coefficients between Cell Speed and selected features , as well as visualization of coefficient differences between modes , were generated using R ( v3 . 2 . 0 , R Foundation for Statistical Computing ) and RStudio ( v0 . 98 . 945 , RStudio , Boston , MA ) .\",\n          \"The Dna2 nuclease-helicase maintains genomic integrity by processing DNA double-strand breaks , Okazaki fragments and stalled replication forks . Dna2 requires ssDNA ends , and is dependent on the ssDNA-binding protein Rpa , which controls cleavage polarity . Here we present the 2 . 3 \\u00c5 structure of intact mouse Dna2 bound to a 15-nucleotide ssDNA . The nuclease active site is embedded in a long , narrow tunnel through which the DNA has to thread . The helicase domain is required for DNA binding but not threading . We also present the structure of a flexibly-tethered Dna2-Rpa interaction that recruits Dna2 to Rpa-coated DNA . We establish that a second Dna2-Rpa interaction is mutually exclusive with Rpa-DNA interactions and mediates the displacement of Rpa from ssDNA . This interaction occurs at the nuclease tunnel entrance and the 5\\u2019 end of the Rpa-DNA complex . Hence , it only displaces Rpa from the 5\\u2019 but not 3\\u2019 end , explaining how Rpa regulates cleavage polarity . \\n Dna2 has nuclease and helicase activities and plays key roles in maintaining genomic integrity . It is involved in the nucleolytic processing of 5\\u2019 flaps during Okazaki fragment maturation , of DNA double-strand breaks ( DSBs ) during homologous-recombination mediated repair , and of stalled replication forks in the intra-S-phase checkpoint ( Bae et al . , 2001; Cejka et al . , 2010; Hu et al . , 2012; Nimonkar et al . , 2011; Zhu et al . , 2008 ) . Dna2 was first identified as a replication mutant required for viability in yeast , and was subsequently shown to be involved in trimming long 5\\u2019 RNA-DNA flaps from Okazaki fragments during replication ( Budd et al . , 1995 ) . Most flaps are cleaved by Fen1 concomitant with their generation by strand displacement during Pol \\u03b4 synthesis on the lagging strand . Flaps that escape early cleavage , or those extended by the Pif1 helicase , get long enough to be coated by the Replication Protein A ( Rpa ) , which renders them resistant to Fen1 ( Bae et al . , 2001; Pike et al . , 2009; Stith et al . , 2008 ) . Dna2 , which can displace Rpa from ssDNA ( Stewart et al . , 2008 ) , trims the flap to a length too short for stable Rpa binding , and restores Fen1 processing ( Ayyagari et al . , 2003; Bae et al . , 2001; Gloor et al . , 2012 ) . The lethality of Dna2 deletion in yeast is attributed to persistent Rpa-coated flaps , which recruit Ddc2 ( ATRIP in metazoa ) and activate the Mec1 ( ATR in metazoa ) DNA-damage checkpoint ( Chen et al . , 2013; Zhu et al . , 2008 ) . Long-flap processing by Dna2 is dependent on Rpa removing ssDNA secondary structure ( Stewart et al . , 2008 ) , an essential Rpa function in many other aspects of DNA metabolism ( Chen et al . , 2013; Fanning et al . , 2006; Symington and Gautier , 2011 ) . Also paralleling other Rpa-dependent processes , the ability of Dna2 to act on Rpa-coated ssDNA is dependent on direct Dna2-Rpa interactions ( Bae et al . , 2003 , 2001 ) . The yeast Dna2\\u25b3405N mutation that reduces Rpa binding also reduces 5\\u2019 flap cleavage and DSB resection in vitro , and renders yeast temperature-sensitive for growth ( Bae et al . , 2003 , 2001; Niu et al . , 2010 ) . In DSB resection , Dna2 acts redundantly with Exo1 ( Gravel et al . , 2008; Mimitou and Symington , 2008; Zhu et al . , 2008 ) . Resection of the 5\\u2019 terminated DNA strand results in a long track of 3\\u2019 overhang ssDNA , which forms a nucleoprotein filament with the Rad51 strand-exchange protein and initiates homologous recombination ( Symington and Gautier , 2011 ) . In vitro , Dna2 , Rpa and the helicase Sgs1 ( BLM in mammals ) constitute the minimal complex that can carry out long-range resection . Resection is dependent on the nuclease activity of Dna2 and the helicase activity of Sgs1/BLM . Rpa is essential for supporting the helicase activity of Sgs1/BLM in part by sequestering the unwound strands , and also for regulating Dna2 by blocking its 3\\u2019 to 5\\u2019 exonuclease activity ( Cejka et al . , 2010; Nimonkar et al . , 2011; Niu et al . , 2010 ) . In cells , Rpa depletion eliminates long-range DSB resection , and also causes the loss or inappropriate annealing of short 3\\u2019 ends generated by Mre11 ( Chen et al . , 2013 ) . In addition to these functions , Dna2 is implicated in preventing the regression of stalled replication forks , which otherwise can generate aberrant structures resembling recombination intermediates and lead to genomic instability ( Hu et al . , 2012 ) . This is dependent on the Dna2 nuclease activity , consistent with the ability of Dna2 to cleave fork structures with regressed leading or lagging nascent strands in vitro ( Hu et al . , 2012 ) . Dna2 contains a PD- ( D/E ) XK superfamily nuclease motif ( Budd et al . , 2000 ) and a 5\\u2019 to 3\\u2019 helicase domain ( Bae and Seo , 2000 ) . It is a ssDNA endonuclease that requires a free end for cleavage , and does not cleave dsDNA , single-stranded gaps , D-loops or RNA ( Bae and Seo , 2000; Kao et al . , 2004 ) . In vitro , isolated Dna2 cleaves ssDNA starting at either end , with multiple rounds of incision degrading ssDNA in both the 5\\u2019 to 3\\u2019 and 3\\u2019 to 5\\u2019 directions ( Bae and Seo , 2000; Masuda-Sasa et al . , 2006 ) . With 5\\u2019 flap DNA , cleavage starts ~\\u223c10 nucleotides ( nts ) from the ssDNA end and continues to within \\u223c5 nts of the duplex ( Bae et al . , 2001; Bae and Seo , 2000; Cejka et al . , 2010; Gloor et al . , 2012; Masuda-Sasa et al . , 2006 ) . It has been suggested that Dna2 loads at the free 5\\u2019 end of the flap and tracks in the 5\\u2019 to 3\\u2019 direction ( Kao et al . , 2004 ) . The helicase and ATPase activities of Dna2 are substantially weaker than those of other helicases ( Budd et al . , 2000; Masuda-Sasa et al . , 2006 ) . The helicase activity can be increased by a high ATP to Mg2+ ratio , but this also inhibits the nuclease activity through ATP sequestering Mg2+ ( Bae and Seo , 2000; Masuda-Sasa et al . , 2006 ) . The 5\\u2019 to 3\\u2019 polarity of the helicase translocation could , in principle , drive the tracking of Dna2 along the 5\\u2019 flap . However , ATPase mutations have a minimal effect on 5\\u2019 flap processing and DSB resection in vitro ( Cejka et al . , 2010; Niu et al . , 2010; Zhu et al . , 2008 ) . And , in vivo , ATPase-inactive yeast Dna2 mutants are viable , although they exhibit impaired growth ( Budd et al . , 2000 ) . To understand the mechanism of action and regulation of this multi-faceted enzyme , we first determined the structure of the intact Dna2-ssDNA complex . The structure revealed that the ssDNA has to thread through a tunnel to bind to Dna2 , with a polarity that precludes the cooperation of the helicase and nuclease activities . The requirement for DNA threading prompted us to investigate how Dna2 gains access to Rpa-coated DNA . We provide the structure of a complex between a Dna2 \\u03b1 helix and the Rpa70 OBN domain , both of which are flexibly-tethered and likely serve to recruit Dna2 to Rpa . We also establish a second Dna2-Rpa interaction that helps to displace Rpa from the 5\\u2019 DNA end , explaining how Rpa restricts the cleavage polarity of Dna2 . \\n We determined the 2 . 3\\u2009\\u00c5 crystal structure of full-length mouse Dna2 ( residues 1 to 1062 ) bound to a ssDNA substrate of 21 nucleotides ( nts ) , 15 of which are well ordered ( Figure 1A and Table 1 ) . The structure also contains an Fe4-S4 iron-sulfur cluster , ADP and two active-site Ca2+ ions , which do not support nuclease activity but can mimic magnesium coordination ( Yang et al . , 2006 ) . 10 . 7554/eLife . 09832 . 003Figure 1 . Structure of the Dna2-ssDNA complex . ( A ) Cartoon representation of the Dna2-ssDNA complex . The \\u03b11 helix , which packs with the hel2A domain of a crystallographic symmetry related protomer , is omitted from this view . The individual domains of Dna2 are colored separately as indicated in ( D ) , ssDNA is red , ADP is shown as sticks , calcium ions are shown as green spheres , the iron-sulfur cluster is shown in a space-filling representation ( nuc: nuclease domain; hel1A: helicase 1A domain , hel2A: helicase 2A domain , OB: oligonucleotide/oligosaccharide binding domain ) . ( B ) View looking up the vertical axis of ( A ) . ( C ) Schematic of the complex illustrating the relative arrangement of the Dna2 domains , and highlighting its cylinder-like shape . Colored as in ( A ) . ( D ) Linear representation of the Dna2 domains and their boundaries; colored as in ( A ) . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 00310 . 7554/eLife . 09832 . 004Figure 1\\u2014figure supplement 1 . Dna2 secondary structure\\u00a0and sequence conservation . Helices are indicated as cylinders , b strands as arrows , segments lacking regular secondary structure as solid lines , and disordered regions as dashed lines . Secondary structure elements are colored as in Figure 1A . The N-terminal extensions , relative to mammalian Dna2 , of the other orthologs are not shown . In the alignment of yeast Dna2 , reliable homology starts with the \\u03b22 strand of the OB domain at residues 44 and 417 of the mouse and yeast orthologs , respectively . The ~360 residue , unique N-terminal segment of yeast Dna2 has an overall low hydrophobicity content indicative of lack of globular structure , and short regions of homology with closely related yeasts suggests that this region acts through linear epitopes , some of which may be acidic/amphipathic helices ( not shown ) . Residues that contact the DNA are marked by \\u201cd\\u201d , active site residues by \\u201cc\\u201d and iron-sulfur cluster ligands by \\u201cf\\u201d . The entire protein structure is within 0 . 9 \\u00c5 C\\u03b1 r . m . s . d . in DNA-bound and apo Dna2 crystals , and the overall DNA conformation is essentially identical in a different crystal form of Dna2 bound to a 21-nt ssDNA ( Table 1 ) . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 00410 . 7554/eLife . 09832 . 005Figure 1\\u2014figure supplement 2 . Dna2 inter-domain interfaces . ( A ) The nuclease domain packs with the OB domain extensively , through both polar and hydrophobic residues . The four residues between the last b strand of the OB domain and the first b strand of the nuclease domain are buried in the interface between the two domains . The OB groove , where DNA or peptides bind in other OB fold proteins , is solvent exposed opposite from its surface that packs with the nuclease domain . Individual domains are colored as in Figure 1A . ( B ) The Dna2 nuclease core fold has a three-helix insertion ( residues 176 to 227 ) that packs with the helicase 1B domain . This three-helix insertion is also present in the AddB nuclease , where it packs with the N-terminal portion of the protein . ( C ) The nuclease domain also links to the helicase domain indirectly , through the stalk domain that is sandwiched between the two . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 00510 . 7554/eLife . 09832 . 006Table 1 . Data collection and refinement statistics . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 006Data SetDna2-5\\u2019 overhang DNA*Dna2-ssDNAapo Dna2apo Dna2 ( SeMet ) DNA2 \\u03b11-RPA 70NSpace groupP22121P212121P212121P212121C121a , b , c ( \\u00c5 ) 87 . 2 , 118 . 5 , 149 . 3120 . 2 , 149 . 2 , 172 . 9120 . 9 , 148 . 6 , 170 . 5120 . 9 , 148 . 6 , 170 . 5134 . 3 , 50 . 9 , 76 . 5\\u03b1 , \\u03b2 , \\u03b3 ( \\u00b0 ) 90 . 0 , 90 . 0 , 90 . 090 . 0 , 90 . 0 , 90 . 090 . 0 , 90 . 0 , 90 . 090 . 0 , 90 . 0 , 90 . 090 . 0 , 103 . 9 , 90 . 0Resolution ( \\u00c5 ) 50 . 0 - 2 . 35 ( 2 . 43 - 2 . 35 ) 30 . 0 \\u2013 3 . 11 ( 3 . 23 - 3 . 11 ) 50 . 0 \\u2013 3 . 0 ( 3 . 11 - 3 . 0 ) 30 . 0 \\u2013 3 . 4 ( 3 . 52 - 3 . 4 ) 60 . 0 \\u2013 1 . 5 ( 1 . 55 - 1 . 5 ) Rsym12 . 6 ( 65 . 6 ) 13 . 7 ( 87 . 5 ) 13 . 1 ( 78 . 7 ) 15 . 8 ( 59 . 3 ) 7 . 4 ( 80 . 1 ) Rpim6 . 3 ( 36 . 6 ) 6 . 6 ( 57 . 1 ) 6 . 8 ( 41 . 9 ) 4 . 0 ( 15 . 1 ) 5 . 1 ( 54 . 1 ) I/\\u03c3 ( I ) 16 . 4 ( 1 . 9 ) 13 . 2 ( 1 . 4 ) 7 . 9 ( 1 . 3 ) 19 . 3 ( 4 . 7 ) 23 . 5 ( 3 . 2 ) Completeness ( % ) 99 . 0 ( 98 . 6 ) 99 . 0 ( 99 . 2 ) 98 . 3 ( 98 . 9 ) 100 . 0 ( 100 ) 84 . 3 ( 79 . 7 ) Redundancy5 . 0 ( 3 . 9 ) 6 . 1 ( 6 . 0 ) 4 . 4 ( 4 . 5 ) 15 . 1 ( 15 . 3 ) 2 . 7 ( 2 . 7 ) RefinementResolution ( \\u00c5 ) 50 . 0-2 . 3630 . 0\\u20133 . 1550 . 0\\u20133 . 030 . 0-1 . 55No . of reflections57 , 87151 , 12254 , 10757 , 455Rwork/Rfree ( % ) 20 . 8/24 . 622 . 3/25 . 621 . 1/24 . 523 . 2/26 . 7Protein atoms8 , 29816 , 53616 , 5363 , 933DNA atoms29067400Cofactor atoms4170700Rmsd bond lengths ( \\u00c5 ) 0 . 0090 . 0090 . 0090 . 007Rmsd bond angles ( \\u00b0 ) B factors ( \\u00c52 ) :proteinDNACa2+waterWilson1 . 4\\u200365 . 5107 . 658 . 949 . 558 . 41 . 41 . 461 . 3*Only the ssDNA is ordered . Values in parentheses are for the highest-resolution shell . The structure consists of a \\u223c310 residue domain that contains the PD- ( D/E ) XK nuclease motif , followed by a \\u223c450 residue , C-terminal helicase domain that has two RecA-like folds characteristic of the SF1 helicase subfamily ( domains 1A and 2A; Figure 1A , B ) . In addition , the structure reveals an OB ( oligonucleotide/oligosaccharide-binding ) fold domain N-terminal to the nuclease domain , and a \\u03b2 barrel domain that occurs between the nuclease and helicase domains and which is held in place by a stalk of two long alpha helices ( Figure 1 and Figure 1\\u2014figure supplement 1A ) . The overall structure has a cylindrical shape with a central tunnel through which the ssDNA threads ( Figure 1A , B ) . The base of the cylinder is formed by the nuclease domain , which adopts a doughnut-like structure with the active site embedded in the central tunnel ( Figure 1C ) . The \\u03b2 barrel and helicase 1A domains pack on top of the nuclease doughnut and extend the cylinder and central tunnel . The helicase 2A domain , which packs with the 1A domain as in the ADP states of other SF1 helicase structures , hangs over the tunnel opening at the top of the cylinder . Most of the DNA-binding sites of the nuclease and helicase 1A domains are inside the tunnel , whereas those of the helicase 2A domain are solvent exposed . The OB domain decorates the exterior of the nuclease domain , and is uninvolved in DNA binding . The nuclease domain is the hub that organizes the overall structure . It packs with the flanking OB and \\u03b2 barrel-stalk domains , as well as the helicase 1B domain ( Figure 1C and Figure 1\\u2014figure supplement 2 ) . The ssDNA is positioned with its 5\\u2019 end at the helicase domain , and its 3\\u2019 end at the nuclease domain ( Figure 1A ) . A 7-nt segment at the 5\\u2019 end contacts first the helicase 2A domain outside the tunnel and then the 1A domain inside the tunnel . The DNA crosses over from the helicase to the nuclease domains over the next two nucleotides , which are in the vicinity of the \\u03b2 barrel domain in the middle of the tunnel , but do not make any protein contacts ( Figure 1A ) . The subsequent 6-nt segment binds to the nuclease domain . Here , the first three nucleotides are fully enclosed by the tunnel , while the last three nucleotides contact the tunnel opening . The DNA bases stack continuously , except for a base step at the helicase , one at the nuclease and one at the transition between the two domains ( Figure 2C ) . Two of the three unstacked base steps are at pyrimidine-pyrimidine pairs , and this may contribute to the DNA binding at a well-defined register . An N-terminal acidic/amphipathic \\u03b1 helix ( \\u03b11; residues 1 to 13 ) packs with the helicase domain of a symmetry-related complex . The \\u03b11 helix is flexibly tethered to the rest of the protein , as the 6 residues that connect it to the OB domain have no electron density in the crystals . As shown later , this helix is one of the two Rpa-binding elements of Dna2 . 10 . 7554/eLife . 09832 . 007Figure 2 . Nuclease and helicase domain structures and DNA contacts . ( A ) Superposition of the Dna2 nuclease domain on the AddB nuclease domain . The N-terminal \\u03b2\\u03b1\\u03b1 extension ( residues 122 to 154 ) and the C-terminal \\u03b1\\u03b1 extension ( residues 384 to 412 ) is colored in orange , with the corresponding elements of AddB in dark cyan . Green spheres are calcium ions . ( B ) DNA contacts and active site residues of the Dna2 nuclease domain . Hydrogen bonds are depicted as green dotted lines , calcium ions as blue spheres , water as a red sphere . ( C ) Diagram showing the contacts depicted in ( B ) and ( E ) . The residues are colored according to the domain they belong as in Figure 1D ( mc: main chain , sc: side chain ) . ( D ) Superposition of the Dna2 helicase domain on Ighmbp2 . Dna2 is colored as in Figure 1D . The Ighmbnp2 1A ( hel1A ) and 2A ( hel2A ) helicase domains are colored gold , its \\u03b2 barrel in light orange , and stalk dark orange . ( E ) DNA contacts of the helicase 1A ( pink ) and 2A ( cyan ) domains , showing residues that are involved in either hydrogen bond ( green dotted lines ) or van der Waals contacts . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 00710 . 7554/eLife . 09832 . 008Figure 2\\u2014figure supplement 1 . Electron density at the nuclease active site and structural similarity of the Dna2 helicase to Upf1-subfamily RNA/DNA helicases . ( A ) Top panel shows a stereo view of the mFo-DFc electron density at the active site , calculated with phases after omitting the two calcium ions ( blue spheres ) , the water molecule ( red sphere ) , a 3-nt segment centered on the scissile phosphate group and the side chain of Lys301 , and subjecting the rest of the entire structure to simulated annealing from 3000\\u00b0K . The dark blue map is contoured at 5 \\u03c3 , and the light green map at 2 . 5 \\u03c3 . The orientation is similar to that of Figure 2B . For comparison , the corresponding residues of the \\u03bb nuclease-DNA complex ( Zhang et al . , 2011 ) are shown as black sticks ( the lysine is mutated to alanine in that structure ) , and the calcium ions and water molecule as black spheres . The Dna2 Ca-1 ion and water molecule have a coordination shell and position very similar to the first magnesium and associated water in \\u03bb nuclease , but the position of Ca-2 is different . The Ca coordination shell and residue labels are shown in bottom panel . For clarity , not shown are the electron density , \\u03bb nuclease , and the His164 side chain to which the green-dotted bond from Ca-1 would be connected . ( B ) Superposition of Dna2 helicase domains 1A ( pink ) and 2A ( cyan ) with Ighmbp2 ( gold ) . DNA is colored in red , RNA is colored in green . The only regions of the Dna2 helicase that do not have counterparts in the Upf1 subfamily are the ~40 residue 1B domain , which packs with the nuclease domain , and the ~20 residue 2B domain , which is distal from the rest of the protein and the DNA ( Figure 1D ) . ( C ) Superposition of individual Dna2 helicase domains 1A ( pink ) and 2A ( cyan ) with Upf1 ( gold ) . DNA is colored in red , RNA is colored in green . Motif III residues are shown as sticks . Supplemental discussion of helicase translocation . Based on the proposed SF1B translocation mechanism ( Saikrishnan et al . , 2009 ) , ATP binding and concomitant closure of the cleft between the 1A and 2A domains would be coupled to domain 2A releasing the DNA and rebinding it at a register shifted by 1 nt in the 5\\u2019 to 3\\u2019 direction , while domain 1A would retain its DNA . On ATP hydrolysis , domain 1A would be the one releasing its DNA and rebinding it at the +1 nt register , restoring the initial open-cleft conformation . In the Dna2-ssDNA structure , however , the rebinding of 1A to DNA at the shifted register will have to await the transient dissociation of the nuclease domain from DNA . The increased half life of the closed cleft would likely also reduce the rate of exchange of ADP for ATP , as the \\u03b3 phosphate binding site of the helicase fold is far less solvent accessible in the closed cleft . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 008 As predicted , the Dna2 nuclease domain contains the core \\u03b1\\u03b2\\u03b2\\u03b2\\u03b1\\u03b2 fold of the PD- ( D/E ) XK nuclease superfamily ( Aravind et al . , 2000; Pingoud et al . , 2005 ) . Based on the DALI server , its closest structural homologs are the bacteriophage \\u03bb exonuclease and the E . coli RecB and B . subtilis AddB nucleases , all of which are involved in the resection of DNA ends during homologous recombination ( Krajewski et al . , 2014; Singleton et al . , 2004; Zhang et al . , 2011 ) . These superimpose on the\\u00a0\\u223c310-residue Dna2 nuclease domain with a \\u223c2\\u2009\\u00c5 root-mean-square deviation ( r . m . s . d . ) in the C\\u03b1 positions of 128 , 112 and 106 residues , respectively . The most extensive similarity is exhibited by AddB , which shares with Dna2 the presence of the iron-sulfur cluster and most of the insertions and extensions that decorate the core fold ( Figure 2A ) ( Krajewski et al . , 2014; Yeeles et al . , 2009 ) . While most of these elements have diverged beyond a \\u223c2\\u2009\\u00c5 r . m . s . d . , their secondary structures , arrangement and structural implications are closely related . In particular , both proteins have a \\u03b2\\u2013\\u03b1\\u2013\\u03b1 N-terminal extension and an \\u03b1\\u2013\\u03b1 C-terminal extension that are stapled together by the iron-sulfur cluster ( Figure 2A ) . This iron-sulfur cluster domain supports a loop that crosses over the catalytic channel and converts it to a tunnel through which the DNA has to thread . \\u03bb exonuclease and RecB also have a crossover loop , but their N- and C-terminal extensions that anchor it are structurally divergent from Dna2 , and , more significantly , they lack the iron-sulfur cluster ( Singleton et al . , 2004; Zhang et al . , 2011 ) . The Dna2 nuclease domain binds to a total of four phosphodiester groups , two before and two after the scissile phosphate group ( Figure 2B , C ) . The scissile phosphate group contacts two calcium ions , one ( Ca-1 ) through a non-bridging oxygen , and another ( Ca-2 ) through the 3\\u2019 bridging oxygen of the preceding base . The Ca-1 ion has an octahedral coordination shell very similar to other PD- ( D/E ) XK nucleases ( Pingoud et al . , 2005; Zhang et al . , 2011 ) . It\\u2019s formed by the side chains of His164 , Asp278 and Glu299 , the Leu300 main chain carbonyl group , the non-bridging oxygen atom of the scissile phosphate group , and a water molecule , which also hydrogen bonds to Lys301 and is positioned for nucleophilic attack on the scissile phosphate ( Figure 2B and Figure 2\\u2014figure supplement 1A ) . Lys301 is buttressed by Gln314 , a motif IV residue characteristic of the RecB and \\u03bb exonuclease families ( Aravind et al . , 2000 ) . The relative position of the second , Ca-2 ion differs from other nucleases ( Figure 2\\u2014figure supplement 1A ) , and it is not clear whether it reflects the use of calcium instead of magnesium , or a divergent aspect of the nuclease mechanism of Dna2 . The base groups of the 6 nt segment stack in two sets of three , with Met157 and other crossover loop side chains wedging in between the Ade12 and Gua13 base groups ( Figure 2B ) . This is very similar to the \\u03bb exonuclease crossover loop , which marks the transition from double stranded to single stranded and is thought to play a key role in unwinding dsDNA , although \\u03bb exonuclease does so without a helicase domain and in the context of a homo-trimeric assembly that contacts the dsDNA of the substrate ( Zhang et al . , 2011 ) . Nevertheless , Met157 and the crossover loop may have an analogous function in the weak strand-separating activity of Dna2 . Dna2\\u2019s combination of the \\u03b2 barrel , stalk and helicase domains , and their relative arrangement are strikingly similar to RNA/DNA helicases of the Upf1 subfamily , which contains Upf1 , Ighmbp2 and Senataxin ( Fairman-Williams et al . , 2010 ) . The \\u223c600-residue assembly of the stalk , \\u03b2 barrel and helicase 1A and 2A domains can be superimposed on the structure of Ighmbp2 with a 1 . 5\\u2009\\u00c5 C\\u03b1 r . m . s . d . over 452 residues , with most of the non-superimposing residues accounted for by a \\u223c10\\u00b0 rotation of the \\u03b2 barrel , which otherwise is structurally conserved ( Figure 2D ) . The structural similarity between Dna2 and Ighmbp2 is actually more extensive than that between Ighmbp2 and Upf1 ( 1 . 8 C\\u03b1 \\u00c5 r . m . s . d for 436 residues ) ( Chakrabarti et al . , 2011; Lim et al . , 2012 ) . The Upf1-like subfamily is one branch of the SF1B family of helicases that translocate in the 5\\u2019 to 3\\u2019 direction ( Fairman-Williams et al . , 2010 ) . The other SF1B branch , represented by the DNA helicase RecD2 ( Saikrishnan et al . , 2009 ) , lacks the barrel and stalk domains . Furthermore , the individual RecD2 1A and 2A domains are not as similar to Dna2 , with alignment r . m . s . d . values of 1 . 8\\u2009\\u00c5 for 134 residues and 1 . 9\\u2009\\u00c5 for 97 residues , respectively , in contrast to the Dna2-Ighmbp2 alignment , where the corresponding values are 1 . 5\\u2009\\u00c5 for 204 residues and 1 . 4\\u2009\\u00c5 for 183 residues , respectively . Together , these structural observations suggest that Dna2 evolved by incorporating an ancestral Upf1-family helicase . DNA binds to Dna2 through both its phosphodiester and base groups ( Figure 2E ) . Dna2-base interactions include Van der Waals contacts from the motif III loop ( Leu795 and Val797 ) , which wedges in between the bases of the last two 2A-bound nucleotides . These contacts are consistent with the proposed role of motif III in preventing DNA sliding during translocation of SF1B helicases ( Saikrishnan et al . , 2009 ) . Dna2-phosphodiester interactions involve protein pockets that are rich in backbone amide and short-side chain hydroxyl groups ( Figure 2E ) . Three consecutive phosphodiester groups , near the 5\\u2019 end of the DNA , bind to the 2A domain ( Figure 2C , E ) . The fourth phosphodiester group is in the cleft between the 2A and 1A domains and does not contact the protein , while the fifth and sixth phosphodiester groups bind to pockets on the 1A domain ( Figure 2C ) . Contacts to the ribose groups are minimal , and they are consistent with the helicase accommodating the 2\\u2019 hydroxyl group of RNA . In fact , the DNA contacts as well as the phosphodiester backbone conformation are very similar to those of the Ighmbp2-RNA , Upf1-RNA complexes ( Figure 2\\u2014figure supplement 1B , C ) . This is in contrast to the RecD2-ssDNA structure , where extensive aromatic and van der Waals contacts with the sugar are thought to discriminate against RNA ( Saikrishnan et al . , 2009 ) . The helicase and ssDNA-dependent ATPase activities of Dna2 are considerably weaker than other helicases ( Budd et al . , 2000; Masuda-Sasa et al . , 2006 ) . One possible explanation , at least for the low ATPase/translocation rate , is the helicase domain being 5\\u2019 to the nuclease domain on the DNA . This would , in principle , make completion of the ATPase/translocation cycle dependent on the nuclease domain releasing its grip on the DNA ( see Figure 2\\u2014figure supplement 1 legend for discussion of translocation ) . The structure explains why the nuclease activity of Dna2 requires the ssDNA to have a free end ( Bae and Seo , 2000; Kao et al . , 2004 ) . The active site is embedded in a \\u223c10\\u2009\\u00c5 wide portion of the tunnel , and the tunnel entrances leading to it are too narrow to accommodate dsDNA or a single-stranded loop of a gap substrate ( Figure 1A , B ) . Since isolated Dna2 can degrade ssDNA with either 5\\u2019 to 3\\u2019 or 3\\u2019 to 5\\u2019 polarity in vitro , the ssDNA must be able to enter and thread through either end of the tunnel . In 5\\u2019 flap processing , threading likely proceeds through initial , transient interactions at the nuclease domain entrance of the tunnel , with subsequent re-binding events occurring further along the tunnel interior . This threading is likely related to that of \\u03bb exonuclease , where a 5\\u2019 terminal phosphate binding site inside the enzyme is proposed to drive the forward movement of the DNA in an electrostatic ratchet mechanism ( Zhang et al . , 2011 ) . The 5\\u2019 to 3\\u2019 nuclease activity was shown to require \\u223c15 nts of ssDNA for optimal affinity and cleavage ( Bae et al . , 2001; Gloor et al . , 2012 ) . This suggests that the nuclease domain-DNA interactions do not provide sufficient binding energy or their half-life is too short relative to the catalytic step of cleavage . We thus presume that the 5\\u2019 end of the ssDNA threads through the nuclease portion of the tunnel without cleavage , until it reaches the helicase domain and engages the DNA-binding pockets there . To investigate this in more detail , we assayed the Dna2 affinity and cleavage of a series of 5\\u2019 overhang substrates with ssDNA lengths that extend successively from the nuclease to the helicase 1A and 2A domains . The ssDNA consisted of deoxythymidine nucleotides , which have minimal secondary structure and thus do not require Rpa for cleavage by Dna2 ( 5\\u2019 ( dT ) 6 to 5\\u2019 ( dT ) 24 ) ( Figure 3\\u2014figure supplement 1A ) . The short substrates that can span only the nuclease domain ( 5\\u2019 ( dT ) 6 and 5\\u2019 ( dT ) 8 ) have Dna2 affinities approximately two orders of magnitude lower than that of 5\\u2019 ( dT ) 24 , and they exhibit minimal cleavage ( Figure 3\\u2014figure supplement 1A , B ) . With the slightly longer 5\\u2019 ( dT ) 8 , low level cleavage occurs 7 and 8 nucleotides from the 5\\u2019 end , indicating that the ssDNA extends to the helicase 1A domain ( Figure 3\\u2014figure supplement 1A ) . The sites of cleavage indicate that Dna2 opens up \\u223c2 base pairs ( bps ) of the duplex , since the tunnel entrance 3\\u2019 to the scissile phosphate is too narrow to accommodate double-stranded DNA . As ATP is neither required , nor has a significant effect , this DNA unwinding is the result of threading , likely analogous to DNA unwinding by \\u03bb exonuclease ( Zhang et al . , 2011 ) . Extending the ssDNA by 2 nts , ( 5\\u2019 ( dT ) 10 ) , results in a major increase in Dna2 affinity and cleavage , with the cleavage sites indicating the engagement of the helicase 2A domain after the unwinding of \\u223c2 bps ( Figure 3\\u2014figure supplement 1A ) . With 5\\u2019 ( dT ) 17 where the ssDNA is long enough to reach both the 1A and 2A domains without duplex unwinding , there is a final increase in Dna2 affinity and cleavage that plateau at the levels of 5\\u2019 ( dT ) 24 . The 3\\u2019 to 5\\u2019 nuclease activity would require the 3\\u2019 end of the ssDNA to enter the tunnel at the helicase 2A domain . As the phosphodiester-binding sites of the 2A domain are fully accessible to bulk solvent , the structure suggests that 3\\u2019 end threading should be more efficient than 5\\u2019 end threading in the absence of ATP . Indeed , Dna2 cleaves a 3\\u2019 overhang ( dT ) 18 substrate at least 3-fold faster than the corresponding 5\\u2019 ( dT ) 18 ( Figure 3A ) . The structure further suggests that the reported inhibition of 3\\u2019 end cleavage by ATP is due to 3\\u2019 end threading being counteracted by the helicase domain moving the DNA in the opposite direction ( Figure 3A ) ( Bae and Seo , 2000; Masuda-Sasa et al . , 2006 ) . 10 . 7554/eLife . 09832 . 009Figure 3 . Dna2 nuclease activity . ( A ) Denaturing PAGE showing ATP inhibits the nuclease activity on 3\\u2019 overhang substrate , while slightly increasing it for the 5\\u2019 overhang substrate . Substrates are at 15 nM . For this and subsequent nuclease assays , cleavage was quantified by loss of substrate and plotted with \\u00b1 s . d . error bars ( n = 3 ) . ( B ) Nuclease time course of 2 nM Dna2 with 10 nM 5\\u2019 RNA-DNA overhang or 5\\u2019 DNA overhang substrates . ATP or AMPPNP is at 1 . 3 mM . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 00910 . 7554/eLife . 09832 . 010Figure 3\\u2014figure supplement 1 . The ssDNA-length dependence of the DNA affinity correlates with cleavage rates . ( A ) Denaturing PAGE showing length dependency and cleavage sites of 5\\u2019 overhang DNA at a 30 nM concentration ( asterisk indicates FAM labeling of the 3\\u2019 end ) in the presence ( top panel ) or absence ( bottom ) of 1 . 3 mM ATP . Also shown are the dissociation constants ( Kd ) of Dna2 ( D278A ) for each substrate , according to the data shown in Figure 3\\u2014figure supplement 1B . ( B ) Electrophoretic mobility shift assays ( EMSA ) of Dna2 ( D278A ) binding to 5\\u2019 overhang DNA substrates . The graph shows fraction bound . Dotted lines are simulated binding curves based on the indicated Kd values . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 010 Taken together , these findings support the model that a major role for the helicase domain is augmenting the DNA affinity of Dna2 . As lack of ATP hydrolysis does not affect overall DNA binding by the helicase 1A and 2A domains , this model is consistent with the Dna2 ATPase-activity being dispensable for viability in yeast , and for DSB end resection in vitro , in contrast to the nuclease activity that is essential for both ( Bae and Seo , 2000; Budd et al . , 2000; Cejka et al . , 2010; Niu et al . , 2010; Zhu et al . , 2008 ) . RNA , which is not cleaved by Dna2 , can substitute for part of DNA in the length-dependency of cleavage , suggesting that it can interact with at least part of Dna2 ( Bae et al . , 2001; Bae and Seo , 2000 ) . In accord , we find that the affinity of Dna2 for a 5\\u2019 ( U ) 24 RNA overhang-DNA duplex is within an order of magnitude of its affinity for a comparable all-DNA substrate ( Figure 3\\u2014figure supplement 1B ) . This observance , coupled with the structural similarity of Dna2 to the Upf1 subfamily of RNA/DNA helicases raises the possibility that the helicase activity facilitates the bypassing of the 5\\u2019 RNA primer of Okazaki fragments . Indeed , ATP but not the non-hydrolyzable ATP analogue AMPPNP stimulates the cleavage of a 5\\u2019 ( U ) 12\\u2013 ( dT ) 12 RNA-DNA overhang substrate by \\u223c50% ( Figure 3B , top panel ) . By contrast , ATP had a minimal effect on the cleavage of the corresponding all-DNA ( dT ) 24 substrate , while AMPPNP inhibited cleavage slightly ( Figure 3B , bottom panel ) . Rpa is a heterotrimeric protein that consists of the Rpa70 , Rpa32 and Rpa14 subunits . Genetic and biochemical studies in budding yeast indicated that Dna2 binds to the Rpa70 subunit , through an interaction between the N-terminal portions of the two proteins ( Bae et al . , 2003 ) . These studies further pointed to additional binding sites on Dna2 and Rpa70 , as deletion of the N-terminal interacting segments reduced but did not eliminate Dna2-Rpa association and the stimulation of the nuclease activity ( Bae et al . , 2003 ) . The N-terminal portion of yeast Dna2 ( residues 1 to 405 ) is poorly conserved in mammalian orthologs and also contains a \\u223c350-residue yeast-specific extension , although it appears to contain an acidic/amphipathic helix analogous to \\u03b11 of mouse Dna2 , and likely encompasses a portion of the OB domain ( Figure 1\\u2014figure supplement 1A ) . We thus tested whether a mouse Dna2 fragment consisting of the \\u03b11 helix and OB domain ( thereafter \\u03b11OB; residues 1 to 122 ) binds to mouse Rpa using a GST pull-down assay . As shown in Figure 4A , GST-\\u03b11OB but not isolated GST binds to heterotrimeric Rpa in a 30 \\u03bcM stoichiometric solution ( lanes 1 and 2 ) . 10 . 7554/eLife . 09832 . 011Figure 4 . Dna2 physically interacts with Rpa . ( A ) GST pull-down assay showing that \\u03b11OB of Dna2 ( residues 1\\u2013122 ) interacts with Rpa70NAB ( 1\\u2013431 ) , but not the rest of Rpa heterotrimer ( Rpa ( -NAB ) ) . ( B ) Schematic drawing of the Rpa trimer , showing the OB domains as rectangles and the winged helix ( WH ) domain of Rpa32 as a sphere . ( C ) ITC curves for the human \\u03b11OB-Rpa70NAB , \\u03b11-Rpa70N and OB-Rpa70AB complexes . ( D ) Structure of the \\u03b11-Rpa70N complex . The \\u03b11 peptide is in yellow and Rpa70N in cyan . For clarity , the main chain amide group of L87 and carbonyl groups of M11 , K13 , F15 are not labeled . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 01110 . 7554/eLife . 09832 . 012Figure 4\\u2014figure supplement 1 . Comparison of the Dna2 \\u03b11\\u2013Rpa70 OBN , p53\\u2013Rpa70 OBN and Dna2 OB structures . ( A ) Superposition of the \\u03b11\\u2013Rpa70 OBN structure on the fusion protein of Rpa70 OBN and p53 . In the fusion protein structure , the Rpa70 OBN ( gray ) groove has two p53 peptides , one ( cyan ) from its fused p53 and another ( green ) from a crystallographic-symmetry related fusion protein . While it was suggested that only the latter represented the solution-state complex , our structure has aspects of both p53-Rpa70 OBN interfaces . An acidic/amphipathic helix from the symmetry-related p53 superimposes with the Dna2 helix , and part of the second p53 peptide overlaps with the Dna2 \\u03b2 turn and associated contacts . \\u03b11 peptide is red . ( B ) The Dna2 OB groove ( cyan ) has an overall hydrophobic- and basic-residue content similar to that of the Rpa70 OBN ( pink ) . The L12 and L45 loops ( labeled ) , which in DNA-binding OB domains partially wrap around the DNA , but are truncated in the Rpa70 OBN domain are , respectively , truncated and absent in the Dna2 OB domain , resulting in a wide open groove . Residues mutated in Figure 5A are labeled . ( C ) ITC titration profile of human Dna2 \\u03b11 ( residues 1\\u201320 ) binding to human Rpa70N ( residues 1\\u2013120 ) . ( D ) ITC titration profile of human Dna2 OB ( residues 21\\u2013122 ) binding to human Rpa70AB ( residues 181\\u2013422 ) . ( E ) ITC titration profile of human Dna2 \\u03b11OB ( residues 1\\u2013122 ) binding to human Rpa70NAB ( residues 1\\u2013422 ) . ( F ) The Dna2 OB domain does not exhibit any detectable DNA-binding at concentrations up to 80 \\u03bcM in an EMSA assay with ( dT ) 20 ssDNA . A shifted , smeary band appearing at 160 \\u03bcM OB has ~ 4% of the total DNA . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 012 The Dna2-interacting N-terminal segment of yeast Rpa70 ( residues 1 to 180 ) consists of an OB fold domain ( named OBN ) that is a known protein-protein interaction site and a ~60 residue flexible linker ( Figure 4B ) ( Bochkareva et al . , 2005; Fan and Pavletich , 2012; Fanning et al . , 2006; Gomes and Wold , 1996 ) . The rest of Rpa70 consists of three OB folds that are the main DNA-binding domains ( named DBD-A , DBD-B and DBD-C; Figure 4B ) ( Bochkarev et al . , 1997; Fan and Pavletich , 2012 ) . Because DBD-A and DBD-B can also participate in protein-protein interactions ( Jiang et al . , 2006; Loo and Melendy , 2004; Yuzhakov et al . , 1999 ) , this raised the possibility that they account for the remainder of Rpa70\\u2019s Dna2 affinity . We tested this using a mouse Rpa70 fragment containing the OBN , DBD-A and DBD-B domains ( residues 1 to 431; thereafter Rpa70NAB ) . As shown in Figure 4A , GST-\\u03b11OB binds to Rpa70NAB ( lanes 3 and 4 ) but not to the Rpa heterotrimer lacking this fragment ( Rpa ( -NAB ) ; lanes 5 to 6 ) . The dissociation constant ( Kd ) of the \\u03b11OB\\u2013Rpa70NAB complex , determined by isothermal titration calorimetry ( ITC ) , is 12 \\u00b1 1 \\u03bcM ( Figure 4C and Figure 4\\u2014figure supplement 1E ) . Using ITC , we found that the Dna2 \\u03b11 helix ( residues 1 to 20 ) binds to the OBN domain of Rpa70 ( residues 1 to 120; thereafter Rpa70N ) , while the Dna2 OB domain ( residues 21 to 122 ) binds to the Rpa70 fragment containing DBD-A and DBD-B ( residues 181 to 422; thereafter Rpa70AB ) , with Kd values of 34 \\u00b1 7 and 46 \\u00b1 10 \\u03bcM , respectively ( Figure 4C and Figure 4\\u2014figure supplement 1 ) . The Kd values of these subcomplexes relative to that of \\u03b11OB\\u2013Rpa70NAB indicate only a low level of cooperativity , consistent with the two interacting elements on both Rpa and Dna2 being separated by flexible linkers . Further dividing Rpa70AB into the individual DBD-A and DBD-B polypeptides failed to show detectable binding to Dna2 OB under the same conditions , indicating both are required ( not shown ) . We co-crystallized a human Dna2 peptide ( residues 1 to 20 ) containing the \\u03b11 helix with the OBN domain of human Rpa70 ( residues 1 to 120 ) . In the 1 . 6\\u2009\\u00c5 structure , residues 6 to 17 of Dna2 form a 2-turn amphipathic helix followed by a \\u03b2 turn , while the rest are disordered ( Figure 4D ) . The peptide binds to a shallow OBN groove that corresponds to the DNA-binding grooves of other OB domains ( Bochkarev et al . , 1997; Fan and Pavletich , 2012 ) . The only other OBN-peptide structure available is of OBN fused to the p53 transactivation domain peptide , where two p53 peptides occupy the OBN groove ( Bochkareva et al . , 2005 ) . The amphipathic helix of Dna2 overlaps with one of the p53 peptides , while the \\u03b2 turn coincides with part of the other p53 peptide ( Figure 4\\u2014figure supplement 1A ) . The mixed basic and hydrophobic character of the OBN groove complements the acidic-hydrophobic nature of the Dna2 peptide . Four arginine guanidinium groups and one backbone amide group of OBN contact two side chain carboxylate and three backbone carbonyl groups from \\u03b11 ( Figure 4D ) . One of these OBN arginine residues ( Arg43 ) splits the otherwise hydrophobic groove , demarcating two hydrophobic pockets . One pocket ( Ile33 , Met57 , Leu87 , Val93 and Ile95 ) binds to the hydrophobic face of the Dna2 helix ( Leu7 , Leu10 and Met11 ) , while the other pocket ( Leu45 and aliphatic portions of Arg31 , Arg43 , and Ser54 ) binds to Phe15 and Trp16 from the \\u03b2 turn of Dna2 . We do not know the structure of the Dna2 OB domain bound to the Rpa70 DBD-A and DBD-B domains , but the OB structure in intact Dna2 is consistent with a role in protein-protein interactions ( Figure 4\\u2014figure supplement 1B ) , and the isolated OB domain polypeptide does not exhibit any DNA-binding in EMSA at concentrations up to 80 \\u03bcM ( Figure 4\\u2014figure supplement 1F ) . A common feature of proteins involved in Rpa-dependent processes is their ability to bind to Rpa , either directly or through accessory factors , and this is thought to form the basis for the displacement of Rpa from ssDNA ( Fanning et al . , 2006; Zou et al . , 2006 ) . Rpa displacement is best understood with the simian virus 40 ( SV40 ) T antigen and related viral replication proteins , where T antigen-Rpa interactions allosterically modulate Rpa\\u2019s ssDNA affinity ( Jiang et al . , 2006; Loo and Melendy , 2004; Yuzhakov et al . , 1999 ) . Because Dna2 has been shown to displace Rpa from 5\\u2019 flap DNA ( Stewart et al . , 2008 ) , we sought to address whether Dna2-Rpa interactions have an analogous , direct role in Rpa displacement , or whether they reflect a simple recruitment process that allows Dna2 to better compete with Rpa for ssDNA . We first confirmed that both the \\u03b11 helix and OB domain of Dna2 are required for the stimulation of the nuclease activity by Rpa . For this , we used a 5\\u2019 overhang substrate with a stem loop secondary structure that makes the Dna2 nuclease activity dependent on Rpa ( 5\\u2019SL24 ) ( Figure 5\\u2014figure supplement 1A ) . In keeping with the findings with the yeast Dna2\\u25b3405N mutant , deletion of the Dna2 \\u03b11 helix reduced cleavage of the Rpa-coated 5\\u2019SL24 by a factor of ~2 compared to intact Dna2 ( Figure 5A ) . The analogous experiment to address the importance of the OB domain was not possible , as the OB-deleted Dna2 is insoluble ( not shown ) . We instead mutated three OB residues at positions corresponding to protein-protein contacts on the Rpa70 OBN domain ( Figure 4\\u2014figure supplement 1B ) . As shown in Figure 5A , two of these mutations ( I82A and R66E ) synergized with \\u03b11 deletion and reduced 5\\u2019SL24 cleavage further , whereas the third ( I110A ) had no discernible effect . To rule out that these mutations do not affect the structural integrity of Dna2 , we tested the Rpa-independent cleavage of the 5\\u2019 ( dT ) 24 substrate and found that it is not affected by neither the OB mutations nor the \\u03b11 deletion ( Figure 5B ) . 10 . 7554/eLife . 09832 . 013Figure 5 . Both Dna2-Rpa interactions are important for Dna2 stimulation , but only one is mutually exclusive with Rpa-ssDNA interactions . ( A ) Cleavage of a 5\\u2019 stem-loop overhang substrate ( 15 nM ) by wild type , \\u03b11-deleted , and OB mutant Dna2 in the presence of 15 nM Rpa . ( B ) Nuclease activity of the same set of enzymes as in ( A ) , but using a 5\\u2019 ( dT ) 24 overhang substrate in the absence of Rpa . ( C ) GST pull-down assay showing that \\u03b11OB and ( dT ) 24 ssDNA bind to Rpa in a partially mutually exclusive manner . Protein and DNA concentrations are in \\u03bcM , and the bar graph shows the quantitation of Rpa70 binding relative to the lane in the absence of DNA , which is set to 1 . Error bars are standard deviations from three repetitions of each experiment . ( D ) The binding of \\u03b11 to the Rpa70N polypeptide is unaffected by ( dT ) 8 ssDNA . GST pull-down assay and quantitation as in ( C ) . ( E ) The binding of OB to the Rpa70AB polypeptide is abolished by ( dT ) 8 ssDNA . GST pull-down assay and quantitation as in ( C ) . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 01310 . 7554/eLife . 09832 . 014Figure 5\\u2014figure supplement 1 . Cleavage of a ssDNA containing secondary structure is Rpa dependent . ( A ) Cleavage of a 5\\u2019 overhang substrate that has a predicted 6 bp stem-loop secondary structure at the 5\\u2019 end of the ssDNA ( 5\\u2019 SL24 ) . Dna2 , at the indicated concentrations , failed to cleave 5\\u2019 SL24 ( 30 nM ) appreciably under conditions where ~95% of the corresponding 5\\u2019 ( dT ) 24 substrate was cleaved , irrespective of ATP . Addition of Rpa , at a stoichiometric amount to ssDNA , stimulated SL24 cleavage by over a factor of 20 , but had only a modest effect on ( dT ) 24 cleavage . In the presence of Rpa , the cleavage of the two substrates was within a factor of two , compared to being nearly two orders of magnitude apart in its absence . ( B ) Cleavage of a 178-nt long 5\\u2019 overhang substrate ( 80 nM ) by Dna2 ( 20 nM ) . As with the shorter 5\\u2019 overhang substrates , ATP ( 1 . 3 mM ) has only a minor stimulatory effect . Because mixed-sequence ssDNA of this length would invariably have some secondary structure ( the lowest energy secondary structure , predicted by the UNAfold server ( mfold . rna . albany . edu ) , is shown at bottom , right ) , cleavage is stimulated by Rpa ( 400 nM ) as with the 5\\u2019 SL24 substrate . For this specific experiment , samples are quantified by integration of final products in the box . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 014 We then addressed whether the \\u03b11 helix and OB domain play a direct role in Rpa displacement , which in principle can occur either through the allosteric destabilization of Rpa-ssDNA interactions as shown for T antigen , or through \\u03b11OB binding to an Rpa site that overlaps with or sterically hinders a ssDNA-binding site . We did not expect the isolated \\u03b11OB , in the absence of the DNA-affinity provided by the nuclease and helicase domains , to displace Rpa from ssDNA , as the affinities of the \\u03b11OB-Rpa and Rpa-ssDNA complexes differ by 5 orders of magnitude ( Kd values of \\u223c12 \\u03bcM and \\u223c100 pM , respectively ) . Instead , we reasoned that if \\u03b11OB has a role destabilizing Rpa-ssDNA interactions , then this should be reflected in ssDNA interfering with \\u03b11OB-Rpa association . As shown in Figure 5C , addition of ( dT ) 24 substantially reduced Rpa binding to GST-\\u03b11OB ( lanes 1 to 3 ) , consistent with \\u03b11OB and ssDNA interacting with Rpa in a mutually exclusive manner . However , while one molar equivalent of ( dT ) 24 reduced the bound Rpa by a factor of \\u223c4 , the remaining Rpa was clearly above the background level of the GST-only reaction ( lanes 4 and 5 ) , suggesting that only one of the two Dna2-Rpa interactions is mutually exclusive with Rpa-ssDNA interactions . Consistent with this , addition of ( dT ) 8 had no discernible effect on the binding of GST-\\u03b11 to Rpa70N ( Figure 5D ) , whereas it eliminated the binding of GST-OB to Rpa70AB in a manner dependent on the stoichiometry of ( dT ) 8 to Rpa70AB ( Figure 5E ) . These findings indicate that the interaction between the Dna2 \\u03b11 helix and the Rpa OBN domain is a simple recruitment step , consistent with both of these elements being flexibly tethered to the remainder of their polypeptides and with their lack of ssDNA affinity . This simple recruitment would be important for Dna2 accessing ssDNA-bound Rpa , where the interaction between the Dna2 OB and Rpa DBD-A\\u2013DBD-B domains would not be initially available . The simple recruitment interaction would also increase the effective concentration of the Dna2 OB at the Rpa DBD-A\\u2013DBD-B , as they immediately follow the \\u03b11 helix and OBN domain , respectively ( Figure 6A ) . 10 . 7554/eLife . 09832 . 015Figure 6 . Dna2 displaces Rpa from 5\\u2019 but not 3\\u2019 overhang DNA . ( A ) Schematic of the proposed mechanism of Dna2 displacing Rpa from a 5\\u2019 overhang DNA . Dna2 is shown as a hollow cylinder , except for its \\u03b11 helix and OB fold domains that are shown as a rectangle and circle , respectively . The label \\u201cnuc\\u201d marks the nuclease tunnel entrance into which the ssDNA would thread , and \\u201chel2A\\u201d marks the helicase tunnel exit where the 5\\u2019 end of the ssDNA would end up after threading . ( B ) Schematic illustrating that at a 3\\u2019 overhang DNA , Dna2-Rpa interactions do not result in a free ssDNA end that can thread into the Dna2 tunnel . ( C ) Nuclease-dead Dna2 ( D278A ) displaces Rpa from 5\\u2019- but not 3\\u2019 overhang DNA . The overhang consists of ( dT ) 26 and the DNA is conjugated to streptavidin ( shown by \\u201cS\\u201d ) through a biotin group at the end of the duplex . Column graph showing quantitation of Rpa32 plots the molar ratio relative to the reaction lacking Dna2 for each DNA substrate ( lanes 2 , 3 for 5\\u2019 overhang DNA , and lanes 5 and 6 for 3\\u2019 overhang DNA ) , or relative to lane 1 for the comparison of Rpa loading onto 5\\u2019- and 3\\u2019-overhang DNA . Quantitation of relative Dna2 loading is similarly shown in the last column graph . Error bars are standard deviations from three repetitions of each experiment . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 015 This juxtaposition would then increase the probability of DBD-A\\u2013DBD-B transiently associating with OB and dissociating from ssDNA . This is plausible , because while the DNA affinity of DBD-A\\u2013DBD-B ( ~50 nM Kd ) is substantially higher than its Dna2 affinity , those of the individual DBD-A ( 2 \\u03bcM Kd ) and DBD-B ( 20 \\u03bcM Kd ) are not , and as with the intact Rpa heterotrimer , they are thought to associate with and dissociate from DNA sequentially ( Arunkumar et al . , 2005; Fan and Pavletich , 2012; Fanning et al . , 2006 ) . By itself , the transient displacement of DNA from DBD-A\\u2013DBD-B will not lead to the release Rpa from ssDNA . However , because the DBDA\\u2013DBDB is at the 5\\u2019 end of the Rpa-DNA complex ( Bochkarev et al . , 1997; Fan and Pavletich , 2012 ) , and the Dna2 OB domain is next to the nuclease tunnel entrance , the transiently free 5\\u2019 end of the DNA will be well-placed to start threading through the nuclease tunnel ( Figure 6A ) . The threading process then should be able to completely dissociate the already weakened Rpa-DNA complex ( Figure 6A ) . At a 3\\u2019 end of DNA , by contrast , the transient dissociation of DBD-A\\u2013DBD-B from DNA will expose an internal ssDNA lacking an end that can be trapped by the Dna2 tunnel , and DBD-A\\u2013DBD-B will revert to their DNA-bound state ( Figure 6B ) . This mechanism of Rpa displacement predicts that the inhibition of 3\\u2019 end cleavage by Rpa is due to the inability of Dna2 to displace Rpa there . To test this prediction , we conjugated 5\\u2019- or 3\\u2019- ( dT ) 26 overhang DNA that was biotinylated on the duplex end onto streptavidin beads , loaded it with Rpa , and then added nuclease-dead Dna2 ( D278A mutant ) . As shown in Figure 6C , Dna2 reduced the amount of Rpa bound to the 5\\u2019 overhang DNA by \\u223c30% ( lanes 2 and 3 ) , whereas it had a minimal effect on the Rpa bound to the 3\\u2019 overhang DNA ( \\u223c5% reduction; lanes 5 and 6 ) . The amount of Rpa displacement was proportional to Dna2 loading , which was substantial with 5\\u2019 overhang DNA ( \\u223c45% of the reaction lacking Rpa; lanes 1 and 3 ) , but negligible with 3\\u2019 overhang DNA ( \\u223c3%; lanes 4 and 6 ) . In the absence of Rpa , by contrast , the amount of Dna2 loading on the two DNA substrates was essentially identical ( lanes 1 and 4 ) , consistent with the inability of Dna2 to load onto the 3\\u2019 overhang being due to its failure to displace Rpa . \\n The Dna2-ssDNA structure shows that the active site and most of the DNA-binding sites are enclosed in a narrow tunnel , necessitating the threading of the DNA through a tunnel end to access the DNA binding sites . The structure also indicates that the translocase activity of the helicase domain does not drive threading . A 5\\u2019 DNA end would have to thread halfway through the tunnel before it can access the helicase domain , while the threading of a 3\\u2019 end starting at the helicase 2A domain would be opposed by the 5\\u2019 to 3\\u2019 polarity of translocation , as we demonstrate . Instead of translocation , the helicase domain appears to be important for providing DNA affinity and for bypassing the 5\\u2019 RNA primer of Okazaki fragments . The structure also precludes the helicase domain tracking on DNA to any significant extent , because the nuclease domain is ahead , in the 5\\u2019 to 3\\u2019 direction of translocation . This is consistent with in vitro studies showing the ATPase activity to be dispensable for 5\\u2019 flap processing and DSB resection ( Cejka et al . , 2010; Niu et al . , 2010; Zhu et al . , 2008 ) , and the fact that this activity is rather low compared to bona-fide helicases ( Bae and Seo , 2000; Masuda-Sasa et al . , 2006 ) . In yeast , ATPase mutations do result in growth defects ( Budd et al . , 2000 ) , and it is possible this is due to the ATPase activity contributing to the bypassing of the RNA primer of Okazaki flaps , as suggested by our in vitro data . This may be reflected in the similarity of the Dna2 helicase domain to the Upf1 family of RNA/DNA helicases , which is extensive enough to indicate that Dna2 picked up an ancestral Upf1-like helicase during its evolution . The requirement for threading necessitates Dna2 having a mechanism to displace Rpa from ssDNA . As our proposed mechanism of Rpa displacement predicts , we find that Dna2 can displace Rpa from a 5\\u2019 but not a 3\\u2019 end , explaining how Rpa dictates the proper end polarity of the nuclease activity of Dna2 . \\n Full-length mouse Dna2 was cloned into a pFastbac1 baculovirus vector engineered with a cleavable N-terminal GST tag and a non-cleavable C-terminal FLAG tag , and was expressed in Hi-5 insect cells ( Life technologies , Carlsbad , CA ) . The recombinant protein was purified first by GST-affinity chromatography and , after cleavage of the GST tag , by anion exchange and gel-filtration chromatography . Purified Dna2 was concentrated to \\u223c20 mg/mL in 20 mM Tris-HCl , 250 mM NaCl , 0 . 3 mM TCEP , pH 8 . 0 . All buffers were degassed before use . The various Dna2 mutants and seleno-methionine substituted protein were purified similarly . Seleno-methionine substituted Dna2 was expressed according to manufacturer\\u2019s protocol ( Expression systems , Davis , CA ) and was purified similarly . For the expression of the mouse Rpa heterotrimer , Rpa70 was cloned into a pFastbac1 vector and Rpa32/Rpa14 were cloned into a modified pFastBac-dual vector with Rpa32 fused to a cleavable N-terminal GST-tag . The Rpa heterotrimer was produced by co-infecting Hi-5 cells with both viruses , and was purified as described ( Fan and Pavletich , 2012 ) . The Rpa heterotrimer with truncated Rpa70 was expressed and purified similarly . GST-tagged mouse Dna2 \\u03b11 ( residues 1\\u201320 ) , \\u03b11OB ( residues 1\\u2013122 ) and OB ( residues 21\\u2013122 ) and Rpa70NAB ( residues 1\\u2013431 ) , Rpa70N ( residues 1\\u2013122 ) , and Rpa70AB ( residues 191\\u2013431 ) fragments , as well as the corresponding human polypeptides used in ITC measurements , were cloned into a pGEX-4T vector and expressed in E . coli BL21DE3 cells . They were purified by GST affinity chromatography , ion exchange and/or heparin chromatography , and gel-filtration chromatography . The corresponding untagged polypeptides were expressed fused to an N-terminal 6-His-sumo tag in E . coli BL21DE3 cells . Following nickel affinity chromatography and cleavage of the 6-His-sumo tag by Ulp1 , they were further purified by ion exchange and/or heparin , and gel-filtration chromatography . Crystals of the Dna2-ADP complex were grown in 4\\u00b0C using the hanging drop vapor diffusion method from a crystallization buffer of 80 mM MES , 250 mM Li2SO4 , 2 mM MgCl2 , 8\\u201312% PEG MME 5000 , 0 . 5 mM TCEP , pH 6 . 5 , containing 12 mg/mL Dna2 and 1 mM ADP . Seleno-methionine substituted Dna2 was crystallized under similar conditions using seeding . Crystals of Dna2 bound to 21-nt ssDNA and ADP ( Dna2-ssDNA in Table 1 ) were grown from a crystallization buffer of 80 mM MES , 20 mM CaCl2 , 10 mM spermidine , 4\\u20139% isopropanol , 0 . 5 mM TCEP , pH 6 . 5 , and 1 mM ADP . They contain two molecules in the asymmetric unit . Crystals of Dna2 bound to 5\\u2019 overhang DNA , which consists of a 17-nt 5\\u2019 overhanging ssDNA and a 6 base pair dsDNA ( Dna2-5\\u2019 overhang DNA in Table 1 ) , grew from a similar condition but in a different space group , and have one complex in the asymmetric unit and higher diffraction limits . As there is no electron density for the duplex , we presume it is disordered . All crystals were cryo-protected in crystallization buffer supplemented with 20\\u201325% glycerol or ethylene glycol , and were flash-frozen in liquid nitrogen . The human DNA2 \\u03b11-RPA70 OBN complex was crystallized by mixing an 8 . 7 mg/ml solution of the RPA70N polypeptide ( residues 1\\u2013120 ) with a 3-fold molar excess of a synthetic DNA2 \\u03b11 peptide ( residues 1\\u201320 ) from 50 mM Tris-HCl , 35% PEG 1500 , 2 mM TCEP , pH 8 . 0 . Crystals were cryo-protected in crystallization buffer supplemented with 20\\u201325% glycerol and flash-frozen in liquid nitrogen . Diffraction data were collected at the 24IDC and 24IDE beamlines of the Advanced Photon Source ( Argonne National Laboratory ) and the X29 beamline of the National Synchrotron Light Source ( Brookhaven National Laboratory ) . Data sets were processed with the HKL2000 suite ( Otwinowski and Minor , 1997 ) . The structure of Dna2-ADP complex was determined using SAD with data collected at the selenium edge ( Bricogne et al . , 2003 ) . The phases were improved using solvent flattening and two-fold NCS averaging with multiple masks with the program DM ( Winn et al . , 2011 ) . The model was built using O ( Jones et al . , 1991 ) and Coot ( Emsley et al . , 2010 ) and refined with REFMAC5 ( Winn et al . , 2011 ) and PHENIX ( Adams et al . , 2010 ) using tight NCS restraints on atom positions . Initial phases for the two Dna2-ADP-ssDNA complexes were obtained by molecular replacement with PHASER ( McCoy et al . , 2007 ) using the apo-Dna2 structure as the search model , and the structures were refined using REFMAC5 ( Winn et al . , 2011 ) and PHENIX ( Adams et al . , 2010 ) , with TLS parameterization of temperature factors of the high resolution Dna2-5\\u2019 overhang DNA complex . The Ramachandran plot of the final model has 90 . 5% , 8 . 9% , 0 . 5% and 0% of the residues in the most favored , additional allowed , generously allowed and disallowed regions . The statistics from data collection and refinement are shown in Table 1 . Figures were generated using PyMOL ( http://www . pymol . org ) . Unless otherwise noted , experiments were performed in a 15 \\u03bcL volume in 20 mM Tris-HCl , 125 mM NaCl , 6 mM MgCl2 , 1 . 3 mM ATP , 0 . 2 mg/mL BSA , 2% glycerol , pH 8 . 0 . Reactions were incubated for 30\\u2009min at 25\\u00b0C and stopped by adding 0 . 5% SDS , 20 mM EDTA and 1 unit of proteinase K . Reactions were analyzed by 16% or 12% denaturing urea-PAGE . For Rpa-containing reactions , Rpa was incubated with DNA for 15 min at 4\\u00b0C before the addition of Dna2 . For reactions using 6-FAM labeled DNA , wet gels were scanned using a fluorescent laser scanner ( Fujifilm FLA 5000 ) , and the bands were quantified with ImageGauge software ( Fujifilm ) . For GST pull-down experiments , 30 \\u03bcM GST-tagged mouse \\u03b11-OB ( residues 1 to 122 ) or GST was incubated with an equal molar amount of full-length Rpa , Rpa70NAB or Rpa ( -NAB ) . Binding reactions ( 40 \\u03bcL ) were carried out in 20 mM Tris-HCl , 80 mM NaCl , 0 . 3 mM TCEP , 2% glycerol , pH 8 . 0 at 4\\u00b0C for 30 min before addition of glutathione beads . After 30 min , the beads were washed three times with binding buffer . Proteins were eluted with 20 mM glutathione and analyzed by SDS-PAGE . Other GST pull-down experiments were carried out similarly , with protein concentrations indicated in the main text or figure legends . For Rpa displacement experiments , 2 \\u03bcM 5\\u2019- or 3\\u2019- ( dT ) 26 overhang DNA that was biotinylated at the duplex end was coupled to magnetic streptavidin beads . After washing , the beads were incubated with a 1 molar equivalent of Rpa for 20\\u2009min in 20 mM Tris-HCl , 125 mM NaCl , 2% glycerol , 0 . 3 mM TCEP , 0 . 01% Tween 20 , pH 8 . 0 , followed by the addition of a one molar equivalent of Rpa or Rpa-Dna2 mixture . The beads were incubated for 45 min with mixing , and after 3 washes the beads were boiled and analyzed by SDS-page . ITC experiments were carried out using a MicroCal ITC200 calorimeter ( Malvern Instruments Inc . , Westborough , MA ) at 20\\u00b0C in a buffer of 20 mM HEPES , 80 mM NaCl , 0 . 2 mM TCEP , pH 7 . 5 . Binding reactions ( 10 \\u03bcl ) containing 0 . 075 or 0 . 3 nM of 32P-labelled DNA substrates with increasing amounts of nuclease-dead Dna2 were carried out in 20 mM Tris-HCl , 80 mM NaCl , 0 . 3 mM TCEP , 0 . 2 mg/mL BSA ( New England Biolabs , Ipswich , MA ) , 2% glycerol , pH 8 . 0 . Reactions were incubated on ice for 30\\u2009min . , followed by electrophoresis at 4\\u00b0C on 4% ( w/v ) native PAGE gels in 1x TB buffer . The dried gels were scanned using a phosphorimager ( GE Typhoon 7000 , GE Healthcare , Pittsburg , PA ) , bands were quantified with ImageGauge software ( Fujifilm ) , and the apparent dissociation constants ( Kd ) were calculated from the equilibrium expression of a one-site binding model . Curve fitting was done by minimizing the sum of the square of the differences between the observed fraction of bound DNA and the fraction predicted from the model .\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"headings\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"keywords\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"id\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 10,\n        \"samples\": [\n          \"journal.ppat.1005553\",\n          \"elife-11384-v1\",\n          \"elife-09832-v3\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"processed_text\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 10,\n        \"samples\": [\n          \"## Keywords: medicine and health sciences, lysosomes, vesicles, pathology and laboratory medicine, hek 293 cells, influenza, pathogens, biological cultures, microbiology, retroviruses, viruses, immunodeficiency viruses, immune receptor signaling, rna viruses, membrane receptor signaling, cellular structures and organelles, endosomes, research and analysis methods, specimen preparation and treatment, infectious diseases, staining, proteins, medical microbiology, hiv, microbial pathogens, cell lines, biochemistry, signal transduction, cell staining, cell biology, viral pathogens, interferons, biology and life sciences, viral diseases, lentivirus, cell signaling, organisms\\n## Abstract\\nPlasmacytoid dendritic cells ( pDC ) are innate immune cells that sense viral nucleic acids through endosomal Toll-like receptor ( TLR ) 7/9 to produce type I interferon ( IFN ) and to differentiate into potent antigen presenting cells ( APC ) . Engagement of TLR7/9 in early endosomes appears to trigger the IRF7 pathway for IFN production whereas engagement in lysosomes seems to trigger the NF-\\u03baB pathway for maturation into APC . We showed previously that HIV-1 ( HIV ) localizes predominantly to early endosomes , not lysosomes , and mainly stimulate IRF7 rather than NF-\\u03baB signaling pathways in pDC . This divergent signaling may contribute to disease progression through production of pro-apoptotic and pro-inflammatory IFN and inadequate maturation of pDCs . We now demonstrate that HIV virions may be re-directed to lysosomes for NF-\\u03baB signaling by either pseudotyping HIV with influenza hemagglutinin envelope or modification of CD4 mediated-intracellular trafficking . These data suggest that HIV envelope-CD4 receptor interactions drive pDC activation toward an immature IFN producing phenotype rather than differentiation into a mature dendritic cell phenotype . \\n\\n## Introduction\\n Type I interferon ( IFN ) plays a dichotomous role in chronic viral infections such as Human Immunodeficiency Virus-1 ( HIV ) , contributing to the control of viral replication during the earliest stages of infection , yet fueling disease progression by activating target cells for infection , decreasing antiviral gene expression , enabling infection with increased reservoir size , and accelerating CD4 T-cell loss [1\\u20138] . Plasmacytoid dendritic cells ( pDC ) are thought to play a significant role in IFN responses during HIV infection , arriving rapidly at sites of mucosal transmission [4] and relocating from blood to lymphoid tissues where they produce pro-apoptotic and pro-inflammatory IFN [9\\u201311] . Cellular mechanisms underlying HIV-stimulated IFN production by pDC are only partially understood . We have previously shown that abundant IFN is produced by pDC upon HIV stimulation through endosomal recognition of genomic RNA by TLR7 . This response requires the presence of HIV envelope protein on viral particles , interactions between CD4 and the viral envelope protein , HIV endocytosis and endosomal acidification; however , co-receptor usage , viral fusion and viral replication are not required [12 , 13] . Cell-to-cell infection seems to amplify pDC responses to HIV , however precise mechanisms underlying differences between cell-free and cell-to-cell pDC activation are not clearly defined [14] . We and others have shown that pDC are highly resistant to HIV infection , and this block to replication is IFN-independent [15 , 16] . In addition to IFN production , pDC can act as antigen-presenting cells ( APC ) to activate T-cell\\u2013mediated adaptive immune responses [17\\u201321] . Acquisition of an APC phenotype requires specific signals that are distinct from the signals that induce large amounts of IFN . We have previously shown that HIV stimulated pDC express low levels of the co-stimulatory molecule CD86 and express Indoleamine 2 , 3-dioxygenase ( IDO ) , a potent inducer of regulatory T cells , indicating that they do not differentiate into mature APC and fail to stimulate potent T cell responses [22 , 23] . However , pDC can differentiate into APC with influenza virus or the synthetic TLR7 agonist R837 and are able to cross-present antigens from HIV-1-infected apoptotic cells to HIV-specific CD8+ T lymphocytes , demonstrating that pDC do not have an intrinsic defect in presentation of HIV antigens , but rather that sensing of HIV does not provide the signals that are required for efficient differentiation of pDC into APC [17] . pDC sense single stranded RNA or unmethylated DNA containing Cytosine\\u2013Guanosine dinucleotides ( CpG ) through Toll-like receptors ( TLR ) 7 and 9 , respectively , located in endosomal compartments . Both TLR7 and TLR9 signal through the adapter protein myeloid differentiation primary response gene 88 ( MyD88 ) . Downstream IFN signaling occurs in response to activation of IFN genes through phosphorylation of interferon regulatory factor 7 ( IRF7 ) , whereas downstream signaling of nuclear factor kappa-light-chain-enhancer of activated B cells ( NF-\\u03baB ) leads to the transcriptional activation of proinflammatory kinases and upregulation of MHC and co-stimulatory molecules necessary for maturation into APC . [12 , 24] . The functional response of pDC to pathogens is flexible . As posited by the spatiotemporal model of pDC sensing [25] , differential pDC activation is likely related to the subcellular location where the TLR senses the pathogen . Thus , engagement of TLR 7/9 in the early endosomes of pDC preferentially triggers the IRF7 signal cascade , leading to type I IFN responses; whereas engagement of TLR7/9 in lysosomes preferentially triggers the NF-\\u03baB signal cascade , leading to the production of proinflammatory cytokines TNF\\u03b1 and IL6 , upregulation of co-stimulatory molecules , and an APC phenotype [25 , 26] . Differential trafficking and therefore sensing of synthetic TLR9-activating CpGs is attributed to sequence-related secondary and tertiary structural features of the CpGs . CpGs which contain phosphodiester backbones and palindromic motifs ( CpGA ) form multimeric complexes and traffic to early endosomes for IRF7 signaling whereas CpGs which contain phosphorothioate backbones and lack palindromic motifs ( CpGB ) traffic as monomers to lysosomes for NF-\\u03baB signaling . Intermediate CpGs ( CpGC ) combine structural elements of both CpGA and CpGB , traffic to both compartments , and stimulate both IRF7 and NF-\\u03baB signaling [27\\u201329] . While the spatiotemporal model of pDC sensing has been most clearly evaluated using synthetic TLR9 agonists ( CpG ) , we have shown that the model also applies to HIV and TLR7 , whereby HIV traffics to early endosomes in pDC , activating IRF7 signaling rather than NF-\\u03baB signaling [22 , 23] . The upstream events that determine activation of each of these pathways , and in particular , HIV virion trafficking in pDC , are currently unknown , however , prior studies suggest that HIV envelope may play a major role [13 , 30 , 31] . Here we demonstrate that HIV trafficking and pDC phenotype is predominantly determined by envelope-CD4 interaction , such that manipulation of HIV envelope or CD4 intracellular trafficking enables modulation of divergent sensing of HIV . \\n\\n## Results\\n We hypothesized that HIV envelope protein interactions with cell surface CD4 determine the intracellular trafficking of HIV and the resultant signaling in pDC , based on the spatiotemporal model of TLR signaling [25] . To test this , we replaced HIV envelope protein with envelope protein from a virus that activates pDC to differentiate into mature pDC , namely influenza virus . Influenza virus hemagglutinin envelope glycoprotein ( HA ) binds to sialic acids on the cell surface to trigger clathrin-dependent endocytosis [32] . We pseudotyped HIV virions with influenza hemagglutinin glycoprotein envelope ( HA-HIV ) and first compared the functional response of purified pDC to HIV , influenza , and HA-HIV , in terms of magnitude and kinetics of TNF\\u03b1 ( TNF ) and IFN production . TNF is produced downstream of NF-\\u03baB signaling and IFN is produced downstream of IRF7 signaling . TNF and IFN were measured by intracellular cytokine staining ( ICS ) at 30 minutes , 2 hours , 6 hours , and 12 hours and in the culture medium by cytokine bead array ( CBA ) and ELISA , respectively , at 2 hours , 6 hours , and 12\\u201324 hours . As previously demonstrated [22] , the response of pDC to HIV was characterized by delayed IFN and TNF responses with IFN predominating at later time points . In comparison , both HA-HIV and influenza stimulated pDC to rapidly produce TNF within 30 minutes- 2 hours , an effect which plateaued by 12\\u201324 hours . Influenza also stimulated early IFN secretion , within 2 hours . HA-HIV induced IFN secretion , albeit at lower levels than Flu itself ( Fig 1A and 1B ) , possibly due to faster trafficking kinetics and early global cytokine shutdown ( as evidenced by earlier IFN and TNF shutdown compared to Flu in ICS Fig 1A ) . Strikingly , TNF was always produced antecedent to IFN , as evidenced by ICS staining , and as has been observed in murine pDC [33] ( Fig 1A ) . After overnight incubation of pDC , HIV stimulated minimal upregulation of CD86 and HLA-DR while HA-HIV and influenza stimulated strong upregulation of CD86 and HLA-DR expression , providing further evidence that HA-HIV and influenza activate NF-\\u03baB signaling/maturation pathways in pDC while HIV does not ( Fig 1C ) . Similar maturation and IFN effects were seen whether pDC were stimulated with X4 lab strain , MN or HIV backbone pNL43-\\u0394Env-vpr+-luc+ pseudotyped with X5 HIV envelopes ( JRFL , REJO , JOTO ) as compared to HIV backbone pNL43-\\u0394Env-vpr+-luc+ pseudotyped with hemagglutinin envelopes H1 and H5 ( S1A Fig ) . A functional measure of pDC maturation is to test whether cells become refractory to re-stimulation by TLR agonists , known as TLR tolerance . We had previously shown that HIV-activated pDC maintain an immature phenotype and are not refractory to re-stimulation to produce IFN . This effect was not due to activation of pDC that had failed to become activated during the previous overnight incubation [22] . We therefore compared the effects of HIV , influenza , HA-HIV , and CpGB , a potent pDC TLR9 maturation stimulus , to inhibit re-stimulation , as a marker of complete pDC maturation . As compared to HIV , HA-HIV inhibited re-stimulation of pDC similarly to influenza and CpGB , thus signifying that HIV pseudotyped with HA matured pDC fully ( Fig 1D ) . Overall , swapping HIV envelope with influenza envelope induced a mature pDC phenotype similar to that induced by influenza activation . Notably , MN HIV , a CXCR4 lab strain of HIV was used in these experiments . However , HIV ( pNL43-\\u0394Env-vpr+-luc+ ) pseudotyped with envelopes JRFL , REJO , or JOTO ( all R5-tropic ) all stimulate pDC to produce IFN , and not to mature , as expected since co-receptor usage is not essential for pDC sensing of HIV ( S1A and S1B Fig ) [12] . Because HA-HIV and influenza similarly stimulated pDC to mature , we sought to investigate whether HA-HIV traffics similarly to influenza in pDC . HIV virions ( pNL43-\\u0394Env-vpr+-luc+ ) pseudotyped with HA and packaging green florescent protein ( GFP ) and HIV virions ( pNL43-\\u0394Env-vpr+-luc+ ) pseudotyped with JRFL ( R5 envelope ) and packaging GFP were generated , using eGFP-Vpr plasmids , as previously described [22] . Influenza packaging GFP were also generated , as previously described [34 , 35] . We found that HA-HIV , similarly to influenza and unlike HIV itself , rapidly trafficked to lysosomes by 30 minutes as evidenced by co-localization with Lysotracker , a dye that traffics to these organelles ( Fig 2A and 2C ) . Both influenza and HA-HIV extensively co-localized with Lysotracker at 30 minutes and 2\\u20134 hours , whereas HIV was barely visible inside the cell at these early time points . After overnight incubation HIV was well visualized inside the cell , but did not co-localize well with lysotracker . Influenza and HA-HIV still seemed to co-localize with lysotracker even though the florescent signal was faded , likely due to lysosomal degradation of the virions ( S2A\\u2013S2D Fig ) . We confirmed that HIV traffics to early endosomal ( EEA1 ) compartments by 18 hours as previously shown [22] , whereas influenza and HA-HIV traffic significantly less to these compartments and the green signal is faded at 18 hours ( Fig 2B and 2D ) . Thus , the nature of the viral envelope seems crucial to determining trafficking of virions in pDC , and for the downstream signaling pathways activated in different intracellular compartments . Our results indicate that viral envelope protein dictates early intracellular trafficking of virions in pDC , suggesting that trafficking is directed by interaction of HIV envelope protein with its cognate receptor . In pDC , sensing of HIV involves CD4-mediated endocytosis [12] . CD4 is a type I integral membrane glycoprotein that can be internalized through clathrin-mediated endocytosis . The intracellular tail of CD4 displays motifs important for its internalization: a dileucine motif that allows interaction with the clathrin adaptor 2 ( AP-2 ) [36] and an adjacent serine , whose phosphorylation augments the affinity of the dileucine motif for AP-2 [36] , thereby regulating CD4 endocytosis . In cells of macrophage-monocyte lineage , CD4 is constitutively endocytosed at low levels through clathrin-coated pits to early and recycling endosomes [37] , as CD4 is serine-phosphorylated to some extent even in unstimulated cells [36] . We first examined whether CD4 itself and CD4-associated targeting motifs are responsible for the predominant localization of HIV in early endosomes . We used HEK 293 T ( HEK ) cells and HEK reporter cells as a model because HEK cells do not express CD4 under native conditions and therefore manipulation of CD4 trafficking and viral-CD4 interactions can be studied more clearly . Moreover , due to technical limitations , it was not possible to transfect or transduce primary pDCs or the Gen2 . 2 pDC cell line to undertake these studies . We engineered hybrid CD4 molecules , mutating its intracytoplasmic tail or swapping it with the intracellular domain of CD205 ( DEC205 ) and Lamp-1 ( Fig 3A ) . CD205 and Lamp-1 contain distinct lysosome-targeting motifs in their intracytoplasmic tail that induce constitutive targeting to late endosomes/lysosomes [38\\u201340] . DEC-205 expresses the coated-pit internalization sequence ( FSSVRY ) and lysosome-targeting motif ( EDE ) , whereas Lamp-1 expresses the lysosomal targeting motif ( YQTI ) . Several mutants were tested , as visualized in Fig 3A: ( 1 ) . CD4-WT ( wild type CD4 ) , ( 2 ) . CD4-STOP ( lacking the cytoplasmic domain ) , ( 3 ) . CD4-DEC ( replacing the CD4 cytoplasmic domain with the DEC-205 cytoplasmic domain to shuttle CD4 to the lysosomes ) , and ( 4 ) . CD4-LAMP ( replacing the CD4 cytoplasmic domain with the Lamp-1 cytoplasmic domain to shuttle CD4 to the lysosomes ) . The CD4 mutant sequences were introduced into lentiviral vectors for stable transduction of HEK cells for microscopy , and HEK-Blue hTLR7-expressing cells to measure NF-\\u03baB activation by HIV . HEK-Blue hTLR7 co-express human TLR7 and an NF-\\u03baB inducible secreted embryonic alkaline phosphatase ( SEAP ) reporter gene . CD4 expression was maintained in the presence of puromycin , and CD4 expression across cell lines was uniform at 65\\u201375% . For these experiments we used JRFL HIV packaging GFP- HIV Gag-iGFP ( GFP HIV ) to track HIV intracellular trafficking . Following incubation with HIV for 2 to 4 hours , CD4-expressing cells bound and endocytosed HIV efficiently , as shown in Fig 3C . This time point was chosen because HIV was not visualized well before 2 hours , and the fluorescent signal was faded after overnight incubation . The main path of viral entry in CD4-expressing HEK cells is CD4-mediated endocytosis as CD4 blockade completely abrogated HIV uptake ( Fig 3B ) . HIV co-localized extensively with CD4 , whatever CD4 construct the cells expressed , in the range of 65% to 80% co-localization per cell , as measured by single cell Mander\\u2019s coefficient ( Fig 3D ) . HEK cells which are not expressing CD4 do not take up HIV as represented by the ( - ) condition in representative images ( Fig 3C ) . Although the intracellular distribution pattern appeared different between the different CD4 constructs , with CD4-WT and CD4-STOP appearing more cell-surface associated and CD4-LAMP and CD4-DEC appearing more internal compartment associated , the overall fluorescence intensities of cell-associated HIV were comparable . To better characterize the intracellular localization of GFP- HIV in HEK cells expressing CD4 hybrids , cells were exposed to GFP-HIV for 2\\u20134 hours , then fixed and stained for the early endosomal marker EEA1 , the recycling endosomal marker transferrin receptor ( TfR ) , or the lysosomal marker LAMP1 . As an additional lysosomal marker , after 2\\u20134 hours of GFP-HIV incubation , lysotracker was added to culture media , and live imaging was performed . Internalized HIV trafficked predominantly to early EEA1+ and TfR+ endosomes in CD4-WT , similar to trafficking of HIV in pDC ( Fig 4 ) . However , HIV trafficked mainly to LAMP-1+ and lysotracker+ lysosomes in CD4-DEC and CD4-LAMP ( Fig 5 ) . Surprisingly HIV trafficked to EEA1+ compartments in CD4-STOP instead of remaining on the cell surface , but did not traffic predominantly to TfR ( Fig 4A\\u20134D ) . As endosomes containing EEA1 ultimately direct trafficking to the degradative machinery of the cell whereas endosomes containing TfR do not [41] , it is likely that some of the mutant CD4-STOP proteins are targeted for degradation to some extent . Overall , these data are consistent with the expected trafficking patterns of each CD4 construct , and indicate that replacing the intracellular tail of CD4 with those of CD205 or LAMP-1 targets CD4 and HIV to the late endosomes/ lysosomes , demonstrating that targeting motifs in CD4 drive the intracellular localization of HIV via CD4-mediated endocytosis . We next examined the functional consequences of CD4 altered trafficking . The lentiviral constructs ( Fig 3A ) were transduced into HEK-Blue cells to measure NF-\\u03baB activation . Notably this cell type is not optimized to produce IFN , therefore only the NF-\\u03baB signaling arm was tested in this system . R848 , a TLR7/8 agonist , was used as a positive control and strong NF-\\u03baB activator that rapidly accumulates in late endosomes [42] . The TLR9 agonist CpGB was used as a negative control in these TLR7-expressing cells and did not induce NF- \\u03baB activation in any condition ( Fig 6 ) . Whereas HIV did not induce NF- \\u03baB activation in CD4-WT and CD4-STOP expressing cells , it induced NF-\\u03baB activation in CD4-DEC and CD4-LAMP expressing cells . These results support the spatiotemporal model of cell signaling , whereby partition into early or late endosomes regulates IFN vs NF-\\u03baB signaling [25] . HIV traffics to early endosomes in pDC and in CD4-WT expressing cells , and induces weak NF-\\u03baB signaling , whereas retargeting CD4 to lysosomes allows for NF-\\u03baB activation by HIV . We also examined more precisely why native CD4 delivers HIV into early endosomes . Endocytosis and intracellular CD4 trafficking is dependent on a dileucine motif in its intracellular domain , and is regulated by phosphorylation of an adjacent Serine ( Ser408 ) . Phosphorylation of Ser408 , as occurs with phorbol ester ( PMA ) , dramatically enhances CD4 delivery from the cell surface to the lysosomes [43] . In contrast to PMA stimulation , HIV activation of human pDCs does not cause marked CD4 internalization at early timepoints . Whether pDC are unstimulated or stimulated with HIV , CD4 internalization is grossly unchanged , whereas PMA stimulation , in the absence or presence of HIV , causes internalization at early timepoints ( Fig 7 ) . To investigate whether CD4 phosphorylation on Ser408 targets CD4 and HIV to late endosomes/ lysosomes , we generated two mutations of the serine residue: one to alanine to abrogate phosphorylation ( CD4-SA ) , and one to the phospho-mimic Glutamic Acid ( CD4-SE ) , and produced HEK cells stably expressing these CD4 mutants . Intracellular localization of HIV and CD4 was monitored by microscopy . As shown in Fig 8 , HIV co-localized with CD4 and appeared less membrane associated in CD4-SE cells , whereas it remained predominantly on the surface of CD4-SA cells . In CD4-SA cells , intracellular HIV accumulated in EEA1+ and TfR+ compartments ( Fig 9 ) , similarly to WT-CD4 cells ( Fig 4 ) , whereas HIV accumulated in Lamp-1+ and lysotracker+ compartments in CD4-SE cells ( Fig 10 ) , similarly to CD4-DEC cells and CD4-Lamp cells ( Fig 5 ) . To test the functional consequences , we transduced HEK-Blue cells with CD4-SA and CD4-SE , as compared to CD4-WT , to measure NF-\\u03baB activation , as above . Whereas HIV did not induce NF-\\u03baB activation in CD4-WT and CD4-SA expressing cells , it induced NF-\\u03baB activation in CD4-SE expressing cells ( Fig 11 ) . Thus , CD4 phosphorylation on Ser408 appears to target CD4 and HIV to late endosomes/ lysosomes , whereas it is routed to early endosomes in its absence ( CD4-SA and CD4-WT ) . Poor specificity of available anti-phospho-Ser408 CD4 antibodies precluded the possibility of studying the phosphorylation state of Ser408 CD4 in primary pDCs after HIV activation . While our data strongly suggest that HIV trafficking and subsequent immune signaling in pDC is driven by HIV envelope/CD4 interactions , we tested an alternative hypothesis of pDC signaling , that the strength of TLR signaling in the early sorting endosomes determines the trafficking and subsequent signaling of TLR agonists . Previous work showed that TLR signaling in other cell types accelerates endosomal maturation through TLR-induced p38 mitogen-activated protein kinase ( p38 ) signaling , as the absence of a TLR/MyD88 signal diminishes phagosome maturation [44] . TLR activation also activates lysosomal function in myeloid DC , which could further influence TLR signaling pathways [45 , 46] . According to this model , strong TLR triggering in the early endosome would accelerate endosomal maturation , and endocytosed viruses or oligonucleotides would rapidly reach late endosomes for NF-\\u03baB signaling [25 , 33] . Using murine pDC differentiated from TLR7-/- and TLR9-/- bone marrow , we tested this hypothesis . As HIV cannot be used in murine immune systems , we tested TLR9 agonists , focusing on CpGB . CpGB has been shown to rapidly traffic to late endosomes/ lysosomes in mouse and human pDC [25 , 26] , and it induces strong NF-\\u03baB signaling and pDC maturation [22] . According to this model where TLR signaling induces endosomal maturation , CpGB would traffic rapidly to lysosomes in wild type ( WT ) or TLR7-/- pDC , but would fail to traffic to lysosomes in murine TLR9-/- pDC . Because MyD88 knockout DCs lacking TLR signaling have been described as having an altered constitutive rate of endosomal maturation [44 , 47] , we also constructed a control oligonucleotide lacking stimulatory CpG motifs ( due to inversion of the CpG motif into a GpC dinucleotide ) which lacks TLR triggering activity . Wild type , TLR7-/- , and TLR9-/- murine pDC were derived from bone marrow with Flt3 ligand , as previously described [48] and purified >85% ( S3A Fig ) . pDCs were incubated with FAM-labeled CpGB or nonactivating FAM-labeled GpC control and were imaged by flow cytometry and live microscopy . We first confirmed the specificity of the various pDC TLR knock out cells by evaluating the expression of the maturation molecule CD86 after overnight incubation of wild type , TLR9-/- , and TLR7-/- pDC with TLR7 agonist R848 and TLR9 agonist FAM-CpGB . As expected , R848 matured WT and TLR9-/- pDC but did not mature TLR7-/- pDC , whereas FAM-CpGB matured WT and TLR7-/- pDC but did not mature TLR9-/- pDC . pDC stimulated with FAM-GpC did not mature WT pDC due to the lack of CpG immunostimulatory motifs ( S3B Fig ) . We then monitored trafficking of FAM-CpGB to lysosomes in WT , TLR7-/- and TLR9-/- pDC by microscopy , by measuring co-localization with Lysotracker Red . Across a z stack spanning the midplane of the cell , we observed that both FAM-CpGB in WT , TLR7-/- and TLR9-/- pDC ( S4A and S4C Fig ) , and FAM-GpC in WT pDC ( S4B and S4D Fig ) , rapidly and extensively trafficked to lysosomes within 15-20min . Thus , TLR activation did not affect intracellular trafficking of TLR agonists in murine pDC , suggesting that an alternative model also exists for human pDC . Altogether , these data demonstrate that divergent HIV-1 sensing by pDC is mediated by CD4-HIV envelope interactions . \\n\\n## Discussion\\n Although many receptors and signaling pathways have been shown to modulate TLR signaling pathways in pDC [49] , it is likely that the functional outcome of TLR signaling is determined upstream and early on . The spatiotemporal model of TLR signaling [25] was proposed to account for the functional flexibility of pDC in response to TLR signaling , and argues that the surface or intracellular localization of TLR signaling initiation determines which downstream signaling pathway is triggered , with differing functional outcomes [50] . This is because each compartment is associated with different adaptors and signaling platforms , specialized in inflammatory cytokine secretion and NF-\\u03baB activation , or signaling through IRF7 and IFN production [25 , 33 , 50] . As HIV steadily traffics to early endosomes in pDC , a compartment associated with IFN signaling [25] , our goal was to understand how early HIV trafficking is regulated in pDC and how it affects pDC functional response . We studied whether HIV trafficking in pDC involves envelope-receptor interaction and targeting signals in endocytic receptors . A hybrid virus was constructed , where HIV envelope was replaced by influenza hemagglutinin envelope , while maintaining all other HIV structural components unchanged ( HA-HIV ) . In contrast to HIV , influenza is rapidly endocytosed by pDC and triggers a strong NF- \\u03baB activation , secretion of inflammatory cytokines , and maturation of pDC [22 , 51] . Strikingly , HA-HIV was rapidly routed to late endosomes/ lysosomes in pDC , contrary to HIV with its native envelope . Furthermore , it induced early secretion of inflammatory cytokines and strong pDC maturation , in a manner and kinetics similar to influenza virus . This demonstrates that virus envelope directly determines HIV trafficking and pDC phenotype . Despite different structural components and nucleic acids , HA-HIV and influenza induced a similar functional response in pDC , which strengthens the importance of viral envelope in determining pDC phenotype . This latter extended to the unresponsiveness of pDC to further stimulation , whereas HIV stimulated pDC could be re-stimulated to produce IFN . HA-HIV triggers IFN secretion , although at lower levels than Flu itself . This may be due to kinetics differences in trafficking to late endosomes and activation of negative signaling pathways or exhaustion . Trafficking to late endosomes , NF-\\u03baB activation and pDC maturation correlates with a state of refractoriness , likely established early during stimulation , and already evidenced by a global cytokine shutdown after a few hours . As shown in Fig 1A , this shutdown occurs earlier for HA-HIV than for Flu , at the time when IFN secretion is starting to be amplified . It is likely HA-HIV traffics significantly faster to late endosomes and triggers early cytokine shutdown . Another example of HIV pseudotyping is VSV-G pseudotyped HIV , where VSV-G from vesicular stomatitis virus is used to allow HIV uptake and infection of many cellular subtypes . The putative receptor for VSV-G has been recently suggested to be LDL receptor [52] , which traffics mainly to recycling endosomes but rarely to lysosomes . In accordance with this trafficking pattern and localization in early endosomes , VSV-G-pseudotyped HIV behaves mostly like HIV with a native envelope to trigger high levels of IFN but little pDC maturation [12] . In addition to demonstrating that viral envelope determines HIV localization and pDC phenotype , HA-pseudotyped HIV may also provide a tool to study HIV antigen presentation and vaccine design [53] , as it enhances expression of MHC and co-stimulatory molecules on pDC , and influenza itself triggers a developmental program suited for antigen presentation in pDC [21 , 54] . If the nature of the viral envelope dictates HIV trafficking in pDC , it may be due to intracellular targeting motifs present in the viral receptor ( s ) [38] . HIV is mainly taken up through CD4 in pDC , and we tested whether altering the intracytoplasmic domain of CD4 would affect HIV trafficking and TLR signaling . Indeed , we observed that swapping CD4 intracytoplasmic domain for DEC205 or Lamp1 intracytoplasmic domain dramatically re-routed HIV into late endosomes/ lysosomes in CD4 expressing cells , whereas intracellular HIV was localized predominantly in early endosomes in cells expressing native CD4 . DEC205 and Lamp1 contain lysosomal targeting motifs which are likely responsible for constitutive CD4 and HIV targeting to late compartments . In addition , redistribution of HIV to late endosomes was accompanied by activation of NF-\\u03baB , not observed when HIV accumulates in early endosomes , again consistent with the spatiotemporal model of TLR signaling . CD4 contains a dileucine motif in its intracytoplasmic domain , which is essential for CD4 endocytosis [36] . In addition , two adjacent Serines , Ser408 and Ser415 , can be phosphorylated and impact CD4 endocytosis and distribution . Completely deleting these motifs by removing the whole CD4 intracytoplasmic domain indeed almost completely abrogated CD4-mediated HIV endocytosis . However , phosphorylation of Ser408 not only enhances CD4 endocytosis [36] , but also redirects CD4 to lysosomal compartments [43] . Ser408 phosphorylation enhances its association with clathrin Adaptor protein AP-1 and AP-2 [36] . In our experiments , mutating Ser408 to Glutamic acid ( CD4-SE ) to mimic Ser408 phosphorylation , induced a complete redistribution of CD4 and HIV into lysosomes ( Fig 10 ) suggesting that in pDC , HIV traffics by default to recycling endosomes due to the CD4 dileucine motif , in the absence of Ser408 phosphorylation . These results are supported by our CD4 internalization studies where we found that HIV-activation of pDC does not seem to alter CD4 internalization , as compared to PMA-activation . Similarly , in HIV infected cells , Nef triggers endocytosis and degradation of CD4 through a dileucine based motif [55 , 56] and CD4 endocytosis and targeting to lysosomes are encoded in different regions of the Nef protein [56 , 57] . The relatively stable localization of HIV in early endosomes , observed for as long as 18 to 24h , remains unexplained . Furthermore , we observed strong co-localization of CD4 with HIV throughout the course of the study . Although the interaction between CD4 and HIV envelope may be stable enough to maintain CD4-HIV co-localization for a prolonged period of time , other explanations are possible . A recent study described in detail how endocytosis of HIV is coupled to dynamin-dependent endocytosis and partial fusion with plasma and endosomal membrane [58] , which may tether HIV envelope and CD4 to the endosomal membrane in the absence of fusion . Furthermore , pDC possess specialized large perinuclear intracellular stores of MHC-I molecules , with characteristics of recycling endosomes in immature pDC , that can be used as sites for rapid MHC-I loading and peptide presentation [21] . These intracellular stores may represent the stable compartment in which HIV accumulates in pDC , and prolonged localization in these early recycling endosomes may ultimately have important consequences for HIV antigen presentation . pDC may harbor HIV in these structures until activated by a maturation stimulus . Indeed , pDC are capable of HIV antigen cross-presentation [17] , and cross-presentation is strongly enhanced by maturation-inducing stimuli [59 , 60] . Upon influenza activation , stored MHC-I molecules are translocated to the cell surface for efficient cross-presentation by pDC [21] , indicating that the process of maturation drives antigen presentation . On the other hand , the non-acidic environment and limited access to MHC-II compartments may prevent efficient MHC-II peptide generation and association with MHC-II molecules . In addition , MHC-II clustering and antigen presentation by pDC is dependent on NF-\\u03baB [51 , 61] , which HIV weakly induces due to localization in early endosomes . The lack of pDC maturation induced by HIV might prevent effective cross-presentation and MHC-II restricted presentation , due to localization in early endosomes and weak NF-\\u03baB activation . Thus , the particular compartmentalization of HIV can affect HIV antigen presentation . As shown here , HA-pseudotyped HIV , which traffics to late endosomes and activates NF- \\u03baB , may serve as a tool to enhance cross-presentation of HIV antigens by pDC . We also tested whether TLR signaling itself alters maturation of endocytic compartments in pDC as was previously demonstrated in the case of murine macrophages , where TLR/ MyD88 signaling induced marked phagosome maturation , possibly through p38 MAP kinase activation [44] . We tested this model in murine pDC , and compared trafficking of CpGB in WT , TLR7-/- or TLR9-/- pDC . We did not observe any difference in trafficking of CpGB in these conditions , and internalized CpGB was rapidly found within late endosomes/ lysosomes whether TLR signaling had occurred or not . As MyD88-/- cells have altered endosomal maturation [47] , we also used a non-stimulatory type B oligonucleotide ( lacking immunostimulatory CpG motifs ) in WT pDC . However , its trafficking was identical to the stimulatory CpGB oligonucleotide . Furthermore , as described previously [44 , 62] , the lack of TLR signaling , as with nonstimulatory GpC , decreased the speed of CpG endocytosis ( S4E Fig ) , however all oligonucleotides localized identically . These data argue against the above model , whereby rate of endocytosis and/ or agonist potency drive its intracellular localization , and establish trafficking as independent of TLR signaling in pDC [47 , 63] . Using purified human pDCs and viruses , we demonstrate that HIV trafficking in pDC at early timepoints is determined by the initial envelope-receptor ( CD4 ) interaction , and is regulated by receptor targeting motifs . Whether this is still valid in the case of cell-associated virus , this remains to be determined . Engineering of viral envelope for increased pDC maturation and antigen presentation ( e . g . HA-HIV ) or for increased IFN secretion [64] may prove useful for vaccine design and modulation of chronic immune activation in HIV disease . \\n\\n## Materials and Methods\\n PBMCs were separated on Ficoll-Hypaque ( Amersham Biosciences ) from buffy coats ( New York Blood Center ) . pDC were purified by BDCA-4 magnetic bead separation ( Miltenyi Biotec ) as described previously [12] , with a purity ranging from 80 to 95% . Cells were cultured in RPMI 1640 Glutamax ( Invitrogen ) with 5% PHS ( Innovative Research , MI ) , gentamycin , and HEPES . HIV-1MN ( X4-tropic ) were produced at the AIDS Vaccine Program , National Cancer Institute as previously described [12 , 23 , 65] . Plasmids encoding JRFL HIV-1 envelope , JOTO HIV-1 envelope , REJO HIV-1 envelope , pNL43-\\u0394Env-vpr+-luc+ and pCAGGS ( human airway trypsin-like protease to cleave HA0 to HA1 and HA2 ) were provided by Carol Weiss ( FDA , Silver Spring , MD ) , plasmids encoding CMV/R Influenza H1 ( A/PR8/8/34 ) ( VRC 7702 ) , CMV/R influenza A/PR/8/1934 NA ( VRC 9776 ) , CMV/8R A/Thailand/1Kan-1/2004 H5 ( VRC 7705 ) , and CMV/8R A/Thailand/1Kan-1/2004 NA ( VRC 7708 ) , and were provided by Gary Nabel and Chih-Jen Wei ( Vaccine Research Center , NIH , Bethesda , MD ) . Plasmids encoding vpr-gfp and MN HIV pNL4-3 \\u0394vpr were obtained from David Ott and Jeffrey Lifson ( AIDS Vaccine Program , Frederick , MD ) . Plasmids encoding HIV Gag-iGFP JRFL were obtained from Benjamin Chen ( Mount Sinai School of Medicine , NY , NY ) . The influenza hemagglutinin viral pseudotypes were generated by calcium phosphate co-transfection of 3 . 0x106 HEK cells in a 10cm2 dish with 10ug HIV core ( pNL43-\\u0394Env-vpr+-luc+ ) , 400ng hemagglutinin envelope ( VRC 7702 , VRC 9776 , VRC 7705 , or VRC 7708 ) , 100ng PR8 NA ( VRC9776 ) , and 100ng HAT pCAGGS , with media change after 6 hours , and viral harvest at 48 hours . HIV pseudotypes were generated by calcium phosphate co-transfection of 3 . 0x106 HEK cells in a 10cm2 dish with 10ug HIV core ( pNL43-\\u0394Env-vpr+-luc+ ) and 6ug Env in pCDNA3 . 1 ( e . g . REJO , JOTO , JRFL ) . HIV Gag-iGFP is a full-length molecular clone of HIV derived from pNL4-3 that packages GFP inserted into the Gag protein between the MA and CA domains of Gag , with JRFL Env cloned into the place of the NL4-3 Env . To generate Gag-iGFP virions for CD4-expressing HEK experiments , 20ug of plasmid was transfected using the calcium phosphate method , with media change at 6 hours and transfection for 48 hours . For all viruses , transfection supernatants were filtered through a 0 . 45uM membrane , pelleted through a 20% sucrose cushion at 25 , 000g for 2hrs at 4\\u00b0C . Pelleted viruses were resuspended in PBS , aliquoted , and stored at -80\\u00b0C until use . HIV virions were quantified using p24 ELISA ( AIDS Vaccine Program ) and HA-HIV viruses were quantified using turkey hemagglutination inhibition assay for hemagglutination unit ( HAU ) as well as p24 ELISA . PR8 influenza was provided by David Levy ( NYU ) and influenza packaging GFP was provided by Jesse Bloom ( Fred Hutchinson Cancer Research Center , Seattle , WA ) as previously described [34 , 35] and were quantified by using turkey hemagglutination inhibition assay for hemagglutination unit . Purified pDC were stimulated at 50 , 000 cells/100uL media at 37\\u00b0 C with 5% CO2 with: MN HIV 300ng ( AIDS Vaccine Program , National Cancer Institute ) , HIV pseudotyped HIV envelopes 300ng , type B CPG oligodeoxyribonucleotide ( CpGB ) 2 ug 5\\u2019 T*C*G*T*C*G*T*T*T*T*G*T*C*G*T*T*T*T*G *T*C*G*T*T*-3\\u2019 where asterisks indicate a phosphorothioate bond ( IDT ) , Resiquimod ( R848 ) 10\\u03bcM ( 3M Corporation , St . Paul , MN ) , influenza virus PR8 HAU 10 heat inactivated at 56\\u00b0C for 30 minutes in a water bath , or HA-HIV HAU 10 . For intracellular staining , brefeldin A was added after 30 minutes , 2 hours , 6 hours , or 12 hours; at 24 hours cells were washed , fixed , and stained with PE-conjugated CD123 PE ( BD Biosciences ) , APC-conjugated TNF\\u03b1 ( eBioscience ) , and FITC-conjugated IFN-\\u03b1 ( BD Biosciences ) in 0 . 05% saponin , and analyzed by FACS . Following incubation of pDCs ( 50 , 000 cells/100uL ) with HIV MN and HIV JRFL , influenza , and HA-HIV , culture supernatants were tested for IFN and TNF by IFN\\u03b1 by ELISA ( PBL Interferon Source ) and human inflammatory kit CBA ( Abcam ) , respectively , following manufacturer instructions . For surface staining after overnight incubation , cells were stained with CD123 PE , CD86 APC , and HLA-DR PerCP ( BD Pharmigen ) , washed , fixed with 4%PFA , and analyzed by FACS . For IFN\\u03b1 restimulation experiment , pDCs were incubated overnight with influenza , HIV , HA-HIV , or CpGB , washed , and then incubated again with the same agonists . Culture supernatants were tested after the first and second overnight incubation for IFN\\u03b1 by ELISA ( PBL Interferon Source ) . CD4 expression plasmid ( pcDNAI ) was provided by Nathaniel Landau ( NYU , NY , NY ) . It was used as a template to mutate CD4 by overlapping PCR . Mutated CD4 were then inserted into pLenti vectors , and lentiviruses were produced using Mirus TransIT co-transfection of 3 . 0x106 HEK cells in a 10cm2 dish for 72 hours with 1 . 45 ug VSVg , 2 . 05 ug RSR-Rev , 2 . 9 ug pMDL plasmids ( provided by Dr Landau ) and 8 . 7 ug pLenti-CD4 constructs ( WT , STOP , DEC , LAMP , SE , SA ) . CD4-STOP was encoded of the first 425 amino-acids , including the transmembrane domain of CD4 . CD4-DEC contained the full extracellular domain of CD4 ( 397 amino acid in the immature form ) and the transmembrane and intracytoplasmic domain of DEC205 ( 56 amino acids ) . CD4-Lamp encoded the full extracellular and transmembrane domain of CD4 , and 12 amino acids of Lamp intracytoplasmic domain . CD4-SE and CD4-SA was identical to CD4 WT , except for a mutation of Serine 408 ( mature form ) for glutamate and alanine , respectively . CD4-expressing HEK cell lines were generated by infecting 3 . 0x106 HEK cells in a 10cm2 dish with lentivirus transfection supernatants and CD4 expression was maintained in the presence of puromycin . C57BL/6 wild type , TLR7-/- and TLR9-/- double-knockout mouse femurs were provided by B . Pulendran , Emory University . Mice were maintained in specific-pathogen-free conditions at the Emory Vaccine Center vivarium in accordance with all animal protocols reviewed and approved by the Institute Animal Care and Use Committee of Emory University . Bone marrow cells were isolated by flushing femurs with PBS supplemented with 2% heat inactivated FBS . BM cells were resuspended in Tris-ammonium chloride at room temperature for 1 minute to lyse RBC , washed , then cultured in RPMI 1640 Glutamax ( Invitrogen ) with 10%FBS , 1nM sodium pyruvate , 10mM HEPES buffer , 100 units/mL penicillin , 100ug/mL streptomycin , 2mM L-glutamine , 1% MEM nonessential amino acids . BM cells were cultured for 8 days at 1 \\u00d7 106 cells/ml in 24-well plates in culture medium supplemented with 200 ng/ml recombinant murine Flt-3 ligand ( Peprotech ) as previously described [48] . After 8 days pDCs were purified from BM cells using the mouse plasmacytoid dendritic cell isolation kit II ( Miltenyi ) with a purity ranging from 85 to 95% and cells were phenotyped by CD11c PerCP and PDCA1 APC ( BD Pharmigen ) . Purified murine pDC ( WT , TLR7-/- , and TLR9-/- ) were stimulated overnight at 50 , 000 cells/100uL media at 37\\u00b0C with 5% CO2 with R848 10\\u03bcM ( 3M ) , 5\\u2019FAM-CpGB 5\\u2019 TCGTCGTTTTGTCGTTTTGTCGTT-3\\u2019 2 ug ( IDT ) , or 5\\u2019FAM-GpC 5\\u2019-TGCTGCTTTTGTGCTTTTGTGCTT-3' 2ug ( IDT ) , both with phosphodiester backbones , and were stained with CD11c PerCP , PDCA1 APC , and CD86 PE ( BD Pharmigen ) . Purified murine pDC ( 200 , 000 cells/200uL ) were stimulated in 0 . 01% Poly L-lysine ( Sigma ) coated 8 chamber polystyrene vessel tissue culture treated glass slides ( CultureSlides BD Falcon ) in 10% FBS culture media ( 200uL ) with FAM-CpGB 2ug or FAM-GpC 2ug and stained with 1uM lysotracker ( Invitrogen ) . Cells were imaged by live microscopy for one hour using the Advanced Precision PersonalDV Imaging system , with temperature ( 37\\u00b0C ) and C02 ( 5% ) humidity control , at a size of 512 X 512 pixels and a bit depth of 16 using a 60X , 1 . 4 N . A . oil objective lens . Images were deconvoluted using the DeltaVision deconvolution system and analyzed using ImageJ . Purified human pDC ( 50 , 000 cells/50uL ) were stimulated in Fibronectin ( Corning ) coated Ibidi imaging chambers \\u03bc-Slide VI0 . 4 ( Ibidi , Madison , WI ) in 5% PHS culture media ( 200uL ) with GFP PR8 influenza 10 HAU , GFP HA-HIV ( 10HAU/500ng p24 ) or GFP JRFL pseudotyped HIV 500ng p24 and stained with 1uM lysotracker ( Invitrogen ) . Live imaging was carried out after 30 minutes , 2\\u20134 hours , or after overnight stimulation of cells , using a Yokogawa CSU-X1 spinning disk mounted on a Zeiss AxioObserver Z1 and controlled by MetaMorph under conditions that were reproduced across all experiments . 488nm and 561nm laser lines were generated by a Prairie Technologies Aurora solid state laser and fluorescence and brightfield ( phase ) images were captured with a Hamamatsu EM-CCD C9100 digital camera set at an EM gain of 11 MHz , a size of 512 X 512 pixels and a bit depth of 16 using a 63X , 1 . 4 N . A . oil objective lens . Temperature ( 37\\u00b0C ) , C02 ( 5% ) , and humidity were controlled using a Tokai Hit incubator . CD4 expressing HEK cells were incubated ( 200 , 000 cells/200 \\u03bcl ) with purified NA/LE mouse anti-human CD4 ( 10 \\u03bcg/ml; BD Pharmigen ) or isotype control purified NA/LE mouse IgG1\\u03ba for 30 minutes . HIV Gag-iGFP was then added , and cells were placed back in culture for 18 hours . Cells were washed , fixed with 4% PFA , and analyzed by FACS . Purified human pDC ( 100 , 000 ) were incubated for 18 hours with GFP PR8 influenza 10 HAU , GFP HA-HIV ( 10HAU/500ng p24 ) or GFP JRFL pseudotyped HIV ( HIV Gag-iGFP JRFL ) 500ng p24 in 8 chamber polystyrene vessel tissue culture treated glass slides ( CultureSlides BD Falcon ) in 10% FBS ( Gibco ) culture media ( 200uL ) then cells were washed x2 , fixed with 4% paraformaldehyde , permeabilized with 0 . 1% Triton X-100 , blocked with 0 . 5% BSA in PBS , stained with mouse anti-EEA1 ( 0 . 5 ug; BD biosciences ) at 4\\u00b0C overnight , washed x 2 then stained with donkey anti-mouse TRITC ( Jackson Immunoresearch ) for one hour at RT , then washed , dried , and mounted in DAPI Anti-fade GOLD ( Vector labs ) . Alternatively pDCs were incubated with media , phorbol 12-myristate 13-acetate 100ng/mL ( PMA ) , HIV , or PMA and HIV for 4 hours then cells were washed x2 , fixed with 4% paraformaldehyde , permeabilized with 0 . 1% Triton X-100 , blocked with 0 . 5% BSA in PBS , and then stained with CD4 PE ( BD Pharmigen ) for one hour , washed x2 , dried , and mounted in DAPI Anti-fade GOLD ( Vector labs ) . HEK cells transduced with CD4 mutants ( 25 , 000 ) were incubated overnight in 8 chamber polystyrene vessel tissue culture treated glass slides ( CultureSlides BD Falcon ) in 10% FBS ( Gibco ) culture media ( 200uL ) . iGFP HIV 500ng was added to culture media for 2\\u20134 hours then cells were washed x2 , fixed with 4% paraformaldehyde , permeabilized with 0 . 1% Triton X-100 , blocked with 0 . 5% BSA in PBS , and then stained with CD4 PE ( BD Pharmigen ) for one hour . Alternatively , cells were stained with mouse anti-EEA1 ( 0 . 5 ug; BD biosciences ) , mouse anti-transferrin receptor ( 0 . 5ug; Invitrogen ) , mouse anti LAMP-1 H4A3 ( 0 . 5ug; Developmental Studies Hybridoma Bank University of Iowa ) at 4\\u00b0C overnight , washed x 2 then stained with donkey anti-mouse TRITC ( Jackson Immunoresearch ) for one hour at RT , then washed , dried , and mounted in DAPI Anti-fade GOLD ( Vector labs ) . Fixed cells were imaged using a Zeiss LSM 880 confocal microscope configured to generate laser lines at 488nm and 561nm , as well as transmitted light . Images were scanned using a 100X , 1 . 46 N . A . oil objective lens at a size of 1024 X 1024 pixels and a bit depth of 12 . Images were captured using sequential ( multitrack ) acquisition to avoid bleedthrough of signal between channels . Colocalization of fluorescently labeled ligands ( green ) with endosomal markers or CD4 receptor ( red ) were analyzed quantitatively using JACoP plugin on ImageJ software or MetaMorph colocalization analysis . Image pair channel files of single image mid-planes were opened in as separate 16-bit ( grey scale ) image files . Individual cell regions were identified using the corresponding brightfield image , images were thresholded , and colocalization analysis was performed using the MetaMorph \\u201cmeasure colocalization\\u201d application , which measures the localization correlation between corresponding pixels in two paired images and provides the Manders correlation [66] . 50\\u2013100 cells were analyzed to generate the individual coefficients and data was plotted using GraphPad Prism software . CD4-expressing HEK-Blue hTLR cell lines were generated by infecting 3 . 0x106 HEK-Blue hTLR cells in a 10cm2 dish with CD4 lentivirus transfection supernatants . Stable CD4 expression of cell lines ( WT , STOP , DEC , LAMP , SA , and SE ) were cultured in the presence of puromycin and CD4 expression was verified by FACS and microscopy . 200 , 000 cells were placed in HEK-blue culture media overnight ( 200uL ) with CpGB , HIV , or R848 and NF-KB activation was measured using the HEK-blue detection system , with blue color depicting NF-\\u03baB activation , as measured by ELISA according to manufacturer specification ( Invivogen ) . Statistical significance was determined by the unpaired Student\\u2019s t test and analysis of variance .\\n\\n\",\n          \"## Keywords: computational and systems biology, cell biology, tools and resources\\n## Abstract\\nMesenchymal ( lamellipodial ) migration is heterogeneous , although whether this reflects progressive variability or discrete , 'switchable' migration modalities , remains unclear . We present an analytical toolbox , based on quantitative single-cell imaging data , to interrogate this heterogeneity . Integrating supervised behavioral classification with multivariate analyses of cell motion , membrane dynamics , cell-matrix adhesion status and F-actin organization , this toolbox here enables the detection and characterization of two quantitatively distinct mesenchymal migration modes , termed 'Continuous' and 'Discontinuous' . Quantitative mode comparisons reveal differences in cell motion , spatiotemporal coordination of membrane protrusion/retraction , and how cells within each mode reorganize with changed cell speed . These modes thus represent distinctive migratory strategies . Additional analyses illuminate the macromolecular- and cellular-scale effects of molecular targeting ( fibronectin , talin , ROCK ) , including 'adaptive switching' between Continuous ( favored at high adhesion/full contraction ) and Discontinuous ( low adhesion/inhibited contraction ) modes . Overall , this analytical toolbox now facilitates the exploration of both spontaneous and adaptive heterogeneity in mesenchymal migration . \\n\\n## Introduction\\n Cell migration is a profoundly heterogeneous phenomenon . Indeed , cells can adopt several substantially different migration modalities , including multicellular , amoeboid , and mesenchymal ( also termed lamellipodial or lamellipodial-driven ) migration , which can all be utilized by a broad range of cell types , as well as lobopodial migration , which has been observed specifically in fibroblasts ( Friedl and Wolf , 2010; Sahai , 2005; Petrie and Yamada , 2015; Petrie et al . , 2014; Welch , 2015; Friedl and Alexander , 2011 ) . These migration modes represent 'prespecified' cellular configurations ( i . e . cell states ) that are favored under particular conditions ( Friedl , 2004 ) . Switch-like conversion between these distinct modes is therefore part of the plastic , adaptive/compensatory response of cells to either environmental modulation ( Liu et al . , 2015; Starke et al . , 2014; Ruprecht et al . , 2015 ) or molecular targeting ( Sahai et al . , 2007; Sanz-Moreno et al . , 2008; Somlyo et al . , 2003; Wolf , 2003 ) . At a finer scale , heterogeneity is also evident within these migration modes , arising either stochastically or as an adaptive response to changing cues ( Geiger et al . , 2009; L\\u00e4mmermann and Sixt , 2009; Lock et al . , 2014; Winograd-Katz et al . , 2009 ) . Yet , partly due to a lack of adequate quantification , it remains unclear to what extent variation within modes occurs either progressively along a continuum or in a switch-like manner between as yet undefined intra-modal subpopulations . Specifically , in the case of amoeboid migration , three discrete sub-modalities have been observed , and these can co-exist under individual conditions ( Welch , 2015; L\\u00e4mmermann and Sixt , 2009; Yoshida , 2006 ) . By contrast , potentially distinct styles of mesenchymal ( lamellipodial ) migration , including keratocyte-like ( Barnhart et al . , 2015; Keren et al . , 2008 ) and fibroblast-like ( Abercrombie et al . , 1977; Theisen et al . , 2012 ) migration , have been described as arising largely in separate cell types or conditions . Therefore , despite some early suggestions ( Lewis et al . , 1982 ) , it has remained uncertain to what extent divergent sub-modalities of mesenchymal migration spontaneously emerge in parallel\\u00a0within uniform cell populations and conditions , and whether these modes are truly quantitatively distinct , or instead represent extremes in a broad phenotypic continuum . These questions are important because each discrete migration mode , with its unique internal logic and dependencies , may respond differently to altered regulatory cues \\u2013 thereby defining a multifaceted adaptive response . Such responses are exemplified by the seminal observation that amoeboid motility provides a compensatory escape route for tumor cells when mesenchymal migration is blocked via inhibition of proteolytic mechanisms or by confinement ( Wolf , 2003 ) . Identifying and characterizing such distinct modes within individual cell types and/or conditions may therefore be key to understanding heterogeneity in response to experimental or even clinical interventions . Unlike amoeboid migration , which requires little or no cell-matrix adhesion ( L\\u00e4mmermann et al . , 2008 ) , mesenchymal migration demands cell adhesion to the extracellular matrix ( ECM ) for effective force application ( Friedl and Wolf , 2010 ) . This adhesion is achieved through the interaction of integrin-mediated cell-matrix adhesion complexes ( CMACs ) with ECM ligands , while applied forces are generated by the F-actin cytoskeleton and actomyosin system ( Geiger et al . , 2009; Humphries et al . , 2015 ) . This arrangement defines the ECM \\u2013 adhesion \\u2013 F-actin axis that directly mediates mesenchymal migration ( Parsons et al . , 2010; Schwartz , 2010 ) . Integrins are central to the ECM \\u2013 adhesion \\u2013 F-actin axis and are both anchored and activated by binding through their extracellular domains to ECM ligands , such as collagen , laminin , and fibronectin ( Lewis et al . , 1982 ) . The composition , topology , and density of ECM ligands therefore play key roles in mesenchymal migration ( Geiger et al . , 2009 ) . Through interaction with integrin tails , talin acts a critical intracellular regulator of integrin activation ( Calderwood et al . , 1999; Kiss et al . , 2015; Moser et al . , 2009; Tadokoro , 2003 ) , and is also one of several CMAC proteins linking integrins to the F-actin cytoskeleton ( Schwarz and Gardel , 2012 ) . This enables cellular force application from F-actin , through CMACs , to the ECM , thereby driving membrane protrusion/retraction and cell translocation ( Small and Resch , 2005 ) . Furthermore , CMACs function both up- and downstream of the small GTPases Rac1 and RhoA ( Raftopoulou and Hall , 2004 ) , which modulate both F-actin polymerization and actomyosin contractility , the latter through the Rho kinase ( ROCK ) -mediated regulation of non-muscle myosin II ( Ridley , 2003; Riento and Ridley , 2003 ) . In fact , CMACs function as hubs for bi-directional chemical and mechanical information transduction across the plasma membrane , providing command and control of mesenchymal migration ( Hynes , 2002; Lock et al . , 2008 ) . Given the pivotal roles of CMACs and F-actin , their assessment can efficiently provide a broad estimation of how the complex machinery underlying mesenchymal migration is organized ( Lock et al . , 2014; Gardel et al . , 2010; Gupton and Waterman-Storer , 2006; Kim and Wirtz , 2013 ) . Such an approach , generally based on quantitative imaging , is especially effective when multiscale data capturing both cell behavior ( migration ) and organization ( e . g . CMAC and F-actin status ) is derived simultaneously on a per cell basis . This facilitates the leveraging of natural or induced heterogeneity to define: i ) the statistical structure of variation ( e . g . progressive or discrete ) within and between cell populations , as well as; ii ) key trends , dependencies and relationships within the cell migration system ( Lock et al . , 2014; Keren et al . , 2008; Kiss et al . , 2015; Kowalewski et al . , 2015; Ku et al . , 2012; Lee et al . , 2015; Lock and Str\\u00f6mblad , 2010 ) . Here , we extend on this approach by employing a tailored analytical toolbox to detect two quantitatively distinct sub-modalities of mesenchymal migration , thus illuminating the discrete nature of variation within this general migration archetype . We employ this unique suite of analytical tools to further characterize key aspects of the behavior , organization , and regulation of these divergent migration strategies . Collectively , this study now provides conceptual and practical capabilities to the cell migration research community , while also highlighting the importance of distinguishing individual mesenchymal migration modes as a vital precursor to understanding mesenchymal migration as a whole . \\n\\n## Results\\n To enable the detection of discrete mesenchymal migration sub-modalities , and thereafter , the comprehensive analysis of cellular organization , regulation , and adaptation underlying each mode , we have integrated a unique combination of analytical tools . These are now made freely available as a Matlab Toolbox: 'The Cell Adhesion and Migration Analysis Toolbox' \\u2013 along with the raw quantitative data underlying this study , sample image data to aid implementation , and explanatory documentation ( see doi:10 . 5061/dryad . 9jh6m ) . The features included in this Toolbox are also specified briefly in 'Materials and methods' . H1299 ( non-small cell lung carcinoma ) cells stably expressing EGFP-paxillin ( marker for CMACs ) and RubyRed-LifeAct ( marker for F-actin ) ( H1299 P/L cells [Lock et al . , 2014] ) were imaged via confocal microscopy on glass coated with 2 . 5 \\u00b5g/ml fibronectin ( FN ) . H1299 P/L cells moved individually and exclusively via mesenchymal ( lamellipodial ) migration . Nonetheless , we observed and classified two qualitatively different migration modes emerging within the clonally derived H1299 P/L cell population . We termed these migration modes 'Discontinuous' and 'Continuous' , reflecting their contrasting stepwise and smooth motion , respectively ( Figure 1 ) . Specifically , cells migrating in the Discontinuous mode cycle in sequence through at least three recognizable stages of movement , which we term 'lateral protrusion' , 'polarization' , and 'tail retraction' , as described in Figure 1A ( also see Video 1 ) . These stages produce dramatic changes in cell morphology and frequent changes in migratory direction . Such directional changes occur at least in part because lateral protrusions typically develop at approximately 90\\u00b0 to the axis of the preceding tail retraction . Overall , the stepwise nature of Discontinuous migration is highly reminiscent of previous descriptions of fibroblast-like migration ( Theisen et al . , 2012; Abercrombie et al . , 1970; Chen , 1981 ) . By contrast , cells migrating in the Continuous mode move progressively , with less frequent changes in cell morphology and motile direction , in a manner analogous to classical keratocyte-like migration ( Figure 1B and Video 2 ) ( Keren et al . , 2008 ) . Importantly , these migration modes emerge spontaneously in parallel under uniform experimental conditions within a clonally\\u00a0derived cell line ( Figure 1\\u2014figure supplement 1 and Video 3 ) . It is noteworthy that cells may traverse the same underlying substrate regions while stably occupying distinct modes ( see Video 3 ) , indicating that local environmental inconsistencies are not the principal cause of differential mode identity . Yet , inter-modal conversion is possible , occurring bi-directionally between modes ( Figure 1\\u2014figure supplement 2 and Videos 4 and 5 ) , indicating their profound plasticity . 10 . 7554/eLife . 11384 . 003Figure 1 . Classification of 'Discontinuous' and 'Continuous' modes of mesenchymal ( lamellipodial ) migration . ( A , B ) Confocal image sequences of individual H1299 cells stably expressing EGFP-paxillin ( cell-matrix adhesion complex [CMAC] marker; upper image panels ) and RubyRed-LifeAct ( F-actin marker; lower image panels ) migrating in either Discontinuous ( A ) or Continuous ( B ) modalities on fibronectin ( FN ) -coated ( 2 . 5 \\u00b5g/ml ) glass . Images are displayed with gray-scale inverted . Numbers denote time ( min ) . Schematics in blue ( Discontinuous , A ) and orange ( Continuous , B ) depict typical cell morphology changes associated with each migration mode . Note stepwise cycles of: lateral protrusion ( in directions of dashed arrows ) ; cell polarization; and tail retraction ( in direction of solid arrows ) that recur during Discontinuous migration , and that lateral protrusion tends to occur at 90\\u00b0 to the preceding tail retraction . During Continuous migration , cell morphology is relatively stable , with many small protrusion ( open arrowheads ) and retraction ( closed arrowheads ) events producing smooth movement . Movies corresponding to cells shown in A and B are available in supporting material ( Videos 1 , 2 ) . ( C ) Quantification of frequencies of Discontinuous and Continuous modes . ( D ) Box plots of cell speed ( \\u00b5m/sec ) per migration mode . Boxplots show median values and inter-quartile ranges ( IQR , 25% to 75% ) . Notches indicate median +/\\u2212 1 . 57 * IQR/\\u221an ( approximates 95% confidence interval of the median , n = number of cell observations , see 'Materials and methods' ) . Statistical discernibility assessed by Wilcoxon rank sum test , p < 0 . 001 . ( E ) Cell trajectories in each mode were assessed via mean squared displacement ( MSD ) analysis and divided into quintiles ( 20% bins ) according to their migration cefficient ( related to speed of movement ) . Median values +/\\u22121 . 57 * IQR/\\u221an ( n = number of cell observations ) of a second MSD measure , persistence time ( related to migration direction stability ) , are plotted per migration coefficient quintile . Number of observations per quintile: Discontinuous = ~ 402; Continuous = ~ 95 . ( F ) A table summarizes results from visual inspection of several cell lines migrating on 2 . 5 \\u00b5g/ml FN , confirming the emergence of either one or both Discontinuous and Continuous migration modes in these cell types . Example image sequences are presented in Figure 1\\u2014figure supplement 3 . ( G ) A table summarizes results from visual inspection of H1299 cells migrating on several different extracellular matrix ligands , confirming the emergence of both migration modes under each condition . Example image sequences are presented in Figure 1\\u2014figure supplement 4 . Scale bars = 20 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00310 . 7554/eLife . 11384 . 004Figure 1\\u2014figure supplement 1 . Discontinuous and Continuous migration modes emerge in parallel under uniform conditions . A confocal image sequence showing the parallel emergence of Discontinuous and Continuous migration modes in H1299 P/L cells labeled with a far red membrane-dye on 5 \\u00b5g/ml fibronectin-coated glass . ( A ) De-noised ( left ) and segmented ( right ) images show a selected region from within a 4 x 4 image montage . Based on observed cell dynamics ( see Video 3 ) , cells were classified as in either the Discontinuous ( e . g . cell in blue box ) or Continuous mode ( e . g . cell in red box ) . ( B ) Stereotypical Discontinuous mode behaviors are displayed in a series of time points ( numbers denote minutes ) following the cell from the blue box in A . Specifically: Tail Retraction is denoted by solid arrows; Lateral Protrusion and Polarization are denoted by dashed double headed arrows; ( C ) The relatively stable morphology of Continuous mode migration is shown by the cell from the red box in A . It is noteworthy that both highlighted cells sequentially traverse the same region of the coated substrate while maintaining distinct migration modes ( see Video 3 ) , indicating that potential inconsistencies in the local environment could not be responsible for the parallel emergence of distinct modes . Scale bars = 10 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00410 . 7554/eLife . 11384 . 005Figure 1\\u2014figure supplement 2 . Discontinuous and Continuous migration modes can spontaneously inter-convert ( mode switching ) . Confocal image sequences show the spontaneous inter-conversion between Discontinuous and Continuous migration modes by H1299 P/L cells labeled with a far red membrane-dye on 5 \\u00b5g/ml fibronectin-coated glass . ( A ) De-noised ( upper ) and segmented ( lower ) time sequences ( numbers denote minutes ) of a single cell spontaneously transitioning ( switching ) from the Continuous migration mode to the Discontinuous migration mode ( see tracked cell in Video 4 ) . The approximate timing of the Mode Transition is indicated . ( B ) De-noised ( upper ) and segmented ( lower ) time sequences of a single cell spontaneously transitioning ( switching ) from the Discontinuous migration mode to the Continuous migration mode ( see tracked cell in Video 5 ) . The approximate timing of the Mode Transition is indicated . Scale bars = 10 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00510 . 7554/eLife . 11384 . 006Figure 1\\u2014figure supplement 3 . Discontinuous and/or Continuous migration modes recur in multiple cell types . Confocal image sequences exemplifying Discontinuous and/or Continuous migration on 2 . 5 \\u00b5g/ml fibronectin-coated glass by several cell types , including: BT549 ( breast epithelial [ductal carcinoma] ) ; MDA-MB-231 ( breast epithelial [adenocarcinoma] ) ; Hep-3 ( hepatocellular carcinoma ) ; and HS578T ( breast epithelial [carcinoma] ) . Cells were labeled with a membrane dye and imaged at 5-min intervals . Images are displayed with gray-scale inverted . Numbers denote time ( min ) . Movies corresponding to image sequences ( in the order shown ) are available in supporting material ( Videos 6\\u201311 , respectively ) . Scale bars = 20 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00610 . 7554/eLife . 11384 . 007Figure 1\\u2014figure supplement 4 . Discontinuous and Continuous migration modes recur in H1299 cells adhering to multiple extracellular matrix ligands . Confocal image sequences exemplifying H1299 cell migration in Discontinuous and Continuous modes on glass coated with various purified extracellular matrix ligands , including: fibronectin ( FN ) ; Laminin; Collagen Type 1; and Vitronectin . Cells were labeled with a membrane dye and imaged at 5-min intervals . Images are displayed with gray-scale inverted . Numbers denote time ( min ) . Schematics of sequences on FN detail the characteristic movement and morphology changes associated with each mode , as shown in Figure 1 . See movies corresponding to image sequences ( in the order shown but excluding fibronectin sequences; as this is shown in Videos 1 , 2 ) ( Videos 12\\u201317 , respectively ) . Scale bars = 20 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00710 . 7554/eLife . 11384 . 008Video 1 . H1299 P/L cell migration in the Discontinuous mode . High-resolution multiscale imaging of a single H1299 P/L cell expressing EGFP-paxillin ( green , CMAC marker ) and RubyRed-LifeAct ( red , F-actin marker ) during migration in the Discontinuous mode on 2 . 5 \\u00b5g/ml fibronectin . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00810 . 7554/eLife . 11384 . 009Video 2 . H1299 P/L cell migration in the Continuous mode . High-resolution multiscale imaging of a single H1299 P/L cell expressing EGFP-paxillin ( green , CMAC marker ) and RubyRed-LifeAct ( red , F-actin marker ) during migration in the Continuous mode on 2 . 5 \\u00b5g/ml fibronectin . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00910 . 7554/eLife . 11384 . 010Video 3 . Parallel emergence of Discontinuous and Continuous modes in H1299 P/L cells . Low-resolution imaging of H1299 P/L cells labeled with a membrane dye shows the parallel emergence of both Discontinuous ( e . g . dark blue outline , trajectory shown ) and Continuous ( e . g . light blue outline , trajectory shown ) modes under uniform conditions . Cells have been segmented and tracked to highlight morphologies and trajectories . Note that two cells in the Continuous mode ( yellow and lilac outlines ) pass through the substrate region traversed by the Discontinuous mode cell ( dark blue ) , yet these cells remain in the Continuous mode . This implies that these migration modes are not simply determined by ( possible ) local variations in , for example , ECM substrate ( 5 \\u00b5g/ml fibronectin ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01010 . 7554/eLife . 11384 . 011Video 4 . Migration mode transition from Continuous to Discontinuous motility . Low-resolution imaging of H1299 P/L cells labeled with a membrane dye , during random migration on 5 \\u00b5g/ml fibronectin . Cells have been segmented and tracked to highlight morphologies and trajectories . Note that the cell with light green outline ( trajectory shown ) transitions from Continuous to Discontinuous migration during the course of imaging . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01110 . 7554/eLife . 11384 . 012Video 5 . Migration mode transition from Discontinuous to Continuous motility . Low-resolution imaging of H1299 P/L cells labeled with a membrane dye , during random migration on 5 \\u00b5g/ml fibronectin . Cells have been segmented and tracked to highlight morphologies and trajectories . Note that the cell with light green outline ( trajectory shown ) transitions from Discontinuous to Continuous migration during the course of imaging . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 012 Under the conditions described above , ~80% of observations showed cells migrating via the Discontinuous mode , with the remaining 20% moving in the Continuous mode ( Figure 1C , see 'Materials and methods' for sample numbers relating to all experimental data ) . Automated cell tracking ( see 'Materials and methods' , [Lock et al . , 2014] ) revealed that cells in the Discontinuous mode migrate significantly faster than during Continuous migration ( Figure 1D ) . By applying mean squared displacement analysis of cell trajectories to assess measures related to cell speed ( migration coefficient ) and directionality ( persistence time ) , we found that Discontinuous migration is less directionally stable at any given speed ( Figure 1E ) . However , both modes of migration show a positive correspondence between speed and directionality , as recently reported ( Maiuri et al . , 2015 ) . Importantly , an extensive comparison of migratory behaviors in several additional cell lines ( Figure 1F , Figure 1\\u2014figure supplement 3 and Videos 6\\u201311 ) and during H1299 cell migration on alternative extracellular matrix ligands ( Figure 1G , Figure 1\\u2014figure supplement 4 and Videos 12\\u201317 ) indicated that Discontinuous and Continuous migration modes are consistently recurring phenomena . 10 . 7554/eLife . 11384 . 013Video 6 . BT549 cell migration in the Discontinuous mode . A single BT549 cell migrating in the Discontinuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01310 . 7554/eLife . 11384 . 014Video 7 . BT549 cell migration in the Continuous mode . A single BT549 cell migrating in the Continuous mode ( upper centre cell ) . Cropped from a larger image montage ( note , montage stitching can cause observable intensity boundaries in image; however , these have no effect on interpretation ) . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are\\u00a0not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01410 . 7554/eLife . 11384 . 015Video 8 . MDA-MB-231 cell migration in the Discontinuous mode . A single MDA-MB-231 cell migrating in the Discontinuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01510 . 7554/eLife . 11384 . 016Video 9 . MDA-MB-231 cell migration in the Continuous mode . MDA-MB-231 cells migrating in the Continuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01610 . 7554/eLife . 11384 . 017Video 10 . Hep3 cell migration in the Discontinuous mode . Hep3 cells migrating in the Discontinuous mode . Cropped from a larger image montage ( note , montage stitching can cause observable intensity boundaries in image; however , these have no effect on interpretation ) . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data is not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01710 . 7554/eLife . 11384 . 018Video 11 . HS578T cell migration in the Continuous mode . A single HS578T cell migrating in the Continuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01810 . 7554/eLife . 11384 . 019Video 12 . H1299 P/L cell migration in the Discontinuous mode on laminin . H1299 P/L cells migrating in the Discontinuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 \\u00b5g/ml laminin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01910 . 7554/eLife . 11384 . 020Video 13 . H1299 P/L cell migration in the Continuous mode on laminin . A single H1299 P/L cell migrating in the Continuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 50 \\u00b5g/ml laminin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 02010 . 7554/eLife . 11384 . 021Video 14 . H1299 P/L cell migration in the Discontinuous mode on collagen type 1 . H1299 P/L cells migrating in the Discontinuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 \\u00b5g/ml collagen type 1 . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 02110 . 7554/eLife . 11384 . 022Video 15 . H1299 P/L cell migration in the Continuous mode on collagen type 1 . H1299 P/L cells migrating in both Discontinuous and Continuous modes . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 5 \\u00b5g/ml collagen type 1 . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 02210 . 7554/eLife . 11384 . 023Video 16 . H1299 P/L cell migration in the Discontinuous mode on vitronectin . A single H1299 P/L cell migrating in the Discontinuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 1 \\u00b5g/ml vitronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 02310 . 7554/eLife . 11384 . 024Video 17 . H1299 P/L cell migration in the Continuous mode on vitronectin . A single H1299 P/L cell migrating in the Continuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 1 \\u00b5g/ml vitronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 024 Cell migration is the product of membrane dynamics that can be divided into protrusive and retractive processes ( Ridley , 2003 ) . We quantitatively compared the spatial and temporal dynamics of membrane protrusions and retractions between and within each migration mode based on defining membrane dynamics over the minimal imaging interval of 5 min ( Figure 2 ) ( Kowalewski et al . , 2015 ) . This revealed that protrusions share similar size ( area ) distributions in both modes ( Figure 2A ) , while retraction events are more extreme in size during Discontinuous migration , that\\u00a0is , more frequently very small or very large ( Figure 2B ) . When compared within\\u00a0each mode , membrane retractions have a broader size distribution than protrusions in the Continuous mode ( Figure 2C ) , although this is far more striking in the Discontinuous mode ( Figure 2D ) . To investigate membrane dynamics operating over longer time-scales , we calculated the probabilities of protrusions and retractions of a given size based on net cell shape/position changes occurring over intervals of between 1 and 15 image frames ( 5 to 75 min ) . This revealed a relatively unstructured pattern of dynamics in Continuous cells , although retractions tended to be smaller and protrusions larger ( Figure 2E ) . In contrast , Discontinuous migration consistently displayed a wide distribution of retraction sizes ( very small and very large ) , with protrusion sizes uniformly moderate ( Figure 2F ) . Notably , the relatively high probability of very large retraction events in Discontinuous cells corresponds with their observed tendency to undergo dramatic tail retraction events . Collectively , these results explain the spatial characteristics underlying stepwise or smooth cell motion during migration in Discontinuous or Continuous modes , respectively . 10 . 7554/eLife . 11384 . 025Figure 2 . Membrane protrusion and retraction dynamics are temporally decoupled and less coordinated during Discontinuous migration . ( A ) Probability distribution of membrane protrusion sizes ( \\u00b5m2 ) per 5-min interval during Discontinuous ( blue ) and Continuous ( orange ) migration . Subtraction of the Continuous from Discontinuous protrusion size probability distribution ( inset ) reveals no substantial or structured difference in these distributions ( probability difference , Y axis ) . ( B ) Probability distribution of membrane retraction sizes ( \\u00b5m2 ) per 5-min interval during Discontinuous ( blue ) or Continuous ( orange ) migration . Subtraction of the Continuous from Discontinuous retraction size probability distribution ( inset ) reveals that retraction sizes tend to be more extreme ( frequently small or large , rarely moderate ) in cells during Discontinuous migration ( probability difference , Y axis ) . ( C ) Subtraction of the protrusion from retraction size probability distribution ( 5 min intervals only ) during Continuous migration reveals little structure in probability differences . ( D ) Subtraction of the protrusion from retraction size probability distribution ( 5 min intervals only ) during Discontinuous migration reveals that retractions in this mode are consistently more extreme in size ( frequently small or large , rarely moderate ) than protrusions . ( E ) Surface plotting of probability differences ( retraction minus protrusion probability , per size ) over various time windows ( 1 to 15 frames; 5 to 75 min ) in cells during Continuous migration . Surface color-coding indicates where protrusions ( blue ) or retractions ( red ) are more common at a particular size . ( F ) Surface plotting of probability differences ( retraction minus protrusion probability , per size ) over various time windows ( 1 to 15 frames; 5 to 75 min ) in cells during Discontinuous migration . ( G ) Signal cross-correlation was calculated between protrusion size and retraction size fluctuations per cell during Continuous migration . Mean cross-correlation values ( Y axis , red = positive , blue = negative ) plotted as a surface , per time lag ( -12 to 12 frames , negative values indicate protrusion leads retraction , positive values indicate retraction leads protrusion ) , per time window ( ranging from 1 to 15 frames; 5 to 75 min ) . ( H ) Signal cross-correlation was calculated between protrusion size and retraction size fluctuations per cell during Discontinuous migration . Mean cross-correlation values ( Y axis , red = positive , blue = negative ) plotted as a surface , per time lag ( -12 to 12 frames , -ve values indicate protrusion leads retraction , +ve values indicate retraction leads protrusion ) , per time window ( ranging from 1 to 15 time points; 5 to 75 min ) . ( I ) Mean ( over all lags , per time window ) absolute cross-correlation values for Continuous ( orange ) and Discontinuous ( blue ) migration , /- 95% confidence intervals , n = number of cells ( see Materials and Methods ) . Statistical discernibility assessed by Friedman testing , p = 0 . 0012 . ( J ) Cells in each migration mode were divided into quintiles ( 20% bins ) based on instantaneous cell speed , and median protrusion areas /- 1 . 57 * interquartile range ( IQR , 25% to 75% ) / \\u221an ( approximates 95% confidence interval of the median , n = number of cell observations , see Materials and Methods ) were calculated per quintile . ( K ) Median retraction areas /- 1 . 57 * IQR/ / \\u221an ( n = number of observations ) were calculated per cell speed quintile . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 025 To understand the temporal dynamics underlying differences in migration mode behavior , we assessed the cross-correlation between protrusion and retraction size signals over time . This revealed strong temporal coordination ( high cross-correlation ) between protrusion and retraction events during Continuous migration , with a single cross-correlation peak around the -1 time-lag . This indicated that protrusion consistently led retraction , but that these dynamics are tightly coupled in time ( Figure 2G ) . This was true regardless of the time frame over which membrane dynamics were defined . In marked contrast , two cross-correlation peaks were apparent in the Discontinuous mode , with one peak stable around the -1 time-lag ( tightly coupled dynamics ) , and a second peak temporally offset with positive lags ( Figure 2H ) . This offset was dependent on the time frame used to define membrane dynamics , but clearly indicates a population of temporally decoupled membrane dynamics , wherein retraction leads protrusion events that are significantly delayed . This confirmed the visual impression of Discontinuous migration as a cyclical , stepwise process with asynchronous retraction ( i . e . tail retraction stage ) preceding protrusion ( i . e . lateral protrusion stage ) . In contrast , the morphological stability observed during Continuous migration is a consequence of the tight coupling of protrusion and retraction , producing motility without large-scale , transient size/shape changes . Visual impressions also suggested the Discontinuous migration mode to be less ordered/coordinated than the Continuous mode . The significantly lower average ( combining all time-lags ) protrusion-retraction cross-correlation values during Discontinuous migration ( Figure 2I ) confirmed reduced coordination of membrane dynamics in this migration mode . To complete our analysis of how membrane dynamics differentially contribute to migration in each mode , we assessed the relationship between cell speed and the size of membrane protrusions ( Figure 2J ) or retractions ( Figure 2K ) . Both modes exhibited similar dependencies . Remarkably , cell speed appears in each case to depend only slightly on the size of membrane protrusions , instead being largely determined by retraction size . The data described above detail significant quantitative differences between both the cell motion and membrane dynamics comprising each migration mode behavior . Yet , how these divergent behaviors emerge from the organization of underlying macromolecular machineries remains unclear . We therefore extended our image analyses ( see Materials and Methods and [Lock et al . , 2014] ) to derive a multivariate dataset of 55 organizational features ( detailed in Figure 3\\u2014figure supplement 1 ) defining cellular-scale morphology as well as the state ( e . g . size , number , density , morphology , localization ) and dynamics ( e . g . motion , stability , rates of area/density change ) of critical macromolecular-scale machineries driving cell migration , namely , cell-matrix adhesion complexes ( CMACs; demarcated by EGFP-paxillin ) and F-actin ( demarcated by RubyRed-LifeAct ) . This multiscale organizational data was measured per cell , per time-point , thereby complementing corresponding measures of migration behavior from the same individual cells ( Figure 3\\u2014figure supplement 1 ) . Such integrated data allowed exploration of the organizational states that give rise to particular cell migration behaviors , based on information embedded in natural cell heterogeneity ( Lock et al . , 2014; Lock and Str\\u00f6mblad , 2010 ) . To first understand how much overlap exists between the organizational states underpinning Discontinuous and Continuous migration modes , we performed supervised clustering ( canonical vectors analysis , CVA ) of all cell observations based on their organizational features only ( Figure 3A ) . Strikingly , the two migration mode populations showed almost no overlap , indicating that their underlying patterns of organization can be quantitatively distinguished . We next visualized these organizational states via parallel coordinate mapping of 15 key organizational features ( Figure 3B ) . This revealed specific instances of dissimilarity in feature values as well as an overview of the collective organizational signatures associated with each mode . To estimate the contribution of specific features to the divergence between modes , we ranked organizational features by the coefficient values assigned to them within the first canonical vector of the CVA ( Figure 3C ) . We then compared feature value distributions between modes for a subset of these top-ranked features , finding highly significant differences in all cases ( Figure 3D\\u2013K ) . These analyses help to define the spectrum of differences in organization that spontaneously arise in parallel with the emergence of Discontinuous and Continuous migration mode behaviors . For instance , cells in the Discontinuous mode are: far smaller; less round ( more protrusive ) ; have fewer , smaller , less dense CMACs ( based on EGFP-paxillin intensity ) with less F-actin association ( based on background-subtracted RubyRed-LifeAct intensity [Li et al . , 2010] ) and shorter lifetimes ( less stable ) , resulting in a drastically decreased total adhesion area . 10 . 7554/eLife . 11384 . 026Figure 3 . Discontinuous and Continuous migration modes reflect quantitatively distinct cell states with unique signatures of underlying organization . ( A ) Canonical vectors analysis ( CVA ) -based multivariate clustering ( based on all 55 organizational features ( see Figure 3\\u2014figure supplement 1 ) , canonical vectors ( CVs ) 1 and 2 displayed , percentages indicate proportion of total variance per CV ) of cell observations ( see Materials and Methods ) during Discontinuous ( blue ) or Continuous ( orange ) migration show modes to be quantitatively distinct . Large circles with black outlines indicate population centers of mass . ( B ) Parallel coordinate mapping of key organizational feature values ( per cell observation ) detail the multivariate signatures associated with cells during Discontinuous ( light blue ) or Continuous ( light orange ) migration . Points of difference between these multivariate signatures are emphasized by plotting of median values for Discontinuous ( blue ) and Continuous ( orange ) modes . Wilcoxon rank sum testing ( per feature ) assessed statistically discernable differences ( p < 0 . 001 ) , except where shown in gray italics . ( C ) A list of organizational features ranked by their contribution ( coefficient values in canonical vector 1 , which contains 99% of total variance ) to the separation of migration modes in the canonical vectors space shown in ( A ) . Gray backgrounds highlight features for which value distributions are compared between Discontinuous ( blue ) and Continuous ( orange ) modes in ( D-K ) . ( D-K ) Features compared indicated on boxplot Y-axes . Boxplots show median values and inter-quartile ranges ( IQR , 25% to 75% ) . Notches indicate median /- 1 . 57 * IQR/ / \\u221an ( approximates 95% confidence interval of the median , n = number of cell observations , as in ( A ) . P values reflect Wilcoxon rank sum testing as in ( B ) . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 02610 . 7554/eLife . 11384 . 027Figure 3\\u2014figure supplement 1 . Definition of organizational and behavioral features . This table defines the names , relevant measurement units , and a brief description of quantitative features derived through multiscale image analysis . Features are numbered 1-60 in correspondence with the order and numbering presented in Spearman's correlation heatmaps in Figure 4 . Note , of these 60 features , 55 define Organizational features ( light gray , used in multivariate clustering analyses ) , while 5 define the Behavioral features assessed herein ( dark gray ) . Additional terms are defined above relevant columns . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 027 Having defined a spectrum of differences in key organizational features underlying migration modes , we next explored how the correlative relationships between these features might also vary . To this end , we mapped the complete network ( including both organizational and behavioral features ) of pairwise inter-feature Spearman\\u2019s correlation coefficients ( rs ) for both Discontinuous ( Figure 4A ) and Continuous ( Figure 4B ) modes . Visual inspection of the heatmaps presented in 4A and 4B suggests that , while some differences in correlations are apparent , general patterns of correspondence are well preserved regardless of migration mode . This is emphasized by calculation of absolute differences in rs values between modes ( Figure 4C ) , wherein the majority of relationships appear virtually unchanged . Indeed , only a very limited number of inter-feature relationships change by more than 0 . 4 rs ( Figure 4D ) . This suggests that changes in inter-feature correlations are unexpectedly selective , given the magnitude of changes in the actual feature values themselves ( Figure 3 ) . To assess this , we compared the observed distribution of absolute differences in rs values to that which would be expected if changes arose randomly from the rs distributions depicted in Figure 4A and 4B ( Figure 4E ) . This confirmed that observed changes are much smaller and less frequent than randomly expected , thus supporting the proposal that reconfiguration of correlative relationships is highly selective . 10 . 7554/eLife . 11384 . 028Figure 4 . Differences in inter-feature relationships between Discontinuous and Continuous migration modes appear highly selective . ( A , B ) Heatmaps of Spearman's rank correlation coefficients ( rs ) for all pairwise combinations between 60 features ( 55 organizational and 5 behavioral ) , derived from cells during ( A ) Discontinuous or ( B ) Continuous migration . Individual correlation coefficient values are color-coded as depicted in the color bar ( blue = negative; red = positive; green = near zero ) . Numbers on X and Y axes ( 1 to 60 ) correspond to the identities of features , as defined in Feature Number column of Figure 3\\u2014figure supplement 1 . ( C ) A heatmap summarizes absolute differences in correlation coefficient values , per feature pair , between Discontinuous and Continuous migration modes . Difference values are color-coded as depicted in the color bar ( blue = no difference; red = large difference ) . ( D ) A histogram shows the frequency distribution of absolute differences in correlation coefficient values between Continuous and Discontinuous migration modes . ( E ) A plot of cumulative distribution functions ( CDFs ) comparing the observed distribution of absolute differences in correlation coefficient values ( between Discontinuous and Continuous modes as in [C] and [D] , black line ) and the differences in coefficient values obtained following randomized permutation of Spearman's correlation pairs ( gray line ) . The permuted distribution ( repeated 100 times for all relationships , all values included in CDF ) shows the expected distribution of coefficient differences if inter-feature correlation changes occur randomly . Comparison of observed and permuted CDFs suggests that observed differences in inter-feature Spearman's correlation values are far more selective than randomly expected . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 028 We next analyzed how organizational features correlated to cell speed in each migration mode . As exemplified in Figure 5A , and more extensively in Figure 5B , we found that a proportion of such correlations were essentially equivalent between modes ( e . g . Cell Area vs Cell Speed rs = negative in both modes; Cell Compactness \\u2212 Cell Speed rs = positive in both modes ) . However , a large number of cell speed correlations were detectable in only one of the two modes ( e . g . Median [CMAC Distance to Border] per Cell vs Cell Speed rs = positive only in Continuous mode; Median [CMAC Area] per Cell vs Cell Speed rs = positive only in Continuous mode ) . Even more salient , a substantial number of correlations to Cell Speed were inverted between modes ( e . g . Median [CMAC Lifetime] per Cell vs Cell Speed rs = negative in Discontinuous mode , positive in Continuous mode; Median [CMAC Average Trailing Edge Speed] per Cell vs Cell Speed rs = positive in Discontinuous mode , negative in Continuous mode ) . Such inversions in organizational feature-to-Cell Speed relationships hint at profound differences in the mechanisms controlling each mode . We also note that , more broadly , most organizational feature-to-Cell Speed correlations were both stronger and more positive during migration in the Continuous mode ( Figure 5C ) . This suggests that Continuous mode migration is more directly dependent on the state of CMAC and F-actin machineries , while alternative machineries may play more dominant roles during Discontinuous migration . 10 . 7554/eLife . 11384 . 029Figure 5 . Cell speed variation is coupled to distinct patterns of remodeling in underlying features and inter-feature relationships within each migration mode . ( A ) A selection of Spearman's correlation coefficient values ( rs ) for organizational feature relationships to cell speed are depicted ( color-coded as in color bar ) . Values reflect medians of populations values . Correlations are considered non-significant ( ns ) if zero is included in the range of the median +/\\u22121 . 57 * inter-quartile range ( IQR , 25% to 75% ) \\u221an ( approximates 95% confidence interval of the median , n = number of cell observations , see 'Materials and methods' ) . This highlights correlations to cell speed that: are equivalent between modes ( e . g . Cell Area , negative correlation in both modes ) ; exist in only one mode ( e . g . Median [cell-matrix adhesion complex ( CMAC ) Area] per Cell , ns in Discontinuous vs positive in Continuous ) ; or are opposite between modes ( e . g . Median [CMAC Lifetime] per Cell , negative in Discontinuous vs positive in Continuous ) . Examples in A are linked to an extensive analysis of cell speed-to-organizational feature correlations in B and C . ( B ) Parallel coordinate mapping of median cell speed-to-organizational feature correlations in Discontinuous ( blue ) and Continuous ( orange ) modes . Relationships are categorized by correlation value similarity or difference between modes , as indicated by the orange;\\u00a0C' ( denotes Continuous ) and blue 'D' ( denotes Discontinuous ) followed by '\\u2212'; ( negative rs ) or\\u00a0'+' ( positive rs ) or 'ns' ( non-significant rs ) . ( C ) Bar graphs depict the magnitude of differences in median cell speed-to-organizational feature correlation values from the Discontinuous to the Continuous mode . Correlations tend to be stronger and more positive in cells during Continuous migration . ( D ) Canonical vectors analysis ( CVA ) -based multivariate clustering ( using all 55 organizational features , canonical vectors [CV] 1 , 2 , and 3 displayed , percentages indicate proportion of total variance per CV ) of cell observations during: slow Discontinuous ( 0\\u201333 . 33% of Discontinuous migration speed values , blue ) ; medium Discontinuous ( 33 . 34\\u201366 . 66% , cyan ) ; fast Discontinuous ( 66 . 67\\u2013100% , green ) ; slow Continuous ( 0\\u201333 . 33% of Continuous migration speed values , yellow ) ; medium Continuous ( 33 . 34\\u201366 . 66% , orange ) ; or fast Continuous ( 66 . 67\\u2013100% , red ) migration . Two orientations of the same three-dimensional clustering are depicted ( upper and lower ) , revealing the separate clustering of each migration mode . Within each mode , progressive differences in cell speed correspond to similarly progressive variations in the position of observations within the multivariate organizational feature ( or cell state ) space . These speed-dependent trends in clustering define trajectories in the cell state space along which cells evolve as cell speed changes . Remarkably , these trajectories are completely distinct when comparing Discontinuous ( dashed lines ) and Continuous ( solid lines ) migration modes . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 029 We have thus far detailed a variety of specific differences between Discontinuous and Continuous migration modes in terms of behavior ( Figure 1 and Figure 2 ) , underlying organizational features ( Figure 3 ) and individual inter-feature correlations ( Figure 4 and Figure 5A\\u2013C ) . We next explored how these differences might translate into systemic dissimilarities in the coupling of cell behavior ( i . e . cell migration speed ) and cell organization within each mode . To this end , we first grouped cell observations according to both migration mode and tertiles of cell speed ( 0\\u221233% [slow]; 33\\u221266% [medium]; 66\\u2212100% [fast] ) . We then performed supervised CVA clustering of these groups ( Figure 5D ) . This enabled assessment of how cells in each mode cluster in the multivariate cell state space , but also of the trajectories defined through cell state space given variability in cell migration speed ( as described previously , [Lock et al . , 2014] ) within each mode . As also observed in Figure 3A ( based on similar but not equivalent clustering ) , we found that cells in Discontinuous or Continuous modes cluster separately . However , in a striking extension of this result , we also found that variability in migration speed within each mode corresponded to completely independent trajectories in cell state space . This means that different patterns of organizational remodeling are associated with varying cell speed within each mode . Thus , profound differences exist between modes in terms of the underlying strategies that produce and control cell migration speed . We have demonstrated how specific organizational feature values diverge in conjunction with the spontaneous emergence of Discontinuous and Continuous migration modes ( Figure 3 ) . Yet , it remains unclear whether these features are tightly coupled to migration mode identity , or only loosely correlated with these behaviors . We therefore assessed whether directed modulation of these features could drive corresponding and predictable changes in the frequency equilibrium between migration modes . To test for such bottom-up regulation , we applied a selection of targeted molecular perturbations for which we and others have established prior knowledge regarding their effects on the organizational features in question . These perturbations included: ECM ligand ( FN ) concentration modulation ( Gupton and Waterman-Storer , 2006 ) ; talin 1 depletion ( Kiss et al . , 2015 ) ; and ROCK inhibition-mediated reduction of actomyosin contractility ( Lock et al . , 2014; Kim and Wirtz , 2013; Hern\\u00e1ndez-Varas et al . , 2015 ) ( Figure 6A\\u2013C ) . Images exemplifying perturbation effects on cell , CMAC and F-actin morphology are presented in Figure 6\\u2014figure supplement 1 . Notably , these molecular perturbations collectively target four core regulatory mechanisms around the ECM \\u2013 adhesion \\u2013 F-actin axis , including integrin ligation; integrin activation/clustering; integrin-F-actin linkage; and actomyosin contractility ( Figure 6A ) . Given the pivotal importance of these mechanisms for mesenchymal migration , the effects of the specific molecular perturbations applied herein may be somewhat predictive of a wide array of related regulatory mechanisms . 10 . 7554/eLife . 11384 . 030Figure 6 . Cell-matrix adhesion and actomyosin contractility control the equilibrium between Continuous and Discontinuous migration modes . ( A ) Schematic summary of molecular mechanisms ( green ) associated with the ECM \\u2013 adhesion \\u2013 F-actin axis ( gray ) targeted here for perturbation to modulate implicated organizational features and , potentially , corresponding migration mode frequencies . Specifically , altering ECM ligand density ( fibronectin concentration , [FN] ) impacts mechanism 1 ) - integrin ligation and subsequent cell-matrix adhesion complex ( CMAC ) formation and maturation . RNAi-mediated knockdown of talin 1 limits both mechanism 2 ) - integrin activation and mechanism 3 ) - integrin-F-actin linkage , thereby also affecting CMAC formation , maturation and stability . Y-27632-mediated inhibition of ROCK disrupts mechanism 4 ) - actomyosin contractility , affecting F-actin dynamics and CMAC maturation . Images exemplifying the effects of each molecular perturbation are presented in Figure 6\\u2014figure supplement 1 . ( B ) Boxplots summarizing the response of the same selection of organizational features to: ECM ligand modulation ( 2 . 5 \\u00b5g/ml FN vs 10 \\u00b5g/ml FN , upper row ) ; talin 1 RNAi ( control siRNA vs talin 1 siRNA , 5 \\u00b5g/ml FN , middle row ) ; or ROCK inhibition ( DMSO vs 6 \\u00b5M Y-27632 , 5 \\u00b5g/ml FN , lower row ) . All boxplots in B show median values per condition and inter-quartile ranges ( IQR , 25% to 75% ) . Notches show the median +/\\u22121 . 57 * IQR/ / \\u221an ( approximates 95% confidence interval of the median , n = number of cell observations , see 'Materials and methods' ) . In each case , statistically discernable differences were assessed by Wilcoxon rank sum testing , with resulting p values <0 . 001 . ( C ) Migration mode frequency responses to each perturbation are depicted ( Discontinuous , blue; Continuous , orange ) . Additional conditions were included for ROCK inhibition ( low panel ) , showing a progressive response to 1 \\u00b5M , 2 \\u00b5M , and 6 \\u00b5M Y-27632 as compared to DMSO vehicle control . Note: in addition to depicting specific perturbation-dependent trends , the matrix of results presented in B and C is used , together with spontaneous feature variations depicted in Figure 3D , E , G\\u2013I , to logically parse organizational features that are consistently coupled , or just occasionally correlated , with migration mode identity . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 03010 . 7554/eLife . 11384 . 031Figure 6\\u2014figure supplement 1 . Comparison of cell , adhesion complex and F-actin morphologies following perturbations targeting the ECM \\u2013 adhesion \\u2013 F-actin axis . Confocal images of H1299 cells comparing: ( A ) low concentration ( 2 . 5 \\u00b5g/ml ) fibronectin ( FN ) -coated glass ( left ) vs high concentration ( 10 \\u00b5g/ml ) FN-coated glass ( right ) ; ( B ) control siRNA treatment ( left ) vs talin 1 siRNA treatment ( right ) ; and ( C ) control ( DMSO ) treatment ( left ) vs ROCK inhibitor ( Y-27632 , 6 \\u00b5M ) treatment ( right ) . EGFP-paxillin ( upper panels ) and RubyRed-LifeAct ( lower panels ) signals are shown to demarcate cell-matrix adhesion complexes ( CMACs ) and F-actin , respectively . Images are displayed with gray-scale inverted . ( D ) Talin 1 knockdown was confirmed by immunoblotting . Scale bars = 20 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 031 Importantly , we found that Continuous mesenchymal migration occurred upon high adhesion ( high FN concentration ) and with unperturbed actomyosin contractility , while Discontinuous migration occurred under conditions of low adhesion ( low FN concentration ) and inhibited contractility ( ROCK-inhibitor ) . Talin knock-down promoted Discontinuous migration ( Figure 6C ) . We then compared: i ) spontaneous differences in feature values between Continuous and Discontinuous migration modes ( displayed in Figure 3D , E , G and I ) ; ii ) perturbation-induced differences in the same features ( Figure 6B ) ; and iii ) perturbation effects on migration mode frequencies ( Figure 6C ) . From this matrix of results , we were able to determine whether a logical coherence was preserved over all these measures . For example , median CMAC Area was spontaneously higher in cells during Continuous migration ( Figure 3I ) , and increasing ECM ligand concentration ( FN ) causes higher CMAC Area values ( Figure 6B , upper left panel ) . It is therefore logically consistent ( if CMAC Area and migration mode behaviors are functionally coupled ) that raising ECM ligand concentration increases the frequency of the Continuous migration mode ( Figure 6C , upper panel ) . In fact , although the applied perturbations produce differing effects on CMAC Area ( increasing or decreasing ) , migration mode frequencies show logically consistent responses in each case . CMAC Area therefore appears likely to be closely coupled to migration mode identity , since there is no evidence that CMAC Area and migration mode identity could be uncoupled . Such coherence was also observed for CMAC Lifetime and Cell Compactness , thus similarly supporting their functional coupling to migration mode determination . In contrast , while the modulation of either ECM ligand concentration or talin 1 expression caused effects on Cell Area and CMAC Number that were consistent with their parallel effects on mode frequencies , ROCK inhibition caused contradictory results: CMAC number and Cell Area were both higher in cells in the Continuous mode ( Figure 3H ) and were increased following ROCK inhibition ( Figure 6B , lower right panel ) , yet ROCK-inhibition caused a decreased frequency of the Continuous mode ( Figure 6C , lower panel ) . These results imply that , although Cell Area and CMAC Number spontaneously correlate with migration mode identities under the original experimental condition , this correspondence can be uncoupled . These organizational features are therefore unlikely to be causal of , or caused by , migration mode identity determination . Taken together , the matrix of results presented delineates organizational features ( CMAC Area , CMAC Lifetime , Cell Compactness ) that may be directly coupled with migration mode determination . More definitively , these results effectively exclude Cell Area and CMAC Number from being ( linearly ) causally linked to this process . Furthermore , these results demonstrate the potent regulatory influence of the specific molecular components fibronectin , talin 1 and ROCK , on migration mode determination . These perturbations also allows us to conclude more broadly that cell-matrix adhesion complexes and actomyosin contractility play key roles in shaping Discontinuous and Continuous mode frequencies . The molecular perturbations described above changed both cellular and macromolecular organization , as well as the frequency equilibrium between Discontinuous and Continuous migration modes . However , it remained unclear to what extent perturbation effects on organizational features may reflect: a ) mode frequency changes ( inter-modal 'adaptive switching' ) , as opposed to b ) organizational remodeling of the modes themselves ( intra-modal 'adaptive stretching' ) . To address the balance between these two adaptive responses , we leveraged data from the comparison of low ( 2 . 5 \\u00b5g/ml ) and high ( 10 \\u00b5g/ml ) FN concentrations displayed in Figure 6 . Using only organizational features , we performed principal component analysis ( PCA ) -based unsupervised clustering of cell observations grouped by both FN concentration and migration mode ( Figure 7A ) . Surprisingly , this revealed that the distances ( in PCA-defined cell state space ) between Discontinuous and Continuous migration modes were far greater than the distances induced within each mode by FN concentration variation . Comparing the heterogeneity of individual cells over time ( Figure 7B and C , Figure 7\\u2014figure supplement 1 ) and the distances between migration mode centers of mass confirmed the substantial nature of average organizational differences between modes ( equal to or greater than the maximal span of variation observed over 8 hr in the individual cells shown in Figure 7B\\u2013C ) . To more intuitively visualize how changing FN concentration shifts the distribution of cell organization within the PCA-defined cell state space ( and between modes ) , we calculated the probability density distributions of cell observations in each mode , for each experimental condition ( Figure 7D\\u2013E ) . This revealed that cell observations in each mode populate high probability-density 'valleys' that are at least partially separated by low probability-density 'ridges' . This visualization also emphasized that the major response to FN modulation was a substantial rebalancing of cell organizational states from one valley ( migration mode ) to the other . Collectively , these observations demonstrate that switching ( between modes ) dominates over stretching ( remodeling within modes ) as an adaptive response to FN concentration modulation . The conceptual relation between these two mechanisms of adaptation in cell state space is schematized in Figure 7F . 10 . 7554/eLife . 11384 . 032Figure 7 . Adaptive switching between migration modes is the dominant perturbation response rather than intra-modal remodeling . ( A ) Principal component analysis ( PCA ) -based clustering of cell observations ( based on all 55 organizational features ) , principal components ( PC ) 1 and 2 displayed , percentages indicate proportion of total variance per PC , color-coded by migration mode and experimental condition: Discontinuous mode on 2 . 5 \\u00b5g/ml FN ( dark blue ) ; Discontinuous mode on 10 \\u00b5g/ml FN ( light blue ) ; Continuous mode on 2 . 5 \\u00b5g/ml FN ( red ) ; Continuous mode on 10 \\u00b5g/ml FN ( orange ) . Large black outlined circles indicate population centers of mass . Note that , despite FN variation exerting large effects on cell organization , differences between migration modes are much larger that differences between conditions within the same mode . ( B ) PCA clustering as in ( A ) , overlaid by the heterogeneous trajectories over time of 4 cells ( I\\u2013IV ) within the PCA-based multivariate state space . Cells I\\u2013IV belong to the spatially corresponding cell populations defined in ( A ) and are relatively constrained within their respective regions of the cell state space . ( C ) Single time point confocal images of EGFP-paxillin from Cells I\\u2013IV from ( B ) . Images are displayed with gray-scale inverted . Corresponding image sequence selections are presented in Figure 7\\u2014figure supplement 1 . Scale bars = 20 \\u00b5m . ( D ) A probability density map based on the coordinates in PCA space of cell observations from the 2 . 5 \\u00b5g/ml FN condition in ( A ) ( view orientation relative to A is rotated 180 degrees ) . Cells in the Discontinuous mode ( dark blue ) define a deep probability valley that partially overlaps with but is largely distinct from a shallow probability valley defined by cells in the Continuous mode ( red ) . The sum of probabilities equals 1 . ( E ) A probability density map based on the coordinates in PCA space of cell observations from the 10 \\u00b5g/ml FN condition in ( A ) ( view orientation relative to A is rotated 180 degrees ) . Cells in the Continuous mode ( orange ) define a deep probability valley that overlaps with but is partly distinct from a shallow probability valley defined by cells in the Discontinuous mode ( light blue ) . The sum of probabilities equals 1 . Centers of mass for each population in ( D ) and ( E ) are indicated by capped vertical lines of matching colors . Arrows in ( D ) and ( E ) signify how decreasing or increasing FN concentration , respectively , causes 'switching' from one probability valley ( migration mode ) to the other . ( F ) Schematic summary of the dual adaptive strategies employed by cells , with respect to mesenchymal migration . Modulation of intracellular or extracellular conditions can cause the remodeling of cellular and macromolecular organization locally within a given mode ( 'adaptive stretching' ) . However , the much larger and more frequent response to such perturbations is 'adaptive switching' between discrete migration modes , resulting in substantial rebalancing of Discontinuous and Continuous migration mode frequencies . ( G ) Dendogram ( qualitative ) indicating the major recognized archetypes of cell migration ( top row ) and the new modalities of Continuous and Discontinuous mesenchymal migration described herein ( bottom row ) . Italicized names in brackets correspond to similar/analogous migration modes/terms . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 03210 . 7554/eLife . 11384 . 033Figure 7\\u2014figure supplement 1 . Confocal image sequences of cells during Discontinuous and Continuous mode migration on glass coated with low or high fibronectin concentrations . Confocal image sequences of H1299 cells during migration on low concentration ( 2 . 5 \\u00b5g/ml ) fibronectin ( FN ) -coated glass ( A and B ) or\\u00a0high concentration ( 10 \\u00b5g/ml ) FN-coated glass ( C and D ) . EGFP-paxillin ( upper panels ) and RubyRed-LifeAct ( lower panels ) signals are shown to demarcate cell-matrix adhesion complexes ( CMACs ) and F-actin , respectively . Images are displayed with gray-scale inverted . Numbers denote time ( min ) . Scale bars = 20 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 033 \\n\\n## Discussion\\n We here present an integrated analytical approach and associated tools , designed to interrogate both spontaneous and induced heterogeneity in single-cell mesenchymal ( lamellipodial ) migration . By deploying these tools , we quantitatively demonstrate that mesenchymal migration is composed of two distinct sub-modalities , each with specific behavioral , organizational , and regulatory characteristics . This marks an important advance that may enhance future investigations of the mesenchymal migration archetype . In particular , interpretations of natural and/or experimentally induced variability will likely become more coherent and precise if data are disaggregated according to Discontinuous or Continuous modes , rather than being unintentionally aggregated across cells with fundamentally different properties and dependencies . Recognition of these modes may therefore facilitate a more accurate understanding of the heterogeneity , adaptability , and regulation of mesenchymal migration as a whole . Our initial classification of cells into Discontinuous and Continuous migration modes was based on observed behavioral differences . Crucially , the same distinctive behaviors emerged in parallel within a variety of cell lines and under a spectrum of conditions , thus supporting the robustness and broad relevance of these behaviors . Yet , given the similarity of Continuous and Discontinuous migrations to previous descriptions of keratocyte-like ( Barnhart et al . , 2015; Keren et al . , 2008 ) or fibroblast-like motility ( Abercrombie et al . , 1977; Theisen et al . , 2012 ) , respectively , the existence of these behaviors per\\u00a0se\\u00a0is not novel . Indeed , early observations even qualitatively suggest the co-existence of similar behaviors ( Lewis et al . , 1982 ) . However , we now present quantitative evidence of the spontaneous , parallel emergence of two distinct yet inter-convertible mesenchymal migration modes , wherein divergent couplings exist between behavioral ( speed ) and organizational characteristics . We thereby provide new insights into the discontinuous structure of heterogeneity within the broad mesenchymal migration archetype . Indeed , by differentiating between progressive ( intra-modal ) and discrete ( inter-modal ) forms of variation , we gained the capacity to compare the roles that these two mechanisms play in adaptive responses to perturbation . Surprisingly , this revealed that adaptive switching between modes is the dominant response to perturbation ( FN concentration modulation ) , with adaptive stretching ( remodelling ) within modes playing only a minor role . The quantitative distinction between Discontinuous and Continuous migration modes is based on systemic differences detected via multivariate clustering of data following visual classification . Importantly , given that cells were classified based on perceived differences in their behavioral dynamics , we excluded behavioral features and focused only on organizational features ( predominantly related to CMAC and F-actin status ) when addressing the question of separation between the modes . As a result , at least four lines of evidence support the discrete nature of Discontinuous and Continuous migration modes . First , supervised multivariate clustering ( canonical vectors analysis , CVA ) of mode-classified cell observations indicated near complete separation between Discontinuous and Continuous modes . The systemic organization underlying these migration modes is therefore , to a very large extent , quantitatively distinct . Second , although separation between modes was not complete when cell observations were clustered in unsupervised PCA space , the topology of probability-density distributions was consistent with the existence of subpopulations in cell state space . Precisely , the high probability-density 'valleys' near the center of mass of each mode contrast with the low probability-density 'ridges' at the boundaries between modes . Importantly , these probability-density distributions are not predetermined outcomes of such PCA clustering , since PCA is unsupervised with respect to mode identity . In fact , these topologies are suggestive of a system comprised of two attractor states ( Huang and Ingber , 2000 ) , whose basins of attraction diverge on either side of a set of potentially unstable or unfavorable configurations . The low probability of these boundary configurations effectively divides mesenchymal migration into two independent sub-modalities . Such attractor states have been described in relation to , for example: gene expression during development and tumorigenesis ( Huang et al . , 2009; Huang , 2009 ) ; cell signaling during motility ( Kim et al . , 2015 ) ; and even in the force-coupled dynamics of CMACs ( Hern\\u00e1ndez-Varas et al . , 2015 ) . Moreover , the attractor state concept appears analogous to the concept of prespecification originally proposed by Friedl to explain the robust , recurrent and yet switchable characteristics of mesenchymal and amoeboid migration modes ( Friedl , 2004 ) . Interestingly , the limited remodelling within Discontinuous and Continuous modes in response to FN modulation ( as opposed to mode switching ) suggests that these modes may be relatively inflexible , providing additional support for their proposed status as prespecified attractor states . Third , further indicating the discrete nature of Discontinuous and Continuous migration modes , CVA clustered cell observations grouped by both migration mode and cell speed were again largely distinct , supporting the validity of the original CVA clustering result . In addition , this approach also provided a fourth compelling line of evidence by revealing completely independent trajectories through cell state space defined by cells migrating at different speeds within each mode . Remarkably , this indicates that changes in cell speed within each mode were coupled to substantially different patterns of organizational remodelling . These systemic differences were composed in part by specific disparities in Spearman\\u2019s correlations between individual organizational features and cell speed within each mode . While a variety of differences were observed , most striking were instances where correlations to cell speed were inverted between modes , such as in the case of adhesion stability ( CMAC lifetime ) \\u2013 a critical parameter previously linked to causal regulation of cell speed ( Lock et al . , 2014 ) . Interestingly , when comparing modes , the independent patterns of cell speed-coupled reorganization arose despite only subtle differences across the entire network of inter-feature Spearman\\u2019s correlations . Indeed , these differences in correlation values were far more limited than would be expected at random , suggesting that a select few features and relationships may control migration mode identity . Taking into account each of these lines of evidence , we conclude that Discontinuous and Continuous migration modes are representative of more than simply cells with different organizational feature values . We propose that these modes in fact represent divergent , yet co-existing strategies of mesenchymal migration . Despite strongly embracing quantitative imaging-based research ( Lock and Str\\u00f6mblad , 2010; Le D\\u00e9v\\u00e9dec et al . , 2010; Masuzzo and Martens , 2015 ) , the cell migration field has seen remarkably little application of automated approaches to the classification of migrating cells . This is in contrast to the study of mitosis , where automated classification has already proven to be a powerful research tool ( Neumann et al . , 2010; Schmitz et al . , 2010 ) . It is likely that this reflects differences in the degree and structure of heterogeneity in these biological processes . For example , mitosis is a relatively stereotyped process - both morphologically and particularly in the terms of the temporal ordering of events . Both characteristics can increase the accuracy of automated classification approaches ( Held et al . , 2010 ) . While cell migration is also more ( e . g . Discontinuous mode ) or less ( e . g . Continuous mode ) a globally time-ordered process , the underlying events appear far less stereotypical . Given these challenges , we have employed a blinded manual classification approach in this study . However , we note that the CVA clustering results \\u2013 even excluding behavioral features ( as herein ) \\u2013 provide evidence that automated classification of migration modes may be feasible in future . This may significantly enhance subsequent quantitative analyses of mode organization , behavior , and regulation . Such automated classification may be in part based on our detailed spatiotemporal analysis of protrusive and retractive membrane dynamics within each mode . This revealed that the major differences between modes were: the size of membrane retraction events; the order and temporal coupling of protrusion and retraction events; and the coordination displayed between protrusive and retractive dynamics . Specifically , cells in the Discontinuous mode produced many very small and occasional very large retraction events , with these retractions tending to lead protrusion , reversing the order detected in Continuous cells . A large time delay also exists between retraction and subsequent protrusion dynamics in Discontinuous cells , and these dynamics are less coordinated ( lower cross-correlation ) . Such substantial differences in membrane dynamics ( especially the inversion of protrusion and retraction orders ) further support the distinct nature of Discontinuous and Continuous migration modes . Furthermore , these analyses both confirmed and explained the visual impressions that originally differentiated Discontinuous from Continuous migration . In addition , given the strong correspondence between retraction ( but not protrusion ) size and cell speed , these analyses now add migration mode and cell speed to motility initiation as aspects of migration where membrane retraction appears to be the deterministic membrane process ( Barnhart et al . , 2015; Kowalewski et al . , 2015; Cramer , 2010 ) . Having quantitatively established the discrete nature and behavioral characteristics of Discontinuous and Continuous migration modes , we sought to understand how their frequency balance is regulated . Importantly , to select candidate molecular regulators , we first considered the differences in organizational feature values that arose spontaneously between the modes . Then , given existing knowledge on how targeting of particular molecular components ( FN concentration [Gupton and Waterman-Storer , 2006]; talin 1 expression [Kiss et al . , 2015]; ROCK signaling [Lock et al . , 2014; Kim and Wirtz , 2013; Hern\\u00e1ndez-Varas et al . , 2015] ) impact these same features , we designed a series of perturbations intended to drive specific changes in these features , while also monitoring the behavioral balance between modes . This demonstrated that each of these molecular factors plays a key role in shaping the equilibrium between modes . Furthermore , these molecular targets also regulate generic characteristics of the pivotal ECM \\u2013 adhesion \\u2013 F-actin axis , including: integrin ligation ( FN concentration ) ; integrin activation and clustering ( talin expression ) ; integrin \\u2013 F-actin linkage ( talin expression ) ; and actomyosin contractility ( ROCK signaling ) . Hence , the effects of these specific perturbations may also be illustrative of how generalized regulatory mechanisms impact the migration mode equilibrium . Taking this perspective , we observe that while Continuous migration arises under conditions of high adhesion and full contractility , Discontinuous migration is preferred given low adhesion and inhibited contractility . It is noteworthy that both modes emerge under conditions that are distinct from those that favour amoeboid migration ( i . e . low adhesion and high contractility ) . Therefore , the switch between Continuous and Discontinuous ( mesenchymal ) migration modes clearly contrasts to the previously identified conversion between mesenchymal and amoeboid migration . This emphasizes the multifaceted adaptive capacity of migrating cells , which likely expands the contexts within which efficient migration is possible . This plasticity is highlighted by the growing variety of distinct migration modalities now recognized ( Figure 7G ) . Finally , by viewing differences in organizational feature values ( both spontaneous and perturbation-induced ) and mode frequencies as a logical matrix , we could identify organizational features that are coupled to migration mode identity under all observed conditions , as well as features that can be explicitly uncoupled . This provides a first step in parsing correlated features from those with significant causal influence over migration mode identity . Specifically , it is notable that CMAC Area and CMAC Lifetime were previously indicated to causally influence cell migration speed ( Lock et al . , 2014; Kim and Wirtz , 2013 ) , while Cell Compactness was causally downstream of cell speed ( Lock et al . , 2014 ) . Thus , the evidence that these features may be functionally coupled to the determination of migration mode identity is consistent with them being integral to migratory regulation . Conversely , we previously found that Cell Area showed no causal relationship to cell speed ( Lock et al . , 2014 ) , and this is again consistent with it being functionally uncoupled from the process of mode identity determination . Overall , three types of insight were revealed through these perturbation-based analyses , including: the regulatory roles of specific molecules ( fibronectin , talin and ROCK ) ; the putative influence of generic regulatory mechanisms; and the nature of links ( coupled vs uncoupled ) between migration mode identity and commonly measured macromolecular features . Nonetheless , much remains to be learned about the proximal and distal determinants of these migration modes . In conclusion , we have presented and applied a quantitative , imaging-based analytic approach to explore the heterogeneity that naturally emerges within the mesenchymal ( lamellipodial ) archetype of cell migration . We thereby characterized two quantitatively distinct migration modes , here termed Continuous and Discontinuous migrations , that co-exist within the broad mesenchymal migration archetype . We compared these migration modes in terms of: their motion and behavioral dynamics; the specific and systemic organization of key machineries ( CMACs and F-actin ) underlying their motility , and; differences in how behavior ( migration speed ) and organization co-evolve within each mode . Through targeted molecular perturbations , we defined both specific molecular and general mechanisms of bottom-up control over migration mode determination , while also beginning to parse potentially functional ( coupled ) relationships ( between features and modes ) from those that are correlated but non-functional ( uncoupled ) . Finally , our systemic analysis of adaptive responses to perturbation has revealed how discrete mechanisms ( inter-modal adaptive switching ) dominate over progressive remodeling ( intra-modal adaptive stretching ) . This study therefore emphasize the importance of distinguishing and comprehending Discontinuous and Continuous migration modes as a necessary precursor to understanding mesenchymal migration in its totality , while simultaneously providing tools and approaches that enable this endeavor . \\n\\n## Materials and methods\\n We have integrated a comprehensive and unique combination of analytical capabilities and made them freely available as a Matlab toolbox: 'The Cell Adhesion and Migration Analysis Toolbox' \\u2013 along with the raw quantitative data underlying this study , sample image data to aid implementation , and explanatory documentation ( see doi:10 . 5061/dryad . 9jh6m ) . Briefly , this toolbox includes the following features utilized in this study: Data import and preprocessing based on images and extracted variables ( see Quantitative Image Analysis , below ) ; Smoothing of cell and adhesion trajectories; Membrane protrusion and retraction dynamics extraction; Migration mode classification; Mean square displacement ( MSD ) of cell trajectories; Variable selection for uni- and multivariate analyses; Univariate statitstical analysis; Variable correlation based heat maps; Canonical vector analysis ( CVA ) ; Principal component analysis ( PCA ) ; and two dimensional kernel density estimation . A more detailed description is available within the Matlab Toolbox . All cell lines were acquired directly from the American Type Culture Collection ( ATCC , Manassas , VA ) , and therefore were not further authenticated . The cell lines used herein are not members of the ATCC list of commonly misidentified cell lines . H1299 ( human non-small cell lung carcinoma , ATCC; ATCC# CRL-5803 , mycoplasma negative ) cells stably expressing EGFP-Paxillin and RubyRed-LifeAct , termed H1299 P/L cells , were established and maintained in RPMI 1640 medium ( Gibco \\u2013 Thermo Fisher Scientific , Waltham , MA ) containing 400 \\u00b5g/ml geneticin ( G-418 sulfate , Gibco ) with 10% fetal bovine serum ( Gibco ) and 1 mM glutamine , as described previously ( Lock et al . , 2014; Kiss et al . , 2015 ) . BT549 ( ductal breast carcinoma , ATCC; ATCC# HTB-122 , mycoplasma negative ) cells were maintained in RPMI 1640 medium ( Gibco ) containing 10% fetal bovine serum ( Gibco ) and 1 mM glutamine . MDA-MB-231 ( adenocarcinoma , ATCC; ATCC# HTB-26 , mycoplasma negative ) , Hep-3 ( hepatocellular carcinoma , ATCC; ATCC# HB-8064 , mycoplasma negative ) and Hs578T ( breast carcinoma , ATCC; ATCC# HTB-126 , mycoplasma negative ) cells were maintained in DMEM ( Gibco ) containing 10% fetal bovine serum ( Gibco ) . All cells were incubated at 37\\u00b0C in 5% CO2 . In preparation for imaging , 96-well optical glass-bottomed plates ( Zell-Kontakt , N\\u00f6rten-Hardenberg , Germany ) were coated with ECM ligands , including either collagen type 1 ( Life Technologies \\u2013\\u00a0Thermo Fisher Scientific ) , laminin ( Sigma-Aldrich , St . Louis , MO ) , fibronectin or vitronectin ( both purified from human plasma as described previously [Smilenov et al . , 1992; Yatohgo et al . , 1988] ) . Coating was performed for 2 hr at 37\\u00b0C followed by blocking with 1% heat-denatured bovine serum albumin ( Sigma-Aldrich ) for 1 hr at 37\\u00b0C . ECM ligand coating concentrations were 10 \\u00b5g/ml except where otherwise indicated ( i . e . where fibronectin concentration was varied ) . RNAi-depletion of talin 1 was performed 48 hr prior to imaging or immunoblotting ( with anti-talin [8d4 , 1:500 , Sigma-Aldrich] and anti-tubulin [DM1A , 1:2000 , Fisher Scientific] antibodies as described previously [Kiss et al . , 2015] ) using the following oligonucleotide sequence: ( 5\\u2019 -GAA GAU GGU UGG CGG CAUU- 3\\u2019 ) ( synthesized by GenePharma , Shanghai , P . R . China ) . A non-targeting oligonucleotide sequence was used as control: ( 5\\u2019 -GCG CGC UUU GUA GGA UUCG- 3\\u2019 ) . Transfection of 2 x 104 cells was performed in 24-well plates using 20 pmol of siRNA and 2 \\u00b5l RNAiMAX ( Invitrogen \\u2013 Thermo Fisher Scientific ) according to the manufacturers instructions . Inhibition of Rho kinase ( ROCK ) was performed starting 1 hr prior to imaging using the Y-27632 inhibitor ( Sigma-Aldrich ) at 1 \\u00b5M , 2 \\u00b5M , or 6 \\u00b5M . Di-methyl sulfoxide ( DMSO , Sigma-Aldrich ) was used as control . Live-cell imaging was initiated 2 hr after replating of 2 x 104 cells per well into ECM-coated 96-well glass-bottomed plates ( see coating details above ) . High-resolution , multiscale imaging ( of CMACs , F-actin and cell migration ) was performed on a Nikon A1R confocal microscope using a PlanApo VC 60X/1 . 4 NA oil-immersion objective ( Nikon , Amsterdam , Netherlands ) . Images were acquired for 8\\u201310 hr at 5 min intervals with a pixel resolution of 0 . 21 \\u00b5m . Long-term ( 16 hr at 5-min intervals ) , low resolution imaging to monitor for inter-modal transitions was performed on a Nikon A1R confocal microscope using a Plan Apo \\u03b3 20X/0 . 75 NA air objective and resonance scanner , producing a pixel resolution of 0 . 82 \\u00b5m . 3\\u00d73 or 4\\u00d74 image montages were acquired and stitched . Prior to imaging , cells were labeled at 1:5000 with a far-red membrane dye ( Cell Mask Deep Red , Thermo Fisher Scientific ) for 2 hr and throughout imaging . Normal culture medium minus fetal bovine serum was used during all imaging , and cells were maintained at 37\\u00b0C in 5% CO2 . Multiscale image sequences ( Nikon ND2 files ) were analyzed using PAD software ( Digital Cell Imaging Laboratories , Keerbergen , Belgium ) , as described previously ( Lock et al . , 2014 ) . Briefly , cell boundaries were detected using the RubyRed-LifeAct signal . Cells touching the image border in any given time frame were excluded . Within each segmented cell , CMACs greater than 0 . 05 \\u00b5m2 in area were segmented based on EGFP-Paxillin signal . Segmented cells and CMACs were then tracked based on nearest neighbor analysis . Cell trajectories were smoothed using smoothing splines . CMACs were excluded where tracking did not contain at least three consecutive time points . CMACs present in the first and last frames of each image sequence were excluded from the calculation of CMAC Lifetime . All segmentation and tracking was manually inspected and iteratively optimized through parameter tuning . Quantitative features characterizing cell , CMAC , and CMAC-associated F-actin properties were then extracted ( Figure 3\\u2014figure supplement 1 ) . CMAC intensity values per channel were corrected by subtraction of mean intensity values in a 1 \\u00b5m radius around the CMAC , excluding other segmented CMACs . CMAC intensity data were further standardized per experimental repeat relative to the median value of CMAC intensities in the size range between 0 . 15 and 0 . 2 \\u00b5m2 within the low FN no INH condition . Cell motion was characterized by instantaneous cell speed or via analysis of mean squared displacement ( MSD ) . Mean square displacement ( MSD ) was calculated using a moving window as previously described ( Khorshidi et al . , 2011 ) . The window size was set to 24 frames ( 2 hr ) . The first 12 time lags ( tlag ) , up to 1 hr , of the MSD were fitted to Furth\\u2019s formula: MSD=4M ( tlag\\u2212tp ( 1\\u2212e\\u2212tlag/tp ) ) , thus measuring the migration coefficient M and the persistence time tp at each time point . The observations were divided into quintiles of migration coefficient and within each such quintile the mean persistence time for each trajectory was calculated . For visualization of parallel mode emergence ( Figure 1\\u2014figure supplement 1 and Video 3 ) and mode transitions ( Figure 1\\u2014figure supplement 2 and Videos 4 , 5 ) , long-term low-resolution image data was segmented and tracked ( following advanced denoising , power 15 ) using a custom general analysis pipeline in NIS-Elements ( v4 . 30 , Nikon ) . Collectively , data sets for comparison of fibronectin concentration , talin 1 expression and ROCK inhibition had structures as follows: 2 . 5 \\u00b5g/ml FN ( 3 experimental repeats , 34 cells , 2525 cell observations , 29651 CMAC observations ) vs 10 \\u00b5g/ml FN ( 19 experimental repeats , 118 cells , 6528 cell observations , 213406 CMAC observations ) ; talin 1 siRNA ( 4 experimental repeats , 101 cells , 6263 cell observations , 81675 CMAC observations ) vs control siRNA ( 4 experimental repeats , 51 cells , 3154 cell observations , 131679 CMAC observations ) ; High-resolution , multiscale imaging of 6\\u00a0\\u00b5M Y27632 ( ROCK inhibitor , 4 experimental repeats , 17 cells , 985 cell observations , 44647 CMAC observations ) vs DMSO control ( 10 experimental repeats , 34 cells , 2666 cell observations , 86794 CMAC observations ) ; long-term , low resolution imaging of 6\\u00b5M Y27632 ( 4 experimental repeats , 73 cells , 4478 cell observations ) vs 2 \\u00b5M Y27632 ( 4 experimental repeats , 71 cells , 5198 cell observations ) vs 1 \\u00b5M Y27632 ( 4 experimental repeats , 69 cells , 4885 cell observations ) vs DMSO control ( 4 experimental repeats , 83 cells , 5968 cell observations ) . Note that the term 'experimental repeats' refers to independent biological repeats , not technical replicates . Data sample sizes were selected based on prior experience from related multiscale analyses in several previous studies ( Lock et al . , 2014 ) . Samples were collected and analyzed without iteration , that\\u00a0is , all relevant available data were used and sample sizes were not altered to increase or decrease statistical power . Cells were classified into Discontinuous or Continuous migration modes according to the behavioral criteria described in Figure 1 based on visual inspection and optimization between two individuals , using a custom platform established in Matlab ( vR2013b , The Mathworks , Natick , MA ) . Single cells were assessed at each time point within each image sequence ( inter-mode transitions , though infrequent , were assessed ) . All data presented herein were assessed concurrently and were both computationally blinded and randomized with respect to experimental conditions . Dead cells and cells in sustained contact were classified as null and excluded from subsequent analyses . Membrane protrusion and retraction dynamics were quantified as described previously ( Kowalewski et al . , 2015 ) . Briefly , consecutive segmented images were compared to define protrusions ( new pixels in cell segment compared to previous frame ) and retractions ( pixels lost from cell segment compared to previous frame ) . To explore dynamics over different time frames , images separated by different intervals were assessed using the same criteria , establishing time window samplings of between 5 min ( 1 frame ) and 75 min ( 15 frames ) . Subtraction of protrusion and retraction size probability distributions was performed as described previously for other feature probability distributions ( Kiss et al . , 2015 ) . Analysis of cross-correlation between protrusion and retraction areas over time was performed on a per cell sequence basis using the Matlab function xcov . Missing values were replaced by the mean value of protrusion/retraction area for that cell trajectory . Time lags of between \\u221260 min ( \\u221212 frames ) and 60 min ( +12 frames ) were assessed , with negative lags indicating that protrusion dynamics lead retraction dynamics , and positive lags indicating that retraction dynamics lead protrusion dynamics . To summarize the mean absolute cross-correlation per migration mode , cross-correlation values at all lags ( \\u221260 to 60 min; \\u221212 to 12 frames ) were averaged for each time window ( 5 min to 75 min; 1 to 15 frames ) , per cell , and then averaged per condition . Nienty-five\\u00a0percent confidence intervals were calculated per time window size . Friedman testing based on distinct cell numbers defined statistically discernable differences in mean protrusion-retraction cross-correlation between modes across the sampled time windows . All membrane dynamics analyses were performed in Matlab ( vR2013b , The Mathworks ) . Supervised multivariate clustering via canonical vectors analysis ( CVA ) was performed based on decompositions of the between-group and within-group covariance matrices following eigenvalue decomposition , as previously described ( Lock et al . , 2014 ) . Feature selection via CVA was performed by ranking absolute feature weighting coefficients from the first canonical vector ( CV1 ) . Unsupervised multivariate clustering using principal component analysis ( PCA ) was performed using singular value decomposition of the normalized data matrix . Probability density surfaces were calculated using a Gaussian kernel density estimation of significantly smaller scale that the overall mode distributions , and hence does not dictate the 'valley' shape of these distributions \\u2013 rather this is a reflection of the true value distribution . Spearman\\u2019s rank correlation coefficients were calculated based on feature values per cell observation . Estimation of the expected distribution of random absolute value changes in Spearman\\u2019s correlation between modes was performed by permuting the identity of correlation values in one mode while holding the other constant . This was repeated 100 times and the cumulative distribution function of all differences is displayed . Pairwise testing for differences in feature distributions ( boxplots , parallel coordinates ) was performed using the Wilcoxon rank sum test ( equivalent to Mann-Whitney test ) given all cell observations . Analyses and visualizations described above were performed in Matlab ( The Mathworks ) . Parallel coordinates-based visualization of selected feature values in each migration mode was generated using Knime ( v2 . 12 . 00 , KNIME . com , Zurich , Switzerland ) . Parallel coordinates-based comparison of Spearman\\u2019s correlation coefficients between Cell Speed and selected features , as well as visualization of coefficient differences between modes , were generated using R ( v3 . 2 . 0 , R Foundation for Statistical Computing ) and RStudio ( v0 . 98 . 945 , RStudio , Boston , MA ) .\\n\\n\",\n          \"## Keywords: chromosomes and gene expression, structural biology and molecular biophysics\\n## Abstract\\nThe Dna2 nuclease-helicase maintains genomic integrity by processing DNA double-strand breaks , Okazaki fragments and stalled replication forks . Dna2 requires ssDNA ends , and is dependent on the ssDNA-binding protein Rpa , which controls cleavage polarity . Here we present the 2 . 3 \\u00c5 structure of intact mouse Dna2 bound to a 15-nucleotide ssDNA . The nuclease active site is embedded in a long , narrow tunnel through which the DNA has to thread . The helicase domain is required for DNA binding but not threading . We also present the structure of a flexibly-tethered Dna2-Rpa interaction that recruits Dna2 to Rpa-coated DNA . We establish that a second Dna2-Rpa interaction is mutually exclusive with Rpa-DNA interactions and mediates the displacement of Rpa from ssDNA . This interaction occurs at the nuclease tunnel entrance and the 5\\u2019 end of the Rpa-DNA complex . Hence , it only displaces Rpa from the 5\\u2019 but not 3\\u2019 end , explaining how Rpa regulates cleavage polarity . \\n\\n## Introduction\\n Dna2 has nuclease and helicase activities and plays key roles in maintaining genomic integrity . It is involved in the nucleolytic processing of 5\\u2019 flaps during Okazaki fragment maturation , of DNA double-strand breaks ( DSBs ) during homologous-recombination mediated repair , and of stalled replication forks in the intra-S-phase checkpoint ( Bae et al . , 2001; Cejka et al . , 2010; Hu et al . , 2012; Nimonkar et al . , 2011; Zhu et al . , 2008 ) . Dna2 was first identified as a replication mutant required for viability in yeast , and was subsequently shown to be involved in trimming long 5\\u2019 RNA-DNA flaps from Okazaki fragments during replication ( Budd et al . , 1995 ) . Most flaps are cleaved by Fen1 concomitant with their generation by strand displacement during Pol \\u03b4 synthesis on the lagging strand . Flaps that escape early cleavage , or those extended by the Pif1 helicase , get long enough to be coated by the Replication Protein A ( Rpa ) , which renders them resistant to Fen1 ( Bae et al . , 2001; Pike et al . , 2009; Stith et al . , 2008 ) . Dna2 , which can displace Rpa from ssDNA ( Stewart et al . , 2008 ) , trims the flap to a length too short for stable Rpa binding , and restores Fen1 processing ( Ayyagari et al . , 2003; Bae et al . , 2001; Gloor et al . , 2012 ) . The lethality of Dna2 deletion in yeast is attributed to persistent Rpa-coated flaps , which recruit Ddc2 ( ATRIP in metazoa ) and activate the Mec1 ( ATR in metazoa ) DNA-damage checkpoint ( Chen et al . , 2013; Zhu et al . , 2008 ) . Long-flap processing by Dna2 is dependent on Rpa removing ssDNA secondary structure ( Stewart et al . , 2008 ) , an essential Rpa function in many other aspects of DNA metabolism ( Chen et al . , 2013; Fanning et al . , 2006; Symington and Gautier , 2011 ) . Also paralleling other Rpa-dependent processes , the ability of Dna2 to act on Rpa-coated ssDNA is dependent on direct Dna2-Rpa interactions ( Bae et al . , 2003 , 2001 ) . The yeast Dna2\\u25b3405N mutation that reduces Rpa binding also reduces 5\\u2019 flap cleavage and DSB resection in vitro , and renders yeast temperature-sensitive for growth ( Bae et al . , 2003 , 2001; Niu et al . , 2010 ) . In DSB resection , Dna2 acts redundantly with Exo1 ( Gravel et al . , 2008; Mimitou and Symington , 2008; Zhu et al . , 2008 ) . Resection of the 5\\u2019 terminated DNA strand results in a long track of 3\\u2019 overhang ssDNA , which forms a nucleoprotein filament with the Rad51 strand-exchange protein and initiates homologous recombination ( Symington and Gautier , 2011 ) . In vitro , Dna2 , Rpa and the helicase Sgs1 ( BLM in mammals ) constitute the minimal complex that can carry out long-range resection . Resection is dependent on the nuclease activity of Dna2 and the helicase activity of Sgs1/BLM . Rpa is essential for supporting the helicase activity of Sgs1/BLM in part by sequestering the unwound strands , and also for regulating Dna2 by blocking its 3\\u2019 to 5\\u2019 exonuclease activity ( Cejka et al . , 2010; Nimonkar et al . , 2011; Niu et al . , 2010 ) . In cells , Rpa depletion eliminates long-range DSB resection , and also causes the loss or inappropriate annealing of short 3\\u2019 ends generated by Mre11 ( Chen et al . , 2013 ) . In addition to these functions , Dna2 is implicated in preventing the regression of stalled replication forks , which otherwise can generate aberrant structures resembling recombination intermediates and lead to genomic instability ( Hu et al . , 2012 ) . This is dependent on the Dna2 nuclease activity , consistent with the ability of Dna2 to cleave fork structures with regressed leading or lagging nascent strands in vitro ( Hu et al . , 2012 ) . Dna2 contains a PD- ( D/E ) XK superfamily nuclease motif ( Budd et al . , 2000 ) and a 5\\u2019 to 3\\u2019 helicase domain ( Bae and Seo , 2000 ) . It is a ssDNA endonuclease that requires a free end for cleavage , and does not cleave dsDNA , single-stranded gaps , D-loops or RNA ( Bae and Seo , 2000; Kao et al . , 2004 ) . In vitro , isolated Dna2 cleaves ssDNA starting at either end , with multiple rounds of incision degrading ssDNA in both the 5\\u2019 to 3\\u2019 and 3\\u2019 to 5\\u2019 directions ( Bae and Seo , 2000; Masuda-Sasa et al . , 2006 ) . With 5\\u2019 flap DNA , cleavage starts ~\\u223c10 nucleotides ( nts ) from the ssDNA end and continues to within \\u223c5 nts of the duplex ( Bae et al . , 2001; Bae and Seo , 2000; Cejka et al . , 2010; Gloor et al . , 2012; Masuda-Sasa et al . , 2006 ) . It has been suggested that Dna2 loads at the free 5\\u2019 end of the flap and tracks in the 5\\u2019 to 3\\u2019 direction ( Kao et al . , 2004 ) . The helicase and ATPase activities of Dna2 are substantially weaker than those of other helicases ( Budd et al . , 2000; Masuda-Sasa et al . , 2006 ) . The helicase activity can be increased by a high ATP to Mg2+ ratio , but this also inhibits the nuclease activity through ATP sequestering Mg2+ ( Bae and Seo , 2000; Masuda-Sasa et al . , 2006 ) . The 5\\u2019 to 3\\u2019 polarity of the helicase translocation could , in principle , drive the tracking of Dna2 along the 5\\u2019 flap . However , ATPase mutations have a minimal effect on 5\\u2019 flap processing and DSB resection in vitro ( Cejka et al . , 2010; Niu et al . , 2010; Zhu et al . , 2008 ) . And , in vivo , ATPase-inactive yeast Dna2 mutants are viable , although they exhibit impaired growth ( Budd et al . , 2000 ) . To understand the mechanism of action and regulation of this multi-faceted enzyme , we first determined the structure of the intact Dna2-ssDNA complex . The structure revealed that the ssDNA has to thread through a tunnel to bind to Dna2 , with a polarity that precludes the cooperation of the helicase and nuclease activities . The requirement for DNA threading prompted us to investigate how Dna2 gains access to Rpa-coated DNA . We provide the structure of a complex between a Dna2 \\u03b1 helix and the Rpa70 OBN domain , both of which are flexibly-tethered and likely serve to recruit Dna2 to Rpa . We also establish a second Dna2-Rpa interaction that helps to displace Rpa from the 5\\u2019 DNA end , explaining how Rpa restricts the cleavage polarity of Dna2 . \\n\\n## Results and discussion\\n We determined the 2 . 3\\u2009\\u00c5 crystal structure of full-length mouse Dna2 ( residues 1 to 1062 ) bound to a ssDNA substrate of 21 nucleotides ( nts ) , 15 of which are well ordered ( Figure 1A and Table 1 ) . The structure also contains an Fe4-S4 iron-sulfur cluster , ADP and two active-site Ca2+ ions , which do not support nuclease activity but can mimic magnesium coordination ( Yang et al . , 2006 ) . 10 . 7554/eLife . 09832 . 003Figure 1 . Structure of the Dna2-ssDNA complex . ( A ) Cartoon representation of the Dna2-ssDNA complex . The \\u03b11 helix , which packs with the hel2A domain of a crystallographic symmetry related protomer , is omitted from this view . The individual domains of Dna2 are colored separately as indicated in ( D ) , ssDNA is red , ADP is shown as sticks , calcium ions are shown as green spheres , the iron-sulfur cluster is shown in a space-filling representation ( nuc: nuclease domain; hel1A: helicase 1A domain , hel2A: helicase 2A domain , OB: oligonucleotide/oligosaccharide binding domain ) . ( B ) View looking up the vertical axis of ( A ) . ( C ) Schematic of the complex illustrating the relative arrangement of the Dna2 domains , and highlighting its cylinder-like shape . Colored as in ( A ) . ( D ) Linear representation of the Dna2 domains and their boundaries; colored as in ( A ) . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 00310 . 7554/eLife . 09832 . 004Figure 1\\u2014figure supplement 1 . Dna2 secondary structure\\u00a0and sequence conservation . Helices are indicated as cylinders , b strands as arrows , segments lacking regular secondary structure as solid lines , and disordered regions as dashed lines . Secondary structure elements are colored as in Figure 1A . The N-terminal extensions , relative to mammalian Dna2 , of the other orthologs are not shown . In the alignment of yeast Dna2 , reliable homology starts with the \\u03b22 strand of the OB domain at residues 44 and 417 of the mouse and yeast orthologs , respectively . The ~360 residue , unique N-terminal segment of yeast Dna2 has an overall low hydrophobicity content indicative of lack of globular structure , and short regions of homology with closely related yeasts suggests that this region acts through linear epitopes , some of which may be acidic/amphipathic helices ( not shown ) . Residues that contact the DNA are marked by \\u201cd\\u201d , active site residues by \\u201cc\\u201d and iron-sulfur cluster ligands by \\u201cf\\u201d . The entire protein structure is within 0 . 9 \\u00c5 C\\u03b1 r . m . s . d . in DNA-bound and apo Dna2 crystals , and the overall DNA conformation is essentially identical in a different crystal form of Dna2 bound to a 21-nt ssDNA ( Table 1 ) . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 00410 . 7554/eLife . 09832 . 005Figure 1\\u2014figure supplement 2 . Dna2 inter-domain interfaces . ( A ) The nuclease domain packs with the OB domain extensively , through both polar and hydrophobic residues . The four residues between the last b strand of the OB domain and the first b strand of the nuclease domain are buried in the interface between the two domains . The OB groove , where DNA or peptides bind in other OB fold proteins , is solvent exposed opposite from its surface that packs with the nuclease domain . Individual domains are colored as in Figure 1A . ( B ) The Dna2 nuclease core fold has a three-helix insertion ( residues 176 to 227 ) that packs with the helicase 1B domain . This three-helix insertion is also present in the AddB nuclease , where it packs with the N-terminal portion of the protein . ( C ) The nuclease domain also links to the helicase domain indirectly , through the stalk domain that is sandwiched between the two . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 00510 . 7554/eLife . 09832 . 006Table 1 . Data collection and refinement statistics . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 006Data SetDna2-5\\u2019 overhang DNA*Dna2-ssDNAapo Dna2apo Dna2 ( SeMet ) DNA2 \\u03b11-RPA 70NSpace groupP22121P212121P212121P212121C121a , b , c ( \\u00c5 ) 87 . 2 , 118 . 5 , 149 . 3120 . 2 , 149 . 2 , 172 . 9120 . 9 , 148 . 6 , 170 . 5120 . 9 , 148 . 6 , 170 . 5134 . 3 , 50 . 9 , 76 . 5\\u03b1 , \\u03b2 , \\u03b3 ( \\u00b0 ) 90 . 0 , 90 . 0 , 90 . 090 . 0 , 90 . 0 , 90 . 090 . 0 , 90 . 0 , 90 . 090 . 0 , 90 . 0 , 90 . 090 . 0 , 103 . 9 , 90 . 0Resolution ( \\u00c5 ) 50 . 0 - 2 . 35 ( 2 . 43 - 2 . 35 ) 30 . 0 \\u2013 3 . 11 ( 3 . 23 - 3 . 11 ) 50 . 0 \\u2013 3 . 0 ( 3 . 11 - 3 . 0 ) 30 . 0 \\u2013 3 . 4 ( 3 . 52 - 3 . 4 ) 60 . 0 \\u2013 1 . 5 ( 1 . 55 - 1 . 5 ) Rsym12 . 6 ( 65 . 6 ) 13 . 7 ( 87 . 5 ) 13 . 1 ( 78 . 7 ) 15 . 8 ( 59 . 3 ) 7 . 4 ( 80 . 1 ) Rpim6 . 3 ( 36 . 6 ) 6 . 6 ( 57 . 1 ) 6 . 8 ( 41 . 9 ) 4 . 0 ( 15 . 1 ) 5 . 1 ( 54 . 1 ) I/\\u03c3 ( I ) 16 . 4 ( 1 . 9 ) 13 . 2 ( 1 . 4 ) 7 . 9 ( 1 . 3 ) 19 . 3 ( 4 . 7 ) 23 . 5 ( 3 . 2 ) Completeness ( % ) 99 . 0 ( 98 . 6 ) 99 . 0 ( 99 . 2 ) 98 . 3 ( 98 . 9 ) 100 . 0 ( 100 ) 84 . 3 ( 79 . 7 ) Redundancy5 . 0 ( 3 . 9 ) 6 . 1 ( 6 . 0 ) 4 . 4 ( 4 . 5 ) 15 . 1 ( 15 . 3 ) 2 . 7 ( 2 . 7 ) RefinementResolution ( \\u00c5 ) 50 . 0-2 . 3630 . 0\\u20133 . 1550 . 0\\u20133 . 030 . 0-1 . 55No . of reflections57 , 87151 , 12254 , 10757 , 455Rwork/Rfree ( % ) 20 . 8/24 . 622 . 3/25 . 621 . 1/24 . 523 . 2/26 . 7Protein atoms8 , 29816 , 53616 , 5363 , 933DNA atoms29067400Cofactor atoms4170700Rmsd bond lengths ( \\u00c5 ) 0 . 0090 . 0090 . 0090 . 007Rmsd bond angles ( \\u00b0 ) B factors ( \\u00c52 ) :proteinDNACa2+waterWilson1 . 4\\u200365 . 5107 . 658 . 949 . 558 . 41 . 41 . 461 . 3*Only the ssDNA is ordered . Values in parentheses are for the highest-resolution shell . The structure consists of a \\u223c310 residue domain that contains the PD- ( D/E ) XK nuclease motif , followed by a \\u223c450 residue , C-terminal helicase domain that has two RecA-like folds characteristic of the SF1 helicase subfamily ( domains 1A and 2A; Figure 1A , B ) . In addition , the structure reveals an OB ( oligonucleotide/oligosaccharide-binding ) fold domain N-terminal to the nuclease domain , and a \\u03b2 barrel domain that occurs between the nuclease and helicase domains and which is held in place by a stalk of two long alpha helices ( Figure 1 and Figure 1\\u2014figure supplement 1A ) . The overall structure has a cylindrical shape with a central tunnel through which the ssDNA threads ( Figure 1A , B ) . The base of the cylinder is formed by the nuclease domain , which adopts a doughnut-like structure with the active site embedded in the central tunnel ( Figure 1C ) . The \\u03b2 barrel and helicase 1A domains pack on top of the nuclease doughnut and extend the cylinder and central tunnel . The helicase 2A domain , which packs with the 1A domain as in the ADP states of other SF1 helicase structures , hangs over the tunnel opening at the top of the cylinder . Most of the DNA-binding sites of the nuclease and helicase 1A domains are inside the tunnel , whereas those of the helicase 2A domain are solvent exposed . The OB domain decorates the exterior of the nuclease domain , and is uninvolved in DNA binding . The nuclease domain is the hub that organizes the overall structure . It packs with the flanking OB and \\u03b2 barrel-stalk domains , as well as the helicase 1B domain ( Figure 1C and Figure 1\\u2014figure supplement 2 ) . The ssDNA is positioned with its 5\\u2019 end at the helicase domain , and its 3\\u2019 end at the nuclease domain ( Figure 1A ) . A 7-nt segment at the 5\\u2019 end contacts first the helicase 2A domain outside the tunnel and then the 1A domain inside the tunnel . The DNA crosses over from the helicase to the nuclease domains over the next two nucleotides , which are in the vicinity of the \\u03b2 barrel domain in the middle of the tunnel , but do not make any protein contacts ( Figure 1A ) . The subsequent 6-nt segment binds to the nuclease domain . Here , the first three nucleotides are fully enclosed by the tunnel , while the last three nucleotides contact the tunnel opening . The DNA bases stack continuously , except for a base step at the helicase , one at the nuclease and one at the transition between the two domains ( Figure 2C ) . Two of the three unstacked base steps are at pyrimidine-pyrimidine pairs , and this may contribute to the DNA binding at a well-defined register . An N-terminal acidic/amphipathic \\u03b1 helix ( \\u03b11; residues 1 to 13 ) packs with the helicase domain of a symmetry-related complex . The \\u03b11 helix is flexibly tethered to the rest of the protein , as the 6 residues that connect it to the OB domain have no electron density in the crystals . As shown later , this helix is one of the two Rpa-binding elements of Dna2 . 10 . 7554/eLife . 09832 . 007Figure 2 . Nuclease and helicase domain structures and DNA contacts . ( A ) Superposition of the Dna2 nuclease domain on the AddB nuclease domain . The N-terminal \\u03b2\\u03b1\\u03b1 extension ( residues 122 to 154 ) and the C-terminal \\u03b1\\u03b1 extension ( residues 384 to 412 ) is colored in orange , with the corresponding elements of AddB in dark cyan . Green spheres are calcium ions . ( B ) DNA contacts and active site residues of the Dna2 nuclease domain . Hydrogen bonds are depicted as green dotted lines , calcium ions as blue spheres , water as a red sphere . ( C ) Diagram showing the contacts depicted in ( B ) and ( E ) . The residues are colored according to the domain they belong as in Figure 1D ( mc: main chain , sc: side chain ) . ( D ) Superposition of the Dna2 helicase domain on Ighmbp2 . Dna2 is colored as in Figure 1D . The Ighmbnp2 1A ( hel1A ) and 2A ( hel2A ) helicase domains are colored gold , its \\u03b2 barrel in light orange , and stalk dark orange . ( E ) DNA contacts of the helicase 1A ( pink ) and 2A ( cyan ) domains , showing residues that are involved in either hydrogen bond ( green dotted lines ) or van der Waals contacts . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 00710 . 7554/eLife . 09832 . 008Figure 2\\u2014figure supplement 1 . Electron density at the nuclease active site and structural similarity of the Dna2 helicase to Upf1-subfamily RNA/DNA helicases . ( A ) Top panel shows a stereo view of the mFo-DFc electron density at the active site , calculated with phases after omitting the two calcium ions ( blue spheres ) , the water molecule ( red sphere ) , a 3-nt segment centered on the scissile phosphate group and the side chain of Lys301 , and subjecting the rest of the entire structure to simulated annealing from 3000\\u00b0K . The dark blue map is contoured at 5 \\u03c3 , and the light green map at 2 . 5 \\u03c3 . The orientation is similar to that of Figure 2B . For comparison , the corresponding residues of the \\u03bb nuclease-DNA complex ( Zhang et al . , 2011 ) are shown as black sticks ( the lysine is mutated to alanine in that structure ) , and the calcium ions and water molecule as black spheres . The Dna2 Ca-1 ion and water molecule have a coordination shell and position very similar to the first magnesium and associated water in \\u03bb nuclease , but the position of Ca-2 is different . The Ca coordination shell and residue labels are shown in bottom panel . For clarity , not shown are the electron density , \\u03bb nuclease , and the His164 side chain to which the green-dotted bond from Ca-1 would be connected . ( B ) Superposition of Dna2 helicase domains 1A ( pink ) and 2A ( cyan ) with Ighmbp2 ( gold ) . DNA is colored in red , RNA is colored in green . The only regions of the Dna2 helicase that do not have counterparts in the Upf1 subfamily are the ~40 residue 1B domain , which packs with the nuclease domain , and the ~20 residue 2B domain , which is distal from the rest of the protein and the DNA ( Figure 1D ) . ( C ) Superposition of individual Dna2 helicase domains 1A ( pink ) and 2A ( cyan ) with Upf1 ( gold ) . DNA is colored in red , RNA is colored in green . Motif III residues are shown as sticks . Supplemental discussion of helicase translocation . Based on the proposed SF1B translocation mechanism ( Saikrishnan et al . , 2009 ) , ATP binding and concomitant closure of the cleft between the 1A and 2A domains would be coupled to domain 2A releasing the DNA and rebinding it at a register shifted by 1 nt in the 5\\u2019 to 3\\u2019 direction , while domain 1A would retain its DNA . On ATP hydrolysis , domain 1A would be the one releasing its DNA and rebinding it at the +1 nt register , restoring the initial open-cleft conformation . In the Dna2-ssDNA structure , however , the rebinding of 1A to DNA at the shifted register will have to await the transient dissociation of the nuclease domain from DNA . The increased half life of the closed cleft would likely also reduce the rate of exchange of ADP for ATP , as the \\u03b3 phosphate binding site of the helicase fold is far less solvent accessible in the closed cleft . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 008 As predicted , the Dna2 nuclease domain contains the core \\u03b1\\u03b2\\u03b2\\u03b2\\u03b1\\u03b2 fold of the PD- ( D/E ) XK nuclease superfamily ( Aravind et al . , 2000; Pingoud et al . , 2005 ) . Based on the DALI server , its closest structural homologs are the bacteriophage \\u03bb exonuclease and the E . coli RecB and B . subtilis AddB nucleases , all of which are involved in the resection of DNA ends during homologous recombination ( Krajewski et al . , 2014; Singleton et al . , 2004; Zhang et al . , 2011 ) . These superimpose on the\\u00a0\\u223c310-residue Dna2 nuclease domain with a \\u223c2\\u2009\\u00c5 root-mean-square deviation ( r . m . s . d . ) in the C\\u03b1 positions of 128 , 112 and 106 residues , respectively . The most extensive similarity is exhibited by AddB , which shares with Dna2 the presence of the iron-sulfur cluster and most of the insertions and extensions that decorate the core fold ( Figure 2A ) ( Krajewski et al . , 2014; Yeeles et al . , 2009 ) . While most of these elements have diverged beyond a \\u223c2\\u2009\\u00c5 r . m . s . d . , their secondary structures , arrangement and structural implications are closely related . In particular , both proteins have a \\u03b2\\u2013\\u03b1\\u2013\\u03b1 N-terminal extension and an \\u03b1\\u2013\\u03b1 C-terminal extension that are stapled together by the iron-sulfur cluster ( Figure 2A ) . This iron-sulfur cluster domain supports a loop that crosses over the catalytic channel and converts it to a tunnel through which the DNA has to thread . \\u03bb exonuclease and RecB also have a crossover loop , but their N- and C-terminal extensions that anchor it are structurally divergent from Dna2 , and , more significantly , they lack the iron-sulfur cluster ( Singleton et al . , 2004; Zhang et al . , 2011 ) . The Dna2 nuclease domain binds to a total of four phosphodiester groups , two before and two after the scissile phosphate group ( Figure 2B , C ) . The scissile phosphate group contacts two calcium ions , one ( Ca-1 ) through a non-bridging oxygen , and another ( Ca-2 ) through the 3\\u2019 bridging oxygen of the preceding base . The Ca-1 ion has an octahedral coordination shell very similar to other PD- ( D/E ) XK nucleases ( Pingoud et al . , 2005; Zhang et al . , 2011 ) . It\\u2019s formed by the side chains of His164 , Asp278 and Glu299 , the Leu300 main chain carbonyl group , the non-bridging oxygen atom of the scissile phosphate group , and a water molecule , which also hydrogen bonds to Lys301 and is positioned for nucleophilic attack on the scissile phosphate ( Figure 2B and Figure 2\\u2014figure supplement 1A ) . Lys301 is buttressed by Gln314 , a motif IV residue characteristic of the RecB and \\u03bb exonuclease families ( Aravind et al . , 2000 ) . The relative position of the second , Ca-2 ion differs from other nucleases ( Figure 2\\u2014figure supplement 1A ) , and it is not clear whether it reflects the use of calcium instead of magnesium , or a divergent aspect of the nuclease mechanism of Dna2 . The base groups of the 6 nt segment stack in two sets of three , with Met157 and other crossover loop side chains wedging in between the Ade12 and Gua13 base groups ( Figure 2B ) . This is very similar to the \\u03bb exonuclease crossover loop , which marks the transition from double stranded to single stranded and is thought to play a key role in unwinding dsDNA , although \\u03bb exonuclease does so without a helicase domain and in the context of a homo-trimeric assembly that contacts the dsDNA of the substrate ( Zhang et al . , 2011 ) . Nevertheless , Met157 and the crossover loop may have an analogous function in the weak strand-separating activity of Dna2 . Dna2\\u2019s combination of the \\u03b2 barrel , stalk and helicase domains , and their relative arrangement are strikingly similar to RNA/DNA helicases of the Upf1 subfamily , which contains Upf1 , Ighmbp2 and Senataxin ( Fairman-Williams et al . , 2010 ) . The \\u223c600-residue assembly of the stalk , \\u03b2 barrel and helicase 1A and 2A domains can be superimposed on the structure of Ighmbp2 with a 1 . 5\\u2009\\u00c5 C\\u03b1 r . m . s . d . over 452 residues , with most of the non-superimposing residues accounted for by a \\u223c10\\u00b0 rotation of the \\u03b2 barrel , which otherwise is structurally conserved ( Figure 2D ) . The structural similarity between Dna2 and Ighmbp2 is actually more extensive than that between Ighmbp2 and Upf1 ( 1 . 8 C\\u03b1 \\u00c5 r . m . s . d for 436 residues ) ( Chakrabarti et al . , 2011; Lim et al . , 2012 ) . The Upf1-like subfamily is one branch of the SF1B family of helicases that translocate in the 5\\u2019 to 3\\u2019 direction ( Fairman-Williams et al . , 2010 ) . The other SF1B branch , represented by the DNA helicase RecD2 ( Saikrishnan et al . , 2009 ) , lacks the barrel and stalk domains . Furthermore , the individual RecD2 1A and 2A domains are not as similar to Dna2 , with alignment r . m . s . d . values of 1 . 8\\u2009\\u00c5 for 134 residues and 1 . 9\\u2009\\u00c5 for 97 residues , respectively , in contrast to the Dna2-Ighmbp2 alignment , where the corresponding values are 1 . 5\\u2009\\u00c5 for 204 residues and 1 . 4\\u2009\\u00c5 for 183 residues , respectively . Together , these structural observations suggest that Dna2 evolved by incorporating an ancestral Upf1-family helicase . DNA binds to Dna2 through both its phosphodiester and base groups ( Figure 2E ) . Dna2-base interactions include Van der Waals contacts from the motif III loop ( Leu795 and Val797 ) , which wedges in between the bases of the last two 2A-bound nucleotides . These contacts are consistent with the proposed role of motif III in preventing DNA sliding during translocation of SF1B helicases ( Saikrishnan et al . , 2009 ) . Dna2-phosphodiester interactions involve protein pockets that are rich in backbone amide and short-side chain hydroxyl groups ( Figure 2E ) . Three consecutive phosphodiester groups , near the 5\\u2019 end of the DNA , bind to the 2A domain ( Figure 2C , E ) . The fourth phosphodiester group is in the cleft between the 2A and 1A domains and does not contact the protein , while the fifth and sixth phosphodiester groups bind to pockets on the 1A domain ( Figure 2C ) . Contacts to the ribose groups are minimal , and they are consistent with the helicase accommodating the 2\\u2019 hydroxyl group of RNA . In fact , the DNA contacts as well as the phosphodiester backbone conformation are very similar to those of the Ighmbp2-RNA , Upf1-RNA complexes ( Figure 2\\u2014figure supplement 1B , C ) . This is in contrast to the RecD2-ssDNA structure , where extensive aromatic and van der Waals contacts with the sugar are thought to discriminate against RNA ( Saikrishnan et al . , 2009 ) . The helicase and ssDNA-dependent ATPase activities of Dna2 are considerably weaker than other helicases ( Budd et al . , 2000; Masuda-Sasa et al . , 2006 ) . One possible explanation , at least for the low ATPase/translocation rate , is the helicase domain being 5\\u2019 to the nuclease domain on the DNA . This would , in principle , make completion of the ATPase/translocation cycle dependent on the nuclease domain releasing its grip on the DNA ( see Figure 2\\u2014figure supplement 1 legend for discussion of translocation ) . The structure explains why the nuclease activity of Dna2 requires the ssDNA to have a free end ( Bae and Seo , 2000; Kao et al . , 2004 ) . The active site is embedded in a \\u223c10\\u2009\\u00c5 wide portion of the tunnel , and the tunnel entrances leading to it are too narrow to accommodate dsDNA or a single-stranded loop of a gap substrate ( Figure 1A , B ) . Since isolated Dna2 can degrade ssDNA with either 5\\u2019 to 3\\u2019 or 3\\u2019 to 5\\u2019 polarity in vitro , the ssDNA must be able to enter and thread through either end of the tunnel . In 5\\u2019 flap processing , threading likely proceeds through initial , transient interactions at the nuclease domain entrance of the tunnel , with subsequent re-binding events occurring further along the tunnel interior . This threading is likely related to that of \\u03bb exonuclease , where a 5\\u2019 terminal phosphate binding site inside the enzyme is proposed to drive the forward movement of the DNA in an electrostatic ratchet mechanism ( Zhang et al . , 2011 ) . The 5\\u2019 to 3\\u2019 nuclease activity was shown to require \\u223c15 nts of ssDNA for optimal affinity and cleavage ( Bae et al . , 2001; Gloor et al . , 2012 ) . This suggests that the nuclease domain-DNA interactions do not provide sufficient binding energy or their half-life is too short relative to the catalytic step of cleavage . We thus presume that the 5\\u2019 end of the ssDNA threads through the nuclease portion of the tunnel without cleavage , until it reaches the helicase domain and engages the DNA-binding pockets there . To investigate this in more detail , we assayed the Dna2 affinity and cleavage of a series of 5\\u2019 overhang substrates with ssDNA lengths that extend successively from the nuclease to the helicase 1A and 2A domains . The ssDNA consisted of deoxythymidine nucleotides , which have minimal secondary structure and thus do not require Rpa for cleavage by Dna2 ( 5\\u2019 ( dT ) 6 to 5\\u2019 ( dT ) 24 ) ( Figure 3\\u2014figure supplement 1A ) . The short substrates that can span only the nuclease domain ( 5\\u2019 ( dT ) 6 and 5\\u2019 ( dT ) 8 ) have Dna2 affinities approximately two orders of magnitude lower than that of 5\\u2019 ( dT ) 24 , and they exhibit minimal cleavage ( Figure 3\\u2014figure supplement 1A , B ) . With the slightly longer 5\\u2019 ( dT ) 8 , low level cleavage occurs 7 and 8 nucleotides from the 5\\u2019 end , indicating that the ssDNA extends to the helicase 1A domain ( Figure 3\\u2014figure supplement 1A ) . The sites of cleavage indicate that Dna2 opens up \\u223c2 base pairs ( bps ) of the duplex , since the tunnel entrance 3\\u2019 to the scissile phosphate is too narrow to accommodate double-stranded DNA . As ATP is neither required , nor has a significant effect , this DNA unwinding is the result of threading , likely analogous to DNA unwinding by \\u03bb exonuclease ( Zhang et al . , 2011 ) . Extending the ssDNA by 2 nts , ( 5\\u2019 ( dT ) 10 ) , results in a major increase in Dna2 affinity and cleavage , with the cleavage sites indicating the engagement of the helicase 2A domain after the unwinding of \\u223c2 bps ( Figure 3\\u2014figure supplement 1A ) . With 5\\u2019 ( dT ) 17 where the ssDNA is long enough to reach both the 1A and 2A domains without duplex unwinding , there is a final increase in Dna2 affinity and cleavage that plateau at the levels of 5\\u2019 ( dT ) 24 . The 3\\u2019 to 5\\u2019 nuclease activity would require the 3\\u2019 end of the ssDNA to enter the tunnel at the helicase 2A domain . As the phosphodiester-binding sites of the 2A domain are fully accessible to bulk solvent , the structure suggests that 3\\u2019 end threading should be more efficient than 5\\u2019 end threading in the absence of ATP . Indeed , Dna2 cleaves a 3\\u2019 overhang ( dT ) 18 substrate at least 3-fold faster than the corresponding 5\\u2019 ( dT ) 18 ( Figure 3A ) . The structure further suggests that the reported inhibition of 3\\u2019 end cleavage by ATP is due to 3\\u2019 end threading being counteracted by the helicase domain moving the DNA in the opposite direction ( Figure 3A ) ( Bae and Seo , 2000; Masuda-Sasa et al . , 2006 ) . 10 . 7554/eLife . 09832 . 009Figure 3 . Dna2 nuclease activity . ( A ) Denaturing PAGE showing ATP inhibits the nuclease activity on 3\\u2019 overhang substrate , while slightly increasing it for the 5\\u2019 overhang substrate . Substrates are at 15 nM . For this and subsequent nuclease assays , cleavage was quantified by loss of substrate and plotted with \\u00b1 s . d . error bars ( n = 3 ) . ( B ) Nuclease time course of 2 nM Dna2 with 10 nM 5\\u2019 RNA-DNA overhang or 5\\u2019 DNA overhang substrates . ATP or AMPPNP is at 1 . 3 mM . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 00910 . 7554/eLife . 09832 . 010Figure 3\\u2014figure supplement 1 . The ssDNA-length dependence of the DNA affinity correlates with cleavage rates . ( A ) Denaturing PAGE showing length dependency and cleavage sites of 5\\u2019 overhang DNA at a 30 nM concentration ( asterisk indicates FAM labeling of the 3\\u2019 end ) in the presence ( top panel ) or absence ( bottom ) of 1 . 3 mM ATP . Also shown are the dissociation constants ( Kd ) of Dna2 ( D278A ) for each substrate , according to the data shown in Figure 3\\u2014figure supplement 1B . ( B ) Electrophoretic mobility shift assays ( EMSA ) of Dna2 ( D278A ) binding to 5\\u2019 overhang DNA substrates . The graph shows fraction bound . Dotted lines are simulated binding curves based on the indicated Kd values . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 010 Taken together , these findings support the model that a major role for the helicase domain is augmenting the DNA affinity of Dna2 . As lack of ATP hydrolysis does not affect overall DNA binding by the helicase 1A and 2A domains , this model is consistent with the Dna2 ATPase-activity being dispensable for viability in yeast , and for DSB end resection in vitro , in contrast to the nuclease activity that is essential for both ( Bae and Seo , 2000; Budd et al . , 2000; Cejka et al . , 2010; Niu et al . , 2010; Zhu et al . , 2008 ) . RNA , which is not cleaved by Dna2 , can substitute for part of DNA in the length-dependency of cleavage , suggesting that it can interact with at least part of Dna2 ( Bae et al . , 2001; Bae and Seo , 2000 ) . In accord , we find that the affinity of Dna2 for a 5\\u2019 ( U ) 24 RNA overhang-DNA duplex is within an order of magnitude of its affinity for a comparable all-DNA substrate ( Figure 3\\u2014figure supplement 1B ) . This observance , coupled with the structural similarity of Dna2 to the Upf1 subfamily of RNA/DNA helicases raises the possibility that the helicase activity facilitates the bypassing of the 5\\u2019 RNA primer of Okazaki fragments . Indeed , ATP but not the non-hydrolyzable ATP analogue AMPPNP stimulates the cleavage of a 5\\u2019 ( U ) 12\\u2013 ( dT ) 12 RNA-DNA overhang substrate by \\u223c50% ( Figure 3B , top panel ) . By contrast , ATP had a minimal effect on the cleavage of the corresponding all-DNA ( dT ) 24 substrate , while AMPPNP inhibited cleavage slightly ( Figure 3B , bottom panel ) . Rpa is a heterotrimeric protein that consists of the Rpa70 , Rpa32 and Rpa14 subunits . Genetic and biochemical studies in budding yeast indicated that Dna2 binds to the Rpa70 subunit , through an interaction between the N-terminal portions of the two proteins ( Bae et al . , 2003 ) . These studies further pointed to additional binding sites on Dna2 and Rpa70 , as deletion of the N-terminal interacting segments reduced but did not eliminate Dna2-Rpa association and the stimulation of the nuclease activity ( Bae et al . , 2003 ) . The N-terminal portion of yeast Dna2 ( residues 1 to 405 ) is poorly conserved in mammalian orthologs and also contains a \\u223c350-residue yeast-specific extension , although it appears to contain an acidic/amphipathic helix analogous to \\u03b11 of mouse Dna2 , and likely encompasses a portion of the OB domain ( Figure 1\\u2014figure supplement 1A ) . We thus tested whether a mouse Dna2 fragment consisting of the \\u03b11 helix and OB domain ( thereafter \\u03b11OB; residues 1 to 122 ) binds to mouse Rpa using a GST pull-down assay . As shown in Figure 4A , GST-\\u03b11OB but not isolated GST binds to heterotrimeric Rpa in a 30 \\u03bcM stoichiometric solution ( lanes 1 and 2 ) . 10 . 7554/eLife . 09832 . 011Figure 4 . Dna2 physically interacts with Rpa . ( A ) GST pull-down assay showing that \\u03b11OB of Dna2 ( residues 1\\u2013122 ) interacts with Rpa70NAB ( 1\\u2013431 ) , but not the rest of Rpa heterotrimer ( Rpa ( -NAB ) ) . ( B ) Schematic drawing of the Rpa trimer , showing the OB domains as rectangles and the winged helix ( WH ) domain of Rpa32 as a sphere . ( C ) ITC curves for the human \\u03b11OB-Rpa70NAB , \\u03b11-Rpa70N and OB-Rpa70AB complexes . ( D ) Structure of the \\u03b11-Rpa70N complex . The \\u03b11 peptide is in yellow and Rpa70N in cyan . For clarity , the main chain amide group of L87 and carbonyl groups of M11 , K13 , F15 are not labeled . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 01110 . 7554/eLife . 09832 . 012Figure 4\\u2014figure supplement 1 . Comparison of the Dna2 \\u03b11\\u2013Rpa70 OBN , p53\\u2013Rpa70 OBN and Dna2 OB structures . ( A ) Superposition of the \\u03b11\\u2013Rpa70 OBN structure on the fusion protein of Rpa70 OBN and p53 . In the fusion protein structure , the Rpa70 OBN ( gray ) groove has two p53 peptides , one ( cyan ) from its fused p53 and another ( green ) from a crystallographic-symmetry related fusion protein . While it was suggested that only the latter represented the solution-state complex , our structure has aspects of both p53-Rpa70 OBN interfaces . An acidic/amphipathic helix from the symmetry-related p53 superimposes with the Dna2 helix , and part of the second p53 peptide overlaps with the Dna2 \\u03b2 turn and associated contacts . \\u03b11 peptide is red . ( B ) The Dna2 OB groove ( cyan ) has an overall hydrophobic- and basic-residue content similar to that of the Rpa70 OBN ( pink ) . The L12 and L45 loops ( labeled ) , which in DNA-binding OB domains partially wrap around the DNA , but are truncated in the Rpa70 OBN domain are , respectively , truncated and absent in the Dna2 OB domain , resulting in a wide open groove . Residues mutated in Figure 5A are labeled . ( C ) ITC titration profile of human Dna2 \\u03b11 ( residues 1\\u201320 ) binding to human Rpa70N ( residues 1\\u2013120 ) . ( D ) ITC titration profile of human Dna2 OB ( residues 21\\u2013122 ) binding to human Rpa70AB ( residues 181\\u2013422 ) . ( E ) ITC titration profile of human Dna2 \\u03b11OB ( residues 1\\u2013122 ) binding to human Rpa70NAB ( residues 1\\u2013422 ) . ( F ) The Dna2 OB domain does not exhibit any detectable DNA-binding at concentrations up to 80 \\u03bcM in an EMSA assay with ( dT ) 20 ssDNA . A shifted , smeary band appearing at 160 \\u03bcM OB has ~ 4% of the total DNA . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 012 The Dna2-interacting N-terminal segment of yeast Rpa70 ( residues 1 to 180 ) consists of an OB fold domain ( named OBN ) that is a known protein-protein interaction site and a ~60 residue flexible linker ( Figure 4B ) ( Bochkareva et al . , 2005; Fan and Pavletich , 2012; Fanning et al . , 2006; Gomes and Wold , 1996 ) . The rest of Rpa70 consists of three OB folds that are the main DNA-binding domains ( named DBD-A , DBD-B and DBD-C; Figure 4B ) ( Bochkarev et al . , 1997; Fan and Pavletich , 2012 ) . Because DBD-A and DBD-B can also participate in protein-protein interactions ( Jiang et al . , 2006; Loo and Melendy , 2004; Yuzhakov et al . , 1999 ) , this raised the possibility that they account for the remainder of Rpa70\\u2019s Dna2 affinity . We tested this using a mouse Rpa70 fragment containing the OBN , DBD-A and DBD-B domains ( residues 1 to 431; thereafter Rpa70NAB ) . As shown in Figure 4A , GST-\\u03b11OB binds to Rpa70NAB ( lanes 3 and 4 ) but not to the Rpa heterotrimer lacking this fragment ( Rpa ( -NAB ) ; lanes 5 to 6 ) . The dissociation constant ( Kd ) of the \\u03b11OB\\u2013Rpa70NAB complex , determined by isothermal titration calorimetry ( ITC ) , is 12 \\u00b1 1 \\u03bcM ( Figure 4C and Figure 4\\u2014figure supplement 1E ) . Using ITC , we found that the Dna2 \\u03b11 helix ( residues 1 to 20 ) binds to the OBN domain of Rpa70 ( residues 1 to 120; thereafter Rpa70N ) , while the Dna2 OB domain ( residues 21 to 122 ) binds to the Rpa70 fragment containing DBD-A and DBD-B ( residues 181 to 422; thereafter Rpa70AB ) , with Kd values of 34 \\u00b1 7 and 46 \\u00b1 10 \\u03bcM , respectively ( Figure 4C and Figure 4\\u2014figure supplement 1 ) . The Kd values of these subcomplexes relative to that of \\u03b11OB\\u2013Rpa70NAB indicate only a low level of cooperativity , consistent with the two interacting elements on both Rpa and Dna2 being separated by flexible linkers . Further dividing Rpa70AB into the individual DBD-A and DBD-B polypeptides failed to show detectable binding to Dna2 OB under the same conditions , indicating both are required ( not shown ) . We co-crystallized a human Dna2 peptide ( residues 1 to 20 ) containing the \\u03b11 helix with the OBN domain of human Rpa70 ( residues 1 to 120 ) . In the 1 . 6\\u2009\\u00c5 structure , residues 6 to 17 of Dna2 form a 2-turn amphipathic helix followed by a \\u03b2 turn , while the rest are disordered ( Figure 4D ) . The peptide binds to a shallow OBN groove that corresponds to the DNA-binding grooves of other OB domains ( Bochkarev et al . , 1997; Fan and Pavletich , 2012 ) . The only other OBN-peptide structure available is of OBN fused to the p53 transactivation domain peptide , where two p53 peptides occupy the OBN groove ( Bochkareva et al . , 2005 ) . The amphipathic helix of Dna2 overlaps with one of the p53 peptides , while the \\u03b2 turn coincides with part of the other p53 peptide ( Figure 4\\u2014figure supplement 1A ) . The mixed basic and hydrophobic character of the OBN groove complements the acidic-hydrophobic nature of the Dna2 peptide . Four arginine guanidinium groups and one backbone amide group of OBN contact two side chain carboxylate and three backbone carbonyl groups from \\u03b11 ( Figure 4D ) . One of these OBN arginine residues ( Arg43 ) splits the otherwise hydrophobic groove , demarcating two hydrophobic pockets . One pocket ( Ile33 , Met57 , Leu87 , Val93 and Ile95 ) binds to the hydrophobic face of the Dna2 helix ( Leu7 , Leu10 and Met11 ) , while the other pocket ( Leu45 and aliphatic portions of Arg31 , Arg43 , and Ser54 ) binds to Phe15 and Trp16 from the \\u03b2 turn of Dna2 . We do not know the structure of the Dna2 OB domain bound to the Rpa70 DBD-A and DBD-B domains , but the OB structure in intact Dna2 is consistent with a role in protein-protein interactions ( Figure 4\\u2014figure supplement 1B ) , and the isolated OB domain polypeptide does not exhibit any DNA-binding in EMSA at concentrations up to 80 \\u03bcM ( Figure 4\\u2014figure supplement 1F ) . A common feature of proteins involved in Rpa-dependent processes is their ability to bind to Rpa , either directly or through accessory factors , and this is thought to form the basis for the displacement of Rpa from ssDNA ( Fanning et al . , 2006; Zou et al . , 2006 ) . Rpa displacement is best understood with the simian virus 40 ( SV40 ) T antigen and related viral replication proteins , where T antigen-Rpa interactions allosterically modulate Rpa\\u2019s ssDNA affinity ( Jiang et al . , 2006; Loo and Melendy , 2004; Yuzhakov et al . , 1999 ) . Because Dna2 has been shown to displace Rpa from 5\\u2019 flap DNA ( Stewart et al . , 2008 ) , we sought to address whether Dna2-Rpa interactions have an analogous , direct role in Rpa displacement , or whether they reflect a simple recruitment process that allows Dna2 to better compete with Rpa for ssDNA . We first confirmed that both the \\u03b11 helix and OB domain of Dna2 are required for the stimulation of the nuclease activity by Rpa . For this , we used a 5\\u2019 overhang substrate with a stem loop secondary structure that makes the Dna2 nuclease activity dependent on Rpa ( 5\\u2019SL24 ) ( Figure 5\\u2014figure supplement 1A ) . In keeping with the findings with the yeast Dna2\\u25b3405N mutant , deletion of the Dna2 \\u03b11 helix reduced cleavage of the Rpa-coated 5\\u2019SL24 by a factor of ~2 compared to intact Dna2 ( Figure 5A ) . The analogous experiment to address the importance of the OB domain was not possible , as the OB-deleted Dna2 is insoluble ( not shown ) . We instead mutated three OB residues at positions corresponding to protein-protein contacts on the Rpa70 OBN domain ( Figure 4\\u2014figure supplement 1B ) . As shown in Figure 5A , two of these mutations ( I82A and R66E ) synergized with \\u03b11 deletion and reduced 5\\u2019SL24 cleavage further , whereas the third ( I110A ) had no discernible effect . To rule out that these mutations do not affect the structural integrity of Dna2 , we tested the Rpa-independent cleavage of the 5\\u2019 ( dT ) 24 substrate and found that it is not affected by neither the OB mutations nor the \\u03b11 deletion ( Figure 5B ) . 10 . 7554/eLife . 09832 . 013Figure 5 . Both Dna2-Rpa interactions are important for Dna2 stimulation , but only one is mutually exclusive with Rpa-ssDNA interactions . ( A ) Cleavage of a 5\\u2019 stem-loop overhang substrate ( 15 nM ) by wild type , \\u03b11-deleted , and OB mutant Dna2 in the presence of 15 nM Rpa . ( B ) Nuclease activity of the same set of enzymes as in ( A ) , but using a 5\\u2019 ( dT ) 24 overhang substrate in the absence of Rpa . ( C ) GST pull-down assay showing that \\u03b11OB and ( dT ) 24 ssDNA bind to Rpa in a partially mutually exclusive manner . Protein and DNA concentrations are in \\u03bcM , and the bar graph shows the quantitation of Rpa70 binding relative to the lane in the absence of DNA , which is set to 1 . Error bars are standard deviations from three repetitions of each experiment . ( D ) The binding of \\u03b11 to the Rpa70N polypeptide is unaffected by ( dT ) 8 ssDNA . GST pull-down assay and quantitation as in ( C ) . ( E ) The binding of OB to the Rpa70AB polypeptide is abolished by ( dT ) 8 ssDNA . GST pull-down assay and quantitation as in ( C ) . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 01310 . 7554/eLife . 09832 . 014Figure 5\\u2014figure supplement 1 . Cleavage of a ssDNA containing secondary structure is Rpa dependent . ( A ) Cleavage of a 5\\u2019 overhang substrate that has a predicted 6 bp stem-loop secondary structure at the 5\\u2019 end of the ssDNA ( 5\\u2019 SL24 ) . Dna2 , at the indicated concentrations , failed to cleave 5\\u2019 SL24 ( 30 nM ) appreciably under conditions where ~95% of the corresponding 5\\u2019 ( dT ) 24 substrate was cleaved , irrespective of ATP . Addition of Rpa , at a stoichiometric amount to ssDNA , stimulated SL24 cleavage by over a factor of 20 , but had only a modest effect on ( dT ) 24 cleavage . In the presence of Rpa , the cleavage of the two substrates was within a factor of two , compared to being nearly two orders of magnitude apart in its absence . ( B ) Cleavage of a 178-nt long 5\\u2019 overhang substrate ( 80 nM ) by Dna2 ( 20 nM ) . As with the shorter 5\\u2019 overhang substrates , ATP ( 1 . 3 mM ) has only a minor stimulatory effect . Because mixed-sequence ssDNA of this length would invariably have some secondary structure ( the lowest energy secondary structure , predicted by the UNAfold server ( mfold . rna . albany . edu ) , is shown at bottom , right ) , cleavage is stimulated by Rpa ( 400 nM ) as with the 5\\u2019 SL24 substrate . For this specific experiment , samples are quantified by integration of final products in the box . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 014 We then addressed whether the \\u03b11 helix and OB domain play a direct role in Rpa displacement , which in principle can occur either through the allosteric destabilization of Rpa-ssDNA interactions as shown for T antigen , or through \\u03b11OB binding to an Rpa site that overlaps with or sterically hinders a ssDNA-binding site . We did not expect the isolated \\u03b11OB , in the absence of the DNA-affinity provided by the nuclease and helicase domains , to displace Rpa from ssDNA , as the affinities of the \\u03b11OB-Rpa and Rpa-ssDNA complexes differ by 5 orders of magnitude ( Kd values of \\u223c12 \\u03bcM and \\u223c100 pM , respectively ) . Instead , we reasoned that if \\u03b11OB has a role destabilizing Rpa-ssDNA interactions , then this should be reflected in ssDNA interfering with \\u03b11OB-Rpa association . As shown in Figure 5C , addition of ( dT ) 24 substantially reduced Rpa binding to GST-\\u03b11OB ( lanes 1 to 3 ) , consistent with \\u03b11OB and ssDNA interacting with Rpa in a mutually exclusive manner . However , while one molar equivalent of ( dT ) 24 reduced the bound Rpa by a factor of \\u223c4 , the remaining Rpa was clearly above the background level of the GST-only reaction ( lanes 4 and 5 ) , suggesting that only one of the two Dna2-Rpa interactions is mutually exclusive with Rpa-ssDNA interactions . Consistent with this , addition of ( dT ) 8 had no discernible effect on the binding of GST-\\u03b11 to Rpa70N ( Figure 5D ) , whereas it eliminated the binding of GST-OB to Rpa70AB in a manner dependent on the stoichiometry of ( dT ) 8 to Rpa70AB ( Figure 5E ) . These findings indicate that the interaction between the Dna2 \\u03b11 helix and the Rpa OBN domain is a simple recruitment step , consistent with both of these elements being flexibly tethered to the remainder of their polypeptides and with their lack of ssDNA affinity . This simple recruitment would be important for Dna2 accessing ssDNA-bound Rpa , where the interaction between the Dna2 OB and Rpa DBD-A\\u2013DBD-B domains would not be initially available . The simple recruitment interaction would also increase the effective concentration of the Dna2 OB at the Rpa DBD-A\\u2013DBD-B , as they immediately follow the \\u03b11 helix and OBN domain , respectively ( Figure 6A ) . 10 . 7554/eLife . 09832 . 015Figure 6 . Dna2 displaces Rpa from 5\\u2019 but not 3\\u2019 overhang DNA . ( A ) Schematic of the proposed mechanism of Dna2 displacing Rpa from a 5\\u2019 overhang DNA . Dna2 is shown as a hollow cylinder , except for its \\u03b11 helix and OB fold domains that are shown as a rectangle and circle , respectively . The label \\u201cnuc\\u201d marks the nuclease tunnel entrance into which the ssDNA would thread , and \\u201chel2A\\u201d marks the helicase tunnel exit where the 5\\u2019 end of the ssDNA would end up after threading . ( B ) Schematic illustrating that at a 3\\u2019 overhang DNA , Dna2-Rpa interactions do not result in a free ssDNA end that can thread into the Dna2 tunnel . ( C ) Nuclease-dead Dna2 ( D278A ) displaces Rpa from 5\\u2019- but not 3\\u2019 overhang DNA . The overhang consists of ( dT ) 26 and the DNA is conjugated to streptavidin ( shown by \\u201cS\\u201d ) through a biotin group at the end of the duplex . Column graph showing quantitation of Rpa32 plots the molar ratio relative to the reaction lacking Dna2 for each DNA substrate ( lanes 2 , 3 for 5\\u2019 overhang DNA , and lanes 5 and 6 for 3\\u2019 overhang DNA ) , or relative to lane 1 for the comparison of Rpa loading onto 5\\u2019- and 3\\u2019-overhang DNA . Quantitation of relative Dna2 loading is similarly shown in the last column graph . Error bars are standard deviations from three repetitions of each experiment . DOI: http://dx . doi . org/10 . 7554/eLife . 09832 . 015 This juxtaposition would then increase the probability of DBD-A\\u2013DBD-B transiently associating with OB and dissociating from ssDNA . This is plausible , because while the DNA affinity of DBD-A\\u2013DBD-B ( ~50 nM Kd ) is substantially higher than its Dna2 affinity , those of the individual DBD-A ( 2 \\u03bcM Kd ) and DBD-B ( 20 \\u03bcM Kd ) are not , and as with the intact Rpa heterotrimer , they are thought to associate with and dissociate from DNA sequentially ( Arunkumar et al . , 2005; Fan and Pavletich , 2012; Fanning et al . , 2006 ) . By itself , the transient displacement of DNA from DBD-A\\u2013DBD-B will not lead to the release Rpa from ssDNA . However , because the DBDA\\u2013DBDB is at the 5\\u2019 end of the Rpa-DNA complex ( Bochkarev et al . , 1997; Fan and Pavletich , 2012 ) , and the Dna2 OB domain is next to the nuclease tunnel entrance , the transiently free 5\\u2019 end of the DNA will be well-placed to start threading through the nuclease tunnel ( Figure 6A ) . The threading process then should be able to completely dissociate the already weakened Rpa-DNA complex ( Figure 6A ) . At a 3\\u2019 end of DNA , by contrast , the transient dissociation of DBD-A\\u2013DBD-B from DNA will expose an internal ssDNA lacking an end that can be trapped by the Dna2 tunnel , and DBD-A\\u2013DBD-B will revert to their DNA-bound state ( Figure 6B ) . This mechanism of Rpa displacement predicts that the inhibition of 3\\u2019 end cleavage by Rpa is due to the inability of Dna2 to displace Rpa there . To test this prediction , we conjugated 5\\u2019- or 3\\u2019- ( dT ) 26 overhang DNA that was biotinylated on the duplex end onto streptavidin beads , loaded it with Rpa , and then added nuclease-dead Dna2 ( D278A mutant ) . As shown in Figure 6C , Dna2 reduced the amount of Rpa bound to the 5\\u2019 overhang DNA by \\u223c30% ( lanes 2 and 3 ) , whereas it had a minimal effect on the Rpa bound to the 3\\u2019 overhang DNA ( \\u223c5% reduction; lanes 5 and 6 ) . The amount of Rpa displacement was proportional to Dna2 loading , which was substantial with 5\\u2019 overhang DNA ( \\u223c45% of the reaction lacking Rpa; lanes 1 and 3 ) , but negligible with 3\\u2019 overhang DNA ( \\u223c3%; lanes 4 and 6 ) . In the absence of Rpa , by contrast , the amount of Dna2 loading on the two DNA substrates was essentially identical ( lanes 1 and 4 ) , consistent with the inability of Dna2 to load onto the 3\\u2019 overhang being due to its failure to displace Rpa . \\n\\n## Conclusions\\n The Dna2-ssDNA structure shows that the active site and most of the DNA-binding sites are enclosed in a narrow tunnel , necessitating the threading of the DNA through a tunnel end to access the DNA binding sites . The structure also indicates that the translocase activity of the helicase domain does not drive threading . A 5\\u2019 DNA end would have to thread halfway through the tunnel before it can access the helicase domain , while the threading of a 3\\u2019 end starting at the helicase 2A domain would be opposed by the 5\\u2019 to 3\\u2019 polarity of translocation , as we demonstrate . Instead of translocation , the helicase domain appears to be important for providing DNA affinity and for bypassing the 5\\u2019 RNA primer of Okazaki fragments . The structure also precludes the helicase domain tracking on DNA to any significant extent , because the nuclease domain is ahead , in the 5\\u2019 to 3\\u2019 direction of translocation . This is consistent with in vitro studies showing the ATPase activity to be dispensable for 5\\u2019 flap processing and DSB resection ( Cejka et al . , 2010; Niu et al . , 2010; Zhu et al . , 2008 ) , and the fact that this activity is rather low compared to bona-fide helicases ( Bae and Seo , 2000; Masuda-Sasa et al . , 2006 ) . In yeast , ATPase mutations do result in growth defects ( Budd et al . , 2000 ) , and it is possible this is due to the ATPase activity contributing to the bypassing of the RNA primer of Okazaki flaps , as suggested by our in vitro data . This may be reflected in the similarity of the Dna2 helicase domain to the Upf1 family of RNA/DNA helicases , which is extensive enough to indicate that Dna2 picked up an ancestral Upf1-like helicase during its evolution . The requirement for threading necessitates Dna2 having a mechanism to displace Rpa from ssDNA . As our proposed mechanism of Rpa displacement predicts , we find that Dna2 can displace Rpa from a 5\\u2019 but not a 3\\u2019 end , explaining how Rpa dictates the proper end polarity of the nuclease activity of Dna2 . \\n\\n## Materials and methods\\n Full-length mouse Dna2 was cloned into a pFastbac1 baculovirus vector engineered with a cleavable N-terminal GST tag and a non-cleavable C-terminal FLAG tag , and was expressed in Hi-5 insect cells ( Life technologies , Carlsbad , CA ) . The recombinant protein was purified first by GST-affinity chromatography and , after cleavage of the GST tag , by anion exchange and gel-filtration chromatography . Purified Dna2 was concentrated to \\u223c20 mg/mL in 20 mM Tris-HCl , 250 mM NaCl , 0 . 3 mM TCEP , pH 8 . 0 . All buffers were degassed before use . The various Dna2 mutants and seleno-methionine substituted protein were purified similarly . Seleno-methionine substituted Dna2 was expressed according to manufacturer\\u2019s protocol ( Expression systems , Davis , CA ) and was purified similarly . For the expression of the mouse Rpa heterotrimer , Rpa70 was cloned into a pFastbac1 vector and Rpa32/Rpa14 were cloned into a modified pFastBac-dual vector with Rpa32 fused to a cleavable N-terminal GST-tag . The Rpa heterotrimer was produced by co-infecting Hi-5 cells with both viruses , and was purified as described ( Fan and Pavletich , 2012 ) . The Rpa heterotrimer with truncated Rpa70 was expressed and purified similarly . GST-tagged mouse Dna2 \\u03b11 ( residues 1\\u201320 ) , \\u03b11OB ( residues 1\\u2013122 ) and OB ( residues 21\\u2013122 ) and Rpa70NAB ( residues 1\\u2013431 ) , Rpa70N ( residues 1\\u2013122 ) , and Rpa70AB ( residues 191\\u2013431 ) fragments , as well as the corresponding human polypeptides used in ITC measurements , were cloned into a pGEX-4T vector and expressed in E . coli BL21DE3 cells . They were purified by GST affinity chromatography , ion exchange and/or heparin chromatography , and gel-filtration chromatography . The corresponding untagged polypeptides were expressed fused to an N-terminal 6-His-sumo tag in E . coli BL21DE3 cells . Following nickel affinity chromatography and cleavage of the 6-His-sumo tag by Ulp1 , they were further purified by ion exchange and/or heparin , and gel-filtration chromatography . Crystals of the Dna2-ADP complex were grown in 4\\u00b0C using the hanging drop vapor diffusion method from a crystallization buffer of 80 mM MES , 250 mM Li2SO4 , 2 mM MgCl2 , 8\\u201312% PEG MME 5000 , 0 . 5 mM TCEP , pH 6 . 5 , containing 12 mg/mL Dna2 and 1 mM ADP . Seleno-methionine substituted Dna2 was crystallized under similar conditions using seeding . Crystals of Dna2 bound to 21-nt ssDNA and ADP ( Dna2-ssDNA in Table 1 ) were grown from a crystallization buffer of 80 mM MES , 20 mM CaCl2 , 10 mM spermidine , 4\\u20139% isopropanol , 0 . 5 mM TCEP , pH 6 . 5 , and 1 mM ADP . They contain two molecules in the asymmetric unit . Crystals of Dna2 bound to 5\\u2019 overhang DNA , which consists of a 17-nt 5\\u2019 overhanging ssDNA and a 6 base pair dsDNA ( Dna2-5\\u2019 overhang DNA in Table 1 ) , grew from a similar condition but in a different space group , and have one complex in the asymmetric unit and higher diffraction limits . As there is no electron density for the duplex , we presume it is disordered . All crystals were cryo-protected in crystallization buffer supplemented with 20\\u201325% glycerol or ethylene glycol , and were flash-frozen in liquid nitrogen . The human DNA2 \\u03b11-RPA70 OBN complex was crystallized by mixing an 8 . 7 mg/ml solution of the RPA70N polypeptide ( residues 1\\u2013120 ) with a 3-fold molar excess of a synthetic DNA2 \\u03b11 peptide ( residues 1\\u201320 ) from 50 mM Tris-HCl , 35% PEG 1500 , 2 mM TCEP , pH 8 . 0 . Crystals were cryo-protected in crystallization buffer supplemented with 20\\u201325% glycerol and flash-frozen in liquid nitrogen . Diffraction data were collected at the 24IDC and 24IDE beamlines of the Advanced Photon Source ( Argonne National Laboratory ) and the X29 beamline of the National Synchrotron Light Source ( Brookhaven National Laboratory ) . Data sets were processed with the HKL2000 suite ( Otwinowski and Minor , 1997 ) . The structure of Dna2-ADP complex was determined using SAD with data collected at the selenium edge ( Bricogne et al . , 2003 ) . The phases were improved using solvent flattening and two-fold NCS averaging with multiple masks with the program DM ( Winn et al . , 2011 ) . The model was built using O ( Jones et al . , 1991 ) and Coot ( Emsley et al . , 2010 ) and refined with REFMAC5 ( Winn et al . , 2011 ) and PHENIX ( Adams et al . , 2010 ) using tight NCS restraints on atom positions . Initial phases for the two Dna2-ADP-ssDNA complexes were obtained by molecular replacement with PHASER ( McCoy et al . , 2007 ) using the apo-Dna2 structure as the search model , and the structures were refined using REFMAC5 ( Winn et al . , 2011 ) and PHENIX ( Adams et al . , 2010 ) , with TLS parameterization of temperature factors of the high resolution Dna2-5\\u2019 overhang DNA complex . The Ramachandran plot of the final model has 90 . 5% , 8 . 9% , 0 . 5% and 0% of the residues in the most favored , additional allowed , generously allowed and disallowed regions . The statistics from data collection and refinement are shown in Table 1 . Figures were generated using PyMOL ( http://www . pymol . org ) . Unless otherwise noted , experiments were performed in a 15 \\u03bcL volume in 20 mM Tris-HCl , 125 mM NaCl , 6 mM MgCl2 , 1 . 3 mM ATP , 0 . 2 mg/mL BSA , 2% glycerol , pH 8 . 0 . Reactions were incubated for 30\\u2009min at 25\\u00b0C and stopped by adding 0 . 5% SDS , 20 mM EDTA and 1 unit of proteinase K . Reactions were analyzed by 16% or 12% denaturing urea-PAGE . For Rpa-containing reactions , Rpa was incubated with DNA for 15 min at 4\\u00b0C before the addition of Dna2 . For reactions using 6-FAM labeled DNA , wet gels were scanned using a fluorescent laser scanner ( Fujifilm FLA 5000 ) , and the bands were quantified with ImageGauge software ( Fujifilm ) . For GST pull-down experiments , 30 \\u03bcM GST-tagged mouse \\u03b11-OB ( residues 1 to 122 ) or GST was incubated with an equal molar amount of full-length Rpa , Rpa70NAB or Rpa ( -NAB ) . Binding reactions ( 40 \\u03bcL ) were carried out in 20 mM Tris-HCl , 80 mM NaCl , 0 . 3 mM TCEP , 2% glycerol , pH 8 . 0 at 4\\u00b0C for 30 min before addition of glutathione beads . After 30 min , the beads were washed three times with binding buffer . Proteins were eluted with 20 mM glutathione and analyzed by SDS-PAGE . Other GST pull-down experiments were carried out similarly , with protein concentrations indicated in the main text or figure legends . For Rpa displacement experiments , 2 \\u03bcM 5\\u2019- or 3\\u2019- ( dT ) 26 overhang DNA that was biotinylated at the duplex end was coupled to magnetic streptavidin beads . After washing , the beads were incubated with a 1 molar equivalent of Rpa for 20\\u2009min in 20 mM Tris-HCl , 125 mM NaCl , 2% glycerol , 0 . 3 mM TCEP , 0 . 01% Tween 20 , pH 8 . 0 , followed by the addition of a one molar equivalent of Rpa or Rpa-Dna2 mixture . The beads were incubated for 45 min with mixing , and after 3 washes the beads were boiled and analyzed by SDS-page . ITC experiments were carried out using a MicroCal ITC200 calorimeter ( Malvern Instruments Inc . , Westborough , MA ) at 20\\u00b0C in a buffer of 20 mM HEPES , 80 mM NaCl , 0 . 2 mM TCEP , pH 7 . 5 . Binding reactions ( 10 \\u03bcl ) containing 0 . 075 or 0 . 3 nM of 32P-labelled DNA substrates with increasing amounts of nuclease-dead Dna2 were carried out in 20 mM Tris-HCl , 80 mM NaCl , 0 . 3 mM TCEP , 0 . 2 mg/mL BSA ( New England Biolabs , Ipswich , MA ) , 2% glycerol , pH 8 . 0 . Reactions were incubated on ice for 30\\u2009min . , followed by electrophoresis at 4\\u00b0C on 4% ( w/v ) native PAGE gels in 1x TB buffer . The dried gels were scanned using a phosphorimager ( GE Typhoon 7000 , GE Healthcare , Pittsburg , PA ) , bands were quantified with ImageGauge software ( Fujifilm ) , and the apparent dissociation constants ( Kd ) were calculated from the equilibrium expression of a one-site binding model . Curve fitting was done by minimizing the sum of the square of the differences between the observed fraction of bound DNA and the fraction predicted from the model .\\n\\n\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"split_sections\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"Materials and methods\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 6,\n        \"samples\": [\n          \"\",\n          \"We have integrated a comprehensive and unique combination of analytical capabilities and made them freely available as a Matlab toolbox: 'The Cell Adhesion and Migration Analysis Toolbox' \\u2013 along with the raw quantitative data underlying this study , sample image data to aid implementation , and explanatory documentation ( see doi:10 . 5061/dryad . 9jh6m ) . Briefly , this toolbox includes the following features utilized in this study: Data import and preprocessing based on images and extracted variables ( see Quantitative Image Analysis , below ) ; Smoothing of cell and adhesion trajectories; Membrane protrusion and retraction dynamics extraction; Migration mode classification; Mean square displacement ( MSD ) of cell trajectories; Variable selection for uni- and multivariate analyses; Univariate statitstical analysis; Variable correlation based heat maps; Canonical vector analysis ( CVA ) ; Principal component analysis ( PCA ) ; and two dimensional kernel density estimation . A more detailed description is available within the Matlab Toolbox . All cell lines were acquired directly from the American Type Culture Collection ( ATCC , Manassas , VA ) , and therefore were not further authenticated . The cell lines used herein are not members of the ATCC list of commonly misidentified cell lines . H1299 ( human non-small cell lung carcinoma , ATCC; ATCC# CRL-5803 , mycoplasma negative ) cells stably expressing EGFP-Paxillin and RubyRed-LifeAct , termed H1299 P/L cells , were established and maintained in RPMI 1640 medium ( Gibco \\u2013 Thermo Fisher Scientific , Waltham , MA ) containing 400 \\u00b5g/ml geneticin ( G-418 sulfate , Gibco ) with 10% fetal bovine serum ( Gibco ) and 1 mM glutamine , as described previously ( Lock et al . , 2014; Kiss et al . , 2015 ) . BT549 ( ductal breast carcinoma , ATCC; ATCC# HTB-122 , mycoplasma negative ) cells were maintained in RPMI 1640 medium ( Gibco ) containing 10% fetal bovine serum ( Gibco ) and 1 mM glutamine . MDA-MB-231 ( adenocarcinoma , ATCC; ATCC# HTB-26 , mycoplasma negative ) , Hep-3 ( hepatocellular carcinoma , ATCC; ATCC# HB-8064 , mycoplasma negative ) and Hs578T ( breast carcinoma , ATCC; ATCC# HTB-126 , mycoplasma negative ) cells were maintained in DMEM ( Gibco ) containing 10% fetal bovine serum ( Gibco ) . All cells were incubated at 37\\u00b0C in 5% CO2 . In preparation for imaging , 96-well optical glass-bottomed plates ( Zell-Kontakt , N\\u00f6rten-Hardenberg , Germany ) were coated with ECM ligands , including either collagen type 1 ( Life Technologies \\u2013\\u00a0Thermo Fisher Scientific ) , laminin ( Sigma-Aldrich , St . Louis , MO ) , fibronectin or vitronectin ( both purified from human plasma as described previously [Smilenov et al . , 1992; Yatohgo et al . , 1988] ) . Coating was performed for 2 hr at 37\\u00b0C followed by blocking with 1% heat-denatured bovine serum albumin ( Sigma-Aldrich ) for 1 hr at 37\\u00b0C . ECM ligand coating concentrations were 10 \\u00b5g/ml except where otherwise indicated ( i . e . where fibronectin concentration was varied ) . RNAi-depletion of talin 1 was performed 48 hr prior to imaging or immunoblotting ( with anti-talin [8d4 , 1:500 , Sigma-Aldrich] and anti-tubulin [DM1A , 1:2000 , Fisher Scientific] antibodies as described previously [Kiss et al . , 2015] ) using the following oligonucleotide sequence: ( 5\\u2019 -GAA GAU GGU UGG CGG CAUU- 3\\u2019 ) ( synthesized by GenePharma , Shanghai , P . R . China ) . A non-targeting oligonucleotide sequence was used as control: ( 5\\u2019 -GCG CGC UUU GUA GGA UUCG- 3\\u2019 ) . Transfection of 2 x 104 cells was performed in 24-well plates using 20 pmol of siRNA and 2 \\u00b5l RNAiMAX ( Invitrogen \\u2013 Thermo Fisher Scientific ) according to the manufacturers instructions . Inhibition of Rho kinase ( ROCK ) was performed starting 1 hr prior to imaging using the Y-27632 inhibitor ( Sigma-Aldrich ) at 1 \\u00b5M , 2 \\u00b5M , or 6 \\u00b5M . Di-methyl sulfoxide ( DMSO , Sigma-Aldrich ) was used as control . Live-cell imaging was initiated 2 hr after replating of 2 x 104 cells per well into ECM-coated 96-well glass-bottomed plates ( see coating details above ) . High-resolution , multiscale imaging ( of CMACs , F-actin and cell migration ) was performed on a Nikon A1R confocal microscope using a PlanApo VC 60X/1 . 4 NA oil-immersion objective ( Nikon , Amsterdam , Netherlands ) . Images were acquired for 8\\u201310 hr at 5 min intervals with a pixel resolution of 0 . 21 \\u00b5m . Long-term ( 16 hr at 5-min intervals ) , low resolution imaging to monitor for inter-modal transitions was performed on a Nikon A1R confocal microscope using a Plan Apo \\u03b3 20X/0 . 75 NA air objective and resonance scanner , producing a pixel resolution of 0 . 82 \\u00b5m . 3\\u00d73 or 4\\u00d74 image montages were acquired and stitched . Prior to imaging , cells were labeled at 1:5000 with a far-red membrane dye ( Cell Mask Deep Red , Thermo Fisher Scientific ) for 2 hr and throughout imaging . Normal culture medium minus fetal bovine serum was used during all imaging , and cells were maintained at 37\\u00b0C in 5% CO2 . Multiscale image sequences ( Nikon ND2 files ) were analyzed using PAD software ( Digital Cell Imaging Laboratories , Keerbergen , Belgium ) , as described previously ( Lock et al . , 2014 ) . Briefly , cell boundaries were detected using the RubyRed-LifeAct signal . Cells touching the image border in any given time frame were excluded . Within each segmented cell , CMACs greater than 0 . 05 \\u00b5m2 in area were segmented based on EGFP-Paxillin signal . Segmented cells and CMACs were then tracked based on nearest neighbor analysis . Cell trajectories were smoothed using smoothing splines . CMACs were excluded where tracking did not contain at least three consecutive time points . CMACs present in the first and last frames of each image sequence were excluded from the calculation of CMAC Lifetime . All segmentation and tracking was manually inspected and iteratively optimized through parameter tuning . Quantitative features characterizing cell , CMAC , and CMAC-associated F-actin properties were then extracted ( Figure 3\\u2014figure supplement 1 ) . CMAC intensity values per channel were corrected by subtraction of mean intensity values in a 1 \\u00b5m radius around the CMAC , excluding other segmented CMACs . CMAC intensity data were further standardized per experimental repeat relative to the median value of CMAC intensities in the size range between 0 . 15 and 0 . 2 \\u00b5m2 within the low FN no INH condition . Cell motion was characterized by instantaneous cell speed or via analysis of mean squared displacement ( MSD ) . Mean square displacement ( MSD ) was calculated using a moving window as previously described ( Khorshidi et al . , 2011 ) . The window size was set to 24 frames ( 2 hr ) . The first 12 time lags ( tlag ) , up to 1 hr , of the MSD were fitted to Furth\\u2019s formula: MSD=4M ( tlag\\u2212tp ( 1\\u2212e\\u2212tlag/tp ) ) , thus measuring the migration coefficient M and the persistence time tp at each time point . The observations were divided into quintiles of migration coefficient and within each such quintile the mean persistence time for each trajectory was calculated . For visualization of parallel mode emergence ( Figure 1\\u2014figure supplement 1 and Video 3 ) and mode transitions ( Figure 1\\u2014figure supplement 2 and Videos 4 , 5 ) , long-term low-resolution image data was segmented and tracked ( following advanced denoising , power 15 ) using a custom general analysis pipeline in NIS-Elements ( v4 . 30 , Nikon ) . Collectively , data sets for comparison of fibronectin concentration , talin 1 expression and ROCK inhibition had structures as follows: 2 . 5 \\u00b5g/ml FN ( 3 experimental repeats , 34 cells , 2525 cell observations , 29651 CMAC observations ) vs 10 \\u00b5g/ml FN ( 19 experimental repeats , 118 cells , 6528 cell observations , 213406 CMAC observations ) ; talin 1 siRNA ( 4 experimental repeats , 101 cells , 6263 cell observations , 81675 CMAC observations ) vs control siRNA ( 4 experimental repeats , 51 cells , 3154 cell observations , 131679 CMAC observations ) ; High-resolution , multiscale imaging of 6\\u00a0\\u00b5M Y27632 ( ROCK inhibitor , 4 experimental repeats , 17 cells , 985 cell observations , 44647 CMAC observations ) vs DMSO control ( 10 experimental repeats , 34 cells , 2666 cell observations , 86794 CMAC observations ) ; long-term , low resolution imaging of 6\\u00b5M Y27632 ( 4 experimental repeats , 73 cells , 4478 cell observations ) vs 2 \\u00b5M Y27632 ( 4 experimental repeats , 71 cells , 5198 cell observations ) vs 1 \\u00b5M Y27632 ( 4 experimental repeats , 69 cells , 4885 cell observations ) vs DMSO control ( 4 experimental repeats , 83 cells , 5968 cell observations ) . Note that the term 'experimental repeats' refers to independent biological repeats , not technical replicates . Data sample sizes were selected based on prior experience from related multiscale analyses in several previous studies ( Lock et al . , 2014 ) . Samples were collected and analyzed without iteration , that\\u00a0is , all relevant available data were used and sample sizes were not altered to increase or decrease statistical power . Cells were classified into Discontinuous or Continuous migration modes according to the behavioral criteria described in Figure 1 based on visual inspection and optimization between two individuals , using a custom platform established in Matlab ( vR2013b , The Mathworks , Natick , MA ) . Single cells were assessed at each time point within each image sequence ( inter-mode transitions , though infrequent , were assessed ) . All data presented herein were assessed concurrently and were both computationally blinded and randomized with respect to experimental conditions . Dead cells and cells in sustained contact were classified as null and excluded from subsequent analyses . Membrane protrusion and retraction dynamics were quantified as described previously ( Kowalewski et al . , 2015 ) . Briefly , consecutive segmented images were compared to define protrusions ( new pixels in cell segment compared to previous frame ) and retractions ( pixels lost from cell segment compared to previous frame ) . To explore dynamics over different time frames , images separated by different intervals were assessed using the same criteria , establishing time window samplings of between 5 min ( 1 frame ) and 75 min ( 15 frames ) . Subtraction of protrusion and retraction size probability distributions was performed as described previously for other feature probability distributions ( Kiss et al . , 2015 ) . Analysis of cross-correlation between protrusion and retraction areas over time was performed on a per cell sequence basis using the Matlab function xcov . Missing values were replaced by the mean value of protrusion/retraction area for that cell trajectory . Time lags of between \\u221260 min ( \\u221212 frames ) and 60 min ( +12 frames ) were assessed , with negative lags indicating that protrusion dynamics lead retraction dynamics , and positive lags indicating that retraction dynamics lead protrusion dynamics . To summarize the mean absolute cross-correlation per migration mode , cross-correlation values at all lags ( \\u221260 to 60 min; \\u221212 to 12 frames ) were averaged for each time window ( 5 min to 75 min; 1 to 15 frames ) , per cell , and then averaged per condition . Nienty-five\\u00a0percent confidence intervals were calculated per time window size . Friedman testing based on distinct cell numbers defined statistically discernable differences in mean protrusion-retraction cross-correlation between modes across the sampled time windows . All membrane dynamics analyses were performed in Matlab ( vR2013b , The Mathworks ) . Supervised multivariate clustering via canonical vectors analysis ( CVA ) was performed based on decompositions of the between-group and within-group covariance matrices following eigenvalue decomposition , as previously described ( Lock et al . , 2014 ) . Feature selection via CVA was performed by ranking absolute feature weighting coefficients from the first canonical vector ( CV1 ) . Unsupervised multivariate clustering using principal component analysis ( PCA ) was performed using singular value decomposition of the normalized data matrix . Probability density surfaces were calculated using a Gaussian kernel density estimation of significantly smaller scale that the overall mode distributions , and hence does not dictate the 'valley' shape of these distributions \\u2013 rather this is a reflection of the true value distribution . Spearman\\u2019s rank correlation coefficients were calculated based on feature values per cell observation . Estimation of the expected distribution of random absolute value changes in Spearman\\u2019s correlation between modes was performed by permuting the identity of correlation values in one mode while holding the other constant . This was repeated 100 times and the cumulative distribution function of all differences is displayed . Pairwise testing for differences in feature distributions ( boxplots , parallel coordinates ) was performed using the Wilcoxon rank sum test ( equivalent to Mann-Whitney test ) given all cell observations . Analyses and visualizations described above were performed in Matlab ( The Mathworks ) . Parallel coordinates-based visualization of selected feature values in each migration mode was generated using Knime ( v2 . 12 . 00 , KNIME . com , Zurich , Switzerland ) . Parallel coordinates-based comparison of Spearman\\u2019s correlation coefficients between Cell Speed and selected features , as well as visualization of coefficient differences between modes , were generated using R ( v3 . 2 . 0 , R Foundation for Statistical Computing ) and RStudio ( v0 . 98 . 945 , RStudio , Boston , MA ) .\",\n          \"E . coli 70S ribosome purification ( Travin et al . , 2019 ) and tRNA synthesis , purification , and charging ( Ad et al . , 2019 ) were performed as previously described . Briefly , 70S ribosomes were purified from E . coli MRE600 cells using sucrose gradients to isolate 30S and 50S ribosomal subunits , followed by subunit reassociation and a second round of sucrose gradient purification . Transfer RNAs were transcribed from PCR DNA templates using T7 RNA polymerase and purified by phenol-chloroform extraction , ethanol precipitation , and column desalting . Flexizyme ribozymes were used to charge the P-site tRNAfMet with either pentafluorobenzoic acid or malonate methyl ester and the A-site tRNAVal with valine ( Goto et al . , 2011 ) . Ribosome-mRNA-tRNA complexes were formed non-enzymatically by incubating 10 \\u00b5M P-site tRNA , 10 \\u00b5M mRNA , and 100 \\u00b5M paromomycin with 1 \\u00b5M ribosomes for 15 min at 37\\u00b0C in buffer AC ( 20 mM Tris pH 7 . 5 , 100 mM NH4Cl , 15 MgCl2 , 0 . 5 mM EDTA , 2 mM DTT , 2 mM spermidine , 0 . 05 mM spermine ) . Then , 10 \\u00b5M A-site tRNA was added and the sample was incubated for an additional 15 min at 37\\u00b0C . Complexes were held at 4\\u00b0C and diluted to 100 nM ribosome concentration in the same buffer immediately before grid preparation . The mRNA of sequence 5\\u2019-GUAUAAGGAGGUAAAAAUGGUAUAACUA-3\\u2019 was chemically synthesized ( IDT ) and was resuspended in water without further purification . The Shine-Dalgarno sequence is shown in bold , and the Met-Val codons are in italics . 300 mesh R1 . 2/1 . 3 UltraAuFoil grids from Quantifoil with an additional amorphous carbon support layer were glow discharged in a Pelco sputter coater . About\\u00a04 \\u00b5L of each sample was deposited onto grids and incubated for 1 min , then washed in buffer AC with 20 mM NH4Cl rather than 100 mM NH4Cl . Grids were plunge-frozen in liquid ethane using a Vitrobot Mark IV with settings: 4\\u00b0C , 100% humidity , blot force 6 , blot time 3 . Movies were collected on a 300-kV Titan Krios microscope with a GIF energy filter and Gatan K3 camera . Super-resolution pixel size was 0 . 355 \\u00c5 , for a physical pixel size of 0 . 71 \\u00c5 . SerialEM ( Schorb et al . , 2019 ) was used to correct astigmatism , perform coma-free alignment , and automate data collection . Movies were collected with the defocus range \\u22120 . 6 to \\u22121 . 5 \\u00b5m and the total dose was 39 . 89 e-/\\u00c52 split over 40 frames . One movie was collected for each hole , with image shift used to collect a series of 3 \\u00d7 3 holes for faster data collection ( Cheng et al . , 2018 ) , and stage shift used to move to the center hole . Based on the 1 . 2/1 . 3 grid hole specification , this should correspond to a maximum image shift of\\u00a0~1 . 8 \\u03bcm , although the true image shift used was not measured . The beam size was chosen such that its diameter was slightly larger than that of the hole , that\\u00a0is , >1 . 2 \\u03bcm , although we have observed variation in the actual hole size compared to the manufacturer specifications . Datasets of 70S ribosome complexes with the two differently charged P-site tRNAs were initially processed separately . Movies were motion-corrected with dose weighting and binned to the recorded physical pixel size ( 0 . 71 \\u00c5 ) within RELION 3 . 0 ( Scheres , 2012 ) using MotionCor2 ( Zheng et al . , 2017 ) . CTF estimation was done with CTFFind4 ( Rohou and Grigorieff , 2015 ) , and micrographs with poor CTF fit as determined by visual inspection were rejected . Particles were auto-picked with RELION\\u2019s Laplacian-of-Gaussian method . The\\u00a02D classification\\u00a0of\\u00a0particles was performed in RELION , and 4\\u00d7 binned particles were used for all classification steps . Particles were separated into 3D classes in cryoSPARC heterogeneous refinement ( Punjani et al . , 2017 ) , using an initial model generated from PDB 1VY4 with A-site and P-site tRNAs ( Polikanov et al . , 2014 ) low-pass filtered to the default 20 \\u00c5\\u00a0resolution , and keeping particles that were sorted into well-resolved 70S ribosome classes . Particles were migrated back to RELION to generate an initial 3D-refined volume ( reference low-pass filtered to 60 \\u00c5 ) on which to perform masked 3D classification without alignment to further sort particles based on A-site tRNA occupancy . CTF Refinement and Bayesian polishing were performed in RELION 3 . 1 before pooling the two datasets together , with nine optics groups defined based on the 3 \\u00d7 3 groups for image shift-based data collection . The resulting 70S ribosome reconstruction was used as input for focused refinements of the 50S and 30S subunits . We used rigid-body docked coordinates for the 70S ribosome , individual ribosomal subunits or domains ( 30S subunit head , 50S subunit central\\u00a0protuberance ) to define the boundaries of the map regions to be used in the focused refinements . Focused refinement of the central\\u00a0protuberance was performed starting from the 50S subunit-focused refinement reconstruction , and head- and platform-focused refinements started from the 30S subunit focused refinement reconstruction . Ewald sphere correction , as implemented in RELION 3 . 1 with the single side-band correction ( Russo and Henderson , 2018; Zivanov et al . , 2018 ) , provided some additional improvements in resolution ( Tables 1\\u20132 ) . In addition to using the 40-frame movies , we used the first three frames corresponding to a\\u00a0~3 electron/\\u00c52 dose to calculate 3D reconstructions , including focused refinements of the 30S and 50S subunits . The focused-refined map of the 30S subunit had a resolution of 2 . 45 \\u00c5 by the map-to-model FSC metric . These maps were used to examine the density for the isoAsp in uS11 , which lacked clear density for the side chain in maps reconstructed from the full 40-frame movies . We also used the maps from the initial three frames to examine connectivity in ribose density , to determine if there is visual evidence for the impact of electron damage . The previous high-resolution structure of the E . coli 70S ribosome ( Noeske et al . , 2015 ) was used as a starting model . We used the \\u2018Fit to Map\\u2019 function in Chimera ( Pettersen et al . , 2004 ) to calibrate the magnification of the cryo-EM map of the 50S ribosomal subunit generated here to maximize correlation , resulting in a pixel size of 0 . 7118 \\u00c5 rather than the recorded 0 . 71 \\u00c5 . Focused-refined maps were transformed into the frame of reference of the 70S ribosome for modeling and refinement , using the \\u2018Fit to Map\\u2019 function in Chimera , and resampling the maps on the 70S ribosome grid . The 50S and 30S subunits were refined separately into their respective focused-refined maps using PHENIX real-space refinement ( RSR; Liebschner et al . , 2019 ) . Protein and rRNA chains were visually inspected in Coot ( Casa\\u00f1al et al . , 2020 ) and manually adjusted where residues did not fit well into the density , making use of B-factor blurred maps where needed to interpret regions of lower resolution . Focused-refined maps on smaller regions were used to make further manual adjustments to the model , alternating with PHENIX RSR . Some parts of the 50S subunit , including H69 , H34 , and the tip of the A-site finger , were modeled based on the 30S subunit focused-refined map . The A-site and P-site tRNAs were modeled as follows: anticodon stem-loops , 30S subunit focused-refined map; P-site tRNA body , 50S subunit focused-refined map , with a B\\u00a0factor of 20 \\u00c52 applied; A-site tRNA body , 30S subunit focused-refined map and 50S subunit focused-refined map with B\\u00a0factors of 20 \\u00c52 applied; tRNA-ACCA 3\\u2019 ends , 50S subunit focused-refined map with B\\u00a0factors of 20\\u201330 \\u00c52 applied . Alignments of uS15 were generated using BLAST ( Altschul et al . , 1997 ) with the E . coli sequence as reference . The model for bL31A ( E . coli gene rpmE ) was manually built into the CP and 30S subunit head domain focused-refined maps before refinement in PHENIX . A model for paromomycin was manually docked into the 30S subunit focused-refined map , followed by real-space refinement in Coot and PHENIX . Comparisons to prior paromomycin structural models ( PDB codes 1J7T , 2VQE , and 4V51; Kurata et al . , 2008; Selmer et al . , 2006; Vicens and Westhof , 2001 ) used least-squares superposition of paromomycin in Coot . Although ring IV is in different conformations in the various paromomycin models , the least-squares superposition is dominated by rings I\\u2013III , which are in nearly identical conformations across models . Ribosome solvation including water molecules , magnesium\\u00a0ions , and polyamines was modeled using a combination of PHENIX ( phenix . douse ) and manual inspection . The phenix . douse feature was run separately on individual focused-refined maps , and the resulting solvent models were combined into the final 30S and 50S subunit models . Due to the fact that the solvent conditions used here contained ammonium ions and no potassium , no effort was made to systematically identify monovalent ion positions . The numbers of various solvent molecules are given in . Along with the individual maps used for model building and refinement , we have also generated a composite map of the 70S ribosome from the focused-refined maps for deposition to the PDB and EMDB for ease of use ( however , experimental maps are recommended for the\\u00a0examination of high-resolution features ) . We made the composite map using the \\u2018Fit in Map\\u2019 and vop commands in Chimera . First , we aligned the unmasked focus-refined maps with the 70S ribosome map using the \\u2018Fit in Map\\u2019 tool . We then used the \\u2018vop resample\\u2019 command to transform these aligned maps to the 70S ribosome grid . After the resampling step , we recorded the map standard deviations as reported in the \\u2018Volume Mean , SD , RMS\\u2019 tool . Then , we added the maps sequentially using \\u2018vop add\\u2019 followed by rescaling the intermediate maps to the starting standard deviation using the \\u2018vop scale\\u2019 command . Initial real-space refinement of the 30S subunit against the focused-refined map using PHENIX resulted in a single chiral volume inversion involving the backbone of N119 in ribosomal protein uS11 , indicating that the L-amino acid was being forced into a D-amino acid chirality , as reported by phenix . real_space_refine . Of the 10 , 564 chiral centers in the 30S subunit model , the C\\u03b1 of N119 had an energy residual nearly two orders of magnitude larger than the next highest deviation . Inspection of the map in this region revealed clear placement for carbonyl oxygens in the backbone , and extra density consistent with an inserted methylene group , as expected for isoAsp . The model of isoAsp at this position was refined into the cryo-EM map using PHENIX RSR , which resolved the stereochemical problem with the C\\u03b1 chiral center . IsoAsp was also built and refined into models of archaeal and eukaryotic uS11 based on cryo-EM maps of an archaeal 30S ribosomal subunit complex ( PDB 6TMF; N\\u00fcrenberg-Goloub et al . , 2020 ) and a yeast 80S ribosome complex ( PDB 6T4Q; Tesina et al . , 2020 ) . These models were refined using PHENIX RSR , and real-space correlations by residue calculated using phenix . model_map_cc . All archaeal genomes were downloaded from the NCBI genome database ( 2618 archaeal genomes , last accessed September 2018 ) . Due to the high number of bacterial genomes available in the NCBI genome database , only one bacterial genome per genus ( 2552 bacterial genomes ) was randomly chosen based on the taxonomy provided by the NCBI ( last accessed in December 2017 ) . The eukaryotic dataset comprises nuclear , mitochondrial , and chloroplast genomes of 10 organisms ( Homo sapiens , Drosophila melanogaster , Saccharomyces cerevisiae , Acanthamoeba castellanii , Arabidopsis thaliana , Chlamydomonas reinhardtii , Phaeodactylum tricornutum , Emiliania huxleyi , Paramecium aurelia , and Naegleria gruberi ) . Genome completeness and contamination were estimated based on the presence of single-copy genes ( SCGs ) as described in Anantharaman et al . , 2016 . Only genomes with completeness\\u00a0>70% and contamination\\u00a0<10% ( based on duplicated copies of the SCGs ) were kept and were further de-replicated using dRep at 95% average nucleotide identity ( version v2 . 0 . 5; Olm et al . , 2017 ) . The most complete genome per cluster was used in downstream analyses . Ribosomal uS11 genes were detected based on matches to the uS11 Pfam domain ( PF00411; Punta et al . , 2012 ) using hmmsearch with an E-value below 0 . 001 ( Eddy , 1998 ) . Amino acid sequences were aligned using the MAFFT software ( version v7 . 453; Katoh and Standley , 2016 ) . The\\u00a0alignment was further trimmed using Trimal ( version 1 . 4 . 22; --gappyout option; Capella-Guti\\u00e9rrez et al . , 2009 ) . Tree reconstruction was performed using IQ-TREE ( version 1 . 6 . 12; Nguyen et al . , 2015 ) , using ModelFinder ( Kalyaanamoorthy et al . , 2017 ) to select the best model of evolution , and with 1000 ultrafast bootstrap ( Hoang et al . , 2018 ) . The tree was visualized with iTol ( version 4; Letunic and Bork , 2019 ) and logos were made using the weblogo server ( Crooks et al . , 2004 ) . 16S and 18S rRNA genes were identified from the prokaryotic and eukaryotic genomes using the method based on hidden Markov model ( HMM ) searches using the cmsearch program from the Infernal package ( Nawrocki et al . , 2009 ) and fully described in Brown et al . , 2015 . The sequences were aligned using the MAFFT software . S21 sequences were retrieved from the huge phage database described in Al-Shayeb et al . , 2020 . Cd-hit was run on the set of S21 sequences to reduce the redundancies ( Fu et al . , 2012; default parameters; version 4 . 8 . 1 ) . Non redundant sequences were used as a query against the database of prokaryotic genomes used for uS11 above using BLASTP ( version 2 . 10 . 0+; e-value 1e-20; Altschul et al . , 1997 ) . Alignment and tree reconstruction were performed as described for uS11 except that we did not perform the alignment trimming step . Similarly to uS11 , the YcaO sequences were identified in prokaryotic genomes based on its PFAM accession ( PF02624; Punta et al . , 2012 ) using hmmsearch with an E-value below 0 . 001 ( Eddy , 1998 ) . Amino acid sequences were aligned using the MAFFT software ( version v7 . 453; Katoh and Standley , 2016 ) . Alignment was further trimmed using Trimal ( version 1 . 4 . 22; --gappyout option; Capella-Guti\\u00e9rrez et al . , 2009 ) . Tree reconstruction was performed using IQ-TREE ( version 1 . 6 . 12; Nguyen et al . , 2015 ) , using ModelFinder ( Kalyaanamoorthy et al . , 2017 ) to select the best model of evolution , and with 1000 ultrafast bootstraps ( Hoang et al . , 2018 ) . The tree was visualized with iTol ( version 4; Letunic and Bork , 2019 ) . The three genes downstream and upstream of each YcaO gene were identified and annotated using the PFAM ( Punta et al . , 2012 ) and the Kegg ( Kanehisa et al . , 2016 ) databases . Masks for each map were generated in two ways . First , to calculate the\\u00a0map-to-model FSC curves for comparisons of the present models with the cryo-EM maps generated here , we used masked maps generated by RELION during postprocessing ( Zivanov et al . , 2018 ) . The effective global resolution of a given map is given at the FSC cutoff of 0 . 5 in Table 2 and Figure 1\\u2014figure supplements 2\\u20133 . Second , we used refined PDB coordinates for the 70S ribosome , individual ribosomal subunits or domains ( 30S subunit head , 50S subunit central\\u00a0protuberance ) for comparisons to the 70S ribosome map or focused-refined maps , and to previously published maps and structural models . Masks for each map were generated in Chimera ( Pettersen et al . , 2004 ) using the relevant PDB coordinates as follows . A 10 \\u00c5\\u00a0resolution map from the coordinates was calculated using molmap , and the surface defined at one standard deviation was used to mask the high-resolution map . For the present models and maps , the effective global resolution of a given map using this second approach was similar or slightly lower than that using the approach in RELION ( within a few hundredths of an \\u00c5 ) . For the recent E . coli 50S subunit structure ( Stojkovi\\u0107 et al . , 2020 ) , we used Chimera to resize the deposited map ( emd_20353 ) to match the dimensions of the maps presented here . Briefly , our atomic coordinates for the 50S subunit were used with the \\u2018Fit to Map\\u2019 function and the voxel size of the deposited map was calibrated to maximize correlation . The resulting voxel size changed from 0 . 822 \\u00c5 to 0 . 8275 \\u00c5 in linear dimension . After rescaling the deposited map , we used phenix . model_map_cc to compare the map with rescaled atomic coordinates deposited in the PDB ( 6PJ6 ) or to the present 50S model , yielding a map-to-model FSC of 0 . 5 at\\u00a0~2 . 5 \\u00c5 . Similar comparisons of the structure of the Staphylococcus aureus 50S subunit to the deposited map ( PDB 6S0Z , emd_10077; Halfon et al . , 2019 ) yielded a map-to-model FSC of 0 . 5 at 2 . 43 \\u00c5 , accounting for a change in voxel linear dimension from 1 . 067 \\u00c5 to 1 . 052 \\u00c5 . For comparisons to the map and model deposited by Pichkur et al . , 2020 ( EMD-10655 and PDB 6XZ7 ) , we removed tRNA bodies , the L1 arm , and the GTPase-associated-center coordinates from 6XZ7 since these regions are disordered or missing in the deposited 50S subunit reconstruction . Previously published E . coli tryptic peptide mass spectrometry ( MS/MS ) raw data was used for the analysis ( Dai et al . , 2017; MassIVE accession: MSV000081144 ) . Peptide searches were performed with MSFragger ( Kong et al . , 2017 ) using the default parameters for a closed search with the following exceptions: additional variable modifications were specified on residues R ( hydroxylation , \\u0394 mass: 15 . 9949 ) and M ( thioamide , \\u0394 mass: 15 . 9772 ) , maximum modifications per peptide set to four , and multiple modifications on a residue were allowed . Spectra were searched against a database of all E . coli proteins plus common contaminants concatenated to a decoy database with all original sequences reversed . Results were analyzed using TPP ( Deutsch et al . , 2015 ) and Skyline ( Pino et al . , 2020 ) . Cryo-EM maps were supersampled in Coot for smoothness . Figure panels showing structural models were prepared using Pymol ( Schr\\u00f6dinger ) and ChimeraX ( Goddard et al . , 2018 ) . Sequence logo figures were\\u00a0made with WebLogo 3 . 7 . 4 ( Crooks et al . , 2004 ) . Phylogenetic trees were visualized with iTol ( version 4; Letunic and Bork , 2019 ) and multiple alignments were visualized with geneious 9 . 0 . 5 ( https://www . geneious . com ) . Ribosome coordinates have been deposited in the Protein Data Bank ( entry 7K00 ) , maps in the EM Database ( entries EMD-22586 , EMD-22607 , EMD-22614 , EMD-22632 , EMD-22635 , EMD-22636 , and EMD-22637 for the 70S ribosome composite map , 70S ribosome , 50S subunit , 30S subunit , 30S subunit head , 30S subunit platform , and 50S subunit CP maps , respectively ) , and raw movies in EMPIAR ( entry EMPIAR-10509 ) .\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"Methods\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"num_unique_values\": 1,\n        \"samples\": [\n          \"\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"Introduction\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 10,\n        \"samples\": [\n          \"Type I interferon ( IFN ) plays a dichotomous role in chronic viral infections such as Human Immunodeficiency Virus-1 ( HIV ) , contributing to the control of viral replication during the earliest stages of infection , yet fueling disease progression by activating target cells for infection , decreasing antiviral gene expression , enabling infection with increased reservoir size , and accelerating CD4 T-cell loss [1\\u20138] . Plasmacytoid dendritic cells ( pDC ) are thought to play a significant role in IFN responses during HIV infection , arriving rapidly at sites of mucosal transmission [4] and relocating from blood to lymphoid tissues where they produce pro-apoptotic and pro-inflammatory IFN [9\\u201311] . Cellular mechanisms underlying HIV-stimulated IFN production by pDC are only partially understood . We have previously shown that abundant IFN is produced by pDC upon HIV stimulation through endosomal recognition of genomic RNA by TLR7 . This response requires the presence of HIV envelope protein on viral particles , interactions between CD4 and the viral envelope protein , HIV endocytosis and endosomal acidification; however , co-receptor usage , viral fusion and viral replication are not required [12 , 13] . Cell-to-cell infection seems to amplify pDC responses to HIV , however precise mechanisms underlying differences between cell-free and cell-to-cell pDC activation are not clearly defined [14] . We and others have shown that pDC are highly resistant to HIV infection , and this block to replication is IFN-independent [15 , 16] . In addition to IFN production , pDC can act as antigen-presenting cells ( APC ) to activate T-cell\\u2013mediated adaptive immune responses [17\\u201321] . Acquisition of an APC phenotype requires specific signals that are distinct from the signals that induce large amounts of IFN . We have previously shown that HIV stimulated pDC express low levels of the co-stimulatory molecule CD86 and express Indoleamine 2 , 3-dioxygenase ( IDO ) , a potent inducer of regulatory T cells , indicating that they do not differentiate into mature APC and fail to stimulate potent T cell responses [22 , 23] . However , pDC can differentiate into APC with influenza virus or the synthetic TLR7 agonist R837 and are able to cross-present antigens from HIV-1-infected apoptotic cells to HIV-specific CD8+ T lymphocytes , demonstrating that pDC do not have an intrinsic defect in presentation of HIV antigens , but rather that sensing of HIV does not provide the signals that are required for efficient differentiation of pDC into APC [17] . pDC sense single stranded RNA or unmethylated DNA containing Cytosine\\u2013Guanosine dinucleotides ( CpG ) through Toll-like receptors ( TLR ) 7 and 9 , respectively , located in endosomal compartments . Both TLR7 and TLR9 signal through the adapter protein myeloid differentiation primary response gene 88 ( MyD88 ) . Downstream IFN signaling occurs in response to activation of IFN genes through phosphorylation of interferon regulatory factor 7 ( IRF7 ) , whereas downstream signaling of nuclear factor kappa-light-chain-enhancer of activated B cells ( NF-\\u03baB ) leads to the transcriptional activation of proinflammatory kinases and upregulation of MHC and co-stimulatory molecules necessary for maturation into APC . [12 , 24] . The functional response of pDC to pathogens is flexible . As posited by the spatiotemporal model of pDC sensing [25] , differential pDC activation is likely related to the subcellular location where the TLR senses the pathogen . Thus , engagement of TLR 7/9 in the early endosomes of pDC preferentially triggers the IRF7 signal cascade , leading to type I IFN responses; whereas engagement of TLR7/9 in lysosomes preferentially triggers the NF-\\u03baB signal cascade , leading to the production of proinflammatory cytokines TNF\\u03b1 and IL6 , upregulation of co-stimulatory molecules , and an APC phenotype [25 , 26] . Differential trafficking and therefore sensing of synthetic TLR9-activating CpGs is attributed to sequence-related secondary and tertiary structural features of the CpGs . CpGs which contain phosphodiester backbones and palindromic motifs ( CpGA ) form multimeric complexes and traffic to early endosomes for IRF7 signaling whereas CpGs which contain phosphorothioate backbones and lack palindromic motifs ( CpGB ) traffic as monomers to lysosomes for NF-\\u03baB signaling . Intermediate CpGs ( CpGC ) combine structural elements of both CpGA and CpGB , traffic to both compartments , and stimulate both IRF7 and NF-\\u03baB signaling [27\\u201329] . While the spatiotemporal model of pDC sensing has been most clearly evaluated using synthetic TLR9 agonists ( CpG ) , we have shown that the model also applies to HIV and TLR7 , whereby HIV traffics to early endosomes in pDC , activating IRF7 signaling rather than NF-\\u03baB signaling [22 , 23] . The upstream events that determine activation of each of these pathways , and in particular , HIV virion trafficking in pDC , are currently unknown , however , prior studies suggest that HIV envelope may play a major role [13 , 30 , 31] . Here we demonstrate that HIV trafficking and pDC phenotype is predominantly determined by envelope-CD4 interaction , such that manipulation of HIV envelope or CD4 intracellular trafficking enables modulation of divergent sensing of HIV .\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"Materials and Methods\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 6,\n        \"samples\": [\n          \"ES cells were generally maintained in NDiff N2B27 media ( Stem Cells , Inc . ; scs-sf-nb-02 ) in the presence of the GSK3 inhibitor CHIR99021 ( Stemgent , 04-0004; 3 \\u00b5M ) and MEK inhibitor PD0325901 ( Stemgent , 04-0006; 1 \\u00b5M ) ( \\u201c+2i\\u201d media ) and were routinely passaged using Accutase ( Sigma , A6964 ) every other day . For differentiation , the media containing inhibitor was removed and replaced with NDiff N2B27 media . Where indicated , ES cells were maintained in serum/LIF conditions in media containing knockout DMEM ( Invitrogen 10829-018 ) , 15% heat inactivated FBS ( Invitrogen 10082-147 ) , 2 mM of Glutamax-1 supplement ( Invitrogen 35050-038 ) , 1% non-essential amino acids ( Invitrogen 11140-035 ) , 50 \\u00b5M 2-mercaptoethanol ( Invitrogen 31350-010 ) and 5\\u00d7105 U of LIF ( Millipore 103 U/ml ) . The ES cells cultured under \\u201c+2i\\u201d conditions were adapted in serum/LIF culture conditions for at least 8 passages before the experiments were performed . Cells were stained for alkaline phosphatase expression using an alkaline phosphatase detection kit as described by the manufacturer's ( Millipore ) . For RNAi , 4\\u00d7104/cm2 cells ( ie 1 . 28\\u00d7104 cells/well of 96 well plate ) were plated out into a mixture of 0 . 3 \\u00b5l of RNAi Max ( Invitrogen ) and 100 nM siRNA in 100 \\u00b5l of \\u201c+2i\\u201d media for 24 hrs . All validation experiments used ON-TARGETplus siRNA SMART pools from Dharmacon . Real time RT-qPCR was carried out as described previously [48] . For assays in 96 well plate format , the same basic protocol was followed except the RNA was obtained using a Fastlane cell RT-PCR kit ( QIAGEN ) . Data were normalized for the average expression of the control genes gapdh , hmbs and tbp . The primer-pairs used for RT-PCR experiments are listed in Table S7 . Western blotting was carried out with the primary antibodies; Erk2 ( 137F5; Cell Signalling , 4695 ) , phospho-ERK ( E10; Cell Signalling , 9106 ) , Dusp1 ( MKP-1; Upstate , 07535 ) , Dusp6 ( MKP-3; Epitomics , 2138-1 ) and Pou5f1 ( Oct-3/4; Santa Cruz , sc-8628 ) . All experiments were carried out in 96-well plates . The lysates were directly harvested in the 2\\u00d7SDS sample buffer followed by sonication ( Bioruptor , Diagenode ) . The proteins were detected using infrared dye-conjugated secondary antibodies ( LI-COR Bioscience , IRDye 800CW [1 in 10 , 000] and IRDye 680LT [1 in 20 , 000] ) , and the signal was collected with a LI-COR Odyssey Infrared Imager and quantified using Odyssey software ( LI-COR Bioscience , Odyssey Infrared Imaging system application software version 3 . 0 . 25 ) . The Ras activities were examined using Ras activation ELISA assay kit ( Millipore ) as described in the manufacturers' instructions . The total lysates used in the ELISA assay was normalised with the quantity of the proteins assayed by the BCA protein assay kit ( ThermoScientific ) . Flow cytometric analysis was carried out using a LSRII flow cytometer and samples were loaded using HTS loader ( BD Biosciences ) . For sampling , media was removed from each well . Single cell suspensions were generated by treating cells with accutase at 37\\u00b0C for 7 mins followed by resuspendion in 0 . 03% BSA/PBS . Dead cells were stained by Sytox Red dead cell stain ( Invitrogen , 5 nM ) . The cells were analysed immediately after sampling . Each sample was analysed with 10 , 000 event counts with the flow rate at 1 \\u00b5l/s . The resulting GFP profile ( green channel ) was created by gating with the right ranges of cell sizes based on forward and reverse scatter plot ( ssc vs fsc; blue channel ) and dead cells were gated away based on the Sytox Red stain profile ( red [APC] channel ) . All liquid handling processes were performed using Biomek robotic system ( Beckman Coulter ) . For the primary screen , Rex1GFPd2 ES were grown in 96 well plates in the presence of \\u201c2i\\u201d and reverse transfection was performed using siGENOME siRNA pools ( Dharmacon; mouse protein kinase [G-013500] , GPCR [G-013600] , druggable [G-014600] and genome [G-015000] libraries ) . 24 hrs later , the \\u201c2i\\u201d media was removed and replaced with fresh NDiff N2 B27 media . After 28 hrs , the levels of GFP in the cells were determined by flow cytometry as described above . Each plate contained 8 control non-targeting siRNAs , and the positive control siRNAs against gsk3\\u03b2 and fgf4 . To take into account slight variations in the timing of pluripotency loss , the ratio of high GFP to low GFP expressing cells was established on each plate based on the non-targeting controls , allowing a threshold to be set as 1 ( ie 50% high GFP and 50% low GFP ) . This threshold was used to determine the ratio of high to low GFP expressing cells in the other wells . The mean plus/minus standard deviation ( SD ) was calculated for each plate , and individual wells were scored positive if they exceeded 2\\u00d7SD above or below this mean . The screen was performed in duplicate , with duplicate plates being analysed on different days . A final list of positive hits was determined by taking siRNAs which scored an average of 2\\u00d7SD across both plates ( or on a single plate where the duplicate well was defective in the case of 30 siRNAs ) , generally with both plates scoring >1 . 5\\u00d7SD above the mean . However , an additional small number of siRNAs were scored as positive where the average score was >2 . 5\\u00d7SD above the mean where only one plate had to score >1 . 5\\u00d7SD above the mean , and also for seven siRNAs where the average score was >1 . 9\\u00d7SD above the mean and both plates scored >1 . 9\\u00d7SD above the mean . For the validation screens , either Rex1GFPd2 or Oct4GFP ES cells were used and screens were performed as above except that ON-TARGETplus siRNA duplexes were used and GFP levels in Oct4GFP ES cells were determined 72 hrs after release from \\u201c2i\\u201d . Individual wells in each screen were scored as positive if the average GFP ( + ) /GFP ( \\u2212 ) ratio exceeded 1 . 25\\u00d7SD above the non-targeting controls across both duplicate plates . Additional hits were considered as positive if they scored >0 . 8\\u00d7SD above the mean in one validation screen and also scored >1 . 0\\u00d7SD above the mean in the other . For the \\u201c1i\\u201d screens , Rex1GFPd2 were used as for the validation screens but only one inhibitor ( ie either CHIR99021 or PD0325901 ) was withdrawn . Wells were scored as positive if the average GFP ( + ) /GFP ( \\u2212 ) ratio exceeded 1 . 5\\u00d7SD above the mean of the non-targeting controls . For constructing networks , lists of gene names were uploaded into STRING [49] with the confidence score set high ( 0 . 40 ) . The resulting networks were saved as * . txt files and then uploaded into Cytoscape ( v . 2 . 7 . 0 ) choosing coexpression , textmining , knowledge and experimental data as proximity criteria . yFiles\\u2192organic network layouts were applied and the positioning and graphic representation of nodes were adjusted manually for increased clarity . GO term analysis was carried out using DAVID Bioinformatics Resources 6 . 7 ( NIH ) [50] . The enriched terms from the functional annotation chart were extracted and manually clustered . Heat maps of GO terms were generated by MultiexperimentViewer ( MeV 4_7_4 ) . GO term summary and visualization was carried out by REVIGO [51] .\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"Abstract\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 10,\n        \"samples\": [\n          \"Plasmacytoid dendritic cells ( pDC ) are innate immune cells that sense viral nucleic acids through endosomal Toll-like receptor ( TLR ) 7/9 to produce type I interferon ( IFN ) and to differentiate into potent antigen presenting cells ( APC ) . Engagement of TLR7/9 in early endosomes appears to trigger the IRF7 pathway for IFN production whereas engagement in lysosomes seems to trigger the NF-\\u03baB pathway for maturation into APC . We showed previously that HIV-1 ( HIV ) localizes predominantly to early endosomes , not lysosomes , and mainly stimulate IRF7 rather than NF-\\u03baB signaling pathways in pDC . This divergent signaling may contribute to disease progression through production of pro-apoptotic and pro-inflammatory IFN and inadequate maturation of pDCs . We now demonstrate that HIV virions may be re-directed to lysosomes for NF-\\u03baB signaling by either pseudotyping HIV with influenza hemagglutinin envelope or modification of CD4 mediated-intracellular trafficking . These data suggest that HIV envelope-CD4 receptor interactions drive pDC activation toward an immature IFN producing phenotype rather than differentiation into a mature dendritic cell phenotype .\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"Results\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 8,\n        \"samples\": [\n          \"To enable the detection of discrete mesenchymal migration sub-modalities , and thereafter , the comprehensive analysis of cellular organization , regulation , and adaptation underlying each mode , we have integrated a unique combination of analytical tools . These are now made freely available as a Matlab Toolbox: 'The Cell Adhesion and Migration Analysis Toolbox' \\u2013 along with the raw quantitative data underlying this study , sample image data to aid implementation , and explanatory documentation ( see doi:10 . 5061/dryad . 9jh6m ) . The features included in this Toolbox are also specified briefly in 'Materials and methods' . H1299 ( non-small cell lung carcinoma ) cells stably expressing EGFP-paxillin ( marker for CMACs ) and RubyRed-LifeAct ( marker for F-actin ) ( H1299 P/L cells [Lock et al . , 2014] ) were imaged via confocal microscopy on glass coated with 2 . 5 \\u00b5g/ml fibronectin ( FN ) . H1299 P/L cells moved individually and exclusively via mesenchymal ( lamellipodial ) migration . Nonetheless , we observed and classified two qualitatively different migration modes emerging within the clonally derived H1299 P/L cell population . We termed these migration modes 'Discontinuous' and 'Continuous' , reflecting their contrasting stepwise and smooth motion , respectively ( Figure 1 ) . Specifically , cells migrating in the Discontinuous mode cycle in sequence through at least three recognizable stages of movement , which we term 'lateral protrusion' , 'polarization' , and 'tail retraction' , as described in Figure 1A ( also see Video 1 ) . These stages produce dramatic changes in cell morphology and frequent changes in migratory direction . Such directional changes occur at least in part because lateral protrusions typically develop at approximately 90\\u00b0 to the axis of the preceding tail retraction . Overall , the stepwise nature of Discontinuous migration is highly reminiscent of previous descriptions of fibroblast-like migration ( Theisen et al . , 2012; Abercrombie et al . , 1970; Chen , 1981 ) . By contrast , cells migrating in the Continuous mode move progressively , with less frequent changes in cell morphology and motile direction , in a manner analogous to classical keratocyte-like migration ( Figure 1B and Video 2 ) ( Keren et al . , 2008 ) . Importantly , these migration modes emerge spontaneously in parallel under uniform experimental conditions within a clonally\\u00a0derived cell line ( Figure 1\\u2014figure supplement 1 and Video 3 ) . It is noteworthy that cells may traverse the same underlying substrate regions while stably occupying distinct modes ( see Video 3 ) , indicating that local environmental inconsistencies are not the principal cause of differential mode identity . Yet , inter-modal conversion is possible , occurring bi-directionally between modes ( Figure 1\\u2014figure supplement 2 and Videos 4 and 5 ) , indicating their profound plasticity . 10 . 7554/eLife . 11384 . 003Figure 1 . Classification of 'Discontinuous' and 'Continuous' modes of mesenchymal ( lamellipodial ) migration . ( A , B ) Confocal image sequences of individual H1299 cells stably expressing EGFP-paxillin ( cell-matrix adhesion complex [CMAC] marker; upper image panels ) and RubyRed-LifeAct ( F-actin marker; lower image panels ) migrating in either Discontinuous ( A ) or Continuous ( B ) modalities on fibronectin ( FN ) -coated ( 2 . 5 \\u00b5g/ml ) glass . Images are displayed with gray-scale inverted . Numbers denote time ( min ) . Schematics in blue ( Discontinuous , A ) and orange ( Continuous , B ) depict typical cell morphology changes associated with each migration mode . Note stepwise cycles of: lateral protrusion ( in directions of dashed arrows ) ; cell polarization; and tail retraction ( in direction of solid arrows ) that recur during Discontinuous migration , and that lateral protrusion tends to occur at 90\\u00b0 to the preceding tail retraction . During Continuous migration , cell morphology is relatively stable , with many small protrusion ( open arrowheads ) and retraction ( closed arrowheads ) events producing smooth movement . Movies corresponding to cells shown in A and B are available in supporting material ( Videos 1 , 2 ) . ( C ) Quantification of frequencies of Discontinuous and Continuous modes . ( D ) Box plots of cell speed ( \\u00b5m/sec ) per migration mode . Boxplots show median values and inter-quartile ranges ( IQR , 25% to 75% ) . Notches indicate median +/\\u2212 1 . 57 * IQR/\\u221an ( approximates 95% confidence interval of the median , n = number of cell observations , see 'Materials and methods' ) . Statistical discernibility assessed by Wilcoxon rank sum test , p < 0 . 001 . ( E ) Cell trajectories in each mode were assessed via mean squared displacement ( MSD ) analysis and divided into quintiles ( 20% bins ) according to their migration cefficient ( related to speed of movement ) . Median values +/\\u22121 . 57 * IQR/\\u221an ( n = number of cell observations ) of a second MSD measure , persistence time ( related to migration direction stability ) , are plotted per migration coefficient quintile . Number of observations per quintile: Discontinuous = ~ 402; Continuous = ~ 95 . ( F ) A table summarizes results from visual inspection of several cell lines migrating on 2 . 5 \\u00b5g/ml FN , confirming the emergence of either one or both Discontinuous and Continuous migration modes in these cell types . Example image sequences are presented in Figure 1\\u2014figure supplement 3 . ( G ) A table summarizes results from visual inspection of H1299 cells migrating on several different extracellular matrix ligands , confirming the emergence of both migration modes under each condition . Example image sequences are presented in Figure 1\\u2014figure supplement 4 . Scale bars = 20 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00310 . 7554/eLife . 11384 . 004Figure 1\\u2014figure supplement 1 . Discontinuous and Continuous migration modes emerge in parallel under uniform conditions . A confocal image sequence showing the parallel emergence of Discontinuous and Continuous migration modes in H1299 P/L cells labeled with a far red membrane-dye on 5 \\u00b5g/ml fibronectin-coated glass . ( A ) De-noised ( left ) and segmented ( right ) images show a selected region from within a 4 x 4 image montage . Based on observed cell dynamics ( see Video 3 ) , cells were classified as in either the Discontinuous ( e . g . cell in blue box ) or Continuous mode ( e . g . cell in red box ) . ( B ) Stereotypical Discontinuous mode behaviors are displayed in a series of time points ( numbers denote minutes ) following the cell from the blue box in A . Specifically: Tail Retraction is denoted by solid arrows; Lateral Protrusion and Polarization are denoted by dashed double headed arrows; ( C ) The relatively stable morphology of Continuous mode migration is shown by the cell from the red box in A . It is noteworthy that both highlighted cells sequentially traverse the same region of the coated substrate while maintaining distinct migration modes ( see Video 3 ) , indicating that potential inconsistencies in the local environment could not be responsible for the parallel emergence of distinct modes . Scale bars = 10 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00410 . 7554/eLife . 11384 . 005Figure 1\\u2014figure supplement 2 . Discontinuous and Continuous migration modes can spontaneously inter-convert ( mode switching ) . Confocal image sequences show the spontaneous inter-conversion between Discontinuous and Continuous migration modes by H1299 P/L cells labeled with a far red membrane-dye on 5 \\u00b5g/ml fibronectin-coated glass . ( A ) De-noised ( upper ) and segmented ( lower ) time sequences ( numbers denote minutes ) of a single cell spontaneously transitioning ( switching ) from the Continuous migration mode to the Discontinuous migration mode ( see tracked cell in Video 4 ) . The approximate timing of the Mode Transition is indicated . ( B ) De-noised ( upper ) and segmented ( lower ) time sequences of a single cell spontaneously transitioning ( switching ) from the Discontinuous migration mode to the Continuous migration mode ( see tracked cell in Video 5 ) . The approximate timing of the Mode Transition is indicated . Scale bars = 10 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00510 . 7554/eLife . 11384 . 006Figure 1\\u2014figure supplement 3 . Discontinuous and/or Continuous migration modes recur in multiple cell types . Confocal image sequences exemplifying Discontinuous and/or Continuous migration on 2 . 5 \\u00b5g/ml fibronectin-coated glass by several cell types , including: BT549 ( breast epithelial [ductal carcinoma] ) ; MDA-MB-231 ( breast epithelial [adenocarcinoma] ) ; Hep-3 ( hepatocellular carcinoma ) ; and HS578T ( breast epithelial [carcinoma] ) . Cells were labeled with a membrane dye and imaged at 5-min intervals . Images are displayed with gray-scale inverted . Numbers denote time ( min ) . Movies corresponding to image sequences ( in the order shown ) are available in supporting material ( Videos 6\\u201311 , respectively ) . Scale bars = 20 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00610 . 7554/eLife . 11384 . 007Figure 1\\u2014figure supplement 4 . Discontinuous and Continuous migration modes recur in H1299 cells adhering to multiple extracellular matrix ligands . Confocal image sequences exemplifying H1299 cell migration in Discontinuous and Continuous modes on glass coated with various purified extracellular matrix ligands , including: fibronectin ( FN ) ; Laminin; Collagen Type 1; and Vitronectin . Cells were labeled with a membrane dye and imaged at 5-min intervals . Images are displayed with gray-scale inverted . Numbers denote time ( min ) . Schematics of sequences on FN detail the characteristic movement and morphology changes associated with each mode , as shown in Figure 1 . See movies corresponding to image sequences ( in the order shown but excluding fibronectin sequences; as this is shown in Videos 1 , 2 ) ( Videos 12\\u201317 , respectively ) . Scale bars = 20 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00710 . 7554/eLife . 11384 . 008Video 1 . H1299 P/L cell migration in the Discontinuous mode . High-resolution multiscale imaging of a single H1299 P/L cell expressing EGFP-paxillin ( green , CMAC marker ) and RubyRed-LifeAct ( red , F-actin marker ) during migration in the Discontinuous mode on 2 . 5 \\u00b5g/ml fibronectin . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00810 . 7554/eLife . 11384 . 009Video 2 . H1299 P/L cell migration in the Continuous mode . High-resolution multiscale imaging of a single H1299 P/L cell expressing EGFP-paxillin ( green , CMAC marker ) and RubyRed-LifeAct ( red , F-actin marker ) during migration in the Continuous mode on 2 . 5 \\u00b5g/ml fibronectin . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 00910 . 7554/eLife . 11384 . 010Video 3 . Parallel emergence of Discontinuous and Continuous modes in H1299 P/L cells . Low-resolution imaging of H1299 P/L cells labeled with a membrane dye shows the parallel emergence of both Discontinuous ( e . g . dark blue outline , trajectory shown ) and Continuous ( e . g . light blue outline , trajectory shown ) modes under uniform conditions . Cells have been segmented and tracked to highlight morphologies and trajectories . Note that two cells in the Continuous mode ( yellow and lilac outlines ) pass through the substrate region traversed by the Discontinuous mode cell ( dark blue ) , yet these cells remain in the Continuous mode . This implies that these migration modes are not simply determined by ( possible ) local variations in , for example , ECM substrate ( 5 \\u00b5g/ml fibronectin ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01010 . 7554/eLife . 11384 . 011Video 4 . Migration mode transition from Continuous to Discontinuous motility . Low-resolution imaging of H1299 P/L cells labeled with a membrane dye , during random migration on 5 \\u00b5g/ml fibronectin . Cells have been segmented and tracked to highlight morphologies and trajectories . Note that the cell with light green outline ( trajectory shown ) transitions from Continuous to Discontinuous migration during the course of imaging . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01110 . 7554/eLife . 11384 . 012Video 5 . Migration mode transition from Discontinuous to Continuous motility . Low-resolution imaging of H1299 P/L cells labeled with a membrane dye , during random migration on 5 \\u00b5g/ml fibronectin . Cells have been segmented and tracked to highlight morphologies and trajectories . Note that the cell with light green outline ( trajectory shown ) transitions from Discontinuous to Continuous migration during the course of imaging . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 012 Under the conditions described above , ~80% of observations showed cells migrating via the Discontinuous mode , with the remaining 20% moving in the Continuous mode ( Figure 1C , see 'Materials and methods' for sample numbers relating to all experimental data ) . Automated cell tracking ( see 'Materials and methods' , [Lock et al . , 2014] ) revealed that cells in the Discontinuous mode migrate significantly faster than during Continuous migration ( Figure 1D ) . By applying mean squared displacement analysis of cell trajectories to assess measures related to cell speed ( migration coefficient ) and directionality ( persistence time ) , we found that Discontinuous migration is less directionally stable at any given speed ( Figure 1E ) . However , both modes of migration show a positive correspondence between speed and directionality , as recently reported ( Maiuri et al . , 2015 ) . Importantly , an extensive comparison of migratory behaviors in several additional cell lines ( Figure 1F , Figure 1\\u2014figure supplement 3 and Videos 6\\u201311 ) and during H1299 cell migration on alternative extracellular matrix ligands ( Figure 1G , Figure 1\\u2014figure supplement 4 and Videos 12\\u201317 ) indicated that Discontinuous and Continuous migration modes are consistently recurring phenomena . 10 . 7554/eLife . 11384 . 013Video 6 . BT549 cell migration in the Discontinuous mode . A single BT549 cell migrating in the Discontinuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01310 . 7554/eLife . 11384 . 014Video 7 . BT549 cell migration in the Continuous mode . A single BT549 cell migrating in the Continuous mode ( upper centre cell ) . Cropped from a larger image montage ( note , montage stitching can cause observable intensity boundaries in image; however , these have no effect on interpretation ) . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are\\u00a0not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01410 . 7554/eLife . 11384 . 015Video 8 . MDA-MB-231 cell migration in the Discontinuous mode . A single MDA-MB-231 cell migrating in the Discontinuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01510 . 7554/eLife . 11384 . 016Video 9 . MDA-MB-231 cell migration in the Continuous mode . MDA-MB-231 cells migrating in the Continuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01610 . 7554/eLife . 11384 . 017Video 10 . Hep3 cell migration in the Discontinuous mode . Hep3 cells migrating in the Discontinuous mode . Cropped from a larger image montage ( note , montage stitching can cause observable intensity boundaries in image; however , these have no effect on interpretation ) . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data is not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01710 . 7554/eLife . 11384 . 018Video 11 . HS578T cell migration in the Continuous mode . A single HS578T cell migrating in the Continuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 . 5 \\u00b5g/ml fibronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01810 . 7554/eLife . 11384 . 019Video 12 . H1299 P/L cell migration in the Discontinuous mode on laminin . H1299 P/L cells migrating in the Discontinuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 \\u00b5g/ml laminin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 01910 . 7554/eLife . 11384 . 020Video 13 . H1299 P/L cell migration in the Continuous mode on laminin . A single H1299 P/L cell migrating in the Continuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 50 \\u00b5g/ml laminin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 02010 . 7554/eLife . 11384 . 021Video 14 . H1299 P/L cell migration in the Discontinuous mode on collagen type 1 . H1299 P/L cells migrating in the Discontinuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 2 \\u00b5g/ml collagen type 1 . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 02110 . 7554/eLife . 11384 . 022Video 15 . H1299 P/L cell migration in the Continuous mode on collagen type 1 . H1299 P/L cells migrating in both Discontinuous and Continuous modes . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 5 \\u00b5g/ml collagen type 1 . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 02210 . 7554/eLife . 11384 . 023Video 16 . H1299 P/L cell migration in the Discontinuous mode on vitronectin . A single H1299 P/L cell migrating in the Discontinuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 1 \\u00b5g/ml vitronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 02310 . 7554/eLife . 11384 . 024Video 17 . H1299 P/L cell migration in the Continuous mode on vitronectin . A single H1299 P/L cell migrating in the Continuous mode . Cropped from a larger image montage . Cells labeled with a membrane dye during random migration on 1 \\u00b5g/ml vitronectin . Image intensity scale inverted and brightness linearly adjusted for visualization ( note: intensity data are not quantified ) . Time in minutes shown . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 024 Cell migration is the product of membrane dynamics that can be divided into protrusive and retractive processes ( Ridley , 2003 ) . We quantitatively compared the spatial and temporal dynamics of membrane protrusions and retractions between and within each migration mode based on defining membrane dynamics over the minimal imaging interval of 5 min ( Figure 2 ) ( Kowalewski et al . , 2015 ) . This revealed that protrusions share similar size ( area ) distributions in both modes ( Figure 2A ) , while retraction events are more extreme in size during Discontinuous migration , that\\u00a0is , more frequently very small or very large ( Figure 2B ) . When compared within\\u00a0each mode , membrane retractions have a broader size distribution than protrusions in the Continuous mode ( Figure 2C ) , although this is far more striking in the Discontinuous mode ( Figure 2D ) . To investigate membrane dynamics operating over longer time-scales , we calculated the probabilities of protrusions and retractions of a given size based on net cell shape/position changes occurring over intervals of between 1 and 15 image frames ( 5 to 75 min ) . This revealed a relatively unstructured pattern of dynamics in Continuous cells , although retractions tended to be smaller and protrusions larger ( Figure 2E ) . In contrast , Discontinuous migration consistently displayed a wide distribution of retraction sizes ( very small and very large ) , with protrusion sizes uniformly moderate ( Figure 2F ) . Notably , the relatively high probability of very large retraction events in Discontinuous cells corresponds with their observed tendency to undergo dramatic tail retraction events . Collectively , these results explain the spatial characteristics underlying stepwise or smooth cell motion during migration in Discontinuous or Continuous modes , respectively . 10 . 7554/eLife . 11384 . 025Figure 2 . Membrane protrusion and retraction dynamics are temporally decoupled and less coordinated during Discontinuous migration . ( A ) Probability distribution of membrane protrusion sizes ( \\u00b5m2 ) per 5-min interval during Discontinuous ( blue ) and Continuous ( orange ) migration . Subtraction of the Continuous from Discontinuous protrusion size probability distribution ( inset ) reveals no substantial or structured difference in these distributions ( probability difference , Y axis ) . ( B ) Probability distribution of membrane retraction sizes ( \\u00b5m2 ) per 5-min interval during Discontinuous ( blue ) or Continuous ( orange ) migration . Subtraction of the Continuous from Discontinuous retraction size probability distribution ( inset ) reveals that retraction sizes tend to be more extreme ( frequently small or large , rarely moderate ) in cells during Discontinuous migration ( probability difference , Y axis ) . ( C ) Subtraction of the protrusion from retraction size probability distribution ( 5 min intervals only ) during Continuous migration reveals little structure in probability differences . ( D ) Subtraction of the protrusion from retraction size probability distribution ( 5 min intervals only ) during Discontinuous migration reveals that retractions in this mode are consistently more extreme in size ( frequently small or large , rarely moderate ) than protrusions . ( E ) Surface plotting of probability differences ( retraction minus protrusion probability , per size ) over various time windows ( 1 to 15 frames; 5 to 75 min ) in cells during Continuous migration . Surface color-coding indicates where protrusions ( blue ) or retractions ( red ) are more common at a particular size . ( F ) Surface plotting of probability differences ( retraction minus protrusion probability , per size ) over various time windows ( 1 to 15 frames; 5 to 75 min ) in cells during Discontinuous migration . ( G ) Signal cross-correlation was calculated between protrusion size and retraction size fluctuations per cell during Continuous migration . Mean cross-correlation values ( Y axis , red = positive , blue = negative ) plotted as a surface , per time lag ( -12 to 12 frames , negative values indicate protrusion leads retraction , positive values indicate retraction leads protrusion ) , per time window ( ranging from 1 to 15 frames; 5 to 75 min ) . ( H ) Signal cross-correlation was calculated between protrusion size and retraction size fluctuations per cell during Discontinuous migration . Mean cross-correlation values ( Y axis , red = positive , blue = negative ) plotted as a surface , per time lag ( -12 to 12 frames , -ve values indicate protrusion leads retraction , +ve values indicate retraction leads protrusion ) , per time window ( ranging from 1 to 15 time points; 5 to 75 min ) . ( I ) Mean ( over all lags , per time window ) absolute cross-correlation values for Continuous ( orange ) and Discontinuous ( blue ) migration , /- 95% confidence intervals , n = number of cells ( see Materials and Methods ) . Statistical discernibility assessed by Friedman testing , p = 0 . 0012 . ( J ) Cells in each migration mode were divided into quintiles ( 20% bins ) based on instantaneous cell speed , and median protrusion areas /- 1 . 57 * interquartile range ( IQR , 25% to 75% ) / \\u221an ( approximates 95% confidence interval of the median , n = number of cell observations , see Materials and Methods ) were calculated per quintile . ( K ) Median retraction areas /- 1 . 57 * IQR/ / \\u221an ( n = number of observations ) were calculated per cell speed quintile . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 025 To understand the temporal dynamics underlying differences in migration mode behavior , we assessed the cross-correlation between protrusion and retraction size signals over time . This revealed strong temporal coordination ( high cross-correlation ) between protrusion and retraction events during Continuous migration , with a single cross-correlation peak around the -1 time-lag . This indicated that protrusion consistently led retraction , but that these dynamics are tightly coupled in time ( Figure 2G ) . This was true regardless of the time frame over which membrane dynamics were defined . In marked contrast , two cross-correlation peaks were apparent in the Discontinuous mode , with one peak stable around the -1 time-lag ( tightly coupled dynamics ) , and a second peak temporally offset with positive lags ( Figure 2H ) . This offset was dependent on the time frame used to define membrane dynamics , but clearly indicates a population of temporally decoupled membrane dynamics , wherein retraction leads protrusion events that are significantly delayed . This confirmed the visual impression of Discontinuous migration as a cyclical , stepwise process with asynchronous retraction ( i . e . tail retraction stage ) preceding protrusion ( i . e . lateral protrusion stage ) . In contrast , the morphological stability observed during Continuous migration is a consequence of the tight coupling of protrusion and retraction , producing motility without large-scale , transient size/shape changes . Visual impressions also suggested the Discontinuous migration mode to be less ordered/coordinated than the Continuous mode . The significantly lower average ( combining all time-lags ) protrusion-retraction cross-correlation values during Discontinuous migration ( Figure 2I ) confirmed reduced coordination of membrane dynamics in this migration mode . To complete our analysis of how membrane dynamics differentially contribute to migration in each mode , we assessed the relationship between cell speed and the size of membrane protrusions ( Figure 2J ) or retractions ( Figure 2K ) . Both modes exhibited similar dependencies . Remarkably , cell speed appears in each case to depend only slightly on the size of membrane protrusions , instead being largely determined by retraction size . The data described above detail significant quantitative differences between both the cell motion and membrane dynamics comprising each migration mode behavior . Yet , how these divergent behaviors emerge from the organization of underlying macromolecular machineries remains unclear . We therefore extended our image analyses ( see Materials and Methods and [Lock et al . , 2014] ) to derive a multivariate dataset of 55 organizational features ( detailed in Figure 3\\u2014figure supplement 1 ) defining cellular-scale morphology as well as the state ( e . g . size , number , density , morphology , localization ) and dynamics ( e . g . motion , stability , rates of area/density change ) of critical macromolecular-scale machineries driving cell migration , namely , cell-matrix adhesion complexes ( CMACs; demarcated by EGFP-paxillin ) and F-actin ( demarcated by RubyRed-LifeAct ) . This multiscale organizational data was measured per cell , per time-point , thereby complementing corresponding measures of migration behavior from the same individual cells ( Figure 3\\u2014figure supplement 1 ) . Such integrated data allowed exploration of the organizational states that give rise to particular cell migration behaviors , based on information embedded in natural cell heterogeneity ( Lock et al . , 2014; Lock and Str\\u00f6mblad , 2010 ) . To first understand how much overlap exists between the organizational states underpinning Discontinuous and Continuous migration modes , we performed supervised clustering ( canonical vectors analysis , CVA ) of all cell observations based on their organizational features only ( Figure 3A ) . Strikingly , the two migration mode populations showed almost no overlap , indicating that their underlying patterns of organization can be quantitatively distinguished . We next visualized these organizational states via parallel coordinate mapping of 15 key organizational features ( Figure 3B ) . This revealed specific instances of dissimilarity in feature values as well as an overview of the collective organizational signatures associated with each mode . To estimate the contribution of specific features to the divergence between modes , we ranked organizational features by the coefficient values assigned to them within the first canonical vector of the CVA ( Figure 3C ) . We then compared feature value distributions between modes for a subset of these top-ranked features , finding highly significant differences in all cases ( Figure 3D\\u2013K ) . These analyses help to define the spectrum of differences in organization that spontaneously arise in parallel with the emergence of Discontinuous and Continuous migration mode behaviors . For instance , cells in the Discontinuous mode are: far smaller; less round ( more protrusive ) ; have fewer , smaller , less dense CMACs ( based on EGFP-paxillin intensity ) with less F-actin association ( based on background-subtracted RubyRed-LifeAct intensity [Li et al . , 2010] ) and shorter lifetimes ( less stable ) , resulting in a drastically decreased total adhesion area . 10 . 7554/eLife . 11384 . 026Figure 3 . Discontinuous and Continuous migration modes reflect quantitatively distinct cell states with unique signatures of underlying organization . ( A ) Canonical vectors analysis ( CVA ) -based multivariate clustering ( based on all 55 organizational features ( see Figure 3\\u2014figure supplement 1 ) , canonical vectors ( CVs ) 1 and 2 displayed , percentages indicate proportion of total variance per CV ) of cell observations ( see Materials and Methods ) during Discontinuous ( blue ) or Continuous ( orange ) migration show modes to be quantitatively distinct . Large circles with black outlines indicate population centers of mass . ( B ) Parallel coordinate mapping of key organizational feature values ( per cell observation ) detail the multivariate signatures associated with cells during Discontinuous ( light blue ) or Continuous ( light orange ) migration . Points of difference between these multivariate signatures are emphasized by plotting of median values for Discontinuous ( blue ) and Continuous ( orange ) modes . Wilcoxon rank sum testing ( per feature ) assessed statistically discernable differences ( p < 0 . 001 ) , except where shown in gray italics . ( C ) A list of organizational features ranked by their contribution ( coefficient values in canonical vector 1 , which contains 99% of total variance ) to the separation of migration modes in the canonical vectors space shown in ( A ) . Gray backgrounds highlight features for which value distributions are compared between Discontinuous ( blue ) and Continuous ( orange ) modes in ( D-K ) . ( D-K ) Features compared indicated on boxplot Y-axes . Boxplots show median values and inter-quartile ranges ( IQR , 25% to 75% ) . Notches indicate median /- 1 . 57 * IQR/ / \\u221an ( approximates 95% confidence interval of the median , n = number of cell observations , as in ( A ) . P values reflect Wilcoxon rank sum testing as in ( B ) . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 02610 . 7554/eLife . 11384 . 027Figure 3\\u2014figure supplement 1 . Definition of organizational and behavioral features . This table defines the names , relevant measurement units , and a brief description of quantitative features derived through multiscale image analysis . Features are numbered 1-60 in correspondence with the order and numbering presented in Spearman's correlation heatmaps in Figure 4 . Note , of these 60 features , 55 define Organizational features ( light gray , used in multivariate clustering analyses ) , while 5 define the Behavioral features assessed herein ( dark gray ) . Additional terms are defined above relevant columns . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 027 Having defined a spectrum of differences in key organizational features underlying migration modes , we next explored how the correlative relationships between these features might also vary . To this end , we mapped the complete network ( including both organizational and behavioral features ) of pairwise inter-feature Spearman\\u2019s correlation coefficients ( rs ) for both Discontinuous ( Figure 4A ) and Continuous ( Figure 4B ) modes . Visual inspection of the heatmaps presented in 4A and 4B suggests that , while some differences in correlations are apparent , general patterns of correspondence are well preserved regardless of migration mode . This is emphasized by calculation of absolute differences in rs values between modes ( Figure 4C ) , wherein the majority of relationships appear virtually unchanged . Indeed , only a very limited number of inter-feature relationships change by more than 0 . 4 rs ( Figure 4D ) . This suggests that changes in inter-feature correlations are unexpectedly selective , given the magnitude of changes in the actual feature values themselves ( Figure 3 ) . To assess this , we compared the observed distribution of absolute differences in rs values to that which would be expected if changes arose randomly from the rs distributions depicted in Figure 4A and 4B ( Figure 4E ) . This confirmed that observed changes are much smaller and less frequent than randomly expected , thus supporting the proposal that reconfiguration of correlative relationships is highly selective . 10 . 7554/eLife . 11384 . 028Figure 4 . Differences in inter-feature relationships between Discontinuous and Continuous migration modes appear highly selective . ( A , B ) Heatmaps of Spearman's rank correlation coefficients ( rs ) for all pairwise combinations between 60 features ( 55 organizational and 5 behavioral ) , derived from cells during ( A ) Discontinuous or ( B ) Continuous migration . Individual correlation coefficient values are color-coded as depicted in the color bar ( blue = negative; red = positive; green = near zero ) . Numbers on X and Y axes ( 1 to 60 ) correspond to the identities of features , as defined in Feature Number column of Figure 3\\u2014figure supplement 1 . ( C ) A heatmap summarizes absolute differences in correlation coefficient values , per feature pair , between Discontinuous and Continuous migration modes . Difference values are color-coded as depicted in the color bar ( blue = no difference; red = large difference ) . ( D ) A histogram shows the frequency distribution of absolute differences in correlation coefficient values between Continuous and Discontinuous migration modes . ( E ) A plot of cumulative distribution functions ( CDFs ) comparing the observed distribution of absolute differences in correlation coefficient values ( between Discontinuous and Continuous modes as in [C] and [D] , black line ) and the differences in coefficient values obtained following randomized permutation of Spearman's correlation pairs ( gray line ) . The permuted distribution ( repeated 100 times for all relationships , all values included in CDF ) shows the expected distribution of coefficient differences if inter-feature correlation changes occur randomly . Comparison of observed and permuted CDFs suggests that observed differences in inter-feature Spearman's correlation values are far more selective than randomly expected . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 028 We next analyzed how organizational features correlated to cell speed in each migration mode . As exemplified in Figure 5A , and more extensively in Figure 5B , we found that a proportion of such correlations were essentially equivalent between modes ( e . g . Cell Area vs Cell Speed rs = negative in both modes; Cell Compactness \\u2212 Cell Speed rs = positive in both modes ) . However , a large number of cell speed correlations were detectable in only one of the two modes ( e . g . Median [CMAC Distance to Border] per Cell vs Cell Speed rs = positive only in Continuous mode; Median [CMAC Area] per Cell vs Cell Speed rs = positive only in Continuous mode ) . Even more salient , a substantial number of correlations to Cell Speed were inverted between modes ( e . g . Median [CMAC Lifetime] per Cell vs Cell Speed rs = negative in Discontinuous mode , positive in Continuous mode; Median [CMAC Average Trailing Edge Speed] per Cell vs Cell Speed rs = positive in Discontinuous mode , negative in Continuous mode ) . Such inversions in organizational feature-to-Cell Speed relationships hint at profound differences in the mechanisms controlling each mode . We also note that , more broadly , most organizational feature-to-Cell Speed correlations were both stronger and more positive during migration in the Continuous mode ( Figure 5C ) . This suggests that Continuous mode migration is more directly dependent on the state of CMAC and F-actin machineries , while alternative machineries may play more dominant roles during Discontinuous migration . 10 . 7554/eLife . 11384 . 029Figure 5 . Cell speed variation is coupled to distinct patterns of remodeling in underlying features and inter-feature relationships within each migration mode . ( A ) A selection of Spearman's correlation coefficient values ( rs ) for organizational feature relationships to cell speed are depicted ( color-coded as in color bar ) . Values reflect medians of populations values . Correlations are considered non-significant ( ns ) if zero is included in the range of the median +/\\u22121 . 57 * inter-quartile range ( IQR , 25% to 75% ) \\u221an ( approximates 95% confidence interval of the median , n = number of cell observations , see 'Materials and methods' ) . This highlights correlations to cell speed that: are equivalent between modes ( e . g . Cell Area , negative correlation in both modes ) ; exist in only one mode ( e . g . Median [cell-matrix adhesion complex ( CMAC ) Area] per Cell , ns in Discontinuous vs positive in Continuous ) ; or are opposite between modes ( e . g . Median [CMAC Lifetime] per Cell , negative in Discontinuous vs positive in Continuous ) . Examples in A are linked to an extensive analysis of cell speed-to-organizational feature correlations in B and C . ( B ) Parallel coordinate mapping of median cell speed-to-organizational feature correlations in Discontinuous ( blue ) and Continuous ( orange ) modes . Relationships are categorized by correlation value similarity or difference between modes , as indicated by the orange;\\u00a0C' ( denotes Continuous ) and blue 'D' ( denotes Discontinuous ) followed by '\\u2212'; ( negative rs ) or\\u00a0'+' ( positive rs ) or 'ns' ( non-significant rs ) . ( C ) Bar graphs depict the magnitude of differences in median cell speed-to-organizational feature correlation values from the Discontinuous to the Continuous mode . Correlations tend to be stronger and more positive in cells during Continuous migration . ( D ) Canonical vectors analysis ( CVA ) -based multivariate clustering ( using all 55 organizational features , canonical vectors [CV] 1 , 2 , and 3 displayed , percentages indicate proportion of total variance per CV ) of cell observations during: slow Discontinuous ( 0\\u201333 . 33% of Discontinuous migration speed values , blue ) ; medium Discontinuous ( 33 . 34\\u201366 . 66% , cyan ) ; fast Discontinuous ( 66 . 67\\u2013100% , green ) ; slow Continuous ( 0\\u201333 . 33% of Continuous migration speed values , yellow ) ; medium Continuous ( 33 . 34\\u201366 . 66% , orange ) ; or fast Continuous ( 66 . 67\\u2013100% , red ) migration . Two orientations of the same three-dimensional clustering are depicted ( upper and lower ) , revealing the separate clustering of each migration mode . Within each mode , progressive differences in cell speed correspond to similarly progressive variations in the position of observations within the multivariate organizational feature ( or cell state ) space . These speed-dependent trends in clustering define trajectories in the cell state space along which cells evolve as cell speed changes . Remarkably , these trajectories are completely distinct when comparing Discontinuous ( dashed lines ) and Continuous ( solid lines ) migration modes . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 029 We have thus far detailed a variety of specific differences between Discontinuous and Continuous migration modes in terms of behavior ( Figure 1 and Figure 2 ) , underlying organizational features ( Figure 3 ) and individual inter-feature correlations ( Figure 4 and Figure 5A\\u2013C ) . We next explored how these differences might translate into systemic dissimilarities in the coupling of cell behavior ( i . e . cell migration speed ) and cell organization within each mode . To this end , we first grouped cell observations according to both migration mode and tertiles of cell speed ( 0\\u221233% [slow]; 33\\u221266% [medium]; 66\\u2212100% [fast] ) . We then performed supervised CVA clustering of these groups ( Figure 5D ) . This enabled assessment of how cells in each mode cluster in the multivariate cell state space , but also of the trajectories defined through cell state space given variability in cell migration speed ( as described previously , [Lock et al . , 2014] ) within each mode . As also observed in Figure 3A ( based on similar but not equivalent clustering ) , we found that cells in Discontinuous or Continuous modes cluster separately . However , in a striking extension of this result , we also found that variability in migration speed within each mode corresponded to completely independent trajectories in cell state space . This means that different patterns of organizational remodeling are associated with varying cell speed within each mode . Thus , profound differences exist between modes in terms of the underlying strategies that produce and control cell migration speed . We have demonstrated how specific organizational feature values diverge in conjunction with the spontaneous emergence of Discontinuous and Continuous migration modes ( Figure 3 ) . Yet , it remains unclear whether these features are tightly coupled to migration mode identity , or only loosely correlated with these behaviors . We therefore assessed whether directed modulation of these features could drive corresponding and predictable changes in the frequency equilibrium between migration modes . To test for such bottom-up regulation , we applied a selection of targeted molecular perturbations for which we and others have established prior knowledge regarding their effects on the organizational features in question . These perturbations included: ECM ligand ( FN ) concentration modulation ( Gupton and Waterman-Storer , 2006 ) ; talin 1 depletion ( Kiss et al . , 2015 ) ; and ROCK inhibition-mediated reduction of actomyosin contractility ( Lock et al . , 2014; Kim and Wirtz , 2013; Hern\\u00e1ndez-Varas et al . , 2015 ) ( Figure 6A\\u2013C ) . Images exemplifying perturbation effects on cell , CMAC and F-actin morphology are presented in Figure 6\\u2014figure supplement 1 . Notably , these molecular perturbations collectively target four core regulatory mechanisms around the ECM \\u2013 adhesion \\u2013 F-actin axis , including integrin ligation; integrin activation/clustering; integrin-F-actin linkage; and actomyosin contractility ( Figure 6A ) . Given the pivotal importance of these mechanisms for mesenchymal migration , the effects of the specific molecular perturbations applied herein may be somewhat predictive of a wide array of related regulatory mechanisms . 10 . 7554/eLife . 11384 . 030Figure 6 . Cell-matrix adhesion and actomyosin contractility control the equilibrium between Continuous and Discontinuous migration modes . ( A ) Schematic summary of molecular mechanisms ( green ) associated with the ECM \\u2013 adhesion \\u2013 F-actin axis ( gray ) targeted here for perturbation to modulate implicated organizational features and , potentially , corresponding migration mode frequencies . Specifically , altering ECM ligand density ( fibronectin concentration , [FN] ) impacts mechanism 1 ) - integrin ligation and subsequent cell-matrix adhesion complex ( CMAC ) formation and maturation . RNAi-mediated knockdown of talin 1 limits both mechanism 2 ) - integrin activation and mechanism 3 ) - integrin-F-actin linkage , thereby also affecting CMAC formation , maturation and stability . Y-27632-mediated inhibition of ROCK disrupts mechanism 4 ) - actomyosin contractility , affecting F-actin dynamics and CMAC maturation . Images exemplifying the effects of each molecular perturbation are presented in Figure 6\\u2014figure supplement 1 . ( B ) Boxplots summarizing the response of the same selection of organizational features to: ECM ligand modulation ( 2 . 5 \\u00b5g/ml FN vs 10 \\u00b5g/ml FN , upper row ) ; talin 1 RNAi ( control siRNA vs talin 1 siRNA , 5 \\u00b5g/ml FN , middle row ) ; or ROCK inhibition ( DMSO vs 6 \\u00b5M Y-27632 , 5 \\u00b5g/ml FN , lower row ) . All boxplots in B show median values per condition and inter-quartile ranges ( IQR , 25% to 75% ) . Notches show the median +/\\u22121 . 57 * IQR/ / \\u221an ( approximates 95% confidence interval of the median , n = number of cell observations , see 'Materials and methods' ) . In each case , statistically discernable differences were assessed by Wilcoxon rank sum testing , with resulting p values <0 . 001 . ( C ) Migration mode frequency responses to each perturbation are depicted ( Discontinuous , blue; Continuous , orange ) . Additional conditions were included for ROCK inhibition ( low panel ) , showing a progressive response to 1 \\u00b5M , 2 \\u00b5M , and 6 \\u00b5M Y-27632 as compared to DMSO vehicle control . Note: in addition to depicting specific perturbation-dependent trends , the matrix of results presented in B and C is used , together with spontaneous feature variations depicted in Figure 3D , E , G\\u2013I , to logically parse organizational features that are consistently coupled , or just occasionally correlated , with migration mode identity . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 03010 . 7554/eLife . 11384 . 031Figure 6\\u2014figure supplement 1 . Comparison of cell , adhesion complex and F-actin morphologies following perturbations targeting the ECM \\u2013 adhesion \\u2013 F-actin axis . Confocal images of H1299 cells comparing: ( A ) low concentration ( 2 . 5 \\u00b5g/ml ) fibronectin ( FN ) -coated glass ( left ) vs high concentration ( 10 \\u00b5g/ml ) FN-coated glass ( right ) ; ( B ) control siRNA treatment ( left ) vs talin 1 siRNA treatment ( right ) ; and ( C ) control ( DMSO ) treatment ( left ) vs ROCK inhibitor ( Y-27632 , 6 \\u00b5M ) treatment ( right ) . EGFP-paxillin ( upper panels ) and RubyRed-LifeAct ( lower panels ) signals are shown to demarcate cell-matrix adhesion complexes ( CMACs ) and F-actin , respectively . Images are displayed with gray-scale inverted . ( D ) Talin 1 knockdown was confirmed by immunoblotting . Scale bars = 20 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 031 Importantly , we found that Continuous mesenchymal migration occurred upon high adhesion ( high FN concentration ) and with unperturbed actomyosin contractility , while Discontinuous migration occurred under conditions of low adhesion ( low FN concentration ) and inhibited contractility ( ROCK-inhibitor ) . Talin knock-down promoted Discontinuous migration ( Figure 6C ) . We then compared: i ) spontaneous differences in feature values between Continuous and Discontinuous migration modes ( displayed in Figure 3D , E , G and I ) ; ii ) perturbation-induced differences in the same features ( Figure 6B ) ; and iii ) perturbation effects on migration mode frequencies ( Figure 6C ) . From this matrix of results , we were able to determine whether a logical coherence was preserved over all these measures . For example , median CMAC Area was spontaneously higher in cells during Continuous migration ( Figure 3I ) , and increasing ECM ligand concentration ( FN ) causes higher CMAC Area values ( Figure 6B , upper left panel ) . It is therefore logically consistent ( if CMAC Area and migration mode behaviors are functionally coupled ) that raising ECM ligand concentration increases the frequency of the Continuous migration mode ( Figure 6C , upper panel ) . In fact , although the applied perturbations produce differing effects on CMAC Area ( increasing or decreasing ) , migration mode frequencies show logically consistent responses in each case . CMAC Area therefore appears likely to be closely coupled to migration mode identity , since there is no evidence that CMAC Area and migration mode identity could be uncoupled . Such coherence was also observed for CMAC Lifetime and Cell Compactness , thus similarly supporting their functional coupling to migration mode determination . In contrast , while the modulation of either ECM ligand concentration or talin 1 expression caused effects on Cell Area and CMAC Number that were consistent with their parallel effects on mode frequencies , ROCK inhibition caused contradictory results: CMAC number and Cell Area were both higher in cells in the Continuous mode ( Figure 3H ) and were increased following ROCK inhibition ( Figure 6B , lower right panel ) , yet ROCK-inhibition caused a decreased frequency of the Continuous mode ( Figure 6C , lower panel ) . These results imply that , although Cell Area and CMAC Number spontaneously correlate with migration mode identities under the original experimental condition , this correspondence can be uncoupled . These organizational features are therefore unlikely to be causal of , or caused by , migration mode identity determination . Taken together , the matrix of results presented delineates organizational features ( CMAC Area , CMAC Lifetime , Cell Compactness ) that may be directly coupled with migration mode determination . More definitively , these results effectively exclude Cell Area and CMAC Number from being ( linearly ) causally linked to this process . Furthermore , these results demonstrate the potent regulatory influence of the specific molecular components fibronectin , talin 1 and ROCK , on migration mode determination . These perturbations also allows us to conclude more broadly that cell-matrix adhesion complexes and actomyosin contractility play key roles in shaping Discontinuous and Continuous mode frequencies . The molecular perturbations described above changed both cellular and macromolecular organization , as well as the frequency equilibrium between Discontinuous and Continuous migration modes . However , it remained unclear to what extent perturbation effects on organizational features may reflect: a ) mode frequency changes ( inter-modal 'adaptive switching' ) , as opposed to b ) organizational remodeling of the modes themselves ( intra-modal 'adaptive stretching' ) . To address the balance between these two adaptive responses , we leveraged data from the comparison of low ( 2 . 5 \\u00b5g/ml ) and high ( 10 \\u00b5g/ml ) FN concentrations displayed in Figure 6 . Using only organizational features , we performed principal component analysis ( PCA ) -based unsupervised clustering of cell observations grouped by both FN concentration and migration mode ( Figure 7A ) . Surprisingly , this revealed that the distances ( in PCA-defined cell state space ) between Discontinuous and Continuous migration modes were far greater than the distances induced within each mode by FN concentration variation . Comparing the heterogeneity of individual cells over time ( Figure 7B and C , Figure 7\\u2014figure supplement 1 ) and the distances between migration mode centers of mass confirmed the substantial nature of average organizational differences between modes ( equal to or greater than the maximal span of variation observed over 8 hr in the individual cells shown in Figure 7B\\u2013C ) . To more intuitively visualize how changing FN concentration shifts the distribution of cell organization within the PCA-defined cell state space ( and between modes ) , we calculated the probability density distributions of cell observations in each mode , for each experimental condition ( Figure 7D\\u2013E ) . This revealed that cell observations in each mode populate high probability-density 'valleys' that are at least partially separated by low probability-density 'ridges' . This visualization also emphasized that the major response to FN modulation was a substantial rebalancing of cell organizational states from one valley ( migration mode ) to the other . Collectively , these observations demonstrate that switching ( between modes ) dominates over stretching ( remodeling within modes ) as an adaptive response to FN concentration modulation . The conceptual relation between these two mechanisms of adaptation in cell state space is schematized in Figure 7F . 10 . 7554/eLife . 11384 . 032Figure 7 . Adaptive switching between migration modes is the dominant perturbation response rather than intra-modal remodeling . ( A ) Principal component analysis ( PCA ) -based clustering of cell observations ( based on all 55 organizational features ) , principal components ( PC ) 1 and 2 displayed , percentages indicate proportion of total variance per PC , color-coded by migration mode and experimental condition: Discontinuous mode on 2 . 5 \\u00b5g/ml FN ( dark blue ) ; Discontinuous mode on 10 \\u00b5g/ml FN ( light blue ) ; Continuous mode on 2 . 5 \\u00b5g/ml FN ( red ) ; Continuous mode on 10 \\u00b5g/ml FN ( orange ) . Large black outlined circles indicate population centers of mass . Note that , despite FN variation exerting large effects on cell organization , differences between migration modes are much larger that differences between conditions within the same mode . ( B ) PCA clustering as in ( A ) , overlaid by the heterogeneous trajectories over time of 4 cells ( I\\u2013IV ) within the PCA-based multivariate state space . Cells I\\u2013IV belong to the spatially corresponding cell populations defined in ( A ) and are relatively constrained within their respective regions of the cell state space . ( C ) Single time point confocal images of EGFP-paxillin from Cells I\\u2013IV from ( B ) . Images are displayed with gray-scale inverted . Corresponding image sequence selections are presented in Figure 7\\u2014figure supplement 1 . Scale bars = 20 \\u00b5m . ( D ) A probability density map based on the coordinates in PCA space of cell observations from the 2 . 5 \\u00b5g/ml FN condition in ( A ) ( view orientation relative to A is rotated 180 degrees ) . Cells in the Discontinuous mode ( dark blue ) define a deep probability valley that partially overlaps with but is largely distinct from a shallow probability valley defined by cells in the Continuous mode ( red ) . The sum of probabilities equals 1 . ( E ) A probability density map based on the coordinates in PCA space of cell observations from the 10 \\u00b5g/ml FN condition in ( A ) ( view orientation relative to A is rotated 180 degrees ) . Cells in the Continuous mode ( orange ) define a deep probability valley that overlaps with but is partly distinct from a shallow probability valley defined by cells in the Discontinuous mode ( light blue ) . The sum of probabilities equals 1 . Centers of mass for each population in ( D ) and ( E ) are indicated by capped vertical lines of matching colors . Arrows in ( D ) and ( E ) signify how decreasing or increasing FN concentration , respectively , causes 'switching' from one probability valley ( migration mode ) to the other . ( F ) Schematic summary of the dual adaptive strategies employed by cells , with respect to mesenchymal migration . Modulation of intracellular or extracellular conditions can cause the remodeling of cellular and macromolecular organization locally within a given mode ( 'adaptive stretching' ) . However , the much larger and more frequent response to such perturbations is 'adaptive switching' between discrete migration modes , resulting in substantial rebalancing of Discontinuous and Continuous migration mode frequencies . ( G ) Dendogram ( qualitative ) indicating the major recognized archetypes of cell migration ( top row ) and the new modalities of Continuous and Discontinuous mesenchymal migration described herein ( bottom row ) . Italicized names in brackets correspond to similar/analogous migration modes/terms . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 03210 . 7554/eLife . 11384 . 033Figure 7\\u2014figure supplement 1 . Confocal image sequences of cells during Discontinuous and Continuous mode migration on glass coated with low or high fibronectin concentrations . Confocal image sequences of H1299 cells during migration on low concentration ( 2 . 5 \\u00b5g/ml ) fibronectin ( FN ) -coated glass ( A and B ) or\\u00a0high concentration ( 10 \\u00b5g/ml ) FN-coated glass ( C and D ) . EGFP-paxillin ( upper panels ) and RubyRed-LifeAct ( lower panels ) signals are shown to demarcate cell-matrix adhesion complexes ( CMACs ) and F-actin , respectively . Images are displayed with gray-scale inverted . Numbers denote time ( min ) . Scale bars = 20 \\u00b5m . DOI: http://dx . doi . org/10 . 7554/eLife . 11384 . 033\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"Results and Discussion\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 2,\n        \"samples\": [\n          \"As in [36] , we simulate a 100 kilobase region of human genome , contributing to a complex disease phenotype and fitness . The region evolves forward in time subject to neutral and deleterious mutation , recombination , selection , and drift . To perform genetic association and heritability estimation studies in silico , we need to impose a trait onto simulated individuals . In doing so , we introduce strong assumptions about the molecular underpinnings of a trait and its evolutionary context . How does the molecular genetic basis of a trait under natural selection influence population genetic signatures in the genome ? This question is very broad , and therefore it was necessary to restrict ourselves to a small subset of molecular and evolutionary scenarios . We analyzed a set of approaches to modeling a single gene region experiencing recurrent unconditionally-deleterious mutation contributing to a quantitative trait subject to Gaussian stabilizing selection . The expected fitness effect of a mutation is always deleterious because trait effects are sampled from an exponential distribution . Therefore , we do not allow for compensatory mutations that may occur in more general models of stabilizing selection . Specifically , we studied three different genetic models and two different demographic models , holding the fitness model as a constant . Parameters are briefly described in Table 1 . We implemented three disease-trait models of the phenotypic form P = G + E . G is the genetic component , and E = N ( 0 , \\u03c3 e 2 ) is the environmental noise expressed as a Gaussian random variable with mean 0 and variance \\u03c3 e 2 . In this context , \\u03c3 e 2 should be thought of as both the contribution from the environment and from the remaining genetic variance at loci in linkage equilibrium with the simulated 100kb region . The genetic models are named the additive co-dominant ( AC ) model , multiplicative recessive ( Mult . recessive; MR ) model and the gene-based recessive ( GBR ) model . The MR model has a parameter , h , that controls the degree of recessivity; we call this model the complete MR ( cMR ) when h = 0 and the incomplete MR ( iMR ) when 0 \\u2264 h \\u2264 1 . Here , h = 1 corresponds to co-dominance , which is different from the typical formulation used when modeling the fitness effects of mutations directly . It is also important to note that here recessivity is being defined in terms of phenotypic effects; this may be unusual for those more accustomed to dealing directly with recessivity for fitness effects . An idealized relationship between dominance for fitness effects and trait effects of a mutation on an unaffected genetic background is shown in S15 Fig . The critical conceptual difference between recessive models is whether dominance is a property of a locus ( nucleotide/SNP ) in a gene or the gene overall . Mathematically , this amounts to whether one first determines diploid genotypes at sites ( and then multiplies across sites to get a total genetic effect ) or calculates a score for each haplotype ( the maternal and paternal alleles ) . For completely co-dominant models , this distinction is irrelevant , however for a model with arbitrary dominance one needs to be more specific . As an example , imagine a compound heterozygote for two biallelic loci , i . e . genotype Ab/aB . In the case of traditional multiplicative recessivity the compound heterozygote is wild type for both loci and therefore wild-type over all; this implies that these loci are in different genes ( or independent functional units of the same gene ) because the mutations are complementary . However , in the case of gene-based recessivity [36] , neither haplotype is wild-type and so the individual is not wild-type; the failure of mutant alleles to complement defines these loci as being in the same gene [35] . For a diploid with mi causative mutations on the ith haplotype , we may define the additive model as G A C = \\u2211 i = 1 2 \\u2211 j = 1 m i c i , j , ( 1 ) where ci , j is the effect size of the jth mutation on the ith haplotype . Each ci , j is sampled from an exponential distribution with mean of \\u03bb , to reflect unconditionally deleterious mutation . In other words , when a new mutation arises its effect c is drawn from an exponential distribution , and remains constant throughout its entire sojourn in the population . The GBR model is the geometric mean of the sum of effect sizes on each haplotype [36] . We sum the causal mutation effects on each allele ( paternal and maternal ) to obtain a haplotype score . We then take the square root of the product of the haplotype scores to determine the total genetic value of the diploid . G G B R = \\u2211 j = 1 m 1 c 1 , j \\u00d7 \\u2211 j = 1 m 2 c 2 , j ( 2 ) Finally , the MR model depends on the number of positions for which a diploid is heterozygous ( mAa ) or homozygous ( maa ) for causative mutations , G M R = \\u220f j = 1 m A a ( 1 + h c j ) \\u220f j = 1 m a a ( 1 + 2 c j ) - 1 . ( 3 ) Thus , h = 0 is a model of multiplicative epistasis with complete recessivity ( cMR ) , and h = 1 closely approximates the additive model when effect sizes are small . Here , phenotypes are subject to Gaussian stabilizing selection with an optimum at zero and standard deviation of \\u03c3s = 1 such that the fitness , w , of a diploid is proportional to a Gaussian function [41] . w = e - P 2 2 \\u03c3 s 2 ( 4 ) The AC and MR models draw no distinction between a \\u201cmutation\\u201d and a \\u201cgene\\u201d ( as discussed in [36] ) . The GBR is also a recessive model , but recessivity is at the level of a haplotype ( or allele ) and is not an inherent property of individual mutations ( see [36] for motivation of this model ) . Viewed in light of the traditional AC and MR models , the recessivity of a site in the GBR model is a function of the local genetic background on which it is found . Based on several qualitative comparisons we find that the GBR model is approximated by iMR models with 0 . 1 \\u2264 h \\u2264 0 . 25 . However , no specific iMR model seems to match well in all aspects . The demographic models are that of a constant sized population ( no growth ) and rapid population expansion ( growth ) . The use of the MR model is inspired by Risch\\u2019s work [30 , 31] , linking a classic evolutionary model of multiple loci interacting multiplicatively [42 , 43] to the the genetic epidemiological parameter relative risk . Risch and Merikangas [44] used this model to calculate the power to detect causal risk variants as a function of their frequency and effect size . Pritchard extended Risch\\u2019s model to consider a trait explicitly as a product of the evolutionary process [17] . Pritchard\\u2019s work demonstrated that the equilibrium frequency distribution suggested an important role for rare deleterious mutations when a trait evolves in a constant sized , randomly mating population with recurrent mutation and constant effect sizes . However , multiplicative epistasis is only one model of gene action . Exploring the effect of different genotype-to-phenotype models on the population and quantitative genetic properties of complex traits is the focus of the current work . The amount of narrow sense heritability , h2 = ( VA ) / ( VP ) , explained by variants across the frequency spectrum is directly related to the effect sizes of those variants [29] . Thus , this measure is an important predictor of statistical power of GWAS and should inform decisions about study design and analysis [45] . Empirically , SNP-based estimates of heritability have inferred negligible dominance variance underlying most quantitative traits [27] . We have a particular interest in the amount of additive variance , VA , that is due to rare alleles and how much of genetic variance , VG , is attributable to VA under different recessive models . We follow the approach of [21] , by calculating the cumulative percent of VG explained by the additive effects of variants less than or equal to frequency x , ( VA;q \\u2264 x ) / ( VG ) . The product of this ratio and broad-sense heritability is an estimate of the narrow-sense heritability , h2 . This calculation is a population-wide equivalent to a SNP-based estimate of heritability in a population sample . In addition we calculate the same distribution for dominance effects ( VD;q \\u2264 x ) / ( VG ) using the orthogonal model of [27] . Methods based on summing effect sizes [29] or the site frequency spectrum [21] would not apply to the GBR model , because the effect of a variant is not independent of other variants ( e . g . , there is intralocus epistasis ) . Therefore , we resort to a regression-based approach , where we regress the genotypes of the population onto the total genetic value as defined in our disease trait models ( see Material and Methods ) . In the limit of Hardy-Weinberg and linkage equilibrium , the regression estimates are equivalent to standard quantitative genetic estimates [29] ( S14 Fig ) . For consistency , we applied the regression approach to all models . Overall , these distributions are substantially different across genetic models , demographic scenarios and model parameters ( Fig 1 ) . Under the AC model , all of VG is explained by additive effects if all variants are included in the calculation; in Fig 1 the solid variance curves reach unity in the AC panel . Low frequency and rare variants ( q < 0 . 01 ) explain a large portion of narrow sense heritability ( 26%\\u201395% ) even in models without rapid population expansion . Further , the variance explained at any given frequency threshold increases asymptotically to unity as a function of increasing \\u03bb ( S4 Fig ) . While the total heritability of a trait in the population is generally insensitive to population size changes ( S1 Fig , see also [21 , 22 , 46] ) , rapid population growth increases the fraction of additive genetic variation due to rare alleles ( Fig 1 ) . Here , increasing \\u03bb corresponds to stronger selection against causative mutations , due to their increased average effect size . Recent work by Zuk et al . [24] , takes a similar approach and relates the allele frequency distribution directly to design of studies for detecting the role of rare variants . However , our findings contrast with those of Zuk et al . [24] and agree with those of Lohmueller [22] , in that we predict that population expansion will substantially increase the heritability , or portion of genetic variance , that is due to rare variants . Our results under the AC model agree with those of Simons et al . [21] , in that we find that increasing strength of selection , increasing \\u03bb in our work , increases the contribution to heritability of rare variants . However , under the GBR model and the cMR model the distribution of genetic variance over risk allele frequency as function \\u03bb is non-monotonic ( Fig 1 and S4 Fig ) . For all recessive models , we find that total VA is less than VG ( Fig 1 ) . For the MR models , all additional genetic variation is explained by the dominance variance component; in Fig 1 the dotted variance curves reach unity in the MR panels . As expected , genetic variation under the MR model with partial recessivity ( h = 0 . 25 ) is primarily additive [29 , 47] , whereas VG under the cMR model ( h = 0 ) is primarily due to dominance . The GBR model shows little dominance variance and is the only model considered here for which the total VG explained by VA+VD is less than the true VG for all \\u03bb . This can be clearly seen in Fig 1 where the dotted curves do not reach unity in the GBR panel . These observations concerning the GBR model are consistent with the finding of [27] that dominance effects of SNPs do not contribute significantly to the heritability for complex traits . Under the GBR model , large trait values are usually due to compound heterozygote genotypes ( e . g . , Ab/aB , where A and B represent different sites in the same gene ) [36] . Therefore , the recessivity is at the level of the gene region while the typical approach to estimating VA and VD assigns effect sizes and dominance to individual mutations . Thus , compound heterozygosity , which is commonly observed for Mendelian diseases ( see [36] and references therein ) would be interpreted as variation due to interactions ( epistasis ) between risk variants . Importantly , the GBR model assumes that such interactions should be local , occurring amongst causal mutations in the same locus . While the GBR model is reflective of the original definition of a gene in which recessive mutations fail to complement , we emphasize that this does not imply that mutations are necessarily exomic . The GBR model is of a general genomic region in which mutations act locally in cis to disrupt the function of that region with respect to a phenotype . The increase in the number of rare alleles due to population growth is a well established theoretical and empirical result [48\\u201361] . The exact relationship between rare alleles [4 , 17 , 26 , 62 , 63] , and the demographic and/or selective scenarios from which they arose [21 , 22 , 64] , and the genetic architecture of common complex diseases in humans is an active area of research . An important parameter dictating the relationships between demography , natural selection , and complex disease risk is the degree of correlation between a variants effect on the disease trait and its effect on fitness [18 , 20\\u201322] . In our simulations , we do not impose an explicit degree of correlation between the phenotypic and fitness effects of a variant . Rather , this correlation is context dependent , varying according to the current genetic burden of the population , the genetic background in which the variant is present and random environmental noise . However , if we re-parameterized our model in terms of [18] , then we would have \\u03c4 \\u2264 0 . 5 ( Gaussian function is greater than or equal to its quadratic approximation ) , which is consistent with recent attempts at estimating that parameter [20 , 65] . Our approach is reflective of weak selection acting directly on the complex disease phenotype , but the degree to which selection acts on genotype is an outcome of the model . While the recent demographic history has little effect on key mean values such as broad-sense heritability of a trait or population genetic burden ( S1 and S3 Figs ) , the structure of the individual components in the population which add up to those mean values varies considerably . The specific predictions with respect to the composition of the populations varies drastically across different modeling approaches . It is therefore necessary to carefully consider the structure of a genetic model in a simulation study . The conclusions reached here also hold when we consider more complex demographic scenarios relevant to human populations . Under the demographic model for European populations from [40] , the additive and GBR models show the same behavior as in Fig 1 ( S17 Fig ) . At all key time points where population size changes , VA = VG for the additive model , and the variance explained by rare mutations depends primarily on \\u03bb ( S17 Fig ) . For the GBR model , VA < VG ( as in Fig 1 ) , and plateaus at the same ratio VA/VG for all time points except immediately after the bottleneck , which results in a short-lived increase in VA/VG that is undetectable by the time growth begins ( S17 Fig ) . All recessive models ( GBR , iMR and cMR ) may show a transient increase in total VG after the bottleneck , depending on the value of \\u03bb ( S18 Fig ) . However , the GBR and iMR models with h > 0 . 25 showed a return to constant population size levels by the final time point . The changes in VA and VG under recessive models is likely due to the transfer of non-additive variation into VA during a bottleneck , which has been studied thoroughly in the theoretical literature [66 , 67] . As in Fig 1 , the genetic model , and not the demographic details , drive the relationship between mutation frequency and additive genetic variance . In agreement with existing literature , site based recessive models show complex dynamics during bottlenecks and population expansion ( S18 and S19 Figs ) . However , with respect to load , the GBR model behaves more like a codominant model and is largely insensitive to changes in population size ( S18 and S19 Figs ) . Thus , complex traits evolving under the GBR model are not expected to show large differences in load between extant human populations . The previous section shows that the relationship between genetic variance and allele frequency in the entire population strongly depends on the genetic model . Recent estimates of variance components from large population samples of unrelated individuals have inferred that dominance variance ( VD ) is negligible for most traits [27] . However , a recent study of more than 104 Swedish twins and 18 traits obtained a contradictory result , inferring significant non-additive variance for most traits , which was interpreted as VD [68] . In this section , we show that this apparent inconsistency is expected under certain models of gene action . We applied GREMLd , MAF-stratified GREMLd ( MS-GREMLd ) , and MAF-stratified Haseman-Elston regression ( see Methods for details ) . We found MS-GREMLd to be numerically unstable on our simulated data , and thus we present results for non-MS-stratified GREMLd . The numerical stability issues likely resulted from some combination of small number of SNPs per region ( O ( 1000 ) ) , low total VG in a region , or high variance in effect sizes across causal mutations [69] . Further , for large \\u03bb , where VG is primarily due to rare alleles ( Fig 1 ) , heritability in a sample may not reflect heritability in the entire population ( S13 Fig ) . Fig 2 shows the GREMLd additive and dominance heritability estimates , as compared to the respective population value , over \\u03bb . Under the cMR model ( h = 0 ) , the dominance component is much larger than the additive component as predicted from Fig 1 . When GREMLd is performed on cMR model data after removing variants with MAF \\u2264 0 . 01 , as done in [27] , the total heritability estimate ( AD ) is quite accurate until \\u03bb \\u2265 0 . 25 where a downward bias is observed . As anticipated , GREMLd using unfiltered data yields results with a slight upward bias [70] . However , for the iMR ( h = 0 . 25 ) model the filtered GREMLd estimates are only accurate for \\u03bb < 0 . 1 reflecting the preponderance of rare causal variants for larger values of \\u03bb . Unfiltered GREMLd estimates under the iMR ( h = 0 . 25 ) model show a slight upward bias for small values of \\u03bb , but are otherwise accurate . This shows that GREMLd is performing as expected under the site-based model for which it is designed . The MS-HE regression results are generally consistent with the GREMLd results . The GREMLd and MS-HE estimates are accurate under the GBR model when \\u03bb is small , because most heritability is additive in that case ( Fig 1 ) . However , under the GBR model , both filtered and unfiltered GREMLd heritability estimates show downward bias when \\u03bb is large ( Fig 2 ) . The MS-HE regression results reveal a similar pattern , which indicates that the downward bias for large values of \\u03bb is not strictly due to removal of rare variants in the filtered GREMLd analysis . Instead , the bias shown for large values of \\u03bb is likely due to the presence of substantial non-additive heritability , which is not captured by the dominance effects of SNPs . In contrast to the variance component methods , our simulated large twin studies provide approximately unbiased estimates of total heritability for large values of \\u03bb , but were biased upward for small effect sizes under the AC and GBR models ( Fig 2 ) . The variance in twin-study estimates was quite large , possibly because only a single locus was simulated rather than the whole genome . Formally , twin studies estimate an additive and a non-additive component of variance and interpreting the non-additive component as epistatic or dominance variance is a matter of perspective . However , the GBR model is inspired by the definition of a gene as a physical region in which recessive mutations leading to the same phenotypic outcome fail to complement [35] , consistent with the allelic heterogeneity observed for human Mendelian disorders ( see [36] for further discussion ) . Thus , the model of recessivity at the level of the gene region is picked up as non-additive variance in twin studies , but missed by variance component methods ( GREML and HE regression ) because the dominance in the GBR model is due to Ab/aB ( compound heterozygotes ) genotypes rather than a/a genotypes ( homozygotes for a specific loss of function variant ) assumed by variance component methods . Thus the contradictory results of applying variance component methods [27] and analysis of large twin studies [68] in order to estimate VA and VD may be interpreted as evidence for a model of gene action such as the GBR , which may be viewed as either recessivity at the haplotype/gene level or intralocus epistasis at the level of causative mutations in a single gene region . Both interpretations are valid . The alternative explanation is that we must assert that one of the study designs is generating artifacts . Both demography and the model of gene action affect the degree to which rare variants contribute to the genetic architecture of a trait ( Fig 1 ) . However , the different mappings of genotype to phenotype from model to model make it difficult to predict a priori the outcomes of GWAS under each model . Therefore , we sought to explicitly examine the performance of statistical methods for GWAS under each genetic and demographic model . We assessed the power of a single marker logistic regression to detect the gene region by calculating the proportion of model replicates in which at least one variant reached genome wide significance at \\u03b1 \\u2264 10\\u22128 ( Fig 3A ) . The basic logistic regression is equivalent to testing for association under the AC model . We simulated both a perfect \\u201cgenotyping chip\\u201d ( all markers with MAF \\u2265 0 . 05 ) and complete re-sequencing including all markers ( Fig 3B ) . One of the most prominent feature of Fig 3 is the curvature of power as a function of \\u03bb . This reflects the competing forces of increasing average genetic effect and decreasing average allele frequency which occurs as \\u03bb increases ( S5 Fig ) . As \\u03bb increases , the total genetic variance explained by the locus increases until the model enters the House-of-cards [71] regime . At which point , the genetic variance is much less dependent on \\u03bb ( S1 Fig ) . When \\u03bb is large , however , the average allele frequency does continue to decrease ( S5 Fig ) which drives power down . Across all genetic models , the single marker logistic regression has less power under population expansion ( Fig 3A ) . The loss of power is attributable to a combination of rapid growth resulting in an excess of rare variants overall [48\\u201361] , and the increasing efficacy of selection against causal variants in growing populations [21] . While complete resequencing is more powerful than a gene-chip design , the relative power gained is modest under growth ( Fig 3A ) . Region-based rare variant association tests behave similarly with respect to population growth ( Fig 3B ) . There are important differences in the behavior of the examined statistical methods across genetic models . We focus first on the single marker tests ( Fig 3A ) . For gene-chip strategies , power increases for \\u201csite-based\\u201d models as recessivity of risk variants increases ( compare power for AC , iMR , and cMR models in Fig 3B ) . This increase in power is due to the well-known fact that recessive risk mutations are shielded from selection when rare ( due to being mostly present as heterozyogtes ) , thus reaching higher frequencies on average ( S5 Fig ) , and that the single-marker test is most powerful when risk variants are common [32] . Further , for the complete multiplicative-recessive model ( cMR ) , the majority of VG is due to common variants ( Fig 1 ) , explaining why resequencing does not increase power for this model ( Fig 3A ) . For single-marker tests , the GBR model predicts large gains in power under resequencing for intermediate \\u03bb ( the mean trait-effect size of newly arising causal mutations ) , similar to the AC or iMR model . But , when \\u03bb is larger power may actually be less under the GBR model than under AC or iMR . For all models , causal mutations are more rare with increasing \\u03bb ( S7 Fig ) . However , as a function of frequency , all VG may be attributed to VA or VD in the site-specific models whereas there is increasing intralocus epistasis in the GBR model as a function of \\u03bb ( Fig 1 ) . It is well-known that the single marker test has lower power when causal mutations have low frequencies , are poorly tagged by more common SNPs , or have small main effects [32 , 72] . Region-based rare variant association tests show many of the same patterns across genetic model and effect size distribution as single marker tests , but there are some interesting differences . The ESM test [36 , 73] is the most powerful method tested for the AC , iMR , and GBR models ( Fig 3b ) , with the c-Alpha test as a close second in some cases . For those models , the power of naive SKAT , linear kernel SKAT and SKAT-O , is always lower than the ESM and c-Alpha tests . This is peculiar since the c-Alpha test statistic is the same as the linear kernel SKAT test . The major difference between SKAT and ESM/c-Alpha is in the evaluation of statistical significance . SKAT uses an analytical approach to determine p-values while the ESM/c-Alpha tests use an explicit permutation approach . This implies that using permutation based p-values results in greater power . Yet , under the cMR model the linear kernel SKAT is the most powerful , followed by c-alpha . The cMR model does not predict a significant burden of rare alleles and so the default beta weights of SKAT are not appropriate , and the linear kernel is superior . The ESM test does poorly on this model because there are not many marginally significant low-frequency markers . It is logical to think that these tests would all perform better if all variants were included . The massive heterogeneity in the performance of region-based rare variant tests across models strongly suggests that multiple methods should be used when prior knowledge of underlying parameters is not available . In agreement with [22 , 74] , we predict that population growth reduces the power to associate variants in a causal gene region with disease status ( Fig 3 ) when the disease also impacts evolutionary fitness . We have recently released software to apply the ESM test to case control data [73] in order to facilitate applying this test to real data . It was noted by [4 , 26] , that an excess of rare significant hits , relative to empirical data , is predicted by AC models where large effect mutations contribute directly to fitness and the disease trait . We confirm that AC models are inconsistent with the empirical data ( Fig 4 ) , except when \\u03bb \\u2264 0 . 01 . The empirical data in Fig 4 represent a pooled data set with the same diseases and quality filters as in [26] , but updated to include more recent data . The data are described in S1 Table , and can be visualized alone more clearly in S16 Fig . Close to half of the data comes from GWAS studies uploaded to the NHGRI database after 2011 , yet the same qualitative pattern is observed . This contradicts the hypothesis that the initial observation of an excess of common significant hits relative to the prediction under an AC model was simply due to small sample sizes and low marker density in early GWAS previously analyzed in [4 , 26] . Yet the initial observation is in fact robust and the meta-pattern provides an appropriate point of comparison when considering the compatibility of explicit population-genetic models with existing GWAS data . The GBR model predicts few rare significant hits and an approximately uniform distribution across the remainder of MAF domain ( Fig 4 ) , even for intermediate and large values of \\u03bb . For smaller values of \\u03bb , the GBR predicts an excess of common significant hits . The more uniform distribution of significant single markers seen under the GBR is consistent with the flatter distribution of genetic variance ( Fig 1 ) . If one considers trying to determine an approximate dominance coefficient in the GBR model , it would be found that there is a distribution of coefficients across sites . Yet , when simulating iMR model , we find that an intermediate degree of dominance , h = 0 . 25 , results in distribution of significant hits which is similar to the GBR results ( Fig 4 ) . Most of the models fail a KS test comparing the simulated and empirical distribution of significant hits ( S21 Fig ) . The cMR ( h = 0 ) model shows a visual excess of intermediate frequency variants ( Fig 4 ) , but this does not result in rejection under the KS test ( S21 Fig ) which is largely insensitive to deviations in the tails . According to the KS test , the remaining models ( AC , GBR , iMR ) perform best when there are fewer data points in the simulated data due to low GWAS power . This suggests that all models would be rejected with enough replicates . We note that there is no compelling reason to expect any specific value of \\u03bb to be a particularly good fit to the empirical data . The empirical data are composed of genome-wide data for multiple traits . We feel that the mutational parameters , \\u03bb and mutation rate to causal variants , are likely to vary across the genome and across traits . Thus , the empirical data reflect a mixture of different underlying models and ascertainment schemes . The reason we emphasize this feature of the data is to demonstrate that models with rare alleles of large effect do not necessarily imply a visual excess of rare significant GWAS hits . In consideration of the rare allele of large effect hypothesis , [62] proposed a model where multiple rare alleles dominate disease risk and create synthetic associations with common SNPs . However , later it was shown that this particular model was inconsistent with GWAS theoretically and empirically [4 , 26 , 75] . Here , we have shown that there exist models in which rare alleles explain a substantial portion of heritability that are not inconsistent with findings from GWAS . We find that the MAF distribution of significant hits in a GWAS varies widely with choice of genetic model . In particular , we confirm the results of Wray et al . [26] , that AC evolutionary models predict an excess of low frequency significant hits unless trait effect sizes are quite small . Also , the cMR model predicts an excess of intermediate and common significant hits . Utilizing a GBR model or an iMR model with h = 0 . 25\\u22120 . 5 , reconciles this inconsistency by simultaneously predicting the importance of rare alleles of large effect and the correct allele frequency distribution among statistically significant single markers . Several empirical observations provide support for the presence of gene-based recessivity underlying variation for some complex traits in humans . The minor allele frequency distribution of significant GWAS hits is relatively flat [4 , 76] , which our results show is consistent with either the presence of small additive effect loci or gene-/site-based partially-recessive loci with intermediate to large effects ( Fig 4 ) . Models with loci of large additive effects predict an excess of rare significant hits . Oppositely , models with complete site-based recessivity predict an excess of common significant hits for all simulated mutation effect size distributions . SNP based estimates of dominance heritability are much lower than estimates of dominance from twins [27 , 68] . Of the models we explored , only the gene-based recessive model with intermediate to large effects is consistent with the difference between twin and SNP based estimates of dominance variance ( Fig 2 ) . Under a site-based recessive model of partial recessivity ( e . g . h = 0 . 25 ) , there should be no significant difference between estimates of dominance variance from SNP and twin studies , provided that the statistical assumptions are met for both approaches ( Fig 2 ) . These results are complementary to the work by Zuk et . al [24] , who show that twin studies can over estimate heritability under a model with gene interactions . It now appears clear that the underlying genetic model does not have the same impact on SNP-based and family based study designs; an issue which should be further explored . Our findings also support a more thorough investigation into the importance of compound heterozygosity in the genetics of complex traits . However , it may be difficult to directly observe non-additive gene-level effects through analysis of individual SNP markers . Additionally , the genetic model appears to be important in the design and analysis of association studies . While changes in population size do affect the relationship between effect size and mutation frequency [48\\u201361] ( Fig 1 and S5 Fig ) , different mappings of genotype to trait value do this in radically different ways for the same demographic history ( Fig 1 ) . From an empirical perspective , our findings suggest that re-sequencing in large samples is likely the best way forward in the face of the allelic heterogeneity imposed by the presence of rare alleles of large effect . Resequencing of candidate genes [77\\u201380] and exomes [40 , 81\\u201386] in case-control panels have observed an abundance of rare variants associated with case status . Here we show that under a model of mutation-selection balance on the genic level , neither current single-marker nor popular multi-marker tests are especially powerful at detecting large genomic regions harboring multiple risk variants ( Fig 3 ) . However , we show that using permutations to derive p-values improves the power of SKAT [69] with a linear kernel ( equivalent test statistic to c-Alpha [38] ) . Similarly , another permutation based test , the ESM test [73] , has more robust power across demographic and genetic models ( Fig 3 ) . Conceptually , cis-effects arise naturally from the original definition of a gene in which mutant recessive alleles fail to complement [35] . We show that cis-effects within a locus , represented by the GBR model , can have an important impact on the population level architecture of a complex trait . This conclusion is important for future simulation studies as well as the interpretation of empirical data . It is important to note that despite our use of the term \\u201cgene-based\\u201d this model may apply to any functional genomic element in which there are multiple mutable sites affecting a trait in cis , not just to genes . From a theoretical perspective , our work motivates the development of a more generalized gene-based model to include arbitrary dominance and arbitrary locus size . Empirically , we find that the GBR model is broadly consistent with a variety of observations from the human statistical genetics literature . Thus , there is an evident need for improved region-based association tests and the development of genetic variance component methods for haplotypes .\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"Discussion\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 8,\n        \"samples\": [\n          \"We here present an integrated analytical approach and associated tools , designed to interrogate both spontaneous and induced heterogeneity in single-cell mesenchymal ( lamellipodial ) migration . By deploying these tools , we quantitatively demonstrate that mesenchymal migration is composed of two distinct sub-modalities , each with specific behavioral , organizational , and regulatory characteristics . This marks an important advance that may enhance future investigations of the mesenchymal migration archetype . In particular , interpretations of natural and/or experimentally induced variability will likely become more coherent and precise if data are disaggregated according to Discontinuous or Continuous modes , rather than being unintentionally aggregated across cells with fundamentally different properties and dependencies . Recognition of these modes may therefore facilitate a more accurate understanding of the heterogeneity , adaptability , and regulation of mesenchymal migration as a whole . Our initial classification of cells into Discontinuous and Continuous migration modes was based on observed behavioral differences . Crucially , the same distinctive behaviors emerged in parallel within a variety of cell lines and under a spectrum of conditions , thus supporting the robustness and broad relevance of these behaviors . Yet , given the similarity of Continuous and Discontinuous migrations to previous descriptions of keratocyte-like ( Barnhart et al . , 2015; Keren et al . , 2008 ) or fibroblast-like motility ( Abercrombie et al . , 1977; Theisen et al . , 2012 ) , respectively , the existence of these behaviors per\\u00a0se\\u00a0is not novel . Indeed , early observations even qualitatively suggest the co-existence of similar behaviors ( Lewis et al . , 1982 ) . However , we now present quantitative evidence of the spontaneous , parallel emergence of two distinct yet inter-convertible mesenchymal migration modes , wherein divergent couplings exist between behavioral ( speed ) and organizational characteristics . We thereby provide new insights into the discontinuous structure of heterogeneity within the broad mesenchymal migration archetype . Indeed , by differentiating between progressive ( intra-modal ) and discrete ( inter-modal ) forms of variation , we gained the capacity to compare the roles that these two mechanisms play in adaptive responses to perturbation . Surprisingly , this revealed that adaptive switching between modes is the dominant response to perturbation ( FN concentration modulation ) , with adaptive stretching ( remodelling ) within modes playing only a minor role . The quantitative distinction between Discontinuous and Continuous migration modes is based on systemic differences detected via multivariate clustering of data following visual classification . Importantly , given that cells were classified based on perceived differences in their behavioral dynamics , we excluded behavioral features and focused only on organizational features ( predominantly related to CMAC and F-actin status ) when addressing the question of separation between the modes . As a result , at least four lines of evidence support the discrete nature of Discontinuous and Continuous migration modes . First , supervised multivariate clustering ( canonical vectors analysis , CVA ) of mode-classified cell observations indicated near complete separation between Discontinuous and Continuous modes . The systemic organization underlying these migration modes is therefore , to a very large extent , quantitatively distinct . Second , although separation between modes was not complete when cell observations were clustered in unsupervised PCA space , the topology of probability-density distributions was consistent with the existence of subpopulations in cell state space . Precisely , the high probability-density 'valleys' near the center of mass of each mode contrast with the low probability-density 'ridges' at the boundaries between modes . Importantly , these probability-density distributions are not predetermined outcomes of such PCA clustering , since PCA is unsupervised with respect to mode identity . In fact , these topologies are suggestive of a system comprised of two attractor states ( Huang and Ingber , 2000 ) , whose basins of attraction diverge on either side of a set of potentially unstable or unfavorable configurations . The low probability of these boundary configurations effectively divides mesenchymal migration into two independent sub-modalities . Such attractor states have been described in relation to , for example: gene expression during development and tumorigenesis ( Huang et al . , 2009; Huang , 2009 ) ; cell signaling during motility ( Kim et al . , 2015 ) ; and even in the force-coupled dynamics of CMACs ( Hern\\u00e1ndez-Varas et al . , 2015 ) . Moreover , the attractor state concept appears analogous to the concept of prespecification originally proposed by Friedl to explain the robust , recurrent and yet switchable characteristics of mesenchymal and amoeboid migration modes ( Friedl , 2004 ) . Interestingly , the limited remodelling within Discontinuous and Continuous modes in response to FN modulation ( as opposed to mode switching ) suggests that these modes may be relatively inflexible , providing additional support for their proposed status as prespecified attractor states . Third , further indicating the discrete nature of Discontinuous and Continuous migration modes , CVA clustered cell observations grouped by both migration mode and cell speed were again largely distinct , supporting the validity of the original CVA clustering result . In addition , this approach also provided a fourth compelling line of evidence by revealing completely independent trajectories through cell state space defined by cells migrating at different speeds within each mode . Remarkably , this indicates that changes in cell speed within each mode were coupled to substantially different patterns of organizational remodelling . These systemic differences were composed in part by specific disparities in Spearman\\u2019s correlations between individual organizational features and cell speed within each mode . While a variety of differences were observed , most striking were instances where correlations to cell speed were inverted between modes , such as in the case of adhesion stability ( CMAC lifetime ) \\u2013 a critical parameter previously linked to causal regulation of cell speed ( Lock et al . , 2014 ) . Interestingly , when comparing modes , the independent patterns of cell speed-coupled reorganization arose despite only subtle differences across the entire network of inter-feature Spearman\\u2019s correlations . Indeed , these differences in correlation values were far more limited than would be expected at random , suggesting that a select few features and relationships may control migration mode identity . Taking into account each of these lines of evidence , we conclude that Discontinuous and Continuous migration modes are representative of more than simply cells with different organizational feature values . We propose that these modes in fact represent divergent , yet co-existing strategies of mesenchymal migration . Despite strongly embracing quantitative imaging-based research ( Lock and Str\\u00f6mblad , 2010; Le D\\u00e9v\\u00e9dec et al . , 2010; Masuzzo and Martens , 2015 ) , the cell migration field has seen remarkably little application of automated approaches to the classification of migrating cells . This is in contrast to the study of mitosis , where automated classification has already proven to be a powerful research tool ( Neumann et al . , 2010; Schmitz et al . , 2010 ) . It is likely that this reflects differences in the degree and structure of heterogeneity in these biological processes . For example , mitosis is a relatively stereotyped process - both morphologically and particularly in the terms of the temporal ordering of events . Both characteristics can increase the accuracy of automated classification approaches ( Held et al . , 2010 ) . While cell migration is also more ( e . g . Discontinuous mode ) or less ( e . g . Continuous mode ) a globally time-ordered process , the underlying events appear far less stereotypical . Given these challenges , we have employed a blinded manual classification approach in this study . However , we note that the CVA clustering results \\u2013 even excluding behavioral features ( as herein ) \\u2013 provide evidence that automated classification of migration modes may be feasible in future . This may significantly enhance subsequent quantitative analyses of mode organization , behavior , and regulation . Such automated classification may be in part based on our detailed spatiotemporal analysis of protrusive and retractive membrane dynamics within each mode . This revealed that the major differences between modes were: the size of membrane retraction events; the order and temporal coupling of protrusion and retraction events; and the coordination displayed between protrusive and retractive dynamics . Specifically , cells in the Discontinuous mode produced many very small and occasional very large retraction events , with these retractions tending to lead protrusion , reversing the order detected in Continuous cells . A large time delay also exists between retraction and subsequent protrusion dynamics in Discontinuous cells , and these dynamics are less coordinated ( lower cross-correlation ) . Such substantial differences in membrane dynamics ( especially the inversion of protrusion and retraction orders ) further support the distinct nature of Discontinuous and Continuous migration modes . Furthermore , these analyses both confirmed and explained the visual impressions that originally differentiated Discontinuous from Continuous migration . In addition , given the strong correspondence between retraction ( but not protrusion ) size and cell speed , these analyses now add migration mode and cell speed to motility initiation as aspects of migration where membrane retraction appears to be the deterministic membrane process ( Barnhart et al . , 2015; Kowalewski et al . , 2015; Cramer , 2010 ) . Having quantitatively established the discrete nature and behavioral characteristics of Discontinuous and Continuous migration modes , we sought to understand how their frequency balance is regulated . Importantly , to select candidate molecular regulators , we first considered the differences in organizational feature values that arose spontaneously between the modes . Then , given existing knowledge on how targeting of particular molecular components ( FN concentration [Gupton and Waterman-Storer , 2006]; talin 1 expression [Kiss et al . , 2015]; ROCK signaling [Lock et al . , 2014; Kim and Wirtz , 2013; Hern\\u00e1ndez-Varas et al . , 2015] ) impact these same features , we designed a series of perturbations intended to drive specific changes in these features , while also monitoring the behavioral balance between modes . This demonstrated that each of these molecular factors plays a key role in shaping the equilibrium between modes . Furthermore , these molecular targets also regulate generic characteristics of the pivotal ECM \\u2013 adhesion \\u2013 F-actin axis , including: integrin ligation ( FN concentration ) ; integrin activation and clustering ( talin expression ) ; integrin \\u2013 F-actin linkage ( talin expression ) ; and actomyosin contractility ( ROCK signaling ) . Hence , the effects of these specific perturbations may also be illustrative of how generalized regulatory mechanisms impact the migration mode equilibrium . Taking this perspective , we observe that while Continuous migration arises under conditions of high adhesion and full contractility , Discontinuous migration is preferred given low adhesion and inhibited contractility . It is noteworthy that both modes emerge under conditions that are distinct from those that favour amoeboid migration ( i . e . low adhesion and high contractility ) . Therefore , the switch between Continuous and Discontinuous ( mesenchymal ) migration modes clearly contrasts to the previously identified conversion between mesenchymal and amoeboid migration . This emphasizes the multifaceted adaptive capacity of migrating cells , which likely expands the contexts within which efficient migration is possible . This plasticity is highlighted by the growing variety of distinct migration modalities now recognized ( Figure 7G ) . Finally , by viewing differences in organizational feature values ( both spontaneous and perturbation-induced ) and mode frequencies as a logical matrix , we could identify organizational features that are coupled to migration mode identity under all observed conditions , as well as features that can be explicitly uncoupled . This provides a first step in parsing correlated features from those with significant causal influence over migration mode identity . Specifically , it is notable that CMAC Area and CMAC Lifetime were previously indicated to causally influence cell migration speed ( Lock et al . , 2014; Kim and Wirtz , 2013 ) , while Cell Compactness was causally downstream of cell speed ( Lock et al . , 2014 ) . Thus , the evidence that these features may be functionally coupled to the determination of migration mode identity is consistent with them being integral to migratory regulation . Conversely , we previously found that Cell Area showed no causal relationship to cell speed ( Lock et al . , 2014 ) , and this is again consistent with it being functionally uncoupled from the process of mode identity determination . Overall , three types of insight were revealed through these perturbation-based analyses , including: the regulatory roles of specific molecules ( fibronectin , talin and ROCK ) ; the putative influence of generic regulatory mechanisms; and the nature of links ( coupled vs uncoupled ) between migration mode identity and commonly measured macromolecular features . Nonetheless , much remains to be learned about the proximal and distal determinants of these migration modes . In conclusion , we have presented and applied a quantitative , imaging-based analytic approach to explore the heterogeneity that naturally emerges within the mesenchymal ( lamellipodial ) archetype of cell migration . We thereby characterized two quantitatively distinct migration modes , here termed Continuous and Discontinuous migrations , that co-exist within the broad mesenchymal migration archetype . We compared these migration modes in terms of: their motion and behavioral dynamics; the specific and systemic organization of key machineries ( CMACs and F-actin ) underlying their motility , and; differences in how behavior ( migration speed ) and organization co-evolve within each mode . Through targeted molecular perturbations , we defined both specific molecular and general mechanisms of bottom-up control over migration mode determination , while also beginning to parse potentially functional ( coupled ) relationships ( between features and modes ) from those that are correlated but non-functional ( uncoupled ) . Finally , our systemic analysis of adaptive responses to perturbation has revealed how discrete mechanisms ( inter-modal adaptive switching ) dominate over progressive remodeling ( intra-modal adaptive stretching ) . This study therefore emphasize the importance of distinguishing and comprehending Discontinuous and Continuous migration modes as a necessary precursor to understanding mesenchymal migration in its totality , while simultaneously providing tools and approaches that enable this endeavor .\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 72
        }
      ]
    }
  ],
  "metadata": {
    "accelerator": "GPU",
    "colab": {
      "gpuType": "V100",
      "machine_shape": "hm",
      "provenance": []
    },
    "kernelspec": {
      "display_name": "Python 3 (ipykernel)",
      "language": "python",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.11.4"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}